# Updated penetrance estimates for recurrent copy number variants - an improved definition and formula.

**DOI:** 10.1038/s41431-025-01948-0
**Authors:** ['ShuxiangGoh', 'TracyDudding-Byth', 'MarkPinese', 'Edwin PKirk']
**Journal:** European journal of human genetics : EJHG
**Year:** 
**Scraped from:** DOI.org
**Scraped by:** agent_b
**Scraping date:** 2025-10-21T10:54:28.872485

## Abstract

Many copy number variants (CNVs) are implicated in neurodevelopmental disability, but exhibit incomplete penetrance. The definition of penetrance is often unclear. In published literature, penetrance typically includes the background risk of disease, while clinicians tend to exclude risks unrelated to the genetic variant. We propose a more clinically relevant definition of penetrance and develop a new formula for this. These changes are applied to existing data sources to produce updated penetrance estimates. Our findings indicate that most CNVs studied have significantly lower penetrance than previously published. Eleven CNVs, previously described as low-penetrant, are recalculated as having a penetrance close to 0% for intellectual disability. These include 1q21.1 proximal duplications [RBM8A], 2q11.2 deletions [TMEM127], 2q13 proximal deletions and duplications [NPHP1], 6q16 duplications [SIM1], 13q12 deletions [CRYL1], 15q11.2 duplications [NIPA1, NIPA2], 15q13.3 duplications [CHRNA7], 16p12.2 duplications [CDR2], 16p13.11 duplications [MYH11] and Xp22.3 duplications [SHOX]. Previous estimates of CNV penetrance, which ranged from 10-40% have been recalculated as 1-10%. In conclusion, many previously published estimates of CNV penetrance are inflated. Re-evaluation of existing data reveals lower and more accurate penetrance estimates for intellectual disability. This has important implications for diagnosis, genetic counselling, and prenatal reporting of recurrent CNVs.

## Full Text

## Abstract
many copy number variants (cnvs) are implicated in neurodevelopmental disability, but exhibit incomplete penetrance. the definition of penetrance is often unclear. in published literature, penetrance typically includes the background risk of disease, while clinicians tend to exclude risks unrelated to the genetic variant. we propose a more clinically relevant definition of penetrance and develop a new formula for this. these changes are applied to existing data sources to produce updated penetrance estimates. our findings indicate that most cnvs studied have significantly lower penetrance than previously published. eleven cnvs, previously described as low-penetrant, are recalculated as having a penetrance close to 0% for intellectual disability. these include 1q21.1 proximal duplications [ rbm8a ], 2q11.2 deletions [ tmem127 ], 2q13 proximal deletions and duplications [ nphp1 ], 6q16 duplications [ sim1 ], 13q12 deletions [ cryl1 ], 15q11.2 duplications [ nipa1 , nipa2 ], 15q13.3 duplications [ chrna7 ], 16p12.2 duplications [ cdr2 ], 16p13.11 duplications [ myh11 ] and xp22.3 duplications [ shox ]. previous estimates of cnv penetrance, which ranged from 10–40% have been recalculated as 1–10%. in conclusion, many previously published estimates of cnv penetrance are inflated. re-evaluation of existing data reveals lower and more accurate penetrance estimates for intellectual disability. this has important implications for diagnosis, genetic counselling, and prenatal reporting of recurrent cnvs. similar content being viewed by others polygenic burden has broader impact on health, cognition, and socioeconomic outcomes than most rare and high-risk copy number variants article open access 01 february 2021 contribution of copy number variations to education, socioeconomic status and cognition from a genome-wide study of 305,401 subjects article 30 august 2024 genome sequencing broadens the range of contributing variants with clinical implications in schizophrenia article open access 01 february 2021 window.datalayer = window.datalayer || []; window.datalayer.push({ recommendations: { recommender: 'semantic', model: 'specter', policy_id: 'na', timestamp: 1761044065, embedded_user: 'null' } }); introduction penetrance describes how often a genetic change leads to a specific phenotype. penetrance is 100% or complete if the genetic change always results in a phenotype. if the phenotype does not always occur, it is termed incomplete penetrance. a non-pathogenic variant should be 0% penetrant. we demonstrate that published estimates of penetrance do not reflect this when using a common bayesian formula for penetrance. we present a revised definition of penetrance that is more clinically useful, along with a new mathematical formula to estimate penetrance. these updated concepts apply to all forms of genetic penetrance, including copy number variations (cnvs), single-nucleotide variants, and other genetic changes. although the exact wording may vary between sources, the commonly published definition of penetrance is [ 1 , 2 , 3 , 4 , 5 , 6 ]: the probability of manifesting a phenotype given that a specific genetic variant is present . in large studies, this is generally [ 3 , 4 , 5 , 7 , 8 , 9 , 10 , 11 , 12 , 13 , 14 , 15 ] calculated as follows: $${\rm{penetrance}}=p\left(d|g\right)=\frac{p\left({g|d}\right).p\left(d\right)}{p\left(g\right)}=\frac{p\left({g|d}\right).p\left(d\right)}{p\left({g|d}\right).p\left(d\right)+p\left({g|}{d}^{c}\right).p\left({d}^{c}\right)}$$ (formula 1) where p(d) is the probability of manifesting the phenotype ( d isease) (equivalently, the prevalence of the phenotype in the general population), p(g) is the probability of having the genetic change ( g enotype), p(d|g) is the probability of manifesting the phenotype given that the genotype is present, p(g|d) is the probability of having the genotype given that the phenotype is present, d c refers to not having the phenotype and p(g|d c ) is the probability of having the genotype given that the phenotype is absent. however, in clinical practice, most clinicians interpret penetrance as the probability that a genetic variant causes a specific phenotype. these represent two different definitions of penetrance, as the latter excludes individuals who have the phenotype due to unrelated causes. two studies [ 8 , 12 ] recognised this distinction and provided an alternate penetrance estimate that attempted to exclude individuals that incidentally shared the phenotype by subtracting the value p(d) from formula 1. we build on this approach by developing formula 2, which is more mathematically precise, to achieve the same outcome. penetrance estimates using formula 1 [ 3 , 4 , 7 , 9 , 10 , 11 , 13 , 14 , 15 , 16 , 17 ] or formula 2 (below) to estimate penetrance are most accurate when considering penetrance for a single pathology. penetrance for multiple phenotypes is not supported by the underlying mathematical framework of the formulas (supplementary method 3.1 ). given that the most common phenotype associated with recurrent cnvs is intellectual disability (id) or developmental delay [ 18 ], this study focuses on cnv penetrance for id. this study addresses six key issues that lead to more accurate penetrance estimates: 1) clarification of the definition of penetrance. 2) an updated formula reflecting the clarified definition. 3) limiting penetrance estimations to a single phenotype of id. 4) determining the prevalence of id. 5) combining these changes to determine updated id penetrance estimates for recurrent cnvs. 6) updating 95% confidence intervals for penetrance estimates. the net outcome is improved penetrance estimates for recurrent cnvs, with many cnvs demonstrating a lower penetrance for id than previously published. methods we propose the following new definition of penetrance : penetrance of a genetic variant is the probability of manifesting a phenotype due to having the genetic variant . the new definition differs from the earlier definition by excluding phenotypes incidental to the genetic variant, with concrete examples provided in supplementary method 1.5 . penetrance is similar to, but distinct from, the attributable fraction among the exposed, or the risk difference, which are commonly seen in other areas of statistics and medicine (supplementary method 1.4 ). a mathematical formula for calculating penetrance can be developed using two different (but equivalent) approaches. both methods begin by recognising that penetrance is the probability of having a phenotype given the presence of a specific genetic change, minus the probability that the phenotype is present due to another unrelated condition. both approaches also take into consideration the possibility of dual diagnoses that may contribute to the phenotype. the two methods use different approaches (supplementary methods 1.1 – 1.4 ), but resultin the same formula: $${\rm{penetrance}}=\frac{p\left(d\right)}{p\left(g\right)}\,.\,\frac{p\left({g|d}\right)\,-\,p\left(g\right)}{1-\,p\left(g\right)-p\left(d\right)+p\left(d\right).p\left({g|d}\right)}$$ (formula 2) p(d), p(g) and p(g|d) were described previously. it should be noted that the data required to calculate either the earlier (formula 1) or new (formula 2) version of penetrance are the same. therefore, data used to derive penetrance estimates using the earlier formula can be used with this new formula – no additional data is required to calculate updated penetrance estimates. most individuals in affected cohorts [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] selected for this study were children from high-income countries with id or developmental delay [ 3 , 4 , 7 , 10 , 14 , 15 , 17 ]. therefore, the parameter p(d) in formula 1 and formula 2, most accurately refers to the prevalence of paediatric id in high-income countries. this was taken to be 1.1% based on data from a systematic review [ 20 ] (supplementary methods 3.2 – 3.3 ). the inclusion criteria for cnvs in table 1 were (i) availability of published penetrance estimates for the cnv, (ii) the ability to verify these estimates using supplied data, (iii) id and/or developmental delay being present in most of the affected cohort (or can be assumed to be present – see supplementary methods 3.2 ) and (iv) the cnv’s primary phenotype includes, or has been suggested to include id. these methods are similar to that of an earlier systematic review [ 21 ], so our data is similar, modified only to remove individuals without id where possible (supplementary methods 2 , 3.1 , 3.2 ). affected cohort data in this study is pooled from 8 studies [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] that met the inclusion criteria listed above. the control cohort ( n = 269,885) is a subset of gnomad data which have been labelled by gnomad as controls, and each one with a relevant cnv for this study was highlighted in the supplemental in the systematic review [ 21 ]. many methods for determining 95% confidence intervals were considered. most [ 3 , 4 , 7 , 9 , 10 , 11 , 14 , 15 , 17 , 19 ] rely on assumptions of normality and symmetry of data, which were not appropriate for rare cnvs due to small datasets. we applied a bootstrap method from an earlier study [ 21 ], modified by running the simulation 10,000 times instead (code provided in supplementary file ). results a systematic review of id [ 20 ] identified 17 studies reporting the prevalence of paediatric id in high income countries. the prevalence ranged from 0.44 to 3.68% (supplementary methods 3.3 ), with a median prevalence of p(d) = 1.1% used in this study. alternate methods for determining the optimal value of p(d) were considered (supplementary methods 3 &amp; supplementary discussion 1 ) but were less accurate. table 1 reports updated penetrance estimates for id using this new approach, alongside estimates using the earlier penetrance formula [ 21 ]. data used to calculate these numbers are taken from a systematic review [ 21 ] and modified to remove individuals without id where possible (supplementary methods 2 ). a larger version of this table considers penetrance estimates using other values of p(d) (supplementary results 1 ). table 1 updated penetrance estimates of 83 recurrent cnvs. full size table table 1 demonstrates that penetrance for id for all cnvs are reduced, except for fully penetrant cnvs which remain fully penetrant. the difference is most noticeable for low penetrant cnvs, which demonstrate an approximate 5–10 fold reduction in penetrance estimates. cnv penetrance estimates are most accurate when interpreted as penetrance for id, rather than for any disease because (i) most datasets used to calculate cnv penetrance are primarily comprised of individuals with id (supplementary methods 3.2 ), (ii) id is the primary phenotype associated with most recurrent cnvs [ 18 ] and (iii) the formula for estimating penetrance is less reliable when calculating multiple phenotypes combined together (supplementary methods 3.1 ). some cnvs, such as the 17q12 [ hnf1b ] deletion, which causes renal cysts and diabetes, require a different affected cohort since id is not prominently associated with the cnv. penetrance estimates for these cnvs are excluded from table 1 , but provided in supplementary results 1 with additional notes. earlier penetrance estimates of 10%, 20% and 40% are re-estimated at 1%, 4% and 10% respectively. fully penetrant conditions remain fully penetrant. a shortcut for converting penetrance estimates from earlier publications to the new penetrance is provided in supplementary results 3 . it is useful to note that penetrance for id in the 17p12 [ pmp22 ] deletion for hereditary neuropathy with liability to pressure palsies and the reciprocal duplication for charcot marie tooth disease is 0%. this is a correct result, as these conditions are not associated with id [ 22 , 23 ]. this also provides supporting evidence that the datasets predominantly contain patients with id rather than phenotypes that might be associated with these two conditions. discussion using a revised penetrance formula with an improved understanding of its parameters resulted in lower penetrance estimates for many cnvs compared to previous publications [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 , 21 ]. reduced cnv penetrance estimates may affect clinical management of patients with these cnvs. reduced cnv penetrance estimates for id that are close to 0% suggest minimal or non-pathogenicity for id. improvements to penetrance estimations our study provides more accurate penetrance estimates due to two key improvements built on prior work [ 5 , 14 , 24 ] in this field. the first improvement involves clarifying the definition of penetrance, so that background risk is excluded. formula 1 (based on the earlier definition) includes the possibility that the genetic change is present but unrelated to the phenotype. formula 2 (based on the newly proposed definition) excludes the background rate of this phenotype. when cnv prevalence is similar in both affected and control groups, formula 1 will report 1–6% penetrance whilst formula 2 will report 0%. stated in another manner, formula 1 erroneously adds the chance that the control cohort has the phenotype, whilst formula 2 excludes this. the second improvement involves a recognition that neither formula 1 nor 2 are suited to calculate penetrance for multiple phenotypes (supplementary methods 3.1 ). whilst an ideal study should focus on a single well-defined condition (such as id), most studies [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] include participants with a variety of conditions (such as id, developmental delay, autism, and congenital malformations). nonetheless, if most individuals have id (or developmental delay which is often an early sign of id), then penetrance for id can still be approximated. the prevalence of id, p(d) was determined from a systematic review to be approximately 1.1% (supplementary methods 3 &amp; supplementary discussion 1 ). using this instead of p(d) = 3% [ 9 ], 4% [ 4 , 10 , 11 , 15 ], 5.12% [ 7 , 12 , 14 ], 5.3% [ 7 , 12 , 14 ], or 13% [ 19 ], led to an estimated 3–12 fold reduction in penetrance estimates for id, compared to penetrance estimates previously published. this change in lowering the value of p(d) to 1.1% requires justification. the affected cohort of three studies were clearly comprised entirely of individuals with id (or infants with developmental delay) [ 4 , 17 , 19 ], justifying the use of 1.1% for these studies. in other studies [ 3 , 7 , 10 , 14 , 15 ], affected cohorts were comprised mostly of individuals with id. supplementary methods 3 justifies using these datasets to approximate the penetrance of id. supplementary discussion 1 explores alternate approaches to penetrance estimation, before concluding on the approach presented here. the datasets are imperfect, so accurate penetrance estimates derived from these datasets are impossible. within these constraints, the value of 1.1% is more accurate than using any other value. the phenotype associated with a cnv different cnvs may be associated with different phenotypes. however, the data sources used to calculate penetrance are the same regardless of the cnv. this has led to confusion regarding the phenotype a cnv might be penetrant for. a key question arises as to whether the differences in penetrance estimates between this publication and earlier studies are due to a focus on id rather than other phenotypes. we argue that for most cnvs, the penetrance listed for id in table 1 using the new method is likely more accurate and clinically-meaningful, regardless of this distinction. we expand on this below, with further details in supplementary discussion 1 : for cnvs associated with a primary phenotype other than id, penetrance estimates are omitted from table 1 . examples include the 17q12 [ hnf1b ] deletion (renal cysts and diabetes) and the 1q21.1 [ rbm8a ] proximal deletion. neither penetrance for id, nor penetrance for another phenotype, can be reliably estimated from these datasets. a different dataset could be used to estimate penetrance for the non-id phenotype. for a cnv for which the primary phenotype is id, the new method is clearly more accurate due to an updated formula and more appropriate choice of the parameter p(d) = 1.1% to represent the prevalence of id. for this group, the presence or absence of other phenotypes for the cnv is irrelevant when it comes to choosing a value for p(d). this group constitutes the majority of the cnvs in table 1 . for cnvs that can cause id in some individuals and congenital malformations without id in other individuals, penetrance estimates presented here are still likely to be more accurate than previous estimates. this is because the affected cohort p(g | d) used to estimate penetrance of neurodevelopmental disorders and congenital malformations in cnvs are primarily composed of individuals with id (supplementary methods 3.2 ), but also includes some individuals without id. therefore, the phenotype “d” is mix of individuals highly biased towards id, supporting the use of 1.1% in the formula as the optimal parameter for the prevalence of the phenotype (supplementary methods 3 ). autism involves a non-homogenous mix of conditions with a definition that has changed significantly over time, whilst onset of schizophrenia is age-related. these issues make penetrance estimation using formula 1 or formula 2 challenging (supplementary methods 3.4 ). sometimes, the phenotype thought to be associated with a cnv may be incidental and its description in relation to the cnv may be due to ascertainment bias. consider a hypothetical cnv with 10% penetrance for id (and therefore 90% of individuals with this cnv are unaffected). multiple authors have assumed a background chance of disability being approximately 5% [ 3 , 4 , 7 , 10 , 11 , 14 , 15 , 17 ]. under these assumptions, 4.5% (5% of 90%) will have a different disability due to causes unrelated to the cnv (incidental phenotypes). individuals with this cnv may therefore have id (10%), other disabilities (4.5%) or be healthy (85.5%). case studies are biased towards those with a phenotype, so case studies may describe other disabilities in up to 30% (4.5/(10 + 4.5)%) of individuals with this cnv. this is higher than what most would consider as the background risk and can be mistaken for being part of the phenotype, but is clearly just an aspect of ascertainment bias. this theoretical example demonstrates how incidental pathology can be misattributed to low penetrant cnvs (with a rate of up to 30% in this example) due purely to ascertainment bias, when in reality, penetrance of these pathologies should be 0%. examples of changes to cnv penetrance seven cnvs are recalculated to have 0% penetrance for id, or have 95% confidence intervals which include 0%. these are 0.9 mb 2q11.2 [ tmem127 ] deletions, 0.2 mb 2q13 [ nphp1 ] proximal deletions and duplications, 0.13 mb 6q16 [ sim1 ] duplications, 0.2 mb 13q12 [ cryl1 ] deletions, 0.3 mb 15q11.2 [bp1-bp2] [ nipa1 , nipa2 ] duplications and 0.5 mb 16p12.2 [ cdr2 ] duplications (previously mismapped to 16p12.1). the prevalence of these cnvs in affected and control cohorts are similar. these 7 cnvs account for 1.8% of individuals in gnomad controls (supplementary result 2 ). four cnvs are calculated to have approximately 1% penetrance for id and 95% confidence intervals that include values ≤ 1%. these are 0.3 mb 1q21.1 [ rbm8a ] proximal duplications, 1.4 mb 15q13.3 [bp4-bp5] [ chrna7 ] duplications,1 mb 16p13.11 [ myh11 ] duplications and xp22.3 [ shox ] duplications. the 95% confidence intervals for these do not include 0% penetrance, suggesting pathogenicity and low (but statistically significant) penetrance. however, it is suggested that these results be interpreted conservatively, as incomplete penetrance may not be the correct explanation for the slight difference in prevalence of this cnv in affected compared to control datasets. differences in genetic ancestry [ 25 ], publication bias and other confounding variables in the way control and affected cohorts are selected may provide alternate explanations [ 21 ]. these four cnvs account for 0.5% of individuals in gnomad controls (supplementary results 2 ). three cnvs with 1–3% penetrance for id may have unclear pathogenicity using these methods. the slight enrichment of these cnvs in affected compared to control cohorts may represent susceptibility to id, but could also represent bias or noise in the dataset. this may differ from cnv to cnv. these cnvs are 0.3 mb 15q11.2 [bp1-bp2] [ nipa1 , nipa2 ] deletions, 0.5 mb 16p12.2 deletions [ cdr2 ] (previously mismapped to 16p12.1) and 0.2–0.3 mb 16p11.2 distal duplications [ sh2b1 ]. phenotypes associated with these cnvs have been published in case series of affected individuals. however, given the high prevalence of individuals with these variants in gnomad controls, it’s likely that many individuals with these cnvs are either healthy or have features that lie within a spectrum of human normal. the earlier example of an incidental phenotype with up to 30% penetrance being erroneously associated with a cnv, demonstrates how ascertainment bias can lead to misattribution of pathogenicity, and is particularly relevant for cnvs in this group. these 3 cnvs account for 0.4% of individuals in gnomad controls (supplementary results 2 ). some recurrent cnvs are rare and data is sparse. cnvs that are present in very few individuals can result in wide 95% confidence intervals for penetrance that span from 0 to 100%. penetrance for these cnvs using these methods are uncertain. more broadly, for any cnv, it may be more appropriate to interpret currently available data for penetrance as lying within a range of potential penetrance estimates (e.g., a 95% confidence interval), rather than as a fixed percentage (supplementary discussion 2 ). implications for clinical care these findings have multiple clinical implications. fourteen cnvs with penetrance close to 0% are seen in approximately 2.7% of individuals labelled by gnomad as controls. in contrast, approximately 3.1% of gnomad controls harbour at least one of the 83 cnvs in table 1 (supplementary results 2 ). equivalently, 14/83 cnvs in table 1 with little evidence for pathogenicity for id or other phenotypes make up the vast majority of recurrent cnvs in gnomad controls with published penetrance estimates, whilst the majority 69/83 of recurrent cnvs that are more penetrant are seen in only a minority of gnomad controls. choosing not to report these 14 cnvs will dramatically reduce the number of healthy individuals who are given a pathogenic cnv result, and dramatically reduce laboratory and clinician workload. reporting these 14 cnvs as pathogenic poses a risk of misattributing a patient’s phenotype to the cnv, or mislabelling a healthy individual. moreover, if a diagnosis is based on one of these cnvs, it may reduce the likelihood of the patient being offered further genomic testing. consequently, the practice of reporting these 14 common cnvs may compromise patient care by limiting a more comprehensive diagnostic evaluation in some countries or settings. large scale studies of some of these cnvs demonstrate that on average, people with some of these 14 cnvs may perform poorer in some cognitive tasks than those without [ 11 ]. however, it is unclear if all such individuals perform poorer, or if some perform poorly and others perform normally, which would also lead to similar graphs in studies that aggregate the results. given how common these 14 cnvs are, the lack of strong correlation with a non-intellectual phenotype and the inability to prove that they cause cognitive reduction in all individuals, it is worth considering that reporting it as pathogenic in an individual may be an example of false positive reporting for that individual. given that normal human intellect spans a large range from iq 70–130 and that there are likely many polygenic factors that contribute to this variable range, it may be more helpful to consider these 14 cnvs as part of a normal spectrum of human genetic changes rather than as pathogenic. a less stringent cut-off could consider 11 cnvs (seen in 2.3% of gnomad controls) instead of 14 cnvs (supplementary result 2 ). in the prenatal setting, identifying a cnv previously reported as being 5–10% penetrant and recognising that an accurate measure of penetrance is closer to 0%, may make the difference between a laboratory choosing to report the cnv or not to report. the choice not to report such cnvs, as suggested by some prenatal reporting guidelines [ 12 , 26 ], can make a significant difference to anxiety and the choices families make in pregnancy. choosing not to report these cnvs can also improve the quality of life for families, by reducing their time spent performing unnecessary research on the cnv. observations from other studies that are consistent with findings from this study our findings have the potential to resolve conflicting observations in the literature regarding pathogenicity of very low penetrant cnvs. for example, the 15q13.3 duplication [ chrna7 ] [bp4-bp5] has been reported as pathogenic [ 4 , 27 , 28 ] and as having little evidence for pathogenicity [ 29 ]. this study clarifies the paradox as being due to the misleading definition of penetrance and resolves it with formula 2. many recent studies suggest caution when assigning pathogenicity to low-penetrant cnvs, especially in the prenatal setting [ 12 , 26 , 30 ]. for example, it has been suggested that the 16p11.2 distal duplication [ sh2b1 ] be not reportable in the prenatal setting [ 26 ], despite its penetrance of 16% [ 21 ] which intuitively appears high enough to warrant reporting. revised penetrance estimates in this study show that penetrance for this cnv is closer to 2.7% (95% ci 1.4–4.4%). our study also raises the possibility that this apparent positive 2.7% penetrance estimate, could be due to other factors such as differences in genetic ancestry or imperfectly ascertained cohorts. therefore, this study provides a mathematical justification for some current published guidelines regarding not reporting low penetrant cnvs. some prospective studies of neonates with cnvs [ 31 , 32 ] demonstrate less evidence of neurodevelopmental disability on follow-up than one might expect based on earlier penetrance estimates. the lower penetrance estimates in this study provide a mathematical reason why this might be so. additionally, it is common in clinical practice to see many low penetrant cnvs being inherited from unaffected parents. this observation can be explained by the lower penetrance estimates published in this study. limitations in estimating penetrance using the bayesian formula penetrance estimates for cnvs provided in this study should be interpreted cautiously. the listed penetrance estimates are for id (because the data sources are biased towards id) rather than for penetrance of other phenotypes. pleiotropy of cnvs means some cnvs might have a higher penetrance for other phenotypes. accurate penetrance estimates also rely on large datasets containing affected and control individuals. while differences in cnv prevalence between these two groups are typically interpreted as evidence for incomplete penetrance of cnvs [ 3 , 4 , 5 , 7 , 8 , 9 , 10 , 11 , 12 , 14 , 15 , 16 , 17 , 19 ], other factors may explain these differences. these include cohort selection biases, age-related onset of symptoms, variable expressivity and methodological differences. in particular, the genetic makeup of affected and control cohorts can be challenging to compare because affected cohorts are typically recruited from one or two hospitals or geographic locations [ 3 , 4 , 10 , 14 , 15 , 17 ] whilst control cohorts are often drawn from diverse studies and locations [ 3 , 4 , 10 , 14 , 33 ]. this disparity raises the possibility that observed increases in cnv prevalence in affected cohorts could stem from differences in genetic ancestry or geography [ 25 ], rather than pathogenicity. the 3q29 [ dlg1 ] [ 21 ], 15q11.2 [ nipa1 , nipa2 ] [ 25 ], and 15q13.3 [ chrna7 ] [bp4-bp5] [ 21 , 25 ] cnvs may represent examples of this. penetrance estimates in this study are based on data that has been collected imperfectly. control cohorts are commonly comprised of adults from outpatient clinics who were enroled as affected individuals for various (non-neurodevelopmental) health issues such as cancer [ 3 , 4 , 14 ], hyperlipidemia [ 3 , 14 ], atherosclerosis [ 14 ], diabetes [ 15 ] or asthma [ 3 , 14 ]. therefore, control cohorts may include those with comorbid id. this can result in fully penetrant conditions being misinterpreted as incompletely penetrant. table 1 flags, with the symbol ‘b’, several cnvs in this category. the opposite may also be true of affected cohorts, particularly if there is no clinical information on the pathology request form for microarray. technological differences between the way affected and control cohorts are identified may also affect penetrance estimates. most cnvs are likely to be detected equally well using either microarray assemblies or cnv callers for exome data [ 34 ]. however, some cnvs may be identified by one method and not the other. if so, then a disparity in prevalence between affected cohorts (which use microarrays) versus the gnomad control cohort (which uses exomes) may lead to erroneously higher or lower penetrance estimates for that particular cnv. despite these potential confounding factors, a significant enrichment of a recurrent cnv in affected compared to control cohorts provides reasonable evidence for pathogenicity, though caution is advised as the aforementioned factors may cause over- or under-estimation of the true penetrance. finally, there is a blurry line between concepts of incomplete penetrance and variable expressivity. a cnv may reduce cognitive performance in some areas, but be insufficient to cause intellectual disability [ 11 ]. in this sense, calling the cnv non-penetrant for id could be misleading, as it does have an effect. on the other hand, if the effect size is small and the individual does not consider themselves impaired, it may be kinder in clinical practice to refer to their ability as lying within a spectrum of human normal, rather than focussing their attention on merely one of many polygenic factors affecting intellect. conclusion we propose an updated definition and formula for genetic penetrance that excludes the background rate of disease. if a genetic change does not predispose to a genetic illness, this genetic change should be considered 0% penetrant for that illness. previous definitions and formulas always result in penetrance &gt;0% due to inclusion of the background risk, which could be misleading for clinicians and patients. imprecise penetrance estimates have the potential to be detrimental by limiting diagnostic options for affected individuals. there is potential for harm in classifying low penetrant cnvs as “pathogenic”, when most individuals are either unaffected or so mildly affected that their symptoms fall within a spectrum of human normal. the updated penetrance estimates presented here may affect interpretation and reporting of cnvs, with implications for genetic counselling in both prenatal and postnatal settings. data availability all relevant data are listed within the paper and its supporting information files (supplementary discussion 1 &amp; supplementary results 1 ). code availability penetrance estimates and their 95% confidence interval can be obtained by running bootstrap_penetrance.m in octave v8.3.0. references carruth ed, young w, beer d, james ca, calkins h, jing l, et al. prevalence and electronic health record-based phenotype of loss-of-function genetic variants in arrhythmogenic right ventricular cardiomyopathy-associated genes. circ genom precis med. 2019;12:e002579. article pubmed pubmed central google scholar chen s, parmigiani g. meta-analysis of brca1 and brca2 penetrance. j clin oncol. 2007;25:1329–33. article pubmed google scholar cooper dn, krawczak m, polychronakos c, tyler-smith c, kehrer-sawatzki h. where genotype is not predictive of phenotype: towards an understanding of the molecular basis of reduced penetrance in human inherited disease. hum genet. 2013;132:1077–130. article cas pubmed pubmed central google scholar kirov g, rees e, walters jt, escott-price v, georgieva l, richards al, et al. the penetrance of copy number variations for schizophrenia and developmental delay. biol psychiatry. 2014;75:378–85. article cas pubmed google scholar vassos, collier e, holden da, patch s, rujescu c, st d, et al. penetrance for copy number variants associated with schizophrenia. hum mol genet. 2010;19:3477–81. article cas pubmed google scholar roberts jd, asaki sy, mazzanti a, bos jm, tuleta i, muir ar, et al. an international multicenter evaluation of type 5 long qt syndrome: a low penetrant primary arrhythmic condition. circulation. 2020;141:429–39. article cas pubmed pubmed central google scholar allach el khattabi l, heide s, caberg jh, andrieux j, doco fenzy m, vincent-delorme c, et al. 16p13.11 microduplication in 45 new patients: refined clinical significance and genotype-phenotype correlations. j med genet. 2018;0:1–7. google scholar chaste p, sanders sj, mohan kn, klei l, song y, murtha mt, et al. modest impact on risk for autism spectrum disorder of rare copy number variants at 15q11.2, specifically breakpoints 1 to 2. autism res. 2014;7:355–62. article pubmed pubmed central google scholar cosemans n, vandenhove l, vogels a, devriendt k, van esch h, van buggenhout g, et al. the clinical relevance of intragenic nrxn1 deletions. j med genet. 2020;57:347–55. article pubmed google scholar isles ar, ingason a, lowther c, walters j, gawlick m, stober g, et al. parental origin of interstitial duplications at 15q11.2-q13.3 in schizophrenia and neurodevelopmental disorders. plos genet. 2016;12:e1005993. article pubmed pubmed central google scholar kendall km, bracher-smith m, fitzpatrick h, lynham a, rees e, escott-price v, et al. cognitive performance and functional outcomes of carriers of pathogenic copy number variants: analysis of the uk biobank. br j psychiatry. 2019;214:297–304. article pubmed pubmed central google scholar maya i, perlman s, shohat m, kahana s, yacobsons, tennet,et al. should we report 15q11.2 bp1-bp2 deletions and duplications in the prenatal setting?.j clin med.2020;9:11. article google scholar mohan kn, cao y, pham j, cheung sw, hoffner l, ou zz, et al. phenotypic association of 15q11.2 cnvs of the region of breakpoints 1-2 (bp1-bp2) in a large cohort of samples referred for genetic diagnosis. j hum genet. 2019;64:253–5. article cas pubmed google scholar rosenfeld ja, coe bp, eichler ee, cuckle h, shaffer lg. estimates of penetrance for recurrent pathogenic copy-number variations. genet med. 2013;15:478–81. article cas pubmed google scholar tropeano m, howley d, gazzellone mj, wilson ce, ahn jw, stavropoulos dj, et al. microduplications at the pseudoautosomal shox locus in autism spectrum disorders and related neurodevelopmental conditions. j med genet. 2016;53:536–47. article cas pubmed google scholar al, shehhi m, forman eb, fitzgerald je, mcinerney v, krawczyk j, et al. nrxn1 deletion syndrome; phenotypic and penetrance data from 34 families. eur j med genet. 2019;62:204–9. article google scholar jønch ae, douard e, moreau c, van dijck a, passeggeri m, kooy f, et al. estimating the effect size of the 15q11.2 bp1-bp2 deletion and its contribution to neurodevelopmental symptoms: recommendations for practice. j med genet. 2019;56:701–10. article pubmed google scholar unique. rare chromosome &amp; gene disorder guides: rare chromosome disorder support group; 2024 available from: https://rarechromo.org/disorder-guides/ . martin cl, wain ke, oetjens mt, tolwinski k, palen e, hare-harris a, et al. identification of neuropsychiatric copy number variants in a health care system population. jama psychiatry. 2020;77:1276–85. article pubmed google scholar maulik pk, mascarenhas mn, mathers cd, dua t, saxena s. prevalence of intellectual disability: a meta-analysis of population-based studies. res dev disabil. 2011;32:419–36. article pubmed google scholar goh s, thiyagarajan l, dudding-byth t, pinese m, kirk ep. a systematic review and pooled analysis of penetrance estimates of copy-number variants associated with neurodevelopment. genet med. 2025;27:101227. article cas pubmed google scholar van paassen bw, van der kooi aj, van spaendonck-zwarts ky, verhamme c, baas f, de visser m. pmp22 related neuropathies: charcot-marie-tooth disease type 1a and hereditary neuropathy with liability to pressure palsies. orphanet j rare dis. 2014;9:38. article pubmed pubmed central google scholar chrestian n hereditary neuropathy with liability to pressure palsies. in: adam mp, feldman j, mirzaa gm, pagon ra, wallace se, bean ljh, et al., editors. genereviews(®). seattle (wa) 1993. benn pa. prenatal counseling and the detection of copy-number variants. genet med. 2013;15:316–7. article pubmed google scholar schultz lm, knighton a, huguet g, saci z, jean-louis m, mollon j, et al. copy-number variants differ in frequency across genetic ancestry groups. hgg adv. 2024;5:100340. cas pubmed pubmed central google scholar muys j, blaumeiser b, jacquemyn y, bandelier c, brison n, bulk s, et al. the belgian microarray prenatal (bemapre) database: a systematic nationwide repository of fetal genomic aberrations. prenat diagn. 2018;38:1120–8. article cas pubmed google scholar zhou d, gochman p, broadnax dd, rapoport jl, ahn k. 15q13.3 duplication in two patients with childhood-onset schizophrenia. am j med genet b neuropsychiatr genet. 2016;171:777–83. article cas pubmed pubmed central google scholar gillentine ma, schaaf cp. the human clinical phenotypes of altered chrna7 copy number. biochem pharmacol. 2015;97:352–62. article cas pubmed pubmed central google scholar clingen. 15q13.3 recurrent region (d-chrna7 to bp5) (includes chrna7 and otud7a): clingen dosage sensitivity curation page; 2018 [updated 10/5/2018. available from: https://dosage.clinicalgenome.org/clingen_region.cgi?id=isca-46295 . cai m, que y, chen x, chen y, liang b, huang h, et al. 16p13.11 microdeletion/microduplication in fetuses: investigation of associated ultrasound phenotypes, genetic anomalies, and pregnancy outcome follow-up. bmc pregnancy childbirth. 2022;22:913. article cas pubmed pubmed central google scholar muys j, jacquemyn y, blaumeiser b, bourlard l, brison n, bulk s, et al. prenatally detected copy number variants in a national cohort: a postnatal follow-up study. prenat diagn. 2020;40:1272–83. article pubmed google scholar smajlagić d, lavrichenko k, berland s, helgeland ø, knudsen gp, vaudel m, et al. population prevalence and inheritance pattern of recurrent cnvs associated with neurodevelopmental disorders in 12,252 newborns and their parents. eur j hum genet. 2021;29:205–15. article pubmed google scholar fu j, liao c, collins r, wang l, ben-isvy d, brand h, et al. rare coding cnvs from exome sequenced individuals in gnomad v4 gnomad: gnomad; 2023 [updated 1/11/2023. available from: https://gnomad.broadinstitute.org/news/2023-11-v4-copy-number-variants/ . babadi m, fu jm, lee sk, smirnov an, gauthier ld, walker m, et al. gatk-gcnv enables the discovery of rare copy number variants from exome sequencing data. nat genet. 2023;55:1589–97. article cas pubmed pubmed central google scholar download references acknowledgements we would like to extend our sincere gratitude to associate professor kate simms for her invaluable assistance in proofreading this manuscript. we would also like to thank dr rhys bowden for early contributions to some of the mathematical aspects of this work. funding open access funding enabled and organized by caul and its member institutions. author information authors and affiliations school of women and children’s health, university of new south wales, sydney, nsw, australia shuxiang goh, mark pinese &amp; edwin p. kirk new south wales health, sydney, nsw, australia shuxiang goh, tracy dudding-byth &amp; edwin p. kirk genetics of learning disability, newcastle, nsw, australia tracy dudding-byth university of newcastle, newcastle, nsw, australia tracy dudding-byth children’s cancer institute, sydney, nsw, australia mark pinese centre for clinical genetics, sydney children’s hospital, sydney, nsw, australia edwin p. kirk nsw health pathology randwick genomics laboratory, sydney, nsw, australia edwin p. kirk authors shuxiang goh view author publications search author on: pubmed google scholar tracy dudding-byth view author publications search author on: pubmed google scholar mark pinese view author publications search author on: pubmed google scholar edwin p. kirk view author publications search author on: pubmed google scholar contributions conceptualization: sg, ek; methodology: sg, ek; data curation: sg; formal analysis: sg; investigation: sg, ek; resources: sg; software: sg; supervision: ek, mp, td; validation: sg, ek; visualization: sg; writing-original draft: sg; writing-review &amp; editing: ek, mp, td. corresponding author correspondence to shuxiang goh . ethics declarations competing interests the authors declare no competing interests. ethics all data used in this study is publicly available. there are no other relevant ethical declarations. additional information publisher’s note springer nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. supplementary information supplementary materials supplementary table 6 &amp; 7 penetrance calculation rights and permissions open access this article is licensed under a creative commons attribution 4.0 international license, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the creative commons licence, and indicate if changes were made. the images or other third party material in this article are included in the article’s creative commons licence, unless indicated otherwise in a credit line to the material. if material is not included in the article’s creative commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. to view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/ . reprints and permissions about this article cite this article goh, s., dudding-byth, t., pinese, m. et al. updated penetrance estimates for recurrent copy number variants – an improved definition and formula. eur j hum genet (2025). https://doi.org/10.1038/s41431-025-01948-0 download citation received : 21 december 2024 revised : 19 august 2025 accepted : 11 september 2025 published : 15 october 2025 doi : https://doi.org/10.1038/s41431-025-01948-0 share this article anyone you share the following link with will be able to read this content: get shareable link sorry, a shareable link is not currently available for this article. copy shareable link to clipboard provided by the springer nature sharedit content-sharing initiative download pdf advertisement window.sn = window.sn || {}; window.sn.libs = window.sn.libs || {}; window.sn.libs.ads = window.sn.libs.ads || {}; window.sn.libs.ads.slotconfig = window.sn.libs.ads.slotconfig || {}; window.sn.libs.ads.slotconfig['right'] = { 'pos': 'right', 'type': 'article', 'path': 's41431-025-01948-0' }; window.sn.libs.ads.slotconfig['kwrd'] = 'clinical+genetics,cytogenetics,neurodevelopmental+disorders'; window.sn.libs.ads.slotconfig['subjmeta'] = '1405,208,2489,366,631'; explore content research articles reviews &amp; analysis news &amp; comment current issue collections follow us on twitter sign up for alerts rss feed about the journal journal information open access fees and funding about the editors about the partner contact for advertisers subscribe publish with us for authors &amp; referees language editing services open access funding submit manuscript window.datalayer = window.datalayer || []; window.datalayer.push({ page: { content: { fundingwidget: "true", } } }); search search articles by subject, keyword or author show results from all journals this journal search advanced search quick links explore articles by subject find a job guide to authors editorial policies european journal of human genetics ( eur j hum genet ) issn 1476-5438 (online) issn 1018-4813 (print) nature.com sitemap about nature portfolio about us press releases press office contact us discover content journals a-z articles by subject protocols.io nature index publishing policies nature portfolio policies open access author &amp; researcher services reprints &amp; permissions research data language editing scientific editing nature masterclasses research solutions libraries &amp; institutions librarian service &amp; tools librarian portal open research recommend to library advertising &amp; partnerships advertising partnerships &amp; services media kits branded content professional development nature awards nature careers nature conferences regional websites nature africa nature china nature india nature japan nature middle east privacy policy use of cookies your privacy choices/manage cookies legal notice accessibility statement terms &amp; conditions your us state privacy rights &copy; 2025 springer nature limited

## Introduction
penetrance describes how often a genetic change leads to a specific phenotype. penetrance is 100% or complete if the genetic change always results in a phenotype. if the phenotype does not always occur, it is termed incomplete penetrance. a non-pathogenic variant should be 0% penetrant. we demonstrate that published estimates of penetrance do not reflect this when using a common bayesian formula for penetrance. we present a revised definition of penetrance that is more clinically useful, along with a new mathematical formula to estimate penetrance. these updated concepts apply to all forms of genetic penetrance, including copy number variations (cnvs), single-nucleotide variants, and other genetic changes. although the exact wording may vary between sources, the commonly published definition of penetrance is [ 1 , 2 , 3 , 4 , 5 , 6 ]: the probability of manifesting a phenotype given that a specific genetic variant is present . in large studies, this is generally [ 3 , 4 , 5 , 7 , 8 , 9 , 10 , 11 , 12 , 13 , 14 , 15 ] calculated as follows: $${\rm{penetrance}}=p\left(d|g\right)=\frac{p\left({g|d}\right).p\left(d\right)}{p\left(g\right)}=\frac{p\left({g|d}\right).p\left(d\right)}{p\left({g|d}\right).p\left(d\right)+p\left({g|}{d}^{c}\right).p\left({d}^{c}\right)}$$ (formula 1) where p(d) is the probability of manifesting the phenotype ( d isease) (equivalently, the prevalence of the phenotype in the general population), p(g) is the probability of having the genetic change ( g enotype), p(d|g) is the probability of manifesting the phenotype given that the genotype is present, p(g|d) is the probability of having the genotype given that the phenotype is present, d c refers to not having the phenotype and p(g|d c ) is the probability of having the genotype given that the phenotype is absent. however, in clinical practice, most clinicians interpret penetrance as the probability that a genetic variant causes a specific phenotype. these represent two different definitions of penetrance, as the latter excludes individuals who have the phenotype due to unrelated causes. two studies [ 8 , 12 ] recognised this distinction and provided an alternate penetrance estimate that attempted to exclude individuals that incidentally shared the phenotype by subtracting the value p(d) from formula 1. we build on this approach by developing formula 2, which is more mathematically precise, to achieve the same outcome. penetrance estimates using formula 1 [ 3 , 4 , 7 , 9 , 10 , 11 , 13 , 14 , 15 , 16 , 17 ] or formula 2 (below) to estimate penetrance are most accurate when considering penetrance for a single pathology. penetrance for multiple phenotypes is not supported by the underlying mathematical framework of the formulas (supplementary method 3.1 ). given that the most common phenotype associated with recurrent cnvs is intellectual disability (id) or developmental delay [ 18 ], this study focuses on cnv penetrance for id. this study addresses six key issues that lead to more accurate penetrance estimates: 1) clarification of the definition of penetrance. 2) an updated formula reflecting the clarified definition. 3) limiting penetrance estimations to a single phenotype of id. 4) determining the prevalence of id. 5) combining these changes to determine updated id penetrance estimates for recurrent cnvs. 6) updating 95% confidence intervals for penetrance estimates. the net outcome is improved penetrance estimates for recurrent cnvs, with many cnvs demonstrating a lower penetrance for id than previously published. methods we propose the following new definition of penetrance : penetrance of a genetic variant is the probability of manifesting a phenotype due to having the genetic variant . the new definition differs from the earlier definition by excluding phenotypes incidental to the genetic variant, with concrete examples provided in supplementary method 1.5 . penetrance is similar to, but distinct from, the attributable fraction among the exposed, or the risk difference, which are commonly seen in other areas of statistics and medicine (supplementary method 1.4 ). a mathematical formula for calculating penetrance can be developed using two different (but equivalent) approaches. both methods begin by recognising that penetrance is the probability of having a phenotype given the presence of a specific genetic change, minus the probability that the phenotype is present due to another unrelated condition. both approaches also take into consideration the possibility of dual diagnoses that may contribute to the phenotype. the two methods use different approaches (supplementary methods 1.1 – 1.4 ), but resultin the same formula: $${\rm{penetrance}}=\frac{p\left(d\right)}{p\left(g\right)}\,.\,\frac{p\left({g|d}\right)\,-\,p\left(g\right)}{1-\,p\left(g\right)-p\left(d\right)+p\left(d\right).p\left({g|d}\right)}$$ (formula 2) p(d), p(g) and p(g|d) were described previously. it should be noted that the data required to calculate either the earlier (formula 1) or new (formula 2) version of penetrance are the same. therefore, data used to derive penetrance estimates using the earlier formula can be used with this new formula – no additional data is required to calculate updated penetrance estimates. most individuals in affected cohorts [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] selected for this study were children from high-income countries with id or developmental delay [ 3 , 4 , 7 , 10 , 14 , 15 , 17 ]. therefore, the parameter p(d) in formula 1 and formula 2, most accurately refers to the prevalence of paediatric id in high-income countries. this was taken to be 1.1% based on data from a systematic review [ 20 ] (supplementary methods 3.2 – 3.3 ). the inclusion criteria for cnvs in table 1 were (i) availability of published penetrance estimates for the cnv, (ii) the ability to verify these estimates using supplied data, (iii) id and/or developmental delay being present in most of the affected cohort (or can be assumed to be present – see supplementary methods 3.2 ) and (iv) the cnv’s primary phenotype includes, or has been suggested to include id. these methods are similar to that of an earlier systematic review [ 21 ], so our data is similar, modified only to remove individuals without id where possible (supplementary methods 2 , 3.1 , 3.2 ). affected cohort data in this study is pooled from 8 studies [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] that met the inclusion criteria listed above. the control cohort ( n = 269,885) is a subset of gnomad data which have been labelled by gnomad as controls, and each one with a relevant cnv for this study was highlighted in the supplemental in the systematic review [ 21 ]. many methods for determining 95% confidence intervals were considered. most [ 3 , 4 , 7 , 9 , 10 , 11 , 14 , 15 , 17 , 19 ] rely on assumptions of normality and symmetry of data, which were not appropriate for rare cnvs due to small datasets. we applied a bootstrap method from an earlier study [ 21 ], modified by running the simulation 10,000 times instead (code provided in supplementary file ). results a systematic review of id [ 20 ] identified 17 studies reporting the prevalence of paediatric id in high income countries. the prevalence ranged from 0.44 to 3.68% (supplementary methods 3.3 ), with a median prevalence of p(d) = 1.1% used in this study. alternate methods for determining the optimal value of p(d) were considered (supplementary methods 3 &amp; supplementary discussion 1 ) but were less accurate. table 1 reports updated penetrance estimates for id using this new approach, alongside estimates using the earlier penetrance formula [ 21 ]. data used to calculate these numbers are taken from a systematic review [ 21 ] and modified to remove individuals without id where possible (supplementary methods 2 ). a larger version of this table considers penetrance estimates using other values of p(d) (supplementary results 1 ). table 1 updated penetrance estimates of 83 recurrent cnvs. full size table table 1 demonstrates that penetrance for id for all cnvs are reduced, except for fully penetrant cnvs which remain fully penetrant. the difference is most noticeable for low penetrant cnvs, which demonstrate an approximate 5–10 fold reduction in penetrance estimates. cnv penetrance estimates are most accurate when interpreted as penetrance for id, rather than for any disease because (i) most datasets used to calculate cnv penetrance are primarily comprised of individuals with id (supplementary methods 3.2 ), (ii) id is the primary phenotype associated with most recurrent cnvs [ 18 ] and (iii) the formula for estimating penetrance is less reliable when calculating multiple phenotypes combined together (supplementary methods 3.1 ). some cnvs, such as the 17q12 [ hnf1b ] deletion, which causes renal cysts and diabetes, require a different affected cohort since id is not prominently associated with the cnv. penetrance estimates for these cnvs are excluded from table 1 , but provided in supplementary results 1 with additional notes. earlier penetrance estimates of 10%, 20% and 40% are re-estimated at 1%, 4% and 10% respectively. fully penetrant conditions remain fully penetrant. a shortcut for converting penetrance estimates from earlier publications to the new penetrance is provided in supplementary results 3 . it is useful to note that penetrance for id in the 17p12 [ pmp22 ] deletion for hereditary neuropathy with liability to pressure palsies and the reciprocal duplication for charcot marie tooth disease is 0%. this is a correct result, as these conditions are not associated with id [ 22 , 23 ]. this also provides supporting evidence that the datasets predominantly contain patients with id rather than phenotypes that might be associated with these two conditions. discussion using a revised penetrance formula with an improved understanding of its parameters resulted in lower penetrance estimates for many cnvs compared to previous publications [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 , 21 ]. reduced cnv penetrance estimates may affect clinical management of patients with these cnvs. reduced cnv penetrance estimates for id that are close to 0% suggest minimal or non-pathogenicity for id. improvements to penetrance estimations our study provides more accurate penetrance estimates due to two key improvements built on prior work [ 5 , 14 , 24 ] in this field. the first improvement involves clarifying the definition of penetrance, so that background risk is excluded. formula 1 (based on the earlier definition) includes the possibility that the genetic change is present but unrelated to the phenotype. formula 2 (based on the newly proposed definition) excludes the background rate of this phenotype. when cnv prevalence is similar in both affected and control groups, formula 1 will report 1–6% penetrance whilst formula 2 will report 0%. stated in another manner, formula 1 erroneously adds the chance that the control cohort has the phenotype, whilst formula 2 excludes this. the second improvement involves a recognition that neither formula 1 nor 2 are suited to calculate penetrance for multiple phenotypes (supplementary methods 3.1 ). whilst an ideal study should focus on a single well-defined condition (such as id), most studies [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] include participants with a variety of conditions (such as id, developmental delay, autism, and congenital malformations). nonetheless, if most individuals have id (or developmental delay which is often an early sign of id), then penetrance for id can still be approximated. the prevalence of id, p(d) was determined from a systematic review to be approximately 1.1% (supplementary methods 3 &amp; supplementary discussion 1 ). using this instead of p(d) = 3% [ 9 ], 4% [ 4 , 10 , 11 , 15 ], 5.12% [ 7 , 12 , 14 ], 5.3% [ 7 , 12 , 14 ], or 13% [ 19 ], led to an estimated 3–12 fold reduction in penetrance estimates for id, compared to penetrance estimates previously published. this change in lowering the value of p(d) to 1.1% requires justification. the affected cohort of three studies were clearly comprised entirely of individuals with id (or infants with developmental delay) [ 4 , 17 , 19 ], justifying the use of 1.1% for these studies. in other studies [ 3 , 7 , 10 , 14 , 15 ], affected cohorts were comprised mostly of individuals with id. supplementary methods 3 justifies using these datasets to approximate the penetrance of id. supplementary discussion 1 explores alternate approaches to penetrance estimation, before concluding on the approach presented here. the datasets are imperfect, so accurate penetrance estimates derived from these datasets are impossible. within these constraints, the value of 1.1% is more accurate than using any other value. the phenotype associated with a cnv different cnvs may be associated with different phenotypes. however, the data sources used to calculate penetrance are the same regardless of the cnv. this has led to confusion regarding the phenotype a cnv might be penetrant for. a key question arises as to whether the differences in penetrance estimates between this publication and earlier studies are due to a focus on id rather than other phenotypes. we argue that for most cnvs, the penetrance listed for id in table 1 using the new method is likely more accurate and clinically-meaningful, regardless of this distinction. we expand on this below, with further details in supplementary discussion 1 : for cnvs associated with a primary phenotype other than id, penetrance estimates are omitted from table 1 . examples include the 17q12 [ hnf1b ] deletion (renal cysts and diabetes) and the 1q21.1 [ rbm8a ] proximal deletion. neither penetrance for id, nor penetrance for another phenotype, can be reliably estimated from these datasets. a different dataset could be used to estimate penetrance for the non-id phenotype. for a cnv for which the primary phenotype is id, the new method is clearly more accurate due to an updated formula and more appropriate choice of the parameter p(d) = 1.1% to represent the prevalence of id. for this group, the presence or absence of other phenotypes for the cnv is irrelevant when it comes to choosing a value for p(d). this group constitutes the majority of the cnvs in table 1 . for cnvs that can cause id in some individuals and congenital malformations without id in other individuals, penetrance estimates presented here are still likely to be more accurate than previous estimates. this is because the affected cohort p(g | d) used to estimate penetrance of neurodevelopmental disorders and congenital malformations in cnvs are primarily composed of individuals with id (supplementary methods 3.2 ), but also includes some individuals without id. therefore, the phenotype “d” is mix of individuals highly biased towards id, supporting the use of 1.1% in the formula as the optimal parameter for the prevalence of the phenotype (supplementary methods 3 ). autism involves a non-homogenous mix of conditions with a definition that has changed significantly over time, whilst onset of schizophrenia is age-related. these issues make penetrance estimation using formula 1 or formula 2 challenging (supplementary methods 3.4 ). sometimes, the phenotype thought to be associated with a cnv may be incidental and its description in relation to the cnv may be due to ascertainment bias. consider a hypothetical cnv with 10% penetrance for id (and therefore 90% of individuals with this cnv are unaffected). multiple authors have assumed a background chance of disability being approximately 5% [ 3 , 4 , 7 , 10 , 11 , 14 , 15 , 17 ]. under these assumptions, 4.5% (5% of 90%) will have a different disability due to causes unrelated to the cnv (incidental phenotypes). individuals with this cnv may therefore have id (10%), other disabilities (4.5%) or be healthy (85.5%). case studies are biased towards those with a phenotype, so case studies may describe other disabilities in up to 30% (4.5/(10 + 4.5)%) of individuals with this cnv. this is higher than what most would consider as the background risk and can be mistaken for being part of the phenotype, but is clearly just an aspect of ascertainment bias. this theoretical example demonstrates how incidental pathology can be misattributed to low penetrant cnvs (with a rate of up to 30% in this example) due purely to ascertainment bias, when in reality, penetrance of these pathologies should be 0%. examples of changes to cnv penetrance seven cnvs are recalculated to have 0% penetrance for id, or have 95% confidence intervals which include 0%. these are 0.9 mb 2q11.2 [ tmem127 ] deletions, 0.2 mb 2q13 [ nphp1 ] proximal deletions and duplications, 0.13 mb 6q16 [ sim1 ] duplications, 0.2 mb 13q12 [ cryl1 ] deletions, 0.3 mb 15q11.2 [bp1-bp2] [ nipa1 , nipa2 ] duplications and 0.5 mb 16p12.2 [ cdr2 ] duplications (previously mismapped to 16p12.1). the prevalence of these cnvs in affected and control cohorts are similar. these 7 cnvs account for 1.8% of individuals in gnomad controls (supplementary result 2 ). four cnvs are calculated to have approximately 1% penetrance for id and 95% confidence intervals that include values ≤ 1%. these are 0.3 mb 1q21.1 [ rbm8a ] proximal duplications, 1.4 mb 15q13.3 [bp4-bp5] [ chrna7 ] duplications,1 mb 16p13.11 [ myh11 ] duplications and xp22.3 [ shox ] duplications. the 95% confidence intervals for these do not include 0% penetrance, suggesting pathogenicity and low (but statistically significant) penetrance. however, it is suggested that these results be interpreted conservatively, as incomplete penetrance may not be the correct explanation for the slight difference in prevalence of this cnv in affected compared to control datasets. differences in genetic ancestry [ 25 ], publication bias and other confounding variables in the way control and affected cohorts are selected may provide alternate explanations [ 21 ]. these four cnvs account for 0.5% of individuals in gnomad controls (supplementary results 2 ). three cnvs with 1–3% penetrance for id may have unclear pathogenicity using these methods. the slight enrichment of these cnvs in affected compared to control cohorts may represent susceptibility to id, but could also represent bias or noise in the dataset. this may differ from cnv to cnv. these cnvs are 0.3 mb 15q11.2 [bp1-bp2] [ nipa1 , nipa2 ] deletions, 0.5 mb 16p12.2 deletions [ cdr2 ] (previously mismapped to 16p12.1) and 0.2–0.3 mb 16p11.2 distal duplications [ sh2b1 ]. phenotypes associated with these cnvs have been published in case series of affected individuals. however, given the high prevalence of individuals with these variants in gnomad controls, it’s likely that many individuals with these cnvs are either healthy or have features that lie within a spectrum of human normal. the earlier example of an incidental phenotype with up to 30% penetrance being erroneously associated with a cnv, demonstrates how ascertainment bias can lead to misattribution of pathogenicity, and is particularly relevant for cnvs in this group. these 3 cnvs account for 0.4% of individuals in gnomad controls (supplementary results 2 ). some recurrent cnvs are rare and data is sparse. cnvs that are present in very few individuals can result in wide 95% confidence intervals for penetrance that span from 0 to 100%. penetrance for these cnvs using these methods are uncertain. more broadly, for any cnv, it may be more appropriate to interpret currently available data for penetrance as lying within a range of potential penetrance estimates (e.g., a 95% confidence interval), rather than as a fixed percentage (supplementary discussion 2 ). implications for clinical care these findings have multiple clinical implications. fourteen cnvs with penetrance close to 0% are seen in approximately 2.7% of individuals labelled by gnomad as controls. in contrast, approximately 3.1% of gnomad controls harbour at least one of the 83 cnvs in table 1 (supplementary results 2 ). equivalently, 14/83 cnvs in table 1 with little evidence for pathogenicity for id or other phenotypes make up the vast majority of recurrent cnvs in gnomad controls with published penetrance estimates, whilst the majority 69/83 of recurrent cnvs that are more penetrant are seen in only a minority of gnomad controls. choosing not to report these 14 cnvs will dramatically reduce the number of healthy individuals who are given a pathogenic cnv result, and dramatically reduce laboratory and clinician workload. reporting these 14 cnvs as pathogenic poses a risk of misattributing a patient’s phenotype to the cnv, or mislabelling a healthy individual. moreover, if a diagnosis is based on one of these cnvs, it may reduce the likelihood of the patient being offered further genomic testing. consequently, the practice of reporting these 14 common cnvs may compromise patient care by limiting a more comprehensive diagnostic evaluation in some countries or settings. large scale studies of some of these cnvs demonstrate that on average, people with some of these 14 cnvs may perform poorer in some cognitive tasks than those without [ 11 ]. however, it is unclear if all such individuals perform poorer, or if some perform poorly and others perform normally, which would also lead to similar graphs in studies that aggregate the results. given how common these 14 cnvs are, the lack of strong correlation with a non-intellectual phenotype and the inability to prove that they cause cognitive reduction in all individuals, it is worth considering that reporting it as pathogenic in an individual may be an example of false positive reporting for that individual. given that normal human intellect spans a large range from iq 70–130 and that there are likely many polygenic factors that contribute to this variable range, it may be more helpful to consider these 14 cnvs as part of a normal spectrum of human genetic changes rather than as pathogenic. a less stringent cut-off could consider 11 cnvs (seen in 2.3% of gnomad controls) instead of 14 cnvs (supplementary result 2 ). in the prenatal setting, identifying a cnv previously reported as being 5–10% penetrant and recognising that an accurate measure of penetrance is closer to 0%, may make the difference between a laboratory choosing to report the cnv or not to report. the choice not to report such cnvs, as suggested by some prenatal reporting guidelines [ 12 , 26 ], can make a significant difference to anxiety and the choices families make in pregnancy. choosing not to report these cnvs can also improve the quality of life for families, by reducing their time spent performing unnecessary research on the cnv. observations from other studies that are consistent with findings from this study our findings have the potential to resolve conflicting observations in the literature regarding pathogenicity of very low penetrant cnvs. for example, the 15q13.3 duplication [ chrna7 ] [bp4-bp5] has been reported as pathogenic [ 4 , 27 , 28 ] and as having little evidence for pathogenicity [ 29 ]. this study clarifies the paradox as being due to the misleading definition of penetrance and resolves it with formula 2. many recent studies suggest caution when assigning pathogenicity to low-penetrant cnvs, especially in the prenatal setting [ 12 , 26 , 30 ]. for example, it has been suggested that the 16p11.2 distal duplication [ sh2b1 ] be not reportable in the prenatal setting [ 26 ], despite its penetrance of 16% [ 21 ] which intuitively appears high enough to warrant reporting. revised penetrance estimates in this study show that penetrance for this cnv is closer to 2.7% (95% ci 1.4–4.4%). our study also raises the possibility that this apparent positive 2.7% penetrance estimate, could be due to other factors such as differences in genetic ancestry or imperfectly ascertained cohorts. therefore, this study provides a mathematical justification for some current published guidelines regarding not reporting low penetrant cnvs. some prospective studies of neonates with cnvs [ 31 , 32 ] demonstrate less evidence of neurodevelopmental disability on follow-up than one might expect based on earlier penetrance estimates. the lower penetrance estimates in this study provide a mathematical reason why this might be so. additionally, it is common in clinical practice to see many low penetrant cnvs being inherited from unaffected parents. this observation can be explained by the lower penetrance estimates published in this study. limitations in estimating penetrance using the bayesian formula penetrance estimates for cnvs provided in this study should be interpreted cautiously. the listed penetrance estimates are for id (because the data sources are biased towards id) rather than for penetrance of other phenotypes. pleiotropy of cnvs means some cnvs might have a higher penetrance for other phenotypes. accurate penetrance estimates also rely on large datasets containing affected and control individuals. while differences in cnv prevalence between these two groups are typically interpreted as evidence for incomplete penetrance of cnvs [ 3 , 4 , 5 , 7 , 8 , 9 , 10 , 11 , 12 , 14 , 15 , 16 , 17 , 19 ], other factors may explain these differences. these include cohort selection biases, age-related onset of symptoms, variable expressivity and methodological differences. in particular, the genetic makeup of affected and control cohorts can be challenging to compare because affected cohorts are typically recruited from one or two hospitals or geographic locations [ 3 , 4 , 10 , 14 , 15 , 17 ] whilst control cohorts are often drawn from diverse studies and locations [ 3 , 4 , 10 , 14 , 33 ]. this disparity raises the possibility that observed increases in cnv prevalence in affected cohorts could stem from differences in genetic ancestry or geography [ 25 ], rather than pathogenicity. the 3q29 [ dlg1 ] [ 21 ], 15q11.2 [ nipa1 , nipa2 ] [ 25 ], and 15q13.3 [ chrna7 ] [bp4-bp5] [ 21 , 25 ] cnvs may represent examples of this. penetrance estimates in this study are based on data that has been collected imperfectly. control cohorts are commonly comprised of adults from outpatient clinics who were enroled as affected individuals for various (non-neurodevelopmental) health issues such as cancer [ 3 , 4 , 14 ], hyperlipidemia [ 3 , 14 ], atherosclerosis [ 14 ], diabetes [ 15 ] or asthma [ 3 , 14 ]. therefore, control cohorts may include those with comorbid id. this can result in fully penetrant conditions being misinterpreted as incompletely penetrant. table 1 flags, with the symbol ‘b’, several cnvs in this category. the opposite may also be true of affected cohorts, particularly if there is no clinical information on the pathology request form for microarray. technological differences between the way affected and control cohorts are identified may also affect penetrance estimates. most cnvs are likely to be detected equally well using either microarray assemblies or cnv callers for exome data [ 34 ]. however, some cnvs may be identified by one method and not the other. if so, then a disparity in prevalence between affected cohorts (which use microarrays) versus the gnomad control cohort (which uses exomes) may lead to erroneously higher or lower penetrance estimates for that particular cnv. despite these potential confounding factors, a significant enrichment of a recurrent cnv in affected compared to control cohorts provides reasonable evidence for pathogenicity, though caution is advised as the aforementioned factors may cause over- or under-estimation of the true penetrance. finally, there is a blurry line between concepts of incomplete penetrance and variable expressivity. a cnv may reduce cognitive performance in some areas, but be insufficient to cause intellectual disability [ 11 ]. in this sense, calling the cnv non-penetrant for id could be misleading, as it does have an effect. on the other hand, if the effect size is small and the individual does not consider themselves impaired, it may be kinder in clinical practice to refer to their ability as lying within a spectrum of human normal, rather than focussing their attention on merely one of many polygenic factors affecting intellect. conclusion we propose an updated definition and formula for genetic penetrance that excludes the background rate of disease. if a genetic change does not predispose to a genetic illness, this genetic change should be considered 0% penetrant for that illness. previous definitions and formulas always result in penetrance &gt;0% due to inclusion of the background risk, which could be misleading for clinicians and patients. imprecise penetrance estimates have the potential to be detrimental by limiting diagnostic options for affected individuals. there is potential for harm in classifying low penetrant cnvs as “pathogenic”, when most individuals are either unaffected or so mildly affected that their symptoms fall within a spectrum of human normal. the updated penetrance estimates presented here may affect interpretation and reporting of cnvs, with implications for genetic counselling in both prenatal and postnatal settings. data availability all relevant data are listed within the paper and its supporting information files (supplementary discussion 1 &amp; supplementary results 1 ). code availability penetrance estimates and their 95% confidence interval can be obtained by running bootstrap_penetrance.m in octave v8.3.0. references carruth ed, young w, beer d, james ca, calkins h, jing l, et al. prevalence and electronic health record-based phenotype of loss-of-function genetic variants in arrhythmogenic right ventricular cardiomyopathy-associated genes. circ genom precis med. 2019;12:e002579. article pubmed pubmed central google scholar chen s, parmigiani g. meta-analysis of brca1 and brca2 penetrance. j clin oncol. 2007;25:1329–33. article pubmed google scholar cooper dn, krawczak m, polychronakos c, tyler-smith c, kehrer-sawatzki h. where genotype is not predictive of phenotype: towards an understanding of the molecular basis of reduced penetrance in human inherited disease. hum genet. 2013;132:1077–130. article cas pubmed pubmed central google scholar kirov g, rees e, walters jt, escott-price v, georgieva l, richards al, et al. the penetrance of copy number variations for schizophrenia and developmental delay. biol psychiatry. 2014;75:378–85. article cas pubmed google scholar vassos, collier e, holden da, patch s, rujescu c, st d, et al. penetrance for copy number variants associated with schizophrenia. hum mol genet. 2010;19:3477–81. article cas pubmed google scholar roberts jd, asaki sy, mazzanti a, bos jm, tuleta i, muir ar, et al. an international multicenter evaluation of type 5 long qt syndrome: a low penetrant primary arrhythmic condition. circulation. 2020;141:429–39. article cas pubmed pubmed central google scholar allach el khattabi l, heide s, caberg jh, andrieux j, doco fenzy m, vincent-delorme c, et al. 16p13.11 microduplication in 45 new patients: refined clinical significance and genotype-phenotype correlations. j med genet. 2018;0:1–7. google scholar chaste p, sanders sj, mohan kn, klei l, song y, murtha mt, et al. modest impact on risk for autism spectrum disorder of rare copy number variants at 15q11.2, specifically breakpoints 1 to 2. autism res. 2014;7:355–62. article pubmed pubmed central google scholar cosemans n, vandenhove l, vogels a, devriendt k, van esch h, van buggenhout g, et al. the clinical relevance of intragenic nrxn1 deletions. j med genet. 2020;57:347–55. article pubmed google scholar isles ar, ingason a, lowther c, walters j, gawlick m, stober g, et al. parental origin of interstitial duplications at 15q11.2-q13.3 in schizophrenia and neurodevelopmental disorders. plos genet. 2016;12:e1005993. article pubmed pubmed central google scholar kendall km, bracher-smith m, fitzpatrick h, lynham a, rees e, escott-price v, et al. cognitive performance and functional outcomes of carriers of pathogenic copy number variants: analysis of the uk biobank. br j psychiatry. 2019;214:297–304. article pubmed pubmed central google scholar maya i, perlman s, shohat m, kahana s, yacobsons, tennet,et al. should we report 15q11.2 bp1-bp2 deletions and duplications in the prenatal setting?.j clin med.2020;9:11. article google scholar mohan kn, cao y, pham j, cheung sw, hoffner l, ou zz, et al. phenotypic association of 15q11.2 cnvs of the region of breakpoints 1-2 (bp1-bp2) in a large cohort of samples referred for genetic diagnosis. j hum genet. 2019;64:253–5. article cas pubmed google scholar rosenfeld ja, coe bp, eichler ee, cuckle h, shaffer lg. estimates of penetrance for recurrent pathogenic copy-number variations. genet med. 2013;15:478–81. article cas pubmed google scholar tropeano m, howley d, gazzellone mj, wilson ce, ahn jw, stavropoulos dj, et al. microduplications at the pseudoautosomal shox locus in autism spectrum disorders and related neurodevelopmental conditions. j med genet. 2016;53:536–47. article cas pubmed google scholar al, shehhi m, forman eb, fitzgerald je, mcinerney v, krawczyk j, et al. nrxn1 deletion syndrome; phenotypic and penetrance data from 34 families. eur j med genet. 2019;62:204–9. article google scholar jønch ae, douard e, moreau c, van dijck a, passeggeri m, kooy f, et al. estimating the effect size of the 15q11.2 bp1-bp2 deletion and its contribution to neurodevelopmental symptoms: recommendations for practice. j med genet. 2019;56:701–10. article pubmed google scholar unique. rare chromosome &amp; gene disorder guides: rare chromosome disorder support group; 2024 available from: https://rarechromo.org/disorder-guides/ . martin cl, wain ke, oetjens mt, tolwinski k, palen e, hare-harris a, et al. identification of neuropsychiatric copy number variants in a health care system population. jama psychiatry. 2020;77:1276–85. article pubmed google scholar maulik pk, mascarenhas mn, mathers cd, dua t, saxena s. prevalence of intellectual disability: a meta-analysis of population-based studies. res dev disabil. 2011;32:419–36. article pubmed google scholar goh s, thiyagarajan l, dudding-byth t, pinese m, kirk ep. a systematic review and pooled analysis of penetrance estimates of copy-number variants associated with neurodevelopment. genet med. 2025;27:101227. article cas pubmed google scholar van paassen bw, van der kooi aj, van spaendonck-zwarts ky, verhamme c, baas f, de visser m. pmp22 related neuropathies: charcot-marie-tooth disease type 1a and hereditary neuropathy with liability to pressure palsies. orphanet j rare dis. 2014;9:38. article pubmed pubmed central google scholar chrestian n hereditary neuropathy with liability to pressure palsies. in: adam mp, feldman j, mirzaa gm, pagon ra, wallace se, bean ljh, et al., editors. genereviews(®). seattle (wa) 1993. benn pa. prenatal counseling and the detection of copy-number variants. genet med. 2013;15:316–7. article pubmed google scholar schultz lm, knighton a, huguet g, saci z, jean-louis m, mollon j, et al. copy-number variants differ in frequency across genetic ancestry groups. hgg adv. 2024;5:100340. cas pubmed pubmed central google scholar muys j, blaumeiser b, jacquemyn y, bandelier c, brison n, bulk s, et al. the belgian microarray prenatal (bemapre) database: a systematic nationwide repository of fetal genomic aberrations. prenat diagn. 2018;38:1120–8. article cas pubmed google scholar zhou d, gochman p, broadnax dd, rapoport jl, ahn k. 15q13.3 duplication in two patients with childhood-onset schizophrenia. am j med genet b neuropsychiatr genet. 2016;171:777–83. article cas pubmed pubmed central google scholar gillentine ma, schaaf cp. the human clinical phenotypes of altered chrna7 copy number. biochem pharmacol. 2015;97:352–62. article cas pubmed pubmed central google scholar clingen. 15q13.3 recurrent region (d-chrna7 to bp5) (includes chrna7 and otud7a): clingen dosage sensitivity curation page; 2018 [updated 10/5/2018. available from: https://dosage.clinicalgenome.org/clingen_region.cgi?id=isca-46295 . cai m, que y, chen x, chen y, liang b, huang h, et al. 16p13.11 microdeletion/microduplication in fetuses: investigation of associated ultrasound phenotypes, genetic anomalies, and pregnancy outcome follow-up. bmc pregnancy childbirth. 2022;22:913. article cas pubmed pubmed central google scholar muys j, jacquemyn y, blaumeiser b, bourlard l, brison n, bulk s, et al. prenatally detected copy number variants in a national cohort: a postnatal follow-up study. prenat diagn. 2020;40:1272–83. article pubmed google scholar smajlagić d, lavrichenko k, berland s, helgeland ø, knudsen gp, vaudel m, et al. population prevalence and inheritance pattern of recurrent cnvs associated with neurodevelopmental disorders in 12,252 newborns and their parents. eur j hum genet. 2021;29:205–15. article pubmed google scholar fu j, liao c, collins r, wang l, ben-isvy d, brand h, et al. rare coding cnvs from exome sequenced individuals in gnomad v4 gnomad: gnomad; 2023 [updated 1/11/2023. available from: https://gnomad.broadinstitute.org/news/2023-11-v4-copy-number-variants/ . babadi m, fu jm, lee sk, smirnov an, gauthier ld, walker m, et al. gatk-gcnv enables the discovery of rare copy number variants from exome sequencing data. nat genet. 2023;55:1589–97. article cas pubmed pubmed central google scholar download references acknowledgements we would like to extend our sincere gratitude to associate professor kate simms for her invaluable assistance in proofreading this manuscript. we would also like to thank dr rhys bowden for early contributions to some of the mathematical aspects of this work. funding open access funding enabled and organized by caul and its member institutions. author information authors and affiliations school of women and children’s health, university of new south wales, sydney, nsw, australia shuxiang goh, mark pinese &amp; edwin p. kirk new south wales health, sydney, nsw, australia shuxiang goh, tracy dudding-byth &amp; edwin p. kirk genetics of learning disability, newcastle, nsw, australia tracy dudding-byth university of newcastle, newcastle, nsw, australia tracy dudding-byth children’s cancer institute, sydney, nsw, australia mark pinese centre for clinical genetics, sydney children’s hospital, sydney, nsw, australia edwin p. kirk nsw health pathology randwick genomics laboratory, sydney, nsw, australia edwin p. kirk authors shuxiang goh view author publications search author on: pubmed google scholar tracy dudding-byth view author publications search author on: pubmed google scholar mark pinese view author publications search author on: pubmed google scholar edwin p. kirk view author publications search author on: pubmed google scholar contributions conceptualization: sg, ek; methodology: sg, ek; data curation: sg; formal analysis: sg; investigation: sg, ek; resources: sg; software: sg; supervision: ek, mp, td; validation: sg, ek; visualization: sg; writing-original draft: sg; writing-review &amp; editing: ek, mp, td. corresponding author correspondence to shuxiang goh . ethics declarations competing interests the authors declare no competing interests. ethics all data used in this study is publicly available. there are no other relevant ethical declarations. additional information publisher’s note springer nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. supplementary information supplementary materials supplementary table 6 &amp; 7 penetrance calculation rights and permissions open access this article is licensed under a creative commons attribution 4.0 international license, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the creative commons licence, and indicate if changes were made. the images or other third party material in this article are included in the article’s creative commons licence, unless indicated otherwise in a credit line to the material. if material is not included in the article’s creative commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. to view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/ . reprints and permissions about this article cite this article goh, s., dudding-byth, t., pinese, m. et al. updated penetrance estimates for recurrent copy number variants – an improved definition and formula. eur j hum genet (2025). https://doi.org/10.1038/s41431-025-01948-0 download citation received : 21 december 2024 revised : 19 august 2025 accepted : 11 september 2025 published : 15 october 2025 doi : https://doi.org/10.1038/s41431-025-01948-0 share this article anyone you share the following link with will be able to read this content: get shareable link sorry, a shareable link is not currently available for this article. copy shareable link to clipboard provided by the springer nature sharedit content-sharing initiative download pdf advertisement window.sn = window.sn || {}; window.sn.libs = window.sn.libs || {}; window.sn.libs.ads = window.sn.libs.ads || {}; window.sn.libs.ads.slotconfig = window.sn.libs.ads.slotconfig || {}; window.sn.libs.ads.slotconfig['right'] = { 'pos': 'right', 'type': 'article', 'path': 's41431-025-01948-0' }; window.sn.libs.ads.slotconfig['kwrd'] = 'clinical+genetics,cytogenetics,neurodevelopmental+disorders'; window.sn.libs.ads.slotconfig['subjmeta'] = '1405,208,2489,366,631'; explore content research articles reviews &amp; analysis news &amp; comment current issue collections follow us on twitter sign up for alerts rss feed about the journal journal information open access fees and funding about the editors about the partner contact for advertisers subscribe publish with us for authors &amp; referees language editing services open access funding submit manuscript window.datalayer = window.datalayer || []; window.datalayer.push({ page: { content: { fundingwidget: "true", } } }); search search articles by subject, keyword or author show results from all journals this journal search advanced search quick links explore articles by subject find a job guide to authors editorial policies european journal of human genetics ( eur j hum genet ) issn 1476-5438 (online) issn 1018-4813 (print) nature.com sitemap about nature portfolio about us press releases press office contact us discover content journals a-z articles by subject protocols.io nature index publishing policies nature portfolio policies open access author &amp; researcher services reprints &amp; permissions research data language editing scientific editing nature masterclasses research solutions libraries &amp; institutions librarian service &amp; tools librarian portal open research recommend to library advertising &amp; partnerships advertising partnerships &amp; services media kits branded content professional development nature awards nature careers nature conferences regional websites nature africa nature china nature india nature japan nature middle east privacy policy use of cookies your privacy choices/manage cookies legal notice accessibility statement terms &amp; conditions your us state privacy rights &copy; 2025 springer nature limited

## Methods
we propose the following new definition of penetrance : penetrance of a genetic variant is the probability of manifesting a phenotype due to having the genetic variant . the new definition differs from the earlier definition by excluding phenotypes incidental to the genetic variant, with concrete examples provided in supplementary method 1.5 . penetrance is similar to, but distinct from, the attributable fraction among the exposed, or the risk difference, which are commonly seen in other areas of statistics and medicine (supplementary method 1.4 ). a mathematical formula for calculating penetrance can be developed using two different (but equivalent) approaches. both methods begin by recognising that penetrance is the probability of having a phenotype given the presence of a specific genetic change, minus the probability that the phenotype is present due to another unrelated condition. both approaches also take into consideration the possibility of dual diagnoses that may contribute to the phenotype. the two methods use different approaches (supplementary methods 1.1 – 1.4 ), but resultin the same formula: $${\rm{penetrance}}=\frac{p\left(d\right)}{p\left(g\right)}\,.\,\frac{p\left({g|d}\right)\,-\,p\left(g\right)}{1-\,p\left(g\right)-p\left(d\right)+p\left(d\right).p\left({g|d}\right)}$$ (formula 2) p(d), p(g) and p(g|d) were described previously. it should be noted that the data required to calculate either the earlier (formula 1) or new (formula 2) version of penetrance are the same. therefore, data used to derive penetrance estimates using the earlier formula can be used with this new formula – no additional data is required to calculate updated penetrance estimates. most individuals in affected cohorts [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] selected for this study were children from high-income countries with id or developmental delay [ 3 , 4 , 7 , 10 , 14 , 15 , 17 ]. therefore, the parameter p(d) in formula 1 and formula 2, most accurately refers to the prevalence of paediatric id in high-income countries. this was taken to be 1.1% based on data from a systematic review [ 20 ] (supplementary methods 3.2 – 3.3 ). the inclusion criteria for cnvs in table 1 were (i) availability of published penetrance estimates for the cnv, (ii) the ability to verify these estimates using supplied data, (iii) id and/or developmental delay being present in most of the affected cohort (or can be assumed to be present – see supplementary methods 3.2 ) and (iv) the cnv’s primary phenotype includes, or has been suggested to include id. these methods are similar to that of an earlier systematic review [ 21 ], so our data is similar, modified only to remove individuals without id where possible (supplementary methods 2 , 3.1 , 3.2 ). affected cohort data in this study is pooled from 8 studies [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] that met the inclusion criteria listed above. the control cohort ( n = 269,885) is a subset of gnomad data which have been labelled by gnomad as controls, and each one with a relevant cnv for this study was highlighted in the supplemental in the systematic review [ 21 ]. many methods for determining 95% confidence intervals were considered. most [ 3 , 4 , 7 , 9 , 10 , 11 , 14 , 15 , 17 , 19 ] rely on assumptions of normality and symmetry of data, which were not appropriate for rare cnvs due to small datasets. we applied a bootstrap method from an earlier study [ 21 ], modified by running the simulation 10,000 times instead (code provided in supplementary file ). results a systematic review of id [ 20 ] identified 17 studies reporting the prevalence of paediatric id in high income countries. the prevalence ranged from 0.44 to 3.68% (supplementary methods 3.3 ), with a median prevalence of p(d) = 1.1% used in this study. alternate methods for determining the optimal value of p(d) were considered (supplementary methods 3 &amp; supplementary discussion 1 ) but were less accurate. table 1 reports updated penetrance estimates for id using this new approach, alongside estimates using the earlier penetrance formula [ 21 ]. data used to calculate these numbers are taken from a systematic review [ 21 ] and modified to remove individuals without id where possible (supplementary methods 2 ). a larger version of this table considers penetrance estimates using other values of p(d) (supplementary results 1 ). table 1 updated penetrance estimates of 83 recurrent cnvs. full size table table 1 demonstrates that penetrance for id for all cnvs are reduced, except for fully penetrant cnvs which remain fully penetrant. the difference is most noticeable for low penetrant cnvs, which demonstrate an approximate 5–10 fold reduction in penetrance estimates. cnv penetrance estimates are most accurate when interpreted as penetrance for id, rather than for any disease because (i) most datasets used to calculate cnv penetrance are primarily comprised of individuals with id (supplementary methods 3.2 ), (ii) id is the primary phenotype associated with most recurrent cnvs [ 18 ] and (iii) the formula for estimating penetrance is less reliable when calculating multiple phenotypes combined together (supplementary methods 3.1 ). some cnvs, such as the 17q12 [ hnf1b ] deletion, which causes renal cysts and diabetes, require a different affected cohort since id is not prominently associated with the cnv. penetrance estimates for these cnvs are excluded from table 1 , but provided in supplementary results 1 with additional notes. earlier penetrance estimates of 10%, 20% and 40% are re-estimated at 1%, 4% and 10% respectively. fully penetrant conditions remain fully penetrant. a shortcut for converting penetrance estimates from earlier publications to the new penetrance is provided in supplementary results 3 . it is useful to note that penetrance for id in the 17p12 [ pmp22 ] deletion for hereditary neuropathy with liability to pressure palsies and the reciprocal duplication for charcot marie tooth disease is 0%. this is a correct result, as these conditions are not associated with id [ 22 , 23 ]. this also provides supporting evidence that the datasets predominantly contain patients with id rather than phenotypes that might be associated with these two conditions. discussion using a revised penetrance formula with an improved understanding of its parameters resulted in lower penetrance estimates for many cnvs compared to previous publications [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 , 21 ]. reduced cnv penetrance estimates may affect clinical management of patients with these cnvs. reduced cnv penetrance estimates for id that are close to 0% suggest minimal or non-pathogenicity for id. improvements to penetrance estimations our study provides more accurate penetrance estimates due to two key improvements built on prior work [ 5 , 14 , 24 ] in this field. the first improvement involves clarifying the definition of penetrance, so that background risk is excluded. formula 1 (based on the earlier definition) includes the possibility that the genetic change is present but unrelated to the phenotype. formula 2 (based on the newly proposed definition) excludes the background rate of this phenotype. when cnv prevalence is similar in both affected and control groups, formula 1 will report 1–6% penetrance whilst formula 2 will report 0%. stated in another manner, formula 1 erroneously adds the chance that the control cohort has the phenotype, whilst formula 2 excludes this. the second improvement involves a recognition that neither formula 1 nor 2 are suited to calculate penetrance for multiple phenotypes (supplementary methods 3.1 ). whilst an ideal study should focus on a single well-defined condition (such as id), most studies [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] include participants with a variety of conditions (such as id, developmental delay, autism, and congenital malformations). nonetheless, if most individuals have id (or developmental delay which is often an early sign of id), then penetrance for id can still be approximated. the prevalence of id, p(d) was determined from a systematic review to be approximately 1.1% (supplementary methods 3 &amp; supplementary discussion 1 ). using this instead of p(d) = 3% [ 9 ], 4% [ 4 , 10 , 11 , 15 ], 5.12% [ 7 , 12 , 14 ], 5.3% [ 7 , 12 , 14 ], or 13% [ 19 ], led to an estimated 3–12 fold reduction in penetrance estimates for id, compared to penetrance estimates previously published. this change in lowering the value of p(d) to 1.1% requires justification. the affected cohort of three studies were clearly comprised entirely of individuals with id (or infants with developmental delay) [ 4 , 17 , 19 ], justifying the use of 1.1% for these studies. in other studies [ 3 , 7 , 10 , 14 , 15 ], affected cohorts were comprised mostly of individuals with id. supplementary methods 3 justifies using these datasets to approximate the penetrance of id. supplementary discussion 1 explores alternate approaches to penetrance estimation, before concluding on the approach presented here. the datasets are imperfect, so accurate penetrance estimates derived from these datasets are impossible. within these constraints, the value of 1.1% is more accurate than using any other value. the phenotype associated with a cnv different cnvs may be associated with different phenotypes. however, the data sources used to calculate penetrance are the same regardless of the cnv. this has led to confusion regarding the phenotype a cnv might be penetrant for. a key question arises as to whether the differences in penetrance estimates between this publication and earlier studies are due to a focus on id rather than other phenotypes. we argue that for most cnvs, the penetrance listed for id in table 1 using the new method is likely more accurate and clinically-meaningful, regardless of this distinction. we expand on this below, with further details in supplementary discussion 1 : for cnvs associated with a primary phenotype other than id, penetrance estimates are omitted from table 1 . examples include the 17q12 [ hnf1b ] deletion (renal cysts and diabetes) and the 1q21.1 [ rbm8a ] proximal deletion. neither penetrance for id, nor penetrance for another phenotype, can be reliably estimated from these datasets. a different dataset could be used to estimate penetrance for the non-id phenotype. for a cnv for which the primary phenotype is id, the new method is clearly more accurate due to an updated formula and more appropriate choice of the parameter p(d) = 1.1% to represent the prevalence of id. for this group, the presence or absence of other phenotypes for the cnv is irrelevant when it comes to choosing a value for p(d). this group constitutes the majority of the cnvs in table 1 . for cnvs that can cause id in some individuals and congenital malformations without id in other individuals, penetrance estimates presented here are still likely to be more accurate than previous estimates. this is because the affected cohort p(g | d) used to estimate penetrance of neurodevelopmental disorders and congenital malformations in cnvs are primarily composed of individuals with id (supplementary methods 3.2 ), but also includes some individuals without id. therefore, the phenotype “d” is mix of individuals highly biased towards id, supporting the use of 1.1% in the formula as the optimal parameter for the prevalence of the phenotype (supplementary methods 3 ). autism involves a non-homogenous mix of conditions with a definition that has changed significantly over time, whilst onset of schizophrenia is age-related. these issues make penetrance estimation using formula 1 or formula 2 challenging (supplementary methods 3.4 ). sometimes, the phenotype thought to be associated with a cnv may be incidental and its description in relation to the cnv may be due to ascertainment bias. consider a hypothetical cnv with 10% penetrance for id (and therefore 90% of individuals with this cnv are unaffected). multiple authors have assumed a background chance of disability being approximately 5% [ 3 , 4 , 7 , 10 , 11 , 14 , 15 , 17 ]. under these assumptions, 4.5% (5% of 90%) will have a different disability due to causes unrelated to the cnv (incidental phenotypes). individuals with this cnv may therefore have id (10%), other disabilities (4.5%) or be healthy (85.5%). case studies are biased towards those with a phenotype, so case studies may describe other disabilities in up to 30% (4.5/(10 + 4.5)%) of individuals with this cnv. this is higher than what most would consider as the background risk and can be mistaken for being part of the phenotype, but is clearly just an aspect of ascertainment bias. this theoretical example demonstrates how incidental pathology can be misattributed to low penetrant cnvs (with a rate of up to 30% in this example) due purely to ascertainment bias, when in reality, penetrance of these pathologies should be 0%. examples of changes to cnv penetrance seven cnvs are recalculated to have 0% penetrance for id, or have 95% confidence intervals which include 0%. these are 0.9 mb 2q11.2 [ tmem127 ] deletions, 0.2 mb 2q13 [ nphp1 ] proximal deletions and duplications, 0.13 mb 6q16 [ sim1 ] duplications, 0.2 mb 13q12 [ cryl1 ] deletions, 0.3 mb 15q11.2 [bp1-bp2] [ nipa1 , nipa2 ] duplications and 0.5 mb 16p12.2 [ cdr2 ] duplications (previously mismapped to 16p12.1). the prevalence of these cnvs in affected and control cohorts are similar. these 7 cnvs account for 1.8% of individuals in gnomad controls (supplementary result 2 ). four cnvs are calculated to have approximately 1% penetrance for id and 95% confidence intervals that include values ≤ 1%. these are 0.3 mb 1q21.1 [ rbm8a ] proximal duplications, 1.4 mb 15q13.3 [bp4-bp5] [ chrna7 ] duplications,1 mb 16p13.11 [ myh11 ] duplications and xp22.3 [ shox ] duplications. the 95% confidence intervals for these do not include 0% penetrance, suggesting pathogenicity and low (but statistically significant) penetrance. however, it is suggested that these results be interpreted conservatively, as incomplete penetrance may not be the correct explanation for the slight difference in prevalence of this cnv in affected compared to control datasets. differences in genetic ancestry [ 25 ], publication bias and other confounding variables in the way control and affected cohorts are selected may provide alternate explanations [ 21 ]. these four cnvs account for 0.5% of individuals in gnomad controls (supplementary results 2 ). three cnvs with 1–3% penetrance for id may have unclear pathogenicity using these methods. the slight enrichment of these cnvs in affected compared to control cohorts may represent susceptibility to id, but could also represent bias or noise in the dataset. this may differ from cnv to cnv. these cnvs are 0.3 mb 15q11.2 [bp1-bp2] [ nipa1 , nipa2 ] deletions, 0.5 mb 16p12.2 deletions [ cdr2 ] (previously mismapped to 16p12.1) and 0.2–0.3 mb 16p11.2 distal duplications [ sh2b1 ]. phenotypes associated with these cnvs have been published in case series of affected individuals. however, given the high prevalence of individuals with these variants in gnomad controls, it’s likely that many individuals with these cnvs are either healthy or have features that lie within a spectrum of human normal. the earlier example of an incidental phenotype with up to 30% penetrance being erroneously associated with a cnv, demonstrates how ascertainment bias can lead to misattribution of pathogenicity, and is particularly relevant for cnvs in this group. these 3 cnvs account for 0.4% of individuals in gnomad controls (supplementary results 2 ). some recurrent cnvs are rare and data is sparse. cnvs that are present in very few individuals can result in wide 95% confidence intervals for penetrance that span from 0 to 100%. penetrance for these cnvs using these methods are uncertain. more broadly, for any cnv, it may be more appropriate to interpret currently available data for penetrance as lying within a range of potential penetrance estimates (e.g., a 95% confidence interval), rather than as a fixed percentage (supplementary discussion 2 ). implications for clinical care these findings have multiple clinical implications. fourteen cnvs with penetrance close to 0% are seen in approximately 2.7% of individuals labelled by gnomad as controls. in contrast, approximately 3.1% of gnomad controls harbour at least one of the 83 cnvs in table 1 (supplementary results 2 ). equivalently, 14/83 cnvs in table 1 with little evidence for pathogenicity for id or other phenotypes make up the vast majority of recurrent cnvs in gnomad controls with published penetrance estimates, whilst the majority 69/83 of recurrent cnvs that are more penetrant are seen in only a minority of gnomad controls. choosing not to report these 14 cnvs will dramatically reduce the number of healthy individuals who are given a pathogenic cnv result, and dramatically reduce laboratory and clinician workload. reporting these 14 cnvs as pathogenic poses a risk of misattributing a patient’s phenotype to the cnv, or mislabelling a healthy individual. moreover, if a diagnosis is based on one of these cnvs, it may reduce the likelihood of the patient being offered further genomic testing. consequently, the practice of reporting these 14 common cnvs may compromise patient care by limiting a more comprehensive diagnostic evaluation in some countries or settings. large scale studies of some of these cnvs demonstrate that on average, people with some of these 14 cnvs may perform poorer in some cognitive tasks than those without [ 11 ]. however, it is unclear if all such individuals perform poorer, or if some perform poorly and others perform normally, which would also lead to similar graphs in studies that aggregate the results. given how common these 14 cnvs are, the lack of strong correlation with a non-intellectual phenotype and the inability to prove that they cause cognitive reduction in all individuals, it is worth considering that reporting it as pathogenic in an individual may be an example of false positive reporting for that individual. given that normal human intellect spans a large range from iq 70–130 and that there are likely many polygenic factors that contribute to this variable range, it may be more helpful to consider these 14 cnvs as part of a normal spectrum of human genetic changes rather than as pathogenic. a less stringent cut-off could consider 11 cnvs (seen in 2.3% of gnomad controls) instead of 14 cnvs (supplementary result 2 ). in the prenatal setting, identifying a cnv previously reported as being 5–10% penetrant and recognising that an accurate measure of penetrance is closer to 0%, may make the difference between a laboratory choosing to report the cnv or not to report. the choice not to report such cnvs, as suggested by some prenatal reporting guidelines [ 12 , 26 ], can make a significant difference to anxiety and the choices families make in pregnancy. choosing not to report these cnvs can also improve the quality of life for families, by reducing their time spent performing unnecessary research on the cnv. observations from other studies that are consistent with findings from this study our findings have the potential to resolve conflicting observations in the literature regarding pathogenicity of very low penetrant cnvs. for example, the 15q13.3 duplication [ chrna7 ] [bp4-bp5] has been reported as pathogenic [ 4 , 27 , 28 ] and as having little evidence for pathogenicity [ 29 ]. this study clarifies the paradox as being due to the misleading definition of penetrance and resolves it with formula 2. many recent studies suggest caution when assigning pathogenicity to low-penetrant cnvs, especially in the prenatal setting [ 12 , 26 , 30 ]. for example, it has been suggested that the 16p11.2 distal duplication [ sh2b1 ] be not reportable in the prenatal setting [ 26 ], despite its penetrance of 16% [ 21 ] which intuitively appears high enough to warrant reporting. revised penetrance estimates in this study show that penetrance for this cnv is closer to 2.7% (95% ci 1.4–4.4%). our study also raises the possibility that this apparent positive 2.7% penetrance estimate, could be due to other factors such as differences in genetic ancestry or imperfectly ascertained cohorts. therefore, this study provides a mathematical justification for some current published guidelines regarding not reporting low penetrant cnvs. some prospective studies of neonates with cnvs [ 31 , 32 ] demonstrate less evidence of neurodevelopmental disability on follow-up than one might expect based on earlier penetrance estimates. the lower penetrance estimates in this study provide a mathematical reason why this might be so. additionally, it is common in clinical practice to see many low penetrant cnvs being inherited from unaffected parents. this observation can be explained by the lower penetrance estimates published in this study. limitations in estimating penetrance using the bayesian formula penetrance estimates for cnvs provided in this study should be interpreted cautiously. the listed penetrance estimates are for id (because the data sources are biased towards id) rather than for penetrance of other phenotypes. pleiotropy of cnvs means some cnvs might have a higher penetrance for other phenotypes. accurate penetrance estimates also rely on large datasets containing affected and control individuals. while differences in cnv prevalence between these two groups are typically interpreted as evidence for incomplete penetrance of cnvs [ 3 , 4 , 5 , 7 , 8 , 9 , 10 , 11 , 12 , 14 , 15 , 16 , 17 , 19 ], other factors may explain these differences. these include cohort selection biases, age-related onset of symptoms, variable expressivity and methodological differences. in particular, the genetic makeup of affected and control cohorts can be challenging to compare because affected cohorts are typically recruited from one or two hospitals or geographic locations [ 3 , 4 , 10 , 14 , 15 , 17 ] whilst control cohorts are often drawn from diverse studies and locations [ 3 , 4 , 10 , 14 , 33 ]. this disparity raises the possibility that observed increases in cnv prevalence in affected cohorts could stem from differences in genetic ancestry or geography [ 25 ], rather than pathogenicity. the 3q29 [ dlg1 ] [ 21 ], 15q11.2 [ nipa1 , nipa2 ] [ 25 ], and 15q13.3 [ chrna7 ] [bp4-bp5] [ 21 , 25 ] cnvs may represent examples of this. penetrance estimates in this study are based on data that has been collected imperfectly. control cohorts are commonly comprised of adults from outpatient clinics who were enroled as affected individuals for various (non-neurodevelopmental) health issues such as cancer [ 3 , 4 , 14 ], hyperlipidemia [ 3 , 14 ], atherosclerosis [ 14 ], diabetes [ 15 ] or asthma [ 3 , 14 ]. therefore, control cohorts may include those with comorbid id. this can result in fully penetrant conditions being misinterpreted as incompletely penetrant. table 1 flags, with the symbol ‘b’, several cnvs in this category. the opposite may also be true of affected cohorts, particularly if there is no clinical information on the pathology request form for microarray. technological differences between the way affected and control cohorts are identified may also affect penetrance estimates. most cnvs are likely to be detected equally well using either microarray assemblies or cnv callers for exome data [ 34 ]. however, some cnvs may be identified by one method and not the other. if so, then a disparity in prevalence between affected cohorts (which use microarrays) versus the gnomad control cohort (which uses exomes) may lead to erroneously higher or lower penetrance estimates for that particular cnv. despite these potential confounding factors, a significant enrichment of a recurrent cnv in affected compared to control cohorts provides reasonable evidence for pathogenicity, though caution is advised as the aforementioned factors may cause over- or under-estimation of the true penetrance. finally, there is a blurry line between concepts of incomplete penetrance and variable expressivity. a cnv may reduce cognitive performance in some areas, but be insufficient to cause intellectual disability [ 11 ]. in this sense, calling the cnv non-penetrant for id could be misleading, as it does have an effect. on the other hand, if the effect size is small and the individual does not consider themselves impaired, it may be kinder in clinical practice to refer to their ability as lying within a spectrum of human normal, rather than focussing their attention on merely one of many polygenic factors affecting intellect. conclusion we propose an updated definition and formula for genetic penetrance that excludes the background rate of disease. if a genetic change does not predispose to a genetic illness, this genetic change should be considered 0% penetrant for that illness. previous definitions and formulas always result in penetrance &gt;0% due to inclusion of the background risk, which could be misleading for clinicians and patients. imprecise penetrance estimates have the potential to be detrimental by limiting diagnostic options for affected individuals. there is potential for harm in classifying low penetrant cnvs as “pathogenic”, when most individuals are either unaffected or so mildly affected that their symptoms fall within a spectrum of human normal. the updated penetrance estimates presented here may affect interpretation and reporting of cnvs, with implications for genetic counselling in both prenatal and postnatal settings. data availability all relevant data are listed within the paper and its supporting information files (supplementary discussion 1 &amp; supplementary results 1 ). code availability penetrance estimates and their 95% confidence interval can be obtained by running bootstrap_penetrance.m in octave v8.3.0. references carruth ed, young w, beer d, james ca, calkins h, jing l, et al. prevalence and electronic health record-based phenotype of loss-of-function genetic variants in arrhythmogenic right ventricular cardiomyopathy-associated genes. circ genom precis med. 2019;12:e002579. article pubmed pubmed central google scholar chen s, parmigiani g. meta-analysis of brca1 and brca2 penetrance. j clin oncol. 2007;25:1329–33. article pubmed google scholar cooper dn, krawczak m, polychronakos c, tyler-smith c, kehrer-sawatzki h. where genotype is not predictive of phenotype: towards an understanding of the molecular basis of reduced penetrance in human inherited disease. hum genet. 2013;132:1077–130. article cas pubmed pubmed central google scholar kirov g, rees e, walters jt, escott-price v, georgieva l, richards al, et al. the penetrance of copy number variations for schizophrenia and developmental delay. biol psychiatry. 2014;75:378–85. article cas pubmed google scholar vassos, collier e, holden da, patch s, rujescu c, st d, et al. penetrance for copy number variants associated with schizophrenia. hum mol genet. 2010;19:3477–81. article cas pubmed google scholar roberts jd, asaki sy, mazzanti a, bos jm, tuleta i, muir ar, et al. an international multicenter evaluation of type 5 long qt syndrome: a low penetrant primary arrhythmic condition. circulation. 2020;141:429–39. article cas pubmed pubmed central google scholar allach el khattabi l, heide s, caberg jh, andrieux j, doco fenzy m, vincent-delorme c, et al. 16p13.11 microduplication in 45 new patients: refined clinical significance and genotype-phenotype correlations. j med genet. 2018;0:1–7. google scholar chaste p, sanders sj, mohan kn, klei l, song y, murtha mt, et al. modest impact on risk for autism spectrum disorder of rare copy number variants at 15q11.2, specifically breakpoints 1 to 2. autism res. 2014;7:355–62. article pubmed pubmed central google scholar cosemans n, vandenhove l, vogels a, devriendt k, van esch h, van buggenhout g, et al. the clinical relevance of intragenic nrxn1 deletions. j med genet. 2020;57:347–55. article pubmed google scholar isles ar, ingason a, lowther c, walters j, gawlick m, stober g, et al. parental origin of interstitial duplications at 15q11.2-q13.3 in schizophrenia and neurodevelopmental disorders. plos genet. 2016;12:e1005993. article pubmed pubmed central google scholar kendall km, bracher-smith m, fitzpatrick h, lynham a, rees e, escott-price v, et al. cognitive performance and functional outcomes of carriers of pathogenic copy number variants: analysis of the uk biobank. br j psychiatry. 2019;214:297–304. article pubmed pubmed central google scholar maya i, perlman s, shohat m, kahana s, yacobsons, tennet,et al. should we report 15q11.2 bp1-bp2 deletions and duplications in the prenatal setting?.j clin med.2020;9:11. article google scholar mohan kn, cao y, pham j, cheung sw, hoffner l, ou zz, et al. phenotypic association of 15q11.2 cnvs of the region of breakpoints 1-2 (bp1-bp2) in a large cohort of samples referred for genetic diagnosis. j hum genet. 2019;64:253–5. article cas pubmed google scholar rosenfeld ja, coe bp, eichler ee, cuckle h, shaffer lg. estimates of penetrance for recurrent pathogenic copy-number variations. genet med. 2013;15:478–81. article cas pubmed google scholar tropeano m, howley d, gazzellone mj, wilson ce, ahn jw, stavropoulos dj, et al. microduplications at the pseudoautosomal shox locus in autism spectrum disorders and related neurodevelopmental conditions. j med genet. 2016;53:536–47. article cas pubmed google scholar al, shehhi m, forman eb, fitzgerald je, mcinerney v, krawczyk j, et al. nrxn1 deletion syndrome; phenotypic and penetrance data from 34 families. eur j med genet. 2019;62:204–9. article google scholar jønch ae, douard e, moreau c, van dijck a, passeggeri m, kooy f, et al. estimating the effect size of the 15q11.2 bp1-bp2 deletion and its contribution to neurodevelopmental symptoms: recommendations for practice. j med genet. 2019;56:701–10. article pubmed google scholar unique. rare chromosome &amp; gene disorder guides: rare chromosome disorder support group; 2024 available from: https://rarechromo.org/disorder-guides/ . martin cl, wain ke, oetjens mt, tolwinski k, palen e, hare-harris a, et al. identification of neuropsychiatric copy number variants in a health care system population. jama psychiatry. 2020;77:1276–85. article pubmed google scholar maulik pk, mascarenhas mn, mathers cd, dua t, saxena s. prevalence of intellectual disability: a meta-analysis of population-based studies. res dev disabil. 2011;32:419–36. article pubmed google scholar goh s, thiyagarajan l, dudding-byth t, pinese m, kirk ep. a systematic review and pooled analysis of penetrance estimates of copy-number variants associated with neurodevelopment. genet med. 2025;27:101227. article cas pubmed google scholar van paassen bw, van der kooi aj, van spaendonck-zwarts ky, verhamme c, baas f, de visser m. pmp22 related neuropathies: charcot-marie-tooth disease type 1a and hereditary neuropathy with liability to pressure palsies. orphanet j rare dis. 2014;9:38. article pubmed pubmed central google scholar chrestian n hereditary neuropathy with liability to pressure palsies. in: adam mp, feldman j, mirzaa gm, pagon ra, wallace se, bean ljh, et al., editors. genereviews(®). seattle (wa) 1993. benn pa. prenatal counseling and the detection of copy-number variants. genet med. 2013;15:316–7. article pubmed google scholar schultz lm, knighton a, huguet g, saci z, jean-louis m, mollon j, et al. copy-number variants differ in frequency across genetic ancestry groups. hgg adv. 2024;5:100340. cas pubmed pubmed central google scholar muys j, blaumeiser b, jacquemyn y, bandelier c, brison n, bulk s, et al. the belgian microarray prenatal (bemapre) database: a systematic nationwide repository of fetal genomic aberrations. prenat diagn. 2018;38:1120–8. article cas pubmed google scholar zhou d, gochman p, broadnax dd, rapoport jl, ahn k. 15q13.3 duplication in two patients with childhood-onset schizophrenia. am j med genet b neuropsychiatr genet. 2016;171:777–83. article cas pubmed pubmed central google scholar gillentine ma, schaaf cp. the human clinical phenotypes of altered chrna7 copy number. biochem pharmacol. 2015;97:352–62. article cas pubmed pubmed central google scholar clingen. 15q13.3 recurrent region (d-chrna7 to bp5) (includes chrna7 and otud7a): clingen dosage sensitivity curation page; 2018 [updated 10/5/2018. available from: https://dosage.clinicalgenome.org/clingen_region.cgi?id=isca-46295 . cai m, que y, chen x, chen y, liang b, huang h, et al. 16p13.11 microdeletion/microduplication in fetuses: investigation of associated ultrasound phenotypes, genetic anomalies, and pregnancy outcome follow-up. bmc pregnancy childbirth. 2022;22:913. article cas pubmed pubmed central google scholar muys j, jacquemyn y, blaumeiser b, bourlard l, brison n, bulk s, et al. prenatally detected copy number variants in a national cohort: a postnatal follow-up study. prenat diagn. 2020;40:1272–83. article pubmed google scholar smajlagić d, lavrichenko k, berland s, helgeland ø, knudsen gp, vaudel m, et al. population prevalence and inheritance pattern of recurrent cnvs associated with neurodevelopmental disorders in 12,252 newborns and their parents. eur j hum genet. 2021;29:205–15. article pubmed google scholar fu j, liao c, collins r, wang l, ben-isvy d, brand h, et al. rare coding cnvs from exome sequenced individuals in gnomad v4 gnomad: gnomad; 2023 [updated 1/11/2023. available from: https://gnomad.broadinstitute.org/news/2023-11-v4-copy-number-variants/ . babadi m, fu jm, lee sk, smirnov an, gauthier ld, walker m, et al. gatk-gcnv enables the discovery of rare copy number variants from exome sequencing data. nat genet. 2023;55:1589–97. article cas pubmed pubmed central google scholar download references acknowledgements we would like to extend our sincere gratitude to associate professor kate simms for her invaluable assistance in proofreading this manuscript. we would also like to thank dr rhys bowden for early contributions to some of the mathematical aspects of this work. funding open access funding enabled and organized by caul and its member institutions. author information authors and affiliations school of women and children’s health, university of new south wales, sydney, nsw, australia shuxiang goh, mark pinese &amp; edwin p. kirk new south wales health, sydney, nsw, australia shuxiang goh, tracy dudding-byth &amp; edwin p. kirk genetics of learning disability, newcastle, nsw, australia tracy dudding-byth university of newcastle, newcastle, nsw, australia tracy dudding-byth children’s cancer institute, sydney, nsw, australia mark pinese centre for clinical genetics, sydney children’s hospital, sydney, nsw, australia edwin p. kirk nsw health pathology randwick genomics laboratory, sydney, nsw, australia edwin p. kirk authors shuxiang goh view author publications search author on: pubmed google scholar tracy dudding-byth view author publications search author on: pubmed google scholar mark pinese view author publications search author on: pubmed google scholar edwin p. kirk view author publications search author on: pubmed google scholar contributions conceptualization: sg, ek; methodology: sg, ek; data curation: sg; formal analysis: sg; investigation: sg, ek; resources: sg; software: sg; supervision: ek, mp, td; validation: sg, ek; visualization: sg; writing-original draft: sg; writing-review &amp; editing: ek, mp, td. corresponding author correspondence to shuxiang goh . ethics declarations competing interests the authors declare no competing interests. ethics all data used in this study is publicly available. there are no other relevant ethical declarations. additional information publisher’s note springer nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. supplementary information supplementary materials supplementary table 6 &amp; 7 penetrance calculation rights and permissions open access this article is licensed under a creative commons attribution 4.0 international license, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the creative commons licence, and indicate if changes were made. the images or other third party material in this article are included in the article’s creative commons licence, unless indicated otherwise in a credit line to the material. if material is not included in the article’s creative commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. to view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/ . reprints and permissions about this article cite this article goh, s., dudding-byth, t., pinese, m. et al. updated penetrance estimates for recurrent copy number variants – an improved definition and formula. eur j hum genet (2025). https://doi.org/10.1038/s41431-025-01948-0 download citation received : 21 december 2024 revised : 19 august 2025 accepted : 11 september 2025 published : 15 october 2025 doi : https://doi.org/10.1038/s41431-025-01948-0 share this article anyone you share the following link with will be able to read this content: get shareable link sorry, a shareable link is not currently available for this article. copy shareable link to clipboard provided by the springer nature sharedit content-sharing initiative download pdf advertisement window.sn = window.sn || {}; window.sn.libs = window.sn.libs || {}; window.sn.libs.ads = window.sn.libs.ads || {}; window.sn.libs.ads.slotconfig = window.sn.libs.ads.slotconfig || {}; window.sn.libs.ads.slotconfig['right'] = { 'pos': 'right', 'type': 'article', 'path': 's41431-025-01948-0' }; window.sn.libs.ads.slotconfig['kwrd'] = 'clinical+genetics,cytogenetics,neurodevelopmental+disorders'; window.sn.libs.ads.slotconfig['subjmeta'] = '1405,208,2489,366,631'; explore content research articles reviews &amp; analysis news &amp; comment current issue collections follow us on twitter sign up for alerts rss feed about the journal journal information open access fees and funding about the editors about the partner contact for advertisers subscribe publish with us for authors &amp; referees language editing services open access funding submit manuscript window.datalayer = window.datalayer || []; window.datalayer.push({ page: { content: { fundingwidget: "true", } } }); search search articles by subject, keyword or author show results from all journals this journal search advanced search quick links explore articles by subject find a job guide to authors editorial policies european journal of human genetics ( eur j hum genet ) issn 1476-5438 (online) issn 1018-4813 (print) nature.com sitemap about nature portfolio about us press releases press office contact us discover content journals a-z articles by subject protocols.io nature index publishing policies nature portfolio policies open access author &amp; researcher services reprints &amp; permissions research data language editing scientific editing nature masterclasses research solutions libraries &amp; institutions librarian service &amp; tools librarian portal open research recommend to library advertising &amp; partnerships advertising partnerships &amp; services media kits branded content professional development nature awards nature careers nature conferences regional websites nature africa nature china nature india nature japan nature middle east privacy policy use of cookies your privacy choices/manage cookies legal notice accessibility statement terms &amp; conditions your us state privacy rights &copy; 2025 springer nature limited

## Results
in a phenotype. if the phenotype does not always occur, it is termed incomplete penetrance. a non-pathogenic variant should be 0% penetrant. we demonstrate that published estimates of penetrance do not reflect this when using a common bayesian formula for penetrance. we present a revised definition of penetrance that is more clinically useful, along with a new mathematical formula to estimate penetrance. these updated concepts apply to all forms of genetic penetrance, including copy number variations (cnvs), single-nucleotide variants, and other genetic changes. although the exact wording may vary between sources, the commonly published definition of penetrance is [ 1 , 2 , 3 , 4 , 5 , 6 ]: the probability of manifesting a phenotype given that a specific genetic variant is present . in large studies, this is generally [ 3 , 4 , 5 , 7 , 8 , 9 , 10 , 11 , 12 , 13 , 14 , 15 ] calculated as follows: $${\rm{penetrance}}=p\left(d|g\right)=\frac{p\left({g|d}\right).p\left(d\right)}{p\left(g\right)}=\frac{p\left({g|d}\right).p\left(d\right)}{p\left({g|d}\right).p\left(d\right)+p\left({g|}{d}^{c}\right).p\left({d}^{c}\right)}$$ (formula 1) where p(d) is the probability of manifesting the phenotype ( d isease) (equivalently, the prevalence of the phenotype in the general population), p(g) is the probability of having the genetic change ( g enotype), p(d|g) is the probability of manifesting the phenotype given that the genotype is present, p(g|d) is the probability of having the genotype given that the phenotype is present, d c refers to not having the phenotype and p(g|d c ) is the probability of having the genotype given that the phenotype is absent. however, in clinical practice, most clinicians interpret penetrance as the probability that a genetic variant causes a specific phenotype. these represent two different definitions of penetrance, as the latter excludes individuals who have the phenotype due to unrelated causes. two studies [ 8 , 12 ] recognised this distinction and provided an alternate penetrance estimate that attempted to exclude individuals that incidentally shared the phenotype by subtracting the value p(d) from formula 1. we build on this approach by developing formula 2, which is more mathematically precise, to achieve the same outcome. penetrance estimates using formula 1 [ 3 , 4 , 7 , 9 , 10 , 11 , 13 , 14 , 15 , 16 , 17 ] or formula 2 (below) to estimate penetrance are most accurate when considering penetrance for a single pathology. penetrance for multiple phenotypes is not supported by the underlying mathematical framework of the formulas (supplementary method 3.1 ). given that the most common phenotype associated with recurrent cnvs is intellectual disability (id) or developmental delay [ 18 ], this study focuses on cnv penetrance for id. this study addresses six key issues that lead to more accurate penetrance estimates: 1) clarification of the definition of penetrance. 2) an updated formula reflecting the clarified definition. 3) limiting penetrance estimations to a single phenotype of id. 4) determining the prevalence of id. 5) combining these changes to determine updated id penetrance estimates for recurrent cnvs. 6) updating 95% confidence intervals for penetrance estimates. the net outcome is improved penetrance estimates for recurrent cnvs, with many cnvs demonstrating a lower penetrance for id than previously published. methods we propose the following new definition of penetrance : penetrance of a genetic variant is the probability of manifesting a phenotype due to having the genetic variant . the new definition differs from the earlier definition by excluding phenotypes incidental to the genetic variant, with concrete examples provided in supplementary method 1.5 . penetrance is similar to, but distinct from, the attributable fraction among the exposed, or the risk difference, which are commonly seen in other areas of statistics and medicine (supplementary method 1.4 ). a mathematical formula for calculating penetrance can be developed using two different (but equivalent) approaches. both methods begin by recognising that penetrance is the probability of having a phenotype given the presence of a specific genetic change, minus the probability that the phenotype is present due to another unrelated condition. both approaches also take into consideration the possibility of dual diagnoses that may contribute to the phenotype. the two methods use different approaches (supplementary methods 1.1 – 1.4 ), but resultin the same formula: $${\rm{penetrance}}=\frac{p\left(d\right)}{p\left(g\right)}\,.\,\frac{p\left({g|d}\right)\,-\,p\left(g\right)}{1-\,p\left(g\right)-p\left(d\right)+p\left(d\right).p\left({g|d}\right)}$$ (formula 2) p(d), p(g) and p(g|d) were described previously. it should be noted that the data required to calculate either the earlier (formula 1) or new (formula 2) version of penetrance are the same. therefore, data used to derive penetrance estimates using the earlier formula can be used with this new formula – no additional data is required to calculate updated penetrance estimates. most individuals in affected cohorts [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] selected for this study were children from high-income countries with id or developmental delay [ 3 , 4 , 7 , 10 , 14 , 15 , 17 ]. therefore, the parameter p(d) in formula 1 and formula 2, most accurately refers to the prevalence of paediatric id in high-income countries. this was taken to be 1.1% based on data from a systematic review [ 20 ] (supplementary methods 3.2 – 3.3 ). the inclusion criteria for cnvs in table 1 were (i) availability of published penetrance estimates for the cnv, (ii) the ability to verify these estimates using supplied data, (iii) id and/or developmental delay being present in most of the affected cohort (or can be assumed to be present – see supplementary methods 3.2 ) and (iv) the cnv’s primary phenotype includes, or has been suggested to include id. these methods are similar to that of an earlier systematic review [ 21 ], so our data is similar, modified only to remove individuals without id where possible (supplementary methods 2 , 3.1 , 3.2 ). affected cohort data in this study is pooled from 8 studies [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] that met the inclusion criteria listed above. the control cohort ( n = 269,885) is a subset of gnomad data which have been labelled by gnomad as controls, and each one with a relevant cnv for this study was highlighted in the supplemental in the systematic review [ 21 ]. many methods for determining 95% confidence intervals were considered. most [ 3 , 4 , 7 , 9 , 10 , 11 , 14 , 15 , 17 , 19 ] rely on assumptions of normality and symmetry of data, which were not appropriate for rare cnvs due to small datasets. we applied a bootstrap method from an earlier study [ 21 ], modified by running the simulation 10,000 times instead (code provided in supplementary file ). results a systematic review of id [ 20 ] identified 17 studies reporting the prevalence of paediatric id in high income countries. the prevalence ranged from 0.44 to 3.68% (supplementary methods 3.3 ), with a median prevalence of p(d) = 1.1% used in this study. alternate methods for determining the optimal value of p(d) were considered (supplementary methods 3 &amp; supplementary discussion 1 ) but were less accurate. table 1 reports updated penetrance estimates for id using this new approach, alongside estimates using the earlier penetrance formula [ 21 ]. data used to calculate these numbers are taken from a systematic review [ 21 ] and modified to remove individuals without id where possible (supplementary methods 2 ). a larger version of this table considers penetrance estimates using other values of p(d) (supplementary results 1 ). table 1 updated penetrance estimates of 83 recurrent cnvs. full size table table 1 demonstrates that penetrance for id for all cnvs are reduced, except for fully penetrant cnvs which remain fully penetrant. the difference is most noticeable for low penetrant cnvs, which demonstrate an approximate 5–10 fold reduction in penetrance estimates. cnv penetrance estimates are most accurate when interpreted as penetrance for id, rather than for any disease because (i) most datasets used to calculate cnv penetrance are primarily comprised of individuals with id (supplementary methods 3.2 ), (ii) id is the primary phenotype associated with most recurrent cnvs [ 18 ] and (iii) the formula for estimating penetrance is less reliable when calculating multiple phenotypes combined together (supplementary methods 3.1 ). some cnvs, such as the 17q12 [ hnf1b ] deletion, which causes renal cysts and diabetes, require a different affected cohort since id is not prominently associated with the cnv. penetrance estimates for these cnvs are excluded from table 1 , but provided in supplementary results 1 with additional notes. earlier penetrance estimates of 10%, 20% and 40% are re-estimated at 1%, 4% and 10% respectively. fully penetrant conditions remain fully penetrant. a shortcut for converting penetrance estimates from earlier publications to the new penetrance is provided in supplementary results 3 . it is useful to note that penetrance for id in the 17p12 [ pmp22 ] deletion for hereditary neuropathy with liability to pressure palsies and the reciprocal duplication for charcot marie tooth disease is 0%. this is a correct result, as these conditions are not associated with id [ 22 , 23 ]. this also provides supporting evidence that the datasets predominantly contain patients with id rather than phenotypes that might be associated with these two conditions. discussion using a revised penetrance formula with an improved understanding of its parameters resulted in lower penetrance estimates for many cnvs compared to previous publications [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 , 21 ]. reduced cnv penetrance estimates may affect clinical management of patients with these cnvs. reduced cnv penetrance estimates for id that are close to 0% suggest minimal or non-pathogenicity for id. improvements to penetrance estimations our study provides more accurate penetrance estimates due to two key improvements built on prior work [ 5 , 14 , 24 ] in this field. the first improvement involves clarifying the definition of penetrance, so that background risk is excluded. formula 1 (based on the earlier definition) includes the possibility that the genetic change is present but unrelated to the phenotype. formula 2 (based on the newly proposed definition) excludes the background rate of this phenotype. when cnv prevalence is similar in both affected and control groups, formula 1 will report 1–6% penetrance whilst formula 2 will report 0%. stated in another manner, formula 1 erroneously adds the chance that the control cohort has the phenotype, whilst formula 2 excludes this. the second improvement involves a recognition that neither formula 1 nor 2 are suited to calculate penetrance for multiple phenotypes (supplementary methods 3.1 ). whilst an ideal study should focus on a single well-defined condition (such as id), most studies [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] include participants with a variety of conditions (such as id, developmental delay, autism, and congenital malformations). nonetheless, if most individuals have id (or developmental delay which is often an early sign of id), then penetrance for id can still be approximated. the prevalence of id, p(d) was determined from a systematic review to be approximately 1.1% (supplementary methods 3 &amp; supplementary discussion 1 ). using this instead of p(d) = 3% [ 9 ], 4% [ 4 , 10 , 11 , 15 ], 5.12% [ 7 , 12 , 14 ], 5.3% [ 7 , 12 , 14 ], or 13% [ 19 ], led to an estimated 3–12 fold reduction in penetrance estimates for id, compared to penetrance estimates previously published. this change in lowering the value of p(d) to 1.1% requires justification. the affected cohort of three studies were clearly comprised entirely of individuals with id (or infants with developmental delay) [ 4 , 17 , 19 ], justifying the use of 1.1% for these studies. in other studies [ 3 , 7 , 10 , 14 , 15 ], affected cohorts were comprised mostly of individuals with id. supplementary methods 3 justifies using these datasets to approximate the penetrance of id. supplementary discussion 1 explores alternate approaches to penetrance estimation, before concluding on the approach presented here. the datasets are imperfect, so accurate penetrance estimates derived from these datasets are impossible. within these constraints, the value of 1.1% is more accurate than using any other value. the phenotype associated with a cnv different cnvs may be associated with different phenotypes. however, the data sources used to calculate penetrance are the same regardless of the cnv. this has led to confusion regarding the phenotype a cnv might be penetrant for. a key question arises as to whether the differences in penetrance estimates between this publication and earlier studies are due to a focus on id rather than other phenotypes. we argue that for most cnvs, the penetrance listed for id in table 1 using the new method is likely more accurate and clinically-meaningful, regardless of this distinction. we expand on this below, with further details in supplementary discussion 1 : for cnvs associated with a primary phenotype other than id, penetrance estimates are omitted from table 1 . examples include the 17q12 [ hnf1b ] deletion (renal cysts and diabetes) and the 1q21.1 [ rbm8a ] proximal deletion. neither penetrance for id, nor penetrance for another phenotype, can be reliably estimated from these datasets. a different dataset could be used to estimate penetrance for the non-id phenotype. for a cnv for which the primary phenotype is id, the new method is clearly more accurate due to an updated formula and more appropriate choice of the parameter p(d) = 1.1% to represent the prevalence of id. for this group, the presence or absence of other phenotypes for the cnv is irrelevant when it comes to choosing a value for p(d). this group constitutes the majority of the cnvs in table 1 . for cnvs that can cause id in some individuals and congenital malformations without id in other individuals, penetrance estimates presented here are still likely to be more accurate than previous estimates. this is because the affected cohort p(g | d) used to estimate penetrance of neurodevelopmental disorders and congenital malformations in cnvs are primarily composed of individuals with id (supplementary methods 3.2 ), but also includes some individuals without id. therefore, the phenotype “d” is mix of individuals highly biased towards id, supporting the use of 1.1% in the formula as the optimal parameter for the prevalence of the phenotype (supplementary methods 3 ). autism involves a non-homogenous mix of conditions with a definition that has changed significantly over time, whilst onset of schizophrenia is age-related. these issues make penetrance estimation using formula 1 or formula 2 challenging (supplementary methods 3.4 ). sometimes, the phenotype thought to be associated with a cnv may be incidental and its description in relation to the cnv may be due to ascertainment bias. consider a hypothetical cnv with 10% penetrance for id (and therefore 90% of individuals with this cnv are unaffected). multiple authors have assumed a background chance of disability being approximately 5% [ 3 , 4 , 7 , 10 , 11 , 14 , 15 , 17 ]. under these assumptions, 4.5% (5% of 90%) will have a different disability due to causes unrelated to the cnv (incidental phenotypes). individuals with this cnv may therefore have id (10%), other disabilities (4.5%) or be healthy (85.5%). case studies are biased towards those with a phenotype, so case studies may describe other disabilities in up to 30% (4.5/(10 + 4.5)%) of individuals with this cnv. this is higher than what most would consider as the background risk and can be mistaken for being part of the phenotype, but is clearly just an aspect of ascertainment bias. this theoretical example demonstrates how incidental pathology can be misattributed to low penetrant cnvs (with a rate of up to 30% in this example) due purely to ascertainment bias, when in reality, penetrance of these pathologies should be 0%. examples of changes to cnv penetrance seven cnvs are recalculated to have 0% penetrance for id, or have 95% confidence intervals which include 0%. these are 0.9 mb 2q11.2 [ tmem127 ] deletions, 0.2 mb 2q13 [ nphp1 ] proximal deletions and duplications, 0.13 mb 6q16 [ sim1 ] duplications, 0.2 mb 13q12 [ cryl1 ] deletions, 0.3 mb 15q11.2 [bp1-bp2] [ nipa1 , nipa2 ] duplications and 0.5 mb 16p12.2 [ cdr2 ] duplications (previously mismapped to 16p12.1). the prevalence of these cnvs in affected and control cohorts are similar. these 7 cnvs account for 1.8% of individuals in gnomad controls (supplementary result 2 ). four cnvs are calculated to have approximately 1% penetrance for id and 95% confidence intervals that include values ≤ 1%. these are 0.3 mb 1q21.1 [ rbm8a ] proximal duplications, 1.4 mb 15q13.3 [bp4-bp5] [ chrna7 ] duplications,1 mb 16p13.11 [ myh11 ] duplications and xp22.3 [ shox ] duplications. the 95% confidence intervals for these do not include 0% penetrance, suggesting pathogenicity and low (but statistically significant) penetrance. however, it is suggested that these results be interpreted conservatively, as incomplete penetrance may not be the correct explanation for the slight difference in prevalence of this cnv in affected compared to control datasets. differences in genetic ancestry [ 25 ], publication bias and other confounding variables in the way control and affected cohorts are selected may provide alternate explanations [ 21 ]. these four cnvs account for 0.5% of individuals in gnomad controls (supplementary results 2 ). three cnvs with 1–3% penetrance for id may have unclear pathogenicity using these methods. the slight enrichment of these cnvs in affected compared to control cohorts may represent susceptibility to id, but could also represent bias or noise in the dataset. this may differ from cnv to cnv. these cnvs are 0.3 mb 15q11.2 [bp1-bp2] [ nipa1 , nipa2 ] deletions, 0.5 mb 16p12.2 deletions [ cdr2 ] (previously mismapped to 16p12.1) and 0.2–0.3 mb 16p11.2 distal duplications [ sh2b1 ]. phenotypes associated with these cnvs have been published in case series of affected individuals. however, given the high prevalence of individuals with these variants in gnomad controls, it’s likely that many individuals with these cnvs are either healthy or have features that lie within a spectrum of human normal. the earlier example of an incidental phenotype with up to 30% penetrance being erroneously associated with a cnv, demonstrates how ascertainment bias can lead to misattribution of pathogenicity, and is particularly relevant for cnvs in this group. these 3 cnvs account for 0.4% of individuals in gnomad controls (supplementary results 2 ). some recurrent cnvs are rare and data is sparse. cnvs that are present in very few individuals can result in wide 95% confidence intervals for penetrance that span from 0 to 100%. penetrance for these cnvs using these methods are uncertain. more broadly, for any cnv, it may be more appropriate to interpret currently available data for penetrance as lying within a range of potential penetrance estimates (e.g., a 95% confidence interval), rather than as a fixed percentage (supplementary discussion 2 ). implications for clinical care these findings have multiple clinical implications. fourteen cnvs with penetrance close to 0% are seen in approximately 2.7% of individuals labelled by gnomad as controls. in contrast, approximately 3.1% of gnomad controls harbour at least one of the 83 cnvs in table 1 (supplementary results 2 ). equivalently, 14/83 cnvs in table 1 with little evidence for pathogenicity for id or other phenotypes make up the vast majority of recurrent cnvs in gnomad controls with published penetrance estimates, whilst the majority 69/83 of recurrent cnvs that are more penetrant are seen in only a minority of gnomad controls. choosing not to report these 14 cnvs will dramatically reduce the number of healthy individuals who are given a pathogenic cnv result, and dramatically reduce laboratory and clinician workload. reporting these 14 cnvs as pathogenic poses a risk of misattributing a patient’s phenotype to the cnv, or mislabelling a healthy individual. moreover, if a diagnosis is based on one of these cnvs, it may reduce the likelihood of the patient being offered further genomic testing. consequently, the practice of reporting these 14 common cnvs may compromise patient care by limiting a more comprehensive diagnostic evaluation in some countries or settings. large scale studies of some of these cnvs demonstrate that on average, people with some of these 14 cnvs may perform poorer in some cognitive tasks than those without [ 11 ]. however, it is unclear if all such individuals perform poorer, or if some perform poorly and others perform normally, which would also lead to similar graphs in studies that aggregate the results. given how common these 14 cnvs are, the lack of strong correlation with a non-intellectual phenotype and the inability to prove that they cause cognitive reduction in all individuals, it is worth considering that reporting it as pathogenic in an individual may be an example of false positive reporting for that individual. given that normal human intellect spans a large range from iq 70–130 and that there are likely many polygenic factors that contribute to this variable range, it may be more helpful to consider these 14 cnvs as part of a normal spectrum of human genetic changes rather than as pathogenic. a less stringent cut-off could consider 11 cnvs (seen in 2.3% of gnomad controls) instead of 14 cnvs (supplementary result 2 ). in the prenatal setting, identifying a cnv previously reported as being 5–10% penetrant and recognising that an accurate measure of penetrance is closer to 0%, may make the difference between a laboratory choosing to report the cnv or not to report. the choice not to report such cnvs, as suggested by some prenatal reporting guidelines [ 12 , 26 ], can make a significant difference to anxiety and the choices families make in pregnancy. choosing not to report these cnvs can also improve the quality of life for families, by reducing their time spent performing unnecessary research on the cnv. observations from other studies that are consistent with findings from this study our findings have the potential to resolve conflicting observations in the literature regarding pathogenicity of very low penetrant cnvs. for example, the 15q13.3 duplication [ chrna7 ] [bp4-bp5] has been reported as pathogenic [ 4 , 27 , 28 ] and as having little evidence for pathogenicity [ 29 ]. this study clarifies the paradox as being due to the misleading definition of penetrance and resolves it with formula 2. many recent studies suggest caution when assigning pathogenicity to low-penetrant cnvs, especially in the prenatal setting [ 12 , 26 , 30 ]. for example, it has been suggested that the 16p11.2 distal duplication [ sh2b1 ] be not reportable in the prenatal setting [ 26 ], despite its penetrance of 16% [ 21 ] which intuitively appears high enough to warrant reporting. revised penetrance estimates in this study show that penetrance for this cnv is closer to 2.7% (95% ci 1.4–4.4%). our study also raises the possibility that this apparent positive 2.7% penetrance estimate, could be due to other factors such as differences in genetic ancestry or imperfectly ascertained cohorts. therefore, this study provides a mathematical justification for some current published guidelines regarding not reporting low penetrant cnvs. some prospective studies of neonates with cnvs [ 31 , 32 ] demonstrate less evidence of neurodevelopmental disability on follow-up than one might expect based on earlier penetrance estimates. the lower penetrance estimates in this study provide a mathematical reason why this might be so. additionally, it is common in clinical practice to see many low penetrant cnvs being inherited from unaffected parents. this observation can be explained by the lower penetrance estimates published in this study. limitations in estimating penetrance using the bayesian formula penetrance estimates for cnvs provided in this study should be interpreted cautiously. the listed penetrance estimates are for id (because the data sources are biased towards id) rather than for penetrance of other phenotypes. pleiotropy of cnvs means some cnvs might have a higher penetrance for other phenotypes. accurate penetrance estimates also rely on large datasets containing affected and control individuals. while differences in cnv prevalence between these two groups are typically interpreted as evidence for incomplete penetrance of cnvs [ 3 , 4 , 5 , 7 , 8 , 9 , 10 , 11 , 12 , 14 , 15 , 16 , 17 , 19 ], other factors may explain these differences. these include cohort selection biases, age-related onset of symptoms, variable expressivity and methodological differences. in particular, the genetic makeup of affected and control cohorts can be challenging to compare because affected cohorts are typically recruited from one or two hospitals or geographic locations [ 3 , 4 , 10 , 14 , 15 , 17 ] whilst control cohorts are often drawn from diverse studies and locations [ 3 , 4 , 10 , 14 , 33 ]. this disparity raises the possibility that observed increases in cnv prevalence in affected cohorts could stem from differences in genetic ancestry or geography [ 25 ], rather than pathogenicity. the 3q29 [ dlg1 ] [ 21 ], 15q11.2 [ nipa1 , nipa2 ] [ 25 ], and 15q13.3 [ chrna7 ] [bp4-bp5] [ 21 , 25 ] cnvs may represent examples of this. penetrance estimates in this study are based on data that has been collected imperfectly. control cohorts are commonly comprised of adults from outpatient clinics who were enroled as affected individuals for various (non-neurodevelopmental) health issues such as cancer [ 3 , 4 , 14 ], hyperlipidemia [ 3 , 14 ], atherosclerosis [ 14 ], diabetes [ 15 ] or asthma [ 3 , 14 ]. therefore, control cohorts may include those with comorbid id. this can result in fully penetrant conditions being misinterpreted as incompletely penetrant. table 1 flags, with the symbol ‘b’, several cnvs in this category. the opposite may also be true of affected cohorts, particularly if there is no clinical information on the pathology request form for microarray. technological differences between the way affected and control cohorts are identified may also affect penetrance estimates. most cnvs are likely to be detected equally well using either microarray assemblies or cnv callers for exome data [ 34 ]. however, some cnvs may be identified by one method and not the other. if so, then a disparity in prevalence between affected cohorts (which use microarrays) versus the gnomad control cohort (which uses exomes) may lead to erroneously higher or lower penetrance estimates for that particular cnv. despite these potential confounding factors, a significant enrichment of a recurrent cnv in affected compared to control cohorts provides reasonable evidence for pathogenicity, though caution is advised as the aforementioned factors may cause over- or under-estimation of the true penetrance. finally, there is a blurry line between concepts of incomplete penetrance and variable expressivity. a cnv may reduce cognitive performance in some areas, but be insufficient to cause intellectual disability [ 11 ]. in this sense, calling the cnv non-penetrant for id could be misleading, as it does have an effect. on the other hand, if the effect size is small and the individual does not consider themselves impaired, it may be kinder in clinical practice to refer to their ability as lying within a spectrum of human normal, rather than focussing their attention on merely one of many polygenic factors affecting intellect. conclusion we propose an updated definition and formula for genetic penetrance that excludes the background rate of disease. if a genetic change does not predispose to a genetic illness, this genetic change should be considered 0% penetrant for that illness. previous definitions and formulas always result in penetrance &gt;0% due to inclusion of the background risk, which could be misleading for clinicians and patients. imprecise penetrance estimates have the potential to be detrimental by limiting diagnostic options for affected individuals. there is potential for harm in classifying low penetrant cnvs as “pathogenic”, when most individuals are either unaffected or so mildly affected that their symptoms fall within a spectrum of human normal. the updated penetrance estimates presented here may affect interpretation and reporting of cnvs, with implications for genetic counselling in both prenatal and postnatal settings. data availability all relevant data are listed within the paper and its supporting information files (supplementary discussion 1 &amp; supplementary results 1 ). code availability penetrance estimates and their 95% confidence interval can be obtained by running bootstrap_penetrance.m in octave v8.3.0. references carruth ed, young w, beer d, james ca, calkins h, jing l, et al. prevalence and electronic health record-based phenotype of loss-of-function genetic variants in arrhythmogenic right ventricular cardiomyopathy-associated genes. circ genom precis med. 2019;12:e002579. article pubmed pubmed central google scholar chen s, parmigiani g. meta-analysis of brca1 and brca2 penetrance. j clin oncol. 2007;25:1329–33. article pubmed google scholar cooper dn, krawczak m, polychronakos c, tyler-smith c, kehrer-sawatzki h. where genotype is not predictive of phenotype: towards an understanding of the molecular basis of reduced penetrance in human inherited disease. hum genet. 2013;132:1077–130. article cas pubmed pubmed central google scholar kirov g, rees e, walters jt, escott-price v, georgieva l, richards al, et al. the penetrance of copy number variations for schizophrenia and developmental delay. biol psychiatry. 2014;75:378–85. article cas pubmed google scholar vassos, collier e, holden da, patch s, rujescu c, st d, et al. penetrance for copy number variants associated with schizophrenia. hum mol genet. 2010;19:3477–81. article cas pubmed google scholar roberts jd, asaki sy, mazzanti a, bos jm, tuleta i, muir ar, et al. an international multicenter evaluation of type 5 long qt syndrome: a low penetrant primary arrhythmic condition. circulation. 2020;141:429–39. article cas pubmed pubmed central google scholar allach el khattabi l, heide s, caberg jh, andrieux j, doco fenzy m, vincent-delorme c, et al. 16p13.11 microduplication in 45 new patients: refined clinical significance and genotype-phenotype correlations. j med genet. 2018;0:1–7. google scholar chaste p, sanders sj, mohan kn, klei l, song y, murtha mt, et al. modest impact on risk for autism spectrum disorder of rare copy number variants at 15q11.2, specifically breakpoints 1 to 2. autism res. 2014;7:355–62. article pubmed pubmed central google scholar cosemans n, vandenhove l, vogels a, devriendt k, van esch h, van buggenhout g, et al. the clinical relevance of intragenic nrxn1 deletions. j med genet. 2020;57:347–55. article pubmed google scholar isles ar, ingason a, lowther c, walters j, gawlick m, stober g, et al. parental origin of interstitial duplications at 15q11.2-q13.3 in schizophrenia and neurodevelopmental disorders. plos genet. 2016;12:e1005993. article pubmed pubmed central google scholar kendall km, bracher-smith m, fitzpatrick h, lynham a, rees e, escott-price v, et al. cognitive performance and functional outcomes of carriers of pathogenic copy number variants: analysis of the uk biobank. br j psychiatry. 2019;214:297–304. article pubmed pubmed central google scholar maya i, perlman s, shohat m, kahana s, yacobsons, tennet,et al. should we report 15q11.2 bp1-bp2 deletions and duplications in the prenatal setting?.j clin med.2020;9:11. article google scholar mohan kn, cao y, pham j, cheung sw, hoffner l, ou zz, et al. phenotypic association of 15q11.2 cnvs of the region of breakpoints 1-2 (bp1-bp2) in a large cohort of samples referred for genetic diagnosis. j hum genet. 2019;64:253–5. article cas pubmed google scholar rosenfeld ja, coe bp, eichler ee, cuckle h, shaffer lg. estimates of penetrance for recurrent pathogenic copy-number variations. genet med. 2013;15:478–81. article cas pubmed google scholar tropeano m, howley d, gazzellone mj, wilson ce, ahn jw, stavropoulos dj, et al. microduplications at the pseudoautosomal shox locus in autism spectrum disorders and related neurodevelopmental conditions. j med genet. 2016;53:536–47. article cas pubmed google scholar al, shehhi m, forman eb, fitzgerald je, mcinerney v, krawczyk j, et al. nrxn1 deletion syndrome; phenotypic and penetrance data from 34 families. eur j med genet. 2019;62:204–9. article google scholar jønch ae, douard e, moreau c, van dijck a, passeggeri m, kooy f, et al. estimating the effect size of the 15q11.2 bp1-bp2 deletion and its contribution to neurodevelopmental symptoms: recommendations for practice. j med genet. 2019;56:701–10. article pubmed google scholar unique. rare chromosome &amp; gene disorder guides: rare chromosome disorder support group; 2024 available from: https://rarechromo.org/disorder-guides/ . martin cl, wain ke, oetjens mt, tolwinski k, palen e, hare-harris a, et al. identification of neuropsychiatric copy number variants in a health care system population. jama psychiatry. 2020;77:1276–85. article pubmed google scholar maulik pk, mascarenhas mn, mathers cd, dua t, saxena s. prevalence of intellectual disability: a meta-analysis of population-based studies. res dev disabil. 2011;32:419–36. article pubmed google scholar goh s, thiyagarajan l, dudding-byth t, pinese m, kirk ep. a systematic review and pooled analysis of penetrance estimates of copy-number variants associated with neurodevelopment. genet med. 2025;27:101227. article cas pubmed google scholar van paassen bw, van der kooi aj, van spaendonck-zwarts ky, verhamme c, baas f, de visser m. pmp22 related neuropathies: charcot-marie-tooth disease type 1a and hereditary neuropathy with liability to pressure palsies. orphanet j rare dis. 2014;9:38. article pubmed pubmed central google scholar chrestian n hereditary neuropathy with liability to pressure palsies. in: adam mp, feldman j, mirzaa gm, pagon ra, wallace se, bean ljh, et al., editors. genereviews(®). seattle (wa) 1993. benn pa. prenatal counseling and the detection of copy-number variants. genet med. 2013;15:316–7. article pubmed google scholar schultz lm, knighton a, huguet g, saci z, jean-louis m, mollon j, et al. copy-number variants differ in frequency across genetic ancestry groups. hgg adv. 2024;5:100340. cas pubmed pubmed central google scholar muys j, blaumeiser b, jacquemyn y, bandelier c, brison n, bulk s, et al. the belgian microarray prenatal (bemapre) database: a systematic nationwide repository of fetal genomic aberrations. prenat diagn. 2018;38:1120–8. article cas pubmed google scholar zhou d, gochman p, broadnax dd, rapoport jl, ahn k. 15q13.3 duplication in two patients with childhood-onset schizophrenia. am j med genet b neuropsychiatr genet. 2016;171:777–83. article cas pubmed pubmed central google scholar gillentine ma, schaaf cp. the human clinical phenotypes of altered chrna7 copy number. biochem pharmacol. 2015;97:352–62. article cas pubmed pubmed central google scholar clingen. 15q13.3 recurrent region (d-chrna7 to bp5) (includes chrna7 and otud7a): clingen dosage sensitivity curation page; 2018 [updated 10/5/2018. available from: https://dosage.clinicalgenome.org/clingen_region.cgi?id=isca-46295 . cai m, que y, chen x, chen y, liang b, huang h, et al. 16p13.11 microdeletion/microduplication in fetuses: investigation of associated ultrasound phenotypes, genetic anomalies, and pregnancy outcome follow-up. bmc pregnancy childbirth. 2022;22:913. article cas pubmed pubmed central google scholar muys j, jacquemyn y, blaumeiser b, bourlard l, brison n, bulk s, et al. prenatally detected copy number variants in a national cohort: a postnatal follow-up study. prenat diagn. 2020;40:1272–83. article pubmed google scholar smajlagić d, lavrichenko k, berland s, helgeland ø, knudsen gp, vaudel m, et al. population prevalence and inheritance pattern of recurrent cnvs associated with neurodevelopmental disorders in 12,252 newborns and their parents. eur j hum genet. 2021;29:205–15. article pubmed google scholar fu j, liao c, collins r, wang l, ben-isvy d, brand h, et al. rare coding cnvs from exome sequenced individuals in gnomad v4 gnomad: gnomad; 2023 [updated 1/11/2023. available from: https://gnomad.broadinstitute.org/news/2023-11-v4-copy-number-variants/ . babadi m, fu jm, lee sk, smirnov an, gauthier ld, walker m, et al. gatk-gcnv enables the discovery of rare copy number variants from exome sequencing data. nat genet. 2023;55:1589–97. article cas pubmed pubmed central google scholar download references acknowledgements we would like to extend our sincere gratitude to associate professor kate simms for her invaluable assistance in proofreading this manuscript. we would also like to thank dr rhys bowden for early contributions to some of the mathematical aspects of this work. funding open access funding enabled and organized by caul and its member institutions. author information authors and affiliations school of women and children’s health, university of new south wales, sydney, nsw, australia shuxiang goh, mark pinese &amp; edwin p. kirk new south wales health, sydney, nsw, australia shuxiang goh, tracy dudding-byth &amp; edwin p. kirk genetics of learning disability, newcastle, nsw, australia tracy dudding-byth university of newcastle, newcastle, nsw, australia tracy dudding-byth children’s cancer institute, sydney, nsw, australia mark pinese centre for clinical genetics, sydney children’s hospital, sydney, nsw, australia edwin p. kirk nsw health pathology randwick genomics laboratory, sydney, nsw, australia edwin p. kirk authors shuxiang goh view author publications search author on: pubmed google scholar tracy dudding-byth view author publications search author on: pubmed google scholar mark pinese view author publications search author on: pubmed google scholar edwin p. kirk view author publications search author on: pubmed google scholar contributions conceptualization: sg, ek; methodology: sg, ek; data curation: sg; formal analysis: sg; investigation: sg, ek; resources: sg; software: sg; supervision: ek, mp, td; validation: sg, ek; visualization: sg; writing-original draft: sg; writing-review &amp; editing: ek, mp, td. corresponding author correspondence to shuxiang goh . ethics declarations competing interests the authors declare no competing interests. ethics all data used in this study is publicly available. there are no other relevant ethical declarations. additional information publisher’s note springer nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. supplementary information supplementary materials supplementary table 6 &amp; 7 penetrance calculation rights and permissions open access this article is licensed under a creative commons attribution 4.0 international license, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the creative commons licence, and indicate if changes were made. the images or other third party material in this article are included in the article’s creative commons licence, unless indicated otherwise in a credit line to the material. if material is not included in the article’s creative commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. to view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/ . reprints and permissions about this article cite this article goh, s., dudding-byth, t., pinese, m. et al. updated penetrance estimates for recurrent copy number variants – an improved definition and formula. eur j hum genet (2025). https://doi.org/10.1038/s41431-025-01948-0 download citation received : 21 december 2024 revised : 19 august 2025 accepted : 11 september 2025 published : 15 october 2025 doi : https://doi.org/10.1038/s41431-025-01948-0 share this article anyone you share the following link with will be able to read this content: get shareable link sorry, a shareable link is not currently available for this article. copy shareable link to clipboard provided by the springer nature sharedit content-sharing initiative download pdf advertisement window.sn = window.sn || {}; window.sn.libs = window.sn.libs || {}; window.sn.libs.ads = window.sn.libs.ads || {}; window.sn.libs.ads.slotconfig = window.sn.libs.ads.slotconfig || {}; window.sn.libs.ads.slotconfig['right'] = { 'pos': 'right', 'type': 'article', 'path': 's41431-025-01948-0' }; window.sn.libs.ads.slotconfig['kwrd'] = 'clinical+genetics,cytogenetics,neurodevelopmental+disorders'; window.sn.libs.ads.slotconfig['subjmeta'] = '1405,208,2489,366,631'; explore content research articles reviews &amp; analysis news &amp; comment current issue collections follow us on twitter sign up for alerts rss feed about the journal journal information open access fees and funding about the editors about the partner contact for advertisers subscribe publish with us for authors &amp; referees language editing services open access funding submit manuscript window.datalayer = window.datalayer || []; window.datalayer.push({ page: { content: { fundingwidget: "true", } } }); search search articles by subject, keyword or author show results from all journals this journal search advanced search quick links explore articles by subject find a job guide to authors editorial policies european journal of human genetics ( eur j hum genet ) issn 1476-5438 (online) issn 1018-4813 (print) nature.com sitemap about nature portfolio about us press releases press office contact us discover content journals a-z articles by subject protocols.io nature index publishing policies nature portfolio policies open access author &amp; researcher services reprints &amp; permissions research data language editing scientific editing nature masterclasses research solutions libraries &amp; institutions librarian service &amp; tools librarian portal open research recommend to library advertising &amp; partnerships advertising partnerships &amp; services media kits branded content professional development nature awards nature careers nature conferences regional websites nature africa nature china nature india nature japan nature middle east privacy policy use of cookies your privacy choices/manage cookies legal notice accessibility statement terms &amp; conditions your us state privacy rights &copy; 2025 springer nature limited

## Discussion
1 ) but were less accurate. table 1 reports updated penetrance estimates for id using this new approach, alongside estimates using the earlier penetrance formula [ 21 ]. data used to calculate these numbers are taken from a systematic review [ 21 ] and modified to remove individuals without id where possible (supplementary methods 2 ). a larger version of this table considers penetrance estimates using other values of p(d) (supplementary results 1 ). table 1 updated penetrance estimates of 83 recurrent cnvs. full size table table 1 demonstrates that penetrance for id for all cnvs are reduced, except for fully penetrant cnvs which remain fully penetrant. the difference is most noticeable for low penetrant cnvs, which demonstrate an approximate 5–10 fold reduction in penetrance estimates. cnv penetrance estimates are most accurate when interpreted as penetrance for id, rather than for any disease because (i) most datasets used to calculate cnv penetrance are primarily comprised of individuals with id (supplementary methods 3.2 ), (ii) id is the primary phenotype associated with most recurrent cnvs [ 18 ] and (iii) the formula for estimating penetrance is less reliable when calculating multiple phenotypes combined together (supplementary methods 3.1 ). some cnvs, such as the 17q12 [ hnf1b ] deletion, which causes renal cysts and diabetes, require a different affected cohort since id is not prominently associated with the cnv. penetrance estimates for these cnvs are excluded from table 1 , but provided in supplementary results 1 with additional notes. earlier penetrance estimates of 10%, 20% and 40% are re-estimated at 1%, 4% and 10% respectively. fully penetrant conditions remain fully penetrant. a shortcut for converting penetrance estimates from earlier publications to the new penetrance is provided in supplementary results 3 . it is useful to note that penetrance for id in the 17p12 [ pmp22 ] deletion for hereditary neuropathy with liability to pressure palsies and the reciprocal duplication for charcot marie tooth disease is 0%. this is a correct result, as these conditions are not associated with id [ 22 , 23 ]. this also provides supporting evidence that the datasets predominantly contain patients with id rather than phenotypes that might be associated with these two conditions. discussion using a revised penetrance formula with an improved understanding of its parameters resulted in lower penetrance estimates for many cnvs compared to previous publications [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 , 21 ]. reduced cnv penetrance estimates may affect clinical management of patients with these cnvs. reduced cnv penetrance estimates for id that are close to 0% suggest minimal or non-pathogenicity for id. improvements to penetrance estimations our study provides more accurate penetrance estimates due to two key improvements built on prior work [ 5 , 14 , 24 ] in this field. the first improvement involves clarifying the definition of penetrance, so that background risk is excluded. formula 1 (based on the earlier definition) includes the possibility that the genetic change is present but unrelated to the phenotype. formula 2 (based on the newly proposed definition) excludes the background rate of this phenotype. when cnv prevalence is similar in both affected and control groups, formula 1 will report 1–6% penetrance whilst formula 2 will report 0%. stated in another manner, formula 1 erroneously adds the chance that the control cohort has the phenotype, whilst formula 2 excludes this. the second improvement involves a recognition that neither formula 1 nor 2 are suited to calculate penetrance for multiple phenotypes (supplementary methods 3.1 ). whilst an ideal study should focus on a single well-defined condition (such as id), most studies [ 3 , 4 , 7 , 10 , 14 , 15 , 17 , 19 ] include participants with a variety of conditions (such as id, developmental delay, autism, and congenital malformations). nonetheless, if most individuals have id (or developmental delay which is often an early sign of id), then penetrance for id can still be approximated. the prevalence of id, p(d) was determined from a systematic review to be approximately 1.1% (supplementary methods 3 &amp; supplementary discussion 1 ). using this instead of p(d) = 3% [ 9 ], 4% [ 4 , 10 , 11 , 15 ], 5.12% [ 7 , 12 , 14 ], 5.3% [ 7 , 12 , 14 ], or 13% [ 19 ], led to an estimated 3–12 fold reduction in penetrance estimates for id, compared to penetrance estimates previously published. this change in lowering the value of p(d) to 1.1% requires justification. the affected cohort of three studies were clearly comprised entirely of individuals with id (or infants with developmental delay) [ 4 , 17 , 19 ], justifying the use of 1.1% for these studies. in other studies [ 3 , 7 , 10 , 14 , 15 ], affected cohorts were comprised mostly of individuals with id. supplementary methods 3 justifies using these datasets to approximate the penetrance of id. supplementary discussion 1 explores alternate approaches to penetrance estimation, before concluding on the approach presented here. the datasets are imperfect, so accurate penetrance estimates derived from these datasets are impossible. within these constraints, the value of 1.1% is more accurate than using any other value. the phenotype associated with a cnv different cnvs may be associated with different phenotypes. however, the data sources used to calculate penetrance are the same regardless of the cnv. this has led to confusion regarding the phenotype a cnv might be penetrant for. a key question arises as to whether the differences in penetrance estimates between this publication and earlier studies are due to a focus on id rather than other phenotypes. we argue that for most cnvs, the penetrance listed for id in table 1 using the new method is likely more accurate and clinically-meaningful, regardless of this distinction. we expand on this below, with further details in supplementary discussion 1 : for cnvs associated with a primary phenotype other than id, penetrance estimates are omitted from table 1 . examples include the 17q12 [ hnf1b ] deletion (renal cysts and diabetes) and the 1q21.1 [ rbm8a ] proximal deletion. neither penetrance for id, nor penetrance for another phenotype, can be reliably estimated from these datasets. a different dataset could be used to estimate penetrance for the non-id phenotype. for a cnv for which the primary phenotype is id, the new method is clearly more accurate due to an updated formula and more appropriate choice of the parameter p(d) = 1.1% to represent the prevalence of id. for this group, the presence or absence of other phenotypes for the cnv is irrelevant when it comes to choosing a value for p(d). this group constitutes the majority of the cnvs in table 1 . for cnvs that can cause id in some individuals and congenital malformations without id in other individuals, penetrance estimates presented here are still likely to be more accurate than previous estimates. this is because the affected cohort p(g | d) used to estimate penetrance of neurodevelopmental disorders and congenital malformations in cnvs are primarily composed of individuals with id (supplementary methods 3.2 ), but also includes some individuals without id. therefore, the phenotype “d” is mix of individuals highly biased towards id, supporting the use of 1.1% in the formula as the optimal parameter for the prevalence of the phenotype (supplementary methods 3 ). autism involves a non-homogenous mix of conditions with a definition that has changed significantly over time, whilst onset of schizophrenia is age-related. these issues make penetrance estimation using formula 1 or formula 2 challenging (supplementary methods 3.4 ). sometimes, the phenotype thought to be associated with a cnv may be incidental and its description in relation to the cnv may be due to ascertainment bias. consider a hypothetical cnv with 10% penetrance for id (and therefore 90% of individuals with this cnv are unaffected). multiple authors have assumed a background chance of disability being approximately 5% [ 3 , 4 , 7 , 10 , 11 , 14 , 15 , 17 ]. under these assumptions, 4.5% (5% of 90%) will have a different disability due to causes unrelated to the cnv (incidental phenotypes). individuals with this cnv may therefore have id (10%), other disabilities (4.5%) or be healthy (85.5%). case studies are biased towards those with a phenotype, so case studies may describe other disabilities in up to 30% (4.5/(10 + 4.5)%) of individuals with this cnv. this is higher than what most would consider as the background risk and can be mistaken for being part of the phenotype, but is clearly just an aspect of ascertainment bias. this theoretical example demonstrates how incidental pathology can be misattributed to low penetrant cnvs (with a rate of up to 30% in this example) due purely to ascertainment bias, when in reality, penetrance of these pathologies should be 0%. examples of changes to cnv penetrance seven cnvs are recalculated to have 0% penetrance for id, or have 95% confidence intervals which include 0%. these are 0.9 mb 2q11.2 [ tmem127 ] deletions, 0.2 mb 2q13 [ nphp1 ] proximal deletions and duplications, 0.13 mb 6q16 [ sim1 ] duplications, 0.2 mb 13q12 [ cryl1 ] deletions, 0.3 mb 15q11.2 [bp1-bp2] [ nipa1 , nipa2 ] duplications and 0.5 mb 16p12.2 [ cdr2 ] duplications (previously mismapped to 16p12.1). the prevalence of these cnvs in affected and control cohorts are similar. these 7 cnvs account for 1.8% of individuals in gnomad controls (supplementary result 2 ). four cnvs are calculated to have approximately 1% penetrance for id and 95% confidence intervals that include values ≤ 1%. these are 0.3 mb 1q21.1 [ rbm8a ] proximal duplications, 1.4 mb 15q13.3 [bp4-bp5] [ chrna7 ] duplications,1 mb 16p13.11 [ myh11 ] duplications and xp22.3 [ shox ] duplications. the 95% confidence intervals for these do not include 0% penetrance, suggesting pathogenicity and low (but statistically significant) penetrance. however, it is suggested that these results be interpreted conservatively, as incomplete penetrance may not be the correct explanation for the slight difference in prevalence of this cnv in affected compared to control datasets. differences in genetic ancestry [ 25 ], publication bias and other confounding variables in the way control and affected cohorts are selected may provide alternate explanations [ 21 ]. these four cnvs account for 0.5% of individuals in gnomad controls (supplementary results 2 ). three cnvs with 1–3% penetrance for id may have unclear pathogenicity using these methods. the slight enrichment of these cnvs in affected compared to control cohorts may represent susceptibility to id, but could also represent bias or noise in the dataset. this may differ from cnv to cnv. these cnvs are 0.3 mb 15q11.2 [bp1-bp2] [ nipa1 , nipa2 ] deletions, 0.5 mb 16p12.2 deletions [ cdr2 ] (previously mismapped to 16p12.1) and 0.2–0.3 mb 16p11.2 distal duplications [ sh2b1 ]. phenotypes associated with these cnvs have been published in case series of affected individuals. however, given the high prevalence of individuals with these variants in gnomad controls, it’s likely that many individuals with these cnvs are either healthy or have features that lie within a spectrum of human normal. the earlier example of an incidental phenotype with up to 30% penetrance being erroneously associated with a cnv, demonstrates how ascertainment bias can lead to misattribution of pathogenicity, and is particularly relevant for cnvs in this group. these 3 cnvs account for 0.4% of individuals in gnomad controls (supplementary results 2 ). some recurrent cnvs are rare and data is sparse. cnvs that are present in very few individuals can result in wide 95% confidence intervals for penetrance that span from 0 to 100%. penetrance for these cnvs using these methods are uncertain. more broadly, for any cnv, it may be more appropriate to interpret currently available data for penetrance as lying within a range of potential penetrance estimates (e.g., a 95% confidence interval), rather than as a fixed percentage (supplementary discussion 2 ). implications for clinical care these findings have multiple clinical implications. fourteen cnvs with penetrance close to 0% are seen in approximately 2.7% of individuals labelled by gnomad as controls. in contrast, approximately 3.1% of gnomad controls harbour at least one of the 83 cnvs in table 1 (supplementary results 2 ). equivalently, 14/83 cnvs in table 1 with little evidence for pathogenicity for id or other phenotypes make up the vast majority of recurrent cnvs in gnomad controls with published penetrance estimates, whilst the majority 69/83 of recurrent cnvs that are more penetrant are seen in only a minority of gnomad controls. choosing not to report these 14 cnvs will dramatically reduce the number of healthy individuals who are given a pathogenic cnv result, and dramatically reduce laboratory and clinician workload. reporting these 14 cnvs as pathogenic poses a risk of misattributing a patient’s phenotype to the cnv, or mislabelling a healthy individual. moreover, if a diagnosis is based on one of these cnvs, it may reduce the likelihood of the patient being offered further genomic testing. consequently, the practice of reporting these 14 common cnvs may compromise patient care by limiting a more comprehensive diagnostic evaluation in some countries or settings. large scale studies of some of these cnvs demonstrate that on average, people with some of these 14 cnvs may perform poorer in some cognitive tasks than those without [ 11 ]. however, it is unclear if all such individuals perform poorer, or if some perform poorly and others perform normally, which would also lead to similar graphs in studies that aggregate the results. given how common these 14 cnvs are, the lack of strong correlation with a non-intellectual phenotype and the inability to prove that they cause cognitive reduction in all individuals, it is worth considering that reporting it as pathogenic in an individual may be an example of false positive reporting for that individual. given that normal human intellect spans a large range from iq 70–130 and that there are likely many polygenic factors that contribute to this variable range, it may be more helpful to consider these 14 cnvs as part of a normal spectrum of human genetic changes rather than as pathogenic. a less stringent cut-off could consider 11 cnvs (seen in 2.3% of gnomad controls) instead of 14 cnvs (supplementary result 2 ). in the prenatal setting, identifying a cnv previously reported as being 5–10% penetrant and recognising that an accurate measure of penetrance is closer to 0%, may make the difference between a laboratory choosing to report the cnv or not to report. the choice not to report such cnvs, as suggested by some prenatal reporting guidelines [ 12 , 26 ], can make a significant difference to anxiety and the choices families make in pregnancy. choosing not to report these cnvs can also improve the quality of life for families, by reducing their time spent performing unnecessary research on the cnv. observations from other studies that are consistent with findings from this study our findings have the potential to resolve conflicting observations in the literature regarding pathogenicity of very low penetrant cnvs. for example, the 15q13.3 duplication [ chrna7 ] [bp4-bp5] has been reported as pathogenic [ 4 , 27 , 28 ] and as having little evidence for pathogenicity [ 29 ]. this study clarifies the paradox as being due to the misleading definition of penetrance and resolves it with formula 2. many recent studies suggest caution when assigning pathogenicity to low-penetrant cnvs, especially in the prenatal setting [ 12 , 26 , 30 ]. for example, it has been suggested that the 16p11.2 distal duplication [ sh2b1 ] be not reportable in the prenatal setting [ 26 ], despite its penetrance of 16% [ 21 ] which intuitively appears high enough to warrant reporting. revised penetrance estimates in this study show that penetrance for this cnv is closer to 2.7% (95% ci 1.4–4.4%). our study also raises the possibility that this apparent positive 2.7% penetrance estimate, could be due to other factors such as differences in genetic ancestry or imperfectly ascertained cohorts. therefore, this study provides a mathematical justification for some current published guidelines regarding not reporting low penetrant cnvs. some prospective studies of neonates with cnvs [ 31 , 32 ] demonstrate less evidence of neurodevelopmental disability on follow-up than one might expect based on earlier penetrance estimates. the lower penetrance estimates in this study provide a mathematical reason why this might be so. additionally, it is common in clinical practice to see many low penetrant cnvs being inherited from unaffected parents. this observation can be explained by the lower penetrance estimates published in this study. limitations in estimating penetrance using the bayesian formula penetrance estimates for cnvs provided in this study should be interpreted cautiously. the listed penetrance estimates are for id (because the data sources are biased towards id) rather than for penetrance of other phenotypes. pleiotropy of cnvs means some cnvs might have a higher penetrance for other phenotypes. accurate penetrance estimates also rely on large datasets containing affected and control individuals. while differences in cnv prevalence between these two groups are typically interpreted as evidence for incomplete penetrance of cnvs [ 3 , 4 , 5 , 7 , 8 , 9 , 10 , 11 , 12 , 14 , 15 , 16 , 17 , 19 ], other factors may explain these differences. these include cohort selection biases, age-related onset of symptoms, variable expressivity and methodological differences. in particular, the genetic makeup of affected and control cohorts can be challenging to compare because affected cohorts are typically recruited from one or two hospitals or geographic locations [ 3 , 4 , 10 , 14 , 15 , 17 ] whilst control cohorts are often drawn from diverse studies and locations [ 3 , 4 , 10 , 14 , 33 ]. this disparity raises the possibility that observed increases in cnv prevalence in affected cohorts could stem from differences in genetic ancestry or geography [ 25 ], rather than pathogenicity. the 3q29 [ dlg1 ] [ 21 ], 15q11.2 [ nipa1 , nipa2 ] [ 25 ], and 15q13.3 [ chrna7 ] [bp4-bp5] [ 21 , 25 ] cnvs may represent examples of this. penetrance estimates in this study are based on data that has been collected imperfectly. control cohorts are commonly comprised of adults from outpatient clinics who were enroled as affected individuals for various (non-neurodevelopmental) health issues such as cancer [ 3 , 4 , 14 ], hyperlipidemia [ 3 , 14 ], atherosclerosis [ 14 ], diabetes [ 15 ] or asthma [ 3 , 14 ]. therefore, control cohorts may include those with comorbid id. this can result in fully penetrant conditions being misinterpreted as incompletely penetrant. table 1 flags, with the symbol ‘b’, several cnvs in this category. the opposite may also be true of affected cohorts, particularly if there is no clinical information on the pathology request form for microarray. technological differences between the way affected and control cohorts are identified may also affect penetrance estimates. most cnvs are likely to be detected equally well using either microarray assemblies or cnv callers for exome data [ 34 ]. however, some cnvs may be identified by one method and not the other. if so, then a disparity in prevalence between affected cohorts (which use microarrays) versus the gnomad control cohort (which uses exomes) may lead to erroneously higher or lower penetrance estimates for that particular cnv. despite these potential confounding factors, a significant enrichment of a recurrent cnv in affected compared to control cohorts provides reasonable evidence for pathogenicity, though caution is advised as the aforementioned factors may cause over- or under-estimation of the true penetrance. finally, there is a blurry line between concepts of incomplete penetrance and variable expressivity. a cnv may reduce cognitive performance in some areas, but be insufficient to cause intellectual disability [ 11 ]. in this sense, calling the cnv non-penetrant for id could be misleading, as it does have an effect. on the other hand, if the effect size is small and the individual does not consider themselves impaired, it may be kinder in clinical practice to refer to their ability as lying within a spectrum of human normal, rather than focussing their attention on merely one of many polygenic factors affecting intellect. conclusion we propose an updated definition and formula for genetic penetrance that excludes the background rate of disease. if a genetic change does not predispose to a genetic illness, this genetic change should be considered 0% penetrant for that illness. previous definitions and formulas always result in penetrance &gt;0% due to inclusion of the background risk, which could be misleading for clinicians and patients. imprecise penetrance estimates have the potential to be detrimental by limiting diagnostic options for affected individuals. there is potential for harm in classifying low penetrant cnvs as “pathogenic”, when most individuals are either unaffected or so mildly affected that their symptoms fall within a spectrum of human normal. the updated penetrance estimates presented here may affect interpretation and reporting of cnvs, with implications for genetic counselling in both prenatal and postnatal settings. data availability all relevant data are listed within the paper and its supporting information files (supplementary discussion 1 &amp; supplementary results 1 ). code availability penetrance estimates and their 95% confidence interval can be obtained by running bootstrap_penetrance.m in octave v8.3.0. references carruth ed, young w, beer d, james ca, calkins h, jing l, et al. prevalence and electronic health record-based phenotype of loss-of-function genetic variants in arrhythmogenic right ventricular cardiomyopathy-associated genes. circ genom precis med. 2019;12:e002579. article pubmed pubmed central google scholar chen s, parmigiani g. meta-analysis of brca1 and brca2 penetrance. j clin oncol. 2007;25:1329–33. article pubmed google scholar cooper dn, krawczak m, polychronakos c, tyler-smith c, kehrer-sawatzki h. where genotype is not predictive of phenotype: towards an understanding of the molecular basis of reduced penetrance in human inherited disease. hum genet. 2013;132:1077–130. article cas pubmed pubmed central google scholar kirov g, rees e, walters jt, escott-price v, georgieva l, richards al, et al. the penetrance of copy number variations for schizophrenia and developmental delay. biol psychiatry. 2014;75:378–85. article cas pubmed google scholar vassos, collier e, holden da, patch s, rujescu c, st d, et al. penetrance for copy number variants associated with schizophrenia. hum mol genet. 2010;19:3477–81. article cas pubmed google scholar roberts jd, asaki sy, mazzanti a, bos jm, tuleta i, muir ar, et al. an international multicenter evaluation of type 5 long qt syndrome: a low penetrant primary arrhythmic condition. circulation. 2020;141:429–39. article cas pubmed pubmed central google scholar allach el khattabi l, heide s, caberg jh, andrieux j, doco fenzy m, vincent-delorme c, et al. 16p13.11 microduplication in 45 new patients: refined clinical significance and genotype-phenotype correlations. j med genet. 2018;0:1–7. google scholar chaste p, sanders sj, mohan kn, klei l, song y, murtha mt, et al. modest impact on risk for autism spectrum disorder of rare copy number variants at 15q11.2, specifically breakpoints 1 to 2. autism res. 2014;7:355–62. article pubmed pubmed central google scholar cosemans n, vandenhove l, vogels a, devriendt k, van esch h, van buggenhout g, et al. the clinical relevance of intragenic nrxn1 deletions. j med genet. 2020;57:347–55. article pubmed google scholar isles ar, ingason a, lowther c, walters j, gawlick m, stober g, et al. parental origin of interstitial duplications at 15q11.2-q13.3 in schizophrenia and neurodevelopmental disorders. plos genet. 2016;12:e1005993. article pubmed pubmed central google scholar kendall km, bracher-smith m, fitzpatrick h, lynham a, rees e, escott-price v, et al. cognitive performance and functional outcomes of carriers of pathogenic copy number variants: analysis of the uk biobank. br j psychiatry. 2019;214:297–304. article pubmed pubmed central google scholar maya i, perlman s, shohat m, kahana s, yacobsons, tennet,et al. should we report 15q11.2 bp1-bp2 deletions and duplications in the prenatal setting?.j clin med.2020;9:11. article google scholar mohan kn, cao y, pham j, cheung sw, hoffner l, ou zz, et al. phenotypic association of 15q11.2 cnvs of the region of breakpoints 1-2 (bp1-bp2) in a large cohort of samples referred for genetic diagnosis. j hum genet. 2019;64:253–5. article cas pubmed google scholar rosenfeld ja, coe bp, eichler ee, cuckle h, shaffer lg. estimates of penetrance for recurrent pathogenic copy-number variations. genet med. 2013;15:478–81. article cas pubmed google scholar tropeano m, howley d, gazzellone mj, wilson ce, ahn jw, stavropoulos dj, et al. microduplications at the pseudoautosomal shox locus in autism spectrum disorders and related neurodevelopmental conditions. j med genet. 2016;53:536–47. article cas pubmed google scholar al, shehhi m, forman eb, fitzgerald je, mcinerney v, krawczyk j, et al. nrxn1 deletion syndrome; phenotypic and penetrance data from 34 families. eur j med genet. 2019;62:204–9. article google scholar jønch ae, douard e, moreau c, van dijck a, passeggeri m, kooy f, et al. estimating the effect size of the 15q11.2 bp1-bp2 deletion and its contribution to neurodevelopmental symptoms: recommendations for practice. j med genet. 2019;56:701–10. article pubmed google scholar unique. rare chromosome &amp; gene disorder guides: rare chromosome disorder support group; 2024 available from: https://rarechromo.org/disorder-guides/ . martin cl, wain ke, oetjens mt, tolwinski k, palen e, hare-harris a, et al. identification of neuropsychiatric copy number variants in a health care system population. jama psychiatry. 2020;77:1276–85. article pubmed google scholar maulik pk, mascarenhas mn, mathers cd, dua t, saxena s. prevalence of intellectual disability: a meta-analysis of population-based studies. res dev disabil. 2011;32:419–36. article pubmed google scholar goh s, thiyagarajan l, dudding-byth t, pinese m, kirk ep. a systematic review and pooled analysis of penetrance estimates of copy-number variants associated with neurodevelopment. genet med. 2025;27:101227. article cas pubmed google scholar van paassen bw, van der kooi aj, van spaendonck-zwarts ky, verhamme c, baas f, de visser m. pmp22 related neuropathies: charcot-marie-tooth disease type 1a and hereditary neuropathy with liability to pressure palsies. orphanet j rare dis. 2014;9:38. article pubmed pubmed central google scholar chrestian n hereditary neuropathy with liability to pressure palsies. in: adam mp, feldman j, mirzaa gm, pagon ra, wallace se, bean ljh, et al., editors. genereviews(®). seattle (wa) 1993. benn pa. prenatal counseling and the detection of copy-number variants. genet med. 2013;15:316–7. article pubmed google scholar schultz lm, knighton a, huguet g, saci z, jean-louis m, mollon j, et al. copy-number variants differ in frequency across genetic ancestry groups. hgg adv. 2024;5:100340. cas pubmed pubmed central google scholar muys j, blaumeiser b, jacquemyn y, bandelier c, brison n, bulk s, et al. the belgian microarray prenatal (bemapre) database: a systematic nationwide repository of fetal genomic aberrations. prenat diagn. 2018;38:1120–8. article cas pubmed google scholar zhou d, gochman p, broadnax dd, rapoport jl, ahn k. 15q13.3 duplication in two patients with childhood-onset schizophrenia. am j med genet b neuropsychiatr genet. 2016;171:777–83. article cas pubmed pubmed central google scholar gillentine ma, schaaf cp. the human clinical phenotypes of altered chrna7 copy number. biochem pharmacol. 2015;97:352–62. article cas pubmed pubmed central google scholar clingen. 15q13.3 recurrent region (d-chrna7 to bp5) (includes chrna7 and otud7a): clingen dosage sensitivity curation page; 2018 [updated 10/5/2018. available from: https://dosage.clinicalgenome.org/clingen_region.cgi?id=isca-46295 . cai m, que y, chen x, chen y, liang b, huang h, et al. 16p13.11 microdeletion/microduplication in fetuses: investigation of associated ultrasound phenotypes, genetic anomalies, and pregnancy outcome follow-up. bmc pregnancy childbirth. 2022;22:913. article cas pubmed pubmed central google scholar muys j, jacquemyn y, blaumeiser b, bourlard l, brison n, bulk s, et al. prenatally detected copy number variants in a national cohort: a postnatal follow-up study. prenat diagn. 2020;40:1272–83. article pubmed google scholar smajlagić d, lavrichenko k, berland s, helgeland ø, knudsen gp, vaudel m, et al. population prevalence and inheritance pattern of recurrent cnvs associated with neurodevelopmental disorders in 12,252 newborns and their parents. eur j hum genet. 2021;29:205–15. article pubmed google scholar fu j, liao c, collins r, wang l, ben-isvy d, brand h, et al. rare coding cnvs from exome sequenced individuals in gnomad v4 gnomad: gnomad; 2023 [updated 1/11/2023. available from: https://gnomad.broadinstitute.org/news/2023-11-v4-copy-number-variants/ . babadi m, fu jm, lee sk, smirnov an, gauthier ld, walker m, et al. gatk-gcnv enables the discovery of rare copy number variants from exome sequencing data. nat genet. 2023;55:1589–97. article cas pubmed pubmed central google scholar download references acknowledgements we would like to extend our sincere gratitude to associate professor kate simms for her invaluable assistance in proofreading this manuscript. we would also like to thank dr rhys bowden for early contributions to some of the mathematical aspects of this work. funding open access funding enabled and organized by caul and its member institutions. author information authors and affiliations school of women and children’s health, university of new south wales, sydney, nsw, australia shuxiang goh, mark pinese &amp; edwin p. kirk new south wales health, sydney, nsw, australia shuxiang goh, tracy dudding-byth &amp; edwin p. kirk genetics of learning disability, newcastle, nsw, australia tracy dudding-byth university of newcastle, newcastle, nsw, australia tracy dudding-byth children’s cancer institute, sydney, nsw, australia mark pinese centre for clinical genetics, sydney children’s hospital, sydney, nsw, australia edwin p. kirk nsw health pathology randwick genomics laboratory, sydney, nsw, australia edwin p. kirk authors shuxiang goh view author publications search author on: pubmed google scholar tracy dudding-byth view author publications search author on: pubmed google scholar mark pinese view author publications search author on: pubmed google scholar edwin p. kirk view author publications search author on: pubmed google scholar contributions conceptualization: sg, ek; methodology: sg, ek; data curation: sg; formal analysis: sg; investigation: sg, ek; resources: sg; software: sg; supervision: ek, mp, td; validation: sg, ek; visualization: sg; writing-original draft: sg; writing-review &amp; editing: ek, mp, td. corresponding author correspondence to shuxiang goh . ethics declarations competing interests the authors declare no competing interests. ethics all data used in this study is publicly available. there are no other relevant ethical declarations. additional information publisher’s note springer nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. supplementary information supplementary materials supplementary table 6 &amp; 7 penetrance calculation rights and permissions open access this article is licensed under a creative commons attribution 4.0 international license, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the creative commons licence, and indicate if changes were made. the images or other third party material in this article are included in the article’s creative commons licence, unless indicated otherwise in a credit line to the material. if material is not included in the article’s creative commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. to view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/ . reprints and permissions about this article cite this article goh, s., dudding-byth, t., pinese, m. et al. updated penetrance estimates for recurrent copy number variants – an improved definition and formula. eur j hum genet (2025). https://doi.org/10.1038/s41431-025-01948-0 download citation received : 21 december 2024 revised : 19 august 2025 accepted : 11 september 2025 published : 15 october 2025 doi : https://doi.org/10.1038/s41431-025-01948-0 share this article anyone you share the following link with will be able to read this content: get shareable link sorry, a shareable link is not currently available for this article. copy shareable link to clipboard provided by the springer nature sharedit content-sharing initiative download pdf advertisement window.sn = window.sn || {}; window.sn.libs = window.sn.libs || {}; window.sn.libs.ads = window.sn.libs.ads || {}; window.sn.libs.ads.slotconfig = window.sn.libs.ads.slotconfig || {}; window.sn.libs.ads.slotconfig['right'] = { 'pos': 'right', 'type': 'article', 'path': 's41431-025-01948-0' }; window.sn.libs.ads.slotconfig['kwrd'] = 'clinical+genetics,cytogenetics,neurodevelopmental+disorders'; window.sn.libs.ads.slotconfig['subjmeta'] = '1405,208,2489,366,631'; explore content research articles reviews &amp; analysis news &amp; comment current issue collections follow us on twitter sign up for alerts rss feed about the journal journal information open access fees and funding about the editors about the partner contact for advertisers subscribe publish with us for authors &amp; referees language editing services open access funding submit manuscript window.datalayer = window.datalayer || []; window.datalayer.push({ page: { content: { fundingwidget: "true", } } }); search search articles by subject, keyword or author show results from all journals this journal search advanced search quick links explore articles by subject find a job guide to authors editorial policies european journal of human genetics ( eur j hum genet ) issn 1476-5438 (online) issn 1018-4813 (print) nature.com sitemap about nature portfolio about us press releases press office contact us discover content journals a-z articles by subject protocols.io nature index publishing policies nature portfolio policies open access author &amp; researcher services reprints &amp; permissions research data language editing scientific editing nature masterclasses research solutions libraries &amp; institutions librarian service &amp; tools librarian portal open research recommend to library advertising &amp; partnerships advertising partnerships &amp; services media kits branded content professional development nature awards nature careers nature conferences regional websites nature africa nature china nature india nature japan nature middle east privacy policy use of cookies your privacy choices/manage cookies legal notice accessibility statement terms &amp; conditions your us state privacy rights &copy; 2025 springer nature limited

## Scraping Notes

- Successfully scraped from DOI.org
