# Conservation of human NMDA receptor subunits and disease variants in zebrafish.

**DOI:** 10.21203/rs.3.rs-7151578/v1
**Authors:** ['Erica RNebet', 'ChristieannAprea', 'Josiah DZoodsma', 'WilliamRaab', 'Howard ISirotkin', 'Lonnie PWollmuth']
**Journal:** Research square
**Year:** 
**Scraped from:** DOI.org
**Scraped by:** agent_b
**Scraping date:** 2025-10-21T11:04:12.508367

## Abstract

NMDA receptors (NMDARs) are widely expressed, glutamate-gated ion channels that play key roles in brain development and function. Variants have been identified in the 
We find that the human NMDAR subunits are generally well conserved across zebrafish paralogs. Moreover, variants classified as pathogenic and putatively pathogenic are highly conserved, reflecting the importance of key protein regions to neurotypical receptor function. Positions with putatively benign and benign variants are less conserved. Across NMDAR domains, the transmembrane domain is most highly conserved, followed by the ligand-binding domain, which maintains conservation of amino acids that participate in the binding of ligands. The N-terminal domain is less well conserved but aligned homology models show high degrees of structural similarity. The C-terminal domain is the most poorly conserved region across zebrafish paralogs, but certain key regions that undergo phosphorylation, palmitoylation, and ubiquitylation as well as protein-binding motifs are better conserved.
Our findings highlight a strong conservation of human NMDAR subunits in zebrafish, with some exceptions. The ligand-binding domain, the transmembrane domain forming the ion channel and the short polypeptide linkers that connect them are highly conserved. The N- and C-terminal domains are less conserved but functional motifs in general, except for the Zn

## Full Text

## Abstract
background nmda receptors (nmdars) are widely expressed, glutamate-gated ion channels that play key roles in brain development and function. variants have been identified in the grin genes encoding nmdar subunits that are linked to neurodevelopmental disorders, among other manifestations. zebrafish are a powerful model to study brain development and function given their rapid development and ease of genetic manipulation. as a result of an ancient genome duplication, zebrafish possess two paralogues for most human nmdar subunits. to evaluate the degree of conservation between human nmdar subunits and their respective zebrafish paralogues, we carried out detailed in silico analyses, with an emphasis on key functional elements. to further assess the suitability of zebrafish for modeling nmdar-associated neurodevelopmental disorders, we analyzed the conservation of positions with identified missense variants. results we find that the human nmdar subunits are generally well conserved across zebrafish paralogs. moreover, variants classified as pathogenic and putatively pathogenic are highly conserved, reflecting the importance of key protein regions to neurotypical receptor function. positions with putatively benign and benign variants are less conserved. across nmdar domains, the transmembrane domain is most highly conserved, followed by the ligand-binding domain, which maintains conservation of amino acids that participate in the binding of ligands. the n-terminal domain is less well conserved but aligned homology models show high degrees of structural similarity. the c-terminal domain is the most poorly conserved region across zebrafish paralogs, but certain key regions that undergo phosphorylation, palmitoylation, and ubiquitylation as well as protein-binding motifs are better conserved. conclusions our findings highlight a strong conservation of human nmdar subunits in zebrafish, with some exceptions. the ligand-binding domain, the transmembrane domain forming the ion channel and the short polypeptide linkers that connect them are highly conserved. the n- and c-terminal domains are less conserved but functional motifs in general, except for the zn 2+ binding site in glun2a paralogues, are more highly conserved relative to the entire domain. overall, our findings support the utility of zebrafish as a model for studying neurodevelopment and disease mechanisms and provide a template for rigorously considering the relationship between human and zebrafish paralogues. grin genes clinvar gnomad sequence alignments disease-associated variants n-terminal domain ligand-binding domain transmembrane domain c-terminal domain ion channel figures figure 1 figure 2 figure 3 figure 4 figure 5 figure 6 figure 7 figure 8 background rapid cell-to-cell signaling in the nervous system occurs at specialized structures called synapses. at these sites, the electrical activity of the upstream or presynaptic neuron is converted into the release of a chemical neurotransmitter. this is then converted back into an electrical signal in the postsynaptic neuron by specialized ion channels that are gated by a neurotransmitter, initiating downstream biological signaling cascades. glutamate, the primary excitatory neurotransmitter in the brain, activates a variety of receptors that are essential for synaptic transmission and plasticity. glutamate-gated ion channels play fundamental roles in signaling in both the developing and mature nervous systems ( 1 – 9 ). the nmda receptor (nmdar) is a prominent glutamate-gated ion channel subtype ( 1 , 2 ). due to its distinct biophysical properties and unique ionotropic and non-ionotropic signaling mechanisms, it contributes to a wide array of nervous and non-nervous system functions ( 2 , 10 – 14 ). numerous missense, frameshift, and nonsense mutations (or variants) have been identified in the genes that encode nmdar subunits and are associated with neurodevelopmental disorders, including autism spectrum disorders (asd), intellectual disability, developmental delay, seizures, and schizophrenia ( 15 – 23 ). however, how these variants lead to clinical phenotypes is poorly understood. zebrafish are a powerful model organism to study neurodevelopmental phenotypes and disorders, including those linked to nmdars ( 19 , 24 – 32 ). zebrafish develop rapidly and share fundamental developmental and signaling pathways with humans and other vertebrates ( 33 ); they are transparent, allowing for dynamic and non-invasive observations of developmental and cellular processes; and they display robust behaviors that are easily assayed at early larval stages. moreover, larval zebrafish readily take up small molecules from their environment, enabling non-invasive and high-throughput readout of nervous system function in the context of behavioral drug screens ( 34 – 36 ). nmdars are heterotetramers composed of two obligatory glun1 (encoded by grin1 ) subunits. they are expressed as either diheterotetramers with two glun1 and two glun2 ( grin2a-d ) subunits or triheterotetramers of some combination of glun2 and/or glun3 ( grin3a-b ) subunits ( 2 ) (fig. 1 a). these nmdar subunits are differentially expressed over the course of neurodevelopment, with glun2a, glun2b, and glun2d being highly expressed during early development, while glun2c, glun3a, and glun3b expression increases into adulthood ( 1 ). moreover, nmdars display distinct functional and pharmacological properties based on their glun2 or glun3 composition ( 2 ). each subunit consists of four principal domains: an n-terminal domain (ntd), a ligand-binding domain (lbd), a transmembrane domain (tmd) that forms the ion channel, and a c-terminal domain (ctd) (fig. 1 b). disease-associated variants (davs) in grin1 , grin2a , grin2b , and grin2d have been identified in pediatric patients with neurodevelopmental disorders ( 15 – 18 , 21 ). these disorders often present with multiple comorbidities, reflecting overlapping neurodevelopmental phenotypes ( 37 ). generally, patients with davs in grin1 present with developmental delay and intellectual disability, though a large proportion of patients also experience seizures ( 15 ). in contrast, grin2a and grin2d are predominantly associated with seizure disorders ( 16 , 18 ). grin2b is a high-risk gene for autism spectrum disorder (asd) ( 17 , 38 , 39 ), as are grin1 and grin2a ( 40 – 42 ). still, patients with such neurodevelopmental disorders generally experience a variety of both central and peripheral nervous system symptoms ( 43 – 46 ). nmda receptors are highly conserved across species ( 47 ). due to an ancient genome duplication event, zebrafish retain two paralogues for most nmdar genes (e.g., grin1a &amp; grin1b ) ( 24 , 48 ). patch clamp electrophysiology experiments show that zebrafish paralogues display similar functional properties to each other and to their mammalian orthologues ( 27 , 28 ). nevertheless, it remains unclear how well the human nmdar genes, including positions with identified missense variants, are conserved in zebrafish. to address this, we investigated nmdar conservation in zebrafish by taking advantage of amino acid sequence alignments ( 28 ) and assessed the conservation of all human nmdar subunits and zebrafish paralogues. we also compared the conservation of positions with missense variants identified in human patients for nmdar subunits expressed during early neurodevelopment. finally, we investigated the conservation of critical functional regions within each structural domain. through this analysis, we define the extent to which critical functional components and regions, with relevance to disease, are conserved across zebrafish nmdars, providing a template to use zebrafish as a model to study nmdars in neurodevelopment and disease pathology. methods nmdar sequences and sequence alignments amino acid sequences for human genes are (uniprot with protein ids): glun1 (q05586-5; ntd splice isoform, nr1-3), glun2a (q12879-1), glun2b (q13224), glun2c (q14957), glun2d (o15399), glun3a (q8tcu5), and glun3b (o60391). for mouse subunits: mglun1 (p35438), mglun2a (p35436), mglun2b (q01097), mglun2c (q01098), mglun2d (q03391), mglun2a (a2air5), and mglun3b (q91zu9). for zebrafish paralogs: zglun1a (f1r366), zglun1b (q6zm67), zglun2aa (a0a8m9pxd6), zglun2ab (f1qde5), zglun2ba (a0a8m9q0i1), zglun2bb (a0a8m1nh89), zglun2ca (e7fh62), zglun2cb (a0a8m3b7k5), zglun2da (i3ni77), zglun2db (e7f3z4), zglun3a (a0a8m3awh5), zglun3ba (a0a8m9p9f6), and zglun3bb (a0a8m6z151). breakpoints for nmdar subunit domains and subdomains are based on structures 6whr (rat glun1-glun2b) and 7eu7 (human glun1-glun2a) ( 49 , 50 ). human and zebrafish protein sequences were aligned and analyzed for amino acid identity and similarity using align sequences protein blast (blastp), which is publicly available through the national center for biotechnology information (ncbi) or emboss needle pairwise sequence alignment (psa). for each nmdar subunit, the zebrafish paralogs were individually aligned to the human protein. amino acid similarity is defined based on the blosum 62 substitution matrix. for the ntd, lbd, and tmd, these alignments were generally straightforward as the sequences do not diverge greatly in length. in contrast, the ctd is quite large, with the zebrafish sequences generally larger than human. for these alignments, we entered the entire sequence into the alignment tool and used the given result, though this led to a fragmentation of the ctd such that certain portions of the sequence were aligned while others were not. we used the resulting domain-specific conservation values to calculate the conservation of the total protein. variants analysis human nmdar variants were identified using clinvar and gnomad (table 2 ). clinvar . results for each gene were downloaded on september 28th, 2023, using clinvar&#x27;s bulk download option ( https://www.ncbi.nlm.nih.gov/clinvar/ ). genomic variants that did not result in protein coding changes were removed from the analysis. in the case of grin3a , protein coding changes in the ppp3r2 gene, a single exon gene whose coding sequence resides within the genomic region of grin3a and is downloaded alongside grin3a due to this, was also removed. in cases where multiple protein coding annotations exist for the same variant, only the annotation fitting the canonical transcript was retained. gnomad : genes were searched for by name on gnomad v4.0.0 ( https://gnomad.broadinstitute.org/ ) and missense, insertion, deletion, and loss-of-function variant information was exported on september 28th, 2023 as comma separated value (csv) files for each gene. hgvs annotation (protein change) was converted to letter-number-letter annotation. gene variants from clinvar and gnomad were merged using r studio 2023.06.1 build 524 based on the letter-number-letter annotation. the resulting merged file was checked for redundant information or variant duplicates, which were removed. missense variants were extracted and grouped by pathogenicity as pathogenic, putatively pathogenic, complex, putatively benign, and benign variants based on the criteria in table 2 . assessment of conservation of positions with identified human variants was performed using the generated sequence alignments. nmdar domain-specific analysis to further refine the relationship between human and zebrafish nmdar proteins, we analyzed the tmd, lbd-tmd linkers, lbd, ntd, and ctd of glun1, glun2a, glun2b, and glun2d for conservation of key structural and functional features. transmembrane domain (tmd) and ligand-binding domain-transmembrane domain (lbd-tmd) linkers. key regions involved in channel gating, mg 2+ block, and ca 2+ flux were identified as described in the literature and analyzed for conservation ( 51 – 57 ). ligand-binding domain (lbd) . key regions and specific amino acids involved in van-der-waals interactions with the α-carbon of the ligand, interactions with the α-amino group, electrostatic interactions with the α-carboxyl group, and interactions with the amino acid side chain were identified as described in ramos-vicente et. al. 2021 and analyzed for conservation ( 47 ). n-terminal domain (ntd) . key amino acid positions required for the binding of zinc and polyamines were identified via a literature search and their conservation assessed ( 2 , 58 , 59 ). we also examined the conservation of putative n-linked glycosylation sites ( 60 ). to generate homology models of the zebrafish ntds, we used the template function in swiss-model ( 61 , 62 ). as reference structures, we used rodent ntd structures 5tq0 (glun1-glun2a in the presence of edta, no bound zn 2+ for glun1 and glun2a), 5tpz (glun1-glun2b apo state for glun1 and glun2b), and 5tpw (glun1-glun2a with zn 2+ in complex with glun2a) ( 58 ). sequences for human (hglun1, hglun2a, and hglun2b) or zebrafish (zglun1a, zglun1b, glun2aa, glun2ab, glun2ba, glun2bb) ntds were entered individually into swiss-model to generate homology models. the subsequent models were then loaded into pymole (human and zebrafish paralog) and aligned. the resultant root mean square deviation (rmsd) is reported. carboxy-terminal domain (ctd) . ctd-specific conservation in zebrafish was assessed for the glun1, glun2a, and glun2b subunits. glun2d was excluded from this analysis due to the poor conservation of the ctd in these paralogs relative to human. to assess functional relevance, we focused on post-translational modifications (ptms), specifically phosphorylation sites (serine, threonine, tyrosine), ubiquitylation sites (lysine), and palmitoylation sites (cysteine), using a combination of experimentally validated data from phosphositeplus (psp) and published literature ( 2 , 11 ). phosphorylation sites with identified kinases or well-characterized functional roles were classified as ‘kinase-regulated phosphorylation sites.’ the remaining phosphorylation sites identified in psp were defined as ‘phosphoproteomic-identified sites.’ note, sites in psp (based on rodents) that did not have a phosphorylation site in humans were excluded from our analysis. in addition, known linear protein-binding motifs were annotated and analyzed for conservation ( 2 , 11 ). to assess conservation of protein-binding motifs in the ctds, we categorized sites as either short linear motifs (slims) or broader interaction ‘regions’. slims were defined as experimentally validated, compact sequence elements (typically &lt; 15 residues) that mediate specific and direct interactions with known protein partners. broader regions were defined as extended sequences (&gt; 15 residues) shown to directly bind proteins in biochemical or structural studies but lacking a discrete consensus sequence. all motifs were curated from published literature where discrete sequences were experimentally validated per motif ( 11 , 63 – 80 ), and sequence conservation was examined. results our goal is to compare the structural similarity of zebrafish nmdar (zglun) paralogues to human nmdars (glun), especially regarding human variants. since mice are a common animal model ( 81 ), as a reference-point for nmdar gene conservation, we initially made global comparisons of both mouse and zebrafish nmdar subunits to human. global comparison of mouse and zebrafish orthologues to human nmdars nmdars are heterotetramers composed of two obligate glycine-binding glun1 subunits and two glutamate-binding glun2 subunits or a combination of glun2 and glycine-binding glun3 subunits ( 1 , 2 ) (fig. 1 a,b). glycine and glutamate act as co-agonists for glun2-containing receptors, with binding of both required for receptor activation, while glun3-containing receptors are activated by glycine binding. as an initial assessment of the structural similarity between nmdar orthologues, we made global sequence alignments using the ncbi blastp or emboss psa tools. not surprisingly, given evolutionary proximity, the mouse subunits are highly conserved relative to human: glun1 (97% identity &amp; similarity), glun2a (95% identity; 98% similarity), glun2b (98% identity; 99% similarity), glun2c (88% identity; 90% similarity), glun2d (97% identity &amp; similarity), glun3a (92% identity; 95% similarity), and glun3b (77% identity; 83% similarity) (table 1 ). on the other hand, the zebrafish paralogues show reduced conservation. the obligatory glun1 subunits are the most highly conserved, both in terms of identity (≥ 84%) and similarity (≥ 89%) (table 1 ). the other subunits show various degrees of identity and similarity: glun2a (≥ 58% identity; ≥79% similarity), glun2b (≥ 65% identity; ≥73% similarity), glun2c (≥ 50% identity; ≥64% similarity), glun2d (≥ 43% identity; ≥54% similarity), glun3a (63% identity; 78% similarity), and glun3b (≥ 54% identity, ≥ 62% similarity) (table 1 ). these subunits, thus, retain ≥ 50% identity with respect to the human protein, except for zglun2da and zglun2db, which only share 43% and 46% sequence identity, respectively (table 1 ). with respect to the zebrafish paralog pairs, there are instances in which one paralog is better conserved than the other. the glun1 paralogs are comparably conserved (table 1 ). however, the zglun2aa paralog is more highly conserved than zglun2ab while the zglun2bb shows increased conservation relative to zglun2ba (table 1 ). the glun2c, glun2d, and glun3b paralogs are conserved to similar extents (table 1 ). nevertheless, despite lower conservation relative to mouse, zebrafish offer many technical advantages (see background). in addition, what is critical is the conservation of specific structural and functional domains and subdomains. table 1 overall protein conservation in mouse and zebrafish relative to human human mouse zebrafish paralogue a zebrafish paralogue b protein length id. sim. protein length id. sim. protein length id. sim. protein length n1 938 97% 97% 938 84% 89% 966 (-28) 86% 90% 937 (-1) n2a 1464 95% 98% 1464 68% 89% 1460 (+ 6) 58% 79% 1445 (-9) n2b 1484 98% 99% 1482 (-2) 65% 73% 1417 (-31) 67% 81% 1770 (+ 322) n2c 1233 88% 90% 1239 (+ 6) 50% 64% 1400 (-176) 59% 79% 1328 (+ 95) n2d 1336 97% 97% 1323 (-13) 43% 54% 1916 (+ 580) 46% 57% 1676 (+ 340) n3a 1115 92% 95% 1135 (+ 20) 63% 78% 1111 (-4) n3b 1043 77% 83% 1003 (-40) 54% 62% 1118 (+ 75) 62% 68% 1114 (+ 71) numbers indicate the percent of mouse or zebrafish paralog conservation with respect to the human protein – ‘id.’ is identity and ‘sim.’ is similarity. the zebrafish a and b paralogs are indicated for each protein. glun3a has only a single paralog in zebrafish. for the mouse and zebrafish protein length, numbers in parentheses indicate the difference in amino acid number relative to human. conservation of specific structural elements in zebrafish paralogues for more rigorous comparisons, we examined the conservation within individual subunit domains and subdomains (fig. 1 ). the structure of nmdars, like all iglurs, consists of 4 domains that are intrinsic to each individual subunit (fig. 1 a,b): the extracellular ntd and lbd; the tmd forming the ion channel as well as the linkers that connect the lbd to the tmd, the lbd-tmd linkers; and the intracellular ctd ( 2 , 53 ). within individual subunits, the lbd is formed by two disparate sequences, s1 and s2, that fold up into the d1 and d2 lobes in the 3-dimensional structure (83, 84). in general, d1 is made up primarily of s1 and d2 of s2, but there is considerable swapping between lobes. the tmd is formed by three transmembrane segments, m1, m3, and m4, and an m2-pore loop. the lbd-tmd linkers are referred to as either s1-m1, m3-s2, and s2-m4 (linear sequence) or d2-m1, m3-d2, and d1-m4 (3-dimensional structure). this distinction is important functionally since d1 and d2 undergo different movements that are translated differently to the tmd ( 85 , 86 ). nevertheless, we will use the s1-m1, m3-s2, and s2-m4 designation for ease of comparison to structures. we began by dividing the zebrafish paralogues into their respective domains and subdomains based on our breakpoints (see methods). interestingly, the amino acid sequences surrounding these breakpoints were well conserved in zebrafish – the majority were identical (not shown). this allowed for ease of division of the zebrafish proteins into their respective domains and subdomains. as an initial assessment of the structural similarity between human and zebrafish nmdars, we made detailed sequence alignments using the ncbi blastp or emboss psa tools (table 1 , fig. 1 ). in terms of length of domains, the zebrafish protein sequence for glun1, glun2, and glun3 subunits was identical to the human sequence for the tmd (m1, m1-m2, m2, m2-m3, m3, &amp; m4) and lbd-tmd linkers (s1-m1, m3-s2, &amp; s2-m4) (fig. 1 c). the lbd (s1 &amp; s2) was also quite similar in length, with no changes in the length of s2 across the paralogs and only slight increases or decreases in amino acid number in the s1 of zglun1a, zglun1b, zglun2ab, zglun2ca, zglun2ca, zglun2da, and zglun2db (fig. 1 c). the ntd and ctd were less conserved in terms of size, wherein almost every zebrafish paralog has a different number of amino acids compared to the human. this is especially notable for the glun2b ntd which either lacks 130 amino acids (zglun2ba) or has an additional 74 amino acids (zglun2bb) as well as for the ctd where, for example, zglun2da and zglun2db have a ctd that is more than twice as long as that in the human protein (fig. 1 c). table 2 variant designation glun designation clinvar gnomad 1 2a 2b 2c 2d 3a 3b pathogenic “pathogenic” absent 49 (5%) 58 (4%) 81 (5%) 0 12 (&lt; 1%) 0 0 putatively pathogenic “uncertain” absent 123 (13%) 214 (15%) 182 (12%) 4 (&lt; 1%) 111 (8%) 5 (&lt; 1%) 8 (&lt; 1%) complex “pathogenic” present 3 (&lt; 1%) 5 (&lt; 1%) 6 (&lt; 1%) 0 1 (&lt; 1%) 0 0 putatively benign “uncertain” present 80 (9%) 320 (22%) 199 (13%) 31 (3%) 223 (17%) 30 (3%) 68 (7%) benign “benign” or absent absent/ present or present 401 (43%) 813 (55%) 672 (45%) 945 (77%) 863 (27%) 59 (5%) 945 (91%) number of positions with missense variants that have been identified in each subunit per designation. percentages report the proportion of the total protein amino acids at which such variants have been identified. we used these sequence alignments to assess the degree of amino acid identity and similarity between domains and subdomains (fig. 1 d,e). the obligatory glun1 subunit showed the highest degree of conservation. the tmd and lbd-tmd linkers are 100% conserved. the lbd is also well conserved with s1 and s2 showing ≥ 92% identity and ≥ 93% similarity. the ntd is somewhat less conserved (≥ 80% identity, ≥ 88% similarity), as is the ctd (≥ 53% identity; ≥58% similarity). this trend of conservation across nmdar subunits – the lbd and tmd are most highly conserved and the ntd and ctd less so – is maintained in the glun2 and glun3 subunits (fig. 1 d,e). generally, the glun2 subunits maintain a higher sequence identity than the glun3 subunits. overall, this sequence analysis suggests that the lbd and tmd are highly conserved whereas the ntd and ctd show less conservation. categorizing the pathogenicity of missense variants nmdars with missense, frameshift, and nonsense variants have been identified in pediatric patients with neurodevelopmental disorders ( 21 , 22 , 87 ). we therefore wanted to assess the extent to which variants are conserved in zebrafish to aid in guiding future studies. for this analysis, we focused solely on missense variants. while nonsense and frameshift mutation can be modelled in various ways, such as by mutating a portion of the protein to induce a loss-of-function, the conservation of the specific protein sequence is somewhat less relevant. in contrast, the single amino acid change associated with missense variants is critical to dissecting how such variants alter receptor function and lead to disease phenotypes. using publicly available datasets of nmdar variants from clinvar and gnomad, we identified missense variants in all nmdar subunits (see methods). we first categorized these variants with regards to their pathogenicity (table 2 ). clinvar reports variants that have been identified in patients with symptomatic disease ( 88 – 90 ), while gnomad reports variants identified in the healthy population ( 91 ). thus, in general, we define variants present in clinvar but absent in gnomad as pathogenic while those with the opposite reporting as benign. with this approach, pathogenic and putatively pathogenic variants were identified in glun1, glun2a, glun2b, and glun2d while minimal to no such variants were identified in glun2c, glun3a, and glun3b (table 2 ). notably, these genes with known pathogenic variants are expressed early in development, contributing to developmental processes while their dysfunction often leads to disease phenotypes ( 21 , 87 ). on the other hand, glun2c, glun3a, and glun3b have no pathogenic and a small number of putatively pathogenic variants (table 2 ). we therefore focus solely on glun1, glun2a, glun2b, and glun2d for further analysis. pathogenic and putatively pathogenic variant positions are well conserved in zebrafish using the criteria in table 2 , amino acid positions with pathogenic missense variants in glun1, glun2a, glun2b, and glun2d are completely conserved in zebrafish paralogues (fig. 2 a-d). putatively pathogenic variants are also conserved in these nmdar subunits, though less so than pathogenic variants. the highest degree of conservation of putatively pathogenic variants is in the glun1 paralogues (≥ 88% identity; 93% similarity). putatively pathogenic variants in the glun2 subunits are also reasonably conserved: zglun2aa (70% identity; 92% similarity), zglun2ab (62% identity; 84% similarity), zglun2ba (65% identity; 71% similarity), zglun2bb (72% identity; 80% similarity), and zglun2da and zglundb (≥ 46% identity; ≥59% similarity) (fig. 2 e,f). in general, the highest level of conservation is in the tmd and tmd-lbd linkers, followed by the lbd, and less so in the ntd and ctd (fig. 2 g,h). benign and putatively benign variant positions show reduced conservation we carried out a comparable analysis as above for benign and putatively benign variants (table 2 ). we hypothesized that positions with more benign variants would be less well conserved than pathogenic ones. indeed, the general trend is consistent with this idea (fig. 3 ). putatively benign variants showed a reduced conservation compared to putatively pathogenic variants (putatively pathogenic vs putatively benign): zglun1a (88% vs. 78% identity; 93% vs. 88% similarity), zglun1b (89% vs. 79% identity; 93% vs. 84% similarity), zglun2aa (70% vs. 60% identity; 92% vs. 86% similarity), zglun2ab (62% vs. 48% identity; 84% vs. 68% similarity), zglun2ba (65% vs. 45% identity; 71% vs. 55% similarity), zglun2bb (72% vs. 55% identity; 80% vs. 70% similarity), zglun2da (46% vs. 47% identity; 59% vs. 61% similarity), and zglun2db (47% vs. 44% identity; 61% vs. 56% similarity) (fig. 2 e,f vs fig. 3 a,b). for these putatively benign variants, the greatest extent of conservation tends to occur in the tmd and lbd-tmd linkers, followed by the lbd, and then the ntd and ctd (fig. 3 c,d). for benign variants, however, the decrease in conservation plateaus, with identity and similarity values comparable to the putatively benign variants, if not slightly higher (fig. 3 e-f). the subdomain-specific conservation trend for these variants follows that seen on the level of the whole receptor (fig. 1 ), wherein the tmd, lbd-tmd linkers, and lbd are generally more conserved and the ntd and ctd less so (fig. 3 g,h). although diverging from our prediction as we do not note a substantial decrease in the conservation of benign variants with respect to the putatively benign variants, this result reflects the overall conservation of the nmdar subunits themselves (benign vs total protein): zglun1 (78% vs. ≥84% identity; 84% vs. ≥89% similarity), zglun2aa (62% vs. 68% identity; 86% vs. 89% similarity), zglun2ab (49% vs. 58% identity; 74% vs. 79% similarity), zglun2ba (54% vs. 65% identity; 64% vs. 73% similarity), and zglun2bb (62% vs. 67% identity; 74% vs. 81% similarity), zglun2da (50% vs. 43% identity; 64% vs. 54% similarity), and zglun2db (49% vs. 46% identity; 62% vs. 57% similarity) (fig. 3 e-f, table 1 ). furthermore, this result reflects the relatively large number of benign variants in comparison to other variant groups (table 2 ). critical tmd and lbd-tmd linker components are completely conserved elements forming the ion channel – the tmd (m1, m1-m2, m2, m2-m3, m3, &amp; m4) – and the lbd-tmd linkers (s1-m1, m3-s2, &amp; s2-m4) were consistently highly conserved (fig. 1 ). nevertheless, we assayed conservation of critical functional motifs. the s1-m1 linker is a critical element for channel gating and is completely conserved (fig. s1 ) ( 56 ). the m2 subdomain contains what is known as the ‘n’ and ‘n + 1’ sites, which participate in mg 2+ block and ca 2+ permeation through the channel ( 2 , 55 ) and are conserved in zebrafish (fig. s1 , dark orange). the tmd m3 segment contains the sytanlaaf motif, which plays a key role in ion channel gating and is known to be highly conserved across species. this motif is conserved in all zebrafish glun1 and glun2 paralogs (fig. s1 , orange). lastly, the ‘drpeer’ motif in m3-s2 of glun1, an important component in the high ca 2+ permeability of nmdars ( 52 ) is conserved (fig. s1 , light orange). hence, core gating and permeation elements of the tmd and associated regions are highly conserved suggesting that basic mechanisms of ion channel gating and block/permeation are preserved. lbd elements required for agonist binding are completely conserved the glun1 lbd binds glycine or d-serine while the glun2 lbd binds glutamate or aspartate (fig. s2a,b). overall, the lbds are highly conserved in zebrafish relative to humans (fig. 1 ), as are specific amino acid side chains in s1 and s2 that are involved in binding endogenous agonists ( 47 ) (fig. s2c). these include sites that participate in van-der waals interactions with the α-carbon of the ligand (darkest red), interactions with the α-amino group (dark red), electrostatic interactions with the α-carboxyl group (red), and interactions with the amino acid side chain (light red) (fig. s2c). conservation of structure-function elements in the ntd in terms of overall sequence comparisons, the ntd shows variable degrees of conservation across nmdar subunits. the glun1 zebrafish paralogs maintain the highest degree of conservation relative to human (80% identity; ≥88% similarity) (fig. 1 ). the glun2 subunits show decreased ntd conservation (49–75% identity; 59–88% similarity) (fig. 1 ). given this reduced conservation, we assayed the conservation of key structure-function motifs in zebrafish paralogues. among its many roles, the ntd participates in interactions with ions and molecules that modulate receptor activity ( 58 , 59 ) (fig. 4 a,b). this is most notable for glun2a and to a lesser extent glun2b as they bind zinc, which inhibits receptor function by reducing channel open probability ( 58 ). in glun2a, four amino acids coordinate zinc binding: glun2a-his44, glun2a-his128, glun2a-glu266, and glun2a-asp282 ( 58 ). of these key sites, three are conserved (fig. 4 c) but in zglun2aa and zglun2ab, a key residue involved in zn 2+ binding, glun2a-his44, is not conserved (fig. 4 c). for glun2b, only two amino acid residues participate in zinc binding: glun2b-his127 and glun2b-glu284 ( 58 ). while glun2b-glu284 is conserved, the glun2b-his127 position in zglun2ba is missing due to alterations in ntd sequence length leading to gaps in the alignments, though it is similar now as a positively charged arginine instead of a histidine in zglun2bb (fig. 4 c). the ntd of glun2b also binds polyamines, which enhance receptor activity ( 59 ). nine amino acids are involved in the binding of polyamines. these sites in zglun2ba and zglun2bb are completely conserved except for one amino acid in zglun2bb that is similar but not identical, now a negatively charged aspartate instead of glutamate (fig. 4 d). most of this functional component of glun2b is, thus, conserved in zebrafish. additionally, the nmdar ntd has glycosylation sites, and their glycosylation regulates the trafficking of nmdars to the cell membrane ( 60 ). given the consensus sequence for arginine (n-) glycosylation (n- x -s/t), glun1 has seven, glun2a has three, and glun2b has three putative glycosylation sites in the ntd ( 60 ). their conservation in the zebrafish paralogues is somewhat variable: zglun1a &amp; zglun1b (5 of 7 sites), zglun2aa (3 of 3 sites), zglun2ab (2 of 3 sites), zglun2ba (2 of 3 sites), and zglun2bb (3 of 3 sites) (fig. 4 e). to the best of our knowledge, specific glycosylation sites in glun2d have yet to be identified. of note, the glun1 glycosylation sites glun1-n203 and glun1-n368 are necessary for receptor surface expression ( 60 ), and these sites are conserved in the zebrafish paralogues. while the critical functional features of the lbd were completely conserved in zebrafish relative to human nmdars, the conservation of key regions in the ntd occasionally diverged, most notably that for zn 2+ binding. to further explore the structural composition of the glun1, glun2a, and glun2b ntds, we used swiss-model to generate homology models for the human and zebrafish nmdar subunits, which we subsequently aligned in pymole to assess conservation at the level of the three-dimensional structure (see methods) (fig. 4 f). in general, this analysis revealed a high degree of similarity between the human ntd and zebrafish paralogs. the zebrafish glun1 ntd models display similar overall topology to human: glun1 vs zglun1a, reference structure 5tp0 (rmsd = 0.071); glun1 vs zglun1a, reference structure 5tpz (rmsd = 0.085); glun1 vs zglun1b, reference structure 5tp0 (rmsd = 0.094); and glun1 vs zglun1b, reference structure 5tpz (rmsd = 0.097) (table s3). the glun2a and glun2b subunits also reflect a high degree of structural similarity: glun2a vs zglun2aa, reference structure 5tp0 (rmsd = 0.085); glun2a vs zglun2ab, reference structure 5tp0 (rmsd = 0.095); glun2a vs zglun2ab, reference structure 5tpw (rmsd = 0.111); glun2b vs zglun2ba, reference structure 5tpz (rmsd = 0.028); and glun2b vs glun2bb, reference structure 5tpz (rmsd = 0.044) (table s3). the only exception here is the structural comparison of glun2a and zglun2aa using reference structure 5tpw, in which the ntd is interacting with zinc (rmsd = 2.149), suggesting that the structural components of zglun2aa may not effectively complex with zinc. interestingly, zglun2ba is &gt; 100 amino acids shorter and zglun2bb is &gt; 70 amino acids longer than human glun2b. though the zglun2ba model is missing an α-helix relative to the human protein, the zglun2bb modelled structure aligns well with its human counterpart, suggesting that the zebrafish sequences maintain a large degree of structural integrity. conservation of post-translational modification (ptm) sites in the ctd the intracellular ctd is a key regulatory domain of nmdar cell biology and function and shows distinct subunit-specific elements ( 11 ). for our analyses, we focused on the glun1-1 ctd splice variant, which is the longest variant containing the c0-c1-c2 cassettes ( 11 ). across the nmdar subunits, the zebrafish ctds are the most poorly conserved (fig. 1 ). with a sequence identity generally around 50–60% for the glun1, glun2a, and glun2b zebrafish paralogs, with some exceptions: zglun1a (53% identity; 58% similarity), zglun1b (54% identity; 59% similarity), zglun2aa (57% identity; 86% identity), zglun2ab (38% identity; 65% similarity), zglun2ba (57% identity, 66% similarity), and zglun2bb (60% identity, 71% similarity) while that of glun2d is much lower (≥ 18% identity, ≥ 28% similarity) (fig. 1 ). given these deviations in protein sequences, we first examined the conservation of ctd regions critical for post-translational modifications (ptms), excluding glun2d due its poor conservation in zebrafish. phosphorylation of nmdar subunit ctds confers key functional roles to the receptor ( 2 , 11 ). initially, we examined experimentally validated sites at which phosphorylation occurs by known kinases and calculated the percent conservation, reported in identity and similarity. similarity, here, is defined as a site at the homologous position that can be phosphorylated (e.g., s, t, or y), if not identical. the glun1 ctd contains phosphorylation sites for src kinase (src), protein kinase c (pkc), and protein kinase a (pka), (fig. 5 a, top ) ( 11 ). note specific sites have also been identified that are also targeted by the serine/threonine protein phosphatase 2b (pp2b). these ptm sites are perfectly conserved in zglun1a, while three of five are conserved in zglun1b (60% identity &amp; similarity) (fig. 5 b). the glun2a ctd is phosphorylated by src, pka, dual specificity tyrosine-phosphorylation-regulated kinase 1a (dyrk1), cyclin-dependent kinase 5 (cdk5), pkc, and calcium/calmodulin-dependent protein kinase iiα (camkiiα) (fig. 5 a, middle ) ( 11 ): zglun2aa (79% identity &amp; similarity) and zglun2ab (64% identity, 71% similarity) (fig. 5 b). the ctd of glun2b is phosphorylated by src, camkiiα, cdk5, pka, pkc, death-associated protein kinase 1 (dapk1), proto-oncogene tyrosine-protein kinase fyn (fyn), and casein kinase ii (ck2) (fig. 5 a, bottom ) ( 11 ). glun2b also contains sites for the protein tyrosine phosphatase non-receptor 11 (ptpn11, or shp2) and protein phosphatase 1 (pp1). the conservation of these ptms is slightly higher than that in glun2a: zglun2ba (73% identity &amp; similarity) and zglun2bb (82% identity, 100% similarity) (fig. 5 b). in addition to these kinase-regulated sites, we also examined the conservation of other known phosphorylation sites in these subunits identified by phosphoproteomics (psp) (see methods). the conservation of these sites in the zebrafish paralogs was generally lower than that of the kinase-regulated sites: glun1 (67% identity, 100% similarity), zglun2aa (51% identity, 66% similarity), zglun2ab (34% identity, 57% similarity), zglun2ba (60% identity &amp; similarity), and zglun2bb (67% identity, 70% similarity) (fig. 5 c). we next examined the conservation of ctd palmitoylation and ubiquitylation sites. glun1 has a single ubiquitylation site (lys860) (fig. 6 a, top ) ( 11 ), and it is conserved in the zebrafish paralogs (fig. 6 c). glun2a contains seven palmitoylation sites (fig. 6 a, middle ) ( 11 ), which are likewise completely conserved in zebrafish (fig. 6 b). the glun2a ctd also has four ubiquitylation sites (fig. 6 a, middle) ( 11 ), of which three are conserved in zglun2aa (75%) and one in zglun2ab (25%) (fig. 6 c). with the same approach for glun2b, we identified eight palmitoylation and six ubiquitylation sites (fig. 6 a, bottom ) ( 11 ). here, all palmitoylation sites are conserved in zebrafish while five of six ubiquitylation sites (83%) are conserved in zglun2ba and all six ubiquitylation sites (100%) are conserved in zglun2bb (fig. 5 b,c). in summary, the global conservation of ptm sites is as follows: zglun1a and zglun1b (89% identity, 100% similarity), zglun2aa (62% identity, 72% similarity), zglun2ab (45% identity, 62% similarity), zglun2ba (66% identity, 67% similarity), and zglun2bb (75% identity, 78% similarity). conservation of protein-binding motifs (pbm) in the ctd in addition to numerous ptm sites, a variety of protein-protein interactions are critical to the role of the ctd in receptor cell biology (surface expression, distribution) and function ( 11 , 63 ). these protein-binding elements include either defined short linear motifs (slims) or broader regions known to serve as protein docking sites. the ctd of glun1 contains a slim known as the kkk/rrr er retention motif, which is conserved in zebrafish ( 66 ) (fig. 7 a,b). it, likewise, has binding regions for calmodulin (cam)/calcium/calmodulin-dependent protein kinase ii (camkii) and α-actinin 2 ( 64 , 65 , 67 , 78 ), which is conserved and yotiao, also known as a-kinase anchoring protein 9 (akap9) ( 67 ), which is generally well conserved: zglun1a (90% identity, 98% similarity) and zglun1b (90% identity, 98% similarity) (fig. 8 a,b). the glun2 subunit ctds contain a variety of pbms. for glun2a, the conservation of slims is: the yxxø motif, where y is tyrosine, x is any amino acid, and ø is a bulky hydrophobic residue like leucine, isoleucine, phenylalanine, methionine, or valine ( 73 ) – both zglun2aa and zglun2ab show 100% identity &amp; similarity; the guanine-nucleotide exchange factor brag2 ( 70 ) – zglun2aa (80% identity &amp; similarity) and zglun2ab (40% identity, 60% similarity); postsynaptic density protein 95 (psd-95) ( 72 ) – zglun2aa (50% identity; 63% similarity) and zglun2ab (38% identity, 63% similarity); and the pdz domain that interacts with the proteins psd-95, discs large (dlg), and zonula occludens-1 (zo-1) ( 11 ) – zglun2aa and zglun2ab (100% identity &amp; similarity) (fig. 7 a,c). interacting regions in glun2a include: ring finger protein 10 (rnf10) ( 70 ) – zglun2aa (51% identity, 72% similarity) and zglun2ab (32% identity, 66% similarity); rabphilin 3a (rph3a) ( 68 ) – zglun2aa (71% identity, 80% similarity) and zglun2ab (49% identity, 71% similarity); flotillin-1 (flot-1) ( 71 ) – zglun2aa (61% identity, 81% similarity) and zglun2ab (31% identity, 52% similarity); and c-terminal binding protein 1 (ctbp1) ( 69 ) – zglun2aa (51% identity, 85% similarity) and zglun2ab (36% identity, 63% similarity) (fig. 8 a,c, s3a,b). for glun2b, the conservation of slims is: the guanine-nucleotide exchange factor brag1 ( 76 ) – zglun2ba (60% identity; 80% similarity) and zglun2bb (60% identity, 100% similarity); psd-95 ( 72 ) – zglun2ba (56% identity &amp; similarity) and zglun2bb (33% identity, 56% similarity); camkii and death-associated protein kinase 1 (dapk1) ( 11 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (92% identity, 100% similarity); synapse-associated protein 102 (sap102) ( 75 ) – zglun2ba (0% identity &amp; similarity) and zglun2bb (50% identity &amp; similarity); the adaptor complex ap2 ( 74 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (100% identity &amp; similarity); and pdz ( 11 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (100% identity &amp; similarity) (fig. 7 a,d). the ctd interaction regions for glun2b are conserved as follows: receptor for activated c kinase 1 (rack1) ( 77 ) – zglun2ba (0% identity, 7% similarity) and zglun2bb (13% identity, 20% similarity); flot-1 ( 71 ) – zglun2ba (23% identity; 47% similarity) and zglun2bb (35% identity, 68% similarity); spectrin ( 79 ) – zglun2ba (40% identity; 59% similarity) and zglun2bb (35% identity, 62% similarity); α-actinin 2 ( 78 ) – zglun2ba (47% identity; 67% similarity) and zglun2bb (51% identity, 76% similarity); and ras protein-specific guanine nucleotide-releasing factor 1 (rasgrf1) ( 80 ) – zglun2ba (59% identity; 74% similarity) and zglun2bb (60% identity, 78% similarity) (fig. 8 a,d, s3a,c). when compared globally across subunits, slims always show greater conservation than regions. this conservation is always greatest in the zebrafish glun1 paralogs, followed by the glun2b and then glun2a paralogs. for the analyzed slims, conservation grouped across each subunit is as follows: zglun1a and zglun1b (100% identity &amp; similarity), zglun2aa (76% identity, 81% similarity), zglun2ab (62% identity, 76% similarity), zglun2ba (78% identity, 81% similarity), and zglun2bb (72% identity, 86% similarity). the regions show the following degrees of conservation: zglun1a and zglun1b (94% identity, 98% similarity), zglun2aa (55% identity, 81% similarity), zglun2ab (36% identity, 62% similarity), zglun2ba (59% identity, 74% similarity), and zglun2bb (60% identity, 78% similarity). discussion here, we compared the conservation of the zebrafish nmdar subunit paralogues, including key structure-function motifs, to their human counterparts. we find that the core gating machinery – the lbd, the lbd-tmd linkers and the tmd – are highly conserved both in terms of overall identity and specific functional motifs. the more peripheral domains – the ntd and ctd – show less conservation overall, though functional motifs are reasonably conserved with the notable exception of zn 2+ binding in the glun2a ntd. notably, disease-associated variants are generally conserved, highlighting that zebrafish represent a useful model to study nmdars. nmdar conservation in zebrafish across most of our examined parameters, the obligatory glun1 subunit shows the highest degree of conservation in zebrafish (fig. 1 ). generally, conservation is then greatest in the glun2(a-d) followed by the glun3(a-b) subunits. within each subunit, sequence conservation is consistently highest in the tmd and lbd-tmd linkers, followed by the lbd. conservation is generally poorer in the ntd and even worse in the ctd (fig. 1 ). a key question with regards to this work is defining what constitutes as a ‘well conserved’ sequence. previous studies regarding the conservation of protein structure and function suggest that sequence identities between 40–70% confer functional conservation ( 92 – 94 ). they regard 50% sequence identity as the threshold below which function drastically diverges ( 93 , 94 ). our sequence conservation analyses almost always led to sequence identities &gt; 50%, except for zglun2da. nevertheless, we carried out more detailed examinations of more poorly conserved domains, e.g., the ntd and ctd, to assess the conservation of key structure-function motifs. nmdar missense variant conservation in zebrafish to date, a multitude of variants, including missense, frameshift, and nonsense mutations, have been identified in the genes encoding the nmdar subunits ( 21 , 22 , 87 ). we categorized identified missense variants by pathogenicity and analyzed the extent of conservation of the positions at which they occur. we focused only on the glun1, glun2a, glun2b, and glun2d subunits for this analysis, as this is where most disease-associated variants have been identified. generally, our results demonstrate a higher degree of variant position conservation with increasing pathogenicity (fig. 2 , 3 ). the sites of pathogenic missense variants are almost perfectly conserved (fig. 2 ). given the high conservation of the amino acid positions at which they occur, these protein regions are likely most essential for receptor function. across our categories of variants, with the exclusion of complex variants, the proportion of the total protein at which missense variants have been identified is lowest in the pathogenic group (table 2 ). it is possible that additional variants that might be categorized as pathogenic are so devastating to development as to be embryonic lethal. this reflects the notion that more integral regions of the receptor are less tolerant to variants ( 95 ). zebrafish, nevertheless, emerge as a useful model for the study of pathogenic nmdar variants. putatively pathogenic variants are conserved to a lesser extent than their pathogenic counterparts (fig. 2 ). according to our designations, these variants have the potential to cause disease, having been identified in patients, but are currently classified as having uncertain significance. thus, it aligns that these variants are comparatively less well conserved – they are likely occurring in less critical regions of the receptor and are likewise less likely to cause disease phenotypes. our subsequent investigation of putatively benign and benign variants revealed a further decrease in variant position conservation. this finding further supported our prediction that variants more highly associated with disease occur at positions with higher degrees of conservation and, thus, are more critical to protein structure and function. the conservation of positions with putatively benign and benign variants is quite comparable. for the benign variants, in particular, conservation percentages also closely reflect total protein conservation values for each paralog (fig. 3 , table 1 ). this follows from the finding that positions with identified benign variants, for most subunits, make up a substantial portion of the protein (table 2 ). nevertheless, this investigation both demonstrates the advantage of the zebrafish model in studying nmdar-associated disease and reflects the notion that the positions most essential for receptor function are those that are best conserved and more likely to induce disease when absent. overall, the trend for variant conservation is the same as that for the whole protein. conservation is always greatest in the tmd and lbd-tmd linkers, followed by the lbd, next the ntd, and last the ctd (fig. 2 , 3 ). these findings suggest that zebrafish serve as an effective model for variants, especially pathogenic and putatively pathogenic variants in both the tmd and lbd-tmd linkers. some caution, however, must be exercised when using this model system for variants in the lbd, ntd, and ctd. tmd, tmd-lbd linker, and lbd conservation our examination of key functional regions and motifs in the tmd, lbd-tmd linkers, and lbd consistently revealed complete conservation in their respective zebrafish paralogs (fig. s1 ,s2). this suggests that the basic mechanism of ion channel gating is conserved in zebrafish relative to human. as these domains and subdomains contribute to the formation of the core of the receptor – the ion channel pore allowing for ion flux – it is expected that they would be most highly conserved across species, since the major functional role of nmdars is current flux during synaptic transmission. ntd conservation the ntd is the first region in which nmdar sequences in zebrafish begin to diverge from those in human. modelling ntd variants in zebrafish must, thus, be approached with caution. for example, polyamine-binding sites in glun2b are almost completely conserved in zebrafish (fig. 4 d), so they have the potential to serve as an effective model for this functional role of nmdars. on the other hand, zinc-binding sites in glun2a and glun2b are more poorly conserved in zebrafish (fig. 4 c), so they would likely serve as a poor model for zinc-related studies in nmdars. this also aligns with our homology model finding that the zglun2aa model aligns well with glun2a when not bound to zinc, but the rmsd of the alignment is substantially poorer when glun2a interacting with zinc is used as the reference structure (table s3). ctd conservation the c-terminal domains (ctds) exhibit the lowest level of overall conservation across nmdar domains (fig. 1 ). this is consistent with the ctd’s classification as an intrinsically disordered region (idr), which lacks fixed secondary or tertiary structure and is more tolerant to evolutionary divergence ( 96 , 97 ). nonetheless, functional conservation within idrs is often maintained through the retention of post-translational modification (ptm) hotspots and short linear motifs (slims) that regulate protein interactions and intracellular signaling ( 98 ). we assessed ctd conservation in glun1, glun2a, and glun2b by analyzing experimentally validated phosphorylation, palmitoylation, and ubiquitylation sites, as well as defined protein-binding motifs. phosphorylation sites with known kinase interactions exhibited relatively high conservation, consistent with their well-characterized regulatory roles (figs. 5 a,b). in contrast, phosphorylation sites without identified kinases were less consistently retained (fig. 5 c). palmitoylation sites were fully conserved across all subunits, while ubiquitylation sites were generally retained, though conservation was notably lower in zglun2ab (fig. 5 f) and may reflect paralogue-specific divergence in degradation or trafficking pathways. protein-binding motif conservation varied by subunit. glun1 retained nearly all canonical interaction motifs, while glun2a and glun2b displayed region-specific conservation. notably well-conserved elements in the glun2 subunits included the pdz-binding motifs as well as docking regions for camkii, dapk1, and ap2 (fig. 6 ). slims overall showed higher conservation, likely due to their short, sequence-specific structure, which may place them under stronger evolutionary constraint than broader, more flexible interaction regions. despite broad sequence divergence, nmdar ctds retain a core set of conserved motifs involved in receptor trafficking, synaptic localization, and plasticity. kinase-regulated phosphorylation sites, palmitoylation sites, and pdz-binding motifs were consistently preserved. conservation of glun1 sites was always greatest in the zebrafish paralogs. among the glun2 subunits, glun2b displayed the highest overall conservation of functional elements, aligning with its essential role in neurodevelopment and synaptic signaling. these findings are especially relevant to the study of disease-associated truncating variants in the nmdar subunits. nonsense variants that introduce premature stop codons in the ctd eliminate key regulatory motifs critical for trafficking, anchoring, or signaling. when these affected regions correspond to conserved functional elements, zebrafish may serve as an appropriate in vitro and in vivo model for studying ctd-mediated disease mechanisms. conversely, for mutations in poorly conserved regions, the translational utility of this model may become limited. conclusions zebrafish represent a valuable system to study nmdar-associated neurodevelopmental disorders, but one must be judicious in choosing which variants to study in this model. our unique approach of sequence alignments per individual nmdar domains and subdomains highlights the high degree of conversation of some regions of the protein relative to others. additionally, our analysis of identified variants indicates that zebrafish are an effective model for pathogenic variants as these occur at conserved locations along the protein. generally, the utility of zebrafish is most apparent for the study of the tmd, lbd-tmd linkers, and lbd as these regions are most well conserved. on the other hand, studying the ntd and ctd in this species requires extra consideration stemming from deviations in sequence conservation. our work will enable future nmdar-related studies that can be effectively and efficiently conducted in zebrafish. abbreviations nmdar n -methyl-d-aspartate receptor ntd n-terminal domain lbd ligand-binding domain tmd transmembrane domain ctd c-terminal domain declarations ethics approval and consent to participate: not applicable. consent for publication: not applicable. availability of data and materials: the datasets used and/or analyzed during the current study are available either at osf (https://osf.io/h2zf9/) or from the corresponding author on reasonable request. competing interests: no competing interests declared. funding: this work was supported by nih grant r01ns088479 to lpw. authors’ contributions: e.r.n., c.a., j.d.z., h.i.s., and l.p.w. contributed to the conceptualization and methodology of this project. e.r.n., c.a., and j.d.z. conducted the investigation and analysis. w.r. assisted with data curation. e.r.n, c.a., and l.p.w. wrote the manuscript, which was revised with insights from j.d.z. and h.i.s. acknowledgements: the authors thank the clinvar and gnomad databases and the swiss-model server for their important contributions to this work. references paoletti p, bellone c, zhou q. nmda receptor subunit diversity: impact on receptor properties, synaptic plasticity and disease. nat rev neurosci. 2013;14(6):383-400. hansen kb, wollmuth lp, bowie d, furukawa h, menniti fs, sobolevsky ai, et al. structure, function, and pharmacology of glutamate receptor ion channels. pharmacol rev. 2021;73(4):298-487. chakraborty a, murphy s, coleman n. the role of nmda receptors in neural stem cell proliferation and differentiation. stem cells dev. 2017;26(11):798-807. nacher j, mcewen bs. the role of n-methyl-d-asparate receptors in neurogenesis. hippocampus. 2006;16(3):267-70. komuro h, rakic p. modulation of neuronal migration by nmda receptors. science. 1993;260(5104):95-7. pearce ia, cambray-deakin ma, burgoyne rd. glutamate acting on nmda receptors stimulates neurite outgrowth from cerebellar granule cells. febs lett. 1987;223(1):143-7. booker sa, wyllie dja. nmda receptor function in inhibitory neurons. neuropharmacology. 2021;196:108609. gambrill ac, barria a. nmda receptor subunit composition controls synaptogenesis and synapse stabilization. proc natl acad sci u s a. 2011;108(14):5855-60. napoli aj, laderwager s, zoodsma jd, biju b, mucollari o, schubel sk, et al. developmental loss of nmda receptors results in supernumerary forebrain neurons through delayed maturation of transit-amplifying neuroblasts. sci rep. 2024;14(1):3395. rajani v, sengar as, salter mw. tripartite signalling by nmda receptors. mol brain. 2020;13(1):23. ishchenko y, carrizales mg, koleske aj. regulation of the nmda receptor by its cytoplasmic domains: (how) is the tail wagging the dog? neuropharmacology. 2021;195:108634. mony l, paoletti p. mechanisms of nmda receptor regulation. curr opin neurobiol. 2023;83:102815. nielsen ac, anderson cl, ens c, boyce akj, thompson rj. non-ionotropic nmdar signalling activates panx1 to induce p2x4r-dependent long-term depression in the hippocampus. j physiol. 2024. barnes sa, thomazeau a, finnie psb, heinrich mj, heynen aj, komiyama nh, et al. non-ionotropic signaling through the nmda receptor glun2b carboxy-terminal domain drives dendritic spine plasticity and reverses fragile x phenotypes. cell rep. 2025;44(3):115311. platzer k, lemke jr. grin1-related neurodevelopmental disorder. genereviews((r)). 2021. strehlow v, myers ka, morgan at, scheffer ie, lemke jr. grin2a-related disorders. genereviews((r)). 2024. platzer k, lemke jr. grin2b-related neurodevelopmental disorder. genereviews((r)). 2021. platzer k, krey i, lemke jr. grin2d-related developmental and epileptic encephalopathy. genereviews((r)). 2022. zehavi y, mandel h, zehavi a, rashid ma, straussberg r, jabur b, et al. de novo grin1 mutations: an emerging cause of severe early infantile encephalopathy. eur j med genet. 2017;60(6):317-20. sabo sl, lahr jm, offer m, weekes a, sceniak mp. grin2b-related neurodevelopmental disorder: current understanding of pathophysiological mechanisms. front synaptic neurosci. 2022;14:1090865. gjerulfsen ce, krey i, klockner c, rubboli g, lemke jr, moller rs. spectrum of nmda receptor variants in neurodevelopmental disorders and epilepsy. methods mol biol. 2024;2799:1-11. amin jb, moody gr, wollmuth lp. from bedside-to-bench: what disease-associated variants are teaching us about the nmda receptor. j physiol. 2021;599(2):397-416. hu c, chen w, myers sj, yuan h, traynelis sf. human grin2b variants in neurodevelopmental disorders. j pharmacol sci. 2016;132(2):115-21. cox ja, kucenas s, voigt mm. molecular characterization and embryonic expression of the family of n-methyl-d-aspartate receptor subunit genes in the zebrafish. dev dyn. 2005;234(3):756-66. veldman mb, lin s. zebrafish as a developmental model organism for pediatric research. pediatr res. 2008;64(5):470-6. choi ty, choi ti, lee yr, choe sk, kim ch. zebrafish as an animal model for biomedical research. exp mol med. 2021;53(3):310-7. zoodsma jd, chan k, bhandiwad aa, golann dr, liu g, syed sa, et al. a model to study nmda receptors in early nervous system development. j neurosci. 2020;40(18):3631-45. zoodsma jd, keegan ej, moody gr, bhandiwad aa, napoli aj, burgess ha, et al. disruption of grin2b, an asd-associated gene, produces social deficits in zebrafish. mol autism. 2022;13(1):38. baraban sc, taylor mr, castro pa, baier h. pentylenetetrazole induced changes in zebrafish behavior, neural activity and c-fos expression. neuroscience. 2005;131(3):759-68. shams s, rihel j, ortiz jg, gerlai r. the zebrafish as a promising tool for modeling human brain disorders: a review based upon an ibns symposium. neurosci biobehav rev. 2018;85:176-90. de abreu ms, genario r, giacomini a, demin ka, lakstygal am, amstislavskaya tg, et al. zebrafish as a model of neurodevelopmental disorders. neuroscience. 2020;445:3-11. weinschutz mendes h, neelakantan u, liu y, fitzpatrick se, chen t, wu w, et al. high-throughput functional analysis of autism genes in zebrafish identifies convergence in dopaminergic and neuroimmune pathways. cell rep. 2023;42(3):112243. howe k, clark md, torroja cf, torrance j, berthelot c, muffato m, et al. the zebrafish reference genome sequence and its relationship to the human genome. nature. 2013;496(7446):498-503. rihel j, schier af. behavioral screening for neuroactive drugs in zebrafish. dev neurobiol. 2012;72(3):373-85. baraban sc, dinday mt, hortopan ga. drug screening in scn1a zebrafish mutant identifies clemizole as a potential dravet syndrome treatment. nat commun. 2013;4:2410. zhang y, kecskes a, copmans d, langlois m, crawford ad, ceulemans b, et al. pharmacological characterization of an antisense knockdown zebrafish model of dravet syndrome: inhibition of epileptic seizures by the serotonin agonist fenfluramine. plos one. 2015;10(5):e0125898. dingemans ajm, jansen s, van reeuwijk j, de leeuw n, pfundt r, schuurs-hoeijmakers j, et al. prevalence of comorbidities in individuals with neurodevelopmental disorders from the aggregated phenomics data of 51,227 pediatric individuals. nat med. 2024;30(7):1994-2003. ohtsuki t, sakurai k, dou h, toru m, yamakawa-kobayashi k, arinami t. mutation analysis of the nmdar2b (grin2b) gene in schizophrenia. mol psychiatry. 2001;6(2):211-6. arnold pd, rosenberg dr, mundo e, tharmalingam s, kennedy jl, richter ma. association of a glutamate (nmda) subunit receptor gene (grin2b) with obsessive-compulsive disorder: a preliminary study. psychopharmacology (berl). 2004;174(4):530-8. saunders ja, tatard-leitman vm, suh j, billingslea en, roberts tp, siegel sj. knockout of nmda receptors in parvalbumin interneurons recreates autism-like phenotypes. autism res. 2013;6(2):69-77. gandal mj, sisti j, klook k, ortinski pi, leitman v, liang y, et al. gabab-mediated rescue of altered excitatory-inhibitory balance, gamma synchrony and behavioral deficits following constitutive nmdar-hypofunction. transl psychiatry. 2012;2(7):e142. barnby g, abbott a, sykes n, morris a, weeks de, mott r, et al. candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at grin2a and abat. am j hum genet. 2005;76(6):950-66. hung ly, margolis kg. autism spectrum disorders and the gastrointestinal tract: insights into mechanisms and clinical relevance. nat rev gastroenterol hepatol. 2024;21(3):142-63. owens ap, mathias cj, iodice v. autonomic dysfunction in autism spectrum disorder. front integr neurosci. 2021;15:787037. orefice ll. peripheral somatosensory neuron dysfunction: emerging roles in autism spectrum disorders. neuroscience. 2020;445:120-9. makris g, agorastos a, chrousos gp, pervanidou p. stress system activation in children and adolescents with autism spectrum disorder. front neurosci. 2021;15:756628. ramos-vicente d, grant sg, bayes a. metazoan evolution and diversity of glutamate receptors and their auxiliary subunits. neuropharmacology. 2021;195:108640. amores a, force a, yan yl, joly l, amemiya c, fritz a, et al. zebrafish hox clusters and vertebrate genome evolution. science. 1998;282(5394):1711-4. chou th, tajima n, romero-hernandez a, furukawa h. structural basis of functional transitions in mammalian nmda receptors. cell. 2020;182(2):357-71 e13. zhang y, ye f, zhang t, lv s, zhou l, du d, et al. structural basis of ketamine action on human nmda receptors. nature. 2021;596(7871):301-5. wollmuth lp, sakmann b. different mechanisms of ca2+ transport in nmda and ca2+-permeable ampa glutamate receptor channels. j gen physiol. 1998;112(5):623-36. watanabe j, beck c, kuner t, premkumar ls, wollmuth lp. drpeer: a motif in the extracellular vestibule conferring high ca2+ flux rates in nmda receptor channels. j neurosci. 2002;22(23):10209-16. wollmuth lp, sobolevsky ai. structure and gating of the glutamate receptor ion channel. trends neurosci. 2004;27(6):321-8. ogden kk, khatri a, traynelis sf, heldt sa. potentiation of glun2c/d nmda receptor subtypes in the amygdala facilitates the retention of fear and extinction learning in mice. neuropsychopharmacology. 2014;39(3):625-37. wollmuth lp. ion permeation in ionotropic glutamate receptors: still dynamic after all these years. curr opin physiol. 2018;2:36-41. mcdaniel mj, ogden kk, kell sa, burger pb, liotta dc, traynelis sf. nmda receptor channel gating control by the pre-m1 helix. j gen physiol. 2020;152(4). wollmuth lp, kuner t, sakmann b. adjacent asparagines in the nr2-subunit of the nmda receptor channel control the voltage-dependent block by extracellular mg2+. j physiol. 1998;506 ( pt 1)(pt 1):13-32. romero-hernandez a, simorowski n, karakas e, furukawa h. molecular basis for subtype specificity and high-affinity zinc inhibition in the glun1-glun2a nmda receptor amino-terminal domain. neuron. 2016;92(6):1324-36. mony l, zhu s, carvalho s, paoletti p. molecular basis of positive allosteric modulation of glun2b nmda receptors by polyamines. embo j. 2011;30(15):3134-46. lichnerova k, kaniakova m, park sp, skrenkova k, wang yx, petralia rs, et al. two n-glycosylation sites in the glun1 subunit are essential for releasing n-methyl-d-aspartate (nmda) receptors from the endoplasmic reticulum. j biol chem. 2015;290(30):18379-90. waterhouse a, bertoni m, bienert s, studer g, tauriello g, gumienny r, et al. swiss-model: homology modelling of protein structures and complexes. nucleic acids res. 2018;46(w1):w296-w303. bienert s, waterhouse a, de beer ta, tauriello g, studer g, bordoli l, et al. the swiss-model repository-new features and functionality. nucleic acids res. 2017;45(d1):d313-d9. warnet xl, bakke krog h, sevillano-quispe og, poulsen h, kjaergaard m. the c-terminal domains of the nmda receptor: how intrinsically disordered tails affect signalling, plasticity and disease. eur j neurosci. 2021;54(8):6713-39. kim k, saneyoshi t, hosokawa t, okamoto k, hayashi y. interplay of enzymatic and structural functions of camkii in long-term potentiation. j neurochem. 2016;139(6):959-72. bej a, ames jb. chemical shift assignments of calmodulin bound to a cytosolic domain of glun2a (residues 1004-1024) from the nmda receptor. biomol nmr assign. 2023;17(1):89-93. horak m, wenthold rj. different roles of c-terminal cassettes in the trafficking of full-length nr1 subunits to the cell surface. j biol chem. 2009;284(15):9683-91. lin jw, wyszynski m, madhavan r, sealock r, kim ju, sheng m. yotiao, a novel protein of neuromuscular junction and brain that interacts with specific splice variants of nmda receptor subunit nr1. j neurosci. 1998;18(6):2017-27. stanic j, carta m, eberini i, pelucchi s, marcello e, genazzani aa, et al. rabphilin 3a retains nmda receptors at synaptic sites through interaction with glun2a/psd-95 complex. nat commun. 2015;6:10181. cousins sl, stephenson fa. identification of c-terminal binding protein 1 as a novel nmda receptor interactor. neurochem res. 2019;44(6):1437-45. franchini l, carrano n, di luca m, gardoni f. synaptic glun2a-containing nmda receptors: from physiology to pathological synaptic plasticity. int j mol sci. 2020;21(4). swanwick cc, shapiro me, yi z, chang k, wenthold rj. nmda receptors interact with flotillin-1 and -2, lipid raft-associated proteins. febs lett. 2009;583(8):1226-30. cousins sl, stephenson fa. identification of n-methyl-d-aspartic acid (nmda) receptor subtype-specific binding sites that mediate direct interactions with scaffold protein psd-95. j biol chem. 2012;287(16):13465-76. vissel b, krupp jj, heinemann sf, westbrook gl. a use-dependent tyrosine dephosphorylation of nmda receptors is independent of ion flux. nat neurosci. 2001;4(6):587-96. lavezzari g, mccallum j, lee r, roche kw. differential binding of the ap-2 adaptor complex and psd-95 to the c-terminus of the nmda receptor subunit nr2b regulates surface expression. neuropharmacology. 2003;45(6):729-37. chen bs, gray ja, sanz-clemente a, wei z, thomas ev, nicoll ra, et al. sap102 mediates synaptic clearance of nmda receptors. cell rep. 2012;2(5):1120-8. elagabani mn, brisevac d, kintscher m, pohle j, kohr g, schmitz d, et al. subunit-selective n-methyl-d-aspartate (nmda) receptor signaling through brefeldin a-resistant arf guanine nucleotide exchange factors brag1 and brag2 during synapse maturation. j biol chem. 2016;291(17):9105-18. yaka r, thornton c, vagts aj, phamluong k, bonci a, ron d. nmda receptor function is regulated by the inhibitory scaffolding protein, rack1. proc natl acad sci u s a. 2002;99(8):5710-5. wyszynski m, lin j, rao a, nigh e, beggs ah, craig am, et al. competitive binding of alpha-actinin and calmodulin to the nmda receptor. nature. 1997;385(6615):439-42. wechsler a, teichberg vi. brain spectrin binding to the nmda receptor is regulated by phosphorylation, calcium and calmodulin. embo j. 1998;17(14):3931-9. krapivinsky g, krapivinsky l, manasian y, ivanov a, tyzio r, pellegrino c, et al. the nmda receptor is coupled to the erk pathway by a direct interaction between nr2b and rasgrf1. neuron. 2003;40(4):775-84. mukherjee p, roy s, ghosh d, nandi sk. role of animal models in biomedical research: a review. lab anim res. 2022;38(1):18. chou th, epstein m, fritzemeier rg, akins ns, paladugu s, ullman ez, et al. molecular mechanism of ligand gating and opening of nmda receptor. nature. 2024;632(8023):209-17. armstrong n, sun y, chen gq, gouaux e. structure of a glutamate-receptor ligand-binding core in complex with kainate. nature. 1998;395(6705):913-7. armstrong n, gouaux e. mechanisms for activation and antagonism of an ampa-sensitive glutamate receptor: crystal structures of the glur2 ligand binding core. neuron. 2000;28(1):165-81. kazi r, gan q, talukder i, markowitz m, salussolia cl, wollmuth lp. asynchronous movements prior to pore opening in nmda receptors. j neurosci. 2013;33(29):12052-66. amin jb, gochman a, he m, certain n, wollmuth lp. nmda receptors require multiple pre-opening gating steps for efficient synaptic activity. neuron. 2021;109(3):488-501 e4. hardingham ge, do kq. linking early-life nmdar hypofunction and oxidative stress in schizophrenia pathogenesis. nat rev neurosci. 2016;17(2):125-34. landrum mj, lee jm, riley gr, jang w, rubinstein ws, church dm, et al. clinvar: public archive of relationships among sequence variation and human phenotype. nucleic acids res. 2014;42(database issue):d980-5. landrum mj, lee jm, benson m, brown g, chao c, chitipiralla s, et al. clinvar: public archive of interpretations of clinically relevant variants. nucleic acids res. 2016;44(d1):d862-8. landrum mj, lee jm, benson m, brown gr, chao c, chitipiralla s, et al. clinvar: improving access to variant interpretations and supporting evidence. nucleic acids res. 2018;46(d1):d1062-d7. gudmundsson s, singer-berk m, watts na, phu w, goodrich jk, solomonson m, et al. variant interpretation using population databases: lessons from gnomad. hum mutat. 2022;43(8):1012-30. tian w, skolnick j. how well is enzyme function conserved as a function of pairwise sequence identity? j mol biol. 2003;333(4):863-82. addou s, rentzsch r, lee d, orengo ca. domain-based and family-specific sequence identity thresholds increase the levels of reliable protein function transfer. j mol biol. 2009;387(2):416-30. sangar v, blankenberg dj, altman n, lesk am. quantitative sequence-function relationships in proteins based on gene ontology. bmc bioinformatics. 2007;8:294. perszyk re, kristensen as, lyuboslavsky p, traynelis sf. three-dimensional missense tolerance ratio analysis. genome res. 2021;31(8):1447-61. uversky vn. intrinsically disordered proteins from a to z. int j biochem cell biol. 2011;43(8):1090-103. singleton md, eisen mb. evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. plos comput biol. 2024;20(4):e1012028. davey ne, cowan jl, shields dc, gibson tj, coldwell mj, edwards rj. slimprints: conservation-based discovery of functional motif fingerprints in intrinsically disordered protein regions. nucleic acids res. 2012;40(21):10628-41. additional declarations no competing interests reported. supplementary files zfishnmdarssupplement20250716.docx cite share download pdf status: under review version 1 posted editorial decision: revision requested 24 sep, 2025 reviews received at journal 19 sep, 2025 reviewers agreed at journal 19 sep, 2025 reviews received at journal 09 sep, 2025 reviews received at journal 18 aug, 2025 reviewers agreed at journal 04 aug, 2025 reviewers agreed at journal 28 jul, 2025 reviewers invited by journal 28 jul, 2025 editor assigned by journal 28 jul, 2025 editor invited by journal 23 jul, 2025 submission checks completed at journal 22 jul, 2025 first submitted to journal 22 jul, 2025 you are reading this latest preprint version research square lets you share your work early, gain feedback from the community, and start making changes to your manuscript prior to peer review in a journal. as a division of research square company, we’re committed to making research communication faster, fairer, and more useful. we do this by developing innovative software and high quality services for the global research community. our growing team is made up of researchers and industry professionals working together to solve the most critical problems facing scientific publishing. also discoverable on platform about our team in review editorial policies advisory board help center resources author services research quality evaluation accessibility api access rss feed manage cookie preferences © research square 2025 | issn 2693-5015 (online) privacy policy terms of service do not sell my personal information {"props":{"pageprops":{"initialdata":{"identity":"rs-7151578","acceptedtermsandconditions":true,"allowdirectsubmit":false,"archivedversions":[],"articletype":"research article","associatedpublications":[],"authors":[{"id":492782335,"identity":"6084adf8-9103-4ca8-8de5-679d2a4ea2bb","order_by":0,"name":"erica r. nebet","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"erica","middlename":"r.","lastname":"nebet","suffix":""},{"id":492782336,"identity":"68eb1705-4eac-413e-bc82-7b9943220d12","order_by":1,"name":"christieann aprea","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"christieann","middlename":"","lastname":"aprea","suffix":""},{"id":492782337,"identity":"776ee37e-7ff9-4a8a-bc43-f965303e90a8","order_by":2,"name":"josiah d. zoodsma","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"josiah","middlename":"d.","lastname":"zoodsma","suffix":""},{"id":492782338,"identity":"ef7f492d-83d8-41a3-aa81-6be2af423f07","order_by":3,"name":"william raab","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"william","middlename":"","lastname":"raab","suffix":""},{"id":492782339,"identity":"6949754f-6430-4ffc-8673-da5548d44bd6","order_by":4,"name":"howard i. sirotkin","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"howard","middlename":"i.","lastname":"sirotkin","suffix":""},{"id":492782340,"identity":"a4f8ecb1-612e-43db-8923-d325a2be05c2","order_by":5,"name":"lonnie p. wollmuth","email":"data:image/png;base64,ivborw0kggoaaaansuheugaaazaaaaayaqmaaabi0h/eaaaablbmvex///8aaabvwtn+aaaacxbiwxmaaa7eaaaoxagvkw4baaabhuleqvriiwnggawjybacphcraclymrua5ae5sdajim0dxwiada6fgaqs4yaxwiigfgyklyknblww9x578kccibgfmbf1m++po+nz5zcfe8y7454cp88bxgdv2zc18jxln0g4w5a4s5mx7tzpwrpcdcfy0o15zxqbs/y2mbvoxajfisdmirgnixfdybcww7kb2hjmphnbehi3ngdgazjwapmh0jiu34bqwv4bp5yghjyehmmwlwcb2jixaee5yyarceyihusxm3psdhtuojawjjn3tikxzm/bzsk55zc08lp3men+qlex5mdvpnbjjc1hefnmw8ck3u5iaizy8sepb8owqxkcjfc5tjdoamcokydxb/fqr8eogawjypgdabgnaum7/svdaaaaaelftksuqmcc","orcid":"","institution":"stony brook university","correspondingauthor":true,"prefix":"","firstname":"lonnie","middlename":"p.","lastname":"wollmuth","suffix":""}],"badges":[],"createdat":"2025-07-17 18:08:21","currentversioncode":1,"declarations":"","doi":"10.21203/rs.3.rs-7151578/v1","doiurl":"https://doi.org/10.21203/rs.3.rs-7151578/v1","draftversion":[],"editorialevents":[],"editorialnote":"","failedworkflow":false,"files":[{"id":87954191,"identity":"93406acd-9b5c-4bbe-b8e4-476bfc8e3591","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":0,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":48663694,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarsmain20250722.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/72a01d08b07202a8dd3432a7.docx"},{"id":87954161,"identity":"09dcee36-3c3d-4bae-a5b6-a8031a04e2f8","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"json","order_by":1,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":8698,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b.json","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/088093afe258318de15a5f12.json"},{"id":87954169,"identity":"c1d3acb0-9e25-4b8a-9228-e0e154b314b2","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":2,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":15353369,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarssupplement20250716.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a3a65bcc549c7954c295d2d1.docx"},{"id":87954163,"identity":"6fde5d88-4da9-48b5-ac47-0b0e0f6354d8","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"xml","order_by":3,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":242943,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b1enriched.xml","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/aad420877d9070a2aedac4cc.xml"},{"id":87954167,"identity":"b5c71071-9fd7-476b-9303-17dcb5b1c845","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"jpeg","order_by":4,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":7224546,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage1.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/239b3c6e0e27c5d5f3a3c485.jpeg"},{"id":87954176,"identity":"66a11160-0f4e-44e0-a6b3-26a6c526f9c1","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":5,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5495318,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage2.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/7c759429dc2c4ccf2d95e1ed.jpeg"},{"id":87954168,"identity":"e3a84d67-d0ce-4822-8529-9e1cc04acd3d","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":6,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5497046,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage3.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/5ab1e35184bc289007b868d7.jpeg"},{"id":87954408,"identity":"4452df13-a563-4564-9a2c-9a250a285ebb","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"jpeg","order_by":7,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4536836,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage4.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/34cad84600342cebbb343c8d.jpeg"},{"id":87954171,"identity":"94757fe0-c77c-46b6-8a13-76b5920bc170","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":8,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4702238,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage5.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/704be5ce4c0cc10b6f98b684.jpeg"},{"id":87954173,"identity":"bbb91acc-360b-43aa-99a5-89fbc5989777","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":9,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4378230,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage6.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/85acc94c54964dbcd4ee6caa.jpeg"},{"id":87954192,"identity":"2447fad7-ffab-4e33-9b2f-a71d2af30f9e","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":10,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5735470,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage7.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3a0654f7ccd9aa40cc5d5f69.jpeg"},{"id":87954413,"identity":"29bec889-0025-4009-9267-6438ea9c5740","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"jpeg","order_by":11,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5327046,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage8.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/c3009b0732aa42082efe8df3.jpeg"},{"id":87954188,"identity":"20ba216a-d094-494f-9894-1937509c8d81","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":12,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":222456,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage1.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/08aba5e904980f68ab32639e.png"},{"id":87954187,"identity":"d99856fc-69f6-4dda-9eeb-4cafd6cc99a0","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":13,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":175317,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage2.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a0ea7afa8616702c9ac66b10.png"},{"id":87954174,"identity":"de47bd92-e161-4fbe-a9b2-437c4b5f0573","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":14,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":140021,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage3.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/bb8abfaa1c199dae266df4dd.png"},{"id":87954412,"identity":"87f7cc6b-e10f-4300-9539-2c2a02855ddb","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"png","order_by":15,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":138717,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage4.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a6b8e16a616611723ae39dab.png"},{"id":87954190,"identity":"ad2ebfe9-5bbc-4128-8d93-eaa9d7470a57","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":16,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":96612,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage5.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/99939544fae92f971073fd51.png"},{"id":87954189,"identity":"84923e46-044f-4cd2-9f5a-b3a9fa41caab","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":17,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":72313,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage6.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/92848d9ff37daaad2b48e9ff.png"},{"id":87954755,"identity":"eaa7ae44-7e94-4117-97cb-d03c1d07f90a","added_by":"auto","created_at":"2025-07-30 18:41:44","extension":"png","order_by":18,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":83342,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage7.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/845b774d666648bfa0a844ab.png"},{"id":87954410,"identity":"13827b86-cf06-4cf3-9952-4b424318037f","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"png","order_by":19,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":117211,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage8.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/af95a59a499464e286bef54b.png"},{"id":87954411,"identity":"7741fc55-f9dc-44b6-9b97-8875d99f5f4b","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"xml","order_by":20,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":240176,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b1structuring.xml","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/dfb8648302675ebf344a2fa6.xml"},{"id":87954756,"identity":"beb85617-8838-4c99-8712-1a7d62a23eff","added_by":"auto","created_at":"2025-07-30 18:41:44","extension":"html","order_by":21,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":257708,"visible":true,"origin":"","legend":"","description":"","filename":"earlyproof.html","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/01f35be0d8432473e2775041.html"},{"id":87954160,"identity":"3920a076-09de-49c0-b764-c081048b3b14","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":1,"title":"figure 1","display":"","copyasset":false,"role":"figure","size":795350,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of human nmdar subunits in zebrafish\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a)\u003c/strong\u003e nmdars are composed of two obligatory glun1 and typically two glun2a-d subunits. the tetrameric complex consists of four domains: extracellular n-terminal (ntd) and ligand-binding (lbd) domains; a transmembrane domain (tmd); and an intracellular c-terminal domain (ctd). glun1 subunits are colored dark blue (ntd), teal (lbd), red (lbd-tmd linkers), light orange (tmd), and gray (ctd); and glun2b subunits gray. 9are (82)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b)\u003c/strong\u003e topology of an individual glun1 subunit. lbd lobes (d1 and d2) are shown; d1 is mainly comprisedof s1 and d2 of s2. the ion channel is formed by 3 transmembrane segments, m1, m3, and m4, and an intracellular m2 pore loop. lbd-tmd linkers are s1-m1 (or d2-m1), m3-s2 (or d2-m3), and s2-m4 (or d1-m4).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c)\u003c/strong\u003e nmdar subunits are separated into 13 structural subdivisions: ntd, lbd (s1 \u0026amp; s2), lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4), tmd (m1, intracellular m1-m2 linker, m2 loop, m2-m3 linker, m3, \u0026amp; m4), and ctd. amino acid length of each zebrafish subdivision is indicated. those sharing amino acid length with their human orthologues are indicated in white, while increasingly darker pink indicates those with increasingly greater amino acid differences (greater (+) or fewer (-)).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(d-e)\u003c/strong\u003e nmdar subunits are grouped as in \u003cstrong\u003e(c)\u003c/strong\u003e. homology between human and zebrafish nmdar subunit paralogues is reported in amino acid identity \u003cstrong\u003e(d)\u003c/strong\u003e or similarity \u003cstrong\u003e(e)\u003c/strong\u003e. heat map with darker green indicating greater percent of homology.\u003c/p\u003e","description":"","filename":"floatimage1.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/5b4e27c91809ed8be6a2b3b5.png"},{"id":87954407,"identity":"47d4fb26-35bf-49ca-aae6-ed196317e2be","added_by":"auto","created_at":"2025-07-30 18:33:43","extension":"png","order_by":2,"title":"figure 2","display":"","copyasset":false,"role":"figure","size":712960,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of positions with pathogenic and putatively pathogenic missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a-b) \u003c/strong\u003epercent conservation of pathogenic variant positions in zebrafish genes, as defined in table 2. conservation is reported in amino acid identity \u003cstrong\u003e(a) \u003c/strong\u003eand similarity \u003cstrong\u003e(b)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c-d)\u003c/strong\u003e conservation of pathogenic variant positions in each nmdar domain, grouped as in figure 1. homology is reported in identity \u003cstrong\u003e(c)\u003c/strong\u003eand similarity \u003cstrong\u003e(d)\u003c/strong\u003e. dashes indicate domains without reported pathogenic variants. along with the percent of conservation, the total number of identified variants in the human protein is indicated in parentheses in the ‘a’ paralog column in \u003cstrong\u003e(c)\u003c/strong\u003e.\u003c/p\u003e","description":"","filename":"floatimage2.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/d7d962b025e722b029324c59.png"},{"id":87954162,"identity":"02953530-369f-4594-b587-6d86079d6a68","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":3,"title":"figure 3","display":"","copyasset":false,"role":"figure","size":667858,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of positions with putatively benign and benign missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a-b) \u003c/strong\u003epercent conservation of putatively benign variant positions in zebrafish genes, as defined in table 2. conservation is reported in identity \u003cstrong\u003e(a) \u003c/strong\u003eand similarity \u003cstrong\u003e(b)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c-d) \u003c/strong\u003econservation of putatively benign variant positions in each nmdar domain, grouped as in figure 2. homology is reported in identity \u003cstrong\u003e(c)\u003c/strong\u003e and similarity \u003cstrong\u003e(d)\u003c/strong\u003e. dashes indicate domains without reported pathogenic variants. along with the percent of conservation, the total number of identified variants in the human protein is indicated in parentheses in the ‘a’ paralog column in \u003cstrong\u003e(c)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(e-h) \u003c/strong\u003eas in a-d but for benign variant positions.\u003c/p\u003e","description":"","filename":"floatimage3.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/069549024abcf2bb76a43e0e.png"},{"id":87954166,"identity":"f30749e8-12a9-42b8-89f5-26186b255ecb","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":4,"title":"figure 4","display":"","copyasset":false,"role":"figure","size":473605,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key functional elements and structure of the nmdar ntd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003etopology of glun2a ntd, showing r1 and r2 subdomains, bound to zinc (red). 5tpw (58)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b) \u003c/strong\u003eclose-up image of structure in \u003cstrong\u003e(a)\u003c/strong\u003ehighlighting amino acids that participate in zinc binding. his44 is demarcated as it is not conserved in the zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c) \u003c/strong\u003econservation of amino acids that participate in ntd zinc binding in glun2a and glun2b, reported in identity (dark red) and similarity (light red). sequences are shown to the right.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(d) \u003c/strong\u003econservation of glun2b ntd regions that participate in interactions with polyamines, reported in identity (dark gray) and similarity (light gray). sequences are shown to the right.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(e) \u003c/strong\u003econservation of ntd glycosylation sites, reported in identity (dark gray).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(f) \u003c/strong\u003ealigned ntd homology models of the human (blue) and zebrafish (gray) protein for glun1 vs zglun1a (reference structure, 5tp0) (\u003cem\u003eleft\u003c/em\u003e), glun2a vs zglun2aa (reference structure, 5tp0) (\u003cem\u003emiddle\u003c/em\u003e), and glun2b vs zglun2bb (reference structure, 5tpz) (\u003cem\u003eright\u003c/em\u003e). secondary structures colored both blue and gray indicate areas of structural overlap.\u003c/p\u003e","description":"","filename":"floatimage4.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3693c16c36a6ea0f931e6343.png"},{"id":87954406,"identity":"16a86e66-9b13-469f-91ca-fe2142d4ce9c","added_by":"auto","created_at":"2025-07-30 18:33:43","extension":"png","order_by":5,"title":"figure 5","display":"","copyasset":false,"role":"figure","size":427432,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key nmdar ctd phosphorylation sites\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd kinase-regulated phosphorylation sites for the glun1, glun2a and glun2b ctds. the specific phosphorylation sites and the corresponding human and zebrafish sequences are indicated. positions sharing identity are indicated in dark gray. similarity is shown in light gray; here, a residue is only considered similar if it is ‘phosphorylatable’ (e.g., contains a s, t, or y) (see methods).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-c) \u003c/strong\u003epercent conservation of kinase-regulated phosphorylation sites \u003cstrong\u003e(b)\u003c/strong\u003e and other phosphorylation sites identified by phosphoproteomics \u003cstrong\u003e(c)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray); similarity is defined in \u003cstrong\u003e(a)\u003c/strong\u003e.\u003c/p\u003e","description":"","filename":"floatimage5.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/25001bb8418306120cd3e005.png"},{"id":87954184,"identity":"a64d1f54-7d76-472b-a396-d9efcf5f86c8","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":6,"title":"figure 6","display":"","copyasset":false,"role":"figure","size":333594,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key nmdar ctd post-translational modification (ptm) sites\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd palmitoylation and ubiquitylation sites for the glun1, glun2a and glun2b ctds. the specific phosphorylation sites and the corresponding human and zebrafish sequences are indicated. positions sharing identity are indicated in dark gray.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-c) \u003c/strong\u003epercent conservation of palmitoylation \u003cstrong\u003e(b)\u003c/strong\u003e and ubiquitylation \u003cstrong\u003e(c)\u003c/strong\u003e sites, reported in identity (dark gray).\u003c/p\u003e","description":"","filename":"floatimage6.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/fac7b19510f29b178f7e3661.png"},{"id":87954182,"identity":"d09e842a-ccf2-41a7-bd3e-0935363761f9","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":7,"title":"figure 7","display":"","copyasset":false,"role":"figure","size":403546,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of nmdar short linear motif (slim) protein-binding motifs (pbms) in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd short linear motifs (slims) for the glun1, glun2a and glun2b ctds.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-d) \u003c/strong\u003epercent conservation of ctd slims in glun1 \u003cstrong\u003e(b)\u003c/strong\u003e, glun2a \u003cstrong\u003e(c)\u003c/strong\u003e, and glun2b \u003cstrong\u003e(d)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray).\u003c/p\u003e","description":"","filename":"floatimage7.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/b6119c0825c4063a5e6bb6ec.png"},{"id":87954181,"identity":"cb3e5a1b-cd98-4155-84bb-c4d2fbb40f7e","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":8,"title":"figure 8","display":"","copyasset":false,"role":"figure","size":518323,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of nmdar protein-binding motifs (pbms) regions in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd regions for the glun1, glun2a and glun2b ctds.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-d) \u003c/strong\u003epercent conservation of ctd regions in glun1 \u003cstrong\u003e(b)\u003c/strong\u003e, glun2a \u003cstrong\u003e(c)\u003c/strong\u003e, and glun2b \u003cstrong\u003e(d)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray).\u003c/p\u003e","description":"","filename":"floatimage8.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/ccea59247fac75e14ad8ad14.png"},{"id":87954866,"identity":"ae52deed-9d7c-4d30-94e3-0b4ac04fe145","added_by":"auto","created_at":"2025-07-30 18:49:46","extension":"pdf","order_by":0,"title":"","display":"","copyasset":false,"role":"manuscript-pdf","size":5866105,"visible":true,"origin":"","legend":"","description":"","filename":"manuscript.pdf","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/24ce2fd9-cdac-44be-947d-44a6c3564886.pdf"},{"id":87954175,"identity":"cf7d81ad-b94d-4e4f-95f1-e45dee844a82","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":0,"title":"","display":"","copyasset":false,"role":"supplement","size":15353369,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarssupplement20250716.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3ada87e108301cd5d027f600.docx"}],"financialinterests":"no competing interests reported.","formattedtitle":"conservation of human nmda receptor subunits and disease variants in zebrafish","fulltext":[{"header":"background","content":"\u003cp\u003erapid cell-to-cell signaling in the nervous system occurs at specialized structures called synapses. at these sites, the electrical activity of the upstream or presynaptic neuron is converted into the release of a chemical neurotransmitter. this is then converted back into an electrical signal in the postsynaptic neuron by specialized ion channels that are gated by a neurotransmitter, initiating downstream biological signaling cascades. glutamate, the primary excitatory neurotransmitter in the brain, activates a variety of receptors that are essential for synaptic transmission and plasticity. glutamate-gated ion channels play fundamental roles in signaling in both the developing and mature nervous systems (\u003cspan additionalcitationids=\"cr2 cr3 cr4 cr5 cr6 cr7 cr8\" citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e–\u003cspan citationid=\"cr9\" class=\"citationref\"\u003e9\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ethe nmda receptor (nmdar) is a prominent glutamate-gated ion channel subtype (\u003cspan citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e, \u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e). due to its distinct biophysical properties and unique ionotropic and non-ionotropic signaling mechanisms, it contributes to a wide array of nervous and non-nervous system functions (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan additionalcitationids=\"cr11 cr12 cr13\" citationid=\"cr10\" class=\"citationref\"\u003e10\u003c/span\u003e–\u003cspan citationid=\"cr14\" class=\"citationref\"\u003e14\u003c/span\u003e). numerous missense, frameshift, and nonsense mutations (or variants) have been identified in the genes that encode nmdar subunits and are associated with neurodevelopmental disorders, including autism spectrum disorders (asd), intellectual disability, developmental delay, seizures, and schizophrenia (\u003cspan additionalcitationids=\"cr16 cr17 cr18 cr19 cr20 cr21 cr22\" citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e–\u003cspan citationid=\"cr23\" class=\"citationref\"\u003e23\u003c/span\u003e). however, how these variants lead to clinical phenotypes is poorly understood.\u003c/p\u003e\u003cp\u003ezebrafish are a powerful model organism to study neurodevelopmental phenotypes and disorders, including those linked to nmdars (\u003cspan citationid=\"cr19\" class=\"citationref\"\u003e19\u003c/span\u003e, \u003cspan additionalcitationids=\"cr25 cr26 cr27 cr28 cr29 cr30 cr31\" citationid=\"cr24\" class=\"citationref\"\u003e24\u003c/span\u003e–\u003cspan citationid=\"cr32\" class=\"citationref\"\u003e32\u003c/span\u003e). zebrafish develop rapidly and share fundamental developmental and signaling pathways with humans and other vertebrates (\u003cspan citationid=\"cr33\" class=\"citationref\"\u003e33\u003c/span\u003e); they are transparent, allowing for dynamic and non-invasive observations of developmental and cellular processes; and they display robust behaviors that are easily assayed at early larval stages. moreover, larval zebrafish readily take up small molecules from their environment, enabling non-invasive and high-throughput readout of nervous system function in the context of behavioral drug screens (\u003cspan additionalcitationids=\"cr35\" citationid=\"cr34\" class=\"citationref\"\u003e34\u003c/span\u003e–\u003cspan citationid=\"cr36\" class=\"citationref\"\u003e36\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enmdars are heterotetramers composed of two obligatory glun1 (encoded by \u003cem\u003egrin1\u003c/em\u003e) subunits. they are expressed as either diheterotetramers with two glun1 and two glun2 (\u003cem\u003egrin2a-d\u003c/em\u003e) subunits or triheterotetramers of some combination of glun2 and/or glun3 (\u003cem\u003egrin3a-b\u003c/em\u003e) subunits (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e) (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003ea). these nmdar subunits are differentially expressed over the course of neurodevelopment, with glun2a, glun2b, and glun2d being highly expressed during early development, while glun2c, glun3a, and glun3b expression increases into adulthood (\u003cspan citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e). moreover, nmdars display distinct functional and pharmacological properties based on their glun2 or glun3 composition (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e). each subunit consists of four principal domains: an n-terminal domain (ntd), a ligand-binding domain (lbd), a transmembrane domain (tmd) that forms the ion channel, and a c-terminal domain (ctd) (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003eb).\u003c/p\u003e\u003cp\u003e\u003c/p\u003e\u003cp\u003edisease-associated variants (davs) in \u003cem\u003egrin1\u003c/em\u003e, \u003cem\u003egrin2a\u003c/em\u003e, \u003cem\u003egrin2b\u003c/em\u003e, and \u003cem\u003egrin2d\u003c/em\u003e have been identified in pediatric patients with neurodevelopmental disorders (\u003cspan additionalcitationids=\"cr16 cr17\" citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e–\u003cspan citationid=\"cr18\" class=\"citationref\"\u003e18\u003c/span\u003e, \u003cspan citationid=\"cr21\" class=\"citationref\"\u003e21\u003c/span\u003e). these disorders often present with multiple comorbidities, reflecting overlapping neurodevelopmental phenotypes (\u003cspan citationid=\"cr37\" class=\"citationref\"\u003e37\u003c/span\u003e). generally, patients with davs in \u003cem\u003egrin1\u003c/em\u003e present with developmental delay and intellectual disability, though a large proportion of patients also experience seizures (\u003cspan citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e). in contrast, \u003cem\u003egrin2a\u003c/em\u003e and \u003cem\u003egrin2d\u003c/em\u003e are predominantly associated with seizure disorders (\u003cspan citationid=\"cr16\" class=\"citationref\"\u003e16\u003c/span\u003e, \u003cspan citationid=\"cr18\" class=\"citationref\"\u003e18\u003c/span\u003e). \u003cem\u003egrin2b\u003c/em\u003e is a high-risk gene for autism spectrum disorder (asd) (\u003cspan citationid=\"cr17\" class=\"citationref\"\u003e17\u003c/span\u003e, \u003cspan citationid=\"cr38\" class=\"citationref\"\u003e38\u003c/span\u003e, \u003cspan citationid=\"cr39\" class=\"citationref\"\u003e39\u003c/span\u003e), as are \u003cem\u003egrin1\u003c/em\u003e and \u003cem\u003egrin2a\u003c/em\u003e (\u003cspan additionalcitationids=\"cr41\" citationid=\"cr40\" class=\"citationref\"\u003e40\u003c/span\u003e–\u003cspan citationid=\"cr42\" class=\"citationref\"\u003e42\u003c/span\u003e). still, patients with such neurodevelopmental disorders generally experience a variety of both central and peripheral nervous system symptoms (\u003cspan additionalcitationids=\"cr44 cr45\" citationid=\"cr43\" class=\"citationref\"\u003e43\u003c/span\u003e–\u003cspan citationid=\"cr46\" class=\"citationref\"\u003e46\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enmda receptors are highly conserved across species (\u003cspan citationid=\"cr47\" class=\"citationref\"\u003e47\u003c/span\u003e). due to an ancient genome duplication event, zebrafish retain two paralogues for most nmdar genes (e.g., \u003cem\u003egrin1a\u003c/em\u003e \u0026amp; \u003cem\u003egrin1b\u003c/em\u003e) (\u003cspan citationid=\"cr24\" class=\"citationref\"\u003e24\u003c/span\u003e, \u003cspan citationid=\"cr48\" class=\"citationref\"\u003e48\u003c/span\u003e). patch clamp electrophysiology experiments show that zebrafish paralogues display similar functional properties to each other and to their mammalian orthologues (\u003cspan citationid=\"cr27\" class=\"citationref\"\u003e27\u003c/span\u003e, \u003cspan citationid=\"cr28\" class=\"citationref\"\u003e28\u003c/span\u003e). nevertheless, it remains unclear how well the human nmdar genes, including positions with identified missense variants, are conserved in zebrafish. to address this, we investigated nmdar conservation in zebrafish by taking advantage of amino acid sequence alignments (\u003cspan citationid=\"cr28\" class=\"citationref\"\u003e28\u003c/span\u003e) and assessed the conservation of all human nmdar subunits and zebrafish paralogues. we also compared the conservation of positions with missense variants identified in human patients for nmdar subunits expressed during early neurodevelopment. finally, we investigated the conservation of critical functional regions within each structural domain. through this analysis, we define the extent to which critical functional components and regions, with relevance to disease, are conserved across zebrafish nmdars, providing a template to use zebrafish as a model to study nmdars in neurodevelopment and disease pathology.\u003c/p\u003e"},{"header":"methods","content":"\u003cp\u003e\u003cb\u003enmdar sequences and sequence alignments\u003c/b\u003e\u003c/p\u003e\u003cp\u003eamino acid sequences for human genes are (uniprot with protein ids): glun1 (q05586-5; ntd splice isoform, nr1-3), glun2a (q12879-1), glun2b (q13224), glun2c (q14957), glun2d (o15399), glun3a (q8tcu5), and glun3b (o60391).\u003c/p\u003e\u003cp\u003efor mouse subunits: mglun1 (p35438), mglun2a (p35436), mglun2b (q01097), mglun2c (q01098), mglun2d (q03391), mglun2a (a2air5), and mglun3b (q91zu9).\u003c/p\u003e\u003cp\u003efor zebrafish paralogs: zglun1a (f1r366), zglun1b (q6zm67), zglun2aa (a0a8m9pxd6), zglun2ab (f1qde5), zglun2ba (a0a8m9q0i1), zglun2bb (a0a8m1nh89), zglun2ca (e7fh62), zglun2cb (a0a8m3b7k5), zglun2da (i3ni77), zglun2db (e7f3z4), zglun3a (a0a8m3awh5), zglun3ba (a0a8m9p9f6), and zglun3bb (a0a8m6z151).\u003c/p\u003e\u003cp\u003ebreakpoints for nmdar subunit domains and subdomains are based on structures 6whr (rat glun1-glun2b) and 7eu7 (human glun1-glun2a) (\u003cspan citationid=\"cr49\" class=\"citationref\"\u003e49\u003c/span\u003e, \u003cspan citationid=\"cr50\" class=\"citationref\"\u003e50\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ehuman and zebrafish protein sequences were aligned and analyzed for amino acid identity and similarity using align sequences protein blast (blastp), which is publicly available through the national center for biotechnology information (ncbi) or emboss needle pairwise sequence alignment (psa). for each nmdar subunit, the zebrafish paralogs were individually aligned to the human protein. amino acid similarity is defined based on the blosum 62 substitution matrix. for the ntd, lbd, and tmd, these alignments were generally straightforward as the sequences do not diverge greatly in length. in contrast, the ctd is quite large, with the zebrafish sequences generally larger than human. for these alignments, we entered the entire sequence into the alignment tool and used the given result, though this led to a fragmentation of the ctd such that certain portions of the sequence were aligned while others were not. we used the resulting domain-specific conservation values to calculate the conservation of the total protein.\u003c/p\u003e\u003cp\u003e\u003cb\u003evariants analysis\u003c/b\u003e\u003c/p\u003e\u003cp\u003ehuman nmdar variants were identified using clinvar and gnomad (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003eclinvar\u003c/em\u003e. results for each gene were downloaded on september 28th, 2023, using clinvar's bulk download option (\u003cspan class=\"externalref\"\u003e\u003cspan class=\"refsource\"\u003ehttps://www.ncbi.nlm.nih.gov/clinvar/\u003c/span\u003e\u003cspan address=\"https://www.ncbi.nlm.nih.gov/clinvar/\" targettype=\"url\" class=\"reftarget\"\u003e\u003c/span\u003e\u003c/span\u003e). genomic variants that did not result in protein coding changes were removed from the analysis. in the case of \u003cem\u003egrin3a\u003c/em\u003e, protein coding changes in the ppp3r2 gene, a single exon gene whose coding sequence resides within the genomic region of \u003cem\u003egrin3a\u003c/em\u003e and is downloaded alongside \u003cem\u003egrin3a\u003c/em\u003e due to this, was also removed. in cases where multiple protein coding annotations exist for the same variant, only the annotation fitting the canonical transcript was retained.\u003c/p\u003e\u003cp\u003e\u003cem\u003egnomad\u003c/em\u003e: genes were searched for by name on gnomad v4.0.0 (\u003cspan class=\"externalref\"\u003e\u003cspan class=\"refsource\"\u003ehttps://gnomad.broadinstitute.org/\u003c/span\u003e\u003cspan address=\"https://gnomad.broadinstitute.org/\" targettype=\"url\" class=\"reftarget\"\u003e\u003c/span\u003e\u003c/span\u003e) and missense, insertion, deletion, and loss-of-function variant information was exported on september 28th, 2023 as comma separated value (csv) files for each gene. hgvs annotation (protein change) was converted to letter-number-letter annotation.\u003c/p\u003e\u003cp\u003egene variants from clinvar and gnomad were merged using r studio 2023.06.1 build 524 based on the letter-number-letter annotation. the resulting merged file was checked for redundant information or variant duplicates, which were removed.\u003c/p\u003e\u003cp\u003emissense variants were extracted and grouped by pathogenicity as pathogenic, putatively pathogenic, complex, putatively benign, and benign variants based on the criteria in table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e. assessment of conservation of positions with identified human variants was performed using the generated sequence alignments.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar domain-specific analysis\u003c/b\u003e\u003c/p\u003e\u003cp\u003eto further refine the relationship between human and zebrafish nmdar proteins, we analyzed the tmd, lbd-tmd linkers, lbd, ntd, and ctd of glun1, glun2a, glun2b, and glun2d for conservation of key structural and functional features.\u003c/p\u003e\u003cp\u003e\u003cem\u003etransmembrane domain (tmd) and ligand-binding domain-transmembrane domain (lbd-tmd) linkers.\u003c/em\u003e key regions involved in channel gating, mg\u003csup\u003e2+\u003c/sup\u003e block, and ca\u003csup\u003e2+\u003c/sup\u003e flux were identified as described in the literature and analyzed for conservation (\u003cspan additionalcitationids=\"cr52 cr53 cr54 cr55 cr56\" citationid=\"cr51\" class=\"citationref\"\u003e51\u003c/span\u003e–\u003cspan citationid=\"cr57\" class=\"citationref\"\u003e57\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003eligand-binding domain (lbd)\u003c/em\u003e. key regions and specific amino acids involved in van-der-waals interactions with the α-carbon of the ligand, interactions with the α-amino group, electrostatic interactions with the α-carboxyl group, and interactions with the amino acid side chain were identified as described in ramos-vicente et. al. 2021 and analyzed for conservation (\u003cspan citationid=\"cr47\" class=\"citationref\"\u003e47\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003en-terminal domain (ntd)\u003c/em\u003e. key amino acid positions required for the binding of zinc and polyamines were identified via a literature search and their conservation assessed (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr58\" class=\"citationref\"\u003e58\u003c/span\u003e, \u003cspan citationid=\"cr59\" class=\"citationref\"\u003e59\u003c/span\u003e). we also examined the conservation of putative n-linked glycosylation sites (\u003cspan citationid=\"cr60\" class=\"citationref\"\u003e60\u003c/span\u003e). to generate homology models of the zebrafish ntds, we used the template function in swiss-model (\u003cspan citationid=\"cr61\" class=\"citationref\"\u003e61\u003c/span\u003e, \u003cspan citationid=\"cr62\" class=\"citationref\"\u003e62\u003c/span\u003e). as reference structures, we used rodent ntd structures 5tq0 (glun1-glun2a in the presence of edta, no bound zn\u003csup\u003e2+\u003c/sup\u003e for glun1 and glun2a), 5tpz (glun1-glun2b apo state for glun1 and glun2b), and 5tpw (glun1-glun2a with zn\u003csup\u003e2+\u003c/sup\u003e in complex with glun2a) (\u003cspan citationid=\"cr58\" class=\"citationref\"\u003e58\u003c/span\u003e). sequences for human (hglun1, hglun2a, and hglun2b) or zebrafish (zglun1a, zglun1b, glun2aa, glun2ab, glun2ba, glun2bb) ntds were entered individually into swiss-model to generate homology models. the subsequent models were then loaded into pymole (human and zebrafish paralog) and aligned. the resultant root mean square deviation (rmsd) is reported.\u003c/p\u003e\u003cp\u003e\u003cem\u003ecarboxy-terminal domain (ctd)\u003c/em\u003e. ctd-specific conservation in zebrafish was assessed for the glun1, glun2a, and glun2b subunits. glun2d was excluded from this analysis due to the poor conservation of the ctd in these paralogs relative to human. to assess functional relevance, we focused on post-translational modifications (ptms), specifically phosphorylation sites (serine, threonine, tyrosine), ubiquitylation sites (lysine), and palmitoylation sites (cysteine), using a combination of experimentally validated data from phosphositeplus (psp) and published literature (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e). phosphorylation sites with identified kinases or well-characterized functional roles were classified as ‘kinase-regulated phosphorylation sites.’ the remaining phosphorylation sites identified in psp were defined as ‘phosphoproteomic-identified sites.’ note, sites in psp (based on rodents) that did not have a phosphorylation site in humans were excluded from our analysis. in addition, known linear protein-binding motifs were annotated and analyzed for conservation (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e). to assess conservation of protein-binding motifs in the ctds, we categorized sites as either short linear motifs (slims) or broader interaction ‘regions’. slims were defined as experimentally validated, compact sequence elements (typically \u0026lt; 15 residues) that mediate specific and direct interactions with known protein partners. broader regions were defined as extended sequences (\u0026gt; 15 residues) shown to directly bind proteins in biochemical or structural studies but lacking a discrete consensus sequence. all motifs were curated from published literature where discrete sequences were experimentally validated per motif (\u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e, \u003cspan additionalcitationids=\"cr64 cr65 cr66 cr67 cr68 cr69 cr70 cr71 cr72 cr73 cr74 cr75 cr76 cr77 cr78 cr79\" citationid=\"cr63\" class=\"citationref\"\u003e63\u003c/span\u003e–\u003cspan citationid=\"cr80\" class=\"citationref\"\u003e80\u003c/span\u003e), and sequence conservation was examined.\u003c/p\u003e"},{"header":"results","content":"\u003cp\u003eour goal is to compare the structural similarity of zebrafish nmdar (zglun) paralogues to human nmdars (glun), especially regarding human variants. since mice are a common animal model (\u003cspan class=\"citationref\"\u003e81\u003c/span\u003e), as a reference-point for nmdar gene conservation, we initially made global comparisons of both mouse and zebrafish nmdar subunits to human.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eglobal comparison of mouse and zebrafish orthologues to human nmdars\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003enmdars are heterotetramers composed of two obligate glycine-binding glun1 subunits and two glutamate-binding glun2 subunits or a combination of glun2 and glycine-binding glun3 subunits (\u003cspan class=\"citationref\"\u003e1\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e2\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ea,b). glycine and glutamate act as co-agonists for glun2-containing receptors, with binding of both required for receptor activation, while glun3-containing receptors are activated by glycine binding. as an initial assessment of the structural similarity between nmdar orthologues, we made global sequence alignments using the ncbi blastp or emboss psa tools. not surprisingly, given evolutionary proximity, the mouse subunits are highly conserved relative to human: glun1 (97% identity \u0026amp; similarity), glun2a (95% identity; 98% similarity), glun2b (98% identity; 99% similarity), glun2c (88% identity; 90% similarity), glun2d (97% identity \u0026amp; similarity), glun3a (92% identity; 95% similarity), and glun3b (77% identity; 83% similarity) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003eon the other hand, the zebrafish paralogues show reduced conservation. the obligatory glun1 subunits are the most highly conserved, both in terms of identity (\u0026ge;\u0026thinsp;84%) and similarity (\u0026ge;\u0026thinsp;89%) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the other subunits show various degrees of identity and similarity: glun2a (\u0026ge;\u0026thinsp;58% identity; \u0026ge;79% similarity), glun2b (\u0026ge;\u0026thinsp;65% identity; \u0026ge;73% similarity), glun2c (\u0026ge;\u0026thinsp;50% identity; \u0026ge;64% similarity), glun2d (\u0026ge;\u0026thinsp;43% identity; \u0026ge;54% similarity), glun3a (63% identity; 78% similarity), and glun3b (\u0026ge;\u0026thinsp;54% identity, \u0026ge;\u0026thinsp;62% similarity) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). these subunits, thus, retain\u0026thinsp;\u0026ge;\u0026thinsp;50% identity with respect to the human protein, except for zglun2da and zglun2db, which only share 43% and 46% sequence identity, respectively (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). with respect to the zebrafish paralog pairs, there are instances in which one paralog is better conserved than the other. the glun1 paralogs are comparably conserved (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). however, the zglun2aa paralog is more highly conserved than zglun2ab while the zglun2bb shows increased conservation relative to zglun2ba (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the glun2c, glun2d, and glun3b paralogs are conserved to similar extents (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003enevertheless, despite lower conservation relative to mouse, zebrafish offer many technical advantages (see background). in addition, what is critical is the conservation of specific structural and functional domains and subdomains.\u003c/p\u003e\n\u003cdiv align=\"left\" class=\"colspec\"\u003e\u003cbr\u003e\u003c/div\u003e\n\u003ctable id=\"tab1\" border=\"1\"\u003e\n \u003ccaption language=\"en\"\u003e\n \u003cdiv class=\"captionnumber\"\u003etable 1\u003c/div\u003e\n \u003cdiv class=\"captioncontent\"\u003e\n \u003cp\u003eoverall protein conservation in mouse and zebrafish relative to human\u003c/p\u003e\n \u003c/div\u003e\n \u003c/caption\u003e\n \u003cthead\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003ehuman\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003emouse\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003ezebrafish\u003c/p\u003e\n \u003cp\u003eparalogue a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003ezebrafish\u003c/p\u003e\n \u003cp\u003eparalogue b\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003c/thead\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en1\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e938\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e938\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e84%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e89%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e966\u003c/p\u003e\n \u003cp\u003e(-28)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e86%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e90%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e937\u003c/p\u003e\n \u003cp\u003e(-1)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2a\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1464\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e95%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e98%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1464\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e68%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e89%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1460\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;6)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e58%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e79%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1445\u003c/p\u003e\n \u003cp\u003e(-9)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2b\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1484\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e98%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e99%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1482\u003c/p\u003e\n \u003cp\u003e(-2)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e65%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e73%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1417\u003c/p\u003e\n \u003cp\u003e(-31)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e67%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e81%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1770 (+\u0026thinsp;322)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2c\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1233\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e88%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e90%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1239\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;6)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e50%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e64%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1400\u003c/p\u003e\n \u003cp\u003e(-176)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e59%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e79%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1328\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;95)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2d\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1336\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1323\u003c/p\u003e\n \u003cp\u003e(-13)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e43%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e54%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1916 (+\u0026thinsp;580)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e46%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e57%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1676 (+\u0026thinsp;340)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en3a\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1115\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e92%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e95%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1135\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;20)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e63%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e78%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1111\u003c/p\u003e\n \u003cp\u003e(-4)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en3b\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1043\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e77%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e83%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1003\u003c/p\u003e\n \u003cp\u003e(-40)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e54%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e62%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1118\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;75)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e62%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e68%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1114\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;71)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003c/p\u003e\n\u003cp\u003enumbers indicate the percent of mouse or zebrafish paralog conservation with respect to the human protein \u0026ndash; \u0026lsquo;id.\u0026rsquo; is identity and \u0026lsquo;sim.\u0026rsquo; is similarity. the zebrafish a and b paralogs are indicated for each protein. glun3a has only a single paralog in zebrafish. for the mouse and zebrafish protein length, numbers in parentheses indicate the difference in amino acid number relative to human.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of specific structural elements in zebrafish paralogues\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003efor more rigorous comparisons, we examined the conservation within individual subunit domains and subdomains (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the structure of nmdars, like all iglurs, consists of 4 domains that are intrinsic to each individual subunit (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ea,b): the extracellular ntd and lbd; the tmd forming the ion channel as well as the linkers that connect the lbd to the tmd, the lbd-tmd linkers; and the intracellular ctd (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e53\u003c/span\u003e). within individual subunits, the lbd is formed by two disparate sequences, s1 and s2, that fold up into the d1 and d2 lobes in the 3-dimensional structure (83, 84). in general, d1 is made up primarily of s1 and d2 of s2, but there is considerable swapping between lobes. the tmd is formed by three transmembrane segments, m1, m3, and m4, and an m2-pore loop. the lbd-tmd linkers are referred to as either s1-m1, m3-s2, and s2-m4 (linear sequence) or d2-m1, m3-d2, and d1-m4 (3-dimensional structure). this distinction is important functionally since d1 and d2 undergo different movements that are translated differently to the tmd (\u003cspan class=\"citationref\"\u003e85\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e86\u003c/span\u003e). nevertheless, we will use the s1-m1, m3-s2, and s2-m4 designation for ease of comparison to structures.\u003c/p\u003e\n\u003cp\u003ewe began by dividing the zebrafish paralogues into their respective domains and subdomains based on our breakpoints (see methods). interestingly, the amino acid sequences surrounding these breakpoints were well conserved in zebrafish \u0026ndash; the majority were identical (not shown). this allowed for ease of division of the zebrafish proteins into their respective domains and subdomains.\u003c/p\u003e\n\u003cp\u003eas an initial assessment of the structural similarity between human and zebrafish nmdars, we made detailed sequence alignments using the ncbi blastp or emboss psa tools (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e, fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). in terms of length of domains, the zebrafish protein sequence for glun1, glun2, and glun3 subunits was identical to the human sequence for the tmd (m1, m1-m2, m2, m2-m3, m3, \u0026amp; m4) and lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec). the lbd (s1 \u0026amp; s2) was also quite similar in length, with no changes in the length of s2 across the paralogs and only slight increases or decreases in amino acid number in the s1 of zglun1a, zglun1b, zglun2ab, zglun2ca, zglun2ca, zglun2da, and zglun2db (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec). the ntd and ctd were less conserved in terms of size, wherein almost every zebrafish paralog has a different number of amino acids compared to the human. this is especially notable for the glun2b ntd which either lacks 130 amino acids (zglun2ba) or has an additional 74 amino acids (zglun2bb) as well as for the ctd where, for example, zglun2da and zglun2db have a ctd that is more than twice as long as that in the human protein (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec).\u003c/p\u003e\n\u003ctable id=\"tab2\" border=\"1\"\u003e\n \u003ccaption language=\"en\"\u003e\n \u003cdiv class=\"captionnumber\"\u003etable 2\u003c/div\u003e\n \u003cdiv class=\"captioncontent\"\u003e\n \u003cp\u003evariant designation\u003c/p\u003e\n \u003c/div\u003e\n \u003c/caption\u003e\n \u003cthead\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eglun\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003edesignation\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eclinvar\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003egnomad\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e1\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2b\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2c\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2d\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e3a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e3b\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003c/thead\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epathogenic\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;pathogenic\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e49\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e58\u003c/p\u003e\n \u003cp\u003e(4%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e81\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e12\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eputatively\u003c/p\u003e\n \u003cp\u003epathogenic\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;uncertain\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e123\u003c/p\u003e\n \u003cp\u003e(13%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e214\u003c/p\u003e\n \u003cp\u003e(15%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e182\u003c/p\u003e\n \u003cp\u003e(12%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e4\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e111\u003c/p\u003e\n \u003cp\u003e(8%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e5\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e8\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003ecomplex\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;pathogenic\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e3\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e5\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e6\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eputatively benign\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;uncertain\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e80\u003c/p\u003e\n \u003cp\u003e(9%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e320\u003c/p\u003e\n \u003cp\u003e(22%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e199\u003c/p\u003e\n \u003cp\u003e(13%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e31\u003c/p\u003e\n \u003cp\u003e(3%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e223\u003c/p\u003e\n \u003cp\u003e(17%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e30\u003c/p\u003e\n \u003cp\u003e(3%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e68\u003c/p\u003e\n \u003cp\u003e(7%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003ebenign\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;benign\u0026rdquo;\u003c/p\u003e\n \u003cp\u003eor\u003c/p\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent/\u003c/p\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003cp\u003eor\u003c/p\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e401\u003c/p\u003e\n \u003cp\u003e(43%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e813\u003c/p\u003e\n \u003cp\u003e(55%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e672\u003c/p\u003e\n \u003cp\u003e(45%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e945\u003c/p\u003e\n \u003cp\u003e(77%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e863\u003c/p\u003e\n \u003cp\u003e(27%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e59\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e945\u003c/p\u003e\n \u003cp\u003e(91%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003c/p\u003e\n\u003cp\u003enumber of positions with missense variants that have been identified in each subunit per designation. percentages report the proportion of the total protein amino acids at which such variants have been identified.\u003c/p\u003e\n\u003cp\u003ewe used these sequence alignments to assess the degree of amino acid identity and similarity between domains and subdomains (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ed,e). the obligatory glun1 subunit showed the highest degree of conservation. the tmd and lbd-tmd linkers are 100% conserved. the lbd is also well conserved with s1 and s2 showing\u0026thinsp;\u0026ge;\u0026thinsp;92% identity and \u0026ge;\u0026thinsp;93% similarity. the ntd is somewhat less conserved (\u0026ge;\u0026thinsp;80% identity, \u0026ge;\u0026thinsp;88% similarity), as is the ctd (\u0026ge;\u0026thinsp;53% identity; \u0026ge;58% similarity).\u003c/p\u003e\n\u003cp\u003ethis trend of conservation across nmdar subunits \u0026ndash; the lbd and tmd are most highly conserved and the ntd and ctd less so \u0026ndash; is maintained in the glun2 and glun3 subunits (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ed,e). generally, the glun2 subunits maintain a higher sequence identity than the glun3 subunits. overall, this sequence analysis suggests that the lbd and tmd are highly conserved whereas the ntd and ctd show less conservation.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecategorizing the pathogenicity of missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003enmdars with missense, frameshift, and nonsense variants have been identified in pediatric patients with neurodevelopmental disorders (\u003cspan class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e22\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e87\u003c/span\u003e). we therefore wanted to assess the extent to which variants are conserved in zebrafish to aid in guiding future studies. for this analysis, we focused solely on missense variants. while nonsense and frameshift mutation can be modelled in various ways, such as by mutating a portion of the protein to induce a loss-of-function, the conservation of the specific protein sequence is somewhat less relevant. in contrast, the single amino acid change associated with missense variants is critical to dissecting how such variants alter receptor function and lead to disease phenotypes.\u003c/p\u003e\n\u003cp\u003eusing publicly available datasets of nmdar variants from clinvar and gnomad, we identified missense variants in all nmdar subunits (see methods). we first categorized these variants with regards to their pathogenicity (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). clinvar reports variants that have been identified in patients with symptomatic disease (\u003cspan class=\"citationref\"\u003e88\u003c/span\u003e\u0026ndash;\u003cspan class=\"citationref\"\u003e90\u003c/span\u003e), while gnomad reports variants identified in the healthy population (\u003cspan class=\"citationref\"\u003e91\u003c/span\u003e). thus, in general, we define variants present in clinvar but absent in gnomad as pathogenic while those with the opposite reporting as benign. with this approach, pathogenic and putatively pathogenic variants were identified in glun1, glun2a, glun2b, and glun2d while minimal to no such variants were identified in glun2c, glun3a, and glun3b (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). notably, these genes with known pathogenic variants are expressed early in development, contributing to developmental processes while their dysfunction often leads to disease phenotypes (\u003cspan class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e87\u003c/span\u003e). on the other hand, glun2c, glun3a, and glun3b have no pathogenic and a small number of putatively pathogenic variants (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). we therefore focus solely on glun1, glun2a, glun2b, and glun2d for further analysis.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003epathogenic and putatively pathogenic variant positions are well conserved in zebrafish\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003eusing the criteria in table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e, amino acid positions with pathogenic missense variants in glun1, glun2a, glun2b, and glun2d are completely conserved in zebrafish paralogues (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ea-d).\u003c/p\u003e\n\u003cp\u003eputatively pathogenic variants are also conserved in these nmdar subunits, though less so than pathogenic variants. the highest degree of conservation of putatively pathogenic variants is in the glun1 paralogues (\u0026ge;\u0026thinsp;88% identity; 93% similarity). putatively pathogenic variants in the glun2 subunits are also reasonably conserved: zglun2aa (70% identity; 92% similarity), zglun2ab (62% identity; 84% similarity), zglun2ba (65% identity; 71% similarity), zglun2bb (72% identity; 80% similarity), and zglun2da and zglundb (\u0026ge;\u0026thinsp;46% identity; \u0026ge;59% similarity) (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ee,f). in general, the highest level of conservation is in the tmd and tmd-lbd linkers, followed by the lbd, and less so in the ntd and ctd (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003eg,h).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ebenign and putatively benign variant positions show reduced conservation\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ewe carried out a comparable analysis as above for benign and putatively benign variants (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). we hypothesized that positions with more benign variants would be less well conserved than pathogenic ones. indeed, the general trend is consistent with this idea (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003e). putatively benign variants showed a reduced conservation compared to putatively pathogenic variants (putatively pathogenic vs putatively benign): zglun1a (88% vs. 78% identity; 93% vs. 88% similarity), zglun1b (89% vs. 79% identity; 93% vs. 84% similarity), zglun2aa (70% vs. 60% identity; 92% vs. 86% similarity), zglun2ab (62% vs. 48% identity; 84% vs. 68% similarity), zglun2ba (65% vs. 45% identity; 71% vs. 55% similarity), zglun2bb (72% vs. 55% identity; 80% vs. 70% similarity), zglun2da (46% vs. 47% identity; 59% vs. 61% similarity), and zglun2db (47% vs. 44% identity; 61% vs. 56% similarity) (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ee,f vs fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ea,b). for these putatively benign variants, the greatest extent of conservation tends to occur in the tmd and lbd-tmd linkers, followed by the lbd, and then the ntd and ctd (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ec,d).\u003c/p\u003e\n\u003cp\u003efor benign variants, however, the decrease in conservation plateaus, with identity and similarity values comparable to the putatively benign variants, if not slightly higher (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ee-f). the subdomain-specific conservation trend for these variants follows that seen on the level of the whole receptor (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e), wherein the tmd, lbd-tmd linkers, and lbd are generally more conserved and the ntd and ctd less so (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003eg,h). although diverging from our prediction as we do not note a substantial decrease in the conservation of benign variants with respect to the putatively benign variants, this result reflects the overall conservation of the nmdar subunits themselves (benign vs total protein): zglun1 (78% vs. \u0026ge;84% identity; 84% vs. \u0026ge;89% similarity), zglun2aa (62% vs. 68% identity; 86% vs. 89% similarity), zglun2ab (49% vs. 58% identity; 74% vs. 79% similarity), zglun2ba (54% vs. 65% identity; 64% vs. 73% similarity), and zglun2bb (62% vs. 67% identity; 74% vs. 81% similarity), zglun2da (50% vs. 43% identity; 64% vs. 54% similarity), and zglun2db (49% vs. 46% identity; 62% vs. 57% similarity) (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ee-f, table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). furthermore, this result reflects the relatively large number of benign variants in comparison to other variant groups (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecritical tmd and lbd-tmd linker components are completely conserved\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003eelements forming the ion channel \u0026ndash; the tmd (m1, m1-m2, m2, m2-m3, m3, \u0026amp; m4) \u0026ndash; and the lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4) were consistently highly conserved (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). nevertheless, we assayed conservation of critical functional motifs. the s1-m1 linker is a critical element for channel gating and is completely conserved (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e) (\u003cspan class=\"citationref\"\u003e56\u003c/span\u003e). the m2 subdomain contains what is known as the \u0026lsquo;n\u0026rsquo; and \u0026lsquo;n\u0026thinsp;+\u0026thinsp;1\u0026rsquo; sites, which participate in mg\u003csup\u003e2+\u003c/sup\u003e block and ca\u003csup\u003e2+\u003c/sup\u003e permeation through the channel (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e55\u003c/span\u003e) and are conserved in zebrafish (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, dark orange). the tmd m3 segment contains the sytanlaaf motif, which plays a key role in ion channel gating and is known to be highly conserved across species. this motif is conserved in all zebrafish glun1 and glun2 paralogs (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, orange). lastly, the \u0026lsquo;drpeer\u0026rsquo; motif in m3-s2 of glun1, an important component in the high ca\u003csup\u003e2+\u003c/sup\u003e permeability of nmdars (\u003cspan class=\"citationref\"\u003e52\u003c/span\u003e) is conserved (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, light orange). hence, core gating and permeation elements of the tmd and associated regions are highly conserved suggesting that basic mechanisms of ion channel gating and block/permeation are preserved.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003elbd elements required for agonist binding are completely conserved\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ethe glun1 lbd binds glycine or d-serine while the glun2 lbd binds glutamate or aspartate (fig. s2a,b). overall, the lbds are highly conserved in zebrafish relative to humans (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e), as are specific amino acid side chains in s1 and s2 that are involved in binding endogenous agonists (\u003cspan class=\"citationref\"\u003e47\u003c/span\u003e) (fig. s2c). these include sites that participate in van-der waals interactions with the \u0026alpha;-carbon of the ligand (darkest red), interactions with the \u0026alpha;-amino group (dark red), electrostatic interactions with the \u0026alpha;-carboxyl group (red), and interactions with the amino acid side chain (light red) (fig. s2c).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of structure-function elements in the ntd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ein terms of overall sequence comparisons, the ntd shows variable degrees of conservation across nmdar subunits. the glun1 zebrafish paralogs maintain the highest degree of conservation relative to human (80% identity; \u0026ge;88% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the glun2 subunits show decreased ntd conservation (49\u0026ndash;75% identity; 59\u0026ndash;88% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). given this reduced conservation, we assayed the conservation of key structure-function motifs in zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003eamong its many roles, the ntd participates in interactions with ions and molecules that modulate receptor activity (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e59\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ea,b). this is most notable for glun2a and to a lesser extent glun2b as they bind zinc, which inhibits receptor function by reducing channel open probability (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). in glun2a, four amino acids coordinate zinc binding: glun2a-his44, glun2a-his128, glun2a-glu266, and glun2a-asp282 (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). of these key sites, three are conserved (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec) but in zglun2aa and zglun2ab, a key residue involved in zn\u003csup\u003e2+\u003c/sup\u003e binding, glun2a-his44, is not conserved (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec). for glun2b, only two amino acid residues participate in zinc binding: glun2b-his127 and glun2b-glu284 (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). while glun2b-glu284 is conserved, the glun2b-his127 position in zglun2ba is missing due to alterations in ntd sequence length leading to gaps in the alignments, though it is similar now as a positively charged arginine instead of a histidine in zglun2bb (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec).\u003c/p\u003e\n\u003cp\u003ethe ntd of glun2b also binds polyamines, which enhance receptor activity (\u003cspan class=\"citationref\"\u003e59\u003c/span\u003e). nine amino acids are involved in the binding of polyamines. these sites in zglun2ba and zglun2bb are completely conserved except for one amino acid in zglun2bb that is similar but not identical, now a negatively charged aspartate instead of glutamate (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ed). most of this functional component of glun2b is, thus, conserved in zebrafish.\u003c/p\u003e\n\u003cp\u003eadditionally, the nmdar ntd has glycosylation sites, and their glycosylation regulates the trafficking of nmdars to the cell membrane (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e). given the consensus sequence for arginine (n-) glycosylation (n-\u003cem\u003ex\u003c/em\u003e-s/t), glun1 has seven, glun2a has three, and glun2b has three putative glycosylation sites in the ntd (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e). their conservation in the zebrafish paralogues is somewhat variable: zglun1a \u0026amp; zglun1b (5 of 7 sites), zglun2aa (3 of 3 sites), zglun2ab (2 of 3 sites), zglun2ba (2 of 3 sites), and zglun2bb (3 of 3 sites) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ee). to the best of our knowledge, specific glycosylation sites in glun2d have yet to be identified. of note, the glun1 glycosylation sites glun1-n203 and glun1-n368 are necessary for receptor surface expression (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e), and these sites are conserved in the zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003ewhile the critical functional features of the lbd were completely conserved in zebrafish relative to human nmdars, the conservation of key regions in the ntd occasionally diverged, most notably that for zn\u003csup\u003e2+\u003c/sup\u003e binding. to further explore the structural composition of the glun1, glun2a, and glun2b ntds, we used swiss-model to generate homology models for the human and zebrafish nmdar subunits, which we subsequently aligned in pymole to assess conservation at the level of the three-dimensional structure (see methods) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ef). in general, this analysis revealed a high degree of similarity between the human ntd and zebrafish paralogs. the zebrafish glun1 ntd models display similar overall topology to human: glun1 vs zglun1a, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.071); glun1 vs zglun1a, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.085); glun1 vs zglun1b, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.094); and glun1 vs zglun1b, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.097) (table s3).\u003c/p\u003e\n\u003cp\u003ethe glun2a and glun2b subunits also reflect a high degree of structural similarity: glun2a vs zglun2aa, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.085); glun2a vs zglun2ab, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.095); glun2a vs zglun2ab, reference structure 5tpw (rmsd\u0026thinsp;=\u0026thinsp;0.111); glun2b vs zglun2ba, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.028); and glun2b vs glun2bb, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.044) (table s3). the only exception here is the structural comparison of glun2a and zglun2aa using reference structure 5tpw, in which the ntd is interacting with zinc (rmsd\u0026thinsp;=\u0026thinsp;2.149), suggesting that the structural components of zglun2aa may not effectively complex with zinc. interestingly, zglun2ba is \u0026gt;\u0026thinsp;100 amino acids shorter and zglun2bb is \u0026gt;\u0026thinsp;70 amino acids longer than human glun2b. though the zglun2ba model is missing an \u0026alpha;-helix relative to the human protein, the zglun2bb modelled structure aligns well with its human counterpart, suggesting that the zebrafish sequences maintain a large degree of structural integrity.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of post-translational modification (ptm) sites in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ethe intracellular ctd is a key regulatory domain of nmdar cell biology and function and shows distinct subunit-specific elements (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). for our analyses, we focused on the glun1-1 ctd splice variant, which is the longest variant containing the c0-c1-c2 cassettes (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003eacross the nmdar subunits, the zebrafish ctds are the most poorly conserved (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). with a sequence identity generally around 50\u0026ndash;60% for the glun1, glun2a, and glun2b zebrafish paralogs, with some exceptions: zglun1a (53% identity; 58% similarity), zglun1b (54% identity; 59% similarity), zglun2aa (57% identity; 86% identity), zglun2ab (38% identity; 65% similarity), zglun2ba (57% identity, 66% similarity), and zglun2bb (60% identity, 71% similarity) while that of glun2d is much lower (\u0026ge;\u0026thinsp;18% identity, \u0026ge;\u0026thinsp;28% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003egiven these deviations in protein sequences, we first examined the conservation of ctd regions critical for post-translational modifications (ptms), excluding glun2d due its poor conservation in zebrafish. phosphorylation of nmdar subunit ctds confers key functional roles to the receptor (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). initially, we examined experimentally validated sites at which phosphorylation occurs by known kinases and calculated the percent conservation, reported in identity and similarity. similarity, here, is defined as a site at the homologous position that can be phosphorylated (e.g., s, t, or y), if not identical. the glun1 ctd contains phosphorylation sites for src kinase (src), protein kinase c (pkc), and protein kinase a (pka), (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003etop\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). note specific sites have also been identified that are also targeted by the serine/threonine protein phosphatase 2b (pp2b). these ptm sites are perfectly conserved in zglun1a, while three of five are conserved in zglun1b (60% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb). the glun2a ctd is phosphorylated by src, pka, dual specificity tyrosine-phosphorylation-regulated kinase 1a (dyrk1), cyclin-dependent kinase 5 (cdk5), pkc, and calcium/calmodulin-dependent protein kinase ii\u0026alpha; (camkii\u0026alpha;) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003emiddle\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e): zglun2aa (79% identity \u0026amp; similarity) and zglun2ab (64% identity, 71% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb). the ctd of glun2b is phosphorylated by src, camkii\u0026alpha;, cdk5, pka, pkc, death-associated protein kinase 1 (dapk1), proto-oncogene tyrosine-protein kinase fyn (fyn), and casein kinase ii (ck2) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003ebottom\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). glun2b also contains sites for the protein tyrosine phosphatase non-receptor 11 (ptpn11, or shp2) and protein phosphatase 1 (pp1). the conservation of these ptms is slightly higher than that in glun2a: zglun2ba (73% identity \u0026amp; similarity) and zglun2bb (82% identity, 100% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb).\u003c/p\u003e\n\u003cp\u003ein addition to these kinase-regulated sites, we also examined the conservation of other known phosphorylation sites in these subunits identified by phosphoproteomics (psp) (see methods). the conservation of these sites in the zebrafish paralogs was generally lower than that of the kinase-regulated sites: glun1 (67% identity, 100% similarity), zglun2aa (51% identity, 66% similarity), zglun2ab (34% identity, 57% similarity), zglun2ba (60% identity \u0026amp; similarity), and zglun2bb (67% identity, 70% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ec).\u003c/p\u003e\n\u003cp\u003ewe next examined the conservation of ctd palmitoylation and ubiquitylation sites. glun1 has a single ubiquitylation site (lys860) (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003etop\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), and it is conserved in the zebrafish paralogs (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ec). glun2a contains seven palmitoylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003emiddle\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), which are likewise completely conserved in zebrafish (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003eb). the glun2a ctd also has four ubiquitylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003emiddle)\u003c/em\u003e (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), of which three are conserved in zglun2aa (75%) and one in zglun2ab (25%) (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ec). with the same approach for glun2b, we identified eight palmitoylation and six ubiquitylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003ebottom\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). here, all palmitoylation sites are conserved in zebrafish while five of six ubiquitylation sites (83%) are conserved in zglun2ba and all six ubiquitylation sites (100%) are conserved in zglun2bb (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb,c).\u003c/p\u003e\n\u003cp\u003ein summary, the global conservation of ptm sites is as follows: zglun1a and zglun1b (89% identity, 100% similarity), zglun2aa (62% identity, 72% similarity), zglun2ab (45% identity, 62% similarity), zglun2ba (66% identity, 67% similarity), and zglun2bb (75% identity, 78% similarity).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of protein-binding motifs (pbm) in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ein addition to numerous ptm sites, a variety of protein-protein interactions are critical to the role of the ctd in receptor cell biology (surface expression, distribution) and function (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e63\u003c/span\u003e). these protein-binding elements include either defined short linear motifs (slims) or broader regions known to serve as protein docking sites.\u003c/p\u003e\n\u003cp\u003ethe ctd of glun1 contains a slim known as the kkk/rrr er retention motif, which is conserved in zebrafish (\u003cspan class=\"citationref\"\u003e66\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,b). it, likewise, has binding regions for calmodulin (cam)/calcium/calmodulin-dependent protein kinase ii (camkii) and \u0026alpha;-actinin 2 (\u003cspan class=\"citationref\"\u003e64\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e65\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e67\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e78\u003c/span\u003e), which is conserved and yotiao, also known as a-kinase anchoring protein 9 (akap9) (\u003cspan class=\"citationref\"\u003e67\u003c/span\u003e), which is generally well conserved: zglun1a (90% identity, 98% similarity) and zglun1b (90% identity, 98% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,b).\u003c/p\u003e\n\u003cp\u003ethe glun2 subunit ctds contain a variety of pbms. for glun2a, the conservation of slims is: the yxx\u0026oslash; motif, where y is tyrosine, x is any amino acid, and \u0026oslash; is a bulky hydrophobic residue like leucine, isoleucine, phenylalanine, methionine, or valine (\u003cspan class=\"citationref\"\u003e73\u003c/span\u003e) \u0026ndash; both zglun2aa and zglun2ab show 100% identity \u0026amp; similarity; the guanine-nucleotide exchange factor brag2 (\u003cspan class=\"citationref\"\u003e70\u003c/span\u003e) \u0026ndash; zglun2aa (80% identity \u0026amp; similarity) and zglun2ab (40% identity, 60% similarity); postsynaptic density protein 95 (psd-95) (\u003cspan class=\"citationref\"\u003e72\u003c/span\u003e) \u0026ndash; zglun2aa (50% identity; 63% similarity) and zglun2ab (38% identity, 63% similarity); and the pdz domain that interacts with the proteins psd-95, discs large (dlg), and zonula occludens-1 (zo-1) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2aa and zglun2ab (100% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,c). interacting regions in glun2a include: ring finger protein 10 (rnf10) (\u003cspan class=\"citationref\"\u003e70\u003c/span\u003e) \u0026ndash; zglun2aa (51% identity, 72% similarity) and zglun2ab (32% identity, 66% similarity); rabphilin 3a (rph3a) (\u003cspan class=\"citationref\"\u003e68\u003c/span\u003e) \u0026ndash; zglun2aa (71% identity, 80% similarity) and zglun2ab (49% identity, 71% similarity); flotillin-1 (flot-1) (\u003cspan class=\"citationref\"\u003e71\u003c/span\u003e) \u0026ndash; zglun2aa (61% identity, 81% similarity) and zglun2ab (31% identity, 52% similarity); and c-terminal binding protein 1 (ctbp1) (\u003cspan class=\"citationref\"\u003e69\u003c/span\u003e) \u0026ndash; zglun2aa (51% identity, 85% similarity) and zglun2ab (36% identity, 63% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,c, s3a,b).\u003c/p\u003e\n\u003cp\u003efor glun2b, the conservation of slims is: the guanine-nucleotide exchange factor brag1 (\u003cspan class=\"citationref\"\u003e76\u003c/span\u003e) \u0026ndash; zglun2ba (60% identity; 80% similarity) and zglun2bb (60% identity, 100% similarity); psd-95 (\u003cspan class=\"citationref\"\u003e72\u003c/span\u003e) \u0026ndash; zglun2ba (56% identity \u0026amp; similarity) and zglun2bb (33% identity, 56% similarity); camkii and death-associated protein kinase 1 (dapk1) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (92% identity, 100% similarity); synapse-associated protein 102 (sap102) (\u003cspan class=\"citationref\"\u003e75\u003c/span\u003e) \u0026ndash; zglun2ba (0% identity \u0026amp; similarity) and zglun2bb (50% identity \u0026amp; similarity); the adaptor complex ap2 (\u003cspan class=\"citationref\"\u003e74\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (100% identity \u0026amp; similarity); and pdz (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (100% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,d). the ctd interaction regions for glun2b are conserved as follows: receptor for activated c kinase 1 (rack1) (\u003cspan class=\"citationref\"\u003e77\u003c/span\u003e) \u0026ndash; zglun2ba (0% identity, 7% similarity) and zglun2bb (13% identity, 20% similarity); flot-1 (\u003cspan class=\"citationref\"\u003e71\u003c/span\u003e) \u0026ndash; zglun2ba (23% identity; 47% similarity) and zglun2bb (35% identity, 68% similarity); spectrin (\u003cspan class=\"citationref\"\u003e79\u003c/span\u003e) \u0026ndash; zglun2ba (40% identity; 59% similarity) and zglun2bb (35% identity, 62% similarity); \u0026alpha;-actinin 2 (\u003cspan class=\"citationref\"\u003e78\u003c/span\u003e) \u0026ndash; zglun2ba (47% identity; 67% similarity) and zglun2bb (51% identity, 76% similarity); and ras protein-specific guanine nucleotide-releasing factor 1 (rasgrf1) (\u003cspan class=\"citationref\"\u003e80\u003c/span\u003e) \u0026ndash; zglun2ba (59% identity; 74% similarity) and zglun2bb (60% identity, 78% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,d, s3a,c).\u003c/p\u003e\n\u003cp\u003ewhen compared globally across subunits, slims always show greater conservation than regions. this conservation is always greatest in the zebrafish glun1 paralogs, followed by the glun2b and then glun2a paralogs. for the analyzed slims, conservation grouped across each subunit is as follows: zglun1a and zglun1b (100% identity \u0026amp; similarity), zglun2aa (76% identity, 81% similarity), zglun2ab (62% identity, 76% similarity), zglun2ba (78% identity, 81% similarity), and zglun2bb (72% identity, 86% similarity). the regions show the following degrees of conservation: zglun1a and zglun1b (94% identity, 98% similarity), zglun2aa (55% identity, 81% similarity), zglun2ab (36% identity, 62% similarity), zglun2ba (59% identity, 74% similarity), and zglun2bb (60% identity, 78% similarity).\u003c/p\u003e"},{"header":"discussion","content":"\u003cp\u003ehere, we compared the conservation of the zebrafish nmdar subunit paralogues, including key structure-function motifs, to their human counterparts. we find that the core gating machinery \u0026ndash; the lbd, the lbd-tmd linkers and the tmd \u0026ndash; are highly conserved both in terms of overall identity and specific functional motifs. the more peripheral domains \u0026ndash; the ntd and ctd \u0026ndash; show less conservation overall, though functional motifs are reasonably conserved with the notable exception of zn\u003csup\u003e2+\u003c/sup\u003e binding in the glun2a ntd. notably, disease-associated variants are generally conserved, highlighting that zebrafish represent a useful model to study nmdars.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar conservation in zebrafish\u003c/b\u003e\u003c/p\u003e\u003cp\u003eacross most of our examined parameters, the obligatory glun1 subunit shows the highest degree of conservation in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e). generally, conservation is then greatest in the glun2(a-d) followed by the glun3(a-b) subunits. within each subunit, sequence conservation is consistently highest in the tmd and lbd-tmd linkers, followed by the lbd. conservation is generally poorer in the ntd and even worse in the ctd (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ea key question with regards to this work is defining what constitutes as a \u0026lsquo;well conserved\u0026rsquo; sequence. previous studies regarding the conservation of protein structure and function suggest that sequence identities between 40\u0026ndash;70% confer functional conservation (\u003cspan additionalcitationids=\"cr93\" citationid=\"cr92\" class=\"citationref\"\u003e92\u003c/span\u003e\u0026ndash;\u003cspan citationid=\"cr94\" class=\"citationref\"\u003e94\u003c/span\u003e). they regard 50% sequence identity as the threshold below which function drastically diverges (\u003cspan citationid=\"cr93\" class=\"citationref\"\u003e93\u003c/span\u003e, \u003cspan citationid=\"cr94\" class=\"citationref\"\u003e94\u003c/span\u003e). our sequence conservation analyses almost always led to sequence identities\u0026thinsp;\u0026gt;\u0026thinsp;50%, except for zglun2da. nevertheless, we carried out more detailed examinations of more poorly conserved domains, e.g., the ntd and ctd, to assess the conservation of key structure-function motifs.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar missense variant conservation in zebrafish\u003c/b\u003e\u003c/p\u003e\u003cp\u003eto date, a multitude of variants, including missense, frameshift, and nonsense mutations, have been identified in the genes encoding the nmdar subunits (\u003cspan citationid=\"cr21\" class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan citationid=\"cr22\" class=\"citationref\"\u003e22\u003c/span\u003e, \u003cspan citationid=\"cr87\" class=\"citationref\"\u003e87\u003c/span\u003e). we categorized identified missense variants by pathogenicity and analyzed the extent of conservation of the positions at which they occur. we focused only on the glun1, glun2a, glun2b, and glun2d subunits for this analysis, as this is where most disease-associated variants have been identified. generally, our results demonstrate a higher degree of variant position conservation with increasing pathogenicity (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e,\u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e). the sites of pathogenic missense variants are almost perfectly conserved (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e). given the high conservation of the amino acid positions at which they occur, these protein regions are likely most essential for receptor function. across our categories of variants, with the exclusion of complex variants, the proportion of the total protein at which missense variants have been identified is lowest in the pathogenic group (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e). it is possible that additional variants that might be categorized as pathogenic are so devastating to development as to be embryonic lethal. this reflects the notion that more integral regions of the receptor are less tolerant to variants (\u003cspan citationid=\"cr95\" class=\"citationref\"\u003e95\u003c/span\u003e). zebrafish, nevertheless, emerge as a useful model for the study of pathogenic nmdar variants.\u003c/p\u003e\u003cp\u003eputatively pathogenic variants are conserved to a lesser extent than their pathogenic counterparts (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e). according to our designations, these variants have the potential to cause disease, having been identified in patients, but are currently classified as having uncertain significance. thus, it aligns that these variants are comparatively less well conserved \u0026ndash; they are likely occurring in less critical regions of the receptor and are likewise less likely to cause disease phenotypes.\u003c/p\u003e\u003cp\u003eour subsequent investigation of putatively benign and benign variants revealed a further decrease in variant position conservation. this finding further supported our prediction that variants more highly associated with disease occur at positions with higher degrees of conservation and, thus, are more critical to protein structure and function. the conservation of positions with putatively benign and benign variants is quite comparable. for the benign variants, in particular, conservation percentages also closely reflect total protein conservation values for each paralog (fig.\u0026nbsp;\u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e, table\u0026nbsp;\u003cspan refid=\"tab1\" class=\"internalref\"\u003e1\u003c/span\u003e). this follows from the finding that positions with identified benign variants, for most subunits, make up a substantial portion of the protein (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enevertheless, this investigation both demonstrates the advantage of the zebrafish model in studying nmdar-associated disease and reflects the notion that the positions most essential for receptor function are those that are best conserved and more likely to induce disease when absent.\u003c/p\u003e\u003cp\u003eoverall, the trend for variant conservation is the same as that for the whole protein. conservation is always greatest in the tmd and lbd-tmd linkers, followed by the lbd, next the ntd, and last the ctd (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e, \u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e). these findings suggest that zebrafish serve as an effective model for variants, especially pathogenic and putatively pathogenic variants in both the tmd and lbd-tmd linkers. some caution, however, must be exercised when using this model system for variants in the lbd, ntd, and ctd.\u003c/p\u003e\u003cp\u003e\u003cb\u003etmd, tmd-lbd linker, and lbd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003eour examination of key functional regions and motifs in the tmd, lbd-tmd linkers, and lbd consistently revealed complete conservation in their respective zebrafish paralogs (fig. \u003cspan refid=\"moesm1\" class=\"internalref\"\u003es1\u003c/span\u003e,s2). this suggests that the basic mechanism of ion channel gating is conserved in zebrafish relative to human. as these domains and subdomains contribute to the formation of the core of the receptor \u0026ndash; the ion channel pore allowing for ion flux \u0026ndash; it is expected that they would be most highly conserved across species, since the major functional role of nmdars is current flux during synaptic transmission.\u003c/p\u003e\u003cp\u003e\u003cb\u003entd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003ethe ntd is the first region in which nmdar sequences in zebrafish begin to diverge from those in human. modelling ntd variants in zebrafish must, thus, be approached with caution. for example, polyamine-binding sites in glun2b are almost completely conserved in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig8\" class=\"internalref\"\u003e4\u003c/span\u003ed), so they have the potential to serve as an effective model for this functional role of nmdars. on the other hand, zinc-binding sites in glun2a and glun2b are more poorly conserved in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig8\" class=\"internalref\"\u003e4\u003c/span\u003ec), so they would likely serve as a poor model for zinc-related studies in nmdars. this also aligns with our homology model finding that the zglun2aa model aligns well with glun2a when not bound to zinc, but the rmsd of the alignment is substantially poorer when glun2a interacting with zinc is used as the reference structure (table s3).\u003c/p\u003e\u003cp\u003e\u003cb\u003ectd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003ethe c-terminal domains (ctds) exhibit the lowest level of overall conservation across nmdar domains (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e). this is consistent with the ctd\u0026rsquo;s classification as an intrinsically disordered region (idr), which lacks fixed secondary or tertiary structure and is more tolerant to evolutionary divergence (\u003cspan citationid=\"cr96\" class=\"citationref\"\u003e96\u003c/span\u003e, \u003cspan citationid=\"cr97\" class=\"citationref\"\u003e97\u003c/span\u003e). nonetheless, functional conservation within idrs is often maintained through the retention of post-translational modification (ptm) hotspots and short linear motifs (slims) that regulate protein interactions and intracellular signaling (\u003cspan citationid=\"cr98\" class=\"citationref\"\u003e98\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ewe assessed ctd conservation in glun1, glun2a, and glun2b by analyzing experimentally validated phosphorylation, palmitoylation, and ubiquitylation sites, as well as defined protein-binding motifs. phosphorylation sites with known kinase interactions exhibited relatively high conservation, consistent with their well-characterized regulatory roles (figs.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ea,b). in contrast, phosphorylation sites without identified kinases were less consistently retained (fig.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ec). palmitoylation sites were fully conserved across all subunits, while ubiquitylation sites were generally retained, though conservation was notably lower in zglun2ab (fig.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ef) and may reflect paralogue-specific divergence in degradation or trafficking pathways.\u003c/p\u003e\u003cp\u003eprotein-binding motif conservation varied by subunit. glun1 retained nearly all canonical interaction motifs, while glun2a and glun2b displayed region-specific conservation. notably well-conserved elements in the glun2 subunits included the pdz-binding motifs as well as docking regions for camkii, dapk1, and ap2 (fig.\u0026nbsp;\u003cspan refid=\"fig12\" class=\"internalref\"\u003e6\u003c/span\u003e). slims overall showed higher conservation, likely due to their short, sequence-specific structure, which may place them under stronger evolutionary constraint than broader, more flexible interaction regions.\u003c/p\u003e\u003cp\u003edespite broad sequence divergence, nmdar ctds retain a core set of conserved motifs involved in receptor trafficking, synaptic localization, and plasticity. kinase-regulated phosphorylation sites, palmitoylation sites, and pdz-binding motifs were consistently preserved. conservation of glun1 sites was always greatest in the zebrafish paralogs. among the glun2 subunits, glun2b displayed the highest overall conservation of functional elements, aligning with its essential role in neurodevelopment and synaptic signaling.\u003c/p\u003e\u003cp\u003ethese findings are especially relevant to the study of disease-associated truncating variants in the nmdar subunits. nonsense variants that introduce premature stop codons in the ctd eliminate key regulatory motifs critical for trafficking, anchoring, or signaling. when these affected regions correspond to conserved functional elements, zebrafish may serve as an appropriate \u003cem\u003ein vitro\u003c/em\u003e and \u003cem\u003ein vivo\u003c/em\u003e model for studying ctd-mediated disease mechanisms. conversely, for mutations in poorly conserved regions, the translational utility of this model may become limited.\u003c/p\u003e"},{"header":"conclusions","content":"\u003cp\u003ezebrafish represent a valuable system to study nmdar-associated neurodevelopmental disorders, but one must be judicious in choosing which variants to study in this model. our unique approach of sequence alignments per individual nmdar domains and subdomains highlights the high degree of conversation of some regions of the protein relative to others. additionally, our analysis of identified variants indicates that zebrafish are an effective model for pathogenic variants as these occur at conserved locations along the protein. generally, the utility of zebrafish is most apparent for the study of the tmd, lbd-tmd linkers, and lbd as these regions are most well conserved. on the other hand, studying the ntd and ctd in this species requires extra consideration stemming from deviations in sequence conservation. our work will enable future nmdar-related studies that can be effectively and efficiently conducted in zebrafish.\u003c/p\u003e"},{"header":"abbreviations","content":"\u003ctable border=\"0\" cellspacing=\"0\" cellpadding=\"0\"\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003enmdar\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003e\u003cem\u003en\u003c/em\u003e-methyl-d-aspartate receptor\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003entd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003en-terminal domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003elbd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003eligand-binding domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003etmd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003etransmembrane domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003ectd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003ec-terminal domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e"},{"header":"declarations","content":"\u003cp\u003e\u003cstrong\u003eethics approval and consent to participate:\u0026nbsp;\u003c/strong\u003enot applicable.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econsent for publication:\u0026nbsp;\u003c/strong\u003enot applicable.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eavailability of data and materials:\u003c/strong\u003e the datasets used and/or analyzed during the current study are available either at osf (https://osf.io/h2zf9/) or from the corresponding author on reasonable request.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecompeting interests:\u0026nbsp;\u003c/strong\u003eno competing interests declared.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003efunding:\u003c/strong\u003e this work was supported by nih grant r01ns088479 to lpw.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eauthors\u0026rsquo; contributions:\u0026nbsp;\u003c/strong\u003ee.r.n., c.a., j.d.z., h.i.s., and l.p.w. contributed to the conceptualization and methodology of this project. e.r.n., c.a., and j.d.z. conducted the investigation and analysis. w.r. assisted with data curation. e.r.n, c.a., and l.p.w. wrote the manuscript, which was revised with insights from j.d.z. and h.i.s.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eacknowledgements:\u0026nbsp;\u003c/strong\u003ethe authors thank the clinvar and gnomad databases and the swiss-model server for their important contributions to this work.\u003c/p\u003e"},{"header":"references","content":"\u003col\u003e\n\u003cli\u003epaoletti p, bellone c, zhou q. nmda receptor subunit diversity: impact on receptor properties, synaptic plasticity and disease. nat rev neurosci. 2013;14(6):383-400.\u003c/li\u003e\n\u003cli\u003ehansen kb, wollmuth lp, bowie d, furukawa h, menniti fs, sobolevsky ai, et al. structure, function, and pharmacology of glutamate receptor ion channels. pharmacol rev. 2021;73(4):298-487.\u003c/li\u003e\n\u003cli\u003echakraborty a, murphy s, coleman n. the role of nmda receptors in neural stem cell proliferation and differentiation. stem cells dev. 2017;26(11):798-807.\u003c/li\u003e\n\u003cli\u003enacher j, mcewen bs. the role of n-methyl-d-asparate receptors in neurogenesis. hippocampus. 2006;16(3):267-70.\u003c/li\u003e\n\u003cli\u003ekomuro h, rakic p. modulation of neuronal migration by nmda receptors. science. 1993;260(5104):95-7.\u003c/li\u003e\n\u003cli\u003epearce ia, cambray-deakin ma, burgoyne rd. glutamate acting on nmda receptors stimulates neurite outgrowth from cerebellar granule cells. febs lett. 1987;223(1):143-7.\u003c/li\u003e\n\u003cli\u003ebooker sa, wyllie dja. nmda receptor function in inhibitory neurons. neuropharmacology. 2021;196:108609.\u003c/li\u003e\n\u003cli\u003egambrill ac, barria a. nmda receptor subunit composition controls synaptogenesis and synapse stabilization. proc natl acad sci u s a. 2011;108(14):5855-60.\u003c/li\u003e\n\u003cli\u003enapoli aj, laderwager s, zoodsma jd, biju b, mucollari o, schubel sk, et al. developmental loss of nmda receptors results in supernumerary forebrain neurons through delayed maturation of transit-amplifying neuroblasts. sci rep. 2024;14(1):3395.\u003c/li\u003e\n\u003cli\u003erajani v, sengar as, salter mw. tripartite signalling by nmda receptors. mol brain. 2020;13(1):23.\u003c/li\u003e\n\u003cli\u003eishchenko y, carrizales mg, koleske aj. regulation of the nmda receptor by its cytoplasmic domains: (how) is the tail wagging the dog? neuropharmacology. 2021;195:108634.\u003c/li\u003e\n\u003cli\u003emony l, paoletti p. mechanisms of nmda receptor regulation. curr opin neurobiol. 2023;83:102815.\u003c/li\u003e\n\u003cli\u003enielsen ac, anderson cl, ens c, boyce akj, thompson rj. non-ionotropic nmdar signalling activates panx1 to induce p2x4r-dependent long-term depression in the hippocampus. j physiol. 2024.\u003c/li\u003e\n\u003cli\u003ebarnes sa, thomazeau a, finnie psb, heinrich mj, heynen aj, komiyama nh, et al. non-ionotropic signaling through the nmda receptor glun2b carboxy-terminal domain drives dendritic spine plasticity and reverses fragile x phenotypes. cell rep. 2025;44(3):115311.\u003c/li\u003e\n\u003cli\u003eplatzer k, lemke jr. grin1-related neurodevelopmental disorder. genereviews((r)). 2021.\u003c/li\u003e\n\u003cli\u003estrehlow v, myers ka, morgan at, scheffer ie, lemke jr. grin2a-related disorders. genereviews((r)). 2024.\u003c/li\u003e\n\u003cli\u003eplatzer k, lemke jr. grin2b-related neurodevelopmental disorder. genereviews((r)). 2021.\u003c/li\u003e\n\u003cli\u003eplatzer k, krey i, lemke jr. grin2d-related developmental and epileptic encephalopathy. genereviews((r)). 2022.\u003c/li\u003e\n\u003cli\u003ezehavi y, mandel h, zehavi a, rashid ma, straussberg r, jabur b, et al. de novo grin1 mutations: an emerging cause of severe early infantile encephalopathy. eur j med genet. 2017;60(6):317-20.\u003c/li\u003e\n\u003cli\u003esabo sl, lahr jm, offer m, weekes a, sceniak mp. grin2b-related neurodevelopmental disorder: current understanding of pathophysiological mechanisms. front synaptic neurosci. 2022;14:1090865.\u003c/li\u003e\n\u003cli\u003egjerulfsen ce, krey i, klockner c, rubboli g, lemke jr, moller rs. spectrum of nmda receptor variants in neurodevelopmental disorders and epilepsy. methods mol biol. 2024;2799:1-11.\u003c/li\u003e\n\u003cli\u003eamin jb, moody gr, wollmuth lp. from bedside-to-bench: what disease-associated variants are teaching us about the nmda receptor. j physiol. 2021;599(2):397-416.\u003c/li\u003e\n\u003cli\u003ehu c, chen w, myers sj, yuan h, traynelis sf. human grin2b variants in neurodevelopmental disorders. j pharmacol sci. 2016;132(2):115-21.\u003c/li\u003e\n\u003cli\u003ecox ja, kucenas s, voigt mm. molecular characterization and embryonic expression of the family of n-methyl-d-aspartate receptor subunit genes in the zebrafish. dev dyn. 2005;234(3):756-66.\u003c/li\u003e\n\u003cli\u003eveldman mb, lin s. zebrafish as a developmental model organism for pediatric research. pediatr res. 2008;64(5):470-6.\u003c/li\u003e\n\u003cli\u003echoi ty, choi ti, lee yr, choe sk, kim ch. zebrafish as an animal model for biomedical research. exp mol med. 2021;53(3):310-7.\u003c/li\u003e\n\u003cli\u003ezoodsma jd, chan k, bhandiwad aa, golann dr, liu g, syed sa, et al. a model to study nmda receptors in early nervous system development. j neurosci. 2020;40(18):3631-45.\u003c/li\u003e\n\u003cli\u003ezoodsma jd, keegan ej, moody gr, bhandiwad aa, napoli aj, burgess ha, et al. disruption of grin2b, an asd-associated gene, produces social deficits in zebrafish. mol autism. 2022;13(1):38.\u003c/li\u003e\n\u003cli\u003ebaraban sc, taylor mr, castro pa, baier h. pentylenetetrazole induced changes in zebrafish behavior, neural activity and c-fos expression. neuroscience. 2005;131(3):759-68.\u003c/li\u003e\n\u003cli\u003eshams s, rihel j, ortiz jg, gerlai r. the zebrafish as a promising tool for modeling human brain disorders: a review based upon an ibns symposium. neurosci biobehav rev. 2018;85:176-90.\u003c/li\u003e\n\u003cli\u003ede abreu ms, genario r, giacomini a, demin ka, lakstygal am, amstislavskaya tg, et al. zebrafish as a model of neurodevelopmental disorders. neuroscience. 2020;445:3-11.\u003c/li\u003e\n\u003cli\u003eweinschutz mendes h, neelakantan u, liu y, fitzpatrick se, chen t, wu w, et al. high-throughput functional analysis of autism genes in zebrafish identifies convergence in dopaminergic and neuroimmune pathways. cell rep. 2023;42(3):112243.\u003c/li\u003e\n\u003cli\u003ehowe k, clark md, torroja cf, torrance j, berthelot c, muffato m, et al. the zebrafish reference genome sequence and its relationship to the human genome. nature. 2013;496(7446):498-503.\u003c/li\u003e\n\u003cli\u003erihel j, schier af. behavioral screening for neuroactive drugs in zebrafish. dev neurobiol. 2012;72(3):373-85.\u003c/li\u003e\n\u003cli\u003ebaraban sc, dinday mt, hortopan ga. drug screening in scn1a zebrafish mutant identifies clemizole as a potential dravet syndrome treatment. nat commun. 2013;4:2410.\u003c/li\u003e\n\u003cli\u003ezhang y, kecskes a, copmans d, langlois m, crawford ad, ceulemans b, et al. pharmacological characterization of an antisense knockdown zebrafish model of dravet syndrome: inhibition of epileptic seizures by the serotonin agonist fenfluramine. plos one. 2015;10(5):e0125898.\u003c/li\u003e\n\u003cli\u003edingemans ajm, jansen s, van reeuwijk j, de leeuw n, pfundt r, schuurs-hoeijmakers j, et al. prevalence of comorbidities in individuals with neurodevelopmental disorders from the aggregated phenomics data of 51,227 pediatric individuals. nat med. 2024;30(7):1994-2003.\u003c/li\u003e\n\u003cli\u003eohtsuki t, sakurai k, dou h, toru m, yamakawa-kobayashi k, arinami t. mutation analysis of the nmdar2b (grin2b) gene in schizophrenia. mol psychiatry. 2001;6(2):211-6.\u003c/li\u003e\n\u003cli\u003earnold pd, rosenberg dr, mundo e, tharmalingam s, kennedy jl, richter ma. association of a glutamate (nmda) subunit receptor gene (grin2b) with obsessive-compulsive disorder: a preliminary study. psychopharmacology (berl). 2004;174(4):530-8.\u003c/li\u003e\n\u003cli\u003esaunders ja, tatard-leitman vm, suh j, billingslea en, roberts tp, siegel sj. knockout of nmda receptors in parvalbumin interneurons recreates autism-like phenotypes. autism res. 2013;6(2):69-77.\u003c/li\u003e\n\u003cli\u003egandal mj, sisti j, klook k, ortinski pi, leitman v, liang y, et al. gabab-mediated rescue of altered excitatory-inhibitory balance, gamma synchrony and behavioral deficits following constitutive nmdar-hypofunction. transl psychiatry. 2012;2(7):e142.\u003c/li\u003e\n\u003cli\u003ebarnby g, abbott a, sykes n, morris a, weeks de, mott r, et al. candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at grin2a and abat. am j hum genet. 2005;76(6):950-66.\u003c/li\u003e\n\u003cli\u003ehung ly, margolis kg. autism spectrum disorders and the gastrointestinal tract: insights into mechanisms and clinical relevance. nat rev gastroenterol hepatol. 2024;21(3):142-63.\u003c/li\u003e\n\u003cli\u003eowens ap, mathias cj, iodice v. autonomic dysfunction in autism spectrum disorder. front integr neurosci. 2021;15:787037.\u003c/li\u003e\n\u003cli\u003eorefice ll. peripheral somatosensory neuron dysfunction: emerging roles in autism spectrum disorders. neuroscience. 2020;445:120-9.\u003c/li\u003e\n\u003cli\u003emakris g, agorastos a, chrousos gp, pervanidou p. stress system activation in children and adolescents with autism spectrum disorder. front neurosci. 2021;15:756628.\u003c/li\u003e\n\u003cli\u003eramos-vicente d, grant sg, bayes a. metazoan evolution and diversity of glutamate receptors and their auxiliary subunits. neuropharmacology. 2021;195:108640.\u003c/li\u003e\n\u003cli\u003eamores a, force a, yan yl, joly l, amemiya c, fritz a, et al. zebrafish hox clusters and vertebrate genome evolution. science. 1998;282(5394):1711-4.\u003c/li\u003e\n\u003cli\u003echou th, tajima n, romero-hernandez a, furukawa h. structural basis of functional transitions in mammalian nmda receptors. cell. 2020;182(2):357-71 e13.\u003c/li\u003e\n\u003cli\u003ezhang y, ye f, zhang t, lv s, zhou l, du d, et al. structural basis of ketamine action on human nmda receptors. nature. 2021;596(7871):301-5.\u003c/li\u003e\n\u003cli\u003ewollmuth lp, sakmann b. different mechanisms of ca2+ transport in nmda and ca2+-permeable ampa glutamate receptor channels. j gen physiol. 1998;112(5):623-36.\u003c/li\u003e\n\u003cli\u003ewatanabe j, beck c, kuner t, premkumar ls, wollmuth lp. drpeer: a motif in the extracellular vestibule conferring high ca2+ flux rates in nmda receptor channels. j neurosci. 2002;22(23):10209-16.\u003c/li\u003e\n\u003cli\u003ewollmuth lp, sobolevsky ai. structure and gating of the glutamate receptor ion channel. trends neurosci. 2004;27(6):321-8.\u003c/li\u003e\n\u003cli\u003eogden kk, khatri a, traynelis sf, heldt sa. potentiation of glun2c/d nmda receptor subtypes in the amygdala facilitates the retention of fear and extinction learning in mice. neuropsychopharmacology. 2014;39(3):625-37.\u003c/li\u003e\n\u003cli\u003ewollmuth lp. ion permeation in ionotropic glutamate receptors: still dynamic after all these years. curr opin physiol. 2018;2:36-41.\u003c/li\u003e\n\u003cli\u003emcdaniel mj, ogden kk, kell sa, burger pb, liotta dc, traynelis sf. nmda receptor channel gating control by the pre-m1 helix. j gen physiol. 2020;152(4).\u003c/li\u003e\n\u003cli\u003ewollmuth lp, kuner t, sakmann b. adjacent asparagines in the nr2-subunit of the nmda receptor channel control the voltage-dependent block by extracellular mg2+. j physiol. 1998;506 ( pt 1)(pt 1):13-32.\u003c/li\u003e\n\u003cli\u003eromero-hernandez a, simorowski n, karakas e, furukawa h. molecular basis for subtype specificity and high-affinity zinc inhibition in the glun1-glun2a nmda receptor amino-terminal domain. neuron. 2016;92(6):1324-36.\u003c/li\u003e\n\u003cli\u003emony l, zhu s, carvalho s, paoletti p. molecular basis of positive allosteric modulation of glun2b nmda receptors by polyamines. embo j. 2011;30(15):3134-46.\u003c/li\u003e\n\u003cli\u003elichnerova k, kaniakova m, park sp, skrenkova k, wang yx, petralia rs, et al. two n-glycosylation sites in the glun1 subunit are essential for releasing n-methyl-d-aspartate (nmda) receptors from the endoplasmic reticulum. j biol chem. 2015;290(30):18379-90.\u003c/li\u003e\n\u003cli\u003ewaterhouse a, bertoni m, bienert s, studer g, tauriello g, gumienny r, et al. swiss-model: homology modelling of protein structures and complexes. nucleic acids res. 2018;46(w1):w296-w303.\u003c/li\u003e\n\u003cli\u003ebienert s, waterhouse a, de beer ta, tauriello g, studer g, bordoli l, et al. the swiss-model repository-new features and functionality. nucleic acids res. 2017;45(d1):d313-d9.\u003c/li\u003e\n\u003cli\u003ewarnet xl, bakke krog h, sevillano-quispe og, poulsen h, kjaergaard m. the c-terminal domains of the nmda receptor: how intrinsically disordered tails affect signalling, plasticity and disease. eur j neurosci. 2021;54(8):6713-39.\u003c/li\u003e\n\u003cli\u003ekim k, saneyoshi t, hosokawa t, okamoto k, hayashi y. interplay of enzymatic and structural functions of camkii in long-term potentiation. j neurochem. 2016;139(6):959-72.\u003c/li\u003e\n\u003cli\u003ebej a, ames jb. chemical shift assignments of calmodulin bound to a cytosolic domain of glun2a (residues 1004-1024) from the nmda receptor. biomol nmr assign. 2023;17(1):89-93.\u003c/li\u003e\n\u003cli\u003ehorak m, wenthold rj. different roles of c-terminal cassettes in the trafficking of full-length nr1 subunits to the cell surface. j biol chem. 2009;284(15):9683-91.\u003c/li\u003e\n\u003cli\u003elin jw, wyszynski m, madhavan r, sealock r, kim ju, sheng m. yotiao, a novel protein of neuromuscular junction and brain that interacts with specific splice variants of nmda receptor subunit nr1. j neurosci. 1998;18(6):2017-27.\u003c/li\u003e\n\u003cli\u003estanic j, carta m, eberini i, pelucchi s, marcello e, genazzani aa, et al. rabphilin 3a retains nmda receptors at synaptic sites through interaction with glun2a/psd-95 complex. nat commun. 2015;6:10181.\u003c/li\u003e\n\u003cli\u003ecousins sl, stephenson fa. identification of c-terminal binding protein 1 as a novel nmda receptor interactor. neurochem res. 2019;44(6):1437-45.\u003c/li\u003e\n\u003cli\u003efranchini l, carrano n, di luca m, gardoni f. synaptic glun2a-containing nmda receptors: from physiology to pathological synaptic plasticity. int j mol sci. 2020;21(4).\u003c/li\u003e\n\u003cli\u003eswanwick cc, shapiro me, yi z, chang k, wenthold rj. nmda receptors interact with flotillin-1 and -2, lipid raft-associated proteins. febs lett. 2009;583(8):1226-30.\u003c/li\u003e\n\u003cli\u003ecousins sl, stephenson fa. identification of n-methyl-d-aspartic acid (nmda) receptor subtype-specific binding sites that mediate direct interactions with scaffold protein psd-95. j biol chem. 2012;287(16):13465-76.\u003c/li\u003e\n\u003cli\u003evissel b, krupp jj, heinemann sf, westbrook gl. a use-dependent tyrosine dephosphorylation of nmda receptors is independent of ion flux. nat neurosci. 2001;4(6):587-96.\u003c/li\u003e\n\u003cli\u003elavezzari g, mccallum j, lee r, roche kw. differential binding of the ap-2 adaptor complex and psd-95 to the c-terminus of the nmda receptor subunit nr2b regulates surface expression. neuropharmacology. 2003;45(6):729-37.\u003c/li\u003e\n\u003cli\u003echen bs, gray ja, sanz-clemente a, wei z, thomas ev, nicoll ra, et al. sap102 mediates synaptic clearance of nmda receptors. cell rep. 2012;2(5):1120-8.\u003c/li\u003e\n\u003cli\u003eelagabani mn, brisevac d, kintscher m, pohle j, kohr g, schmitz d, et al. subunit-selective n-methyl-d-aspartate (nmda) receptor signaling through brefeldin a-resistant arf guanine nucleotide exchange factors brag1 and brag2 during synapse maturation. j biol chem. 2016;291(17):9105-18.\u003c/li\u003e\n\u003cli\u003eyaka r, thornton c, vagts aj, phamluong k, bonci a, ron d. nmda receptor function is regulated by the inhibitory scaffolding protein, rack1. proc natl acad sci u s a. 2002;99(8):5710-5.\u003c/li\u003e\n\u003cli\u003ewyszynski m, lin j, rao a, nigh e, beggs ah, craig am, et al. competitive binding of alpha-actinin and calmodulin to the nmda receptor. nature. 1997;385(6615):439-42.\u003c/li\u003e\n\u003cli\u003ewechsler a, teichberg vi. brain spectrin binding to the nmda receptor is regulated by phosphorylation, calcium and calmodulin. embo j. 1998;17(14):3931-9.\u003c/li\u003e\n\u003cli\u003ekrapivinsky g, krapivinsky l, manasian y, ivanov a, tyzio r, pellegrino c, et al. the nmda receptor is coupled to the erk pathway by a direct interaction between nr2b and rasgrf1. neuron. 2003;40(4):775-84.\u003c/li\u003e\n\u003cli\u003emukherjee p, roy s, ghosh d, nandi sk. role of animal models in biomedical research: a review. lab anim res. 2022;38(1):18.\u003c/li\u003e\n\u003cli\u003echou th, epstein m, fritzemeier rg, akins ns, paladugu s, ullman ez, et al. molecular mechanism of ligand gating and opening of nmda receptor. nature. 2024;632(8023):209-17.\u003c/li\u003e\n\u003cli\u003earmstrong n, sun y, chen gq, gouaux e. structure of a glutamate-receptor ligand-binding core in complex with kainate. nature. 1998;395(6705):913-7.\u003c/li\u003e\n\u003cli\u003earmstrong n, gouaux e. mechanisms for activation and antagonism of an ampa-sensitive glutamate receptor: crystal structures of the glur2 ligand binding core. neuron. 2000;28(1):165-81.\u003c/li\u003e\n\u003cli\u003ekazi r, gan q, talukder i, markowitz m, salussolia cl, wollmuth lp. asynchronous movements prior to pore opening in nmda receptors. j neurosci. 2013;33(29):12052-66.\u003c/li\u003e\n\u003cli\u003eamin jb, gochman a, he m, certain n, wollmuth lp. nmda receptors require multiple pre-opening gating steps for efficient synaptic activity. neuron. 2021;109(3):488-501 e4.\u003c/li\u003e\n\u003cli\u003ehardingham ge, do kq. linking early-life nmdar hypofunction and oxidative stress in schizophrenia pathogenesis. nat rev neurosci. 2016;17(2):125-34.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, riley gr, jang w, rubinstein ws, church dm, et al. clinvar: public archive of relationships among sequence variation and human phenotype. nucleic acids res. 2014;42(database issue):d980-5.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, benson m, brown g, chao c, chitipiralla s, et al. clinvar: public archive of interpretations of clinically relevant variants. nucleic acids res. 2016;44(d1):d862-8.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, benson m, brown gr, chao c, chitipiralla s, et al. clinvar: improving access to variant interpretations and supporting evidence. nucleic acids res. 2018;46(d1):d1062-d7.\u003c/li\u003e\n\u003cli\u003egudmundsson s, singer-berk m, watts na, phu w, goodrich jk, solomonson m, et al. variant interpretation using population databases: lessons from gnomad. hum mutat. 2022;43(8):1012-30.\u003c/li\u003e\n\u003cli\u003etian w, skolnick j. how well is enzyme function conserved as a function of pairwise sequence identity? j mol biol. 2003;333(4):863-82.\u003c/li\u003e\n\u003cli\u003eaddou s, rentzsch r, lee d, orengo ca. domain-based and family-specific sequence identity thresholds increase the levels of reliable protein function transfer. j mol biol. 2009;387(2):416-30.\u003c/li\u003e\n\u003cli\u003esangar v, blankenberg dj, altman n, lesk am. quantitative sequence-function relationships in proteins based on gene ontology. bmc bioinformatics. 2007;8:294.\u003c/li\u003e\n\u003cli\u003eperszyk re, kristensen as, lyuboslavsky p, traynelis sf. three-dimensional missense tolerance ratio analysis. genome res. 2021;31(8):1447-61.\u003c/li\u003e\n\u003cli\u003euversky vn. intrinsically disordered proteins from a to z. int j biochem cell biol. 2011;43(8):1090-103.\u003c/li\u003e\n\u003cli\u003esingleton md, eisen mb. evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. plos comput biol. 2024;20(4):e1012028.\u003c/li\u003e\n\u003cli\u003edavey ne, cowan jl, shields dc, gibson tj, coldwell mj, edwards rj. slimprints: conservation-based discovery of functional motif fingerprints in intrinsically disordered protein regions. nucleic acids res. 2012;40(21):10628-41.\u003c/li\u003e\n\u003c/ol\u003e"}],"fulltextsource":"","fulltext":"","funders":[],"hasadminpriorityonworkflow":false,"hasmanuscriptdocx":true,"hasoptedintopreprint":true,"haspassedjournalqc":"","hasanypriority":false,"hidejournal":false,"highlight":"","institution":"","isacceptedbyjournal":false,"isauthorsuppliedpdf":false,"isdeskrejected":"","ishiddenfromsearch":false,"isinqc":false,"isinworkflow":false,"ispdf":false,"ispdfuptodate":true,"iswithdrawnorretracted":false,"journal":{"display":true,"email":"info@researchsquare.com","identity":"bmc-genomics","isnaturejournal":false,"hasqc":true,"allowdirectsubmit":false,"externalidentity":"gics","sidebox":"learn more about [bmc genomics](http://bmcgenomics.biomedcentral.com/)","snpcode":"","submissionurl":"https://www.editorialmanager.com/gics","title":"bmc genomics","twitterhandle":"#bmcgenomics","acdcenabled":true,"dfaenabled":false,"editorialsystem":"em","reportingportfolio":"bmc series","inreviewenabled":true,"inreviewrevisionsenabled":true},"keywords":"grin genes, clinvar, gnomad, sequence alignments, disease-associated variants, n-terminal domain, ligand-binding domain, transmembrane domain, c-terminal domain, ion channel","lastpublisheddoi":"10.21203/rs.3.rs-7151578/v1","lastpublisheddoiurl":"https://doi.org/10.21203/rs.3.rs-7151578/v1","license":{"name":"cc by 4.0","url":"https://creativecommons.org/licenses/by/4.0/"},"manuscriptabstract":"\u003ch2\u003ebackground\u003c/h2\u003e\u003cp\u003enmda receptors (nmdars) are widely expressed, glutamate-gated ion channels that play key roles in brain development and function. variants have been identified in the \u003cem\u003egrin\u003c/em\u003e genes encoding nmdar subunits that are linked to neurodevelopmental disorders, among other manifestations. zebrafish are a powerful model to study brain development and function given their rapid development and ease of genetic manipulation. as a result of an ancient genome duplication, zebrafish possess two paralogues for most human nmdar subunits. to evaluate the degree of conservation between human nmdar subunits and their respective zebrafish paralogues, we carried out detailed \u003cem\u003ein silico\u003c/em\u003e analyses, with an emphasis on key functional elements. to further assess the suitability of zebrafish for modeling nmdar-associated neurodevelopmental disorders, we analyzed the conservation of positions with identified missense variants.\u003c/p\u003e\u003ch2\u003eresults\u003c/h2\u003e\u003cp\u003ewe find that the human nmdar subunits are generally well conserved across zebrafish paralogs. moreover, variants classified as pathogenic and putatively pathogenic are highly conserved, reflecting the importance of key protein regions to neurotypical receptor function. positions with putatively benign and benign variants are less conserved. across nmdar domains, the transmembrane domain is most highly conserved, followed by the ligand-binding domain, which maintains conservation of amino acids that participate in the binding of ligands. the n-terminal domain is less well conserved but aligned homology models show high degrees of structural similarity. the c-terminal domain is the most poorly conserved region across zebrafish paralogs, but certain key regions that undergo phosphorylation, palmitoylation, and ubiquitylation as well as protein-binding motifs are better conserved.\u003c/p\u003e\u003ch2\u003econclusions\u003c/h2\u003e\u003cp\u003eour findings highlight a strong conservation of human nmdar subunits in zebrafish, with some exceptions. the ligand-binding domain, the transmembrane domain forming the ion channel and the short polypeptide linkers that connect them are highly conserved. the n- and c-terminal domains are less conserved but functional motifs in general, except for the zn\u003csup\u003e2+\u003c/sup\u003e binding site in glun2a paralogues, are more highly conserved relative to the entire domain. overall, our findings support the utility of zebrafish as a model for studying neurodevelopment and disease mechanisms and provide a template for rigorously considering the relationship between human and zebrafish paralogues.\u003c/p\u003e","manuscripttitle":"conservation of human nmda receptor subunits and disease variants in zebrafish","msid":"","msnumber":"","nondraftversions":[{"code":1,"date":"2025-07-30 18:25:39","doi":"10.21203/rs.3.rs-7151578/v1","editorialevents":[{"type":"communitycomments","content":0},{"type":"decision","content":"revision requested","date":"2025-09-24t19:23:39+00:00","index":"","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-09-19t21:34:21+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"269262391104669611722251259561774961128","date":"2025-09-19t14:02:55+00:00","index":"hide","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-09-09t17:52:57+00:00","index":"hide","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-08-18t16:11:58+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"201191728820772676273599414131783545114","date":"2025-08-04t10:39:46+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"230034227985411933590242857371107944843","date":"2025-07-28t13:05:21+00:00","index":"hide","fulltext":""},{"type":"reviewersinvited","content":"","date":"2025-07-28t07:38:37+00:00","index":"","fulltext":""},{"type":"editorassigned","content":"","date":"2025-07-28t07:10:47+00:00","index":"","fulltext":""},{"type":"editorinvited","content":"","date":"2025-07-23t18:56:26+00:00","index":"","fulltext":""},{"type":"checkscomplete","content":"","date":"2025-07-22t16:36:43+00:00","index":"","fulltext":""},{"type":"submitted","content":"bmc genomics","date":"2025-07-22t16:32:19+00:00","index":"","fulltext":""}],"status":"published","journal":{"display":true,"email":"info@researchsquare.com","identity":"bmc-genomics","isnaturejournal":false,"hasqc":true,"allowdirectsubmit":false,"externalidentity":"gics","sidebox":"learn more about [bmc genomics](http://bmcgenomics.biomedcentral.com/)","snpcode":"","submissionurl":"https://www.editorialmanager.com/gics","title":"bmc genomics","twitterhandle":"#bmcgenomics","acdcenabled":true,"dfaenabled":false,"editorialsystem":"em","reportingportfolio":"bmc series","inreviewenabled":true,"inreviewrevisionsenabled":true}}],"origin":"","owneridentity":"2b85bbad-8f38-4c35-89b3-5949e6b6d3c0","owner":[],"posteddate":"july 30th, 2025","published":true,"recenteditorialevents":[],"rejectedjournal":[],"revision":"","amendment":"","status":"under-review","subjectareas":[],"tags":[],"updatedat":"2025-10-14t16:08:28+00:00","versionofrecord":[],"versioncreatedat":"2025-07-30 18:25:39","video":"","vordoi":"","vordoiurl":"","workflowstages":[]},"version":"v1","identity":"rs-7151578","journalconfig":"researchsquare"},"__n_ssp":true},"page":"/article/[identity]/[[...version]]","query":{"identity":"rs-7151578","version":["v1"]},"buildid":"idgtm6tzxtmdq-mqt1zbo","isfallback":false,"dynamicids":[87228],"gssp":true,"scriptloader":[]}

## Introduction
rgba(0,0,0,0.85);color:white;padding:1rem;box-sizing:border-box;}.lightbox-wrapper .yarl__slide_description,.lightbox-wrapper .yarl__slide_description *{-webkit-user-select:text !important;-moz-user-select:text !important;-ms-user-select:text !important;user-select:text !important;pointer-events:auto !important;} browse preprints in review journals covid-19 preprints aje video bytes research tools research promotion aje professional editing aje rubriq about preprint platform in review editorial policies research quality evaluation our team advisory board help center sign in submit a preprint cite share download pdf research article conservation of human nmda receptor subunits and disease variants in zebrafish erica r. nebet, christieann aprea, josiah d. zoodsma, william raab, and 2 more this is a preprint; it has not been peer reviewed by a journal. https://doi.org/ 10.21203/rs.3.rs-7151578/v1 this work is licensed under a cc by 4.0 license status: under review version 1 posted 12 you are reading this latest preprint version abstract background nmda receptors (nmdars) are widely expressed, glutamate-gated ion channels that play key roles in brain development and function. variants have been identified in the grin genes encoding nmdar subunits that are linked to neurodevelopmental disorders, among other manifestations. zebrafish are a powerful model to study brain development and function given their rapid development and ease of genetic manipulation. as a result of an ancient genome duplication, zebrafish possess two paralogues for most human nmdar subunits. to evaluate the degree of conservation between human nmdar subunits and their respective zebrafish paralogues, we carried out detailed in silico analyses, with an emphasis on key functional elements. to further assess the suitability of zebrafish for modeling nmdar-associated neurodevelopmental disorders, we analyzed the conservation of positions with identified missense variants. results we find that the human nmdar subunits are generally well conserved across zebrafish paralogs. moreover, variants classified as pathogenic and putatively pathogenic are highly conserved, reflecting the importance of key protein regions to neurotypical receptor function. positions with putatively benign and benign variants are less conserved. across nmdar domains, the transmembrane domain is most highly conserved, followed by the ligand-binding domain, which maintains conservation of amino acids that participate in the binding of ligands. the n-terminal domain is less well conserved but aligned homology models show high degrees of structural similarity. the c-terminal domain is the most poorly conserved region across zebrafish paralogs, but certain key regions that undergo phosphorylation, palmitoylation, and ubiquitylation as well as protein-binding motifs are better conserved. conclusions our findings highlight a strong conservation of human nmdar subunits in zebrafish, with some exceptions. the ligand-binding domain, the transmembrane domain forming the ion channel and the short polypeptide linkers that connect them are highly conserved. the n- and c-terminal domains are less conserved but functional motifs in general, except for the zn 2+ binding site in glun2a paralogues, are more highly conserved relative to the entire domain. overall, our findings support the utility of zebrafish as a model for studying neurodevelopment and disease mechanisms and provide a template for rigorously considering the relationship between human and zebrafish paralogues. grin genes clinvar gnomad sequence alignments disease-associated variants n-terminal domain ligand-binding domain transmembrane domain c-terminal domain ion channel figures figure 1 figure 2 figure 3 figure 4 figure 5 figure 6 figure 7 figure 8 background rapid cell-to-cell signaling in the nervous system occurs at specialized structures called synapses. at these sites, the electrical activity of the upstream or presynaptic neuron is converted into the release of a chemical neurotransmitter. this is then converted back into an electrical signal in the postsynaptic neuron by specialized ion channels that are gated by a neurotransmitter, initiating downstream biological signaling cascades. glutamate, the primary excitatory neurotransmitter in the brain, activates a variety of receptors that are essential for synaptic transmission and plasticity. glutamate-gated ion channels play fundamental roles in signaling in both the developing and mature nervous systems ( 1 – 9 ). the nmda receptor (nmdar) is a prominent glutamate-gated ion channel subtype ( 1 , 2 ). due to its distinct biophysical properties and unique ionotropic and non-ionotropic signaling mechanisms, it contributes to a wide array of nervous and non-nervous system functions ( 2 , 10 – 14 ). numerous missense, frameshift, and nonsense mutations (or variants) have been identified in the genes that encode nmdar subunits and are associated with neurodevelopmental disorders, including autism spectrum disorders (asd), intellectual disability, developmental delay, seizures, and schizophrenia ( 15 – 23 ). however, how these variants lead to clinical phenotypes is poorly understood. zebrafish are a powerful model organism to study neurodevelopmental phenotypes and disorders, including those linked to nmdars ( 19 , 24 – 32 ). zebrafish develop rapidly and share fundamental developmental and signaling pathways with humans and other vertebrates ( 33 ); they are transparent, allowing for dynamic and non-invasive observations of developmental and cellular processes; and they display robust behaviors that are easily assayed at early larval stages. moreover, larval zebrafish readily take up small molecules from their environment, enabling non-invasive and high-throughput readout of nervous system function in the context of behavioral drug screens ( 34 – 36 ). nmdars are heterotetramers composed of two obligatory glun1 (encoded by grin1 ) subunits. they are expressed as either diheterotetramers with two glun1 and two glun2 ( grin2a-d ) subunits or triheterotetramers of some combination of glun2 and/or glun3 ( grin3a-b ) subunits ( 2 ) (fig. 1 a). these nmdar subunits are differentially expressed over the course of neurodevelopment, with glun2a, glun2b, and glun2d being highly expressed during early development, while glun2c, glun3a, and glun3b expression increases into adulthood ( 1 ). moreover, nmdars display distinct functional and pharmacological properties based on their glun2 or glun3 composition ( 2 ). each subunit consists of four principal domains: an n-terminal domain (ntd), a ligand-binding domain (lbd), a transmembrane domain (tmd) that forms the ion channel, and a c-terminal domain (ctd) (fig. 1 b). disease-associated variants (davs) in grin1 , grin2a , grin2b , and grin2d have been identified in pediatric patients with neurodevelopmental disorders ( 15 – 18 , 21 ). these disorders often present with multiple comorbidities, reflecting overlapping neurodevelopmental phenotypes ( 37 ). generally, patients with davs in grin1 present with developmental delay and intellectual disability, though a large proportion of patients also experience seizures ( 15 ). in contrast, grin2a and grin2d are predominantly associated with seizure disorders ( 16 , 18 ). grin2b is a high-risk gene for autism spectrum disorder (asd) ( 17 , 38 , 39 ), as are grin1 and grin2a ( 40 – 42 ). still, patients with such neurodevelopmental disorders generally experience a variety of both central and peripheral nervous system symptoms ( 43 – 46 ). nmda receptors are highly conserved across species ( 47 ). due to an ancient genome duplication event, zebrafish retain two paralogues for most nmdar genes (e.g., grin1a &amp; grin1b ) ( 24 , 48 ). patch clamp electrophysiology experiments show that zebrafish paralogues display similar functional properties to each other and to their mammalian orthologues ( 27 , 28 ). nevertheless, it remains unclear how well the human nmdar genes, including positions with identified missense variants, are conserved in zebrafish. to address this, we investigated nmdar conservation in zebrafish by taking advantage of amino acid sequence alignments ( 28 ) and assessed the conservation of all human nmdar subunits and zebrafish paralogues. we also compared the conservation of positions with missense variants identified in human patients for nmdar subunits expressed during early neurodevelopment. finally, we investigated the conservation of critical functional regions within each structural domain. through this analysis, we define the extent to which critical functional components and regions, with relevance to disease, are conserved across zebrafish nmdars, providing a template to use zebrafish as a model to study nmdars in neurodevelopment and disease pathology. methods nmdar sequences and sequence alignments amino acid sequences for human genes are (uniprot with protein ids): glun1 (q05586-5; ntd splice isoform, nr1-3), glun2a (q12879-1), glun2b (q13224), glun2c (q14957), glun2d (o15399), glun3a (q8tcu5), and glun3b (o60391). for mouse subunits: mglun1 (p35438), mglun2a (p35436), mglun2b (q01097), mglun2c (q01098), mglun2d (q03391), mglun2a (a2air5), and mglun3b (q91zu9). for zebrafish paralogs: zglun1a (f1r366), zglun1b (q6zm67), zglun2aa (a0a8m9pxd6), zglun2ab (f1qde5), zglun2ba (a0a8m9q0i1), zglun2bb (a0a8m1nh89), zglun2ca (e7fh62), zglun2cb (a0a8m3b7k5), zglun2da (i3ni77), zglun2db (e7f3z4), zglun3a (a0a8m3awh5), zglun3ba (a0a8m9p9f6), and zglun3bb (a0a8m6z151). breakpoints for nmdar subunit domains and subdomains are based on structures 6whr (rat glun1-glun2b) and 7eu7 (human glun1-glun2a) ( 49 , 50 ). human and zebrafish protein sequences were aligned and analyzed for amino acid identity and similarity using align sequences protein blast (blastp), which is publicly available through the national center for biotechnology information (ncbi) or emboss needle pairwise sequence alignment (psa). for each nmdar subunit, the zebrafish paralogs were individually aligned to the human protein. amino acid similarity is defined based on the blosum 62 substitution matrix. for the ntd, lbd, and tmd, these alignments were generally straightforward as the sequences do not diverge greatly in length. in contrast, the ctd is quite large, with the zebrafish sequences generally larger than human. for these alignments, we entered the entire sequence into the alignment tool and used the given result, though this led to a fragmentation of the ctd such that certain portions of the sequence were aligned while others were not. we used the resulting domain-specific conservation values to calculate the conservation of the total protein. variants analysis human nmdar variants were identified using clinvar and gnomad (table 2 ). clinvar . results for each gene were downloaded on september 28th, 2023, using clinvar&#x27;s bulk download option ( https://www.ncbi.nlm.nih.gov/clinvar/ ). genomic variants that did not result in protein coding changes were removed from the analysis. in the case of grin3a , protein coding changes in the ppp3r2 gene, a single exon gene whose coding sequence resides within the genomic region of grin3a and is downloaded alongside grin3a due to this, was also removed. in cases where multiple protein coding annotations exist for the same variant, only the annotation fitting the canonical transcript was retained. gnomad : genes were searched for by name on gnomad v4.0.0 ( https://gnomad.broadinstitute.org/ ) and missense, insertion, deletion, and loss-of-function variant information was exported on september 28th, 2023 as comma separated value (csv) files for each gene. hgvs annotation (protein change) was converted to letter-number-letter annotation. gene variants from clinvar and gnomad were merged using r studio 2023.06.1 build 524 based on the letter-number-letter annotation. the resulting merged file was checked for redundant information or variant duplicates, which were removed. missense variants were extracted and grouped by pathogenicity as pathogenic, putatively pathogenic, complex, putatively benign, and benign variants based on the criteria in table 2 . assessment of conservation of positions with identified human variants was performed using the generated sequence alignments. nmdar domain-specific analysis to further refine the relationship between human and zebrafish nmdar proteins, we analyzed the tmd, lbd-tmd linkers, lbd, ntd, and ctd of glun1, glun2a, glun2b, and glun2d for conservation of key structural and functional features. transmembrane domain (tmd) and ligand-binding domain-transmembrane domain (lbd-tmd) linkers. key regions involved in channel gating, mg 2+ block, and ca 2+ flux were identified as described in the literature and analyzed for conservation ( 51 – 57 ). ligand-binding domain (lbd) . key regions and specific amino acids involved in van-der-waals interactions with the α-carbon of the ligand, interactions with the α-amino group, electrostatic interactions with the α-carboxyl group, and interactions with the amino acid side chain were identified as described in ramos-vicente et. al. 2021 and analyzed for conservation ( 47 ). n-terminal domain (ntd) . key amino acid positions required for the binding of zinc and polyamines were identified via a literature search and their conservation assessed ( 2 , 58 , 59 ). we also examined the conservation of putative n-linked glycosylation sites ( 60 ). to generate homology models of the zebrafish ntds, we used the template function in swiss-model ( 61 , 62 ). as reference structures, we used rodent ntd structures 5tq0 (glun1-glun2a in the presence of edta, no bound zn 2+ for glun1 and glun2a), 5tpz (glun1-glun2b apo state for glun1 and glun2b), and 5tpw (glun1-glun2a with zn 2+ in complex with glun2a) ( 58 ). sequences for human (hglun1, hglun2a, and hglun2b) or zebrafish (zglun1a, zglun1b, glun2aa, glun2ab, glun2ba, glun2bb) ntds were entered individually into swiss-model to generate homology models. the subsequent models were then loaded into pymole (human and zebrafish paralog) and aligned. the resultant root mean square deviation (rmsd) is reported. carboxy-terminal domain (ctd) . ctd-specific conservation in zebrafish was assessed for the glun1, glun2a, and glun2b subunits. glun2d was excluded from this analysis due to the poor conservation of the ctd in these paralogs relative to human. to assess functional relevance, we focused on post-translational modifications (ptms), specifically phosphorylation sites (serine, threonine, tyrosine), ubiquitylation sites (lysine), and palmitoylation sites (cysteine), using a combination of experimentally validated data from phosphositeplus (psp) and published literature ( 2 , 11 ). phosphorylation sites with identified kinases or well-characterized functional roles were classified as ‘kinase-regulated phosphorylation sites.’ the remaining phosphorylation sites identified in psp were defined as ‘phosphoproteomic-identified sites.’ note, sites in psp (based on rodents) that did not have a phosphorylation site in humans were excluded from our analysis. in addition, known linear protein-binding motifs were annotated and analyzed for conservation ( 2 , 11 ). to assess conservation of protein-binding motifs in the ctds, we categorized sites as either short linear motifs (slims) or broader interaction ‘regions’. slims were defined as experimentally validated, compact sequence elements (typically &lt; 15 residues) that mediate specific and direct interactions with known protein partners. broader regions were defined as extended sequences (&gt; 15 residues) shown to directly bind proteins in biochemical or structural studies but lacking a discrete consensus sequence. all motifs were curated from published literature where discrete sequences were experimentally validated per motif ( 11 , 63 – 80 ), and sequence conservation was examined. results our goal is to compare the structural similarity of zebrafish nmdar (zglun) paralogues to human nmdars (glun), especially regarding human variants. since mice are a common animal model ( 81 ), as a reference-point for nmdar gene conservation, we initially made global comparisons of both mouse and zebrafish nmdar subunits to human. global comparison of mouse and zebrafish orthologues to human nmdars nmdars are heterotetramers composed of two obligate glycine-binding glun1 subunits and two glutamate-binding glun2 subunits or a combination of glun2 and glycine-binding glun3 subunits ( 1 , 2 ) (fig. 1 a,b). glycine and glutamate act as co-agonists for glun2-containing receptors, with binding of both required for receptor activation, while glun3-containing receptors are activated by glycine binding. as an initial assessment of the structural similarity between nmdar orthologues, we made global sequence alignments using the ncbi blastp or emboss psa tools. not surprisingly, given evolutionary proximity, the mouse subunits are highly conserved relative to human: glun1 (97% identity &amp; similarity), glun2a (95% identity; 98% similarity), glun2b (98% identity; 99% similarity), glun2c (88% identity; 90% similarity), glun2d (97% identity &amp; similarity), glun3a (92% identity; 95% similarity), and glun3b (77% identity; 83% similarity) (table 1 ). on the other hand, the zebrafish paralogues show reduced conservation. the obligatory glun1 subunits are the most highly conserved, both in terms of identity (≥ 84%) and similarity (≥ 89%) (table 1 ). the other subunits show various degrees of identity and similarity: glun2a (≥ 58% identity; ≥79% similarity), glun2b (≥ 65% identity; ≥73% similarity), glun2c (≥ 50% identity; ≥64% similarity), glun2d (≥ 43% identity; ≥54% similarity), glun3a (63% identity; 78% similarity), and glun3b (≥ 54% identity, ≥ 62% similarity) (table 1 ). these subunits, thus, retain ≥ 50% identity with respect to the human protein, except for zglun2da and zglun2db, which only share 43% and 46% sequence identity, respectively (table 1 ). with respect to the zebrafish paralog pairs, there are instances in which one paralog is better conserved than the other. the glun1 paralogs are comparably conserved (table 1 ). however, the zglun2aa paralog is more highly conserved than zglun2ab while the zglun2bb shows increased conservation relative to zglun2ba (table 1 ). the glun2c, glun2d, and glun3b paralogs are conserved to similar extents (table 1 ). nevertheless, despite lower conservation relative to mouse, zebrafish offer many technical advantages (see background). in addition, what is critical is the conservation of specific structural and functional domains and subdomains. table 1 overall protein conservation in mouse and zebrafish relative to human human mouse zebrafish paralogue a zebrafish paralogue b protein length id. sim. protein length id. sim. protein length id. sim. protein length n1 938 97% 97% 938 84% 89% 966 (-28) 86% 90% 937 (-1) n2a 1464 95% 98% 1464 68% 89% 1460 (+ 6) 58% 79% 1445 (-9) n2b 1484 98% 99% 1482 (-2) 65% 73% 1417 (-31) 67% 81% 1770 (+ 322) n2c 1233 88% 90% 1239 (+ 6) 50% 64% 1400 (-176) 59% 79% 1328 (+ 95) n2d 1336 97% 97% 1323 (-13) 43% 54% 1916 (+ 580) 46% 57% 1676 (+ 340) n3a 1115 92% 95% 1135 (+ 20) 63% 78% 1111 (-4) n3b 1043 77% 83% 1003 (-40) 54% 62% 1118 (+ 75) 62% 68% 1114 (+ 71) numbers indicate the percent of mouse or zebrafish paralog conservation with respect to the human protein – ‘id.’ is identity and ‘sim.’ is similarity. the zebrafish a and b paralogs are indicated for each protein. glun3a has only a single paralog in zebrafish. for the mouse and zebrafish protein length, numbers in parentheses indicate the difference in amino acid number relative to human. conservation of specific structural elements in zebrafish paralogues for more rigorous comparisons, we examined the conservation within individual subunit domains and subdomains (fig. 1 ). the structure of nmdars, like all iglurs, consists of 4 domains that are intrinsic to each individual subunit (fig. 1 a,b): the extracellular ntd and lbd; the tmd forming the ion channel as well as the linkers that connect the lbd to the tmd, the lbd-tmd linkers; and the intracellular ctd ( 2 , 53 ). within individual subunits, the lbd is formed by two disparate sequences, s1 and s2, that fold up into the d1 and d2 lobes in the 3-dimensional structure (83, 84). in general, d1 is made up primarily of s1 and d2 of s2, but there is considerable swapping between lobes. the tmd is formed by three transmembrane segments, m1, m3, and m4, and an m2-pore loop. the lbd-tmd linkers are referred to as either s1-m1, m3-s2, and s2-m4 (linear sequence) or d2-m1, m3-d2, and d1-m4 (3-dimensional structure). this distinction is important functionally since d1 and d2 undergo different movements that are translated differently to the tmd ( 85 , 86 ). nevertheless, we will use the s1-m1, m3-s2, and s2-m4 designation for ease of comparison to structures. we began by dividing the zebrafish paralogues into their respective domains and subdomains based on our breakpoints (see methods). interestingly, the amino acid sequences surrounding these breakpoints were well conserved in zebrafish – the majority were identical (not shown). this allowed for ease of division of the zebrafish proteins into their respective domains and subdomains. as an initial assessment of the structural similarity between human and zebrafish nmdars, we made detailed sequence alignments using the ncbi blastp or emboss psa tools (table 1 , fig. 1 ). in terms of length of domains, the zebrafish protein sequence for glun1, glun2, and glun3 subunits was identical to the human sequence for the tmd (m1, m1-m2, m2, m2-m3, m3, &amp; m4) and lbd-tmd linkers (s1-m1, m3-s2, &amp; s2-m4) (fig. 1 c). the lbd (s1 &amp; s2) was also quite similar in length, with no changes in the length of s2 across the paralogs and only slight increases or decreases in amino acid number in the s1 of zglun1a, zglun1b, zglun2ab, zglun2ca, zglun2ca, zglun2da, and zglun2db (fig. 1 c). the ntd and ctd were less conserved in terms of size, wherein almost every zebrafish paralog has a different number of amino acids compared to the human. this is especially notable for the glun2b ntd which either lacks 130 amino acids (zglun2ba) or has an additional 74 amino acids (zglun2bb) as well as for the ctd where, for example, zglun2da and zglun2db have a ctd that is more than twice as long as that in the human protein (fig. 1 c). table 2 variant designation glun designation clinvar gnomad 1 2a 2b 2c 2d 3a 3b pathogenic “pathogenic” absent 49 (5%) 58 (4%) 81 (5%) 0 12 (&lt; 1%) 0 0 putatively pathogenic “uncertain” absent 123 (13%) 214 (15%) 182 (12%) 4 (&lt; 1%) 111 (8%) 5 (&lt; 1%) 8 (&lt; 1%) complex “pathogenic” present 3 (&lt; 1%) 5 (&lt; 1%) 6 (&lt; 1%) 0 1 (&lt; 1%) 0 0 putatively benign “uncertain” present 80 (9%) 320 (22%) 199 (13%) 31 (3%) 223 (17%) 30 (3%) 68 (7%) benign “benign” or absent absent/ present or present 401 (43%) 813 (55%) 672 (45%) 945 (77%) 863 (27%) 59 (5%) 945 (91%) number of positions with missense variants that have been identified in each subunit per designation. percentages report the proportion of the total protein amino acids at which such variants have been identified. we used these sequence alignments to assess the degree of amino acid identity and similarity between domains and subdomains (fig. 1 d,e). the obligatory glun1 subunit showed the highest degree of conservation. the tmd and lbd-tmd linkers are 100% conserved. the lbd is also well conserved with s1 and s2 showing ≥ 92% identity and ≥ 93% similarity. the ntd is somewhat less conserved (≥ 80% identity, ≥ 88% similarity), as is the ctd (≥ 53% identity; ≥58% similarity). this trend of conservation across nmdar subunits – the lbd and tmd are most highly conserved and the ntd and ctd less so – is maintained in the glun2 and glun3 subunits (fig. 1 d,e). generally, the glun2 subunits maintain a higher sequence identity than the glun3 subunits. overall, this sequence analysis suggests that the lbd and tmd are highly conserved whereas the ntd and ctd show less conservation. categorizing the pathogenicity of missense variants nmdars with missense, frameshift, and nonsense variants have been identified in pediatric patients with neurodevelopmental disorders ( 21 , 22 , 87 ). we therefore wanted to assess the extent to which variants are conserved in zebrafish to aid in guiding future studies. for this analysis, we focused solely on missense variants. while nonsense and frameshift mutation can be modelled in various ways, such as by mutating a portion of the protein to induce a loss-of-function, the conservation of the specific protein sequence is somewhat less relevant. in contrast, the single amino acid change associated with missense variants is critical to dissecting how such variants alter receptor function and lead to disease phenotypes. using publicly available datasets of nmdar variants from clinvar and gnomad, we identified missense variants in all nmdar subunits (see methods). we first categorized these variants with regards to their pathogenicity (table 2 ). clinvar reports variants that have been identified in patients with symptomatic disease ( 88 – 90 ), while gnomad reports variants identified in the healthy population ( 91 ). thus, in general, we define variants present in clinvar but absent in gnomad as pathogenic while those with the opposite reporting as benign. with this approach, pathogenic and putatively pathogenic variants were identified in glun1, glun2a, glun2b, and glun2d while minimal to no such variants were identified in glun2c, glun3a, and glun3b (table 2 ). notably, these genes with known pathogenic variants are expressed early in development, contributing to developmental processes while their dysfunction often leads to disease phenotypes ( 21 , 87 ). on the other hand, glun2c, glun3a, and glun3b have no pathogenic and a small number of putatively pathogenic variants (table 2 ). we therefore focus solely on glun1, glun2a, glun2b, and glun2d for further analysis. pathogenic and putatively pathogenic variant positions are well conserved in zebrafish using the criteria in table 2 , amino acid positions with pathogenic missense variants in glun1, glun2a, glun2b, and glun2d are completely conserved in zebrafish paralogues (fig. 2 a-d). putatively pathogenic variants are also conserved in these nmdar subunits, though less so than pathogenic variants. the highest degree of conservation of putatively pathogenic variants is in the glun1 paralogues (≥ 88% identity; 93% similarity). putatively pathogenic variants in the glun2 subunits are also reasonably conserved: zglun2aa (70% identity; 92% similarity), zglun2ab (62% identity; 84% similarity), zglun2ba (65% identity; 71% similarity), zglun2bb (72% identity; 80% similarity), and zglun2da and zglundb (≥ 46% identity; ≥59% similarity) (fig. 2 e,f). in general, the highest level of conservation is in the tmd and tmd-lbd linkers, followed by the lbd, and less so in the ntd and ctd (fig. 2 g,h). benign and putatively benign variant positions show reduced conservation we carried out a comparable analysis as above for benign and putatively benign variants (table 2 ). we hypothesized that positions with more benign variants would be less well conserved than pathogenic ones. indeed, the general trend is consistent with this idea (fig. 3 ). putatively benign variants showed a reduced conservation compared to putatively pathogenic variants (putatively pathogenic vs putatively benign): zglun1a (88% vs. 78% identity; 93% vs. 88% similarity), zglun1b (89% vs. 79% identity; 93% vs. 84% similarity), zglun2aa (70% vs. 60% identity; 92% vs. 86% similarity), zglun2ab (62% vs. 48% identity; 84% vs. 68% similarity), zglun2ba (65% vs. 45% identity; 71% vs. 55% similarity), zglun2bb (72% vs. 55% identity; 80% vs. 70% similarity), zglun2da (46% vs. 47% identity; 59% vs. 61% similarity), and zglun2db (47% vs. 44% identity; 61% vs. 56% similarity) (fig. 2 e,f vs fig. 3 a,b). for these putatively benign variants, the greatest extent of conservation tends to occur in the tmd and lbd-tmd linkers, followed by the lbd, and then the ntd and ctd (fig. 3 c,d). for benign variants, however, the decrease in conservation plateaus, with identity and similarity values comparable to the putatively benign variants, if not slightly higher (fig. 3 e-f). the subdomain-specific conservation trend for these variants follows that seen on the level of the whole receptor (fig. 1 ), wherein the tmd, lbd-tmd linkers, and lbd are generally more conserved and the ntd and ctd less so (fig. 3 g,h). although diverging from our prediction as we do not note a substantial decrease in the conservation of benign variants with respect to the putatively benign variants, this result reflects the overall conservation of the nmdar subunits themselves (benign vs total protein): zglun1 (78% vs. ≥84% identity; 84% vs. ≥89% similarity), zglun2aa (62% vs. 68% identity; 86% vs. 89% similarity), zglun2ab (49% vs. 58% identity; 74% vs. 79% similarity), zglun2ba (54% vs. 65% identity; 64% vs. 73% similarity), and zglun2bb (62% vs. 67% identity; 74% vs. 81% similarity), zglun2da (50% vs. 43% identity; 64% vs. 54% similarity), and zglun2db (49% vs. 46% identity; 62% vs. 57% similarity) (fig. 3 e-f, table 1 ). furthermore, this result reflects the relatively large number of benign variants in comparison to other variant groups (table 2 ). critical tmd and lbd-tmd linker components are completely conserved elements forming the ion channel – the tmd (m1, m1-m2, m2, m2-m3, m3, &amp; m4) – and the lbd-tmd linkers (s1-m1, m3-s2, &amp; s2-m4) were consistently highly conserved (fig. 1 ). nevertheless, we assayed conservation of critical functional motifs. the s1-m1 linker is a critical element for channel gating and is completely conserved (fig. s1 ) ( 56 ). the m2 subdomain contains what is known as the ‘n’ and ‘n + 1’ sites, which participate in mg 2+ block and ca 2+ permeation through the channel ( 2 , 55 ) and are conserved in zebrafish (fig. s1 , dark orange). the tmd m3 segment contains the sytanlaaf motif, which plays a key role in ion channel gating and is known to be highly conserved across species. this motif is conserved in all zebrafish glun1 and glun2 paralogs (fig. s1 , orange). lastly, the ‘drpeer’ motif in m3-s2 of glun1, an important component in the high ca 2+ permeability of nmdars ( 52 ) is conserved (fig. s1 , light orange). hence, core gating and permeation elements of the tmd and associated regions are highly conserved suggesting that basic mechanisms of ion channel gating and block/permeation are preserved. lbd elements required for agonist binding are completely conserved the glun1 lbd binds glycine or d-serine while the glun2 lbd binds glutamate or aspartate (fig. s2a,b). overall, the lbds are highly conserved in zebrafish relative to humans (fig. 1 ), as are specific amino acid side chains in s1 and s2 that are involved in binding endogenous agonists ( 47 ) (fig. s2c). these include sites that participate in van-der waals interactions with the α-carbon of the ligand (darkest red), interactions with the α-amino group (dark red), electrostatic interactions with the α-carboxyl group (red), and interactions with the amino acid side chain (light red) (fig. s2c). conservation of structure-function elements in the ntd in terms of overall sequence comparisons, the ntd shows variable degrees of conservation across nmdar subunits. the glun1 zebrafish paralogs maintain the highest degree of conservation relative to human (80% identity; ≥88% similarity) (fig. 1 ). the glun2 subunits show decreased ntd conservation (49–75% identity; 59–88% similarity) (fig. 1 ). given this reduced conservation, we assayed the conservation of key structure-function motifs in zebrafish paralogues. among its many roles, the ntd participates in interactions with ions and molecules that modulate receptor activity ( 58 , 59 ) (fig. 4 a,b). this is most notable for glun2a and to a lesser extent glun2b as they bind zinc, which inhibits receptor function by reducing channel open probability ( 58 ). in glun2a, four amino acids coordinate zinc binding: glun2a-his44, glun2a-his128, glun2a-glu266, and glun2a-asp282 ( 58 ). of these key sites, three are conserved (fig. 4 c) but in zglun2aa and zglun2ab, a key residue involved in zn 2+ binding, glun2a-his44, is not conserved (fig. 4 c). for glun2b, only two amino acid residues participate in zinc binding: glun2b-his127 and glun2b-glu284 ( 58 ). while glun2b-glu284 is conserved, the glun2b-his127 position in zglun2ba is missing due to alterations in ntd sequence length leading to gaps in the alignments, though it is similar now as a positively charged arginine instead of a histidine in zglun2bb (fig. 4 c). the ntd of glun2b also binds polyamines, which enhance receptor activity ( 59 ). nine amino acids are involved in the binding of polyamines. these sites in zglun2ba and zglun2bb are completely conserved except for one amino acid in zglun2bb that is similar but not identical, now a negatively charged aspartate instead of glutamate (fig. 4 d). most of this functional component of glun2b is, thus, conserved in zebrafish. additionally, the nmdar ntd has glycosylation sites, and their glycosylation regulates the trafficking of nmdars to the cell membrane ( 60 ). given the consensus sequence for arginine (n-) glycosylation (n- x -s/t), glun1 has seven, glun2a has three, and glun2b has three putative glycosylation sites in the ntd ( 60 ). their conservation in the zebrafish paralogues is somewhat variable: zglun1a &amp; zglun1b (5 of 7 sites), zglun2aa (3 of 3 sites), zglun2ab (2 of 3 sites), zglun2ba (2 of 3 sites), and zglun2bb (3 of 3 sites) (fig. 4 e). to the best of our knowledge, specific glycosylation sites in glun2d have yet to be identified. of note, the glun1 glycosylation sites glun1-n203 and glun1-n368 are necessary for receptor surface expression ( 60 ), and these sites are conserved in the zebrafish paralogues. while the critical functional features of the lbd were completely conserved in zebrafish relative to human nmdars, the conservation of key regions in the ntd occasionally diverged, most notably that for zn 2+ binding. to further explore the structural composition of the glun1, glun2a, and glun2b ntds, we used swiss-model to generate homology models for the human and zebrafish nmdar subunits, which we subsequently aligned in pymole to assess conservation at the level of the three-dimensional structure (see methods) (fig. 4 f). in general, this analysis revealed a high degree of similarity between the human ntd and zebrafish paralogs. the zebrafish glun1 ntd models display similar overall topology to human: glun1 vs zglun1a, reference structure 5tp0 (rmsd = 0.071); glun1 vs zglun1a, reference structure 5tpz (rmsd = 0.085); glun1 vs zglun1b, reference structure 5tp0 (rmsd = 0.094); and glun1 vs zglun1b, reference structure 5tpz (rmsd = 0.097) (table s3). the glun2a and glun2b subunits also reflect a high degree of structural similarity: glun2a vs zglun2aa, reference structure 5tp0 (rmsd = 0.085); glun2a vs zglun2ab, reference structure 5tp0 (rmsd = 0.095); glun2a vs zglun2ab, reference structure 5tpw (rmsd = 0.111); glun2b vs zglun2ba, reference structure 5tpz (rmsd = 0.028); and glun2b vs glun2bb, reference structure 5tpz (rmsd = 0.044) (table s3). the only exception here is the structural comparison of glun2a and zglun2aa using reference structure 5tpw, in which the ntd is interacting with zinc (rmsd = 2.149), suggesting that the structural components of zglun2aa may not effectively complex with zinc. interestingly, zglun2ba is &gt; 100 amino acids shorter and zglun2bb is &gt; 70 amino acids longer than human glun2b. though the zglun2ba model is missing an α-helix relative to the human protein, the zglun2bb modelled structure aligns well with its human counterpart, suggesting that the zebrafish sequences maintain a large degree of structural integrity. conservation of post-translational modification (ptm) sites in the ctd the intracellular ctd is a key regulatory domain of nmdar cell biology and function and shows distinct subunit-specific elements ( 11 ). for our analyses, we focused on the glun1-1 ctd splice variant, which is the longest variant containing the c0-c1-c2 cassettes ( 11 ). across the nmdar subunits, the zebrafish ctds are the most poorly conserved (fig. 1 ). with a sequence identity generally around 50–60% for the glun1, glun2a, and glun2b zebrafish paralogs, with some exceptions: zglun1a (53% identity; 58% similarity), zglun1b (54% identity; 59% similarity), zglun2aa (57% identity; 86% identity), zglun2ab (38% identity; 65% similarity), zglun2ba (57% identity, 66% similarity), and zglun2bb (60% identity, 71% similarity) while that of glun2d is much lower (≥ 18% identity, ≥ 28% similarity) (fig. 1 ). given these deviations in protein sequences, we first examined the conservation of ctd regions critical for post-translational modifications (ptms), excluding glun2d due its poor conservation in zebrafish. phosphorylation of nmdar subunit ctds confers key functional roles to the receptor ( 2 , 11 ). initially, we examined experimentally validated sites at which phosphorylation occurs by known kinases and calculated the percent conservation, reported in identity and similarity. similarity, here, is defined as a site at the homologous position that can be phosphorylated (e.g., s, t, or y), if not identical. the glun1 ctd contains phosphorylation sites for src kinase (src), protein kinase c (pkc), and protein kinase a (pka), (fig. 5 a, top ) ( 11 ). note specific sites have also been identified that are also targeted by the serine/threonine protein phosphatase 2b (pp2b). these ptm sites are perfectly conserved in zglun1a, while three of five are conserved in zglun1b (60% identity &amp; similarity) (fig. 5 b). the glun2a ctd is phosphorylated by src, pka, dual specificity tyrosine-phosphorylation-regulated kinase 1a (dyrk1), cyclin-dependent kinase 5 (cdk5), pkc, and calcium/calmodulin-dependent protein kinase iiα (camkiiα) (fig. 5 a, middle ) ( 11 ): zglun2aa (79% identity &amp; similarity) and zglun2ab (64% identity, 71% similarity) (fig. 5 b). the ctd of glun2b is phosphorylated by src, camkiiα, cdk5, pka, pkc, death-associated protein kinase 1 (dapk1), proto-oncogene tyrosine-protein kinase fyn (fyn), and casein kinase ii (ck2) (fig. 5 a, bottom ) ( 11 ). glun2b also contains sites for the protein tyrosine phosphatase non-receptor 11 (ptpn11, or shp2) and protein phosphatase 1 (pp1). the conservation of these ptms is slightly higher than that in glun2a: zglun2ba (73% identity &amp; similarity) and zglun2bb (82% identity, 100% similarity) (fig. 5 b). in addition to these kinase-regulated sites, we also examined the conservation of other known phosphorylation sites in these subunits identified by phosphoproteomics (psp) (see methods). the conservation of these sites in the zebrafish paralogs was generally lower than that of the kinase-regulated sites: glun1 (67% identity, 100% similarity), zglun2aa (51% identity, 66% similarity), zglun2ab (34% identity, 57% similarity), zglun2ba (60% identity &amp; similarity), and zglun2bb (67% identity, 70% similarity) (fig. 5 c). we next examined the conservation of ctd palmitoylation and ubiquitylation sites. glun1 has a single ubiquitylation site (lys860) (fig. 6 a, top ) ( 11 ), and it is conserved in the zebrafish paralogs (fig. 6 c). glun2a contains seven palmitoylation sites (fig. 6 a, middle ) ( 11 ), which are likewise completely conserved in zebrafish (fig. 6 b). the glun2a ctd also has four ubiquitylation sites (fig. 6 a, middle) ( 11 ), of which three are conserved in zglun2aa (75%) and one in zglun2ab (25%) (fig. 6 c). with the same approach for glun2b, we identified eight palmitoylation and six ubiquitylation sites (fig. 6 a, bottom ) ( 11 ). here, all palmitoylation sites are conserved in zebrafish while five of six ubiquitylation sites (83%) are conserved in zglun2ba and all six ubiquitylation sites (100%) are conserved in zglun2bb (fig. 5 b,c). in summary, the global conservation of ptm sites is as follows: zglun1a and zglun1b (89% identity, 100% similarity), zglun2aa (62% identity, 72% similarity), zglun2ab (45% identity, 62% similarity), zglun2ba (66% identity, 67% similarity), and zglun2bb (75% identity, 78% similarity). conservation of protein-binding motifs (pbm) in the ctd in addition to numerous ptm sites, a variety of protein-protein interactions are critical to the role of the ctd in receptor cell biology (surface expression, distribution) and function ( 11 , 63 ). these protein-binding elements include either defined short linear motifs (slims) or broader regions known to serve as protein docking sites. the ctd of glun1 contains a slim known as the kkk/rrr er retention motif, which is conserved in zebrafish ( 66 ) (fig. 7 a,b). it, likewise, has binding regions for calmodulin (cam)/calcium/calmodulin-dependent protein kinase ii (camkii) and α-actinin 2 ( 64 , 65 , 67 , 78 ), which is conserved and yotiao, also known as a-kinase anchoring protein 9 (akap9) ( 67 ), which is generally well conserved: zglun1a (90% identity, 98% similarity) and zglun1b (90% identity, 98% similarity) (fig. 8 a,b). the glun2 subunit ctds contain a variety of pbms. for glun2a, the conservation of slims is: the yxxø motif, where y is tyrosine, x is any amino acid, and ø is a bulky hydrophobic residue like leucine, isoleucine, phenylalanine, methionine, or valine ( 73 ) – both zglun2aa and zglun2ab show 100% identity &amp; similarity; the guanine-nucleotide exchange factor brag2 ( 70 ) – zglun2aa (80% identity &amp; similarity) and zglun2ab (40% identity, 60% similarity); postsynaptic density protein 95 (psd-95) ( 72 ) – zglun2aa (50% identity; 63% similarity) and zglun2ab (38% identity, 63% similarity); and the pdz domain that interacts with the proteins psd-95, discs large (dlg), and zonula occludens-1 (zo-1) ( 11 ) – zglun2aa and zglun2ab (100% identity &amp; similarity) (fig. 7 a,c). interacting regions in glun2a include: ring finger protein 10 (rnf10) ( 70 ) – zglun2aa (51% identity, 72% similarity) and zglun2ab (32% identity, 66% similarity); rabphilin 3a (rph3a) ( 68 ) – zglun2aa (71% identity, 80% similarity) and zglun2ab (49% identity, 71% similarity); flotillin-1 (flot-1) ( 71 ) – zglun2aa (61% identity, 81% similarity) and zglun2ab (31% identity, 52% similarity); and c-terminal binding protein 1 (ctbp1) ( 69 ) – zglun2aa (51% identity, 85% similarity) and zglun2ab (36% identity, 63% similarity) (fig. 8 a,c, s3a,b). for glun2b, the conservation of slims is: the guanine-nucleotide exchange factor brag1 ( 76 ) – zglun2ba (60% identity; 80% similarity) and zglun2bb (60% identity, 100% similarity); psd-95 ( 72 ) – zglun2ba (56% identity &amp; similarity) and zglun2bb (33% identity, 56% similarity); camkii and death-associated protein kinase 1 (dapk1) ( 11 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (92% identity, 100% similarity); synapse-associated protein 102 (sap102) ( 75 ) – zglun2ba (0% identity &amp; similarity) and zglun2bb (50% identity &amp; similarity); the adaptor complex ap2 ( 74 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (100% identity &amp; similarity); and pdz ( 11 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (100% identity &amp; similarity) (fig. 7 a,d). the ctd interaction regions for glun2b are conserved as follows: receptor for activated c kinase 1 (rack1) ( 77 ) – zglun2ba (0% identity, 7% similarity) and zglun2bb (13% identity, 20% similarity); flot-1 ( 71 ) – zglun2ba (23% identity; 47% similarity) and zglun2bb (35% identity, 68% similarity); spectrin ( 79 ) – zglun2ba (40% identity; 59% similarity) and zglun2bb (35% identity, 62% similarity); α-actinin 2 ( 78 ) – zglun2ba (47% identity; 67% similarity) and zglun2bb (51% identity, 76% similarity); and ras protein-specific guanine nucleotide-releasing factor 1 (rasgrf1) ( 80 ) – zglun2ba (59% identity; 74% similarity) and zglun2bb (60% identity, 78% similarity) (fig. 8 a,d, s3a,c). when compared globally across subunits, slims always show greater conservation than regions. this conservation is always greatest in the zebrafish glun1 paralogs, followed by the glun2b and then glun2a paralogs. for the analyzed slims, conservation grouped across each subunit is as follows: zglun1a and zglun1b (100% identity &amp; similarity), zglun2aa (76% identity, 81% similarity), zglun2ab (62% identity, 76% similarity), zglun2ba (78% identity, 81% similarity), and zglun2bb (72% identity, 86% similarity). the regions show the following degrees of conservation: zglun1a and zglun1b (94% identity, 98% similarity), zglun2aa (55% identity, 81% similarity), zglun2ab (36% identity, 62% similarity), zglun2ba (59% identity, 74% similarity), and zglun2bb (60% identity, 78% similarity). discussion here, we compared the conservation of the zebrafish nmdar subunit paralogues, including key structure-function motifs, to their human counterparts. we find that the core gating machinery – the lbd, the lbd-tmd linkers and the tmd – are highly conserved both in terms of overall identity and specific functional motifs. the more peripheral domains – the ntd and ctd – show less conservation overall, though functional motifs are reasonably conserved with the notable exception of zn 2+ binding in the glun2a ntd. notably, disease-associated variants are generally conserved, highlighting that zebrafish represent a useful model to study nmdars. nmdar conservation in zebrafish across most of our examined parameters, the obligatory glun1 subunit shows the highest degree of conservation in zebrafish (fig. 1 ). generally, conservation is then greatest in the glun2(a-d) followed by the glun3(a-b) subunits. within each subunit, sequence conservation is consistently highest in the tmd and lbd-tmd linkers, followed by the lbd. conservation is generally poorer in the ntd and even worse in the ctd (fig. 1 ). a key question with regards to this work is defining what constitutes as a ‘well conserved’ sequence. previous studies regarding the conservation of protein structure and function suggest that sequence identities between 40–70% confer functional conservation ( 92 – 94 ). they regard 50% sequence identity as the threshold below which function drastically diverges ( 93 , 94 ). our sequence conservation analyses almost always led to sequence identities &gt; 50%, except for zglun2da. nevertheless, we carried out more detailed examinations of more poorly conserved domains, e.g., the ntd and ctd, to assess the conservation of key structure-function motifs. nmdar missense variant conservation in zebrafish to date, a multitude of variants, including missense, frameshift, and nonsense mutations, have been identified in the genes encoding the nmdar subunits ( 21 , 22 , 87 ). we categorized identified missense variants by pathogenicity and analyzed the extent of conservation of the positions at which they occur. we focused only on the glun1, glun2a, glun2b, and glun2d subunits for this analysis, as this is where most disease-associated variants have been identified. generally, our results demonstrate a higher degree of variant position conservation with increasing pathogenicity (fig. 2 , 3 ). the sites of pathogenic missense variants are almost perfectly conserved (fig. 2 ). given the high conservation of the amino acid positions at which they occur, these protein regions are likely most essential for receptor function. across our categories of variants, with the exclusion of complex variants, the proportion of the total protein at which missense variants have been identified is lowest in the pathogenic group (table 2 ). it is possible that additional variants that might be categorized as pathogenic are so devastating to development as to be embryonic lethal. this reflects the notion that more integral regions of the receptor are less tolerant to variants ( 95 ). zebrafish, nevertheless, emerge as a useful model for the study of pathogenic nmdar variants. putatively pathogenic variants are conserved to a lesser extent than their pathogenic counterparts (fig. 2 ). according to our designations, these variants have the potential to cause disease, having been identified in patients, but are currently classified as having uncertain significance. thus, it aligns that these variants are comparatively less well conserved – they are likely occurring in less critical regions of the receptor and are likewise less likely to cause disease phenotypes. our subsequent investigation of putatively benign and benign variants revealed a further decrease in variant position conservation. this finding further supported our prediction that variants more highly associated with disease occur at positions with higher degrees of conservation and, thus, are more critical to protein structure and function. the conservation of positions with putatively benign and benign variants is quite comparable. for the benign variants, in particular, conservation percentages also closely reflect total protein conservation values for each paralog (fig. 3 , table 1 ). this follows from the finding that positions with identified benign variants, for most subunits, make up a substantial portion of the protein (table 2 ). nevertheless, this investigation both demonstrates the advantage of the zebrafish model in studying nmdar-associated disease and reflects the notion that the positions most essential for receptor function are those that are best conserved and more likely to induce disease when absent. overall, the trend for variant conservation is the same as that for the whole protein. conservation is always greatest in the tmd and lbd-tmd linkers, followed by the lbd, next the ntd, and last the ctd (fig. 2 , 3 ). these findings suggest that zebrafish serve as an effective model for variants, especially pathogenic and putatively pathogenic variants in both the tmd and lbd-tmd linkers. some caution, however, must be exercised when using this model system for variants in the lbd, ntd, and ctd. tmd, tmd-lbd linker, and lbd conservation our examination of key functional regions and motifs in the tmd, lbd-tmd linkers, and lbd consistently revealed complete conservation in their respective zebrafish paralogs (fig. s1 ,s2). this suggests that the basic mechanism of ion channel gating is conserved in zebrafish relative to human. as these domains and subdomains contribute to the formation of the core of the receptor – the ion channel pore allowing for ion flux – it is expected that they would be most highly conserved across species, since the major functional role of nmdars is current flux during synaptic transmission. ntd conservation the ntd is the first region in which nmdar sequences in zebrafish begin to diverge from those in human. modelling ntd variants in zebrafish must, thus, be approached with caution. for example, polyamine-binding sites in glun2b are almost completely conserved in zebrafish (fig. 4 d), so they have the potential to serve as an effective model for this functional role of nmdars. on the other hand, zinc-binding sites in glun2a and glun2b are more poorly conserved in zebrafish (fig. 4 c), so they would likely serve as a poor model for zinc-related studies in nmdars. this also aligns with our homology model finding that the zglun2aa model aligns well with glun2a when not bound to zinc, but the rmsd of the alignment is substantially poorer when glun2a interacting with zinc is used as the reference structure (table s3). ctd conservation the c-terminal domains (ctds) exhibit the lowest level of overall conservation across nmdar domains (fig. 1 ). this is consistent with the ctd’s classification as an intrinsically disordered region (idr), which lacks fixed secondary or tertiary structure and is more tolerant to evolutionary divergence ( 96 , 97 ). nonetheless, functional conservation within idrs is often maintained through the retention of post-translational modification (ptm) hotspots and short linear motifs (slims) that regulate protein interactions and intracellular signaling ( 98 ). we assessed ctd conservation in glun1, glun2a, and glun2b by analyzing experimentally validated phosphorylation, palmitoylation, and ubiquitylation sites, as well as defined protein-binding motifs. phosphorylation sites with known kinase interactions exhibited relatively high conservation, consistent with their well-characterized regulatory roles (figs. 5 a,b). in contrast, phosphorylation sites without identified kinases were less consistently retained (fig. 5 c). palmitoylation sites were fully conserved across all subunits, while ubiquitylation sites were generally retained, though conservation was notably lower in zglun2ab (fig. 5 f) and may reflect paralogue-specific divergence in degradation or trafficking pathways. protein-binding motif conservation varied by subunit. glun1 retained nearly all canonical interaction motifs, while glun2a and glun2b displayed region-specific conservation. notably well-conserved elements in the glun2 subunits included the pdz-binding motifs as well as docking regions for camkii, dapk1, and ap2 (fig. 6 ). slims overall showed higher conservation, likely due to their short, sequence-specific structure, which may place them under stronger evolutionary constraint than broader, more flexible interaction regions. despite broad sequence divergence, nmdar ctds retain a core set of conserved motifs involved in receptor trafficking, synaptic localization, and plasticity. kinase-regulated phosphorylation sites, palmitoylation sites, and pdz-binding motifs were consistently preserved. conservation of glun1 sites was always greatest in the zebrafish paralogs. among the glun2 subunits, glun2b displayed the highest overall conservation of functional elements, aligning with its essential role in neurodevelopment and synaptic signaling. these findings are especially relevant to the study of disease-associated truncating variants in the nmdar subunits. nonsense variants that introduce premature stop codons in the ctd eliminate key regulatory motifs critical for trafficking, anchoring, or signaling. when these affected regions correspond to conserved functional elements, zebrafish may serve as an appropriate in vitro and in vivo model for studying ctd-mediated disease mechanisms. conversely, for mutations in poorly conserved regions, the translational utility of this model may become limited. conclusions zebrafish represent a valuable system to study nmdar-associated neurodevelopmental disorders, but one must be judicious in choosing which variants to study in this model. our unique approach of sequence alignments per individual nmdar domains and subdomains highlights the high degree of conversation of some regions of the protein relative to others. additionally, our analysis of identified variants indicates that zebrafish are an effective model for pathogenic variants as these occur at conserved locations along the protein. generally, the utility of zebrafish is most apparent for the study of the tmd, lbd-tmd linkers, and lbd as these regions are most well conserved. on the other hand, studying the ntd and ctd in this species requires extra consideration stemming from deviations in sequence conservation. our work will enable future nmdar-related studies that can be effectively and efficiently conducted in zebrafish. abbreviations nmdar n -methyl-d-aspartate receptor ntd n-terminal domain lbd ligand-binding domain tmd transmembrane domain ctd c-terminal domain declarations ethics approval and consent to participate: not applicable. consent for publication: not applicable. availability of data and materials: the datasets used and/or analyzed during the current study are available either at osf (https://osf.io/h2zf9/) or from the corresponding author on reasonable request. competing interests: no competing interests declared. funding: this work was supported by nih grant r01ns088479 to lpw. authors’ contributions: e.r.n., c.a., j.d.z., h.i.s., and l.p.w. contributed to the conceptualization and methodology of this project. e.r.n., c.a., and j.d.z. conducted the investigation and analysis. w.r. assisted with data curation. e.r.n, c.a., and l.p.w. wrote the manuscript, which was revised with insights from j.d.z. and h.i.s. acknowledgements: the authors thank the clinvar and gnomad databases and the swiss-model server for their important contributions to this work. references paoletti p, bellone c, zhou q. nmda receptor subunit diversity: impact on receptor properties, synaptic plasticity and disease. nat rev neurosci. 2013;14(6):383-400. hansen kb, wollmuth lp, bowie d, furukawa h, menniti fs, sobolevsky ai, et al. structure, function, and pharmacology of glutamate receptor ion channels. pharmacol rev. 2021;73(4):298-487. chakraborty a, murphy s, coleman n. the role of nmda receptors in neural stem cell proliferation and differentiation. stem cells dev. 2017;26(11):798-807. nacher j, mcewen bs. the role of n-methyl-d-asparate receptors in neurogenesis. hippocampus. 2006;16(3):267-70. komuro h, rakic p. modulation of neuronal migration by nmda receptors. science. 1993;260(5104):95-7. pearce ia, cambray-deakin ma, burgoyne rd. glutamate acting on nmda receptors stimulates neurite outgrowth from cerebellar granule cells. febs lett. 1987;223(1):143-7. booker sa, wyllie dja. nmda receptor function in inhibitory neurons. neuropharmacology. 2021;196:108609. gambrill ac, barria a. nmda receptor subunit composition controls synaptogenesis and synapse stabilization. proc natl acad sci u s a. 2011;108(14):5855-60. napoli aj, laderwager s, zoodsma jd, biju b, mucollari o, schubel sk, et al. developmental loss of nmda receptors results in supernumerary forebrain neurons through delayed maturation of transit-amplifying neuroblasts. sci rep. 2024;14(1):3395. rajani v, sengar as, salter mw. tripartite signalling by nmda receptors. mol brain. 2020;13(1):23. ishchenko y, carrizales mg, koleske aj. regulation of the nmda receptor by its cytoplasmic domains: (how) is the tail wagging the dog? neuropharmacology. 2021;195:108634. mony l, paoletti p. mechanisms of nmda receptor regulation. curr opin neurobiol. 2023;83:102815. nielsen ac, anderson cl, ens c, boyce akj, thompson rj. non-ionotropic nmdar signalling activates panx1 to induce p2x4r-dependent long-term depression in the hippocampus. j physiol. 2024. barnes sa, thomazeau a, finnie psb, heinrich mj, heynen aj, komiyama nh, et al. non-ionotropic signaling through the nmda receptor glun2b carboxy-terminal domain drives dendritic spine plasticity and reverses fragile x phenotypes. cell rep. 2025;44(3):115311. platzer k, lemke jr. grin1-related neurodevelopmental disorder. genereviews((r)). 2021. strehlow v, myers ka, morgan at, scheffer ie, lemke jr. grin2a-related disorders. genereviews((r)). 2024. platzer k, lemke jr. grin2b-related neurodevelopmental disorder. genereviews((r)). 2021. platzer k, krey i, lemke jr. grin2d-related developmental and epileptic encephalopathy. genereviews((r)). 2022. zehavi y, mandel h, zehavi a, rashid ma, straussberg r, jabur b, et al. de novo grin1 mutations: an emerging cause of severe early infantile encephalopathy. eur j med genet. 2017;60(6):317-20. sabo sl, lahr jm, offer m, weekes a, sceniak mp. grin2b-related neurodevelopmental disorder: current understanding of pathophysiological mechanisms. front synaptic neurosci. 2022;14:1090865. gjerulfsen ce, krey i, klockner c, rubboli g, lemke jr, moller rs. spectrum of nmda receptor variants in neurodevelopmental disorders and epilepsy. methods mol biol. 2024;2799:1-11. amin jb, moody gr, wollmuth lp. from bedside-to-bench: what disease-associated variants are teaching us about the nmda receptor. j physiol. 2021;599(2):397-416. hu c, chen w, myers sj, yuan h, traynelis sf. human grin2b variants in neurodevelopmental disorders. j pharmacol sci. 2016;132(2):115-21. cox ja, kucenas s, voigt mm. molecular characterization and embryonic expression of the family of n-methyl-d-aspartate receptor subunit genes in the zebrafish. dev dyn. 2005;234(3):756-66. veldman mb, lin s. zebrafish as a developmental model organism for pediatric research. pediatr res. 2008;64(5):470-6. choi ty, choi ti, lee yr, choe sk, kim ch. zebrafish as an animal model for biomedical research. exp mol med. 2021;53(3):310-7. zoodsma jd, chan k, bhandiwad aa, golann dr, liu g, syed sa, et al. a model to study nmda receptors in early nervous system development. j neurosci. 2020;40(18):3631-45. zoodsma jd, keegan ej, moody gr, bhandiwad aa, napoli aj, burgess ha, et al. disruption of grin2b, an asd-associated gene, produces social deficits in zebrafish. mol autism. 2022;13(1):38. baraban sc, taylor mr, castro pa, baier h. pentylenetetrazole induced changes in zebrafish behavior, neural activity and c-fos expression. neuroscience. 2005;131(3):759-68. shams s, rihel j, ortiz jg, gerlai r. the zebrafish as a promising tool for modeling human brain disorders: a review based upon an ibns symposium. neurosci biobehav rev. 2018;85:176-90. de abreu ms, genario r, giacomini a, demin ka, lakstygal am, amstislavskaya tg, et al. zebrafish as a model of neurodevelopmental disorders. neuroscience. 2020;445:3-11. weinschutz mendes h, neelakantan u, liu y, fitzpatrick se, chen t, wu w, et al. high-throughput functional analysis of autism genes in zebrafish identifies convergence in dopaminergic and neuroimmune pathways. cell rep. 2023;42(3):112243. howe k, clark md, torroja cf, torrance j, berthelot c, muffato m, et al. the zebrafish reference genome sequence and its relationship to the human genome. nature. 2013;496(7446):498-503. rihel j, schier af. behavioral screening for neuroactive drugs in zebrafish. dev neurobiol. 2012;72(3):373-85. baraban sc, dinday mt, hortopan ga. drug screening in scn1a zebrafish mutant identifies clemizole as a potential dravet syndrome treatment. nat commun. 2013;4:2410. zhang y, kecskes a, copmans d, langlois m, crawford ad, ceulemans b, et al. pharmacological characterization of an antisense knockdown zebrafish model of dravet syndrome: inhibition of epileptic seizures by the serotonin agonist fenfluramine. plos one. 2015;10(5):e0125898. dingemans ajm, jansen s, van reeuwijk j, de leeuw n, pfundt r, schuurs-hoeijmakers j, et al. prevalence of comorbidities in individuals with neurodevelopmental disorders from the aggregated phenomics data of 51,227 pediatric individuals. nat med. 2024;30(7):1994-2003. ohtsuki t, sakurai k, dou h, toru m, yamakawa-kobayashi k, arinami t. mutation analysis of the nmdar2b (grin2b) gene in schizophrenia. mol psychiatry. 2001;6(2):211-6. arnold pd, rosenberg dr, mundo e, tharmalingam s, kennedy jl, richter ma. association of a glutamate (nmda) subunit receptor gene (grin2b) with obsessive-compulsive disorder: a preliminary study. psychopharmacology (berl). 2004;174(4):530-8. saunders ja, tatard-leitman vm, suh j, billingslea en, roberts tp, siegel sj. knockout of nmda receptors in parvalbumin interneurons recreates autism-like phenotypes. autism res. 2013;6(2):69-77. gandal mj, sisti j, klook k, ortinski pi, leitman v, liang y, et al. gabab-mediated rescue of altered excitatory-inhibitory balance, gamma synchrony and behavioral deficits following constitutive nmdar-hypofunction. transl psychiatry. 2012;2(7):e142. barnby g, abbott a, sykes n, morris a, weeks de, mott r, et al. candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at grin2a and abat. am j hum genet. 2005;76(6):950-66. hung ly, margolis kg. autism spectrum disorders and the gastrointestinal tract: insights into mechanisms and clinical relevance. nat rev gastroenterol hepatol. 2024;21(3):142-63. owens ap, mathias cj, iodice v. autonomic dysfunction in autism spectrum disorder. front integr neurosci. 2021;15:787037. orefice ll. peripheral somatosensory neuron dysfunction: emerging roles in autism spectrum disorders. neuroscience. 2020;445:120-9. makris g, agorastos a, chrousos gp, pervanidou p. stress system activation in children and adolescents with autism spectrum disorder. front neurosci. 2021;15:756628. ramos-vicente d, grant sg, bayes a. metazoan evolution and diversity of glutamate receptors and their auxiliary subunits. neuropharmacology. 2021;195:108640. amores a, force a, yan yl, joly l, amemiya c, fritz a, et al. zebrafish hox clusters and vertebrate genome evolution. science. 1998;282(5394):1711-4. chou th, tajima n, romero-hernandez a, furukawa h. structural basis of functional transitions in mammalian nmda receptors. cell. 2020;182(2):357-71 e13. zhang y, ye f, zhang t, lv s, zhou l, du d, et al. structural basis of ketamine action on human nmda receptors. nature. 2021;596(7871):301-5. wollmuth lp, sakmann b. different mechanisms of ca2+ transport in nmda and ca2+-permeable ampa glutamate receptor channels. j gen physiol. 1998;112(5):623-36. watanabe j, beck c, kuner t, premkumar ls, wollmuth lp. drpeer: a motif in the extracellular vestibule conferring high ca2+ flux rates in nmda receptor channels. j neurosci. 2002;22(23):10209-16. wollmuth lp, sobolevsky ai. structure and gating of the glutamate receptor ion channel. trends neurosci. 2004;27(6):321-8. ogden kk, khatri a, traynelis sf, heldt sa. potentiation of glun2c/d nmda receptor subtypes in the amygdala facilitates the retention of fear and extinction learning in mice. neuropsychopharmacology. 2014;39(3):625-37. wollmuth lp. ion permeation in ionotropic glutamate receptors: still dynamic after all these years. curr opin physiol. 2018;2:36-41. mcdaniel mj, ogden kk, kell sa, burger pb, liotta dc, traynelis sf. nmda receptor channel gating control by the pre-m1 helix. j gen physiol. 2020;152(4). wollmuth lp, kuner t, sakmann b. adjacent asparagines in the nr2-subunit of the nmda receptor channel control the voltage-dependent block by extracellular mg2+. j physiol. 1998;506 ( pt 1)(pt 1):13-32. romero-hernandez a, simorowski n, karakas e, furukawa h. molecular basis for subtype specificity and high-affinity zinc inhibition in the glun1-glun2a nmda receptor amino-terminal domain. neuron. 2016;92(6):1324-36. mony l, zhu s, carvalho s, paoletti p. molecular basis of positive allosteric modulation of glun2b nmda receptors by polyamines. embo j. 2011;30(15):3134-46. lichnerova k, kaniakova m, park sp, skrenkova k, wang yx, petralia rs, et al. two n-glycosylation sites in the glun1 subunit are essential for releasing n-methyl-d-aspartate (nmda) receptors from the endoplasmic reticulum. j biol chem. 2015;290(30):18379-90. waterhouse a, bertoni m, bienert s, studer g, tauriello g, gumienny r, et al. swiss-model: homology modelling of protein structures and complexes. nucleic acids res. 2018;46(w1):w296-w303. bienert s, waterhouse a, de beer ta, tauriello g, studer g, bordoli l, et al. the swiss-model repository-new features and functionality. nucleic acids res. 2017;45(d1):d313-d9. warnet xl, bakke krog h, sevillano-quispe og, poulsen h, kjaergaard m. the c-terminal domains of the nmda receptor: how intrinsically disordered tails affect signalling, plasticity and disease. eur j neurosci. 2021;54(8):6713-39. kim k, saneyoshi t, hosokawa t, okamoto k, hayashi y. interplay of enzymatic and structural functions of camkii in long-term potentiation. j neurochem. 2016;139(6):959-72. bej a, ames jb. chemical shift assignments of calmodulin bound to a cytosolic domain of glun2a (residues 1004-1024) from the nmda receptor. biomol nmr assign. 2023;17(1):89-93. horak m, wenthold rj. different roles of c-terminal cassettes in the trafficking of full-length nr1 subunits to the cell surface. j biol chem. 2009;284(15):9683-91. lin jw, wyszynski m, madhavan r, sealock r, kim ju, sheng m. yotiao, a novel protein of neuromuscular junction and brain that interacts with specific splice variants of nmda receptor subunit nr1. j neurosci. 1998;18(6):2017-27. stanic j, carta m, eberini i, pelucchi s, marcello e, genazzani aa, et al. rabphilin 3a retains nmda receptors at synaptic sites through interaction with glun2a/psd-95 complex. nat commun. 2015;6:10181. cousins sl, stephenson fa. identification of c-terminal binding protein 1 as a novel nmda receptor interactor. neurochem res. 2019;44(6):1437-45. franchini l, carrano n, di luca m, gardoni f. synaptic glun2a-containing nmda receptors: from physiology to pathological synaptic plasticity. int j mol sci. 2020;21(4). swanwick cc, shapiro me, yi z, chang k, wenthold rj. nmda receptors interact with flotillin-1 and -2, lipid raft-associated proteins. febs lett. 2009;583(8):1226-30. cousins sl, stephenson fa. identification of n-methyl-d-aspartic acid (nmda) receptor subtype-specific binding sites that mediate direct interactions with scaffold protein psd-95. j biol chem. 2012;287(16):13465-76. vissel b, krupp jj, heinemann sf, westbrook gl. a use-dependent tyrosine dephosphorylation of nmda receptors is independent of ion flux. nat neurosci. 2001;4(6):587-96. lavezzari g, mccallum j, lee r, roche kw. differential binding of the ap-2 adaptor complex and psd-95 to the c-terminus of the nmda receptor subunit nr2b regulates surface expression. neuropharmacology. 2003;45(6):729-37. chen bs, gray ja, sanz-clemente a, wei z, thomas ev, nicoll ra, et al. sap102 mediates synaptic clearance of nmda receptors. cell rep. 2012;2(5):1120-8. elagabani mn, brisevac d, kintscher m, pohle j, kohr g, schmitz d, et al. subunit-selective n-methyl-d-aspartate (nmda) receptor signaling through brefeldin a-resistant arf guanine nucleotide exchange factors brag1 and brag2 during synapse maturation. j biol chem. 2016;291(17):9105-18. yaka r, thornton c, vagts aj, phamluong k, bonci a, ron d. nmda receptor function is regulated by the inhibitory scaffolding protein, rack1. proc natl acad sci u s a. 2002;99(8):5710-5. wyszynski m, lin j, rao a, nigh e, beggs ah, craig am, et al. competitive binding of alpha-actinin and calmodulin to the nmda receptor. nature. 1997;385(6615):439-42. wechsler a, teichberg vi. brain spectrin binding to the nmda receptor is regulated by phosphorylation, calcium and calmodulin. embo j. 1998;17(14):3931-9. krapivinsky g, krapivinsky l, manasian y, ivanov a, tyzio r, pellegrino c, et al. the nmda receptor is coupled to the erk pathway by a direct interaction between nr2b and rasgrf1. neuron. 2003;40(4):775-84. mukherjee p, roy s, ghosh d, nandi sk. role of animal models in biomedical research: a review. lab anim res. 2022;38(1):18. chou th, epstein m, fritzemeier rg, akins ns, paladugu s, ullman ez, et al. molecular mechanism of ligand gating and opening of nmda receptor. nature. 2024;632(8023):209-17. armstrong n, sun y, chen gq, gouaux e. structure of a glutamate-receptor ligand-binding core in complex with kainate. nature. 1998;395(6705):913-7. armstrong n, gouaux e. mechanisms for activation and antagonism of an ampa-sensitive glutamate receptor: crystal structures of the glur2 ligand binding core. neuron. 2000;28(1):165-81. kazi r, gan q, talukder i, markowitz m, salussolia cl, wollmuth lp. asynchronous movements prior to pore opening in nmda receptors. j neurosci. 2013;33(29):12052-66. amin jb, gochman a, he m, certain n, wollmuth lp. nmda receptors require multiple pre-opening gating steps for efficient synaptic activity. neuron. 2021;109(3):488-501 e4. hardingham ge, do kq. linking early-life nmdar hypofunction and oxidative stress in schizophrenia pathogenesis. nat rev neurosci. 2016;17(2):125-34. landrum mj, lee jm, riley gr, jang w, rubinstein ws, church dm, et al. clinvar: public archive of relationships among sequence variation and human phenotype. nucleic acids res. 2014;42(database issue):d980-5. landrum mj, lee jm, benson m, brown g, chao c, chitipiralla s, et al. clinvar: public archive of interpretations of clinically relevant variants. nucleic acids res. 2016;44(d1):d862-8. landrum mj, lee jm, benson m, brown gr, chao c, chitipiralla s, et al. clinvar: improving access to variant interpretations and supporting evidence. nucleic acids res. 2018;46(d1):d1062-d7. gudmundsson s, singer-berk m, watts na, phu w, goodrich jk, solomonson m, et al. variant interpretation using population databases: lessons from gnomad. hum mutat. 2022;43(8):1012-30. tian w, skolnick j. how well is enzyme function conserved as a function of pairwise sequence identity? j mol biol. 2003;333(4):863-82. addou s, rentzsch r, lee d, orengo ca. domain-based and family-specific sequence identity thresholds increase the levels of reliable protein function transfer. j mol biol. 2009;387(2):416-30. sangar v, blankenberg dj, altman n, lesk am. quantitative sequence-function relationships in proteins based on gene ontology. bmc bioinformatics. 2007;8:294. perszyk re, kristensen as, lyuboslavsky p, traynelis sf. three-dimensional missense tolerance ratio analysis. genome res. 2021;31(8):1447-61. uversky vn. intrinsically disordered proteins from a to z. int j biochem cell biol. 2011;43(8):1090-103. singleton md, eisen mb. evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. plos comput biol. 2024;20(4):e1012028. davey ne, cowan jl, shields dc, gibson tj, coldwell mj, edwards rj. slimprints: conservation-based discovery of functional motif fingerprints in intrinsically disordered protein regions. nucleic acids res. 2012;40(21):10628-41. additional declarations no competing interests reported. supplementary files zfishnmdarssupplement20250716.docx cite share download pdf status: under review version 1 posted editorial decision: revision requested 24 sep, 2025 reviews received at journal 19 sep, 2025 reviewers agreed at journal 19 sep, 2025 reviews received at journal 09 sep, 2025 reviews received at journal 18 aug, 2025 reviewers agreed at journal 04 aug, 2025 reviewers agreed at journal 28 jul, 2025 reviewers invited by journal 28 jul, 2025 editor assigned by journal 28 jul, 2025 editor invited by journal 23 jul, 2025 submission checks completed at journal 22 jul, 2025 first submitted to journal 22 jul, 2025 you are reading this latest preprint version research square lets you share your work early, gain feedback from the community, and start making changes to your manuscript prior to peer review in a journal. as a division of research square company, we’re committed to making research communication faster, fairer, and more useful. we do this by developing innovative software and high quality services for the global research community. our growing team is made up of researchers and industry professionals working together to solve the most critical problems facing scientific publishing. also discoverable on platform about our team in review editorial policies advisory board help center resources author services research quality evaluation accessibility api access rss feed manage cookie preferences © research square 2025 | issn 2693-5015 (online) privacy policy terms of service do not sell my personal information {"props":{"pageprops":{"initialdata":{"identity":"rs-7151578","acceptedtermsandconditions":true,"allowdirectsubmit":false,"archivedversions":[],"articletype":"research article","associatedpublications":[],"authors":[{"id":492782335,"identity":"6084adf8-9103-4ca8-8de5-679d2a4ea2bb","order_by":0,"name":"erica r. nebet","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"erica","middlename":"r.","lastname":"nebet","suffix":""},{"id":492782336,"identity":"68eb1705-4eac-413e-bc82-7b9943220d12","order_by":1,"name":"christieann aprea","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"christieann","middlename":"","lastname":"aprea","suffix":""},{"id":492782337,"identity":"776ee37e-7ff9-4a8a-bc43-f965303e90a8","order_by":2,"name":"josiah d. zoodsma","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"josiah","middlename":"d.","lastname":"zoodsma","suffix":""},{"id":492782338,"identity":"ef7f492d-83d8-41a3-aa81-6be2af423f07","order_by":3,"name":"william raab","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"william","middlename":"","lastname":"raab","suffix":""},{"id":492782339,"identity":"6949754f-6430-4ffc-8673-da5548d44bd6","order_by":4,"name":"howard i. sirotkin","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"howard","middlename":"i.","lastname":"sirotkin","suffix":""},{"id":492782340,"identity":"a4f8ecb1-612e-43db-8923-d325a2be05c2","order_by":5,"name":"lonnie p. wollmuth","email":"data:image/png;base64,ivborw0kggoaaaansuheugaaazaaaaayaqmaaabi0h/eaaaablbmvex///8aaabvwtn+aaaacxbiwxmaaa7eaaaoxagvkw4baaabhuleqvriiwnggawjybacphcraclymrua5ae5sdajim0dxwiada6fgaqs4yaxwiigfgyklyknblww9x578kccibgfmbf1m++po+nz5zcfe8y7454cp88bxgdv2zc18jxln0g4w5a4s5mx7tzpwrpcdcfy0o15zxqbs/y2mbvoxajfisdmirgnixfdybcww7kb2hjmphnbehi3ngdgazjwapmh0jiu34bqwv4bp5yghjyehmmwlwcb2jixaee5yyarceyihusxm3psdhtuojawjjn3tikxzm/bzsk55zc08lp3men+qlex5mdvpnbjjc1hefnmw8ck3u5iaizy8sepb8owqxkcjfc5tjdoamcokydxb/fqr8eogawjypgdabgnaum7/svdaaaaaelftksuqmcc","orcid":"","institution":"stony brook university","correspondingauthor":true,"prefix":"","firstname":"lonnie","middlename":"p.","lastname":"wollmuth","suffix":""}],"badges":[],"createdat":"2025-07-17 18:08:21","currentversioncode":1,"declarations":"","doi":"10.21203/rs.3.rs-7151578/v1","doiurl":"https://doi.org/10.21203/rs.3.rs-7151578/v1","draftversion":[],"editorialevents":[],"editorialnote":"","failedworkflow":false,"files":[{"id":87954191,"identity":"93406acd-9b5c-4bbe-b8e4-476bfc8e3591","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":0,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":48663694,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarsmain20250722.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/72a01d08b07202a8dd3432a7.docx"},{"id":87954161,"identity":"09dcee36-3c3d-4bae-a5b6-a8031a04e2f8","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"json","order_by":1,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":8698,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b.json","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/088093afe258318de15a5f12.json"},{"id":87954169,"identity":"c1d3acb0-9e25-4b8a-9228-e0e154b314b2","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":2,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":15353369,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarssupplement20250716.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a3a65bcc549c7954c295d2d1.docx"},{"id":87954163,"identity":"6fde5d88-4da9-48b5-ac47-0b0e0f6354d8","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"xml","order_by":3,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":242943,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b1enriched.xml","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/aad420877d9070a2aedac4cc.xml"},{"id":87954167,"identity":"b5c71071-9fd7-476b-9303-17dcb5b1c845","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"jpeg","order_by":4,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":7224546,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage1.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/239b3c6e0e27c5d5f3a3c485.jpeg"},{"id":87954176,"identity":"66a11160-0f4e-44e0-a6b3-26a6c526f9c1","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":5,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5495318,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage2.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/7c759429dc2c4ccf2d95e1ed.jpeg"},{"id":87954168,"identity":"e3a84d67-d0ce-4822-8529-9e1cc04acd3d","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":6,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5497046,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage3.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/5ab1e35184bc289007b868d7.jpeg"},{"id":87954408,"identity":"4452df13-a563-4564-9a2c-9a250a285ebb","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"jpeg","order_by":7,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4536836,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage4.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/34cad84600342cebbb343c8d.jpeg"},{"id":87954171,"identity":"94757fe0-c77c-46b6-8a13-76b5920bc170","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":8,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4702238,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage5.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/704be5ce4c0cc10b6f98b684.jpeg"},{"id":87954173,"identity":"bbb91acc-360b-43aa-99a5-89fbc5989777","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":9,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4378230,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage6.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/85acc94c54964dbcd4ee6caa.jpeg"},{"id":87954192,"identity":"2447fad7-ffab-4e33-9b2f-a71d2af30f9e","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":10,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5735470,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage7.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3a0654f7ccd9aa40cc5d5f69.jpeg"},{"id":87954413,"identity":"29bec889-0025-4009-9267-6438ea9c5740","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"jpeg","order_by":11,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5327046,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage8.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/c3009b0732aa42082efe8df3.jpeg"},{"id":87954188,"identity":"20ba216a-d094-494f-9894-1937509c8d81","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":12,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":222456,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage1.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/08aba5e904980f68ab32639e.png"},{"id":87954187,"identity":"d99856fc-69f6-4dda-9eeb-4cafd6cc99a0","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":13,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":175317,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage2.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a0ea7afa8616702c9ac66b10.png"},{"id":87954174,"identity":"de47bd92-e161-4fbe-a9b2-437c4b5f0573","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":14,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":140021,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage3.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/bb8abfaa1c199dae266df4dd.png"},{"id":87954412,"identity":"87f7cc6b-e10f-4300-9539-2c2a02855ddb","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"png","order_by":15,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":138717,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage4.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a6b8e16a616611723ae39dab.png"},{"id":87954190,"identity":"ad2ebfe9-5bbc-4128-8d93-eaa9d7470a57","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":16,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":96612,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage5.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/99939544fae92f971073fd51.png"},{"id":87954189,"identity":"84923e46-044f-4cd2-9f5a-b3a9fa41caab","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":17,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":72313,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage6.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/92848d9ff37daaad2b48e9ff.png"},{"id":87954755,"identity":"eaa7ae44-7e94-4117-97cb-d03c1d07f90a","added_by":"auto","created_at":"2025-07-30 18:41:44","extension":"png","order_by":18,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":83342,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage7.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/845b774d666648bfa0a844ab.png"},{"id":87954410,"identity":"13827b86-cf06-4cf3-9952-4b424318037f","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"png","order_by":19,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":117211,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage8.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/af95a59a499464e286bef54b.png"},{"id":87954411,"identity":"7741fc55-f9dc-44b6-9b97-8875d99f5f4b","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"xml","order_by":20,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":240176,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b1structuring.xml","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/dfb8648302675ebf344a2fa6.xml"},{"id":87954756,"identity":"beb85617-8838-4c99-8712-1a7d62a23eff","added_by":"auto","created_at":"2025-07-30 18:41:44","extension":"html","order_by":21,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":257708,"visible":true,"origin":"","legend":"","description":"","filename":"earlyproof.html","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/01f35be0d8432473e2775041.html"},{"id":87954160,"identity":"3920a076-09de-49c0-b764-c081048b3b14","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":1,"title":"figure 1","display":"","copyasset":false,"role":"figure","size":795350,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of human nmdar subunits in zebrafish\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a)\u003c/strong\u003e nmdars are composed of two obligatory glun1 and typically two glun2a-d subunits. the tetrameric complex consists of four domains: extracellular n-terminal (ntd) and ligand-binding (lbd) domains; a transmembrane domain (tmd); and an intracellular c-terminal domain (ctd). glun1 subunits are colored dark blue (ntd), teal (lbd), red (lbd-tmd linkers), light orange (tmd), and gray (ctd); and glun2b subunits gray. 9are (82)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b)\u003c/strong\u003e topology of an individual glun1 subunit. lbd lobes (d1 and d2) are shown; d1 is mainly comprisedof s1 and d2 of s2. the ion channel is formed by 3 transmembrane segments, m1, m3, and m4, and an intracellular m2 pore loop. lbd-tmd linkers are s1-m1 (or d2-m1), m3-s2 (or d2-m3), and s2-m4 (or d1-m4).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c)\u003c/strong\u003e nmdar subunits are separated into 13 structural subdivisions: ntd, lbd (s1 \u0026amp; s2), lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4), tmd (m1, intracellular m1-m2 linker, m2 loop, m2-m3 linker, m3, \u0026amp; m4), and ctd. amino acid length of each zebrafish subdivision is indicated. those sharing amino acid length with their human orthologues are indicated in white, while increasingly darker pink indicates those with increasingly greater amino acid differences (greater (+) or fewer (-)).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(d-e)\u003c/strong\u003e nmdar subunits are grouped as in \u003cstrong\u003e(c)\u003c/strong\u003e. homology between human and zebrafish nmdar subunit paralogues is reported in amino acid identity \u003cstrong\u003e(d)\u003c/strong\u003e or similarity \u003cstrong\u003e(e)\u003c/strong\u003e. heat map with darker green indicating greater percent of homology.\u003c/p\u003e","description":"","filename":"floatimage1.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/5b4e27c91809ed8be6a2b3b5.png"},{"id":87954407,"identity":"47d4fb26-35bf-49ca-aae6-ed196317e2be","added_by":"auto","created_at":"2025-07-30 18:33:43","extension":"png","order_by":2,"title":"figure 2","display":"","copyasset":false,"role":"figure","size":712960,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of positions with pathogenic and putatively pathogenic missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a-b) \u003c/strong\u003epercent conservation of pathogenic variant positions in zebrafish genes, as defined in table 2. conservation is reported in amino acid identity \u003cstrong\u003e(a) \u003c/strong\u003eand similarity \u003cstrong\u003e(b)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c-d)\u003c/strong\u003e conservation of pathogenic variant positions in each nmdar domain, grouped as in figure 1. homology is reported in identity \u003cstrong\u003e(c)\u003c/strong\u003eand similarity \u003cstrong\u003e(d)\u003c/strong\u003e. dashes indicate domains without reported pathogenic variants. along with the percent of conservation, the total number of identified variants in the human protein is indicated in parentheses in the ‘a’ paralog column in \u003cstrong\u003e(c)\u003c/strong\u003e.\u003c/p\u003e","description":"","filename":"floatimage2.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/d7d962b025e722b029324c59.png"},{"id":87954162,"identity":"02953530-369f-4594-b587-6d86079d6a68","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":3,"title":"figure 3","display":"","copyasset":false,"role":"figure","size":667858,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of positions with putatively benign and benign missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a-b) \u003c/strong\u003epercent conservation of putatively benign variant positions in zebrafish genes, as defined in table 2. conservation is reported in identity \u003cstrong\u003e(a) \u003c/strong\u003eand similarity \u003cstrong\u003e(b)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c-d) \u003c/strong\u003econservation of putatively benign variant positions in each nmdar domain, grouped as in figure 2. homology is reported in identity \u003cstrong\u003e(c)\u003c/strong\u003e and similarity \u003cstrong\u003e(d)\u003c/strong\u003e. dashes indicate domains without reported pathogenic variants. along with the percent of conservation, the total number of identified variants in the human protein is indicated in parentheses in the ‘a’ paralog column in \u003cstrong\u003e(c)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(e-h) \u003c/strong\u003eas in a-d but for benign variant positions.\u003c/p\u003e","description":"","filename":"floatimage3.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/069549024abcf2bb76a43e0e.png"},{"id":87954166,"identity":"f30749e8-12a9-42b8-89f5-26186b255ecb","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":4,"title":"figure 4","display":"","copyasset":false,"role":"figure","size":473605,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key functional elements and structure of the nmdar ntd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003etopology of glun2a ntd, showing r1 and r2 subdomains, bound to zinc (red). 5tpw (58)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b) \u003c/strong\u003eclose-up image of structure in \u003cstrong\u003e(a)\u003c/strong\u003ehighlighting amino acids that participate in zinc binding. his44 is demarcated as it is not conserved in the zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c) \u003c/strong\u003econservation of amino acids that participate in ntd zinc binding in glun2a and glun2b, reported in identity (dark red) and similarity (light red). sequences are shown to the right.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(d) \u003c/strong\u003econservation of glun2b ntd regions that participate in interactions with polyamines, reported in identity (dark gray) and similarity (light gray). sequences are shown to the right.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(e) \u003c/strong\u003econservation of ntd glycosylation sites, reported in identity (dark gray).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(f) \u003c/strong\u003ealigned ntd homology models of the human (blue) and zebrafish (gray) protein for glun1 vs zglun1a (reference structure, 5tp0) (\u003cem\u003eleft\u003c/em\u003e), glun2a vs zglun2aa (reference structure, 5tp0) (\u003cem\u003emiddle\u003c/em\u003e), and glun2b vs zglun2bb (reference structure, 5tpz) (\u003cem\u003eright\u003c/em\u003e). secondary structures colored both blue and gray indicate areas of structural overlap.\u003c/p\u003e","description":"","filename":"floatimage4.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3693c16c36a6ea0f931e6343.png"},{"id":87954406,"identity":"16a86e66-9b13-469f-91ca-fe2142d4ce9c","added_by":"auto","created_at":"2025-07-30 18:33:43","extension":"png","order_by":5,"title":"figure 5","display":"","copyasset":false,"role":"figure","size":427432,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key nmdar ctd phosphorylation sites\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd kinase-regulated phosphorylation sites for the glun1, glun2a and glun2b ctds. the specific phosphorylation sites and the corresponding human and zebrafish sequences are indicated. positions sharing identity are indicated in dark gray. similarity is shown in light gray; here, a residue is only considered similar if it is ‘phosphorylatable’ (e.g., contains a s, t, or y) (see methods).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-c) \u003c/strong\u003epercent conservation of kinase-regulated phosphorylation sites \u003cstrong\u003e(b)\u003c/strong\u003e and other phosphorylation sites identified by phosphoproteomics \u003cstrong\u003e(c)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray); similarity is defined in \u003cstrong\u003e(a)\u003c/strong\u003e.\u003c/p\u003e","description":"","filename":"floatimage5.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/25001bb8418306120cd3e005.png"},{"id":87954184,"identity":"a64d1f54-7d76-472b-a396-d9efcf5f86c8","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":6,"title":"figure 6","display":"","copyasset":false,"role":"figure","size":333594,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key nmdar ctd post-translational modification (ptm) sites\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd palmitoylation and ubiquitylation sites for the glun1, glun2a and glun2b ctds. the specific phosphorylation sites and the corresponding human and zebrafish sequences are indicated. positions sharing identity are indicated in dark gray.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-c) \u003c/strong\u003epercent conservation of palmitoylation \u003cstrong\u003e(b)\u003c/strong\u003e and ubiquitylation \u003cstrong\u003e(c)\u003c/strong\u003e sites, reported in identity (dark gray).\u003c/p\u003e","description":"","filename":"floatimage6.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/fac7b19510f29b178f7e3661.png"},{"id":87954182,"identity":"d09e842a-ccf2-41a7-bd3e-0935363761f9","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":7,"title":"figure 7","display":"","copyasset":false,"role":"figure","size":403546,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of nmdar short linear motif (slim) protein-binding motifs (pbms) in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd short linear motifs (slims) for the glun1, glun2a and glun2b ctds.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-d) \u003c/strong\u003epercent conservation of ctd slims in glun1 \u003cstrong\u003e(b)\u003c/strong\u003e, glun2a \u003cstrong\u003e(c)\u003c/strong\u003e, and glun2b \u003cstrong\u003e(d)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray).\u003c/p\u003e","description":"","filename":"floatimage7.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/b6119c0825c4063a5e6bb6ec.png"},{"id":87954181,"identity":"cb3e5a1b-cd98-4155-84bb-c4d2fbb40f7e","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":8,"title":"figure 8","display":"","copyasset":false,"role":"figure","size":518323,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of nmdar protein-binding motifs (pbms) regions in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd regions for the glun1, glun2a and glun2b ctds.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-d) \u003c/strong\u003epercent conservation of ctd regions in glun1 \u003cstrong\u003e(b)\u003c/strong\u003e, glun2a \u003cstrong\u003e(c)\u003c/strong\u003e, and glun2b \u003cstrong\u003e(d)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray).\u003c/p\u003e","description":"","filename":"floatimage8.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/ccea59247fac75e14ad8ad14.png"},{"id":87954866,"identity":"ae52deed-9d7c-4d30-94e3-0b4ac04fe145","added_by":"auto","created_at":"2025-07-30 18:49:46","extension":"pdf","order_by":0,"title":"","display":"","copyasset":false,"role":"manuscript-pdf","size":5866105,"visible":true,"origin":"","legend":"","description":"","filename":"manuscript.pdf","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/24ce2fd9-cdac-44be-947d-44a6c3564886.pdf"},{"id":87954175,"identity":"cf7d81ad-b94d-4e4f-95f1-e45dee844a82","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":0,"title":"","display":"","copyasset":false,"role":"supplement","size":15353369,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarssupplement20250716.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3ada87e108301cd5d027f600.docx"}],"financialinterests":"no competing interests reported.","formattedtitle":"conservation of human nmda receptor subunits and disease variants in zebrafish","fulltext":[{"header":"background","content":"\u003cp\u003erapid cell-to-cell signaling in the nervous system occurs at specialized structures called synapses. at these sites, the electrical activity of the upstream or presynaptic neuron is converted into the release of a chemical neurotransmitter. this is then converted back into an electrical signal in the postsynaptic neuron by specialized ion channels that are gated by a neurotransmitter, initiating downstream biological signaling cascades. glutamate, the primary excitatory neurotransmitter in the brain, activates a variety of receptors that are essential for synaptic transmission and plasticity. glutamate-gated ion channels play fundamental roles in signaling in both the developing and mature nervous systems (\u003cspan additionalcitationids=\"cr2 cr3 cr4 cr5 cr6 cr7 cr8\" citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e–\u003cspan citationid=\"cr9\" class=\"citationref\"\u003e9\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ethe nmda receptor (nmdar) is a prominent glutamate-gated ion channel subtype (\u003cspan citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e, \u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e). due to its distinct biophysical properties and unique ionotropic and non-ionotropic signaling mechanisms, it contributes to a wide array of nervous and non-nervous system functions (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan additionalcitationids=\"cr11 cr12 cr13\" citationid=\"cr10\" class=\"citationref\"\u003e10\u003c/span\u003e–\u003cspan citationid=\"cr14\" class=\"citationref\"\u003e14\u003c/span\u003e). numerous missense, frameshift, and nonsense mutations (or variants) have been identified in the genes that encode nmdar subunits and are associated with neurodevelopmental disorders, including autism spectrum disorders (asd), intellectual disability, developmental delay, seizures, and schizophrenia (\u003cspan additionalcitationids=\"cr16 cr17 cr18 cr19 cr20 cr21 cr22\" citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e–\u003cspan citationid=\"cr23\" class=\"citationref\"\u003e23\u003c/span\u003e). however, how these variants lead to clinical phenotypes is poorly understood.\u003c/p\u003e\u003cp\u003ezebrafish are a powerful model organism to study neurodevelopmental phenotypes and disorders, including those linked to nmdars (\u003cspan citationid=\"cr19\" class=\"citationref\"\u003e19\u003c/span\u003e, \u003cspan additionalcitationids=\"cr25 cr26 cr27 cr28 cr29 cr30 cr31\" citationid=\"cr24\" class=\"citationref\"\u003e24\u003c/span\u003e–\u003cspan citationid=\"cr32\" class=\"citationref\"\u003e32\u003c/span\u003e). zebrafish develop rapidly and share fundamental developmental and signaling pathways with humans and other vertebrates (\u003cspan citationid=\"cr33\" class=\"citationref\"\u003e33\u003c/span\u003e); they are transparent, allowing for dynamic and non-invasive observations of developmental and cellular processes; and they display robust behaviors that are easily assayed at early larval stages. moreover, larval zebrafish readily take up small molecules from their environment, enabling non-invasive and high-throughput readout of nervous system function in the context of behavioral drug screens (\u003cspan additionalcitationids=\"cr35\" citationid=\"cr34\" class=\"citationref\"\u003e34\u003c/span\u003e–\u003cspan citationid=\"cr36\" class=\"citationref\"\u003e36\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enmdars are heterotetramers composed of two obligatory glun1 (encoded by \u003cem\u003egrin1\u003c/em\u003e) subunits. they are expressed as either diheterotetramers with two glun1 and two glun2 (\u003cem\u003egrin2a-d\u003c/em\u003e) subunits or triheterotetramers of some combination of glun2 and/or glun3 (\u003cem\u003egrin3a-b\u003c/em\u003e) subunits (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e) (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003ea). these nmdar subunits are differentially expressed over the course of neurodevelopment, with glun2a, glun2b, and glun2d being highly expressed during early development, while glun2c, glun3a, and glun3b expression increases into adulthood (\u003cspan citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e). moreover, nmdars display distinct functional and pharmacological properties based on their glun2 or glun3 composition (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e). each subunit consists of four principal domains: an n-terminal domain (ntd), a ligand-binding domain (lbd), a transmembrane domain (tmd) that forms the ion channel, and a c-terminal domain (ctd) (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003eb).\u003c/p\u003e\u003cp\u003e\u003c/p\u003e\u003cp\u003edisease-associated variants (davs) in \u003cem\u003egrin1\u003c/em\u003e, \u003cem\u003egrin2a\u003c/em\u003e, \u003cem\u003egrin2b\u003c/em\u003e, and \u003cem\u003egrin2d\u003c/em\u003e have been identified in pediatric patients with neurodevelopmental disorders (\u003cspan additionalcitationids=\"cr16 cr17\" citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e–\u003cspan citationid=\"cr18\" class=\"citationref\"\u003e18\u003c/span\u003e, \u003cspan citationid=\"cr21\" class=\"citationref\"\u003e21\u003c/span\u003e). these disorders often present with multiple comorbidities, reflecting overlapping neurodevelopmental phenotypes (\u003cspan citationid=\"cr37\" class=\"citationref\"\u003e37\u003c/span\u003e). generally, patients with davs in \u003cem\u003egrin1\u003c/em\u003e present with developmental delay and intellectual disability, though a large proportion of patients also experience seizures (\u003cspan citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e). in contrast, \u003cem\u003egrin2a\u003c/em\u003e and \u003cem\u003egrin2d\u003c/em\u003e are predominantly associated with seizure disorders (\u003cspan citationid=\"cr16\" class=\"citationref\"\u003e16\u003c/span\u003e, \u003cspan citationid=\"cr18\" class=\"citationref\"\u003e18\u003c/span\u003e). \u003cem\u003egrin2b\u003c/em\u003e is a high-risk gene for autism spectrum disorder (asd) (\u003cspan citationid=\"cr17\" class=\"citationref\"\u003e17\u003c/span\u003e, \u003cspan citationid=\"cr38\" class=\"citationref\"\u003e38\u003c/span\u003e, \u003cspan citationid=\"cr39\" class=\"citationref\"\u003e39\u003c/span\u003e), as are \u003cem\u003egrin1\u003c/em\u003e and \u003cem\u003egrin2a\u003c/em\u003e (\u003cspan additionalcitationids=\"cr41\" citationid=\"cr40\" class=\"citationref\"\u003e40\u003c/span\u003e–\u003cspan citationid=\"cr42\" class=\"citationref\"\u003e42\u003c/span\u003e). still, patients with such neurodevelopmental disorders generally experience a variety of both central and peripheral nervous system symptoms (\u003cspan additionalcitationids=\"cr44 cr45\" citationid=\"cr43\" class=\"citationref\"\u003e43\u003c/span\u003e–\u003cspan citationid=\"cr46\" class=\"citationref\"\u003e46\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enmda receptors are highly conserved across species (\u003cspan citationid=\"cr47\" class=\"citationref\"\u003e47\u003c/span\u003e). due to an ancient genome duplication event, zebrafish retain two paralogues for most nmdar genes (e.g., \u003cem\u003egrin1a\u003c/em\u003e \u0026amp; \u003cem\u003egrin1b\u003c/em\u003e) (\u003cspan citationid=\"cr24\" class=\"citationref\"\u003e24\u003c/span\u003e, \u003cspan citationid=\"cr48\" class=\"citationref\"\u003e48\u003c/span\u003e). patch clamp electrophysiology experiments show that zebrafish paralogues display similar functional properties to each other and to their mammalian orthologues (\u003cspan citationid=\"cr27\" class=\"citationref\"\u003e27\u003c/span\u003e, \u003cspan citationid=\"cr28\" class=\"citationref\"\u003e28\u003c/span\u003e). nevertheless, it remains unclear how well the human nmdar genes, including positions with identified missense variants, are conserved in zebrafish. to address this, we investigated nmdar conservation in zebrafish by taking advantage of amino acid sequence alignments (\u003cspan citationid=\"cr28\" class=\"citationref\"\u003e28\u003c/span\u003e) and assessed the conservation of all human nmdar subunits and zebrafish paralogues. we also compared the conservation of positions with missense variants identified in human patients for nmdar subunits expressed during early neurodevelopment. finally, we investigated the conservation of critical functional regions within each structural domain. through this analysis, we define the extent to which critical functional components and regions, with relevance to disease, are conserved across zebrafish nmdars, providing a template to use zebrafish as a model to study nmdars in neurodevelopment and disease pathology.\u003c/p\u003e"},{"header":"methods","content":"\u003cp\u003e\u003cb\u003enmdar sequences and sequence alignments\u003c/b\u003e\u003c/p\u003e\u003cp\u003eamino acid sequences for human genes are (uniprot with protein ids): glun1 (q05586-5; ntd splice isoform, nr1-3), glun2a (q12879-1), glun2b (q13224), glun2c (q14957), glun2d (o15399), glun3a (q8tcu5), and glun3b (o60391).\u003c/p\u003e\u003cp\u003efor mouse subunits: mglun1 (p35438), mglun2a (p35436), mglun2b (q01097), mglun2c (q01098), mglun2d (q03391), mglun2a (a2air5), and mglun3b (q91zu9).\u003c/p\u003e\u003cp\u003efor zebrafish paralogs: zglun1a (f1r366), zglun1b (q6zm67), zglun2aa (a0a8m9pxd6), zglun2ab (f1qde5), zglun2ba (a0a8m9q0i1), zglun2bb (a0a8m1nh89), zglun2ca (e7fh62), zglun2cb (a0a8m3b7k5), zglun2da (i3ni77), zglun2db (e7f3z4), zglun3a (a0a8m3awh5), zglun3ba (a0a8m9p9f6), and zglun3bb (a0a8m6z151).\u003c/p\u003e\u003cp\u003ebreakpoints for nmdar subunit domains and subdomains are based on structures 6whr (rat glun1-glun2b) and 7eu7 (human glun1-glun2a) (\u003cspan citationid=\"cr49\" class=\"citationref\"\u003e49\u003c/span\u003e, \u003cspan citationid=\"cr50\" class=\"citationref\"\u003e50\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ehuman and zebrafish protein sequences were aligned and analyzed for amino acid identity and similarity using align sequences protein blast (blastp), which is publicly available through the national center for biotechnology information (ncbi) or emboss needle pairwise sequence alignment (psa). for each nmdar subunit, the zebrafish paralogs were individually aligned to the human protein. amino acid similarity is defined based on the blosum 62 substitution matrix. for the ntd, lbd, and tmd, these alignments were generally straightforward as the sequences do not diverge greatly in length. in contrast, the ctd is quite large, with the zebrafish sequences generally larger than human. for these alignments, we entered the entire sequence into the alignment tool and used the given result, though this led to a fragmentation of the ctd such that certain portions of the sequence were aligned while others were not. we used the resulting domain-specific conservation values to calculate the conservation of the total protein.\u003c/p\u003e\u003cp\u003e\u003cb\u003evariants analysis\u003c/b\u003e\u003c/p\u003e\u003cp\u003ehuman nmdar variants were identified using clinvar and gnomad (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003eclinvar\u003c/em\u003e. results for each gene were downloaded on september 28th, 2023, using clinvar's bulk download option (\u003cspan class=\"externalref\"\u003e\u003cspan class=\"refsource\"\u003ehttps://www.ncbi.nlm.nih.gov/clinvar/\u003c/span\u003e\u003cspan address=\"https://www.ncbi.nlm.nih.gov/clinvar/\" targettype=\"url\" class=\"reftarget\"\u003e\u003c/span\u003e\u003c/span\u003e). genomic variants that did not result in protein coding changes were removed from the analysis. in the case of \u003cem\u003egrin3a\u003c/em\u003e, protein coding changes in the ppp3r2 gene, a single exon gene whose coding sequence resides within the genomic region of \u003cem\u003egrin3a\u003c/em\u003e and is downloaded alongside \u003cem\u003egrin3a\u003c/em\u003e due to this, was also removed. in cases where multiple protein coding annotations exist for the same variant, only the annotation fitting the canonical transcript was retained.\u003c/p\u003e\u003cp\u003e\u003cem\u003egnomad\u003c/em\u003e: genes were searched for by name on gnomad v4.0.0 (\u003cspan class=\"externalref\"\u003e\u003cspan class=\"refsource\"\u003ehttps://gnomad.broadinstitute.org/\u003c/span\u003e\u003cspan address=\"https://gnomad.broadinstitute.org/\" targettype=\"url\" class=\"reftarget\"\u003e\u003c/span\u003e\u003c/span\u003e) and missense, insertion, deletion, and loss-of-function variant information was exported on september 28th, 2023 as comma separated value (csv) files for each gene. hgvs annotation (protein change) was converted to letter-number-letter annotation.\u003c/p\u003e\u003cp\u003egene variants from clinvar and gnomad were merged using r studio 2023.06.1 build 524 based on the letter-number-letter annotation. the resulting merged file was checked for redundant information or variant duplicates, which were removed.\u003c/p\u003e\u003cp\u003emissense variants were extracted and grouped by pathogenicity as pathogenic, putatively pathogenic, complex, putatively benign, and benign variants based on the criteria in table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e. assessment of conservation of positions with identified human variants was performed using the generated sequence alignments.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar domain-specific analysis\u003c/b\u003e\u003c/p\u003e\u003cp\u003eto further refine the relationship between human and zebrafish nmdar proteins, we analyzed the tmd, lbd-tmd linkers, lbd, ntd, and ctd of glun1, glun2a, glun2b, and glun2d for conservation of key structural and functional features.\u003c/p\u003e\u003cp\u003e\u003cem\u003etransmembrane domain (tmd) and ligand-binding domain-transmembrane domain (lbd-tmd) linkers.\u003c/em\u003e key regions involved in channel gating, mg\u003csup\u003e2+\u003c/sup\u003e block, and ca\u003csup\u003e2+\u003c/sup\u003e flux were identified as described in the literature and analyzed for conservation (\u003cspan additionalcitationids=\"cr52 cr53 cr54 cr55 cr56\" citationid=\"cr51\" class=\"citationref\"\u003e51\u003c/span\u003e–\u003cspan citationid=\"cr57\" class=\"citationref\"\u003e57\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003eligand-binding domain (lbd)\u003c/em\u003e. key regions and specific amino acids involved in van-der-waals interactions with the α-carbon of the ligand, interactions with the α-amino group, electrostatic interactions with the α-carboxyl group, and interactions with the amino acid side chain were identified as described in ramos-vicente et. al. 2021 and analyzed for conservation (\u003cspan citationid=\"cr47\" class=\"citationref\"\u003e47\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003en-terminal domain (ntd)\u003c/em\u003e. key amino acid positions required for the binding of zinc and polyamines were identified via a literature search and their conservation assessed (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr58\" class=\"citationref\"\u003e58\u003c/span\u003e, \u003cspan citationid=\"cr59\" class=\"citationref\"\u003e59\u003c/span\u003e). we also examined the conservation of putative n-linked glycosylation sites (\u003cspan citationid=\"cr60\" class=\"citationref\"\u003e60\u003c/span\u003e). to generate homology models of the zebrafish ntds, we used the template function in swiss-model (\u003cspan citationid=\"cr61\" class=\"citationref\"\u003e61\u003c/span\u003e, \u003cspan citationid=\"cr62\" class=\"citationref\"\u003e62\u003c/span\u003e). as reference structures, we used rodent ntd structures 5tq0 (glun1-glun2a in the presence of edta, no bound zn\u003csup\u003e2+\u003c/sup\u003e for glun1 and glun2a), 5tpz (glun1-glun2b apo state for glun1 and glun2b), and 5tpw (glun1-glun2a with zn\u003csup\u003e2+\u003c/sup\u003e in complex with glun2a) (\u003cspan citationid=\"cr58\" class=\"citationref\"\u003e58\u003c/span\u003e). sequences for human (hglun1, hglun2a, and hglun2b) or zebrafish (zglun1a, zglun1b, glun2aa, glun2ab, glun2ba, glun2bb) ntds were entered individually into swiss-model to generate homology models. the subsequent models were then loaded into pymole (human and zebrafish paralog) and aligned. the resultant root mean square deviation (rmsd) is reported.\u003c/p\u003e\u003cp\u003e\u003cem\u003ecarboxy-terminal domain (ctd)\u003c/em\u003e. ctd-specific conservation in zebrafish was assessed for the glun1, glun2a, and glun2b subunits. glun2d was excluded from this analysis due to the poor conservation of the ctd in these paralogs relative to human. to assess functional relevance, we focused on post-translational modifications (ptms), specifically phosphorylation sites (serine, threonine, tyrosine), ubiquitylation sites (lysine), and palmitoylation sites (cysteine), using a combination of experimentally validated data from phosphositeplus (psp) and published literature (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e). phosphorylation sites with identified kinases or well-characterized functional roles were classified as ‘kinase-regulated phosphorylation sites.’ the remaining phosphorylation sites identified in psp were defined as ‘phosphoproteomic-identified sites.’ note, sites in psp (based on rodents) that did not have a phosphorylation site in humans were excluded from our analysis. in addition, known linear protein-binding motifs were annotated and analyzed for conservation (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e). to assess conservation of protein-binding motifs in the ctds, we categorized sites as either short linear motifs (slims) or broader interaction ‘regions’. slims were defined as experimentally validated, compact sequence elements (typically \u0026lt; 15 residues) that mediate specific and direct interactions with known protein partners. broader regions were defined as extended sequences (\u0026gt; 15 residues) shown to directly bind proteins in biochemical or structural studies but lacking a discrete consensus sequence. all motifs were curated from published literature where discrete sequences were experimentally validated per motif (\u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e, \u003cspan additionalcitationids=\"cr64 cr65 cr66 cr67 cr68 cr69 cr70 cr71 cr72 cr73 cr74 cr75 cr76 cr77 cr78 cr79\" citationid=\"cr63\" class=\"citationref\"\u003e63\u003c/span\u003e–\u003cspan citationid=\"cr80\" class=\"citationref\"\u003e80\u003c/span\u003e), and sequence conservation was examined.\u003c/p\u003e"},{"header":"results","content":"\u003cp\u003eour goal is to compare the structural similarity of zebrafish nmdar (zglun) paralogues to human nmdars (glun), especially regarding human variants. since mice are a common animal model (\u003cspan class=\"citationref\"\u003e81\u003c/span\u003e), as a reference-point for nmdar gene conservation, we initially made global comparisons of both mouse and zebrafish nmdar subunits to human.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eglobal comparison of mouse and zebrafish orthologues to human nmdars\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003enmdars are heterotetramers composed of two obligate glycine-binding glun1 subunits and two glutamate-binding glun2 subunits or a combination of glun2 and glycine-binding glun3 subunits (\u003cspan class=\"citationref\"\u003e1\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e2\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ea,b). glycine and glutamate act as co-agonists for glun2-containing receptors, with binding of both required for receptor activation, while glun3-containing receptors are activated by glycine binding. as an initial assessment of the structural similarity between nmdar orthologues, we made global sequence alignments using the ncbi blastp or emboss psa tools. not surprisingly, given evolutionary proximity, the mouse subunits are highly conserved relative to human: glun1 (97% identity \u0026amp; similarity), glun2a (95% identity; 98% similarity), glun2b (98% identity; 99% similarity), glun2c (88% identity; 90% similarity), glun2d (97% identity \u0026amp; similarity), glun3a (92% identity; 95% similarity), and glun3b (77% identity; 83% similarity) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003eon the other hand, the zebrafish paralogues show reduced conservation. the obligatory glun1 subunits are the most highly conserved, both in terms of identity (\u0026ge;\u0026thinsp;84%) and similarity (\u0026ge;\u0026thinsp;89%) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the other subunits show various degrees of identity and similarity: glun2a (\u0026ge;\u0026thinsp;58% identity; \u0026ge;79% similarity), glun2b (\u0026ge;\u0026thinsp;65% identity; \u0026ge;73% similarity), glun2c (\u0026ge;\u0026thinsp;50% identity; \u0026ge;64% similarity), glun2d (\u0026ge;\u0026thinsp;43% identity; \u0026ge;54% similarity), glun3a (63% identity; 78% similarity), and glun3b (\u0026ge;\u0026thinsp;54% identity, \u0026ge;\u0026thinsp;62% similarity) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). these subunits, thus, retain\u0026thinsp;\u0026ge;\u0026thinsp;50% identity with respect to the human protein, except for zglun2da and zglun2db, which only share 43% and 46% sequence identity, respectively (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). with respect to the zebrafish paralog pairs, there are instances in which one paralog is better conserved than the other. the glun1 paralogs are comparably conserved (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). however, the zglun2aa paralog is more highly conserved than zglun2ab while the zglun2bb shows increased conservation relative to zglun2ba (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the glun2c, glun2d, and glun3b paralogs are conserved to similar extents (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003enevertheless, despite lower conservation relative to mouse, zebrafish offer many technical advantages (see background). in addition, what is critical is the conservation of specific structural and functional domains and subdomains.\u003c/p\u003e\n\u003cdiv align=\"left\" class=\"colspec\"\u003e\u003cbr\u003e\u003c/div\u003e\n\u003ctable id=\"tab1\" border=\"1\"\u003e\n \u003ccaption language=\"en\"\u003e\n \u003cdiv class=\"captionnumber\"\u003etable 1\u003c/div\u003e\n \u003cdiv class=\"captioncontent\"\u003e\n \u003cp\u003eoverall protein conservation in mouse and zebrafish relative to human\u003c/p\u003e\n \u003c/div\u003e\n \u003c/caption\u003e\n \u003cthead\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003ehuman\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003emouse\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003ezebrafish\u003c/p\u003e\n \u003cp\u003eparalogue a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003ezebrafish\u003c/p\u003e\n \u003cp\u003eparalogue b\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003c/thead\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en1\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e938\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e938\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e84%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e89%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e966\u003c/p\u003e\n \u003cp\u003e(-28)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e86%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e90%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e937\u003c/p\u003e\n \u003cp\u003e(-1)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2a\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1464\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e95%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e98%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1464\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e68%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e89%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1460\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;6)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e58%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e79%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1445\u003c/p\u003e\n \u003cp\u003e(-9)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2b\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1484\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e98%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e99%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1482\u003c/p\u003e\n \u003cp\u003e(-2)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e65%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e73%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1417\u003c/p\u003e\n \u003cp\u003e(-31)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e67%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e81%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1770 (+\u0026thinsp;322)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2c\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1233\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e88%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e90%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1239\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;6)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e50%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e64%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1400\u003c/p\u003e\n \u003cp\u003e(-176)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e59%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e79%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1328\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;95)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2d\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1336\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1323\u003c/p\u003e\n \u003cp\u003e(-13)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e43%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e54%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1916 (+\u0026thinsp;580)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e46%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e57%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1676 (+\u0026thinsp;340)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en3a\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1115\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e92%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e95%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1135\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;20)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e63%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e78%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1111\u003c/p\u003e\n \u003cp\u003e(-4)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en3b\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1043\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e77%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e83%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1003\u003c/p\u003e\n \u003cp\u003e(-40)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e54%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e62%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1118\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;75)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e62%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e68%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1114\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;71)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003c/p\u003e\n\u003cp\u003enumbers indicate the percent of mouse or zebrafish paralog conservation with respect to the human protein \u0026ndash; \u0026lsquo;id.\u0026rsquo; is identity and \u0026lsquo;sim.\u0026rsquo; is similarity. the zebrafish a and b paralogs are indicated for each protein. glun3a has only a single paralog in zebrafish. for the mouse and zebrafish protein length, numbers in parentheses indicate the difference in amino acid number relative to human.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of specific structural elements in zebrafish paralogues\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003efor more rigorous comparisons, we examined the conservation within individual subunit domains and subdomains (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the structure of nmdars, like all iglurs, consists of 4 domains that are intrinsic to each individual subunit (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ea,b): the extracellular ntd and lbd; the tmd forming the ion channel as well as the linkers that connect the lbd to the tmd, the lbd-tmd linkers; and the intracellular ctd (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e53\u003c/span\u003e). within individual subunits, the lbd is formed by two disparate sequences, s1 and s2, that fold up into the d1 and d2 lobes in the 3-dimensional structure (83, 84). in general, d1 is made up primarily of s1 and d2 of s2, but there is considerable swapping between lobes. the tmd is formed by three transmembrane segments, m1, m3, and m4, and an m2-pore loop. the lbd-tmd linkers are referred to as either s1-m1, m3-s2, and s2-m4 (linear sequence) or d2-m1, m3-d2, and d1-m4 (3-dimensional structure). this distinction is important functionally since d1 and d2 undergo different movements that are translated differently to the tmd (\u003cspan class=\"citationref\"\u003e85\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e86\u003c/span\u003e). nevertheless, we will use the s1-m1, m3-s2, and s2-m4 designation for ease of comparison to structures.\u003c/p\u003e\n\u003cp\u003ewe began by dividing the zebrafish paralogues into their respective domains and subdomains based on our breakpoints (see methods). interestingly, the amino acid sequences surrounding these breakpoints were well conserved in zebrafish \u0026ndash; the majority were identical (not shown). this allowed for ease of division of the zebrafish proteins into their respective domains and subdomains.\u003c/p\u003e\n\u003cp\u003eas an initial assessment of the structural similarity between human and zebrafish nmdars, we made detailed sequence alignments using the ncbi blastp or emboss psa tools (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e, fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). in terms of length of domains, the zebrafish protein sequence for glun1, glun2, and glun3 subunits was identical to the human sequence for the tmd (m1, m1-m2, m2, m2-m3, m3, \u0026amp; m4) and lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec). the lbd (s1 \u0026amp; s2) was also quite similar in length, with no changes in the length of s2 across the paralogs and only slight increases or decreases in amino acid number in the s1 of zglun1a, zglun1b, zglun2ab, zglun2ca, zglun2ca, zglun2da, and zglun2db (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec). the ntd and ctd were less conserved in terms of size, wherein almost every zebrafish paralog has a different number of amino acids compared to the human. this is especially notable for the glun2b ntd which either lacks 130 amino acids (zglun2ba) or has an additional 74 amino acids (zglun2bb) as well as for the ctd where, for example, zglun2da and zglun2db have a ctd that is more than twice as long as that in the human protein (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec).\u003c/p\u003e\n\u003ctable id=\"tab2\" border=\"1\"\u003e\n \u003ccaption language=\"en\"\u003e\n \u003cdiv class=\"captionnumber\"\u003etable 2\u003c/div\u003e\n \u003cdiv class=\"captioncontent\"\u003e\n \u003cp\u003evariant designation\u003c/p\u003e\n \u003c/div\u003e\n \u003c/caption\u003e\n \u003cthead\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eglun\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003edesignation\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eclinvar\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003egnomad\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e1\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2b\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2c\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2d\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e3a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e3b\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003c/thead\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epathogenic\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;pathogenic\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e49\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e58\u003c/p\u003e\n \u003cp\u003e(4%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e81\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e12\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eputatively\u003c/p\u003e\n \u003cp\u003epathogenic\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;uncertain\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e123\u003c/p\u003e\n \u003cp\u003e(13%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e214\u003c/p\u003e\n \u003cp\u003e(15%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e182\u003c/p\u003e\n \u003cp\u003e(12%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e4\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e111\u003c/p\u003e\n \u003cp\u003e(8%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e5\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e8\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003ecomplex\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;pathogenic\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e3\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e5\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e6\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eputatively benign\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;uncertain\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e80\u003c/p\u003e\n \u003cp\u003e(9%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e320\u003c/p\u003e\n \u003cp\u003e(22%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e199\u003c/p\u003e\n \u003cp\u003e(13%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e31\u003c/p\u003e\n \u003cp\u003e(3%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e223\u003c/p\u003e\n \u003cp\u003e(17%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e30\u003c/p\u003e\n \u003cp\u003e(3%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e68\u003c/p\u003e\n \u003cp\u003e(7%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003ebenign\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;benign\u0026rdquo;\u003c/p\u003e\n \u003cp\u003eor\u003c/p\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent/\u003c/p\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003cp\u003eor\u003c/p\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e401\u003c/p\u003e\n \u003cp\u003e(43%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e813\u003c/p\u003e\n \u003cp\u003e(55%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e672\u003c/p\u003e\n \u003cp\u003e(45%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e945\u003c/p\u003e\n \u003cp\u003e(77%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e863\u003c/p\u003e\n \u003cp\u003e(27%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e59\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e945\u003c/p\u003e\n \u003cp\u003e(91%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003c/p\u003e\n\u003cp\u003enumber of positions with missense variants that have been identified in each subunit per designation. percentages report the proportion of the total protein amino acids at which such variants have been identified.\u003c/p\u003e\n\u003cp\u003ewe used these sequence alignments to assess the degree of amino acid identity and similarity between domains and subdomains (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ed,e). the obligatory glun1 subunit showed the highest degree of conservation. the tmd and lbd-tmd linkers are 100% conserved. the lbd is also well conserved with s1 and s2 showing\u0026thinsp;\u0026ge;\u0026thinsp;92% identity and \u0026ge;\u0026thinsp;93% similarity. the ntd is somewhat less conserved (\u0026ge;\u0026thinsp;80% identity, \u0026ge;\u0026thinsp;88% similarity), as is the ctd (\u0026ge;\u0026thinsp;53% identity; \u0026ge;58% similarity).\u003c/p\u003e\n\u003cp\u003ethis trend of conservation across nmdar subunits \u0026ndash; the lbd and tmd are most highly conserved and the ntd and ctd less so \u0026ndash; is maintained in the glun2 and glun3 subunits (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ed,e). generally, the glun2 subunits maintain a higher sequence identity than the glun3 subunits. overall, this sequence analysis suggests that the lbd and tmd are highly conserved whereas the ntd and ctd show less conservation.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecategorizing the pathogenicity of missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003enmdars with missense, frameshift, and nonsense variants have been identified in pediatric patients with neurodevelopmental disorders (\u003cspan class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e22\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e87\u003c/span\u003e). we therefore wanted to assess the extent to which variants are conserved in zebrafish to aid in guiding future studies. for this analysis, we focused solely on missense variants. while nonsense and frameshift mutation can be modelled in various ways, such as by mutating a portion of the protein to induce a loss-of-function, the conservation of the specific protein sequence is somewhat less relevant. in contrast, the single amino acid change associated with missense variants is critical to dissecting how such variants alter receptor function and lead to disease phenotypes.\u003c/p\u003e\n\u003cp\u003eusing publicly available datasets of nmdar variants from clinvar and gnomad, we identified missense variants in all nmdar subunits (see methods). we first categorized these variants with regards to their pathogenicity (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). clinvar reports variants that have been identified in patients with symptomatic disease (\u003cspan class=\"citationref\"\u003e88\u003c/span\u003e\u0026ndash;\u003cspan class=\"citationref\"\u003e90\u003c/span\u003e), while gnomad reports variants identified in the healthy population (\u003cspan class=\"citationref\"\u003e91\u003c/span\u003e). thus, in general, we define variants present in clinvar but absent in gnomad as pathogenic while those with the opposite reporting as benign. with this approach, pathogenic and putatively pathogenic variants were identified in glun1, glun2a, glun2b, and glun2d while minimal to no such variants were identified in glun2c, glun3a, and glun3b (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). notably, these genes with known pathogenic variants are expressed early in development, contributing to developmental processes while their dysfunction often leads to disease phenotypes (\u003cspan class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e87\u003c/span\u003e). on the other hand, glun2c, glun3a, and glun3b have no pathogenic and a small number of putatively pathogenic variants (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). we therefore focus solely on glun1, glun2a, glun2b, and glun2d for further analysis.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003epathogenic and putatively pathogenic variant positions are well conserved in zebrafish\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003eusing the criteria in table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e, amino acid positions with pathogenic missense variants in glun1, glun2a, glun2b, and glun2d are completely conserved in zebrafish paralogues (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ea-d).\u003c/p\u003e\n\u003cp\u003eputatively pathogenic variants are also conserved in these nmdar subunits, though less so than pathogenic variants. the highest degree of conservation of putatively pathogenic variants is in the glun1 paralogues (\u0026ge;\u0026thinsp;88% identity; 93% similarity). putatively pathogenic variants in the glun2 subunits are also reasonably conserved: zglun2aa (70% identity; 92% similarity), zglun2ab (62% identity; 84% similarity), zglun2ba (65% identity; 71% similarity), zglun2bb (72% identity; 80% similarity), and zglun2da and zglundb (\u0026ge;\u0026thinsp;46% identity; \u0026ge;59% similarity) (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ee,f). in general, the highest level of conservation is in the tmd and tmd-lbd linkers, followed by the lbd, and less so in the ntd and ctd (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003eg,h).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ebenign and putatively benign variant positions show reduced conservation\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ewe carried out a comparable analysis as above for benign and putatively benign variants (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). we hypothesized that positions with more benign variants would be less well conserved than pathogenic ones. indeed, the general trend is consistent with this idea (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003e). putatively benign variants showed a reduced conservation compared to putatively pathogenic variants (putatively pathogenic vs putatively benign): zglun1a (88% vs. 78% identity; 93% vs. 88% similarity), zglun1b (89% vs. 79% identity; 93% vs. 84% similarity), zglun2aa (70% vs. 60% identity; 92% vs. 86% similarity), zglun2ab (62% vs. 48% identity; 84% vs. 68% similarity), zglun2ba (65% vs. 45% identity; 71% vs. 55% similarity), zglun2bb (72% vs. 55% identity; 80% vs. 70% similarity), zglun2da (46% vs. 47% identity; 59% vs. 61% similarity), and zglun2db (47% vs. 44% identity; 61% vs. 56% similarity) (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ee,f vs fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ea,b). for these putatively benign variants, the greatest extent of conservation tends to occur in the tmd and lbd-tmd linkers, followed by the lbd, and then the ntd and ctd (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ec,d).\u003c/p\u003e\n\u003cp\u003efor benign variants, however, the decrease in conservation plateaus, with identity and similarity values comparable to the putatively benign variants, if not slightly higher (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ee-f). the subdomain-specific conservation trend for these variants follows that seen on the level of the whole receptor (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e), wherein the tmd, lbd-tmd linkers, and lbd are generally more conserved and the ntd and ctd less so (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003eg,h). although diverging from our prediction as we do not note a substantial decrease in the conservation of benign variants with respect to the putatively benign variants, this result reflects the overall conservation of the nmdar subunits themselves (benign vs total protein): zglun1 (78% vs. \u0026ge;84% identity; 84% vs. \u0026ge;89% similarity), zglun2aa (62% vs. 68% identity; 86% vs. 89% similarity), zglun2ab (49% vs. 58% identity; 74% vs. 79% similarity), zglun2ba (54% vs. 65% identity; 64% vs. 73% similarity), and zglun2bb (62% vs. 67% identity; 74% vs. 81% similarity), zglun2da (50% vs. 43% identity; 64% vs. 54% similarity), and zglun2db (49% vs. 46% identity; 62% vs. 57% similarity) (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ee-f, table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). furthermore, this result reflects the relatively large number of benign variants in comparison to other variant groups (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecritical tmd and lbd-tmd linker components are completely conserved\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003eelements forming the ion channel \u0026ndash; the tmd (m1, m1-m2, m2, m2-m3, m3, \u0026amp; m4) \u0026ndash; and the lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4) were consistently highly conserved (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). nevertheless, we assayed conservation of critical functional motifs. the s1-m1 linker is a critical element for channel gating and is completely conserved (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e) (\u003cspan class=\"citationref\"\u003e56\u003c/span\u003e). the m2 subdomain contains what is known as the \u0026lsquo;n\u0026rsquo; and \u0026lsquo;n\u0026thinsp;+\u0026thinsp;1\u0026rsquo; sites, which participate in mg\u003csup\u003e2+\u003c/sup\u003e block and ca\u003csup\u003e2+\u003c/sup\u003e permeation through the channel (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e55\u003c/span\u003e) and are conserved in zebrafish (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, dark orange). the tmd m3 segment contains the sytanlaaf motif, which plays a key role in ion channel gating and is known to be highly conserved across species. this motif is conserved in all zebrafish glun1 and glun2 paralogs (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, orange). lastly, the \u0026lsquo;drpeer\u0026rsquo; motif in m3-s2 of glun1, an important component in the high ca\u003csup\u003e2+\u003c/sup\u003e permeability of nmdars (\u003cspan class=\"citationref\"\u003e52\u003c/span\u003e) is conserved (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, light orange). hence, core gating and permeation elements of the tmd and associated regions are highly conserved suggesting that basic mechanisms of ion channel gating and block/permeation are preserved.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003elbd elements required for agonist binding are completely conserved\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ethe glun1 lbd binds glycine or d-serine while the glun2 lbd binds glutamate or aspartate (fig. s2a,b). overall, the lbds are highly conserved in zebrafish relative to humans (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e), as are specific amino acid side chains in s1 and s2 that are involved in binding endogenous agonists (\u003cspan class=\"citationref\"\u003e47\u003c/span\u003e) (fig. s2c). these include sites that participate in van-der waals interactions with the \u0026alpha;-carbon of the ligand (darkest red), interactions with the \u0026alpha;-amino group (dark red), electrostatic interactions with the \u0026alpha;-carboxyl group (red), and interactions with the amino acid side chain (light red) (fig. s2c).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of structure-function elements in the ntd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ein terms of overall sequence comparisons, the ntd shows variable degrees of conservation across nmdar subunits. the glun1 zebrafish paralogs maintain the highest degree of conservation relative to human (80% identity; \u0026ge;88% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the glun2 subunits show decreased ntd conservation (49\u0026ndash;75% identity; 59\u0026ndash;88% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). given this reduced conservation, we assayed the conservation of key structure-function motifs in zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003eamong its many roles, the ntd participates in interactions with ions and molecules that modulate receptor activity (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e59\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ea,b). this is most notable for glun2a and to a lesser extent glun2b as they bind zinc, which inhibits receptor function by reducing channel open probability (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). in glun2a, four amino acids coordinate zinc binding: glun2a-his44, glun2a-his128, glun2a-glu266, and glun2a-asp282 (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). of these key sites, three are conserved (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec) but in zglun2aa and zglun2ab, a key residue involved in zn\u003csup\u003e2+\u003c/sup\u003e binding, glun2a-his44, is not conserved (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec). for glun2b, only two amino acid residues participate in zinc binding: glun2b-his127 and glun2b-glu284 (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). while glun2b-glu284 is conserved, the glun2b-his127 position in zglun2ba is missing due to alterations in ntd sequence length leading to gaps in the alignments, though it is similar now as a positively charged arginine instead of a histidine in zglun2bb (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec).\u003c/p\u003e\n\u003cp\u003ethe ntd of glun2b also binds polyamines, which enhance receptor activity (\u003cspan class=\"citationref\"\u003e59\u003c/span\u003e). nine amino acids are involved in the binding of polyamines. these sites in zglun2ba and zglun2bb are completely conserved except for one amino acid in zglun2bb that is similar but not identical, now a negatively charged aspartate instead of glutamate (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ed). most of this functional component of glun2b is, thus, conserved in zebrafish.\u003c/p\u003e\n\u003cp\u003eadditionally, the nmdar ntd has glycosylation sites, and their glycosylation regulates the trafficking of nmdars to the cell membrane (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e). given the consensus sequence for arginine (n-) glycosylation (n-\u003cem\u003ex\u003c/em\u003e-s/t), glun1 has seven, glun2a has three, and glun2b has three putative glycosylation sites in the ntd (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e). their conservation in the zebrafish paralogues is somewhat variable: zglun1a \u0026amp; zglun1b (5 of 7 sites), zglun2aa (3 of 3 sites), zglun2ab (2 of 3 sites), zglun2ba (2 of 3 sites), and zglun2bb (3 of 3 sites) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ee). to the best of our knowledge, specific glycosylation sites in glun2d have yet to be identified. of note, the glun1 glycosylation sites glun1-n203 and glun1-n368 are necessary for receptor surface expression (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e), and these sites are conserved in the zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003ewhile the critical functional features of the lbd were completely conserved in zebrafish relative to human nmdars, the conservation of key regions in the ntd occasionally diverged, most notably that for zn\u003csup\u003e2+\u003c/sup\u003e binding. to further explore the structural composition of the glun1, glun2a, and glun2b ntds, we used swiss-model to generate homology models for the human and zebrafish nmdar subunits, which we subsequently aligned in pymole to assess conservation at the level of the three-dimensional structure (see methods) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ef). in general, this analysis revealed a high degree of similarity between the human ntd and zebrafish paralogs. the zebrafish glun1 ntd models display similar overall topology to human: glun1 vs zglun1a, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.071); glun1 vs zglun1a, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.085); glun1 vs zglun1b, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.094); and glun1 vs zglun1b, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.097) (table s3).\u003c/p\u003e\n\u003cp\u003ethe glun2a and glun2b subunits also reflect a high degree of structural similarity: glun2a vs zglun2aa, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.085); glun2a vs zglun2ab, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.095); glun2a vs zglun2ab, reference structure 5tpw (rmsd\u0026thinsp;=\u0026thinsp;0.111); glun2b vs zglun2ba, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.028); and glun2b vs glun2bb, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.044) (table s3). the only exception here is the structural comparison of glun2a and zglun2aa using reference structure 5tpw, in which the ntd is interacting with zinc (rmsd\u0026thinsp;=\u0026thinsp;2.149), suggesting that the structural components of zglun2aa may not effectively complex with zinc. interestingly, zglun2ba is \u0026gt;\u0026thinsp;100 amino acids shorter and zglun2bb is \u0026gt;\u0026thinsp;70 amino acids longer than human glun2b. though the zglun2ba model is missing an \u0026alpha;-helix relative to the human protein, the zglun2bb modelled structure aligns well with its human counterpart, suggesting that the zebrafish sequences maintain a large degree of structural integrity.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of post-translational modification (ptm) sites in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ethe intracellular ctd is a key regulatory domain of nmdar cell biology and function and shows distinct subunit-specific elements (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). for our analyses, we focused on the glun1-1 ctd splice variant, which is the longest variant containing the c0-c1-c2 cassettes (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003eacross the nmdar subunits, the zebrafish ctds are the most poorly conserved (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). with a sequence identity generally around 50\u0026ndash;60% for the glun1, glun2a, and glun2b zebrafish paralogs, with some exceptions: zglun1a (53% identity; 58% similarity), zglun1b (54% identity; 59% similarity), zglun2aa (57% identity; 86% identity), zglun2ab (38% identity; 65% similarity), zglun2ba (57% identity, 66% similarity), and zglun2bb (60% identity, 71% similarity) while that of glun2d is much lower (\u0026ge;\u0026thinsp;18% identity, \u0026ge;\u0026thinsp;28% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003egiven these deviations in protein sequences, we first examined the conservation of ctd regions critical for post-translational modifications (ptms), excluding glun2d due its poor conservation in zebrafish. phosphorylation of nmdar subunit ctds confers key functional roles to the receptor (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). initially, we examined experimentally validated sites at which phosphorylation occurs by known kinases and calculated the percent conservation, reported in identity and similarity. similarity, here, is defined as a site at the homologous position that can be phosphorylated (e.g., s, t, or y), if not identical. the glun1 ctd contains phosphorylation sites for src kinase (src), protein kinase c (pkc), and protein kinase a (pka), (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003etop\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). note specific sites have also been identified that are also targeted by the serine/threonine protein phosphatase 2b (pp2b). these ptm sites are perfectly conserved in zglun1a, while three of five are conserved in zglun1b (60% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb). the glun2a ctd is phosphorylated by src, pka, dual specificity tyrosine-phosphorylation-regulated kinase 1a (dyrk1), cyclin-dependent kinase 5 (cdk5), pkc, and calcium/calmodulin-dependent protein kinase ii\u0026alpha; (camkii\u0026alpha;) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003emiddle\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e): zglun2aa (79% identity \u0026amp; similarity) and zglun2ab (64% identity, 71% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb). the ctd of glun2b is phosphorylated by src, camkii\u0026alpha;, cdk5, pka, pkc, death-associated protein kinase 1 (dapk1), proto-oncogene tyrosine-protein kinase fyn (fyn), and casein kinase ii (ck2) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003ebottom\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). glun2b also contains sites for the protein tyrosine phosphatase non-receptor 11 (ptpn11, or shp2) and protein phosphatase 1 (pp1). the conservation of these ptms is slightly higher than that in glun2a: zglun2ba (73% identity \u0026amp; similarity) and zglun2bb (82% identity, 100% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb).\u003c/p\u003e\n\u003cp\u003ein addition to these kinase-regulated sites, we also examined the conservation of other known phosphorylation sites in these subunits identified by phosphoproteomics (psp) (see methods). the conservation of these sites in the zebrafish paralogs was generally lower than that of the kinase-regulated sites: glun1 (67% identity, 100% similarity), zglun2aa (51% identity, 66% similarity), zglun2ab (34% identity, 57% similarity), zglun2ba (60% identity \u0026amp; similarity), and zglun2bb (67% identity, 70% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ec).\u003c/p\u003e\n\u003cp\u003ewe next examined the conservation of ctd palmitoylation and ubiquitylation sites. glun1 has a single ubiquitylation site (lys860) (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003etop\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), and it is conserved in the zebrafish paralogs (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ec). glun2a contains seven palmitoylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003emiddle\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), which are likewise completely conserved in zebrafish (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003eb). the glun2a ctd also has four ubiquitylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003emiddle)\u003c/em\u003e (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), of which three are conserved in zglun2aa (75%) and one in zglun2ab (25%) (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ec). with the same approach for glun2b, we identified eight palmitoylation and six ubiquitylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003ebottom\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). here, all palmitoylation sites are conserved in zebrafish while five of six ubiquitylation sites (83%) are conserved in zglun2ba and all six ubiquitylation sites (100%) are conserved in zglun2bb (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb,c).\u003c/p\u003e\n\u003cp\u003ein summary, the global conservation of ptm sites is as follows: zglun1a and zglun1b (89% identity, 100% similarity), zglun2aa (62% identity, 72% similarity), zglun2ab (45% identity, 62% similarity), zglun2ba (66% identity, 67% similarity), and zglun2bb (75% identity, 78% similarity).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of protein-binding motifs (pbm) in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ein addition to numerous ptm sites, a variety of protein-protein interactions are critical to the role of the ctd in receptor cell biology (surface expression, distribution) and function (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e63\u003c/span\u003e). these protein-binding elements include either defined short linear motifs (slims) or broader regions known to serve as protein docking sites.\u003c/p\u003e\n\u003cp\u003ethe ctd of glun1 contains a slim known as the kkk/rrr er retention motif, which is conserved in zebrafish (\u003cspan class=\"citationref\"\u003e66\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,b). it, likewise, has binding regions for calmodulin (cam)/calcium/calmodulin-dependent protein kinase ii (camkii) and \u0026alpha;-actinin 2 (\u003cspan class=\"citationref\"\u003e64\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e65\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e67\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e78\u003c/span\u003e), which is conserved and yotiao, also known as a-kinase anchoring protein 9 (akap9) (\u003cspan class=\"citationref\"\u003e67\u003c/span\u003e), which is generally well conserved: zglun1a (90% identity, 98% similarity) and zglun1b (90% identity, 98% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,b).\u003c/p\u003e\n\u003cp\u003ethe glun2 subunit ctds contain a variety of pbms. for glun2a, the conservation of slims is: the yxx\u0026oslash; motif, where y is tyrosine, x is any amino acid, and \u0026oslash; is a bulky hydrophobic residue like leucine, isoleucine, phenylalanine, methionine, or valine (\u003cspan class=\"citationref\"\u003e73\u003c/span\u003e) \u0026ndash; both zglun2aa and zglun2ab show 100% identity \u0026amp; similarity; the guanine-nucleotide exchange factor brag2 (\u003cspan class=\"citationref\"\u003e70\u003c/span\u003e) \u0026ndash; zglun2aa (80% identity \u0026amp; similarity) and zglun2ab (40% identity, 60% similarity); postsynaptic density protein 95 (psd-95) (\u003cspan class=\"citationref\"\u003e72\u003c/span\u003e) \u0026ndash; zglun2aa (50% identity; 63% similarity) and zglun2ab (38% identity, 63% similarity); and the pdz domain that interacts with the proteins psd-95, discs large (dlg), and zonula occludens-1 (zo-1) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2aa and zglun2ab (100% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,c). interacting regions in glun2a include: ring finger protein 10 (rnf10) (\u003cspan class=\"citationref\"\u003e70\u003c/span\u003e) \u0026ndash; zglun2aa (51% identity, 72% similarity) and zglun2ab (32% identity, 66% similarity); rabphilin 3a (rph3a) (\u003cspan class=\"citationref\"\u003e68\u003c/span\u003e) \u0026ndash; zglun2aa (71% identity, 80% similarity) and zglun2ab (49% identity, 71% similarity); flotillin-1 (flot-1) (\u003cspan class=\"citationref\"\u003e71\u003c/span\u003e) \u0026ndash; zglun2aa (61% identity, 81% similarity) and zglun2ab (31% identity, 52% similarity); and c-terminal binding protein 1 (ctbp1) (\u003cspan class=\"citationref\"\u003e69\u003c/span\u003e) \u0026ndash; zglun2aa (51% identity, 85% similarity) and zglun2ab (36% identity, 63% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,c, s3a,b).\u003c/p\u003e\n\u003cp\u003efor glun2b, the conservation of slims is: the guanine-nucleotide exchange factor brag1 (\u003cspan class=\"citationref\"\u003e76\u003c/span\u003e) \u0026ndash; zglun2ba (60% identity; 80% similarity) and zglun2bb (60% identity, 100% similarity); psd-95 (\u003cspan class=\"citationref\"\u003e72\u003c/span\u003e) \u0026ndash; zglun2ba (56% identity \u0026amp; similarity) and zglun2bb (33% identity, 56% similarity); camkii and death-associated protein kinase 1 (dapk1) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (92% identity, 100% similarity); synapse-associated protein 102 (sap102) (\u003cspan class=\"citationref\"\u003e75\u003c/span\u003e) \u0026ndash; zglun2ba (0% identity \u0026amp; similarity) and zglun2bb (50% identity \u0026amp; similarity); the adaptor complex ap2 (\u003cspan class=\"citationref\"\u003e74\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (100% identity \u0026amp; similarity); and pdz (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (100% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,d). the ctd interaction regions for glun2b are conserved as follows: receptor for activated c kinase 1 (rack1) (\u003cspan class=\"citationref\"\u003e77\u003c/span\u003e) \u0026ndash; zglun2ba (0% identity, 7% similarity) and zglun2bb (13% identity, 20% similarity); flot-1 (\u003cspan class=\"citationref\"\u003e71\u003c/span\u003e) \u0026ndash; zglun2ba (23% identity; 47% similarity) and zglun2bb (35% identity, 68% similarity); spectrin (\u003cspan class=\"citationref\"\u003e79\u003c/span\u003e) \u0026ndash; zglun2ba (40% identity; 59% similarity) and zglun2bb (35% identity, 62% similarity); \u0026alpha;-actinin 2 (\u003cspan class=\"citationref\"\u003e78\u003c/span\u003e) \u0026ndash; zglun2ba (47% identity; 67% similarity) and zglun2bb (51% identity, 76% similarity); and ras protein-specific guanine nucleotide-releasing factor 1 (rasgrf1) (\u003cspan class=\"citationref\"\u003e80\u003c/span\u003e) \u0026ndash; zglun2ba (59% identity; 74% similarity) and zglun2bb (60% identity, 78% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,d, s3a,c).\u003c/p\u003e\n\u003cp\u003ewhen compared globally across subunits, slims always show greater conservation than regions. this conservation is always greatest in the zebrafish glun1 paralogs, followed by the glun2b and then glun2a paralogs. for the analyzed slims, conservation grouped across each subunit is as follows: zglun1a and zglun1b (100% identity \u0026amp; similarity), zglun2aa (76% identity, 81% similarity), zglun2ab (62% identity, 76% similarity), zglun2ba (78% identity, 81% similarity), and zglun2bb (72% identity, 86% similarity). the regions show the following degrees of conservation: zglun1a and zglun1b (94% identity, 98% similarity), zglun2aa (55% identity, 81% similarity), zglun2ab (36% identity, 62% similarity), zglun2ba (59% identity, 74% similarity), and zglun2bb (60% identity, 78% similarity).\u003c/p\u003e"},{"header":"discussion","content":"\u003cp\u003ehere, we compared the conservation of the zebrafish nmdar subunit paralogues, including key structure-function motifs, to their human counterparts. we find that the core gating machinery \u0026ndash; the lbd, the lbd-tmd linkers and the tmd \u0026ndash; are highly conserved both in terms of overall identity and specific functional motifs. the more peripheral domains \u0026ndash; the ntd and ctd \u0026ndash; show less conservation overall, though functional motifs are reasonably conserved with the notable exception of zn\u003csup\u003e2+\u003c/sup\u003e binding in the glun2a ntd. notably, disease-associated variants are generally conserved, highlighting that zebrafish represent a useful model to study nmdars.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar conservation in zebrafish\u003c/b\u003e\u003c/p\u003e\u003cp\u003eacross most of our examined parameters, the obligatory glun1 subunit shows the highest degree of conservation in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e). generally, conservation is then greatest in the glun2(a-d) followed by the glun3(a-b) subunits. within each subunit, sequence conservation is consistently highest in the tmd and lbd-tmd linkers, followed by the lbd. conservation is generally poorer in the ntd and even worse in the ctd (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ea key question with regards to this work is defining what constitutes as a \u0026lsquo;well conserved\u0026rsquo; sequence. previous studies regarding the conservation of protein structure and function suggest that sequence identities between 40\u0026ndash;70% confer functional conservation (\u003cspan additionalcitationids=\"cr93\" citationid=\"cr92\" class=\"citationref\"\u003e92\u003c/span\u003e\u0026ndash;\u003cspan citationid=\"cr94\" class=\"citationref\"\u003e94\u003c/span\u003e). they regard 50% sequence identity as the threshold below which function drastically diverges (\u003cspan citationid=\"cr93\" class=\"citationref\"\u003e93\u003c/span\u003e, \u003cspan citationid=\"cr94\" class=\"citationref\"\u003e94\u003c/span\u003e). our sequence conservation analyses almost always led to sequence identities\u0026thinsp;\u0026gt;\u0026thinsp;50%, except for zglun2da. nevertheless, we carried out more detailed examinations of more poorly conserved domains, e.g., the ntd and ctd, to assess the conservation of key structure-function motifs.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar missense variant conservation in zebrafish\u003c/b\u003e\u003c/p\u003e\u003cp\u003eto date, a multitude of variants, including missense, frameshift, and nonsense mutations, have been identified in the genes encoding the nmdar subunits (\u003cspan citationid=\"cr21\" class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan citationid=\"cr22\" class=\"citationref\"\u003e22\u003c/span\u003e, \u003cspan citationid=\"cr87\" class=\"citationref\"\u003e87\u003c/span\u003e). we categorized identified missense variants by pathogenicity and analyzed the extent of conservation of the positions at which they occur. we focused only on the glun1, glun2a, glun2b, and glun2d subunits for this analysis, as this is where most disease-associated variants have been identified. generally, our results demonstrate a higher degree of variant position conservation with increasing pathogenicity (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e,\u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e). the sites of pathogenic missense variants are almost perfectly conserved (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e). given the high conservation of the amino acid positions at which they occur, these protein regions are likely most essential for receptor function. across our categories of variants, with the exclusion of complex variants, the proportion of the total protein at which missense variants have been identified is lowest in the pathogenic group (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e). it is possible that additional variants that might be categorized as pathogenic are so devastating to development as to be embryonic lethal. this reflects the notion that more integral regions of the receptor are less tolerant to variants (\u003cspan citationid=\"cr95\" class=\"citationref\"\u003e95\u003c/span\u003e). zebrafish, nevertheless, emerge as a useful model for the study of pathogenic nmdar variants.\u003c/p\u003e\u003cp\u003eputatively pathogenic variants are conserved to a lesser extent than their pathogenic counterparts (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e). according to our designations, these variants have the potential to cause disease, having been identified in patients, but are currently classified as having uncertain significance. thus, it aligns that these variants are comparatively less well conserved \u0026ndash; they are likely occurring in less critical regions of the receptor and are likewise less likely to cause disease phenotypes.\u003c/p\u003e\u003cp\u003eour subsequent investigation of putatively benign and benign variants revealed a further decrease in variant position conservation. this finding further supported our prediction that variants more highly associated with disease occur at positions with higher degrees of conservation and, thus, are more critical to protein structure and function. the conservation of positions with putatively benign and benign variants is quite comparable. for the benign variants, in particular, conservation percentages also closely reflect total protein conservation values for each paralog (fig.\u0026nbsp;\u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e, table\u0026nbsp;\u003cspan refid=\"tab1\" class=\"internalref\"\u003e1\u003c/span\u003e). this follows from the finding that positions with identified benign variants, for most subunits, make up a substantial portion of the protein (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enevertheless, this investigation both demonstrates the advantage of the zebrafish model in studying nmdar-associated disease and reflects the notion that the positions most essential for receptor function are those that are best conserved and more likely to induce disease when absent.\u003c/p\u003e\u003cp\u003eoverall, the trend for variant conservation is the same as that for the whole protein. conservation is always greatest in the tmd and lbd-tmd linkers, followed by the lbd, next the ntd, and last the ctd (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e, \u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e). these findings suggest that zebrafish serve as an effective model for variants, especially pathogenic and putatively pathogenic variants in both the tmd and lbd-tmd linkers. some caution, however, must be exercised when using this model system for variants in the lbd, ntd, and ctd.\u003c/p\u003e\u003cp\u003e\u003cb\u003etmd, tmd-lbd linker, and lbd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003eour examination of key functional regions and motifs in the tmd, lbd-tmd linkers, and lbd consistently revealed complete conservation in their respective zebrafish paralogs (fig. \u003cspan refid=\"moesm1\" class=\"internalref\"\u003es1\u003c/span\u003e,s2). this suggests that the basic mechanism of ion channel gating is conserved in zebrafish relative to human. as these domains and subdomains contribute to the formation of the core of the receptor \u0026ndash; the ion channel pore allowing for ion flux \u0026ndash; it is expected that they would be most highly conserved across species, since the major functional role of nmdars is current flux during synaptic transmission.\u003c/p\u003e\u003cp\u003e\u003cb\u003entd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003ethe ntd is the first region in which nmdar sequences in zebrafish begin to diverge from those in human. modelling ntd variants in zebrafish must, thus, be approached with caution. for example, polyamine-binding sites in glun2b are almost completely conserved in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig8\" class=\"internalref\"\u003e4\u003c/span\u003ed), so they have the potential to serve as an effective model for this functional role of nmdars. on the other hand, zinc-binding sites in glun2a and glun2b are more poorly conserved in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig8\" class=\"internalref\"\u003e4\u003c/span\u003ec), so they would likely serve as a poor model for zinc-related studies in nmdars. this also aligns with our homology model finding that the zglun2aa model aligns well with glun2a when not bound to zinc, but the rmsd of the alignment is substantially poorer when glun2a interacting with zinc is used as the reference structure (table s3).\u003c/p\u003e\u003cp\u003e\u003cb\u003ectd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003ethe c-terminal domains (ctds) exhibit the lowest level of overall conservation across nmdar domains (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e). this is consistent with the ctd\u0026rsquo;s classification as an intrinsically disordered region (idr), which lacks fixed secondary or tertiary structure and is more tolerant to evolutionary divergence (\u003cspan citationid=\"cr96\" class=\"citationref\"\u003e96\u003c/span\u003e, \u003cspan citationid=\"cr97\" class=\"citationref\"\u003e97\u003c/span\u003e). nonetheless, functional conservation within idrs is often maintained through the retention of post-translational modification (ptm) hotspots and short linear motifs (slims) that regulate protein interactions and intracellular signaling (\u003cspan citationid=\"cr98\" class=\"citationref\"\u003e98\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ewe assessed ctd conservation in glun1, glun2a, and glun2b by analyzing experimentally validated phosphorylation, palmitoylation, and ubiquitylation sites, as well as defined protein-binding motifs. phosphorylation sites with known kinase interactions exhibited relatively high conservation, consistent with their well-characterized regulatory roles (figs.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ea,b). in contrast, phosphorylation sites without identified kinases were less consistently retained (fig.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ec). palmitoylation sites were fully conserved across all subunits, while ubiquitylation sites were generally retained, though conservation was notably lower in zglun2ab (fig.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ef) and may reflect paralogue-specific divergence in degradation or trafficking pathways.\u003c/p\u003e\u003cp\u003eprotein-binding motif conservation varied by subunit. glun1 retained nearly all canonical interaction motifs, while glun2a and glun2b displayed region-specific conservation. notably well-conserved elements in the glun2 subunits included the pdz-binding motifs as well as docking regions for camkii, dapk1, and ap2 (fig.\u0026nbsp;\u003cspan refid=\"fig12\" class=\"internalref\"\u003e6\u003c/span\u003e). slims overall showed higher conservation, likely due to their short, sequence-specific structure, which may place them under stronger evolutionary constraint than broader, more flexible interaction regions.\u003c/p\u003e\u003cp\u003edespite broad sequence divergence, nmdar ctds retain a core set of conserved motifs involved in receptor trafficking, synaptic localization, and plasticity. kinase-regulated phosphorylation sites, palmitoylation sites, and pdz-binding motifs were consistently preserved. conservation of glun1 sites was always greatest in the zebrafish paralogs. among the glun2 subunits, glun2b displayed the highest overall conservation of functional elements, aligning with its essential role in neurodevelopment and synaptic signaling.\u003c/p\u003e\u003cp\u003ethese findings are especially relevant to the study of disease-associated truncating variants in the nmdar subunits. nonsense variants that introduce premature stop codons in the ctd eliminate key regulatory motifs critical for trafficking, anchoring, or signaling. when these affected regions correspond to conserved functional elements, zebrafish may serve as an appropriate \u003cem\u003ein vitro\u003c/em\u003e and \u003cem\u003ein vivo\u003c/em\u003e model for studying ctd-mediated disease mechanisms. conversely, for mutations in poorly conserved regions, the translational utility of this model may become limited.\u003c/p\u003e"},{"header":"conclusions","content":"\u003cp\u003ezebrafish represent a valuable system to study nmdar-associated neurodevelopmental disorders, but one must be judicious in choosing which variants to study in this model. our unique approach of sequence alignments per individual nmdar domains and subdomains highlights the high degree of conversation of some regions of the protein relative to others. additionally, our analysis of identified variants indicates that zebrafish are an effective model for pathogenic variants as these occur at conserved locations along the protein. generally, the utility of zebrafish is most apparent for the study of the tmd, lbd-tmd linkers, and lbd as these regions are most well conserved. on the other hand, studying the ntd and ctd in this species requires extra consideration stemming from deviations in sequence conservation. our work will enable future nmdar-related studies that can be effectively and efficiently conducted in zebrafish.\u003c/p\u003e"},{"header":"abbreviations","content":"\u003ctable border=\"0\" cellspacing=\"0\" cellpadding=\"0\"\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003enmdar\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003e\u003cem\u003en\u003c/em\u003e-methyl-d-aspartate receptor\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003entd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003en-terminal domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003elbd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003eligand-binding domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003etmd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003etransmembrane domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003ectd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003ec-terminal domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e"},{"header":"declarations","content":"\u003cp\u003e\u003cstrong\u003eethics approval and consent to participate:\u0026nbsp;\u003c/strong\u003enot applicable.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econsent for publication:\u0026nbsp;\u003c/strong\u003enot applicable.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eavailability of data and materials:\u003c/strong\u003e the datasets used and/or analyzed during the current study are available either at osf (https://osf.io/h2zf9/) or from the corresponding author on reasonable request.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecompeting interests:\u0026nbsp;\u003c/strong\u003eno competing interests declared.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003efunding:\u003c/strong\u003e this work was supported by nih grant r01ns088479 to lpw.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eauthors\u0026rsquo; contributions:\u0026nbsp;\u003c/strong\u003ee.r.n., c.a., j.d.z., h.i.s., and l.p.w. contributed to the conceptualization and methodology of this project. e.r.n., c.a., and j.d.z. conducted the investigation and analysis. w.r. assisted with data curation. e.r.n, c.a., and l.p.w. wrote the manuscript, which was revised with insights from j.d.z. and h.i.s.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eacknowledgements:\u0026nbsp;\u003c/strong\u003ethe authors thank the clinvar and gnomad databases and the swiss-model server for their important contributions to this work.\u003c/p\u003e"},{"header":"references","content":"\u003col\u003e\n\u003cli\u003epaoletti p, bellone c, zhou q. nmda receptor subunit diversity: impact on receptor properties, synaptic plasticity and disease. nat rev neurosci. 2013;14(6):383-400.\u003c/li\u003e\n\u003cli\u003ehansen kb, wollmuth lp, bowie d, furukawa h, menniti fs, sobolevsky ai, et al. structure, function, and pharmacology of glutamate receptor ion channels. pharmacol rev. 2021;73(4):298-487.\u003c/li\u003e\n\u003cli\u003echakraborty a, murphy s, coleman n. the role of nmda receptors in neural stem cell proliferation and differentiation. stem cells dev. 2017;26(11):798-807.\u003c/li\u003e\n\u003cli\u003enacher j, mcewen bs. the role of n-methyl-d-asparate receptors in neurogenesis. hippocampus. 2006;16(3):267-70.\u003c/li\u003e\n\u003cli\u003ekomuro h, rakic p. modulation of neuronal migration by nmda receptors. science. 1993;260(5104):95-7.\u003c/li\u003e\n\u003cli\u003epearce ia, cambray-deakin ma, burgoyne rd. glutamate acting on nmda receptors stimulates neurite outgrowth from cerebellar granule cells. febs lett. 1987;223(1):143-7.\u003c/li\u003e\n\u003cli\u003ebooker sa, wyllie dja. nmda receptor function in inhibitory neurons. neuropharmacology. 2021;196:108609.\u003c/li\u003e\n\u003cli\u003egambrill ac, barria a. nmda receptor subunit composition controls synaptogenesis and synapse stabilization. proc natl acad sci u s a. 2011;108(14):5855-60.\u003c/li\u003e\n\u003cli\u003enapoli aj, laderwager s, zoodsma jd, biju b, mucollari o, schubel sk, et al. developmental loss of nmda receptors results in supernumerary forebrain neurons through delayed maturation of transit-amplifying neuroblasts. sci rep. 2024;14(1):3395.\u003c/li\u003e\n\u003cli\u003erajani v, sengar as, salter mw. tripartite signalling by nmda receptors. mol brain. 2020;13(1):23.\u003c/li\u003e\n\u003cli\u003eishchenko y, carrizales mg, koleske aj. regulation of the nmda receptor by its cytoplasmic domains: (how) is the tail wagging the dog? neuropharmacology. 2021;195:108634.\u003c/li\u003e\n\u003cli\u003emony l, paoletti p. mechanisms of nmda receptor regulation. curr opin neurobiol. 2023;83:102815.\u003c/li\u003e\n\u003cli\u003enielsen ac, anderson cl, ens c, boyce akj, thompson rj. non-ionotropic nmdar signalling activates panx1 to induce p2x4r-dependent long-term depression in the hippocampus. j physiol. 2024.\u003c/li\u003e\n\u003cli\u003ebarnes sa, thomazeau a, finnie psb, heinrich mj, heynen aj, komiyama nh, et al. non-ionotropic signaling through the nmda receptor glun2b carboxy-terminal domain drives dendritic spine plasticity and reverses fragile x phenotypes. cell rep. 2025;44(3):115311.\u003c/li\u003e\n\u003cli\u003eplatzer k, lemke jr. grin1-related neurodevelopmental disorder. genereviews((r)). 2021.\u003c/li\u003e\n\u003cli\u003estrehlow v, myers ka, morgan at, scheffer ie, lemke jr. grin2a-related disorders. genereviews((r)). 2024.\u003c/li\u003e\n\u003cli\u003eplatzer k, lemke jr. grin2b-related neurodevelopmental disorder. genereviews((r)). 2021.\u003c/li\u003e\n\u003cli\u003eplatzer k, krey i, lemke jr. grin2d-related developmental and epileptic encephalopathy. genereviews((r)). 2022.\u003c/li\u003e\n\u003cli\u003ezehavi y, mandel h, zehavi a, rashid ma, straussberg r, jabur b, et al. de novo grin1 mutations: an emerging cause of severe early infantile encephalopathy. eur j med genet. 2017;60(6):317-20.\u003c/li\u003e\n\u003cli\u003esabo sl, lahr jm, offer m, weekes a, sceniak mp. grin2b-related neurodevelopmental disorder: current understanding of pathophysiological mechanisms. front synaptic neurosci. 2022;14:1090865.\u003c/li\u003e\n\u003cli\u003egjerulfsen ce, krey i, klockner c, rubboli g, lemke jr, moller rs. spectrum of nmda receptor variants in neurodevelopmental disorders and epilepsy. methods mol biol. 2024;2799:1-11.\u003c/li\u003e\n\u003cli\u003eamin jb, moody gr, wollmuth lp. from bedside-to-bench: what disease-associated variants are teaching us about the nmda receptor. j physiol. 2021;599(2):397-416.\u003c/li\u003e\n\u003cli\u003ehu c, chen w, myers sj, yuan h, traynelis sf. human grin2b variants in neurodevelopmental disorders. j pharmacol sci. 2016;132(2):115-21.\u003c/li\u003e\n\u003cli\u003ecox ja, kucenas s, voigt mm. molecular characterization and embryonic expression of the family of n-methyl-d-aspartate receptor subunit genes in the zebrafish. dev dyn. 2005;234(3):756-66.\u003c/li\u003e\n\u003cli\u003eveldman mb, lin s. zebrafish as a developmental model organism for pediatric research. pediatr res. 2008;64(5):470-6.\u003c/li\u003e\n\u003cli\u003echoi ty, choi ti, lee yr, choe sk, kim ch. zebrafish as an animal model for biomedical research. exp mol med. 2021;53(3):310-7.\u003c/li\u003e\n\u003cli\u003ezoodsma jd, chan k, bhandiwad aa, golann dr, liu g, syed sa, et al. a model to study nmda receptors in early nervous system development. j neurosci. 2020;40(18):3631-45.\u003c/li\u003e\n\u003cli\u003ezoodsma jd, keegan ej, moody gr, bhandiwad aa, napoli aj, burgess ha, et al. disruption of grin2b, an asd-associated gene, produces social deficits in zebrafish. mol autism. 2022;13(1):38.\u003c/li\u003e\n\u003cli\u003ebaraban sc, taylor mr, castro pa, baier h. pentylenetetrazole induced changes in zebrafish behavior, neural activity and c-fos expression. neuroscience. 2005;131(3):759-68.\u003c/li\u003e\n\u003cli\u003eshams s, rihel j, ortiz jg, gerlai r. the zebrafish as a promising tool for modeling human brain disorders: a review based upon an ibns symposium. neurosci biobehav rev. 2018;85:176-90.\u003c/li\u003e\n\u003cli\u003ede abreu ms, genario r, giacomini a, demin ka, lakstygal am, amstislavskaya tg, et al. zebrafish as a model of neurodevelopmental disorders. neuroscience. 2020;445:3-11.\u003c/li\u003e\n\u003cli\u003eweinschutz mendes h, neelakantan u, liu y, fitzpatrick se, chen t, wu w, et al. high-throughput functional analysis of autism genes in zebrafish identifies convergence in dopaminergic and neuroimmune pathways. cell rep. 2023;42(3):112243.\u003c/li\u003e\n\u003cli\u003ehowe k, clark md, torroja cf, torrance j, berthelot c, muffato m, et al. the zebrafish reference genome sequence and its relationship to the human genome. nature. 2013;496(7446):498-503.\u003c/li\u003e\n\u003cli\u003erihel j, schier af. behavioral screening for neuroactive drugs in zebrafish. dev neurobiol. 2012;72(3):373-85.\u003c/li\u003e\n\u003cli\u003ebaraban sc, dinday mt, hortopan ga. drug screening in scn1a zebrafish mutant identifies clemizole as a potential dravet syndrome treatment. nat commun. 2013;4:2410.\u003c/li\u003e\n\u003cli\u003ezhang y, kecskes a, copmans d, langlois m, crawford ad, ceulemans b, et al. pharmacological characterization of an antisense knockdown zebrafish model of dravet syndrome: inhibition of epileptic seizures by the serotonin agonist fenfluramine. plos one. 2015;10(5):e0125898.\u003c/li\u003e\n\u003cli\u003edingemans ajm, jansen s, van reeuwijk j, de leeuw n, pfundt r, schuurs-hoeijmakers j, et al. prevalence of comorbidities in individuals with neurodevelopmental disorders from the aggregated phenomics data of 51,227 pediatric individuals. nat med. 2024;30(7):1994-2003.\u003c/li\u003e\n\u003cli\u003eohtsuki t, sakurai k, dou h, toru m, yamakawa-kobayashi k, arinami t. mutation analysis of the nmdar2b (grin2b) gene in schizophrenia. mol psychiatry. 2001;6(2):211-6.\u003c/li\u003e\n\u003cli\u003earnold pd, rosenberg dr, mundo e, tharmalingam s, kennedy jl, richter ma. association of a glutamate (nmda) subunit receptor gene (grin2b) with obsessive-compulsive disorder: a preliminary study. psychopharmacology (berl). 2004;174(4):530-8.\u003c/li\u003e\n\u003cli\u003esaunders ja, tatard-leitman vm, suh j, billingslea en, roberts tp, siegel sj. knockout of nmda receptors in parvalbumin interneurons recreates autism-like phenotypes. autism res. 2013;6(2):69-77.\u003c/li\u003e\n\u003cli\u003egandal mj, sisti j, klook k, ortinski pi, leitman v, liang y, et al. gabab-mediated rescue of altered excitatory-inhibitory balance, gamma synchrony and behavioral deficits following constitutive nmdar-hypofunction. transl psychiatry. 2012;2(7):e142.\u003c/li\u003e\n\u003cli\u003ebarnby g, abbott a, sykes n, morris a, weeks de, mott r, et al. candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at grin2a and abat. am j hum genet. 2005;76(6):950-66.\u003c/li\u003e\n\u003cli\u003ehung ly, margolis kg. autism spectrum disorders and the gastrointestinal tract: insights into mechanisms and clinical relevance. nat rev gastroenterol hepatol. 2024;21(3):142-63.\u003c/li\u003e\n\u003cli\u003eowens ap, mathias cj, iodice v. autonomic dysfunction in autism spectrum disorder. front integr neurosci. 2021;15:787037.\u003c/li\u003e\n\u003cli\u003eorefice ll. peripheral somatosensory neuron dysfunction: emerging roles in autism spectrum disorders. neuroscience. 2020;445:120-9.\u003c/li\u003e\n\u003cli\u003emakris g, agorastos a, chrousos gp, pervanidou p. stress system activation in children and adolescents with autism spectrum disorder. front neurosci. 2021;15:756628.\u003c/li\u003e\n\u003cli\u003eramos-vicente d, grant sg, bayes a. metazoan evolution and diversity of glutamate receptors and their auxiliary subunits. neuropharmacology. 2021;195:108640.\u003c/li\u003e\n\u003cli\u003eamores a, force a, yan yl, joly l, amemiya c, fritz a, et al. zebrafish hox clusters and vertebrate genome evolution. science. 1998;282(5394):1711-4.\u003c/li\u003e\n\u003cli\u003echou th, tajima n, romero-hernandez a, furukawa h. structural basis of functional transitions in mammalian nmda receptors. cell. 2020;182(2):357-71 e13.\u003c/li\u003e\n\u003cli\u003ezhang y, ye f, zhang t, lv s, zhou l, du d, et al. structural basis of ketamine action on human nmda receptors. nature. 2021;596(7871):301-5.\u003c/li\u003e\n\u003cli\u003ewollmuth lp, sakmann b. different mechanisms of ca2+ transport in nmda and ca2+-permeable ampa glutamate receptor channels. j gen physiol. 1998;112(5):623-36.\u003c/li\u003e\n\u003cli\u003ewatanabe j, beck c, kuner t, premkumar ls, wollmuth lp. drpeer: a motif in the extracellular vestibule conferring high ca2+ flux rates in nmda receptor channels. j neurosci. 2002;22(23):10209-16.\u003c/li\u003e\n\u003cli\u003ewollmuth lp, sobolevsky ai. structure and gating of the glutamate receptor ion channel. trends neurosci. 2004;27(6):321-8.\u003c/li\u003e\n\u003cli\u003eogden kk, khatri a, traynelis sf, heldt sa. potentiation of glun2c/d nmda receptor subtypes in the amygdala facilitates the retention of fear and extinction learning in mice. neuropsychopharmacology. 2014;39(3):625-37.\u003c/li\u003e\n\u003cli\u003ewollmuth lp. ion permeation in ionotropic glutamate receptors: still dynamic after all these years. curr opin physiol. 2018;2:36-41.\u003c/li\u003e\n\u003cli\u003emcdaniel mj, ogden kk, kell sa, burger pb, liotta dc, traynelis sf. nmda receptor channel gating control by the pre-m1 helix. j gen physiol. 2020;152(4).\u003c/li\u003e\n\u003cli\u003ewollmuth lp, kuner t, sakmann b. adjacent asparagines in the nr2-subunit of the nmda receptor channel control the voltage-dependent block by extracellular mg2+. j physiol. 1998;506 ( pt 1)(pt 1):13-32.\u003c/li\u003e\n\u003cli\u003eromero-hernandez a, simorowski n, karakas e, furukawa h. molecular basis for subtype specificity and high-affinity zinc inhibition in the glun1-glun2a nmda receptor amino-terminal domain. neuron. 2016;92(6):1324-36.\u003c/li\u003e\n\u003cli\u003emony l, zhu s, carvalho s, paoletti p. molecular basis of positive allosteric modulation of glun2b nmda receptors by polyamines. embo j. 2011;30(15):3134-46.\u003c/li\u003e\n\u003cli\u003elichnerova k, kaniakova m, park sp, skrenkova k, wang yx, petralia rs, et al. two n-glycosylation sites in the glun1 subunit are essential for releasing n-methyl-d-aspartate (nmda) receptors from the endoplasmic reticulum. j biol chem. 2015;290(30):18379-90.\u003c/li\u003e\n\u003cli\u003ewaterhouse a, bertoni m, bienert s, studer g, tauriello g, gumienny r, et al. swiss-model: homology modelling of protein structures and complexes. nucleic acids res. 2018;46(w1):w296-w303.\u003c/li\u003e\n\u003cli\u003ebienert s, waterhouse a, de beer ta, tauriello g, studer g, bordoli l, et al. the swiss-model repository-new features and functionality. nucleic acids res. 2017;45(d1):d313-d9.\u003c/li\u003e\n\u003cli\u003ewarnet xl, bakke krog h, sevillano-quispe og, poulsen h, kjaergaard m. the c-terminal domains of the nmda receptor: how intrinsically disordered tails affect signalling, plasticity and disease. eur j neurosci. 2021;54(8):6713-39.\u003c/li\u003e\n\u003cli\u003ekim k, saneyoshi t, hosokawa t, okamoto k, hayashi y. interplay of enzymatic and structural functions of camkii in long-term potentiation. j neurochem. 2016;139(6):959-72.\u003c/li\u003e\n\u003cli\u003ebej a, ames jb. chemical shift assignments of calmodulin bound to a cytosolic domain of glun2a (residues 1004-1024) from the nmda receptor. biomol nmr assign. 2023;17(1):89-93.\u003c/li\u003e\n\u003cli\u003ehorak m, wenthold rj. different roles of c-terminal cassettes in the trafficking of full-length nr1 subunits to the cell surface. j biol chem. 2009;284(15):9683-91.\u003c/li\u003e\n\u003cli\u003elin jw, wyszynski m, madhavan r, sealock r, kim ju, sheng m. yotiao, a novel protein of neuromuscular junction and brain that interacts with specific splice variants of nmda receptor subunit nr1. j neurosci. 1998;18(6):2017-27.\u003c/li\u003e\n\u003cli\u003estanic j, carta m, eberini i, pelucchi s, marcello e, genazzani aa, et al. rabphilin 3a retains nmda receptors at synaptic sites through interaction with glun2a/psd-95 complex. nat commun. 2015;6:10181.\u003c/li\u003e\n\u003cli\u003ecousins sl, stephenson fa. identification of c-terminal binding protein 1 as a novel nmda receptor interactor. neurochem res. 2019;44(6):1437-45.\u003c/li\u003e\n\u003cli\u003efranchini l, carrano n, di luca m, gardoni f. synaptic glun2a-containing nmda receptors: from physiology to pathological synaptic plasticity. int j mol sci. 2020;21(4).\u003c/li\u003e\n\u003cli\u003eswanwick cc, shapiro me, yi z, chang k, wenthold rj. nmda receptors interact with flotillin-1 and -2, lipid raft-associated proteins. febs lett. 2009;583(8):1226-30.\u003c/li\u003e\n\u003cli\u003ecousins sl, stephenson fa. identification of n-methyl-d-aspartic acid (nmda) receptor subtype-specific binding sites that mediate direct interactions with scaffold protein psd-95. j biol chem. 2012;287(16):13465-76.\u003c/li\u003e\n\u003cli\u003evissel b, krupp jj, heinemann sf, westbrook gl. a use-dependent tyrosine dephosphorylation of nmda receptors is independent of ion flux. nat neurosci. 2001;4(6):587-96.\u003c/li\u003e\n\u003cli\u003elavezzari g, mccallum j, lee r, roche kw. differential binding of the ap-2 adaptor complex and psd-95 to the c-terminus of the nmda receptor subunit nr2b regulates surface expression. neuropharmacology. 2003;45(6):729-37.\u003c/li\u003e\n\u003cli\u003echen bs, gray ja, sanz-clemente a, wei z, thomas ev, nicoll ra, et al. sap102 mediates synaptic clearance of nmda receptors. cell rep. 2012;2(5):1120-8.\u003c/li\u003e\n\u003cli\u003eelagabani mn, brisevac d, kintscher m, pohle j, kohr g, schmitz d, et al. subunit-selective n-methyl-d-aspartate (nmda) receptor signaling through brefeldin a-resistant arf guanine nucleotide exchange factors brag1 and brag2 during synapse maturation. j biol chem. 2016;291(17):9105-18.\u003c/li\u003e\n\u003cli\u003eyaka r, thornton c, vagts aj, phamluong k, bonci a, ron d. nmda receptor function is regulated by the inhibitory scaffolding protein, rack1. proc natl acad sci u s a. 2002;99(8):5710-5.\u003c/li\u003e\n\u003cli\u003ewyszynski m, lin j, rao a, nigh e, beggs ah, craig am, et al. competitive binding of alpha-actinin and calmodulin to the nmda receptor. nature. 1997;385(6615):439-42.\u003c/li\u003e\n\u003cli\u003ewechsler a, teichberg vi. brain spectrin binding to the nmda receptor is regulated by phosphorylation, calcium and calmodulin. embo j. 1998;17(14):3931-9.\u003c/li\u003e\n\u003cli\u003ekrapivinsky g, krapivinsky l, manasian y, ivanov a, tyzio r, pellegrino c, et al. the nmda receptor is coupled to the erk pathway by a direct interaction between nr2b and rasgrf1. neuron. 2003;40(4):775-84.\u003c/li\u003e\n\u003cli\u003emukherjee p, roy s, ghosh d, nandi sk. role of animal models in biomedical research: a review. lab anim res. 2022;38(1):18.\u003c/li\u003e\n\u003cli\u003echou th, epstein m, fritzemeier rg, akins ns, paladugu s, ullman ez, et al. molecular mechanism of ligand gating and opening of nmda receptor. nature. 2024;632(8023):209-17.\u003c/li\u003e\n\u003cli\u003earmstrong n, sun y, chen gq, gouaux e. structure of a glutamate-receptor ligand-binding core in complex with kainate. nature. 1998;395(6705):913-7.\u003c/li\u003e\n\u003cli\u003earmstrong n, gouaux e. mechanisms for activation and antagonism of an ampa-sensitive glutamate receptor: crystal structures of the glur2 ligand binding core. neuron. 2000;28(1):165-81.\u003c/li\u003e\n\u003cli\u003ekazi r, gan q, talukder i, markowitz m, salussolia cl, wollmuth lp. asynchronous movements prior to pore opening in nmda receptors. j neurosci. 2013;33(29):12052-66.\u003c/li\u003e\n\u003cli\u003eamin jb, gochman a, he m, certain n, wollmuth lp. nmda receptors require multiple pre-opening gating steps for efficient synaptic activity. neuron. 2021;109(3):488-501 e4.\u003c/li\u003e\n\u003cli\u003ehardingham ge, do kq. linking early-life nmdar hypofunction and oxidative stress in schizophrenia pathogenesis. nat rev neurosci. 2016;17(2):125-34.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, riley gr, jang w, rubinstein ws, church dm, et al. clinvar: public archive of relationships among sequence variation and human phenotype. nucleic acids res. 2014;42(database issue):d980-5.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, benson m, brown g, chao c, chitipiralla s, et al. clinvar: public archive of interpretations of clinically relevant variants. nucleic acids res. 2016;44(d1):d862-8.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, benson m, brown gr, chao c, chitipiralla s, et al. clinvar: improving access to variant interpretations and supporting evidence. nucleic acids res. 2018;46(d1):d1062-d7.\u003c/li\u003e\n\u003cli\u003egudmundsson s, singer-berk m, watts na, phu w, goodrich jk, solomonson m, et al. variant interpretation using population databases: lessons from gnomad. hum mutat. 2022;43(8):1012-30.\u003c/li\u003e\n\u003cli\u003etian w, skolnick j. how well is enzyme function conserved as a function of pairwise sequence identity? j mol biol. 2003;333(4):863-82.\u003c/li\u003e\n\u003cli\u003eaddou s, rentzsch r, lee d, orengo ca. domain-based and family-specific sequence identity thresholds increase the levels of reliable protein function transfer. j mol biol. 2009;387(2):416-30.\u003c/li\u003e\n\u003cli\u003esangar v, blankenberg dj, altman n, lesk am. quantitative sequence-function relationships in proteins based on gene ontology. bmc bioinformatics. 2007;8:294.\u003c/li\u003e\n\u003cli\u003eperszyk re, kristensen as, lyuboslavsky p, traynelis sf. three-dimensional missense tolerance ratio analysis. genome res. 2021;31(8):1447-61.\u003c/li\u003e\n\u003cli\u003euversky vn. intrinsically disordered proteins from a to z. int j biochem cell biol. 2011;43(8):1090-103.\u003c/li\u003e\n\u003cli\u003esingleton md, eisen mb. evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. plos comput biol. 2024;20(4):e1012028.\u003c/li\u003e\n\u003cli\u003edavey ne, cowan jl, shields dc, gibson tj, coldwell mj, edwards rj. slimprints: conservation-based discovery of functional motif fingerprints in intrinsically disordered protein regions. nucleic acids res. 2012;40(21):10628-41.\u003c/li\u003e\n\u003c/ol\u003e"}],"fulltextsource":"","fulltext":"","funders":[],"hasadminpriorityonworkflow":false,"hasmanuscriptdocx":true,"hasoptedintopreprint":true,"haspassedjournalqc":"","hasanypriority":false,"hidejournal":false,"highlight":"","institution":"","isacceptedbyjournal":false,"isauthorsuppliedpdf":false,"isdeskrejected":"","ishiddenfromsearch":false,"isinqc":false,"isinworkflow":false,"ispdf":false,"ispdfuptodate":true,"iswithdrawnorretracted":false,"journal":{"display":true,"email":"info@researchsquare.com","identity":"bmc-genomics","isnaturejournal":false,"hasqc":true,"allowdirectsubmit":false,"externalidentity":"gics","sidebox":"learn more about [bmc genomics](http://bmcgenomics.biomedcentral.com/)","snpcode":"","submissionurl":"https://www.editorialmanager.com/gics","title":"bmc genomics","twitterhandle":"#bmcgenomics","acdcenabled":true,"dfaenabled":false,"editorialsystem":"em","reportingportfolio":"bmc series","inreviewenabled":true,"inreviewrevisionsenabled":true},"keywords":"grin genes, clinvar, gnomad, sequence alignments, disease-associated variants, n-terminal domain, ligand-binding domain, transmembrane domain, c-terminal domain, ion channel","lastpublisheddoi":"10.21203/rs.3.rs-7151578/v1","lastpublisheddoiurl":"https://doi.org/10.21203/rs.3.rs-7151578/v1","license":{"name":"cc by 4.0","url":"https://creativecommons.org/licenses/by/4.0/"},"manuscriptabstract":"\u003ch2\u003ebackground\u003c/h2\u003e\u003cp\u003enmda receptors (nmdars) are widely expressed, glutamate-gated ion channels that play key roles in brain development and function. variants have been identified in the \u003cem\u003egrin\u003c/em\u003e genes encoding nmdar subunits that are linked to neurodevelopmental disorders, among other manifestations. zebrafish are a powerful model to study brain development and function given their rapid development and ease of genetic manipulation. as a result of an ancient genome duplication, zebrafish possess two paralogues for most human nmdar subunits. to evaluate the degree of conservation between human nmdar subunits and their respective zebrafish paralogues, we carried out detailed \u003cem\u003ein silico\u003c/em\u003e analyses, with an emphasis on key functional elements. to further assess the suitability of zebrafish for modeling nmdar-associated neurodevelopmental disorders, we analyzed the conservation of positions with identified missense variants.\u003c/p\u003e\u003ch2\u003eresults\u003c/h2\u003e\u003cp\u003ewe find that the human nmdar subunits are generally well conserved across zebrafish paralogs. moreover, variants classified as pathogenic and putatively pathogenic are highly conserved, reflecting the importance of key protein regions to neurotypical receptor function. positions with putatively benign and benign variants are less conserved. across nmdar domains, the transmembrane domain is most highly conserved, followed by the ligand-binding domain, which maintains conservation of amino acids that participate in the binding of ligands. the n-terminal domain is less well conserved but aligned homology models show high degrees of structural similarity. the c-terminal domain is the most poorly conserved region across zebrafish paralogs, but certain key regions that undergo phosphorylation, palmitoylation, and ubiquitylation as well as protein-binding motifs are better conserved.\u003c/p\u003e\u003ch2\u003econclusions\u003c/h2\u003e\u003cp\u003eour findings highlight a strong conservation of human nmdar subunits in zebrafish, with some exceptions. the ligand-binding domain, the transmembrane domain forming the ion channel and the short polypeptide linkers that connect them are highly conserved. the n- and c-terminal domains are less conserved but functional motifs in general, except for the zn\u003csup\u003e2+\u003c/sup\u003e binding site in glun2a paralogues, are more highly conserved relative to the entire domain. overall, our findings support the utility of zebrafish as a model for studying neurodevelopment and disease mechanisms and provide a template for rigorously considering the relationship between human and zebrafish paralogues.\u003c/p\u003e","manuscripttitle":"conservation of human nmda receptor subunits and disease variants in zebrafish","msid":"","msnumber":"","nondraftversions":[{"code":1,"date":"2025-07-30 18:25:39","doi":"10.21203/rs.3.rs-7151578/v1","editorialevents":[{"type":"communitycomments","content":0},{"type":"decision","content":"revision requested","date":"2025-09-24t19:23:39+00:00","index":"","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-09-19t21:34:21+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"269262391104669611722251259561774961128","date":"2025-09-19t14:02:55+00:00","index":"hide","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-09-09t17:52:57+00:00","index":"hide","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-08-18t16:11:58+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"201191728820772676273599414131783545114","date":"2025-08-04t10:39:46+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"230034227985411933590242857371107944843","date":"2025-07-28t13:05:21+00:00","index":"hide","fulltext":""},{"type":"reviewersinvited","content":"","date":"2025-07-28t07:38:37+00:00","index":"","fulltext":""},{"type":"editorassigned","content":"","date":"2025-07-28t07:10:47+00:00","index":"","fulltext":""},{"type":"editorinvited","content":"","date":"2025-07-23t18:56:26+00:00","index":"","fulltext":""},{"type":"checkscomplete","content":"","date":"2025-07-22t16:36:43+00:00","index":"","fulltext":""},{"type":"submitted","content":"bmc genomics","date":"2025-07-22t16:32:19+00:00","index":"","fulltext":""}],"status":"published","journal":{"display":true,"email":"info@researchsquare.com","identity":"bmc-genomics","isnaturejournal":false,"hasqc":true,"allowdirectsubmit":false,"externalidentity":"gics","sidebox":"learn more about [bmc genomics](http://bmcgenomics.biomedcentral.com/)","snpcode":"","submissionurl":"https://www.editorialmanager.com/gics","title":"bmc genomics","twitterhandle":"#bmcgenomics","acdcenabled":true,"dfaenabled":false,"editorialsystem":"em","reportingportfolio":"bmc series","inreviewenabled":true,"inreviewrevisionsenabled":true}}],"origin":"","owneridentity":"2b85bbad-8f38-4c35-89b3-5949e6b6d3c0","owner":[],"posteddate":"july 30th, 2025","published":true,"recenteditorialevents":[],"rejectedjournal":[],"revision":"","amendment":"","status":"under-review","subjectareas":[],"tags":[],"updatedat":"2025-10-14t16:08:28+00:00","versionofrecord":[],"versioncreatedat":"2025-07-30 18:25:39","video":"","vordoi":"","vordoiurl":"","workflowstages":[]},"version":"v1","identity":"rs-7151578","journalconfig":"researchsquare"},"__n_ssp":true},"page":"/article/[identity]/[[...version]]","query":{"identity":"rs-7151578","version":["v1"]},"buildid":"idgtm6tzxtmdq-mqt1zbo","isfallback":false,"dynamicids":[87228],"gssp":true,"scriptloader":[]}

## Methods
nmdar sequences and sequence alignments amino acid sequences for human genes are (uniprot with protein ids): glun1 (q05586-5; ntd splice isoform, nr1-3), glun2a (q12879-1), glun2b (q13224), glun2c (q14957), glun2d (o15399), glun3a (q8tcu5), and glun3b (o60391). for mouse subunits: mglun1 (p35438), mglun2a (p35436), mglun2b (q01097), mglun2c (q01098), mglun2d (q03391), mglun2a (a2air5), and mglun3b (q91zu9). for zebrafish paralogs: zglun1a (f1r366), zglun1b (q6zm67), zglun2aa (a0a8m9pxd6), zglun2ab (f1qde5), zglun2ba (a0a8m9q0i1), zglun2bb (a0a8m1nh89), zglun2ca (e7fh62), zglun2cb (a0a8m3b7k5), zglun2da (i3ni77), zglun2db (e7f3z4), zglun3a (a0a8m3awh5), zglun3ba (a0a8m9p9f6), and zglun3bb (a0a8m6z151). breakpoints for nmdar subunit domains and subdomains are based on structures 6whr (rat glun1-glun2b) and 7eu7 (human glun1-glun2a) ( 49 , 50 ). human and zebrafish protein sequences were aligned and analyzed for amino acid identity and similarity using align sequences protein blast (blastp), which is publicly available through the national center for biotechnology information (ncbi) or emboss needle pairwise sequence alignment (psa). for each nmdar subunit, the zebrafish paralogs were individually aligned to the human protein. amino acid similarity is defined based on the blosum 62 substitution matrix. for the ntd, lbd, and tmd, these alignments were generally straightforward as the sequences do not diverge greatly in length. in contrast, the ctd is quite large, with the zebrafish sequences generally larger than human. for these alignments, we entered the entire sequence into the alignment tool and used the given result, though this led to a fragmentation of the ctd such that certain portions of the sequence were aligned while others were not. we used the resulting domain-specific conservation values to calculate the conservation of the total protein. variants analysis human nmdar variants were identified using clinvar and gnomad (table 2 ). clinvar . results for each gene were downloaded on september 28th, 2023, using clinvar&#x27;s bulk download option ( https://www.ncbi.nlm.nih.gov/clinvar/ ). genomic variants that did not result in protein coding changes were removed from the analysis. in the case of grin3a , protein coding changes in the ppp3r2 gene, a single exon gene whose coding sequence resides within the genomic region of grin3a and is downloaded alongside grin3a due to this, was also removed. in cases where multiple protein coding annotations exist for the same variant, only the annotation fitting the canonical transcript was retained. gnomad : genes were searched for by name on gnomad v4.0.0 ( https://gnomad.broadinstitute.org/ ) and missense, insertion, deletion, and loss-of-function variant information was exported on september 28th, 2023 as comma separated value (csv) files for each gene. hgvs annotation (protein change) was converted to letter-number-letter annotation. gene variants from clinvar and gnomad were merged using r studio 2023.06.1 build 524 based on the letter-number-letter annotation. the resulting merged file was checked for redundant information or variant duplicates, which were removed. missense variants were extracted and grouped by pathogenicity as pathogenic, putatively pathogenic, complex, putatively benign, and benign variants based on the criteria in table 2 . assessment of conservation of positions with identified human variants was performed using the generated sequence alignments. nmdar domain-specific analysis to further refine the relationship between human and zebrafish nmdar proteins, we analyzed the tmd, lbd-tmd linkers, lbd, ntd, and ctd of glun1, glun2a, glun2b, and glun2d for conservation of key structural and functional features. transmembrane domain (tmd) and ligand-binding domain-transmembrane domain (lbd-tmd) linkers. key regions involved in channel gating, mg 2+ block, and ca 2+ flux were identified as described in the literature and analyzed for conservation ( 51 – 57 ). ligand-binding domain (lbd) . key regions and specific amino acids involved in van-der-waals interactions with the α-carbon of the ligand, interactions with the α-amino group, electrostatic interactions with the α-carboxyl group, and interactions with the amino acid side chain were identified as described in ramos-vicente et. al. 2021 and analyzed for conservation ( 47 ). n-terminal domain (ntd) . key amino acid positions required for the binding of zinc and polyamines were identified via a literature search and their conservation assessed ( 2 , 58 , 59 ). we also examined the conservation of putative n-linked glycosylation sites ( 60 ). to generate homology models of the zebrafish ntds, we used the template function in swiss-model ( 61 , 62 ). as reference structures, we used rodent ntd structures 5tq0 (glun1-glun2a in the presence of edta, no bound zn 2+ for glun1 and glun2a), 5tpz (glun1-glun2b apo state for glun1 and glun2b), and 5tpw (glun1-glun2a with zn 2+ in complex with glun2a) ( 58 ). sequences for human (hglun1, hglun2a, and hglun2b) or zebrafish (zglun1a, zglun1b, glun2aa, glun2ab, glun2ba, glun2bb) ntds were entered individually into swiss-model to generate homology models. the subsequent models were then loaded into pymole (human and zebrafish paralog) and aligned. the resultant root mean square deviation (rmsd) is reported. carboxy-terminal domain (ctd) . ctd-specific conservation in zebrafish was assessed for the glun1, glun2a, and glun2b subunits. glun2d was excluded from this analysis due to the poor conservation of the ctd in these paralogs relative to human. to assess functional relevance, we focused on post-translational modifications (ptms), specifically phosphorylation sites (serine, threonine, tyrosine), ubiquitylation sites (lysine), and palmitoylation sites (cysteine), using a combination of experimentally validated data from phosphositeplus (psp) and published literature ( 2 , 11 ). phosphorylation sites with identified kinases or well-characterized functional roles were classified as ‘kinase-regulated phosphorylation sites.’ the remaining phosphorylation sites identified in psp were defined as ‘phosphoproteomic-identified sites.’ note, sites in psp (based on rodents) that did not have a phosphorylation site in humans were excluded from our analysis. in addition, known linear protein-binding motifs were annotated and analyzed for conservation ( 2 , 11 ). to assess conservation of protein-binding motifs in the ctds, we categorized sites as either short linear motifs (slims) or broader interaction ‘regions’. slims were defined as experimentally validated, compact sequence elements (typically &lt; 15 residues) that mediate specific and direct interactions with known protein partners. broader regions were defined as extended sequences (&gt; 15 residues) shown to directly bind proteins in biochemical or structural studies but lacking a discrete consensus sequence. all motifs were curated from published literature where discrete sequences were experimentally validated per motif ( 11 , 63 – 80 ), and sequence conservation was examined. results our goal is to compare the structural similarity of zebrafish nmdar (zglun) paralogues to human nmdars (glun), especially regarding human variants. since mice are a common animal model ( 81 ), as a reference-point for nmdar gene conservation, we initially made global comparisons of both mouse and zebrafish nmdar subunits to human. global comparison of mouse and zebrafish orthologues to human nmdars nmdars are heterotetramers composed of two obligate glycine-binding glun1 subunits and two glutamate-binding glun2 subunits or a combination of glun2 and glycine-binding glun3 subunits ( 1 , 2 ) (fig. 1 a,b). glycine and glutamate act as co-agonists for glun2-containing receptors, with binding of both required for receptor activation, while glun3-containing receptors are activated by glycine binding. as an initial assessment of the structural similarity between nmdar orthologues, we made global sequence alignments using the ncbi blastp or emboss psa tools. not surprisingly, given evolutionary proximity, the mouse subunits are highly conserved relative to human: glun1 (97% identity &amp; similarity), glun2a (95% identity; 98% similarity), glun2b (98% identity; 99% similarity), glun2c (88% identity; 90% similarity), glun2d (97% identity &amp; similarity), glun3a (92% identity; 95% similarity), and glun3b (77% identity; 83% similarity) (table 1 ). on the other hand, the zebrafish paralogues show reduced conservation. the obligatory glun1 subunits are the most highly conserved, both in terms of identity (≥ 84%) and similarity (≥ 89%) (table 1 ). the other subunits show various degrees of identity and similarity: glun2a (≥ 58% identity; ≥79% similarity), glun2b (≥ 65% identity; ≥73% similarity), glun2c (≥ 50% identity; ≥64% similarity), glun2d (≥ 43% identity; ≥54% similarity), glun3a (63% identity; 78% similarity), and glun3b (≥ 54% identity, ≥ 62% similarity) (table 1 ). these subunits, thus, retain ≥ 50% identity with respect to the human protein, except for zglun2da and zglun2db, which only share 43% and 46% sequence identity, respectively (table 1 ). with respect to the zebrafish paralog pairs, there are instances in which one paralog is better conserved than the other. the glun1 paralogs are comparably conserved (table 1 ). however, the zglun2aa paralog is more highly conserved than zglun2ab while the zglun2bb shows increased conservation relative to zglun2ba (table 1 ). the glun2c, glun2d, and glun3b paralogs are conserved to similar extents (table 1 ). nevertheless, despite lower conservation relative to mouse, zebrafish offer many technical advantages (see background). in addition, what is critical is the conservation of specific structural and functional domains and subdomains. table 1 overall protein conservation in mouse and zebrafish relative to human human mouse zebrafish paralogue a zebrafish paralogue b protein length id. sim. protein length id. sim. protein length id. sim. protein length n1 938 97% 97% 938 84% 89% 966 (-28) 86% 90% 937 (-1) n2a 1464 95% 98% 1464 68% 89% 1460 (+ 6) 58% 79% 1445 (-9) n2b 1484 98% 99% 1482 (-2) 65% 73% 1417 (-31) 67% 81% 1770 (+ 322) n2c 1233 88% 90% 1239 (+ 6) 50% 64% 1400 (-176) 59% 79% 1328 (+ 95) n2d 1336 97% 97% 1323 (-13) 43% 54% 1916 (+ 580) 46% 57% 1676 (+ 340) n3a 1115 92% 95% 1135 (+ 20) 63% 78% 1111 (-4) n3b 1043 77% 83% 1003 (-40) 54% 62% 1118 (+ 75) 62% 68% 1114 (+ 71) numbers indicate the percent of mouse or zebrafish paralog conservation with respect to the human protein – ‘id.’ is identity and ‘sim.’ is similarity. the zebrafish a and b paralogs are indicated for each protein. glun3a has only a single paralog in zebrafish. for the mouse and zebrafish protein length, numbers in parentheses indicate the difference in amino acid number relative to human. conservation of specific structural elements in zebrafish paralogues for more rigorous comparisons, we examined the conservation within individual subunit domains and subdomains (fig. 1 ). the structure of nmdars, like all iglurs, consists of 4 domains that are intrinsic to each individual subunit (fig. 1 a,b): the extracellular ntd and lbd; the tmd forming the ion channel as well as the linkers that connect the lbd to the tmd, the lbd-tmd linkers; and the intracellular ctd ( 2 , 53 ). within individual subunits, the lbd is formed by two disparate sequences, s1 and s2, that fold up into the d1 and d2 lobes in the 3-dimensional structure (83, 84). in general, d1 is made up primarily of s1 and d2 of s2, but there is considerable swapping between lobes. the tmd is formed by three transmembrane segments, m1, m3, and m4, and an m2-pore loop. the lbd-tmd linkers are referred to as either s1-m1, m3-s2, and s2-m4 (linear sequence) or d2-m1, m3-d2, and d1-m4 (3-dimensional structure). this distinction is important functionally since d1 and d2 undergo different movements that are translated differently to the tmd ( 85 , 86 ). nevertheless, we will use the s1-m1, m3-s2, and s2-m4 designation for ease of comparison to structures. we began by dividing the zebrafish paralogues into their respective domains and subdomains based on our breakpoints (see methods). interestingly, the amino acid sequences surrounding these breakpoints were well conserved in zebrafish – the majority were identical (not shown). this allowed for ease of division of the zebrafish proteins into their respective domains and subdomains. as an initial assessment of the structural similarity between human and zebrafish nmdars, we made detailed sequence alignments using the ncbi blastp or emboss psa tools (table 1 , fig. 1 ). in terms of length of domains, the zebrafish protein sequence for glun1, glun2, and glun3 subunits was identical to the human sequence for the tmd (m1, m1-m2, m2, m2-m3, m3, &amp; m4) and lbd-tmd linkers (s1-m1, m3-s2, &amp; s2-m4) (fig. 1 c). the lbd (s1 &amp; s2) was also quite similar in length, with no changes in the length of s2 across the paralogs and only slight increases or decreases in amino acid number in the s1 of zglun1a, zglun1b, zglun2ab, zglun2ca, zglun2ca, zglun2da, and zglun2db (fig. 1 c). the ntd and ctd were less conserved in terms of size, wherein almost every zebrafish paralog has a different number of amino acids compared to the human. this is especially notable for the glun2b ntd which either lacks 130 amino acids (zglun2ba) or has an additional 74 amino acids (zglun2bb) as well as for the ctd where, for example, zglun2da and zglun2db have a ctd that is more than twice as long as that in the human protein (fig. 1 c). table 2 variant designation glun designation clinvar gnomad 1 2a 2b 2c 2d 3a 3b pathogenic “pathogenic” absent 49 (5%) 58 (4%) 81 (5%) 0 12 (&lt; 1%) 0 0 putatively pathogenic “uncertain” absent 123 (13%) 214 (15%) 182 (12%) 4 (&lt; 1%) 111 (8%) 5 (&lt; 1%) 8 (&lt; 1%) complex “pathogenic” present 3 (&lt; 1%) 5 (&lt; 1%) 6 (&lt; 1%) 0 1 (&lt; 1%) 0 0 putatively benign “uncertain” present 80 (9%) 320 (22%) 199 (13%) 31 (3%) 223 (17%) 30 (3%) 68 (7%) benign “benign” or absent absent/ present or present 401 (43%) 813 (55%) 672 (45%) 945 (77%) 863 (27%) 59 (5%) 945 (91%) number of positions with missense variants that have been identified in each subunit per designation. percentages report the proportion of the total protein amino acids at which such variants have been identified. we used these sequence alignments to assess the degree of amino acid identity and similarity between domains and subdomains (fig. 1 d,e). the obligatory glun1 subunit showed the highest degree of conservation. the tmd and lbd-tmd linkers are 100% conserved. the lbd is also well conserved with s1 and s2 showing ≥ 92% identity and ≥ 93% similarity. the ntd is somewhat less conserved (≥ 80% identity, ≥ 88% similarity), as is the ctd (≥ 53% identity; ≥58% similarity). this trend of conservation across nmdar subunits – the lbd and tmd are most highly conserved and the ntd and ctd less so – is maintained in the glun2 and glun3 subunits (fig. 1 d,e). generally, the glun2 subunits maintain a higher sequence identity than the glun3 subunits. overall, this sequence analysis suggests that the lbd and tmd are highly conserved whereas the ntd and ctd show less conservation. categorizing the pathogenicity of missense variants nmdars with missense, frameshift, and nonsense variants have been identified in pediatric patients with neurodevelopmental disorders ( 21 , 22 , 87 ). we therefore wanted to assess the extent to which variants are conserved in zebrafish to aid in guiding future studies. for this analysis, we focused solely on missense variants. while nonsense and frameshift mutation can be modelled in various ways, such as by mutating a portion of the protein to induce a loss-of-function, the conservation of the specific protein sequence is somewhat less relevant. in contrast, the single amino acid change associated with missense variants is critical to dissecting how such variants alter receptor function and lead to disease phenotypes. using publicly available datasets of nmdar variants from clinvar and gnomad, we identified missense variants in all nmdar subunits (see methods). we first categorized these variants with regards to their pathogenicity (table 2 ). clinvar reports variants that have been identified in patients with symptomatic disease ( 88 – 90 ), while gnomad reports variants identified in the healthy population ( 91 ). thus, in general, we define variants present in clinvar but absent in gnomad as pathogenic while those with the opposite reporting as benign. with this approach, pathogenic and putatively pathogenic variants were identified in glun1, glun2a, glun2b, and glun2d while minimal to no such variants were identified in glun2c, glun3a, and glun3b (table 2 ). notably, these genes with known pathogenic variants are expressed early in development, contributing to developmental processes while their dysfunction often leads to disease phenotypes ( 21 , 87 ). on the other hand, glun2c, glun3a, and glun3b have no pathogenic and a small number of putatively pathogenic variants (table 2 ). we therefore focus solely on glun1, glun2a, glun2b, and glun2d for further analysis. pathogenic and putatively pathogenic variant positions are well conserved in zebrafish using the criteria in table 2 , amino acid positions with pathogenic missense variants in glun1, glun2a, glun2b, and glun2d are completely conserved in zebrafish paralogues (fig. 2 a-d). putatively pathogenic variants are also conserved in these nmdar subunits, though less so than pathogenic variants. the highest degree of conservation of putatively pathogenic variants is in the glun1 paralogues (≥ 88% identity; 93% similarity). putatively pathogenic variants in the glun2 subunits are also reasonably conserved: zglun2aa (70% identity; 92% similarity), zglun2ab (62% identity; 84% similarity), zglun2ba (65% identity; 71% similarity), zglun2bb (72% identity; 80% similarity), and zglun2da and zglundb (≥ 46% identity; ≥59% similarity) (fig. 2 e,f). in general, the highest level of conservation is in the tmd and tmd-lbd linkers, followed by the lbd, and less so in the ntd and ctd (fig. 2 g,h). benign and putatively benign variant positions show reduced conservation we carried out a comparable analysis as above for benign and putatively benign variants (table 2 ). we hypothesized that positions with more benign variants would be less well conserved than pathogenic ones. indeed, the general trend is consistent with this idea (fig. 3 ). putatively benign variants showed a reduced conservation compared to putatively pathogenic variants (putatively pathogenic vs putatively benign): zglun1a (88% vs. 78% identity; 93% vs. 88% similarity), zglun1b (89% vs. 79% identity; 93% vs. 84% similarity), zglun2aa (70% vs. 60% identity; 92% vs. 86% similarity), zglun2ab (62% vs. 48% identity; 84% vs. 68% similarity), zglun2ba (65% vs. 45% identity; 71% vs. 55% similarity), zglun2bb (72% vs. 55% identity; 80% vs. 70% similarity), zglun2da (46% vs. 47% identity; 59% vs. 61% similarity), and zglun2db (47% vs. 44% identity; 61% vs. 56% similarity) (fig. 2 e,f vs fig. 3 a,b). for these putatively benign variants, the greatest extent of conservation tends to occur in the tmd and lbd-tmd linkers, followed by the lbd, and then the ntd and ctd (fig. 3 c,d). for benign variants, however, the decrease in conservation plateaus, with identity and similarity values comparable to the putatively benign variants, if not slightly higher (fig. 3 e-f). the subdomain-specific conservation trend for these variants follows that seen on the level of the whole receptor (fig. 1 ), wherein the tmd, lbd-tmd linkers, and lbd are generally more conserved and the ntd and ctd less so (fig. 3 g,h). although diverging from our prediction as we do not note a substantial decrease in the conservation of benign variants with respect to the putatively benign variants, this result reflects the overall conservation of the nmdar subunits themselves (benign vs total protein): zglun1 (78% vs. ≥84% identity; 84% vs. ≥89% similarity), zglun2aa (62% vs. 68% identity; 86% vs. 89% similarity), zglun2ab (49% vs. 58% identity; 74% vs. 79% similarity), zglun2ba (54% vs. 65% identity; 64% vs. 73% similarity), and zglun2bb (62% vs. 67% identity; 74% vs. 81% similarity), zglun2da (50% vs. 43% identity; 64% vs. 54% similarity), and zglun2db (49% vs. 46% identity; 62% vs. 57% similarity) (fig. 3 e-f, table 1 ). furthermore, this result reflects the relatively large number of benign variants in comparison to other variant groups (table 2 ). critical tmd and lbd-tmd linker components are completely conserved elements forming the ion channel – the tmd (m1, m1-m2, m2, m2-m3, m3, &amp; m4) – and the lbd-tmd linkers (s1-m1, m3-s2, &amp; s2-m4) were consistently highly conserved (fig. 1 ). nevertheless, we assayed conservation of critical functional motifs. the s1-m1 linker is a critical element for channel gating and is completely conserved (fig. s1 ) ( 56 ). the m2 subdomain contains what is known as the ‘n’ and ‘n + 1’ sites, which participate in mg 2+ block and ca 2+ permeation through the channel ( 2 , 55 ) and are conserved in zebrafish (fig. s1 , dark orange). the tmd m3 segment contains the sytanlaaf motif, which plays a key role in ion channel gating and is known to be highly conserved across species. this motif is conserved in all zebrafish glun1 and glun2 paralogs (fig. s1 , orange). lastly, the ‘drpeer’ motif in m3-s2 of glun1, an important component in the high ca 2+ permeability of nmdars ( 52 ) is conserved (fig. s1 , light orange). hence, core gating and permeation elements of the tmd and associated regions are highly conserved suggesting that basic mechanisms of ion channel gating and block/permeation are preserved. lbd elements required for agonist binding are completely conserved the glun1 lbd binds glycine or d-serine while the glun2 lbd binds glutamate or aspartate (fig. s2a,b). overall, the lbds are highly conserved in zebrafish relative to humans (fig. 1 ), as are specific amino acid side chains in s1 and s2 that are involved in binding endogenous agonists ( 47 ) (fig. s2c). these include sites that participate in van-der waals interactions with the α-carbon of the ligand (darkest red), interactions with the α-amino group (dark red), electrostatic interactions with the α-carboxyl group (red), and interactions with the amino acid side chain (light red) (fig. s2c). conservation of structure-function elements in the ntd in terms of overall sequence comparisons, the ntd shows variable degrees of conservation across nmdar subunits. the glun1 zebrafish paralogs maintain the highest degree of conservation relative to human (80% identity; ≥88% similarity) (fig. 1 ). the glun2 subunits show decreased ntd conservation (49–75% identity; 59–88% similarity) (fig. 1 ). given this reduced conservation, we assayed the conservation of key structure-function motifs in zebrafish paralogues. among its many roles, the ntd participates in interactions with ions and molecules that modulate receptor activity ( 58 , 59 ) (fig. 4 a,b). this is most notable for glun2a and to a lesser extent glun2b as they bind zinc, which inhibits receptor function by reducing channel open probability ( 58 ). in glun2a, four amino acids coordinate zinc binding: glun2a-his44, glun2a-his128, glun2a-glu266, and glun2a-asp282 ( 58 ). of these key sites, three are conserved (fig. 4 c) but in zglun2aa and zglun2ab, a key residue involved in zn 2+ binding, glun2a-his44, is not conserved (fig. 4 c). for glun2b, only two amino acid residues participate in zinc binding: glun2b-his127 and glun2b-glu284 ( 58 ). while glun2b-glu284 is conserved, the glun2b-his127 position in zglun2ba is missing due to alterations in ntd sequence length leading to gaps in the alignments, though it is similar now as a positively charged arginine instead of a histidine in zglun2bb (fig. 4 c). the ntd of glun2b also binds polyamines, which enhance receptor activity ( 59 ). nine amino acids are involved in the binding of polyamines. these sites in zglun2ba and zglun2bb are completely conserved except for one amino acid in zglun2bb that is similar but not identical, now a negatively charged aspartate instead of glutamate (fig. 4 d). most of this functional component of glun2b is, thus, conserved in zebrafish. additionally, the nmdar ntd has glycosylation sites, and their glycosylation regulates the trafficking of nmdars to the cell membrane ( 60 ). given the consensus sequence for arginine (n-) glycosylation (n- x -s/t), glun1 has seven, glun2a has three, and glun2b has three putative glycosylation sites in the ntd ( 60 ). their conservation in the zebrafish paralogues is somewhat variable: zglun1a &amp; zglun1b (5 of 7 sites), zglun2aa (3 of 3 sites), zglun2ab (2 of 3 sites), zglun2ba (2 of 3 sites), and zglun2bb (3 of 3 sites) (fig. 4 e). to the best of our knowledge, specific glycosylation sites in glun2d have yet to be identified. of note, the glun1 glycosylation sites glun1-n203 and glun1-n368 are necessary for receptor surface expression ( 60 ), and these sites are conserved in the zebrafish paralogues. while the critical functional features of the lbd were completely conserved in zebrafish relative to human nmdars, the conservation of key regions in the ntd occasionally diverged, most notably that for zn 2+ binding. to further explore the structural composition of the glun1, glun2a, and glun2b ntds, we used swiss-model to generate homology models for the human and zebrafish nmdar subunits, which we subsequently aligned in pymole to assess conservation at the level of the three-dimensional structure (see methods) (fig. 4 f). in general, this analysis revealed a high degree of similarity between the human ntd and zebrafish paralogs. the zebrafish glun1 ntd models display similar overall topology to human: glun1 vs zglun1a, reference structure 5tp0 (rmsd = 0.071); glun1 vs zglun1a, reference structure 5tpz (rmsd = 0.085); glun1 vs zglun1b, reference structure 5tp0 (rmsd = 0.094); and glun1 vs zglun1b, reference structure 5tpz (rmsd = 0.097) (table s3). the glun2a and glun2b subunits also reflect a high degree of structural similarity: glun2a vs zglun2aa, reference structure 5tp0 (rmsd = 0.085); glun2a vs zglun2ab, reference structure 5tp0 (rmsd = 0.095); glun2a vs zglun2ab, reference structure 5tpw (rmsd = 0.111); glun2b vs zglun2ba, reference structure 5tpz (rmsd = 0.028); and glun2b vs glun2bb, reference structure 5tpz (rmsd = 0.044) (table s3). the only exception here is the structural comparison of glun2a and zglun2aa using reference structure 5tpw, in which the ntd is interacting with zinc (rmsd = 2.149), suggesting that the structural components of zglun2aa may not effectively complex with zinc. interestingly, zglun2ba is &gt; 100 amino acids shorter and zglun2bb is &gt; 70 amino acids longer than human glun2b. though the zglun2ba model is missing an α-helix relative to the human protein, the zglun2bb modelled structure aligns well with its human counterpart, suggesting that the zebrafish sequences maintain a large degree of structural integrity. conservation of post-translational modification (ptm) sites in the ctd the intracellular ctd is a key regulatory domain of nmdar cell biology and function and shows distinct subunit-specific elements ( 11 ). for our analyses, we focused on the glun1-1 ctd splice variant, which is the longest variant containing the c0-c1-c2 cassettes ( 11 ). across the nmdar subunits, the zebrafish ctds are the most poorly conserved (fig. 1 ). with a sequence identity generally around 50–60% for the glun1, glun2a, and glun2b zebrafish paralogs, with some exceptions: zglun1a (53% identity; 58% similarity), zglun1b (54% identity; 59% similarity), zglun2aa (57% identity; 86% identity), zglun2ab (38% identity; 65% similarity), zglun2ba (57% identity, 66% similarity), and zglun2bb (60% identity, 71% similarity) while that of glun2d is much lower (≥ 18% identity, ≥ 28% similarity) (fig. 1 ). given these deviations in protein sequences, we first examined the conservation of ctd regions critical for post-translational modifications (ptms), excluding glun2d due its poor conservation in zebrafish. phosphorylation of nmdar subunit ctds confers key functional roles to the receptor ( 2 , 11 ). initially, we examined experimentally validated sites at which phosphorylation occurs by known kinases and calculated the percent conservation, reported in identity and similarity. similarity, here, is defined as a site at the homologous position that can be phosphorylated (e.g., s, t, or y), if not identical. the glun1 ctd contains phosphorylation sites for src kinase (src), protein kinase c (pkc), and protein kinase a (pka), (fig. 5 a, top ) ( 11 ). note specific sites have also been identified that are also targeted by the serine/threonine protein phosphatase 2b (pp2b). these ptm sites are perfectly conserved in zglun1a, while three of five are conserved in zglun1b (60% identity &amp; similarity) (fig. 5 b). the glun2a ctd is phosphorylated by src, pka, dual specificity tyrosine-phosphorylation-regulated kinase 1a (dyrk1), cyclin-dependent kinase 5 (cdk5), pkc, and calcium/calmodulin-dependent protein kinase iiα (camkiiα) (fig. 5 a, middle ) ( 11 ): zglun2aa (79% identity &amp; similarity) and zglun2ab (64% identity, 71% similarity) (fig. 5 b). the ctd of glun2b is phosphorylated by src, camkiiα, cdk5, pka, pkc, death-associated protein kinase 1 (dapk1), proto-oncogene tyrosine-protein kinase fyn (fyn), and casein kinase ii (ck2) (fig. 5 a, bottom ) ( 11 ). glun2b also contains sites for the protein tyrosine phosphatase non-receptor 11 (ptpn11, or shp2) and protein phosphatase 1 (pp1). the conservation of these ptms is slightly higher than that in glun2a: zglun2ba (73% identity &amp; similarity) and zglun2bb (82% identity, 100% similarity) (fig. 5 b). in addition to these kinase-regulated sites, we also examined the conservation of other known phosphorylation sites in these subunits identified by phosphoproteomics (psp) (see methods). the conservation of these sites in the zebrafish paralogs was generally lower than that of the kinase-regulated sites: glun1 (67% identity, 100% similarity), zglun2aa (51% identity, 66% similarity), zglun2ab (34% identity, 57% similarity), zglun2ba (60% identity &amp; similarity), and zglun2bb (67% identity, 70% similarity) (fig. 5 c). we next examined the conservation of ctd palmitoylation and ubiquitylation sites. glun1 has a single ubiquitylation site (lys860) (fig. 6 a, top ) ( 11 ), and it is conserved in the zebrafish paralogs (fig. 6 c). glun2a contains seven palmitoylation sites (fig. 6 a, middle ) ( 11 ), which are likewise completely conserved in zebrafish (fig. 6 b). the glun2a ctd also has four ubiquitylation sites (fig. 6 a, middle) ( 11 ), of which three are conserved in zglun2aa (75%) and one in zglun2ab (25%) (fig. 6 c). with the same approach for glun2b, we identified eight palmitoylation and six ubiquitylation sites (fig. 6 a, bottom ) ( 11 ). here, all palmitoylation sites are conserved in zebrafish while five of six ubiquitylation sites (83%) are conserved in zglun2ba and all six ubiquitylation sites (100%) are conserved in zglun2bb (fig. 5 b,c). in summary, the global conservation of ptm sites is as follows: zglun1a and zglun1b (89% identity, 100% similarity), zglun2aa (62% identity, 72% similarity), zglun2ab (45% identity, 62% similarity), zglun2ba (66% identity, 67% similarity), and zglun2bb (75% identity, 78% similarity). conservation of protein-binding motifs (pbm) in the ctd in addition to numerous ptm sites, a variety of protein-protein interactions are critical to the role of the ctd in receptor cell biology (surface expression, distribution) and function ( 11 , 63 ). these protein-binding elements include either defined short linear motifs (slims) or broader regions known to serve as protein docking sites. the ctd of glun1 contains a slim known as the kkk/rrr er retention motif, which is conserved in zebrafish ( 66 ) (fig. 7 a,b). it, likewise, has binding regions for calmodulin (cam)/calcium/calmodulin-dependent protein kinase ii (camkii) and α-actinin 2 ( 64 , 65 , 67 , 78 ), which is conserved and yotiao, also known as a-kinase anchoring protein 9 (akap9) ( 67 ), which is generally well conserved: zglun1a (90% identity, 98% similarity) and zglun1b (90% identity, 98% similarity) (fig. 8 a,b). the glun2 subunit ctds contain a variety of pbms. for glun2a, the conservation of slims is: the yxxø motif, where y is tyrosine, x is any amino acid, and ø is a bulky hydrophobic residue like leucine, isoleucine, phenylalanine, methionine, or valine ( 73 ) – both zglun2aa and zglun2ab show 100% identity &amp; similarity; the guanine-nucleotide exchange factor brag2 ( 70 ) – zglun2aa (80% identity &amp; similarity) and zglun2ab (40% identity, 60% similarity); postsynaptic density protein 95 (psd-95) ( 72 ) – zglun2aa (50% identity; 63% similarity) and zglun2ab (38% identity, 63% similarity); and the pdz domain that interacts with the proteins psd-95, discs large (dlg), and zonula occludens-1 (zo-1) ( 11 ) – zglun2aa and zglun2ab (100% identity &amp; similarity) (fig. 7 a,c). interacting regions in glun2a include: ring finger protein 10 (rnf10) ( 70 ) – zglun2aa (51% identity, 72% similarity) and zglun2ab (32% identity, 66% similarity); rabphilin 3a (rph3a) ( 68 ) – zglun2aa (71% identity, 80% similarity) and zglun2ab (49% identity, 71% similarity); flotillin-1 (flot-1) ( 71 ) – zglun2aa (61% identity, 81% similarity) and zglun2ab (31% identity, 52% similarity); and c-terminal binding protein 1 (ctbp1) ( 69 ) – zglun2aa (51% identity, 85% similarity) and zglun2ab (36% identity, 63% similarity) (fig. 8 a,c, s3a,b). for glun2b, the conservation of slims is: the guanine-nucleotide exchange factor brag1 ( 76 ) – zglun2ba (60% identity; 80% similarity) and zglun2bb (60% identity, 100% similarity); psd-95 ( 72 ) – zglun2ba (56% identity &amp; similarity) and zglun2bb (33% identity, 56% similarity); camkii and death-associated protein kinase 1 (dapk1) ( 11 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (92% identity, 100% similarity); synapse-associated protein 102 (sap102) ( 75 ) – zglun2ba (0% identity &amp; similarity) and zglun2bb (50% identity &amp; similarity); the adaptor complex ap2 ( 74 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (100% identity &amp; similarity); and pdz ( 11 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (100% identity &amp; similarity) (fig. 7 a,d). the ctd interaction regions for glun2b are conserved as follows: receptor for activated c kinase 1 (rack1) ( 77 ) – zglun2ba (0% identity, 7% similarity) and zglun2bb (13% identity, 20% similarity); flot-1 ( 71 ) – zglun2ba (23% identity; 47% similarity) and zglun2bb (35% identity, 68% similarity); spectrin ( 79 ) – zglun2ba (40% identity; 59% similarity) and zglun2bb (35% identity, 62% similarity); α-actinin 2 ( 78 ) – zglun2ba (47% identity; 67% similarity) and zglun2bb (51% identity, 76% similarity); and ras protein-specific guanine nucleotide-releasing factor 1 (rasgrf1) ( 80 ) – zglun2ba (59% identity; 74% similarity) and zglun2bb (60% identity, 78% similarity) (fig. 8 a,d, s3a,c). when compared globally across subunits, slims always show greater conservation than regions. this conservation is always greatest in the zebrafish glun1 paralogs, followed by the glun2b and then glun2a paralogs. for the analyzed slims, conservation grouped across each subunit is as follows: zglun1a and zglun1b (100% identity &amp; similarity), zglun2aa (76% identity, 81% similarity), zglun2ab (62% identity, 76% similarity), zglun2ba (78% identity, 81% similarity), and zglun2bb (72% identity, 86% similarity). the regions show the following degrees of conservation: zglun1a and zglun1b (94% identity, 98% similarity), zglun2aa (55% identity, 81% similarity), zglun2ab (36% identity, 62% similarity), zglun2ba (59% identity, 74% similarity), and zglun2bb (60% identity, 78% similarity). discussion here, we compared the conservation of the zebrafish nmdar subunit paralogues, including key structure-function motifs, to their human counterparts. we find that the core gating machinery – the lbd, the lbd-tmd linkers and the tmd – are highly conserved both in terms of overall identity and specific functional motifs. the more peripheral domains – the ntd and ctd – show less conservation overall, though functional motifs are reasonably conserved with the notable exception of zn 2+ binding in the glun2a ntd. notably, disease-associated variants are generally conserved, highlighting that zebrafish represent a useful model to study nmdars. nmdar conservation in zebrafish across most of our examined parameters, the obligatory glun1 subunit shows the highest degree of conservation in zebrafish (fig. 1 ). generally, conservation is then greatest in the glun2(a-d) followed by the glun3(a-b) subunits. within each subunit, sequence conservation is consistently highest in the tmd and lbd-tmd linkers, followed by the lbd. conservation is generally poorer in the ntd and even worse in the ctd (fig. 1 ). a key question with regards to this work is defining what constitutes as a ‘well conserved’ sequence. previous studies regarding the conservation of protein structure and function suggest that sequence identities between 40–70% confer functional conservation ( 92 – 94 ). they regard 50% sequence identity as the threshold below which function drastically diverges ( 93 , 94 ). our sequence conservation analyses almost always led to sequence identities &gt; 50%, except for zglun2da. nevertheless, we carried out more detailed examinations of more poorly conserved domains, e.g., the ntd and ctd, to assess the conservation of key structure-function motifs. nmdar missense variant conservation in zebrafish to date, a multitude of variants, including missense, frameshift, and nonsense mutations, have been identified in the genes encoding the nmdar subunits ( 21 , 22 , 87 ). we categorized identified missense variants by pathogenicity and analyzed the extent of conservation of the positions at which they occur. we focused only on the glun1, glun2a, glun2b, and glun2d subunits for this analysis, as this is where most disease-associated variants have been identified. generally, our results demonstrate a higher degree of variant position conservation with increasing pathogenicity (fig. 2 , 3 ). the sites of pathogenic missense variants are almost perfectly conserved (fig. 2 ). given the high conservation of the amino acid positions at which they occur, these protein regions are likely most essential for receptor function. across our categories of variants, with the exclusion of complex variants, the proportion of the total protein at which missense variants have been identified is lowest in the pathogenic group (table 2 ). it is possible that additional variants that might be categorized as pathogenic are so devastating to development as to be embryonic lethal. this reflects the notion that more integral regions of the receptor are less tolerant to variants ( 95 ). zebrafish, nevertheless, emerge as a useful model for the study of pathogenic nmdar variants. putatively pathogenic variants are conserved to a lesser extent than their pathogenic counterparts (fig. 2 ). according to our designations, these variants have the potential to cause disease, having been identified in patients, but are currently classified as having uncertain significance. thus, it aligns that these variants are comparatively less well conserved – they are likely occurring in less critical regions of the receptor and are likewise less likely to cause disease phenotypes. our subsequent investigation of putatively benign and benign variants revealed a further decrease in variant position conservation. this finding further supported our prediction that variants more highly associated with disease occur at positions with higher degrees of conservation and, thus, are more critical to protein structure and function. the conservation of positions with putatively benign and benign variants is quite comparable. for the benign variants, in particular, conservation percentages also closely reflect total protein conservation values for each paralog (fig. 3 , table 1 ). this follows from the finding that positions with identified benign variants, for most subunits, make up a substantial portion of the protein (table 2 ). nevertheless, this investigation both demonstrates the advantage of the zebrafish model in studying nmdar-associated disease and reflects the notion that the positions most essential for receptor function are those that are best conserved and more likely to induce disease when absent. overall, the trend for variant conservation is the same as that for the whole protein. conservation is always greatest in the tmd and lbd-tmd linkers, followed by the lbd, next the ntd, and last the ctd (fig. 2 , 3 ). these findings suggest that zebrafish serve as an effective model for variants, especially pathogenic and putatively pathogenic variants in both the tmd and lbd-tmd linkers. some caution, however, must be exercised when using this model system for variants in the lbd, ntd, and ctd. tmd, tmd-lbd linker, and lbd conservation our examination of key functional regions and motifs in the tmd, lbd-tmd linkers, and lbd consistently revealed complete conservation in their respective zebrafish paralogs (fig. s1 ,s2). this suggests that the basic mechanism of ion channel gating is conserved in zebrafish relative to human. as these domains and subdomains contribute to the formation of the core of the receptor – the ion channel pore allowing for ion flux – it is expected that they would be most highly conserved across species, since the major functional role of nmdars is current flux during synaptic transmission. ntd conservation the ntd is the first region in which nmdar sequences in zebrafish begin to diverge from those in human. modelling ntd variants in zebrafish must, thus, be approached with caution. for example, polyamine-binding sites in glun2b are almost completely conserved in zebrafish (fig. 4 d), so they have the potential to serve as an effective model for this functional role of nmdars. on the other hand, zinc-binding sites in glun2a and glun2b are more poorly conserved in zebrafish (fig. 4 c), so they would likely serve as a poor model for zinc-related studies in nmdars. this also aligns with our homology model finding that the zglun2aa model aligns well with glun2a when not bound to zinc, but the rmsd of the alignment is substantially poorer when glun2a interacting with zinc is used as the reference structure (table s3). ctd conservation the c-terminal domains (ctds) exhibit the lowest level of overall conservation across nmdar domains (fig. 1 ). this is consistent with the ctd’s classification as an intrinsically disordered region (idr), which lacks fixed secondary or tertiary structure and is more tolerant to evolutionary divergence ( 96 , 97 ). nonetheless, functional conservation within idrs is often maintained through the retention of post-translational modification (ptm) hotspots and short linear motifs (slims) that regulate protein interactions and intracellular signaling ( 98 ). we assessed ctd conservation in glun1, glun2a, and glun2b by analyzing experimentally validated phosphorylation, palmitoylation, and ubiquitylation sites, as well as defined protein-binding motifs. phosphorylation sites with known kinase interactions exhibited relatively high conservation, consistent with their well-characterized regulatory roles (figs. 5 a,b). in contrast, phosphorylation sites without identified kinases were less consistently retained (fig. 5 c). palmitoylation sites were fully conserved across all subunits, while ubiquitylation sites were generally retained, though conservation was notably lower in zglun2ab (fig. 5 f) and may reflect paralogue-specific divergence in degradation or trafficking pathways. protein-binding motif conservation varied by subunit. glun1 retained nearly all canonical interaction motifs, while glun2a and glun2b displayed region-specific conservation. notably well-conserved elements in the glun2 subunits included the pdz-binding motifs as well as docking regions for camkii, dapk1, and ap2 (fig. 6 ). slims overall showed higher conservation, likely due to their short, sequence-specific structure, which may place them under stronger evolutionary constraint than broader, more flexible interaction regions. despite broad sequence divergence, nmdar ctds retain a core set of conserved motifs involved in receptor trafficking, synaptic localization, and plasticity. kinase-regulated phosphorylation sites, palmitoylation sites, and pdz-binding motifs were consistently preserved. conservation of glun1 sites was always greatest in the zebrafish paralogs. among the glun2 subunits, glun2b displayed the highest overall conservation of functional elements, aligning with its essential role in neurodevelopment and synaptic signaling. these findings are especially relevant to the study of disease-associated truncating variants in the nmdar subunits. nonsense variants that introduce premature stop codons in the ctd eliminate key regulatory motifs critical for trafficking, anchoring, or signaling. when these affected regions correspond to conserved functional elements, zebrafish may serve as an appropriate in vitro and in vivo model for studying ctd-mediated disease mechanisms. conversely, for mutations in poorly conserved regions, the translational utility of this model may become limited. conclusions zebrafish represent a valuable system to study nmdar-associated neurodevelopmental disorders, but one must be judicious in choosing which variants to study in this model. our unique approach of sequence alignments per individual nmdar domains and subdomains highlights the high degree of conversation of some regions of the protein relative to others. additionally, our analysis of identified variants indicates that zebrafish are an effective model for pathogenic variants as these occur at conserved locations along the protein. generally, the utility of zebrafish is most apparent for the study of the tmd, lbd-tmd linkers, and lbd as these regions are most well conserved. on the other hand, studying the ntd and ctd in this species requires extra consideration stemming from deviations in sequence conservation. our work will enable future nmdar-related studies that can be effectively and efficiently conducted in zebrafish. abbreviations nmdar n -methyl-d-aspartate receptor ntd n-terminal domain lbd ligand-binding domain tmd transmembrane domain ctd c-terminal domain declarations ethics approval and consent to participate: not applicable. consent for publication: not applicable. availability of data and materials: the datasets used and/or analyzed during the current study are available either at osf (https://osf.io/h2zf9/) or from the corresponding author on reasonable request. competing interests: no competing interests declared. funding: this work was supported by nih grant r01ns088479 to lpw. authors’ contributions: e.r.n., c.a., j.d.z., h.i.s., and l.p.w. contributed to the conceptualization and methodology of this project. e.r.n., c.a., and j.d.z. conducted the investigation and analysis. w.r. assisted with data curation. e.r.n, c.a., and l.p.w. wrote the manuscript, which was revised with insights from j.d.z. and h.i.s. acknowledgements: the authors thank the clinvar and gnomad databases and the swiss-model server for their important contributions to this work. references paoletti p, bellone c, zhou q. nmda receptor subunit diversity: impact on receptor properties, synaptic plasticity and disease. nat rev neurosci. 2013;14(6):383-400. hansen kb, wollmuth lp, bowie d, furukawa h, menniti fs, sobolevsky ai, et al. structure, function, and pharmacology of glutamate receptor ion channels. pharmacol rev. 2021;73(4):298-487. chakraborty a, murphy s, coleman n. the role of nmda receptors in neural stem cell proliferation and differentiation. stem cells dev. 2017;26(11):798-807. nacher j, mcewen bs. the role of n-methyl-d-asparate receptors in neurogenesis. hippocampus. 2006;16(3):267-70. komuro h, rakic p. modulation of neuronal migration by nmda receptors. science. 1993;260(5104):95-7. pearce ia, cambray-deakin ma, burgoyne rd. glutamate acting on nmda receptors stimulates neurite outgrowth from cerebellar granule cells. febs lett. 1987;223(1):143-7. booker sa, wyllie dja. nmda receptor function in inhibitory neurons. neuropharmacology. 2021;196:108609. gambrill ac, barria a. nmda receptor subunit composition controls synaptogenesis and synapse stabilization. proc natl acad sci u s a. 2011;108(14):5855-60. napoli aj, laderwager s, zoodsma jd, biju b, mucollari o, schubel sk, et al. developmental loss of nmda receptors results in supernumerary forebrain neurons through delayed maturation of transit-amplifying neuroblasts. sci rep. 2024;14(1):3395. rajani v, sengar as, salter mw. tripartite signalling by nmda receptors. mol brain. 2020;13(1):23. ishchenko y, carrizales mg, koleske aj. regulation of the nmda receptor by its cytoplasmic domains: (how) is the tail wagging the dog? neuropharmacology. 2021;195:108634. mony l, paoletti p. mechanisms of nmda receptor regulation. curr opin neurobiol. 2023;83:102815. nielsen ac, anderson cl, ens c, boyce akj, thompson rj. non-ionotropic nmdar signalling activates panx1 to induce p2x4r-dependent long-term depression in the hippocampus. j physiol. 2024. barnes sa, thomazeau a, finnie psb, heinrich mj, heynen aj, komiyama nh, et al. non-ionotropic signaling through the nmda receptor glun2b carboxy-terminal domain drives dendritic spine plasticity and reverses fragile x phenotypes. cell rep. 2025;44(3):115311. platzer k, lemke jr. grin1-related neurodevelopmental disorder. genereviews((r)). 2021. strehlow v, myers ka, morgan at, scheffer ie, lemke jr. grin2a-related disorders. genereviews((r)). 2024. platzer k, lemke jr. grin2b-related neurodevelopmental disorder. genereviews((r)). 2021. platzer k, krey i, lemke jr. grin2d-related developmental and epileptic encephalopathy. genereviews((r)). 2022. zehavi y, mandel h, zehavi a, rashid ma, straussberg r, jabur b, et al. de novo grin1 mutations: an emerging cause of severe early infantile encephalopathy. eur j med genet. 2017;60(6):317-20. sabo sl, lahr jm, offer m, weekes a, sceniak mp. grin2b-related neurodevelopmental disorder: current understanding of pathophysiological mechanisms. front synaptic neurosci. 2022;14:1090865. gjerulfsen ce, krey i, klockner c, rubboli g, lemke jr, moller rs. spectrum of nmda receptor variants in neurodevelopmental disorders and epilepsy. methods mol biol. 2024;2799:1-11. amin jb, moody gr, wollmuth lp. from bedside-to-bench: what disease-associated variants are teaching us about the nmda receptor. j physiol. 2021;599(2):397-416. hu c, chen w, myers sj, yuan h, traynelis sf. human grin2b variants in neurodevelopmental disorders. j pharmacol sci. 2016;132(2):115-21. cox ja, kucenas s, voigt mm. molecular characterization and embryonic expression of the family of n-methyl-d-aspartate receptor subunit genes in the zebrafish. dev dyn. 2005;234(3):756-66. veldman mb, lin s. zebrafish as a developmental model organism for pediatric research. pediatr res. 2008;64(5):470-6. choi ty, choi ti, lee yr, choe sk, kim ch. zebrafish as an animal model for biomedical research. exp mol med. 2021;53(3):310-7. zoodsma jd, chan k, bhandiwad aa, golann dr, liu g, syed sa, et al. a model to study nmda receptors in early nervous system development. j neurosci. 2020;40(18):3631-45. zoodsma jd, keegan ej, moody gr, bhandiwad aa, napoli aj, burgess ha, et al. disruption of grin2b, an asd-associated gene, produces social deficits in zebrafish. mol autism. 2022;13(1):38. baraban sc, taylor mr, castro pa, baier h. pentylenetetrazole induced changes in zebrafish behavior, neural activity and c-fos expression. neuroscience. 2005;131(3):759-68. shams s, rihel j, ortiz jg, gerlai r. the zebrafish as a promising tool for modeling human brain disorders: a review based upon an ibns symposium. neurosci biobehav rev. 2018;85:176-90. de abreu ms, genario r, giacomini a, demin ka, lakstygal am, amstislavskaya tg, et al. zebrafish as a model of neurodevelopmental disorders. neuroscience. 2020;445:3-11. weinschutz mendes h, neelakantan u, liu y, fitzpatrick se, chen t, wu w, et al. high-throughput functional analysis of autism genes in zebrafish identifies convergence in dopaminergic and neuroimmune pathways. cell rep. 2023;42(3):112243. howe k, clark md, torroja cf, torrance j, berthelot c, muffato m, et al. the zebrafish reference genome sequence and its relationship to the human genome. nature. 2013;496(7446):498-503. rihel j, schier af. behavioral screening for neuroactive drugs in zebrafish. dev neurobiol. 2012;72(3):373-85. baraban sc, dinday mt, hortopan ga. drug screening in scn1a zebrafish mutant identifies clemizole as a potential dravet syndrome treatment. nat commun. 2013;4:2410. zhang y, kecskes a, copmans d, langlois m, crawford ad, ceulemans b, et al. pharmacological characterization of an antisense knockdown zebrafish model of dravet syndrome: inhibition of epileptic seizures by the serotonin agonist fenfluramine. plos one. 2015;10(5):e0125898. dingemans ajm, jansen s, van reeuwijk j, de leeuw n, pfundt r, schuurs-hoeijmakers j, et al. prevalence of comorbidities in individuals with neurodevelopmental disorders from the aggregated phenomics data of 51,227 pediatric individuals. nat med. 2024;30(7):1994-2003. ohtsuki t, sakurai k, dou h, toru m, yamakawa-kobayashi k, arinami t. mutation analysis of the nmdar2b (grin2b) gene in schizophrenia. mol psychiatry. 2001;6(2):211-6. arnold pd, rosenberg dr, mundo e, tharmalingam s, kennedy jl, richter ma. association of a glutamate (nmda) subunit receptor gene (grin2b) with obsessive-compulsive disorder: a preliminary study. psychopharmacology (berl). 2004;174(4):530-8. saunders ja, tatard-leitman vm, suh j, billingslea en, roberts tp, siegel sj. knockout of nmda receptors in parvalbumin interneurons recreates autism-like phenotypes. autism res. 2013;6(2):69-77. gandal mj, sisti j, klook k, ortinski pi, leitman v, liang y, et al. gabab-mediated rescue of altered excitatory-inhibitory balance, gamma synchrony and behavioral deficits following constitutive nmdar-hypofunction. transl psychiatry. 2012;2(7):e142. barnby g, abbott a, sykes n, morris a, weeks de, mott r, et al. candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at grin2a and abat. am j hum genet. 2005;76(6):950-66. hung ly, margolis kg. autism spectrum disorders and the gastrointestinal tract: insights into mechanisms and clinical relevance. nat rev gastroenterol hepatol. 2024;21(3):142-63. owens ap, mathias cj, iodice v. autonomic dysfunction in autism spectrum disorder. front integr neurosci. 2021;15:787037. orefice ll. peripheral somatosensory neuron dysfunction: emerging roles in autism spectrum disorders. neuroscience. 2020;445:120-9. makris g, agorastos a, chrousos gp, pervanidou p. stress system activation in children and adolescents with autism spectrum disorder. front neurosci. 2021;15:756628. ramos-vicente d, grant sg, bayes a. metazoan evolution and diversity of glutamate receptors and their auxiliary subunits. neuropharmacology. 2021;195:108640. amores a, force a, yan yl, joly l, amemiya c, fritz a, et al. zebrafish hox clusters and vertebrate genome evolution. science. 1998;282(5394):1711-4. chou th, tajima n, romero-hernandez a, furukawa h. structural basis of functional transitions in mammalian nmda receptors. cell. 2020;182(2):357-71 e13. zhang y, ye f, zhang t, lv s, zhou l, du d, et al. structural basis of ketamine action on human nmda receptors. nature. 2021;596(7871):301-5. wollmuth lp, sakmann b. different mechanisms of ca2+ transport in nmda and ca2+-permeable ampa glutamate receptor channels. j gen physiol. 1998;112(5):623-36. watanabe j, beck c, kuner t, premkumar ls, wollmuth lp. drpeer: a motif in the extracellular vestibule conferring high ca2+ flux rates in nmda receptor channels. j neurosci. 2002;22(23):10209-16. wollmuth lp, sobolevsky ai. structure and gating of the glutamate receptor ion channel. trends neurosci. 2004;27(6):321-8. ogden kk, khatri a, traynelis sf, heldt sa. potentiation of glun2c/d nmda receptor subtypes in the amygdala facilitates the retention of fear and extinction learning in mice. neuropsychopharmacology. 2014;39(3):625-37. wollmuth lp. ion permeation in ionotropic glutamate receptors: still dynamic after all these years. curr opin physiol. 2018;2:36-41. mcdaniel mj, ogden kk, kell sa, burger pb, liotta dc, traynelis sf. nmda receptor channel gating control by the pre-m1 helix. j gen physiol. 2020;152(4). wollmuth lp, kuner t, sakmann b. adjacent asparagines in the nr2-subunit of the nmda receptor channel control the voltage-dependent block by extracellular mg2+. j physiol. 1998;506 ( pt 1)(pt 1):13-32. romero-hernandez a, simorowski n, karakas e, furukawa h. molecular basis for subtype specificity and high-affinity zinc inhibition in the glun1-glun2a nmda receptor amino-terminal domain. neuron. 2016;92(6):1324-36. mony l, zhu s, carvalho s, paoletti p. molecular basis of positive allosteric modulation of glun2b nmda receptors by polyamines. embo j. 2011;30(15):3134-46. lichnerova k, kaniakova m, park sp, skrenkova k, wang yx, petralia rs, et al. two n-glycosylation sites in the glun1 subunit are essential for releasing n-methyl-d-aspartate (nmda) receptors from the endoplasmic reticulum. j biol chem. 2015;290(30):18379-90. waterhouse a, bertoni m, bienert s, studer g, tauriello g, gumienny r, et al. swiss-model: homology modelling of protein structures and complexes. nucleic acids res. 2018;46(w1):w296-w303. bienert s, waterhouse a, de beer ta, tauriello g, studer g, bordoli l, et al. the swiss-model repository-new features and functionality. nucleic acids res. 2017;45(d1):d313-d9. warnet xl, bakke krog h, sevillano-quispe og, poulsen h, kjaergaard m. the c-terminal domains of the nmda receptor: how intrinsically disordered tails affect signalling, plasticity and disease. eur j neurosci. 2021;54(8):6713-39. kim k, saneyoshi t, hosokawa t, okamoto k, hayashi y. interplay of enzymatic and structural functions of camkii in long-term potentiation. j neurochem. 2016;139(6):959-72. bej a, ames jb. chemical shift assignments of calmodulin bound to a cytosolic domain of glun2a (residues 1004-1024) from the nmda receptor. biomol nmr assign. 2023;17(1):89-93. horak m, wenthold rj. different roles of c-terminal cassettes in the trafficking of full-length nr1 subunits to the cell surface. j biol chem. 2009;284(15):9683-91. lin jw, wyszynski m, madhavan r, sealock r, kim ju, sheng m. yotiao, a novel protein of neuromuscular junction and brain that interacts with specific splice variants of nmda receptor subunit nr1. j neurosci. 1998;18(6):2017-27. stanic j, carta m, eberini i, pelucchi s, marcello e, genazzani aa, et al. rabphilin 3a retains nmda receptors at synaptic sites through interaction with glun2a/psd-95 complex. nat commun. 2015;6:10181. cousins sl, stephenson fa. identification of c-terminal binding protein 1 as a novel nmda receptor interactor. neurochem res. 2019;44(6):1437-45. franchini l, carrano n, di luca m, gardoni f. synaptic glun2a-containing nmda receptors: from physiology to pathological synaptic plasticity. int j mol sci. 2020;21(4). swanwick cc, shapiro me, yi z, chang k, wenthold rj. nmda receptors interact with flotillin-1 and -2, lipid raft-associated proteins. febs lett. 2009;583(8):1226-30. cousins sl, stephenson fa. identification of n-methyl-d-aspartic acid (nmda) receptor subtype-specific binding sites that mediate direct interactions with scaffold protein psd-95. j biol chem. 2012;287(16):13465-76. vissel b, krupp jj, heinemann sf, westbrook gl. a use-dependent tyrosine dephosphorylation of nmda receptors is independent of ion flux. nat neurosci. 2001;4(6):587-96. lavezzari g, mccallum j, lee r, roche kw. differential binding of the ap-2 adaptor complex and psd-95 to the c-terminus of the nmda receptor subunit nr2b regulates surface expression. neuropharmacology. 2003;45(6):729-37. chen bs, gray ja, sanz-clemente a, wei z, thomas ev, nicoll ra, et al. sap102 mediates synaptic clearance of nmda receptors. cell rep. 2012;2(5):1120-8. elagabani mn, brisevac d, kintscher m, pohle j, kohr g, schmitz d, et al. subunit-selective n-methyl-d-aspartate (nmda) receptor signaling through brefeldin a-resistant arf guanine nucleotide exchange factors brag1 and brag2 during synapse maturation. j biol chem. 2016;291(17):9105-18. yaka r, thornton c, vagts aj, phamluong k, bonci a, ron d. nmda receptor function is regulated by the inhibitory scaffolding protein, rack1. proc natl acad sci u s a. 2002;99(8):5710-5. wyszynski m, lin j, rao a, nigh e, beggs ah, craig am, et al. competitive binding of alpha-actinin and calmodulin to the nmda receptor. nature. 1997;385(6615):439-42. wechsler a, teichberg vi. brain spectrin binding to the nmda receptor is regulated by phosphorylation, calcium and calmodulin. embo j. 1998;17(14):3931-9. krapivinsky g, krapivinsky l, manasian y, ivanov a, tyzio r, pellegrino c, et al. the nmda receptor is coupled to the erk pathway by a direct interaction between nr2b and rasgrf1. neuron. 2003;40(4):775-84. mukherjee p, roy s, ghosh d, nandi sk. role of animal models in biomedical research: a review. lab anim res. 2022;38(1):18. chou th, epstein m, fritzemeier rg, akins ns, paladugu s, ullman ez, et al. molecular mechanism of ligand gating and opening of nmda receptor. nature. 2024;632(8023):209-17. armstrong n, sun y, chen gq, gouaux e. structure of a glutamate-receptor ligand-binding core in complex with kainate. nature. 1998;395(6705):913-7. armstrong n, gouaux e. mechanisms for activation and antagonism of an ampa-sensitive glutamate receptor: crystal structures of the glur2 ligand binding core. neuron. 2000;28(1):165-81. kazi r, gan q, talukder i, markowitz m, salussolia cl, wollmuth lp. asynchronous movements prior to pore opening in nmda receptors. j neurosci. 2013;33(29):12052-66. amin jb, gochman a, he m, certain n, wollmuth lp. nmda receptors require multiple pre-opening gating steps for efficient synaptic activity. neuron. 2021;109(3):488-501 e4. hardingham ge, do kq. linking early-life nmdar hypofunction and oxidative stress in schizophrenia pathogenesis. nat rev neurosci. 2016;17(2):125-34. landrum mj, lee jm, riley gr, jang w, rubinstein ws, church dm, et al. clinvar: public archive of relationships among sequence variation and human phenotype. nucleic acids res. 2014;42(database issue):d980-5. landrum mj, lee jm, benson m, brown g, chao c, chitipiralla s, et al. clinvar: public archive of interpretations of clinically relevant variants. nucleic acids res. 2016;44(d1):d862-8. landrum mj, lee jm, benson m, brown gr, chao c, chitipiralla s, et al. clinvar: improving access to variant interpretations and supporting evidence. nucleic acids res. 2018;46(d1):d1062-d7. gudmundsson s, singer-berk m, watts na, phu w, goodrich jk, solomonson m, et al. variant interpretation using population databases: lessons from gnomad. hum mutat. 2022;43(8):1012-30. tian w, skolnick j. how well is enzyme function conserved as a function of pairwise sequence identity? j mol biol. 2003;333(4):863-82. addou s, rentzsch r, lee d, orengo ca. domain-based and family-specific sequence identity thresholds increase the levels of reliable protein function transfer. j mol biol. 2009;387(2):416-30. sangar v, blankenberg dj, altman n, lesk am. quantitative sequence-function relationships in proteins based on gene ontology. bmc bioinformatics. 2007;8:294. perszyk re, kristensen as, lyuboslavsky p, traynelis sf. three-dimensional missense tolerance ratio analysis. genome res. 2021;31(8):1447-61. uversky vn. intrinsically disordered proteins from a to z. int j biochem cell biol. 2011;43(8):1090-103. singleton md, eisen mb. evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. plos comput biol. 2024;20(4):e1012028. davey ne, cowan jl, shields dc, gibson tj, coldwell mj, edwards rj. slimprints: conservation-based discovery of functional motif fingerprints in intrinsically disordered protein regions. nucleic acids res. 2012;40(21):10628-41. additional declarations no competing interests reported. supplementary files zfishnmdarssupplement20250716.docx cite share download pdf status: under review version 1 posted editorial decision: revision requested 24 sep, 2025 reviews received at journal 19 sep, 2025 reviewers agreed at journal 19 sep, 2025 reviews received at journal 09 sep, 2025 reviews received at journal 18 aug, 2025 reviewers agreed at journal 04 aug, 2025 reviewers agreed at journal 28 jul, 2025 reviewers invited by journal 28 jul, 2025 editor assigned by journal 28 jul, 2025 editor invited by journal 23 jul, 2025 submission checks completed at journal 22 jul, 2025 first submitted to journal 22 jul, 2025 you are reading this latest preprint version research square lets you share your work early, gain feedback from the community, and start making changes to your manuscript prior to peer review in a journal. as a division of research square company, we’re committed to making research communication faster, fairer, and more useful. we do this by developing innovative software and high quality services for the global research community. our growing team is made up of researchers and industry professionals working together to solve the most critical problems facing scientific publishing. also discoverable on platform about our team in review editorial policies advisory board help center resources author services research quality evaluation accessibility api access rss feed manage cookie preferences © research square 2025 | issn 2693-5015 (online) privacy policy terms of service do not sell my personal information {"props":{"pageprops":{"initialdata":{"identity":"rs-7151578","acceptedtermsandconditions":true,"allowdirectsubmit":false,"archivedversions":[],"articletype":"research article","associatedpublications":[],"authors":[{"id":492782335,"identity":"6084adf8-9103-4ca8-8de5-679d2a4ea2bb","order_by":0,"name":"erica r. nebet","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"erica","middlename":"r.","lastname":"nebet","suffix":""},{"id":492782336,"identity":"68eb1705-4eac-413e-bc82-7b9943220d12","order_by":1,"name":"christieann aprea","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"christieann","middlename":"","lastname":"aprea","suffix":""},{"id":492782337,"identity":"776ee37e-7ff9-4a8a-bc43-f965303e90a8","order_by":2,"name":"josiah d. zoodsma","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"josiah","middlename":"d.","lastname":"zoodsma","suffix":""},{"id":492782338,"identity":"ef7f492d-83d8-41a3-aa81-6be2af423f07","order_by":3,"name":"william raab","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"william","middlename":"","lastname":"raab","suffix":""},{"id":492782339,"identity":"6949754f-6430-4ffc-8673-da5548d44bd6","order_by":4,"name":"howard i. sirotkin","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"howard","middlename":"i.","lastname":"sirotkin","suffix":""},{"id":492782340,"identity":"a4f8ecb1-612e-43db-8923-d325a2be05c2","order_by":5,"name":"lonnie p. wollmuth","email":"data:image/png;base64,ivborw0kggoaaaansuheugaaazaaaaayaqmaaabi0h/eaaaablbmvex///8aaabvwtn+aaaacxbiwxmaaa7eaaaoxagvkw4baaabhuleqvriiwnggawjybacphcraclymrua5ae5sdajim0dxwiada6fgaqs4yaxwiigfgyklyknblww9x578kccibgfmbf1m++po+nz5zcfe8y7454cp88bxgdv2zc18jxln0g4w5a4s5mx7tzpwrpcdcfy0o15zxqbs/y2mbvoxajfisdmirgnixfdybcww7kb2hjmphnbehi3ngdgazjwapmh0jiu34bqwv4bp5yghjyehmmwlwcb2jixaee5yyarceyihusxm3psdhtuojawjjn3tikxzm/bzsk55zc08lp3men+qlex5mdvpnbjjc1hefnmw8ck3u5iaizy8sepb8owqxkcjfc5tjdoamcokydxb/fqr8eogawjypgdabgnaum7/svdaaaaaelftksuqmcc","orcid":"","institution":"stony brook university","correspondingauthor":true,"prefix":"","firstname":"lonnie","middlename":"p.","lastname":"wollmuth","suffix":""}],"badges":[],"createdat":"2025-07-17 18:08:21","currentversioncode":1,"declarations":"","doi":"10.21203/rs.3.rs-7151578/v1","doiurl":"https://doi.org/10.21203/rs.3.rs-7151578/v1","draftversion":[],"editorialevents":[],"editorialnote":"","failedworkflow":false,"files":[{"id":87954191,"identity":"93406acd-9b5c-4bbe-b8e4-476bfc8e3591","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":0,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":48663694,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarsmain20250722.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/72a01d08b07202a8dd3432a7.docx"},{"id":87954161,"identity":"09dcee36-3c3d-4bae-a5b6-a8031a04e2f8","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"json","order_by":1,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":8698,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b.json","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/088093afe258318de15a5f12.json"},{"id":87954169,"identity":"c1d3acb0-9e25-4b8a-9228-e0e154b314b2","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":2,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":15353369,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarssupplement20250716.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a3a65bcc549c7954c295d2d1.docx"},{"id":87954163,"identity":"6fde5d88-4da9-48b5-ac47-0b0e0f6354d8","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"xml","order_by":3,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":242943,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b1enriched.xml","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/aad420877d9070a2aedac4cc.xml"},{"id":87954167,"identity":"b5c71071-9fd7-476b-9303-17dcb5b1c845","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"jpeg","order_by":4,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":7224546,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage1.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/239b3c6e0e27c5d5f3a3c485.jpeg"},{"id":87954176,"identity":"66a11160-0f4e-44e0-a6b3-26a6c526f9c1","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":5,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5495318,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage2.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/7c759429dc2c4ccf2d95e1ed.jpeg"},{"id":87954168,"identity":"e3a84d67-d0ce-4822-8529-9e1cc04acd3d","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":6,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5497046,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage3.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/5ab1e35184bc289007b868d7.jpeg"},{"id":87954408,"identity":"4452df13-a563-4564-9a2c-9a250a285ebb","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"jpeg","order_by":7,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4536836,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage4.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/34cad84600342cebbb343c8d.jpeg"},{"id":87954171,"identity":"94757fe0-c77c-46b6-8a13-76b5920bc170","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":8,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4702238,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage5.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/704be5ce4c0cc10b6f98b684.jpeg"},{"id":87954173,"identity":"bbb91acc-360b-43aa-99a5-89fbc5989777","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":9,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4378230,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage6.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/85acc94c54964dbcd4ee6caa.jpeg"},{"id":87954192,"identity":"2447fad7-ffab-4e33-9b2f-a71d2af30f9e","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":10,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5735470,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage7.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3a0654f7ccd9aa40cc5d5f69.jpeg"},{"id":87954413,"identity":"29bec889-0025-4009-9267-6438ea9c5740","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"jpeg","order_by":11,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5327046,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage8.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/c3009b0732aa42082efe8df3.jpeg"},{"id":87954188,"identity":"20ba216a-d094-494f-9894-1937509c8d81","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":12,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":222456,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage1.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/08aba5e904980f68ab32639e.png"},{"id":87954187,"identity":"d99856fc-69f6-4dda-9eeb-4cafd6cc99a0","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":13,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":175317,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage2.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a0ea7afa8616702c9ac66b10.png"},{"id":87954174,"identity":"de47bd92-e161-4fbe-a9b2-437c4b5f0573","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":14,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":140021,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage3.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/bb8abfaa1c199dae266df4dd.png"},{"id":87954412,"identity":"87f7cc6b-e10f-4300-9539-2c2a02855ddb","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"png","order_by":15,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":138717,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage4.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a6b8e16a616611723ae39dab.png"},{"id":87954190,"identity":"ad2ebfe9-5bbc-4128-8d93-eaa9d7470a57","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":16,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":96612,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage5.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/99939544fae92f971073fd51.png"},{"id":87954189,"identity":"84923e46-044f-4cd2-9f5a-b3a9fa41caab","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":17,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":72313,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage6.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/92848d9ff37daaad2b48e9ff.png"},{"id":87954755,"identity":"eaa7ae44-7e94-4117-97cb-d03c1d07f90a","added_by":"auto","created_at":"2025-07-30 18:41:44","extension":"png","order_by":18,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":83342,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage7.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/845b774d666648bfa0a844ab.png"},{"id":87954410,"identity":"13827b86-cf06-4cf3-9952-4b424318037f","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"png","order_by":19,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":117211,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage8.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/af95a59a499464e286bef54b.png"},{"id":87954411,"identity":"7741fc55-f9dc-44b6-9b97-8875d99f5f4b","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"xml","order_by":20,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":240176,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b1structuring.xml","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/dfb8648302675ebf344a2fa6.xml"},{"id":87954756,"identity":"beb85617-8838-4c99-8712-1a7d62a23eff","added_by":"auto","created_at":"2025-07-30 18:41:44","extension":"html","order_by":21,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":257708,"visible":true,"origin":"","legend":"","description":"","filename":"earlyproof.html","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/01f35be0d8432473e2775041.html"},{"id":87954160,"identity":"3920a076-09de-49c0-b764-c081048b3b14","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":1,"title":"figure 1","display":"","copyasset":false,"role":"figure","size":795350,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of human nmdar subunits in zebrafish\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a)\u003c/strong\u003e nmdars are composed of two obligatory glun1 and typically two glun2a-d subunits. the tetrameric complex consists of four domains: extracellular n-terminal (ntd) and ligand-binding (lbd) domains; a transmembrane domain (tmd); and an intracellular c-terminal domain (ctd). glun1 subunits are colored dark blue (ntd), teal (lbd), red (lbd-tmd linkers), light orange (tmd), and gray (ctd); and glun2b subunits gray. 9are (82)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b)\u003c/strong\u003e topology of an individual glun1 subunit. lbd lobes (d1 and d2) are shown; d1 is mainly comprisedof s1 and d2 of s2. the ion channel is formed by 3 transmembrane segments, m1, m3, and m4, and an intracellular m2 pore loop. lbd-tmd linkers are s1-m1 (or d2-m1), m3-s2 (or d2-m3), and s2-m4 (or d1-m4).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c)\u003c/strong\u003e nmdar subunits are separated into 13 structural subdivisions: ntd, lbd (s1 \u0026amp; s2), lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4), tmd (m1, intracellular m1-m2 linker, m2 loop, m2-m3 linker, m3, \u0026amp; m4), and ctd. amino acid length of each zebrafish subdivision is indicated. those sharing amino acid length with their human orthologues are indicated in white, while increasingly darker pink indicates those with increasingly greater amino acid differences (greater (+) or fewer (-)).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(d-e)\u003c/strong\u003e nmdar subunits are grouped as in \u003cstrong\u003e(c)\u003c/strong\u003e. homology between human and zebrafish nmdar subunit paralogues is reported in amino acid identity \u003cstrong\u003e(d)\u003c/strong\u003e or similarity \u003cstrong\u003e(e)\u003c/strong\u003e. heat map with darker green indicating greater percent of homology.\u003c/p\u003e","description":"","filename":"floatimage1.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/5b4e27c91809ed8be6a2b3b5.png"},{"id":87954407,"identity":"47d4fb26-35bf-49ca-aae6-ed196317e2be","added_by":"auto","created_at":"2025-07-30 18:33:43","extension":"png","order_by":2,"title":"figure 2","display":"","copyasset":false,"role":"figure","size":712960,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of positions with pathogenic and putatively pathogenic missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a-b) \u003c/strong\u003epercent conservation of pathogenic variant positions in zebrafish genes, as defined in table 2. conservation is reported in amino acid identity \u003cstrong\u003e(a) \u003c/strong\u003eand similarity \u003cstrong\u003e(b)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c-d)\u003c/strong\u003e conservation of pathogenic variant positions in each nmdar domain, grouped as in figure 1. homology is reported in identity \u003cstrong\u003e(c)\u003c/strong\u003eand similarity \u003cstrong\u003e(d)\u003c/strong\u003e. dashes indicate domains without reported pathogenic variants. along with the percent of conservation, the total number of identified variants in the human protein is indicated in parentheses in the ‘a’ paralog column in \u003cstrong\u003e(c)\u003c/strong\u003e.\u003c/p\u003e","description":"","filename":"floatimage2.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/d7d962b025e722b029324c59.png"},{"id":87954162,"identity":"02953530-369f-4594-b587-6d86079d6a68","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":3,"title":"figure 3","display":"","copyasset":false,"role":"figure","size":667858,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of positions with putatively benign and benign missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a-b) \u003c/strong\u003epercent conservation of putatively benign variant positions in zebrafish genes, as defined in table 2. conservation is reported in identity \u003cstrong\u003e(a) \u003c/strong\u003eand similarity \u003cstrong\u003e(b)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c-d) \u003c/strong\u003econservation of putatively benign variant positions in each nmdar domain, grouped as in figure 2. homology is reported in identity \u003cstrong\u003e(c)\u003c/strong\u003e and similarity \u003cstrong\u003e(d)\u003c/strong\u003e. dashes indicate domains without reported pathogenic variants. along with the percent of conservation, the total number of identified variants in the human protein is indicated in parentheses in the ‘a’ paralog column in \u003cstrong\u003e(c)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(e-h) \u003c/strong\u003eas in a-d but for benign variant positions.\u003c/p\u003e","description":"","filename":"floatimage3.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/069549024abcf2bb76a43e0e.png"},{"id":87954166,"identity":"f30749e8-12a9-42b8-89f5-26186b255ecb","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":4,"title":"figure 4","display":"","copyasset":false,"role":"figure","size":473605,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key functional elements and structure of the nmdar ntd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003etopology of glun2a ntd, showing r1 and r2 subdomains, bound to zinc (red). 5tpw (58)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b) \u003c/strong\u003eclose-up image of structure in \u003cstrong\u003e(a)\u003c/strong\u003ehighlighting amino acids that participate in zinc binding. his44 is demarcated as it is not conserved in the zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c) \u003c/strong\u003econservation of amino acids that participate in ntd zinc binding in glun2a and glun2b, reported in identity (dark red) and similarity (light red). sequences are shown to the right.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(d) \u003c/strong\u003econservation of glun2b ntd regions that participate in interactions with polyamines, reported in identity (dark gray) and similarity (light gray). sequences are shown to the right.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(e) \u003c/strong\u003econservation of ntd glycosylation sites, reported in identity (dark gray).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(f) \u003c/strong\u003ealigned ntd homology models of the human (blue) and zebrafish (gray) protein for glun1 vs zglun1a (reference structure, 5tp0) (\u003cem\u003eleft\u003c/em\u003e), glun2a vs zglun2aa (reference structure, 5tp0) (\u003cem\u003emiddle\u003c/em\u003e), and glun2b vs zglun2bb (reference structure, 5tpz) (\u003cem\u003eright\u003c/em\u003e). secondary structures colored both blue and gray indicate areas of structural overlap.\u003c/p\u003e","description":"","filename":"floatimage4.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3693c16c36a6ea0f931e6343.png"},{"id":87954406,"identity":"16a86e66-9b13-469f-91ca-fe2142d4ce9c","added_by":"auto","created_at":"2025-07-30 18:33:43","extension":"png","order_by":5,"title":"figure 5","display":"","copyasset":false,"role":"figure","size":427432,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key nmdar ctd phosphorylation sites\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd kinase-regulated phosphorylation sites for the glun1, glun2a and glun2b ctds. the specific phosphorylation sites and the corresponding human and zebrafish sequences are indicated. positions sharing identity are indicated in dark gray. similarity is shown in light gray; here, a residue is only considered similar if it is ‘phosphorylatable’ (e.g., contains a s, t, or y) (see methods).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-c) \u003c/strong\u003epercent conservation of kinase-regulated phosphorylation sites \u003cstrong\u003e(b)\u003c/strong\u003e and other phosphorylation sites identified by phosphoproteomics \u003cstrong\u003e(c)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray); similarity is defined in \u003cstrong\u003e(a)\u003c/strong\u003e.\u003c/p\u003e","description":"","filename":"floatimage5.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/25001bb8418306120cd3e005.png"},{"id":87954184,"identity":"a64d1f54-7d76-472b-a396-d9efcf5f86c8","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":6,"title":"figure 6","display":"","copyasset":false,"role":"figure","size":333594,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key nmdar ctd post-translational modification (ptm) sites\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd palmitoylation and ubiquitylation sites for the glun1, glun2a and glun2b ctds. the specific phosphorylation sites and the corresponding human and zebrafish sequences are indicated. positions sharing identity are indicated in dark gray.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-c) \u003c/strong\u003epercent conservation of palmitoylation \u003cstrong\u003e(b)\u003c/strong\u003e and ubiquitylation \u003cstrong\u003e(c)\u003c/strong\u003e sites, reported in identity (dark gray).\u003c/p\u003e","description":"","filename":"floatimage6.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/fac7b19510f29b178f7e3661.png"},{"id":87954182,"identity":"d09e842a-ccf2-41a7-bd3e-0935363761f9","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":7,"title":"figure 7","display":"","copyasset":false,"role":"figure","size":403546,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of nmdar short linear motif (slim) protein-binding motifs (pbms) in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd short linear motifs (slims) for the glun1, glun2a and glun2b ctds.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-d) \u003c/strong\u003epercent conservation of ctd slims in glun1 \u003cstrong\u003e(b)\u003c/strong\u003e, glun2a \u003cstrong\u003e(c)\u003c/strong\u003e, and glun2b \u003cstrong\u003e(d)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray).\u003c/p\u003e","description":"","filename":"floatimage7.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/b6119c0825c4063a5e6bb6ec.png"},{"id":87954181,"identity":"cb3e5a1b-cd98-4155-84bb-c4d2fbb40f7e","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":8,"title":"figure 8","display":"","copyasset":false,"role":"figure","size":518323,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of nmdar protein-binding motifs (pbms) regions in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd regions for the glun1, glun2a and glun2b ctds.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-d) \u003c/strong\u003epercent conservation of ctd regions in glun1 \u003cstrong\u003e(b)\u003c/strong\u003e, glun2a \u003cstrong\u003e(c)\u003c/strong\u003e, and glun2b \u003cstrong\u003e(d)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray).\u003c/p\u003e","description":"","filename":"floatimage8.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/ccea59247fac75e14ad8ad14.png"},{"id":87954866,"identity":"ae52deed-9d7c-4d30-94e3-0b4ac04fe145","added_by":"auto","created_at":"2025-07-30 18:49:46","extension":"pdf","order_by":0,"title":"","display":"","copyasset":false,"role":"manuscript-pdf","size":5866105,"visible":true,"origin":"","legend":"","description":"","filename":"manuscript.pdf","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/24ce2fd9-cdac-44be-947d-44a6c3564886.pdf"},{"id":87954175,"identity":"cf7d81ad-b94d-4e4f-95f1-e45dee844a82","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":0,"title":"","display":"","copyasset":false,"role":"supplement","size":15353369,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarssupplement20250716.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3ada87e108301cd5d027f600.docx"}],"financialinterests":"no competing interests reported.","formattedtitle":"conservation of human nmda receptor subunits and disease variants in zebrafish","fulltext":[{"header":"background","content":"\u003cp\u003erapid cell-to-cell signaling in the nervous system occurs at specialized structures called synapses. at these sites, the electrical activity of the upstream or presynaptic neuron is converted into the release of a chemical neurotransmitter. this is then converted back into an electrical signal in the postsynaptic neuron by specialized ion channels that are gated by a neurotransmitter, initiating downstream biological signaling cascades. glutamate, the primary excitatory neurotransmitter in the brain, activates a variety of receptors that are essential for synaptic transmission and plasticity. glutamate-gated ion channels play fundamental roles in signaling in both the developing and mature nervous systems (\u003cspan additionalcitationids=\"cr2 cr3 cr4 cr5 cr6 cr7 cr8\" citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e–\u003cspan citationid=\"cr9\" class=\"citationref\"\u003e9\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ethe nmda receptor (nmdar) is a prominent glutamate-gated ion channel subtype (\u003cspan citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e, \u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e). due to its distinct biophysical properties and unique ionotropic and non-ionotropic signaling mechanisms, it contributes to a wide array of nervous and non-nervous system functions (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan additionalcitationids=\"cr11 cr12 cr13\" citationid=\"cr10\" class=\"citationref\"\u003e10\u003c/span\u003e–\u003cspan citationid=\"cr14\" class=\"citationref\"\u003e14\u003c/span\u003e). numerous missense, frameshift, and nonsense mutations (or variants) have been identified in the genes that encode nmdar subunits and are associated with neurodevelopmental disorders, including autism spectrum disorders (asd), intellectual disability, developmental delay, seizures, and schizophrenia (\u003cspan additionalcitationids=\"cr16 cr17 cr18 cr19 cr20 cr21 cr22\" citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e–\u003cspan citationid=\"cr23\" class=\"citationref\"\u003e23\u003c/span\u003e). however, how these variants lead to clinical phenotypes is poorly understood.\u003c/p\u003e\u003cp\u003ezebrafish are a powerful model organism to study neurodevelopmental phenotypes and disorders, including those linked to nmdars (\u003cspan citationid=\"cr19\" class=\"citationref\"\u003e19\u003c/span\u003e, \u003cspan additionalcitationids=\"cr25 cr26 cr27 cr28 cr29 cr30 cr31\" citationid=\"cr24\" class=\"citationref\"\u003e24\u003c/span\u003e–\u003cspan citationid=\"cr32\" class=\"citationref\"\u003e32\u003c/span\u003e). zebrafish develop rapidly and share fundamental developmental and signaling pathways with humans and other vertebrates (\u003cspan citationid=\"cr33\" class=\"citationref\"\u003e33\u003c/span\u003e); they are transparent, allowing for dynamic and non-invasive observations of developmental and cellular processes; and they display robust behaviors that are easily assayed at early larval stages. moreover, larval zebrafish readily take up small molecules from their environment, enabling non-invasive and high-throughput readout of nervous system function in the context of behavioral drug screens (\u003cspan additionalcitationids=\"cr35\" citationid=\"cr34\" class=\"citationref\"\u003e34\u003c/span\u003e–\u003cspan citationid=\"cr36\" class=\"citationref\"\u003e36\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enmdars are heterotetramers composed of two obligatory glun1 (encoded by \u003cem\u003egrin1\u003c/em\u003e) subunits. they are expressed as either diheterotetramers with two glun1 and two glun2 (\u003cem\u003egrin2a-d\u003c/em\u003e) subunits or triheterotetramers of some combination of glun2 and/or glun3 (\u003cem\u003egrin3a-b\u003c/em\u003e) subunits (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e) (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003ea). these nmdar subunits are differentially expressed over the course of neurodevelopment, with glun2a, glun2b, and glun2d being highly expressed during early development, while glun2c, glun3a, and glun3b expression increases into adulthood (\u003cspan citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e). moreover, nmdars display distinct functional and pharmacological properties based on their glun2 or glun3 composition (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e). each subunit consists of four principal domains: an n-terminal domain (ntd), a ligand-binding domain (lbd), a transmembrane domain (tmd) that forms the ion channel, and a c-terminal domain (ctd) (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003eb).\u003c/p\u003e\u003cp\u003e\u003c/p\u003e\u003cp\u003edisease-associated variants (davs) in \u003cem\u003egrin1\u003c/em\u003e, \u003cem\u003egrin2a\u003c/em\u003e, \u003cem\u003egrin2b\u003c/em\u003e, and \u003cem\u003egrin2d\u003c/em\u003e have been identified in pediatric patients with neurodevelopmental disorders (\u003cspan additionalcitationids=\"cr16 cr17\" citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e–\u003cspan citationid=\"cr18\" class=\"citationref\"\u003e18\u003c/span\u003e, \u003cspan citationid=\"cr21\" class=\"citationref\"\u003e21\u003c/span\u003e). these disorders often present with multiple comorbidities, reflecting overlapping neurodevelopmental phenotypes (\u003cspan citationid=\"cr37\" class=\"citationref\"\u003e37\u003c/span\u003e). generally, patients with davs in \u003cem\u003egrin1\u003c/em\u003e present with developmental delay and intellectual disability, though a large proportion of patients also experience seizures (\u003cspan citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e). in contrast, \u003cem\u003egrin2a\u003c/em\u003e and \u003cem\u003egrin2d\u003c/em\u003e are predominantly associated with seizure disorders (\u003cspan citationid=\"cr16\" class=\"citationref\"\u003e16\u003c/span\u003e, \u003cspan citationid=\"cr18\" class=\"citationref\"\u003e18\u003c/span\u003e). \u003cem\u003egrin2b\u003c/em\u003e is a high-risk gene for autism spectrum disorder (asd) (\u003cspan citationid=\"cr17\" class=\"citationref\"\u003e17\u003c/span\u003e, \u003cspan citationid=\"cr38\" class=\"citationref\"\u003e38\u003c/span\u003e, \u003cspan citationid=\"cr39\" class=\"citationref\"\u003e39\u003c/span\u003e), as are \u003cem\u003egrin1\u003c/em\u003e and \u003cem\u003egrin2a\u003c/em\u003e (\u003cspan additionalcitationids=\"cr41\" citationid=\"cr40\" class=\"citationref\"\u003e40\u003c/span\u003e–\u003cspan citationid=\"cr42\" class=\"citationref\"\u003e42\u003c/span\u003e). still, patients with such neurodevelopmental disorders generally experience a variety of both central and peripheral nervous system symptoms (\u003cspan additionalcitationids=\"cr44 cr45\" citationid=\"cr43\" class=\"citationref\"\u003e43\u003c/span\u003e–\u003cspan citationid=\"cr46\" class=\"citationref\"\u003e46\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enmda receptors are highly conserved across species (\u003cspan citationid=\"cr47\" class=\"citationref\"\u003e47\u003c/span\u003e). due to an ancient genome duplication event, zebrafish retain two paralogues for most nmdar genes (e.g., \u003cem\u003egrin1a\u003c/em\u003e \u0026amp; \u003cem\u003egrin1b\u003c/em\u003e) (\u003cspan citationid=\"cr24\" class=\"citationref\"\u003e24\u003c/span\u003e, \u003cspan citationid=\"cr48\" class=\"citationref\"\u003e48\u003c/span\u003e). patch clamp electrophysiology experiments show that zebrafish paralogues display similar functional properties to each other and to their mammalian orthologues (\u003cspan citationid=\"cr27\" class=\"citationref\"\u003e27\u003c/span\u003e, \u003cspan citationid=\"cr28\" class=\"citationref\"\u003e28\u003c/span\u003e). nevertheless, it remains unclear how well the human nmdar genes, including positions with identified missense variants, are conserved in zebrafish. to address this, we investigated nmdar conservation in zebrafish by taking advantage of amino acid sequence alignments (\u003cspan citationid=\"cr28\" class=\"citationref\"\u003e28\u003c/span\u003e) and assessed the conservation of all human nmdar subunits and zebrafish paralogues. we also compared the conservation of positions with missense variants identified in human patients for nmdar subunits expressed during early neurodevelopment. finally, we investigated the conservation of critical functional regions within each structural domain. through this analysis, we define the extent to which critical functional components and regions, with relevance to disease, are conserved across zebrafish nmdars, providing a template to use zebrafish as a model to study nmdars in neurodevelopment and disease pathology.\u003c/p\u003e"},{"header":"methods","content":"\u003cp\u003e\u003cb\u003enmdar sequences and sequence alignments\u003c/b\u003e\u003c/p\u003e\u003cp\u003eamino acid sequences for human genes are (uniprot with protein ids): glun1 (q05586-5; ntd splice isoform, nr1-3), glun2a (q12879-1), glun2b (q13224), glun2c (q14957), glun2d (o15399), glun3a (q8tcu5), and glun3b (o60391).\u003c/p\u003e\u003cp\u003efor mouse subunits: mglun1 (p35438), mglun2a (p35436), mglun2b (q01097), mglun2c (q01098), mglun2d (q03391), mglun2a (a2air5), and mglun3b (q91zu9).\u003c/p\u003e\u003cp\u003efor zebrafish paralogs: zglun1a (f1r366), zglun1b (q6zm67), zglun2aa (a0a8m9pxd6), zglun2ab (f1qde5), zglun2ba (a0a8m9q0i1), zglun2bb (a0a8m1nh89), zglun2ca (e7fh62), zglun2cb (a0a8m3b7k5), zglun2da (i3ni77), zglun2db (e7f3z4), zglun3a (a0a8m3awh5), zglun3ba (a0a8m9p9f6), and zglun3bb (a0a8m6z151).\u003c/p\u003e\u003cp\u003ebreakpoints for nmdar subunit domains and subdomains are based on structures 6whr (rat glun1-glun2b) and 7eu7 (human glun1-glun2a) (\u003cspan citationid=\"cr49\" class=\"citationref\"\u003e49\u003c/span\u003e, \u003cspan citationid=\"cr50\" class=\"citationref\"\u003e50\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ehuman and zebrafish protein sequences were aligned and analyzed for amino acid identity and similarity using align sequences protein blast (blastp), which is publicly available through the national center for biotechnology information (ncbi) or emboss needle pairwise sequence alignment (psa). for each nmdar subunit, the zebrafish paralogs were individually aligned to the human protein. amino acid similarity is defined based on the blosum 62 substitution matrix. for the ntd, lbd, and tmd, these alignments were generally straightforward as the sequences do not diverge greatly in length. in contrast, the ctd is quite large, with the zebrafish sequences generally larger than human. for these alignments, we entered the entire sequence into the alignment tool and used the given result, though this led to a fragmentation of the ctd such that certain portions of the sequence were aligned while others were not. we used the resulting domain-specific conservation values to calculate the conservation of the total protein.\u003c/p\u003e\u003cp\u003e\u003cb\u003evariants analysis\u003c/b\u003e\u003c/p\u003e\u003cp\u003ehuman nmdar variants were identified using clinvar and gnomad (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003eclinvar\u003c/em\u003e. results for each gene were downloaded on september 28th, 2023, using clinvar's bulk download option (\u003cspan class=\"externalref\"\u003e\u003cspan class=\"refsource\"\u003ehttps://www.ncbi.nlm.nih.gov/clinvar/\u003c/span\u003e\u003cspan address=\"https://www.ncbi.nlm.nih.gov/clinvar/\" targettype=\"url\" class=\"reftarget\"\u003e\u003c/span\u003e\u003c/span\u003e). genomic variants that did not result in protein coding changes were removed from the analysis. in the case of \u003cem\u003egrin3a\u003c/em\u003e, protein coding changes in the ppp3r2 gene, a single exon gene whose coding sequence resides within the genomic region of \u003cem\u003egrin3a\u003c/em\u003e and is downloaded alongside \u003cem\u003egrin3a\u003c/em\u003e due to this, was also removed. in cases where multiple protein coding annotations exist for the same variant, only the annotation fitting the canonical transcript was retained.\u003c/p\u003e\u003cp\u003e\u003cem\u003egnomad\u003c/em\u003e: genes were searched for by name on gnomad v4.0.0 (\u003cspan class=\"externalref\"\u003e\u003cspan class=\"refsource\"\u003ehttps://gnomad.broadinstitute.org/\u003c/span\u003e\u003cspan address=\"https://gnomad.broadinstitute.org/\" targettype=\"url\" class=\"reftarget\"\u003e\u003c/span\u003e\u003c/span\u003e) and missense, insertion, deletion, and loss-of-function variant information was exported on september 28th, 2023 as comma separated value (csv) files for each gene. hgvs annotation (protein change) was converted to letter-number-letter annotation.\u003c/p\u003e\u003cp\u003egene variants from clinvar and gnomad were merged using r studio 2023.06.1 build 524 based on the letter-number-letter annotation. the resulting merged file was checked for redundant information or variant duplicates, which were removed.\u003c/p\u003e\u003cp\u003emissense variants were extracted and grouped by pathogenicity as pathogenic, putatively pathogenic, complex, putatively benign, and benign variants based on the criteria in table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e. assessment of conservation of positions with identified human variants was performed using the generated sequence alignments.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar domain-specific analysis\u003c/b\u003e\u003c/p\u003e\u003cp\u003eto further refine the relationship between human and zebrafish nmdar proteins, we analyzed the tmd, lbd-tmd linkers, lbd, ntd, and ctd of glun1, glun2a, glun2b, and glun2d for conservation of key structural and functional features.\u003c/p\u003e\u003cp\u003e\u003cem\u003etransmembrane domain (tmd) and ligand-binding domain-transmembrane domain (lbd-tmd) linkers.\u003c/em\u003e key regions involved in channel gating, mg\u003csup\u003e2+\u003c/sup\u003e block, and ca\u003csup\u003e2+\u003c/sup\u003e flux were identified as described in the literature and analyzed for conservation (\u003cspan additionalcitationids=\"cr52 cr53 cr54 cr55 cr56\" citationid=\"cr51\" class=\"citationref\"\u003e51\u003c/span\u003e–\u003cspan citationid=\"cr57\" class=\"citationref\"\u003e57\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003eligand-binding domain (lbd)\u003c/em\u003e. key regions and specific amino acids involved in van-der-waals interactions with the α-carbon of the ligand, interactions with the α-amino group, electrostatic interactions with the α-carboxyl group, and interactions with the amino acid side chain were identified as described in ramos-vicente et. al. 2021 and analyzed for conservation (\u003cspan citationid=\"cr47\" class=\"citationref\"\u003e47\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003en-terminal domain (ntd)\u003c/em\u003e. key amino acid positions required for the binding of zinc and polyamines were identified via a literature search and their conservation assessed (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr58\" class=\"citationref\"\u003e58\u003c/span\u003e, \u003cspan citationid=\"cr59\" class=\"citationref\"\u003e59\u003c/span\u003e). we also examined the conservation of putative n-linked glycosylation sites (\u003cspan citationid=\"cr60\" class=\"citationref\"\u003e60\u003c/span\u003e). to generate homology models of the zebrafish ntds, we used the template function in swiss-model (\u003cspan citationid=\"cr61\" class=\"citationref\"\u003e61\u003c/span\u003e, \u003cspan citationid=\"cr62\" class=\"citationref\"\u003e62\u003c/span\u003e). as reference structures, we used rodent ntd structures 5tq0 (glun1-glun2a in the presence of edta, no bound zn\u003csup\u003e2+\u003c/sup\u003e for glun1 and glun2a), 5tpz (glun1-glun2b apo state for glun1 and glun2b), and 5tpw (glun1-glun2a with zn\u003csup\u003e2+\u003c/sup\u003e in complex with glun2a) (\u003cspan citationid=\"cr58\" class=\"citationref\"\u003e58\u003c/span\u003e). sequences for human (hglun1, hglun2a, and hglun2b) or zebrafish (zglun1a, zglun1b, glun2aa, glun2ab, glun2ba, glun2bb) ntds were entered individually into swiss-model to generate homology models. the subsequent models were then loaded into pymole (human and zebrafish paralog) and aligned. the resultant root mean square deviation (rmsd) is reported.\u003c/p\u003e\u003cp\u003e\u003cem\u003ecarboxy-terminal domain (ctd)\u003c/em\u003e. ctd-specific conservation in zebrafish was assessed for the glun1, glun2a, and glun2b subunits. glun2d was excluded from this analysis due to the poor conservation of the ctd in these paralogs relative to human. to assess functional relevance, we focused on post-translational modifications (ptms), specifically phosphorylation sites (serine, threonine, tyrosine), ubiquitylation sites (lysine), and palmitoylation sites (cysteine), using a combination of experimentally validated data from phosphositeplus (psp) and published literature (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e). phosphorylation sites with identified kinases or well-characterized functional roles were classified as ‘kinase-regulated phosphorylation sites.’ the remaining phosphorylation sites identified in psp were defined as ‘phosphoproteomic-identified sites.’ note, sites in psp (based on rodents) that did not have a phosphorylation site in humans were excluded from our analysis. in addition, known linear protein-binding motifs were annotated and analyzed for conservation (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e). to assess conservation of protein-binding motifs in the ctds, we categorized sites as either short linear motifs (slims) or broader interaction ‘regions’. slims were defined as experimentally validated, compact sequence elements (typically \u0026lt; 15 residues) that mediate specific and direct interactions with known protein partners. broader regions were defined as extended sequences (\u0026gt; 15 residues) shown to directly bind proteins in biochemical or structural studies but lacking a discrete consensus sequence. all motifs were curated from published literature where discrete sequences were experimentally validated per motif (\u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e, \u003cspan additionalcitationids=\"cr64 cr65 cr66 cr67 cr68 cr69 cr70 cr71 cr72 cr73 cr74 cr75 cr76 cr77 cr78 cr79\" citationid=\"cr63\" class=\"citationref\"\u003e63\u003c/span\u003e–\u003cspan citationid=\"cr80\" class=\"citationref\"\u003e80\u003c/span\u003e), and sequence conservation was examined.\u003c/p\u003e"},{"header":"results","content":"\u003cp\u003eour goal is to compare the structural similarity of zebrafish nmdar (zglun) paralogues to human nmdars (glun), especially regarding human variants. since mice are a common animal model (\u003cspan class=\"citationref\"\u003e81\u003c/span\u003e), as a reference-point for nmdar gene conservation, we initially made global comparisons of both mouse and zebrafish nmdar subunits to human.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eglobal comparison of mouse and zebrafish orthologues to human nmdars\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003enmdars are heterotetramers composed of two obligate glycine-binding glun1 subunits and two glutamate-binding glun2 subunits or a combination of glun2 and glycine-binding glun3 subunits (\u003cspan class=\"citationref\"\u003e1\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e2\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ea,b). glycine and glutamate act as co-agonists for glun2-containing receptors, with binding of both required for receptor activation, while glun3-containing receptors are activated by glycine binding. as an initial assessment of the structural similarity between nmdar orthologues, we made global sequence alignments using the ncbi blastp or emboss psa tools. not surprisingly, given evolutionary proximity, the mouse subunits are highly conserved relative to human: glun1 (97% identity \u0026amp; similarity), glun2a (95% identity; 98% similarity), glun2b (98% identity; 99% similarity), glun2c (88% identity; 90% similarity), glun2d (97% identity \u0026amp; similarity), glun3a (92% identity; 95% similarity), and glun3b (77% identity; 83% similarity) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003eon the other hand, the zebrafish paralogues show reduced conservation. the obligatory glun1 subunits are the most highly conserved, both in terms of identity (\u0026ge;\u0026thinsp;84%) and similarity (\u0026ge;\u0026thinsp;89%) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the other subunits show various degrees of identity and similarity: glun2a (\u0026ge;\u0026thinsp;58% identity; \u0026ge;79% similarity), glun2b (\u0026ge;\u0026thinsp;65% identity; \u0026ge;73% similarity), glun2c (\u0026ge;\u0026thinsp;50% identity; \u0026ge;64% similarity), glun2d (\u0026ge;\u0026thinsp;43% identity; \u0026ge;54% similarity), glun3a (63% identity; 78% similarity), and glun3b (\u0026ge;\u0026thinsp;54% identity, \u0026ge;\u0026thinsp;62% similarity) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). these subunits, thus, retain\u0026thinsp;\u0026ge;\u0026thinsp;50% identity with respect to the human protein, except for zglun2da and zglun2db, which only share 43% and 46% sequence identity, respectively (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). with respect to the zebrafish paralog pairs, there are instances in which one paralog is better conserved than the other. the glun1 paralogs are comparably conserved (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). however, the zglun2aa paralog is more highly conserved than zglun2ab while the zglun2bb shows increased conservation relative to zglun2ba (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the glun2c, glun2d, and glun3b paralogs are conserved to similar extents (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003enevertheless, despite lower conservation relative to mouse, zebrafish offer many technical advantages (see background). in addition, what is critical is the conservation of specific structural and functional domains and subdomains.\u003c/p\u003e\n\u003cdiv align=\"left\" class=\"colspec\"\u003e\u003cbr\u003e\u003c/div\u003e\n\u003ctable id=\"tab1\" border=\"1\"\u003e\n \u003ccaption language=\"en\"\u003e\n \u003cdiv class=\"captionnumber\"\u003etable 1\u003c/div\u003e\n \u003cdiv class=\"captioncontent\"\u003e\n \u003cp\u003eoverall protein conservation in mouse and zebrafish relative to human\u003c/p\u003e\n \u003c/div\u003e\n \u003c/caption\u003e\n \u003cthead\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003ehuman\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003emouse\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003ezebrafish\u003c/p\u003e\n \u003cp\u003eparalogue a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003ezebrafish\u003c/p\u003e\n \u003cp\u003eparalogue b\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003c/thead\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en1\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e938\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e938\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e84%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e89%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e966\u003c/p\u003e\n \u003cp\u003e(-28)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e86%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e90%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e937\u003c/p\u003e\n \u003cp\u003e(-1)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2a\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1464\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e95%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e98%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1464\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e68%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e89%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1460\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;6)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e58%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e79%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1445\u003c/p\u003e\n \u003cp\u003e(-9)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2b\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1484\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e98%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e99%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1482\u003c/p\u003e\n \u003cp\u003e(-2)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e65%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e73%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1417\u003c/p\u003e\n \u003cp\u003e(-31)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e67%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e81%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1770 (+\u0026thinsp;322)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2c\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1233\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e88%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e90%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1239\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;6)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e50%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e64%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1400\u003c/p\u003e\n \u003cp\u003e(-176)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e59%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e79%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1328\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;95)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2d\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1336\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1323\u003c/p\u003e\n \u003cp\u003e(-13)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e43%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e54%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1916 (+\u0026thinsp;580)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e46%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e57%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1676 (+\u0026thinsp;340)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en3a\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1115\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e92%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e95%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1135\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;20)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e63%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e78%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1111\u003c/p\u003e\n \u003cp\u003e(-4)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en3b\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1043\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e77%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e83%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1003\u003c/p\u003e\n \u003cp\u003e(-40)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e54%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e62%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1118\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;75)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e62%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e68%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1114\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;71)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003c/p\u003e\n\u003cp\u003enumbers indicate the percent of mouse or zebrafish paralog conservation with respect to the human protein \u0026ndash; \u0026lsquo;id.\u0026rsquo; is identity and \u0026lsquo;sim.\u0026rsquo; is similarity. the zebrafish a and b paralogs are indicated for each protein. glun3a has only a single paralog in zebrafish. for the mouse and zebrafish protein length, numbers in parentheses indicate the difference in amino acid number relative to human.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of specific structural elements in zebrafish paralogues\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003efor more rigorous comparisons, we examined the conservation within individual subunit domains and subdomains (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the structure of nmdars, like all iglurs, consists of 4 domains that are intrinsic to each individual subunit (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ea,b): the extracellular ntd and lbd; the tmd forming the ion channel as well as the linkers that connect the lbd to the tmd, the lbd-tmd linkers; and the intracellular ctd (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e53\u003c/span\u003e). within individual subunits, the lbd is formed by two disparate sequences, s1 and s2, that fold up into the d1 and d2 lobes in the 3-dimensional structure (83, 84). in general, d1 is made up primarily of s1 and d2 of s2, but there is considerable swapping between lobes. the tmd is formed by three transmembrane segments, m1, m3, and m4, and an m2-pore loop. the lbd-tmd linkers are referred to as either s1-m1, m3-s2, and s2-m4 (linear sequence) or d2-m1, m3-d2, and d1-m4 (3-dimensional structure). this distinction is important functionally since d1 and d2 undergo different movements that are translated differently to the tmd (\u003cspan class=\"citationref\"\u003e85\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e86\u003c/span\u003e). nevertheless, we will use the s1-m1, m3-s2, and s2-m4 designation for ease of comparison to structures.\u003c/p\u003e\n\u003cp\u003ewe began by dividing the zebrafish paralogues into their respective domains and subdomains based on our breakpoints (see methods). interestingly, the amino acid sequences surrounding these breakpoints were well conserved in zebrafish \u0026ndash; the majority were identical (not shown). this allowed for ease of division of the zebrafish proteins into their respective domains and subdomains.\u003c/p\u003e\n\u003cp\u003eas an initial assessment of the structural similarity between human and zebrafish nmdars, we made detailed sequence alignments using the ncbi blastp or emboss psa tools (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e, fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). in terms of length of domains, the zebrafish protein sequence for glun1, glun2, and glun3 subunits was identical to the human sequence for the tmd (m1, m1-m2, m2, m2-m3, m3, \u0026amp; m4) and lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec). the lbd (s1 \u0026amp; s2) was also quite similar in length, with no changes in the length of s2 across the paralogs and only slight increases or decreases in amino acid number in the s1 of zglun1a, zglun1b, zglun2ab, zglun2ca, zglun2ca, zglun2da, and zglun2db (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec). the ntd and ctd were less conserved in terms of size, wherein almost every zebrafish paralog has a different number of amino acids compared to the human. this is especially notable for the glun2b ntd which either lacks 130 amino acids (zglun2ba) or has an additional 74 amino acids (zglun2bb) as well as for the ctd where, for example, zglun2da and zglun2db have a ctd that is more than twice as long as that in the human protein (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec).\u003c/p\u003e\n\u003ctable id=\"tab2\" border=\"1\"\u003e\n \u003ccaption language=\"en\"\u003e\n \u003cdiv class=\"captionnumber\"\u003etable 2\u003c/div\u003e\n \u003cdiv class=\"captioncontent\"\u003e\n \u003cp\u003evariant designation\u003c/p\u003e\n \u003c/div\u003e\n \u003c/caption\u003e\n \u003cthead\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eglun\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003edesignation\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eclinvar\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003egnomad\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e1\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2b\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2c\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2d\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e3a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e3b\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003c/thead\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epathogenic\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;pathogenic\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e49\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e58\u003c/p\u003e\n \u003cp\u003e(4%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e81\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e12\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eputatively\u003c/p\u003e\n \u003cp\u003epathogenic\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;uncertain\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e123\u003c/p\u003e\n \u003cp\u003e(13%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e214\u003c/p\u003e\n \u003cp\u003e(15%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e182\u003c/p\u003e\n \u003cp\u003e(12%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e4\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e111\u003c/p\u003e\n \u003cp\u003e(8%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e5\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e8\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003ecomplex\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;pathogenic\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e3\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e5\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e6\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eputatively benign\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;uncertain\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e80\u003c/p\u003e\n \u003cp\u003e(9%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e320\u003c/p\u003e\n \u003cp\u003e(22%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e199\u003c/p\u003e\n \u003cp\u003e(13%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e31\u003c/p\u003e\n \u003cp\u003e(3%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e223\u003c/p\u003e\n \u003cp\u003e(17%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e30\u003c/p\u003e\n \u003cp\u003e(3%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e68\u003c/p\u003e\n \u003cp\u003e(7%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003ebenign\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;benign\u0026rdquo;\u003c/p\u003e\n \u003cp\u003eor\u003c/p\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent/\u003c/p\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003cp\u003eor\u003c/p\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e401\u003c/p\u003e\n \u003cp\u003e(43%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e813\u003c/p\u003e\n \u003cp\u003e(55%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e672\u003c/p\u003e\n \u003cp\u003e(45%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e945\u003c/p\u003e\n \u003cp\u003e(77%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e863\u003c/p\u003e\n \u003cp\u003e(27%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e59\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e945\u003c/p\u003e\n \u003cp\u003e(91%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003c/p\u003e\n\u003cp\u003enumber of positions with missense variants that have been identified in each subunit per designation. percentages report the proportion of the total protein amino acids at which such variants have been identified.\u003c/p\u003e\n\u003cp\u003ewe used these sequence alignments to assess the degree of amino acid identity and similarity between domains and subdomains (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ed,e). the obligatory glun1 subunit showed the highest degree of conservation. the tmd and lbd-tmd linkers are 100% conserved. the lbd is also well conserved with s1 and s2 showing\u0026thinsp;\u0026ge;\u0026thinsp;92% identity and \u0026ge;\u0026thinsp;93% similarity. the ntd is somewhat less conserved (\u0026ge;\u0026thinsp;80% identity, \u0026ge;\u0026thinsp;88% similarity), as is the ctd (\u0026ge;\u0026thinsp;53% identity; \u0026ge;58% similarity).\u003c/p\u003e\n\u003cp\u003ethis trend of conservation across nmdar subunits \u0026ndash; the lbd and tmd are most highly conserved and the ntd and ctd less so \u0026ndash; is maintained in the glun2 and glun3 subunits (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ed,e). generally, the glun2 subunits maintain a higher sequence identity than the glun3 subunits. overall, this sequence analysis suggests that the lbd and tmd are highly conserved whereas the ntd and ctd show less conservation.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecategorizing the pathogenicity of missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003enmdars with missense, frameshift, and nonsense variants have been identified in pediatric patients with neurodevelopmental disorders (\u003cspan class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e22\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e87\u003c/span\u003e). we therefore wanted to assess the extent to which variants are conserved in zebrafish to aid in guiding future studies. for this analysis, we focused solely on missense variants. while nonsense and frameshift mutation can be modelled in various ways, such as by mutating a portion of the protein to induce a loss-of-function, the conservation of the specific protein sequence is somewhat less relevant. in contrast, the single amino acid change associated with missense variants is critical to dissecting how such variants alter receptor function and lead to disease phenotypes.\u003c/p\u003e\n\u003cp\u003eusing publicly available datasets of nmdar variants from clinvar and gnomad, we identified missense variants in all nmdar subunits (see methods). we first categorized these variants with regards to their pathogenicity (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). clinvar reports variants that have been identified in patients with symptomatic disease (\u003cspan class=\"citationref\"\u003e88\u003c/span\u003e\u0026ndash;\u003cspan class=\"citationref\"\u003e90\u003c/span\u003e), while gnomad reports variants identified in the healthy population (\u003cspan class=\"citationref\"\u003e91\u003c/span\u003e). thus, in general, we define variants present in clinvar but absent in gnomad as pathogenic while those with the opposite reporting as benign. with this approach, pathogenic and putatively pathogenic variants were identified in glun1, glun2a, glun2b, and glun2d while minimal to no such variants were identified in glun2c, glun3a, and glun3b (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). notably, these genes with known pathogenic variants are expressed early in development, contributing to developmental processes while their dysfunction often leads to disease phenotypes (\u003cspan class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e87\u003c/span\u003e). on the other hand, glun2c, glun3a, and glun3b have no pathogenic and a small number of putatively pathogenic variants (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). we therefore focus solely on glun1, glun2a, glun2b, and glun2d for further analysis.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003epathogenic and putatively pathogenic variant positions are well conserved in zebrafish\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003eusing the criteria in table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e, amino acid positions with pathogenic missense variants in glun1, glun2a, glun2b, and glun2d are completely conserved in zebrafish paralogues (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ea-d).\u003c/p\u003e\n\u003cp\u003eputatively pathogenic variants are also conserved in these nmdar subunits, though less so than pathogenic variants. the highest degree of conservation of putatively pathogenic variants is in the glun1 paralogues (\u0026ge;\u0026thinsp;88% identity; 93% similarity). putatively pathogenic variants in the glun2 subunits are also reasonably conserved: zglun2aa (70% identity; 92% similarity), zglun2ab (62% identity; 84% similarity), zglun2ba (65% identity; 71% similarity), zglun2bb (72% identity; 80% similarity), and zglun2da and zglundb (\u0026ge;\u0026thinsp;46% identity; \u0026ge;59% similarity) (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ee,f). in general, the highest level of conservation is in the tmd and tmd-lbd linkers, followed by the lbd, and less so in the ntd and ctd (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003eg,h).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ebenign and putatively benign variant positions show reduced conservation\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ewe carried out a comparable analysis as above for benign and putatively benign variants (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). we hypothesized that positions with more benign variants would be less well conserved than pathogenic ones. indeed, the general trend is consistent with this idea (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003e). putatively benign variants showed a reduced conservation compared to putatively pathogenic variants (putatively pathogenic vs putatively benign): zglun1a (88% vs. 78% identity; 93% vs. 88% similarity), zglun1b (89% vs. 79% identity; 93% vs. 84% similarity), zglun2aa (70% vs. 60% identity; 92% vs. 86% similarity), zglun2ab (62% vs. 48% identity; 84% vs. 68% similarity), zglun2ba (65% vs. 45% identity; 71% vs. 55% similarity), zglun2bb (72% vs. 55% identity; 80% vs. 70% similarity), zglun2da (46% vs. 47% identity; 59% vs. 61% similarity), and zglun2db (47% vs. 44% identity; 61% vs. 56% similarity) (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ee,f vs fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ea,b). for these putatively benign variants, the greatest extent of conservation tends to occur in the tmd and lbd-tmd linkers, followed by the lbd, and then the ntd and ctd (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ec,d).\u003c/p\u003e\n\u003cp\u003efor benign variants, however, the decrease in conservation plateaus, with identity and similarity values comparable to the putatively benign variants, if not slightly higher (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ee-f). the subdomain-specific conservation trend for these variants follows that seen on the level of the whole receptor (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e), wherein the tmd, lbd-tmd linkers, and lbd are generally more conserved and the ntd and ctd less so (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003eg,h). although diverging from our prediction as we do not note a substantial decrease in the conservation of benign variants with respect to the putatively benign variants, this result reflects the overall conservation of the nmdar subunits themselves (benign vs total protein): zglun1 (78% vs. \u0026ge;84% identity; 84% vs. \u0026ge;89% similarity), zglun2aa (62% vs. 68% identity; 86% vs. 89% similarity), zglun2ab (49% vs. 58% identity; 74% vs. 79% similarity), zglun2ba (54% vs. 65% identity; 64% vs. 73% similarity), and zglun2bb (62% vs. 67% identity; 74% vs. 81% similarity), zglun2da (50% vs. 43% identity; 64% vs. 54% similarity), and zglun2db (49% vs. 46% identity; 62% vs. 57% similarity) (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ee-f, table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). furthermore, this result reflects the relatively large number of benign variants in comparison to other variant groups (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecritical tmd and lbd-tmd linker components are completely conserved\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003eelements forming the ion channel \u0026ndash; the tmd (m1, m1-m2, m2, m2-m3, m3, \u0026amp; m4) \u0026ndash; and the lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4) were consistently highly conserved (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). nevertheless, we assayed conservation of critical functional motifs. the s1-m1 linker is a critical element for channel gating and is completely conserved (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e) (\u003cspan class=\"citationref\"\u003e56\u003c/span\u003e). the m2 subdomain contains what is known as the \u0026lsquo;n\u0026rsquo; and \u0026lsquo;n\u0026thinsp;+\u0026thinsp;1\u0026rsquo; sites, which participate in mg\u003csup\u003e2+\u003c/sup\u003e block and ca\u003csup\u003e2+\u003c/sup\u003e permeation through the channel (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e55\u003c/span\u003e) and are conserved in zebrafish (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, dark orange). the tmd m3 segment contains the sytanlaaf motif, which plays a key role in ion channel gating and is known to be highly conserved across species. this motif is conserved in all zebrafish glun1 and glun2 paralogs (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, orange). lastly, the \u0026lsquo;drpeer\u0026rsquo; motif in m3-s2 of glun1, an important component in the high ca\u003csup\u003e2+\u003c/sup\u003e permeability of nmdars (\u003cspan class=\"citationref\"\u003e52\u003c/span\u003e) is conserved (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, light orange). hence, core gating and permeation elements of the tmd and associated regions are highly conserved suggesting that basic mechanisms of ion channel gating and block/permeation are preserved.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003elbd elements required for agonist binding are completely conserved\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ethe glun1 lbd binds glycine or d-serine while the glun2 lbd binds glutamate or aspartate (fig. s2a,b). overall, the lbds are highly conserved in zebrafish relative to humans (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e), as are specific amino acid side chains in s1 and s2 that are involved in binding endogenous agonists (\u003cspan class=\"citationref\"\u003e47\u003c/span\u003e) (fig. s2c). these include sites that participate in van-der waals interactions with the \u0026alpha;-carbon of the ligand (darkest red), interactions with the \u0026alpha;-amino group (dark red), electrostatic interactions with the \u0026alpha;-carboxyl group (red), and interactions with the amino acid side chain (light red) (fig. s2c).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of structure-function elements in the ntd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ein terms of overall sequence comparisons, the ntd shows variable degrees of conservation across nmdar subunits. the glun1 zebrafish paralogs maintain the highest degree of conservation relative to human (80% identity; \u0026ge;88% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the glun2 subunits show decreased ntd conservation (49\u0026ndash;75% identity; 59\u0026ndash;88% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). given this reduced conservation, we assayed the conservation of key structure-function motifs in zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003eamong its many roles, the ntd participates in interactions with ions and molecules that modulate receptor activity (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e59\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ea,b). this is most notable for glun2a and to a lesser extent glun2b as they bind zinc, which inhibits receptor function by reducing channel open probability (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). in glun2a, four amino acids coordinate zinc binding: glun2a-his44, glun2a-his128, glun2a-glu266, and glun2a-asp282 (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). of these key sites, three are conserved (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec) but in zglun2aa and zglun2ab, a key residue involved in zn\u003csup\u003e2+\u003c/sup\u003e binding, glun2a-his44, is not conserved (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec). for glun2b, only two amino acid residues participate in zinc binding: glun2b-his127 and glun2b-glu284 (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). while glun2b-glu284 is conserved, the glun2b-his127 position in zglun2ba is missing due to alterations in ntd sequence length leading to gaps in the alignments, though it is similar now as a positively charged arginine instead of a histidine in zglun2bb (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec).\u003c/p\u003e\n\u003cp\u003ethe ntd of glun2b also binds polyamines, which enhance receptor activity (\u003cspan class=\"citationref\"\u003e59\u003c/span\u003e). nine amino acids are involved in the binding of polyamines. these sites in zglun2ba and zglun2bb are completely conserved except for one amino acid in zglun2bb that is similar but not identical, now a negatively charged aspartate instead of glutamate (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ed). most of this functional component of glun2b is, thus, conserved in zebrafish.\u003c/p\u003e\n\u003cp\u003eadditionally, the nmdar ntd has glycosylation sites, and their glycosylation regulates the trafficking of nmdars to the cell membrane (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e). given the consensus sequence for arginine (n-) glycosylation (n-\u003cem\u003ex\u003c/em\u003e-s/t), glun1 has seven, glun2a has three, and glun2b has three putative glycosylation sites in the ntd (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e). their conservation in the zebrafish paralogues is somewhat variable: zglun1a \u0026amp; zglun1b (5 of 7 sites), zglun2aa (3 of 3 sites), zglun2ab (2 of 3 sites), zglun2ba (2 of 3 sites), and zglun2bb (3 of 3 sites) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ee). to the best of our knowledge, specific glycosylation sites in glun2d have yet to be identified. of note, the glun1 glycosylation sites glun1-n203 and glun1-n368 are necessary for receptor surface expression (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e), and these sites are conserved in the zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003ewhile the critical functional features of the lbd were completely conserved in zebrafish relative to human nmdars, the conservation of key regions in the ntd occasionally diverged, most notably that for zn\u003csup\u003e2+\u003c/sup\u003e binding. to further explore the structural composition of the glun1, glun2a, and glun2b ntds, we used swiss-model to generate homology models for the human and zebrafish nmdar subunits, which we subsequently aligned in pymole to assess conservation at the level of the three-dimensional structure (see methods) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ef). in general, this analysis revealed a high degree of similarity between the human ntd and zebrafish paralogs. the zebrafish glun1 ntd models display similar overall topology to human: glun1 vs zglun1a, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.071); glun1 vs zglun1a, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.085); glun1 vs zglun1b, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.094); and glun1 vs zglun1b, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.097) (table s3).\u003c/p\u003e\n\u003cp\u003ethe glun2a and glun2b subunits also reflect a high degree of structural similarity: glun2a vs zglun2aa, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.085); glun2a vs zglun2ab, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.095); glun2a vs zglun2ab, reference structure 5tpw (rmsd\u0026thinsp;=\u0026thinsp;0.111); glun2b vs zglun2ba, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.028); and glun2b vs glun2bb, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.044) (table s3). the only exception here is the structural comparison of glun2a and zglun2aa using reference structure 5tpw, in which the ntd is interacting with zinc (rmsd\u0026thinsp;=\u0026thinsp;2.149), suggesting that the structural components of zglun2aa may not effectively complex with zinc. interestingly, zglun2ba is \u0026gt;\u0026thinsp;100 amino acids shorter and zglun2bb is \u0026gt;\u0026thinsp;70 amino acids longer than human glun2b. though the zglun2ba model is missing an \u0026alpha;-helix relative to the human protein, the zglun2bb modelled structure aligns well with its human counterpart, suggesting that the zebrafish sequences maintain a large degree of structural integrity.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of post-translational modification (ptm) sites in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ethe intracellular ctd is a key regulatory domain of nmdar cell biology and function and shows distinct subunit-specific elements (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). for our analyses, we focused on the glun1-1 ctd splice variant, which is the longest variant containing the c0-c1-c2 cassettes (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003eacross the nmdar subunits, the zebrafish ctds are the most poorly conserved (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). with a sequence identity generally around 50\u0026ndash;60% for the glun1, glun2a, and glun2b zebrafish paralogs, with some exceptions: zglun1a (53% identity; 58% similarity), zglun1b (54% identity; 59% similarity), zglun2aa (57% identity; 86% identity), zglun2ab (38% identity; 65% similarity), zglun2ba (57% identity, 66% similarity), and zglun2bb (60% identity, 71% similarity) while that of glun2d is much lower (\u0026ge;\u0026thinsp;18% identity, \u0026ge;\u0026thinsp;28% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003egiven these deviations in protein sequences, we first examined the conservation of ctd regions critical for post-translational modifications (ptms), excluding glun2d due its poor conservation in zebrafish. phosphorylation of nmdar subunit ctds confers key functional roles to the receptor (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). initially, we examined experimentally validated sites at which phosphorylation occurs by known kinases and calculated the percent conservation, reported in identity and similarity. similarity, here, is defined as a site at the homologous position that can be phosphorylated (e.g., s, t, or y), if not identical. the glun1 ctd contains phosphorylation sites for src kinase (src), protein kinase c (pkc), and protein kinase a (pka), (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003etop\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). note specific sites have also been identified that are also targeted by the serine/threonine protein phosphatase 2b (pp2b). these ptm sites are perfectly conserved in zglun1a, while three of five are conserved in zglun1b (60% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb). the glun2a ctd is phosphorylated by src, pka, dual specificity tyrosine-phosphorylation-regulated kinase 1a (dyrk1), cyclin-dependent kinase 5 (cdk5), pkc, and calcium/calmodulin-dependent protein kinase ii\u0026alpha; (camkii\u0026alpha;) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003emiddle\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e): zglun2aa (79% identity \u0026amp; similarity) and zglun2ab (64% identity, 71% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb). the ctd of glun2b is phosphorylated by src, camkii\u0026alpha;, cdk5, pka, pkc, death-associated protein kinase 1 (dapk1), proto-oncogene tyrosine-protein kinase fyn (fyn), and casein kinase ii (ck2) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003ebottom\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). glun2b also contains sites for the protein tyrosine phosphatase non-receptor 11 (ptpn11, or shp2) and protein phosphatase 1 (pp1). the conservation of these ptms is slightly higher than that in glun2a: zglun2ba (73% identity \u0026amp; similarity) and zglun2bb (82% identity, 100% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb).\u003c/p\u003e\n\u003cp\u003ein addition to these kinase-regulated sites, we also examined the conservation of other known phosphorylation sites in these subunits identified by phosphoproteomics (psp) (see methods). the conservation of these sites in the zebrafish paralogs was generally lower than that of the kinase-regulated sites: glun1 (67% identity, 100% similarity), zglun2aa (51% identity, 66% similarity), zglun2ab (34% identity, 57% similarity), zglun2ba (60% identity \u0026amp; similarity), and zglun2bb (67% identity, 70% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ec).\u003c/p\u003e\n\u003cp\u003ewe next examined the conservation of ctd palmitoylation and ubiquitylation sites. glun1 has a single ubiquitylation site (lys860) (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003etop\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), and it is conserved in the zebrafish paralogs (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ec). glun2a contains seven palmitoylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003emiddle\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), which are likewise completely conserved in zebrafish (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003eb). the glun2a ctd also has four ubiquitylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003emiddle)\u003c/em\u003e (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), of which three are conserved in zglun2aa (75%) and one in zglun2ab (25%) (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ec). with the same approach for glun2b, we identified eight palmitoylation and six ubiquitylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003ebottom\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). here, all palmitoylation sites are conserved in zebrafish while five of six ubiquitylation sites (83%) are conserved in zglun2ba and all six ubiquitylation sites (100%) are conserved in zglun2bb (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb,c).\u003c/p\u003e\n\u003cp\u003ein summary, the global conservation of ptm sites is as follows: zglun1a and zglun1b (89% identity, 100% similarity), zglun2aa (62% identity, 72% similarity), zglun2ab (45% identity, 62% similarity), zglun2ba (66% identity, 67% similarity), and zglun2bb (75% identity, 78% similarity).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of protein-binding motifs (pbm) in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ein addition to numerous ptm sites, a variety of protein-protein interactions are critical to the role of the ctd in receptor cell biology (surface expression, distribution) and function (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e63\u003c/span\u003e). these protein-binding elements include either defined short linear motifs (slims) or broader regions known to serve as protein docking sites.\u003c/p\u003e\n\u003cp\u003ethe ctd of glun1 contains a slim known as the kkk/rrr er retention motif, which is conserved in zebrafish (\u003cspan class=\"citationref\"\u003e66\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,b). it, likewise, has binding regions for calmodulin (cam)/calcium/calmodulin-dependent protein kinase ii (camkii) and \u0026alpha;-actinin 2 (\u003cspan class=\"citationref\"\u003e64\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e65\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e67\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e78\u003c/span\u003e), which is conserved and yotiao, also known as a-kinase anchoring protein 9 (akap9) (\u003cspan class=\"citationref\"\u003e67\u003c/span\u003e), which is generally well conserved: zglun1a (90% identity, 98% similarity) and zglun1b (90% identity, 98% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,b).\u003c/p\u003e\n\u003cp\u003ethe glun2 subunit ctds contain a variety of pbms. for glun2a, the conservation of slims is: the yxx\u0026oslash; motif, where y is tyrosine, x is any amino acid, and \u0026oslash; is a bulky hydrophobic residue like leucine, isoleucine, phenylalanine, methionine, or valine (\u003cspan class=\"citationref\"\u003e73\u003c/span\u003e) \u0026ndash; both zglun2aa and zglun2ab show 100% identity \u0026amp; similarity; the guanine-nucleotide exchange factor brag2 (\u003cspan class=\"citationref\"\u003e70\u003c/span\u003e) \u0026ndash; zglun2aa (80% identity \u0026amp; similarity) and zglun2ab (40% identity, 60% similarity); postsynaptic density protein 95 (psd-95) (\u003cspan class=\"citationref\"\u003e72\u003c/span\u003e) \u0026ndash; zglun2aa (50% identity; 63% similarity) and zglun2ab (38% identity, 63% similarity); and the pdz domain that interacts with the proteins psd-95, discs large (dlg), and zonula occludens-1 (zo-1) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2aa and zglun2ab (100% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,c). interacting regions in glun2a include: ring finger protein 10 (rnf10) (\u003cspan class=\"citationref\"\u003e70\u003c/span\u003e) \u0026ndash; zglun2aa (51% identity, 72% similarity) and zglun2ab (32% identity, 66% similarity); rabphilin 3a (rph3a) (\u003cspan class=\"citationref\"\u003e68\u003c/span\u003e) \u0026ndash; zglun2aa (71% identity, 80% similarity) and zglun2ab (49% identity, 71% similarity); flotillin-1 (flot-1) (\u003cspan class=\"citationref\"\u003e71\u003c/span\u003e) \u0026ndash; zglun2aa (61% identity, 81% similarity) and zglun2ab (31% identity, 52% similarity); and c-terminal binding protein 1 (ctbp1) (\u003cspan class=\"citationref\"\u003e69\u003c/span\u003e) \u0026ndash; zglun2aa (51% identity, 85% similarity) and zglun2ab (36% identity, 63% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,c, s3a,b).\u003c/p\u003e\n\u003cp\u003efor glun2b, the conservation of slims is: the guanine-nucleotide exchange factor brag1 (\u003cspan class=\"citationref\"\u003e76\u003c/span\u003e) \u0026ndash; zglun2ba (60% identity; 80% similarity) and zglun2bb (60% identity, 100% similarity); psd-95 (\u003cspan class=\"citationref\"\u003e72\u003c/span\u003e) \u0026ndash; zglun2ba (56% identity \u0026amp; similarity) and zglun2bb (33% identity, 56% similarity); camkii and death-associated protein kinase 1 (dapk1) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (92% identity, 100% similarity); synapse-associated protein 102 (sap102) (\u003cspan class=\"citationref\"\u003e75\u003c/span\u003e) \u0026ndash; zglun2ba (0% identity \u0026amp; similarity) and zglun2bb (50% identity \u0026amp; similarity); the adaptor complex ap2 (\u003cspan class=\"citationref\"\u003e74\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (100% identity \u0026amp; similarity); and pdz (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (100% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,d). the ctd interaction regions for glun2b are conserved as follows: receptor for activated c kinase 1 (rack1) (\u003cspan class=\"citationref\"\u003e77\u003c/span\u003e) \u0026ndash; zglun2ba (0% identity, 7% similarity) and zglun2bb (13% identity, 20% similarity); flot-1 (\u003cspan class=\"citationref\"\u003e71\u003c/span\u003e) \u0026ndash; zglun2ba (23% identity; 47% similarity) and zglun2bb (35% identity, 68% similarity); spectrin (\u003cspan class=\"citationref\"\u003e79\u003c/span\u003e) \u0026ndash; zglun2ba (40% identity; 59% similarity) and zglun2bb (35% identity, 62% similarity); \u0026alpha;-actinin 2 (\u003cspan class=\"citationref\"\u003e78\u003c/span\u003e) \u0026ndash; zglun2ba (47% identity; 67% similarity) and zglun2bb (51% identity, 76% similarity); and ras protein-specific guanine nucleotide-releasing factor 1 (rasgrf1) (\u003cspan class=\"citationref\"\u003e80\u003c/span\u003e) \u0026ndash; zglun2ba (59% identity; 74% similarity) and zglun2bb (60% identity, 78% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,d, s3a,c).\u003c/p\u003e\n\u003cp\u003ewhen compared globally across subunits, slims always show greater conservation than regions. this conservation is always greatest in the zebrafish glun1 paralogs, followed by the glun2b and then glun2a paralogs. for the analyzed slims, conservation grouped across each subunit is as follows: zglun1a and zglun1b (100% identity \u0026amp; similarity), zglun2aa (76% identity, 81% similarity), zglun2ab (62% identity, 76% similarity), zglun2ba (78% identity, 81% similarity), and zglun2bb (72% identity, 86% similarity). the regions show the following degrees of conservation: zglun1a and zglun1b (94% identity, 98% similarity), zglun2aa (55% identity, 81% similarity), zglun2ab (36% identity, 62% similarity), zglun2ba (59% identity, 74% similarity), and zglun2bb (60% identity, 78% similarity).\u003c/p\u003e"},{"header":"discussion","content":"\u003cp\u003ehere, we compared the conservation of the zebrafish nmdar subunit paralogues, including key structure-function motifs, to their human counterparts. we find that the core gating machinery \u0026ndash; the lbd, the lbd-tmd linkers and the tmd \u0026ndash; are highly conserved both in terms of overall identity and specific functional motifs. the more peripheral domains \u0026ndash; the ntd and ctd \u0026ndash; show less conservation overall, though functional motifs are reasonably conserved with the notable exception of zn\u003csup\u003e2+\u003c/sup\u003e binding in the glun2a ntd. notably, disease-associated variants are generally conserved, highlighting that zebrafish represent a useful model to study nmdars.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar conservation in zebrafish\u003c/b\u003e\u003c/p\u003e\u003cp\u003eacross most of our examined parameters, the obligatory glun1 subunit shows the highest degree of conservation in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e). generally, conservation is then greatest in the glun2(a-d) followed by the glun3(a-b) subunits. within each subunit, sequence conservation is consistently highest in the tmd and lbd-tmd linkers, followed by the lbd. conservation is generally poorer in the ntd and even worse in the ctd (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ea key question with regards to this work is defining what constitutes as a \u0026lsquo;well conserved\u0026rsquo; sequence. previous studies regarding the conservation of protein structure and function suggest that sequence identities between 40\u0026ndash;70% confer functional conservation (\u003cspan additionalcitationids=\"cr93\" citationid=\"cr92\" class=\"citationref\"\u003e92\u003c/span\u003e\u0026ndash;\u003cspan citationid=\"cr94\" class=\"citationref\"\u003e94\u003c/span\u003e). they regard 50% sequence identity as the threshold below which function drastically diverges (\u003cspan citationid=\"cr93\" class=\"citationref\"\u003e93\u003c/span\u003e, \u003cspan citationid=\"cr94\" class=\"citationref\"\u003e94\u003c/span\u003e). our sequence conservation analyses almost always led to sequence identities\u0026thinsp;\u0026gt;\u0026thinsp;50%, except for zglun2da. nevertheless, we carried out more detailed examinations of more poorly conserved domains, e.g., the ntd and ctd, to assess the conservation of key structure-function motifs.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar missense variant conservation in zebrafish\u003c/b\u003e\u003c/p\u003e\u003cp\u003eto date, a multitude of variants, including missense, frameshift, and nonsense mutations, have been identified in the genes encoding the nmdar subunits (\u003cspan citationid=\"cr21\" class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan citationid=\"cr22\" class=\"citationref\"\u003e22\u003c/span\u003e, \u003cspan citationid=\"cr87\" class=\"citationref\"\u003e87\u003c/span\u003e). we categorized identified missense variants by pathogenicity and analyzed the extent of conservation of the positions at which they occur. we focused only on the glun1, glun2a, glun2b, and glun2d subunits for this analysis, as this is where most disease-associated variants have been identified. generally, our results demonstrate a higher degree of variant position conservation with increasing pathogenicity (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e,\u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e). the sites of pathogenic missense variants are almost perfectly conserved (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e). given the high conservation of the amino acid positions at which they occur, these protein regions are likely most essential for receptor function. across our categories of variants, with the exclusion of complex variants, the proportion of the total protein at which missense variants have been identified is lowest in the pathogenic group (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e). it is possible that additional variants that might be categorized as pathogenic are so devastating to development as to be embryonic lethal. this reflects the notion that more integral regions of the receptor are less tolerant to variants (\u003cspan citationid=\"cr95\" class=\"citationref\"\u003e95\u003c/span\u003e). zebrafish, nevertheless, emerge as a useful model for the study of pathogenic nmdar variants.\u003c/p\u003e\u003cp\u003eputatively pathogenic variants are conserved to a lesser extent than their pathogenic counterparts (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e). according to our designations, these variants have the potential to cause disease, having been identified in patients, but are currently classified as having uncertain significance. thus, it aligns that these variants are comparatively less well conserved \u0026ndash; they are likely occurring in less critical regions of the receptor and are likewise less likely to cause disease phenotypes.\u003c/p\u003e\u003cp\u003eour subsequent investigation of putatively benign and benign variants revealed a further decrease in variant position conservation. this finding further supported our prediction that variants more highly associated with disease occur at positions with higher degrees of conservation and, thus, are more critical to protein structure and function. the conservation of positions with putatively benign and benign variants is quite comparable. for the benign variants, in particular, conservation percentages also closely reflect total protein conservation values for each paralog (fig.\u0026nbsp;\u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e, table\u0026nbsp;\u003cspan refid=\"tab1\" class=\"internalref\"\u003e1\u003c/span\u003e). this follows from the finding that positions with identified benign variants, for most subunits, make up a substantial portion of the protein (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enevertheless, this investigation both demonstrates the advantage of the zebrafish model in studying nmdar-associated disease and reflects the notion that the positions most essential for receptor function are those that are best conserved and more likely to induce disease when absent.\u003c/p\u003e\u003cp\u003eoverall, the trend for variant conservation is the same as that for the whole protein. conservation is always greatest in the tmd and lbd-tmd linkers, followed by the lbd, next the ntd, and last the ctd (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e, \u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e). these findings suggest that zebrafish serve as an effective model for variants, especially pathogenic and putatively pathogenic variants in both the tmd and lbd-tmd linkers. some caution, however, must be exercised when using this model system for variants in the lbd, ntd, and ctd.\u003c/p\u003e\u003cp\u003e\u003cb\u003etmd, tmd-lbd linker, and lbd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003eour examination of key functional regions and motifs in the tmd, lbd-tmd linkers, and lbd consistently revealed complete conservation in their respective zebrafish paralogs (fig. \u003cspan refid=\"moesm1\" class=\"internalref\"\u003es1\u003c/span\u003e,s2). this suggests that the basic mechanism of ion channel gating is conserved in zebrafish relative to human. as these domains and subdomains contribute to the formation of the core of the receptor \u0026ndash; the ion channel pore allowing for ion flux \u0026ndash; it is expected that they would be most highly conserved across species, since the major functional role of nmdars is current flux during synaptic transmission.\u003c/p\u003e\u003cp\u003e\u003cb\u003entd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003ethe ntd is the first region in which nmdar sequences in zebrafish begin to diverge from those in human. modelling ntd variants in zebrafish must, thus, be approached with caution. for example, polyamine-binding sites in glun2b are almost completely conserved in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig8\" class=\"internalref\"\u003e4\u003c/span\u003ed), so they have the potential to serve as an effective model for this functional role of nmdars. on the other hand, zinc-binding sites in glun2a and glun2b are more poorly conserved in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig8\" class=\"internalref\"\u003e4\u003c/span\u003ec), so they would likely serve as a poor model for zinc-related studies in nmdars. this also aligns with our homology model finding that the zglun2aa model aligns well with glun2a when not bound to zinc, but the rmsd of the alignment is substantially poorer when glun2a interacting with zinc is used as the reference structure (table s3).\u003c/p\u003e\u003cp\u003e\u003cb\u003ectd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003ethe c-terminal domains (ctds) exhibit the lowest level of overall conservation across nmdar domains (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e). this is consistent with the ctd\u0026rsquo;s classification as an intrinsically disordered region (idr), which lacks fixed secondary or tertiary structure and is more tolerant to evolutionary divergence (\u003cspan citationid=\"cr96\" class=\"citationref\"\u003e96\u003c/span\u003e, \u003cspan citationid=\"cr97\" class=\"citationref\"\u003e97\u003c/span\u003e). nonetheless, functional conservation within idrs is often maintained through the retention of post-translational modification (ptm) hotspots and short linear motifs (slims) that regulate protein interactions and intracellular signaling (\u003cspan citationid=\"cr98\" class=\"citationref\"\u003e98\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ewe assessed ctd conservation in glun1, glun2a, and glun2b by analyzing experimentally validated phosphorylation, palmitoylation, and ubiquitylation sites, as well as defined protein-binding motifs. phosphorylation sites with known kinase interactions exhibited relatively high conservation, consistent with their well-characterized regulatory roles (figs.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ea,b). in contrast, phosphorylation sites without identified kinases were less consistently retained (fig.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ec). palmitoylation sites were fully conserved across all subunits, while ubiquitylation sites were generally retained, though conservation was notably lower in zglun2ab (fig.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ef) and may reflect paralogue-specific divergence in degradation or trafficking pathways.\u003c/p\u003e\u003cp\u003eprotein-binding motif conservation varied by subunit. glun1 retained nearly all canonical interaction motifs, while glun2a and glun2b displayed region-specific conservation. notably well-conserved elements in the glun2 subunits included the pdz-binding motifs as well as docking regions for camkii, dapk1, and ap2 (fig.\u0026nbsp;\u003cspan refid=\"fig12\" class=\"internalref\"\u003e6\u003c/span\u003e). slims overall showed higher conservation, likely due to their short, sequence-specific structure, which may place them under stronger evolutionary constraint than broader, more flexible interaction regions.\u003c/p\u003e\u003cp\u003edespite broad sequence divergence, nmdar ctds retain a core set of conserved motifs involved in receptor trafficking, synaptic localization, and plasticity. kinase-regulated phosphorylation sites, palmitoylation sites, and pdz-binding motifs were consistently preserved. conservation of glun1 sites was always greatest in the zebrafish paralogs. among the glun2 subunits, glun2b displayed the highest overall conservation of functional elements, aligning with its essential role in neurodevelopment and synaptic signaling.\u003c/p\u003e\u003cp\u003ethese findings are especially relevant to the study of disease-associated truncating variants in the nmdar subunits. nonsense variants that introduce premature stop codons in the ctd eliminate key regulatory motifs critical for trafficking, anchoring, or signaling. when these affected regions correspond to conserved functional elements, zebrafish may serve as an appropriate \u003cem\u003ein vitro\u003c/em\u003e and \u003cem\u003ein vivo\u003c/em\u003e model for studying ctd-mediated disease mechanisms. conversely, for mutations in poorly conserved regions, the translational utility of this model may become limited.\u003c/p\u003e"},{"header":"conclusions","content":"\u003cp\u003ezebrafish represent a valuable system to study nmdar-associated neurodevelopmental disorders, but one must be judicious in choosing which variants to study in this model. our unique approach of sequence alignments per individual nmdar domains and subdomains highlights the high degree of conversation of some regions of the protein relative to others. additionally, our analysis of identified variants indicates that zebrafish are an effective model for pathogenic variants as these occur at conserved locations along the protein. generally, the utility of zebrafish is most apparent for the study of the tmd, lbd-tmd linkers, and lbd as these regions are most well conserved. on the other hand, studying the ntd and ctd in this species requires extra consideration stemming from deviations in sequence conservation. our work will enable future nmdar-related studies that can be effectively and efficiently conducted in zebrafish.\u003c/p\u003e"},{"header":"abbreviations","content":"\u003ctable border=\"0\" cellspacing=\"0\" cellpadding=\"0\"\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003enmdar\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003e\u003cem\u003en\u003c/em\u003e-methyl-d-aspartate receptor\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003entd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003en-terminal domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003elbd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003eligand-binding domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003etmd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003etransmembrane domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003ectd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003ec-terminal domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e"},{"header":"declarations","content":"\u003cp\u003e\u003cstrong\u003eethics approval and consent to participate:\u0026nbsp;\u003c/strong\u003enot applicable.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econsent for publication:\u0026nbsp;\u003c/strong\u003enot applicable.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eavailability of data and materials:\u003c/strong\u003e the datasets used and/or analyzed during the current study are available either at osf (https://osf.io/h2zf9/) or from the corresponding author on reasonable request.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecompeting interests:\u0026nbsp;\u003c/strong\u003eno competing interests declared.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003efunding:\u003c/strong\u003e this work was supported by nih grant r01ns088479 to lpw.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eauthors\u0026rsquo; contributions:\u0026nbsp;\u003c/strong\u003ee.r.n., c.a., j.d.z., h.i.s., and l.p.w. contributed to the conceptualization and methodology of this project. e.r.n., c.a., and j.d.z. conducted the investigation and analysis. w.r. assisted with data curation. e.r.n, c.a., and l.p.w. wrote the manuscript, which was revised with insights from j.d.z. and h.i.s.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eacknowledgements:\u0026nbsp;\u003c/strong\u003ethe authors thank the clinvar and gnomad databases and the swiss-model server for their important contributions to this work.\u003c/p\u003e"},{"header":"references","content":"\u003col\u003e\n\u003cli\u003epaoletti p, bellone c, zhou q. nmda receptor subunit diversity: impact on receptor properties, synaptic plasticity and disease. nat rev neurosci. 2013;14(6):383-400.\u003c/li\u003e\n\u003cli\u003ehansen kb, wollmuth lp, bowie d, furukawa h, menniti fs, sobolevsky ai, et al. structure, function, and pharmacology of glutamate receptor ion channels. pharmacol rev. 2021;73(4):298-487.\u003c/li\u003e\n\u003cli\u003echakraborty a, murphy s, coleman n. the role of nmda receptors in neural stem cell proliferation and differentiation. stem cells dev. 2017;26(11):798-807.\u003c/li\u003e\n\u003cli\u003enacher j, mcewen bs. the role of n-methyl-d-asparate receptors in neurogenesis. hippocampus. 2006;16(3):267-70.\u003c/li\u003e\n\u003cli\u003ekomuro h, rakic p. modulation of neuronal migration by nmda receptors. science. 1993;260(5104):95-7.\u003c/li\u003e\n\u003cli\u003epearce ia, cambray-deakin ma, burgoyne rd. glutamate acting on nmda receptors stimulates neurite outgrowth from cerebellar granule cells. febs lett. 1987;223(1):143-7.\u003c/li\u003e\n\u003cli\u003ebooker sa, wyllie dja. nmda receptor function in inhibitory neurons. neuropharmacology. 2021;196:108609.\u003c/li\u003e\n\u003cli\u003egambrill ac, barria a. nmda receptor subunit composition controls synaptogenesis and synapse stabilization. proc natl acad sci u s a. 2011;108(14):5855-60.\u003c/li\u003e\n\u003cli\u003enapoli aj, laderwager s, zoodsma jd, biju b, mucollari o, schubel sk, et al. developmental loss of nmda receptors results in supernumerary forebrain neurons through delayed maturation of transit-amplifying neuroblasts. sci rep. 2024;14(1):3395.\u003c/li\u003e\n\u003cli\u003erajani v, sengar as, salter mw. tripartite signalling by nmda receptors. mol brain. 2020;13(1):23.\u003c/li\u003e\n\u003cli\u003eishchenko y, carrizales mg, koleske aj. regulation of the nmda receptor by its cytoplasmic domains: (how) is the tail wagging the dog? neuropharmacology. 2021;195:108634.\u003c/li\u003e\n\u003cli\u003emony l, paoletti p. mechanisms of nmda receptor regulation. curr opin neurobiol. 2023;83:102815.\u003c/li\u003e\n\u003cli\u003enielsen ac, anderson cl, ens c, boyce akj, thompson rj. non-ionotropic nmdar signalling activates panx1 to induce p2x4r-dependent long-term depression in the hippocampus. j physiol. 2024.\u003c/li\u003e\n\u003cli\u003ebarnes sa, thomazeau a, finnie psb, heinrich mj, heynen aj, komiyama nh, et al. non-ionotropic signaling through the nmda receptor glun2b carboxy-terminal domain drives dendritic spine plasticity and reverses fragile x phenotypes. cell rep. 2025;44(3):115311.\u003c/li\u003e\n\u003cli\u003eplatzer k, lemke jr. grin1-related neurodevelopmental disorder. genereviews((r)). 2021.\u003c/li\u003e\n\u003cli\u003estrehlow v, myers ka, morgan at, scheffer ie, lemke jr. grin2a-related disorders. genereviews((r)). 2024.\u003c/li\u003e\n\u003cli\u003eplatzer k, lemke jr. grin2b-related neurodevelopmental disorder. genereviews((r)). 2021.\u003c/li\u003e\n\u003cli\u003eplatzer k, krey i, lemke jr. grin2d-related developmental and epileptic encephalopathy. genereviews((r)). 2022.\u003c/li\u003e\n\u003cli\u003ezehavi y, mandel h, zehavi a, rashid ma, straussberg r, jabur b, et al. de novo grin1 mutations: an emerging cause of severe early infantile encephalopathy. eur j med genet. 2017;60(6):317-20.\u003c/li\u003e\n\u003cli\u003esabo sl, lahr jm, offer m, weekes a, sceniak mp. grin2b-related neurodevelopmental disorder: current understanding of pathophysiological mechanisms. front synaptic neurosci. 2022;14:1090865.\u003c/li\u003e\n\u003cli\u003egjerulfsen ce, krey i, klockner c, rubboli g, lemke jr, moller rs. spectrum of nmda receptor variants in neurodevelopmental disorders and epilepsy. methods mol biol. 2024;2799:1-11.\u003c/li\u003e\n\u003cli\u003eamin jb, moody gr, wollmuth lp. from bedside-to-bench: what disease-associated variants are teaching us about the nmda receptor. j physiol. 2021;599(2):397-416.\u003c/li\u003e\n\u003cli\u003ehu c, chen w, myers sj, yuan h, traynelis sf. human grin2b variants in neurodevelopmental disorders. j pharmacol sci. 2016;132(2):115-21.\u003c/li\u003e\n\u003cli\u003ecox ja, kucenas s, voigt mm. molecular characterization and embryonic expression of the family of n-methyl-d-aspartate receptor subunit genes in the zebrafish. dev dyn. 2005;234(3):756-66.\u003c/li\u003e\n\u003cli\u003eveldman mb, lin s. zebrafish as a developmental model organism for pediatric research. pediatr res. 2008;64(5):470-6.\u003c/li\u003e\n\u003cli\u003echoi ty, choi ti, lee yr, choe sk, kim ch. zebrafish as an animal model for biomedical research. exp mol med. 2021;53(3):310-7.\u003c/li\u003e\n\u003cli\u003ezoodsma jd, chan k, bhandiwad aa, golann dr, liu g, syed sa, et al. a model to study nmda receptors in early nervous system development. j neurosci. 2020;40(18):3631-45.\u003c/li\u003e\n\u003cli\u003ezoodsma jd, keegan ej, moody gr, bhandiwad aa, napoli aj, burgess ha, et al. disruption of grin2b, an asd-associated gene, produces social deficits in zebrafish. mol autism. 2022;13(1):38.\u003c/li\u003e\n\u003cli\u003ebaraban sc, taylor mr, castro pa, baier h. pentylenetetrazole induced changes in zebrafish behavior, neural activity and c-fos expression. neuroscience. 2005;131(3):759-68.\u003c/li\u003e\n\u003cli\u003eshams s, rihel j, ortiz jg, gerlai r. the zebrafish as a promising tool for modeling human brain disorders: a review based upon an ibns symposium. neurosci biobehav rev. 2018;85:176-90.\u003c/li\u003e\n\u003cli\u003ede abreu ms, genario r, giacomini a, demin ka, lakstygal am, amstislavskaya tg, et al. zebrafish as a model of neurodevelopmental disorders. neuroscience. 2020;445:3-11.\u003c/li\u003e\n\u003cli\u003eweinschutz mendes h, neelakantan u, liu y, fitzpatrick se, chen t, wu w, et al. high-throughput functional analysis of autism genes in zebrafish identifies convergence in dopaminergic and neuroimmune pathways. cell rep. 2023;42(3):112243.\u003c/li\u003e\n\u003cli\u003ehowe k, clark md, torroja cf, torrance j, berthelot c, muffato m, et al. the zebrafish reference genome sequence and its relationship to the human genome. nature. 2013;496(7446):498-503.\u003c/li\u003e\n\u003cli\u003erihel j, schier af. behavioral screening for neuroactive drugs in zebrafish. dev neurobiol. 2012;72(3):373-85.\u003c/li\u003e\n\u003cli\u003ebaraban sc, dinday mt, hortopan ga. drug screening in scn1a zebrafish mutant identifies clemizole as a potential dravet syndrome treatment. nat commun. 2013;4:2410.\u003c/li\u003e\n\u003cli\u003ezhang y, kecskes a, copmans d, langlois m, crawford ad, ceulemans b, et al. pharmacological characterization of an antisense knockdown zebrafish model of dravet syndrome: inhibition of epileptic seizures by the serotonin agonist fenfluramine. plos one. 2015;10(5):e0125898.\u003c/li\u003e\n\u003cli\u003edingemans ajm, jansen s, van reeuwijk j, de leeuw n, pfundt r, schuurs-hoeijmakers j, et al. prevalence of comorbidities in individuals with neurodevelopmental disorders from the aggregated phenomics data of 51,227 pediatric individuals. nat med. 2024;30(7):1994-2003.\u003c/li\u003e\n\u003cli\u003eohtsuki t, sakurai k, dou h, toru m, yamakawa-kobayashi k, arinami t. mutation analysis of the nmdar2b (grin2b) gene in schizophrenia. mol psychiatry. 2001;6(2):211-6.\u003c/li\u003e\n\u003cli\u003earnold pd, rosenberg dr, mundo e, tharmalingam s, kennedy jl, richter ma. association of a glutamate (nmda) subunit receptor gene (grin2b) with obsessive-compulsive disorder: a preliminary study. psychopharmacology (berl). 2004;174(4):530-8.\u003c/li\u003e\n\u003cli\u003esaunders ja, tatard-leitman vm, suh j, billingslea en, roberts tp, siegel sj. knockout of nmda receptors in parvalbumin interneurons recreates autism-like phenotypes. autism res. 2013;6(2):69-77.\u003c/li\u003e\n\u003cli\u003egandal mj, sisti j, klook k, ortinski pi, leitman v, liang y, et al. gabab-mediated rescue of altered excitatory-inhibitory balance, gamma synchrony and behavioral deficits following constitutive nmdar-hypofunction. transl psychiatry. 2012;2(7):e142.\u003c/li\u003e\n\u003cli\u003ebarnby g, abbott a, sykes n, morris a, weeks de, mott r, et al. candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at grin2a and abat. am j hum genet. 2005;76(6):950-66.\u003c/li\u003e\n\u003cli\u003ehung ly, margolis kg. autism spectrum disorders and the gastrointestinal tract: insights into mechanisms and clinical relevance. nat rev gastroenterol hepatol. 2024;21(3):142-63.\u003c/li\u003e\n\u003cli\u003eowens ap, mathias cj, iodice v. autonomic dysfunction in autism spectrum disorder. front integr neurosci. 2021;15:787037.\u003c/li\u003e\n\u003cli\u003eorefice ll. peripheral somatosensory neuron dysfunction: emerging roles in autism spectrum disorders. neuroscience. 2020;445:120-9.\u003c/li\u003e\n\u003cli\u003emakris g, agorastos a, chrousos gp, pervanidou p. stress system activation in children and adolescents with autism spectrum disorder. front neurosci. 2021;15:756628.\u003c/li\u003e\n\u003cli\u003eramos-vicente d, grant sg, bayes a. metazoan evolution and diversity of glutamate receptors and their auxiliary subunits. neuropharmacology. 2021;195:108640.\u003c/li\u003e\n\u003cli\u003eamores a, force a, yan yl, joly l, amemiya c, fritz a, et al. zebrafish hox clusters and vertebrate genome evolution. science. 1998;282(5394):1711-4.\u003c/li\u003e\n\u003cli\u003echou th, tajima n, romero-hernandez a, furukawa h. structural basis of functional transitions in mammalian nmda receptors. cell. 2020;182(2):357-71 e13.\u003c/li\u003e\n\u003cli\u003ezhang y, ye f, zhang t, lv s, zhou l, du d, et al. structural basis of ketamine action on human nmda receptors. nature. 2021;596(7871):301-5.\u003c/li\u003e\n\u003cli\u003ewollmuth lp, sakmann b. different mechanisms of ca2+ transport in nmda and ca2+-permeable ampa glutamate receptor channels. j gen physiol. 1998;112(5):623-36.\u003c/li\u003e\n\u003cli\u003ewatanabe j, beck c, kuner t, premkumar ls, wollmuth lp. drpeer: a motif in the extracellular vestibule conferring high ca2+ flux rates in nmda receptor channels. j neurosci. 2002;22(23):10209-16.\u003c/li\u003e\n\u003cli\u003ewollmuth lp, sobolevsky ai. structure and gating of the glutamate receptor ion channel. trends neurosci. 2004;27(6):321-8.\u003c/li\u003e\n\u003cli\u003eogden kk, khatri a, traynelis sf, heldt sa. potentiation of glun2c/d nmda receptor subtypes in the amygdala facilitates the retention of fear and extinction learning in mice. neuropsychopharmacology. 2014;39(3):625-37.\u003c/li\u003e\n\u003cli\u003ewollmuth lp. ion permeation in ionotropic glutamate receptors: still dynamic after all these years. curr opin physiol. 2018;2:36-41.\u003c/li\u003e\n\u003cli\u003emcdaniel mj, ogden kk, kell sa, burger pb, liotta dc, traynelis sf. nmda receptor channel gating control by the pre-m1 helix. j gen physiol. 2020;152(4).\u003c/li\u003e\n\u003cli\u003ewollmuth lp, kuner t, sakmann b. adjacent asparagines in the nr2-subunit of the nmda receptor channel control the voltage-dependent block by extracellular mg2+. j physiol. 1998;506 ( pt 1)(pt 1):13-32.\u003c/li\u003e\n\u003cli\u003eromero-hernandez a, simorowski n, karakas e, furukawa h. molecular basis for subtype specificity and high-affinity zinc inhibition in the glun1-glun2a nmda receptor amino-terminal domain. neuron. 2016;92(6):1324-36.\u003c/li\u003e\n\u003cli\u003emony l, zhu s, carvalho s, paoletti p. molecular basis of positive allosteric modulation of glun2b nmda receptors by polyamines. embo j. 2011;30(15):3134-46.\u003c/li\u003e\n\u003cli\u003elichnerova k, kaniakova m, park sp, skrenkova k, wang yx, petralia rs, et al. two n-glycosylation sites in the glun1 subunit are essential for releasing n-methyl-d-aspartate (nmda) receptors from the endoplasmic reticulum. j biol chem. 2015;290(30):18379-90.\u003c/li\u003e\n\u003cli\u003ewaterhouse a, bertoni m, bienert s, studer g, tauriello g, gumienny r, et al. swiss-model: homology modelling of protein structures and complexes. nucleic acids res. 2018;46(w1):w296-w303.\u003c/li\u003e\n\u003cli\u003ebienert s, waterhouse a, de beer ta, tauriello g, studer g, bordoli l, et al. the swiss-model repository-new features and functionality. nucleic acids res. 2017;45(d1):d313-d9.\u003c/li\u003e\n\u003cli\u003ewarnet xl, bakke krog h, sevillano-quispe og, poulsen h, kjaergaard m. the c-terminal domains of the nmda receptor: how intrinsically disordered tails affect signalling, plasticity and disease. eur j neurosci. 2021;54(8):6713-39.\u003c/li\u003e\n\u003cli\u003ekim k, saneyoshi t, hosokawa t, okamoto k, hayashi y. interplay of enzymatic and structural functions of camkii in long-term potentiation. j neurochem. 2016;139(6):959-72.\u003c/li\u003e\n\u003cli\u003ebej a, ames jb. chemical shift assignments of calmodulin bound to a cytosolic domain of glun2a (residues 1004-1024) from the nmda receptor. biomol nmr assign. 2023;17(1):89-93.\u003c/li\u003e\n\u003cli\u003ehorak m, wenthold rj. different roles of c-terminal cassettes in the trafficking of full-length nr1 subunits to the cell surface. j biol chem. 2009;284(15):9683-91.\u003c/li\u003e\n\u003cli\u003elin jw, wyszynski m, madhavan r, sealock r, kim ju, sheng m. yotiao, a novel protein of neuromuscular junction and brain that interacts with specific splice variants of nmda receptor subunit nr1. j neurosci. 1998;18(6):2017-27.\u003c/li\u003e\n\u003cli\u003estanic j, carta m, eberini i, pelucchi s, marcello e, genazzani aa, et al. rabphilin 3a retains nmda receptors at synaptic sites through interaction with glun2a/psd-95 complex. nat commun. 2015;6:10181.\u003c/li\u003e\n\u003cli\u003ecousins sl, stephenson fa. identification of c-terminal binding protein 1 as a novel nmda receptor interactor. neurochem res. 2019;44(6):1437-45.\u003c/li\u003e\n\u003cli\u003efranchini l, carrano n, di luca m, gardoni f. synaptic glun2a-containing nmda receptors: from physiology to pathological synaptic plasticity. int j mol sci. 2020;21(4).\u003c/li\u003e\n\u003cli\u003eswanwick cc, shapiro me, yi z, chang k, wenthold rj. nmda receptors interact with flotillin-1 and -2, lipid raft-associated proteins. febs lett. 2009;583(8):1226-30.\u003c/li\u003e\n\u003cli\u003ecousins sl, stephenson fa. identification of n-methyl-d-aspartic acid (nmda) receptor subtype-specific binding sites that mediate direct interactions with scaffold protein psd-95. j biol chem. 2012;287(16):13465-76.\u003c/li\u003e\n\u003cli\u003evissel b, krupp jj, heinemann sf, westbrook gl. a use-dependent tyrosine dephosphorylation of nmda receptors is independent of ion flux. nat neurosci. 2001;4(6):587-96.\u003c/li\u003e\n\u003cli\u003elavezzari g, mccallum j, lee r, roche kw. differential binding of the ap-2 adaptor complex and psd-95 to the c-terminus of the nmda receptor subunit nr2b regulates surface expression. neuropharmacology. 2003;45(6):729-37.\u003c/li\u003e\n\u003cli\u003echen bs, gray ja, sanz-clemente a, wei z, thomas ev, nicoll ra, et al. sap102 mediates synaptic clearance of nmda receptors. cell rep. 2012;2(5):1120-8.\u003c/li\u003e\n\u003cli\u003eelagabani mn, brisevac d, kintscher m, pohle j, kohr g, schmitz d, et al. subunit-selective n-methyl-d-aspartate (nmda) receptor signaling through brefeldin a-resistant arf guanine nucleotide exchange factors brag1 and brag2 during synapse maturation. j biol chem. 2016;291(17):9105-18.\u003c/li\u003e\n\u003cli\u003eyaka r, thornton c, vagts aj, phamluong k, bonci a, ron d. nmda receptor function is regulated by the inhibitory scaffolding protein, rack1. proc natl acad sci u s a. 2002;99(8):5710-5.\u003c/li\u003e\n\u003cli\u003ewyszynski m, lin j, rao a, nigh e, beggs ah, craig am, et al. competitive binding of alpha-actinin and calmodulin to the nmda receptor. nature. 1997;385(6615):439-42.\u003c/li\u003e\n\u003cli\u003ewechsler a, teichberg vi. brain spectrin binding to the nmda receptor is regulated by phosphorylation, calcium and calmodulin. embo j. 1998;17(14):3931-9.\u003c/li\u003e\n\u003cli\u003ekrapivinsky g, krapivinsky l, manasian y, ivanov a, tyzio r, pellegrino c, et al. the nmda receptor is coupled to the erk pathway by a direct interaction between nr2b and rasgrf1. neuron. 2003;40(4):775-84.\u003c/li\u003e\n\u003cli\u003emukherjee p, roy s, ghosh d, nandi sk. role of animal models in biomedical research: a review. lab anim res. 2022;38(1):18.\u003c/li\u003e\n\u003cli\u003echou th, epstein m, fritzemeier rg, akins ns, paladugu s, ullman ez, et al. molecular mechanism of ligand gating and opening of nmda receptor. nature. 2024;632(8023):209-17.\u003c/li\u003e\n\u003cli\u003earmstrong n, sun y, chen gq, gouaux e. structure of a glutamate-receptor ligand-binding core in complex with kainate. nature. 1998;395(6705):913-7.\u003c/li\u003e\n\u003cli\u003earmstrong n, gouaux e. mechanisms for activation and antagonism of an ampa-sensitive glutamate receptor: crystal structures of the glur2 ligand binding core. neuron. 2000;28(1):165-81.\u003c/li\u003e\n\u003cli\u003ekazi r, gan q, talukder i, markowitz m, salussolia cl, wollmuth lp. asynchronous movements prior to pore opening in nmda receptors. j neurosci. 2013;33(29):12052-66.\u003c/li\u003e\n\u003cli\u003eamin jb, gochman a, he m, certain n, wollmuth lp. nmda receptors require multiple pre-opening gating steps for efficient synaptic activity. neuron. 2021;109(3):488-501 e4.\u003c/li\u003e\n\u003cli\u003ehardingham ge, do kq. linking early-life nmdar hypofunction and oxidative stress in schizophrenia pathogenesis. nat rev neurosci. 2016;17(2):125-34.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, riley gr, jang w, rubinstein ws, church dm, et al. clinvar: public archive of relationships among sequence variation and human phenotype. nucleic acids res. 2014;42(database issue):d980-5.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, benson m, brown g, chao c, chitipiralla s, et al. clinvar: public archive of interpretations of clinically relevant variants. nucleic acids res. 2016;44(d1):d862-8.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, benson m, brown gr, chao c, chitipiralla s, et al. clinvar: improving access to variant interpretations and supporting evidence. nucleic acids res. 2018;46(d1):d1062-d7.\u003c/li\u003e\n\u003cli\u003egudmundsson s, singer-berk m, watts na, phu w, goodrich jk, solomonson m, et al. variant interpretation using population databases: lessons from gnomad. hum mutat. 2022;43(8):1012-30.\u003c/li\u003e\n\u003cli\u003etian w, skolnick j. how well is enzyme function conserved as a function of pairwise sequence identity? j mol biol. 2003;333(4):863-82.\u003c/li\u003e\n\u003cli\u003eaddou s, rentzsch r, lee d, orengo ca. domain-based and family-specific sequence identity thresholds increase the levels of reliable protein function transfer. j mol biol. 2009;387(2):416-30.\u003c/li\u003e\n\u003cli\u003esangar v, blankenberg dj, altman n, lesk am. quantitative sequence-function relationships in proteins based on gene ontology. bmc bioinformatics. 2007;8:294.\u003c/li\u003e\n\u003cli\u003eperszyk re, kristensen as, lyuboslavsky p, traynelis sf. three-dimensional missense tolerance ratio analysis. genome res. 2021;31(8):1447-61.\u003c/li\u003e\n\u003cli\u003euversky vn. intrinsically disordered proteins from a to z. int j biochem cell biol. 2011;43(8):1090-103.\u003c/li\u003e\n\u003cli\u003esingleton md, eisen mb. evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. plos comput biol. 2024;20(4):e1012028.\u003c/li\u003e\n\u003cli\u003edavey ne, cowan jl, shields dc, gibson tj, coldwell mj, edwards rj. slimprints: conservation-based discovery of functional motif fingerprints in intrinsically disordered protein regions. nucleic acids res. 2012;40(21):10628-41.\u003c/li\u003e\n\u003c/ol\u003e"}],"fulltextsource":"","fulltext":"","funders":[],"hasadminpriorityonworkflow":false,"hasmanuscriptdocx":true,"hasoptedintopreprint":true,"haspassedjournalqc":"","hasanypriority":false,"hidejournal":false,"highlight":"","institution":"","isacceptedbyjournal":false,"isauthorsuppliedpdf":false,"isdeskrejected":"","ishiddenfromsearch":false,"isinqc":false,"isinworkflow":false,"ispdf":false,"ispdfuptodate":true,"iswithdrawnorretracted":false,"journal":{"display":true,"email":"info@researchsquare.com","identity":"bmc-genomics","isnaturejournal":false,"hasqc":true,"allowdirectsubmit":false,"externalidentity":"gics","sidebox":"learn more about [bmc genomics](http://bmcgenomics.biomedcentral.com/)","snpcode":"","submissionurl":"https://www.editorialmanager.com/gics","title":"bmc genomics","twitterhandle":"#bmcgenomics","acdcenabled":true,"dfaenabled":false,"editorialsystem":"em","reportingportfolio":"bmc series","inreviewenabled":true,"inreviewrevisionsenabled":true},"keywords":"grin genes, clinvar, gnomad, sequence alignments, disease-associated variants, n-terminal domain, ligand-binding domain, transmembrane domain, c-terminal domain, ion channel","lastpublisheddoi":"10.21203/rs.3.rs-7151578/v1","lastpublisheddoiurl":"https://doi.org/10.21203/rs.3.rs-7151578/v1","license":{"name":"cc by 4.0","url":"https://creativecommons.org/licenses/by/4.0/"},"manuscriptabstract":"\u003ch2\u003ebackground\u003c/h2\u003e\u003cp\u003enmda receptors (nmdars) are widely expressed, glutamate-gated ion channels that play key roles in brain development and function. variants have been identified in the \u003cem\u003egrin\u003c/em\u003e genes encoding nmdar subunits that are linked to neurodevelopmental disorders, among other manifestations. zebrafish are a powerful model to study brain development and function given their rapid development and ease of genetic manipulation. as a result of an ancient genome duplication, zebrafish possess two paralogues for most human nmdar subunits. to evaluate the degree of conservation between human nmdar subunits and their respective zebrafish paralogues, we carried out detailed \u003cem\u003ein silico\u003c/em\u003e analyses, with an emphasis on key functional elements. to further assess the suitability of zebrafish for modeling nmdar-associated neurodevelopmental disorders, we analyzed the conservation of positions with identified missense variants.\u003c/p\u003e\u003ch2\u003eresults\u003c/h2\u003e\u003cp\u003ewe find that the human nmdar subunits are generally well conserved across zebrafish paralogs. moreover, variants classified as pathogenic and putatively pathogenic are highly conserved, reflecting the importance of key protein regions to neurotypical receptor function. positions with putatively benign and benign variants are less conserved. across nmdar domains, the transmembrane domain is most highly conserved, followed by the ligand-binding domain, which maintains conservation of amino acids that participate in the binding of ligands. the n-terminal domain is less well conserved but aligned homology models show high degrees of structural similarity. the c-terminal domain is the most poorly conserved region across zebrafish paralogs, but certain key regions that undergo phosphorylation, palmitoylation, and ubiquitylation as well as protein-binding motifs are better conserved.\u003c/p\u003e\u003ch2\u003econclusions\u003c/h2\u003e\u003cp\u003eour findings highlight a strong conservation of human nmdar subunits in zebrafish, with some exceptions. the ligand-binding domain, the transmembrane domain forming the ion channel and the short polypeptide linkers that connect them are highly conserved. the n- and c-terminal domains are less conserved but functional motifs in general, except for the zn\u003csup\u003e2+\u003c/sup\u003e binding site in glun2a paralogues, are more highly conserved relative to the entire domain. overall, our findings support the utility of zebrafish as a model for studying neurodevelopment and disease mechanisms and provide a template for rigorously considering the relationship between human and zebrafish paralogues.\u003c/p\u003e","manuscripttitle":"conservation of human nmda receptor subunits and disease variants in zebrafish","msid":"","msnumber":"","nondraftversions":[{"code":1,"date":"2025-07-30 18:25:39","doi":"10.21203/rs.3.rs-7151578/v1","editorialevents":[{"type":"communitycomments","content":0},{"type":"decision","content":"revision requested","date":"2025-09-24t19:23:39+00:00","index":"","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-09-19t21:34:21+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"269262391104669611722251259561774961128","date":"2025-09-19t14:02:55+00:00","index":"hide","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-09-09t17:52:57+00:00","index":"hide","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-08-18t16:11:58+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"201191728820772676273599414131783545114","date":"2025-08-04t10:39:46+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"230034227985411933590242857371107944843","date":"2025-07-28t13:05:21+00:00","index":"hide","fulltext":""},{"type":"reviewersinvited","content":"","date":"2025-07-28t07:38:37+00:00","index":"","fulltext":""},{"type":"editorassigned","content":"","date":"2025-07-28t07:10:47+00:00","index":"","fulltext":""},{"type":"editorinvited","content":"","date":"2025-07-23t18:56:26+00:00","index":"","fulltext":""},{"type":"checkscomplete","content":"","date":"2025-07-22t16:36:43+00:00","index":"","fulltext":""},{"type":"submitted","content":"bmc genomics","date":"2025-07-22t16:32:19+00:00","index":"","fulltext":""}],"status":"published","journal":{"display":true,"email":"info@researchsquare.com","identity":"bmc-genomics","isnaturejournal":false,"hasqc":true,"allowdirectsubmit":false,"externalidentity":"gics","sidebox":"learn more about [bmc genomics](http://bmcgenomics.biomedcentral.com/)","snpcode":"","submissionurl":"https://www.editorialmanager.com/gics","title":"bmc genomics","twitterhandle":"#bmcgenomics","acdcenabled":true,"dfaenabled":false,"editorialsystem":"em","reportingportfolio":"bmc series","inreviewenabled":true,"inreviewrevisionsenabled":true}}],"origin":"","owneridentity":"2b85bbad-8f38-4c35-89b3-5949e6b6d3c0","owner":[],"posteddate":"july 30th, 2025","published":true,"recenteditorialevents":[],"rejectedjournal":[],"revision":"","amendment":"","status":"under-review","subjectareas":[],"tags":[],"updatedat":"2025-10-14t16:08:28+00:00","versionofrecord":[],"versioncreatedat":"2025-07-30 18:25:39","video":"","vordoi":"","vordoiurl":"","workflowstages":[]},"version":"v1","identity":"rs-7151578","journalconfig":"researchsquare"},"__n_ssp":true},"page":"/article/[identity]/[[...version]]","query":{"identity":"rs-7151578","version":["v1"]},"buildid":"idgtm6tzxtmdq-mqt1zbo","isfallback":false,"dynamicids":[87228],"gssp":true,"scriptloader":[]}

## Results
we find that the human nmdar subunits are generally well conserved across zebrafish paralogs. moreover, variants classified as pathogenic and putatively pathogenic are highly conserved, reflecting the importance of key protein regions to neurotypical receptor function. positions with putatively benign and benign variants are less conserved. across nmdar domains, the transmembrane domain is most highly conserved, followed by the ligand-binding domain, which maintains conservation of amino acids that participate in the binding of ligands. the n-terminal domain is less well conserved but aligned homology models show high degrees of structural similarity. the c-terminal domain is the most poorly conserved region across zebrafish paralogs, but certain key regions that undergo phosphorylation, palmitoylation, and ubiquitylation as well as protein-binding motifs are better conserved. conclusions our findings highlight a strong conservation of human nmdar subunits in zebrafish, with some exceptions. the ligand-binding domain, the transmembrane domain forming the ion channel and the short polypeptide linkers that connect them are highly conserved. the n- and c-terminal domains are less conserved but functional motifs in general, except for the zn 2+ binding site in glun2a paralogues, are more highly conserved relative to the entire domain. overall, our findings support the utility of zebrafish as a model for studying neurodevelopment and disease mechanisms and provide a template for rigorously considering the relationship between human and zebrafish paralogues. grin genes clinvar gnomad sequence alignments disease-associated variants n-terminal domain ligand-binding domain transmembrane domain c-terminal domain ion channel figures figure 1 figure 2 figure 3 figure 4 figure 5 figure 6 figure 7 figure 8 background rapid cell-to-cell signaling in the nervous system occurs at specialized structures called synapses. at these sites, the electrical activity of the upstream or presynaptic neuron is converted into the release of a chemical neurotransmitter. this is then converted back into an electrical signal in the postsynaptic neuron by specialized ion channels that are gated by a neurotransmitter, initiating downstream biological signaling cascades. glutamate, the primary excitatory neurotransmitter in the brain, activates a variety of receptors that are essential for synaptic transmission and plasticity. glutamate-gated ion channels play fundamental roles in signaling in both the developing and mature nervous systems ( 1 – 9 ). the nmda receptor (nmdar) is a prominent glutamate-gated ion channel subtype ( 1 , 2 ). due to its distinct biophysical properties and unique ionotropic and non-ionotropic signaling mechanisms, it contributes to a wide array of nervous and non-nervous system functions ( 2 , 10 – 14 ). numerous missense, frameshift, and nonsense mutations (or variants) have been identified in the genes that encode nmdar subunits and are associated with neurodevelopmental disorders, including autism spectrum disorders (asd), intellectual disability, developmental delay, seizures, and schizophrenia ( 15 – 23 ). however, how these variants lead to clinical phenotypes is poorly understood. zebrafish are a powerful model organism to study neurodevelopmental phenotypes and disorders, including those linked to nmdars ( 19 , 24 – 32 ). zebrafish develop rapidly and share fundamental developmental and signaling pathways with humans and other vertebrates ( 33 ); they are transparent, allowing for dynamic and non-invasive observations of developmental and cellular processes; and they display robust behaviors that are easily assayed at early larval stages. moreover, larval zebrafish readily take up small molecules from their environment, enabling non-invasive and high-throughput readout of nervous system function in the context of behavioral drug screens ( 34 – 36 ). nmdars are heterotetramers composed of two obligatory glun1 (encoded by grin1 ) subunits. they are expressed as either diheterotetramers with two glun1 and two glun2 ( grin2a-d ) subunits or triheterotetramers of some combination of glun2 and/or glun3 ( grin3a-b ) subunits ( 2 ) (fig. 1 a). these nmdar subunits are differentially expressed over the course of neurodevelopment, with glun2a, glun2b, and glun2d being highly expressed during early development, while glun2c, glun3a, and glun3b expression increases into adulthood ( 1 ). moreover, nmdars display distinct functional and pharmacological properties based on their glun2 or glun3 composition ( 2 ). each subunit consists of four principal domains: an n-terminal domain (ntd), a ligand-binding domain (lbd), a transmembrane domain (tmd) that forms the ion channel, and a c-terminal domain (ctd) (fig. 1 b). disease-associated variants (davs) in grin1 , grin2a , grin2b , and grin2d have been identified in pediatric patients with neurodevelopmental disorders ( 15 – 18 , 21 ). these disorders often present with multiple comorbidities, reflecting overlapping neurodevelopmental phenotypes ( 37 ). generally, patients with davs in grin1 present with developmental delay and intellectual disability, though a large proportion of patients also experience seizures ( 15 ). in contrast, grin2a and grin2d are predominantly associated with seizure disorders ( 16 , 18 ). grin2b is a high-risk gene for autism spectrum disorder (asd) ( 17 , 38 , 39 ), as are grin1 and grin2a ( 40 – 42 ). still, patients with such neurodevelopmental disorders generally experience a variety of both central and peripheral nervous system symptoms ( 43 – 46 ). nmda receptors are highly conserved across species ( 47 ). due to an ancient genome duplication event, zebrafish retain two paralogues for most nmdar genes (e.g., grin1a &amp; grin1b ) ( 24 , 48 ). patch clamp electrophysiology experiments show that zebrafish paralogues display similar functional properties to each other and to their mammalian orthologues ( 27 , 28 ). nevertheless, it remains unclear how well the human nmdar genes, including positions with identified missense variants, are conserved in zebrafish. to address this, we investigated nmdar conservation in zebrafish by taking advantage of amino acid sequence alignments ( 28 ) and assessed the conservation of all human nmdar subunits and zebrafish paralogues. we also compared the conservation of positions with missense variants identified in human patients for nmdar subunits expressed during early neurodevelopment. finally, we investigated the conservation of critical functional regions within each structural domain. through this analysis, we define the extent to which critical functional components and regions, with relevance to disease, are conserved across zebrafish nmdars, providing a template to use zebrafish as a model to study nmdars in neurodevelopment and disease pathology. methods nmdar sequences and sequence alignments amino acid sequences for human genes are (uniprot with protein ids): glun1 (q05586-5; ntd splice isoform, nr1-3), glun2a (q12879-1), glun2b (q13224), glun2c (q14957), glun2d (o15399), glun3a (q8tcu5), and glun3b (o60391). for mouse subunits: mglun1 (p35438), mglun2a (p35436), mglun2b (q01097), mglun2c (q01098), mglun2d (q03391), mglun2a (a2air5), and mglun3b (q91zu9). for zebrafish paralogs: zglun1a (f1r366), zglun1b (q6zm67), zglun2aa (a0a8m9pxd6), zglun2ab (f1qde5), zglun2ba (a0a8m9q0i1), zglun2bb (a0a8m1nh89), zglun2ca (e7fh62), zglun2cb (a0a8m3b7k5), zglun2da (i3ni77), zglun2db (e7f3z4), zglun3a (a0a8m3awh5), zglun3ba (a0a8m9p9f6), and zglun3bb (a0a8m6z151). breakpoints for nmdar subunit domains and subdomains are based on structures 6whr (rat glun1-glun2b) and 7eu7 (human glun1-glun2a) ( 49 , 50 ). human and zebrafish protein sequences were aligned and analyzed for amino acid identity and similarity using align sequences protein blast (blastp), which is publicly available through the national center for biotechnology information (ncbi) or emboss needle pairwise sequence alignment (psa). for each nmdar subunit, the zebrafish paralogs were individually aligned to the human protein. amino acid similarity is defined based on the blosum 62 substitution matrix. for the ntd, lbd, and tmd, these alignments were generally straightforward as the sequences do not diverge greatly in length. in contrast, the ctd is quite large, with the zebrafish sequences generally larger than human. for these alignments, we entered the entire sequence into the alignment tool and used the given result, though this led to a fragmentation of the ctd such that certain portions of the sequence were aligned while others were not. we used the resulting domain-specific conservation values to calculate the conservation of the total protein. variants analysis human nmdar variants were identified using clinvar and gnomad (table 2 ). clinvar . results for each gene were downloaded on september 28th, 2023, using clinvar&#x27;s bulk download option ( https://www.ncbi.nlm.nih.gov/clinvar/ ). genomic variants that did not result in protein coding changes were removed from the analysis. in the case of grin3a , protein coding changes in the ppp3r2 gene, a single exon gene whose coding sequence resides within the genomic region of grin3a and is downloaded alongside grin3a due to this, was also removed. in cases where multiple protein coding annotations exist for the same variant, only the annotation fitting the canonical transcript was retained. gnomad : genes were searched for by name on gnomad v4.0.0 ( https://gnomad.broadinstitute.org/ ) and missense, insertion, deletion, and loss-of-function variant information was exported on september 28th, 2023 as comma separated value (csv) files for each gene. hgvs annotation (protein change) was converted to letter-number-letter annotation. gene variants from clinvar and gnomad were merged using r studio 2023.06.1 build 524 based on the letter-number-letter annotation. the resulting merged file was checked for redundant information or variant duplicates, which were removed. missense variants were extracted and grouped by pathogenicity as pathogenic, putatively pathogenic, complex, putatively benign, and benign variants based on the criteria in table 2 . assessment of conservation of positions with identified human variants was performed using the generated sequence alignments. nmdar domain-specific analysis to further refine the relationship between human and zebrafish nmdar proteins, we analyzed the tmd, lbd-tmd linkers, lbd, ntd, and ctd of glun1, glun2a, glun2b, and glun2d for conservation of key structural and functional features. transmembrane domain (tmd) and ligand-binding domain-transmembrane domain (lbd-tmd) linkers. key regions involved in channel gating, mg 2+ block, and ca 2+ flux were identified as described in the literature and analyzed for conservation ( 51 – 57 ). ligand-binding domain (lbd) . key regions and specific amino acids involved in van-der-waals interactions with the α-carbon of the ligand, interactions with the α-amino group, electrostatic interactions with the α-carboxyl group, and interactions with the amino acid side chain were identified as described in ramos-vicente et. al. 2021 and analyzed for conservation ( 47 ). n-terminal domain (ntd) . key amino acid positions required for the binding of zinc and polyamines were identified via a literature search and their conservation assessed ( 2 , 58 , 59 ). we also examined the conservation of putative n-linked glycosylation sites ( 60 ). to generate homology models of the zebrafish ntds, we used the template function in swiss-model ( 61 , 62 ). as reference structures, we used rodent ntd structures 5tq0 (glun1-glun2a in the presence of edta, no bound zn 2+ for glun1 and glun2a), 5tpz (glun1-glun2b apo state for glun1 and glun2b), and 5tpw (glun1-glun2a with zn 2+ in complex with glun2a) ( 58 ). sequences for human (hglun1, hglun2a, and hglun2b) or zebrafish (zglun1a, zglun1b, glun2aa, glun2ab, glun2ba, glun2bb) ntds were entered individually into swiss-model to generate homology models. the subsequent models were then loaded into pymole (human and zebrafish paralog) and aligned. the resultant root mean square deviation (rmsd) is reported. carboxy-terminal domain (ctd) . ctd-specific conservation in zebrafish was assessed for the glun1, glun2a, and glun2b subunits. glun2d was excluded from this analysis due to the poor conservation of the ctd in these paralogs relative to human. to assess functional relevance, we focused on post-translational modifications (ptms), specifically phosphorylation sites (serine, threonine, tyrosine), ubiquitylation sites (lysine), and palmitoylation sites (cysteine), using a combination of experimentally validated data from phosphositeplus (psp) and published literature ( 2 , 11 ). phosphorylation sites with identified kinases or well-characterized functional roles were classified as ‘kinase-regulated phosphorylation sites.’ the remaining phosphorylation sites identified in psp were defined as ‘phosphoproteomic-identified sites.’ note, sites in psp (based on rodents) that did not have a phosphorylation site in humans were excluded from our analysis. in addition, known linear protein-binding motifs were annotated and analyzed for conservation ( 2 , 11 ). to assess conservation of protein-binding motifs in the ctds, we categorized sites as either short linear motifs (slims) or broader interaction ‘regions’. slims were defined as experimentally validated, compact sequence elements (typically &lt; 15 residues) that mediate specific and direct interactions with known protein partners. broader regions were defined as extended sequences (&gt; 15 residues) shown to directly bind proteins in biochemical or structural studies but lacking a discrete consensus sequence. all motifs were curated from published literature where discrete sequences were experimentally validated per motif ( 11 , 63 – 80 ), and sequence conservation was examined. results our goal is to compare the structural similarity of zebrafish nmdar (zglun) paralogues to human nmdars (glun), especially regarding human variants. since mice are a common animal model ( 81 ), as a reference-point for nmdar gene conservation, we initially made global comparisons of both mouse and zebrafish nmdar subunits to human. global comparison of mouse and zebrafish orthologues to human nmdars nmdars are heterotetramers composed of two obligate glycine-binding glun1 subunits and two glutamate-binding glun2 subunits or a combination of glun2 and glycine-binding glun3 subunits ( 1 , 2 ) (fig. 1 a,b). glycine and glutamate act as co-agonists for glun2-containing receptors, with binding of both required for receptor activation, while glun3-containing receptors are activated by glycine binding. as an initial assessment of the structural similarity between nmdar orthologues, we made global sequence alignments using the ncbi blastp or emboss psa tools. not surprisingly, given evolutionary proximity, the mouse subunits are highly conserved relative to human: glun1 (97% identity &amp; similarity), glun2a (95% identity; 98% similarity), glun2b (98% identity; 99% similarity), glun2c (88% identity; 90% similarity), glun2d (97% identity &amp; similarity), glun3a (92% identity; 95% similarity), and glun3b (77% identity; 83% similarity) (table 1 ). on the other hand, the zebrafish paralogues show reduced conservation. the obligatory glun1 subunits are the most highly conserved, both in terms of identity (≥ 84%) and similarity (≥ 89%) (table 1 ). the other subunits show various degrees of identity and similarity: glun2a (≥ 58% identity; ≥79% similarity), glun2b (≥ 65% identity; ≥73% similarity), glun2c (≥ 50% identity; ≥64% similarity), glun2d (≥ 43% identity; ≥54% similarity), glun3a (63% identity; 78% similarity), and glun3b (≥ 54% identity, ≥ 62% similarity) (table 1 ). these subunits, thus, retain ≥ 50% identity with respect to the human protein, except for zglun2da and zglun2db, which only share 43% and 46% sequence identity, respectively (table 1 ). with respect to the zebrafish paralog pairs, there are instances in which one paralog is better conserved than the other. the glun1 paralogs are comparably conserved (table 1 ). however, the zglun2aa paralog is more highly conserved than zglun2ab while the zglun2bb shows increased conservation relative to zglun2ba (table 1 ). the glun2c, glun2d, and glun3b paralogs are conserved to similar extents (table 1 ). nevertheless, despite lower conservation relative to mouse, zebrafish offer many technical advantages (see background). in addition, what is critical is the conservation of specific structural and functional domains and subdomains. table 1 overall protein conservation in mouse and zebrafish relative to human human mouse zebrafish paralogue a zebrafish paralogue b protein length id. sim. protein length id. sim. protein length id. sim. protein length n1 938 97% 97% 938 84% 89% 966 (-28) 86% 90% 937 (-1) n2a 1464 95% 98% 1464 68% 89% 1460 (+ 6) 58% 79% 1445 (-9) n2b 1484 98% 99% 1482 (-2) 65% 73% 1417 (-31) 67% 81% 1770 (+ 322) n2c 1233 88% 90% 1239 (+ 6) 50% 64% 1400 (-176) 59% 79% 1328 (+ 95) n2d 1336 97% 97% 1323 (-13) 43% 54% 1916 (+ 580) 46% 57% 1676 (+ 340) n3a 1115 92% 95% 1135 (+ 20) 63% 78% 1111 (-4) n3b 1043 77% 83% 1003 (-40) 54% 62% 1118 (+ 75) 62% 68% 1114 (+ 71) numbers indicate the percent of mouse or zebrafish paralog conservation with respect to the human protein – ‘id.’ is identity and ‘sim.’ is similarity. the zebrafish a and b paralogs are indicated for each protein. glun3a has only a single paralog in zebrafish. for the mouse and zebrafish protein length, numbers in parentheses indicate the difference in amino acid number relative to human. conservation of specific structural elements in zebrafish paralogues for more rigorous comparisons, we examined the conservation within individual subunit domains and subdomains (fig. 1 ). the structure of nmdars, like all iglurs, consists of 4 domains that are intrinsic to each individual subunit (fig. 1 a,b): the extracellular ntd and lbd; the tmd forming the ion channel as well as the linkers that connect the lbd to the tmd, the lbd-tmd linkers; and the intracellular ctd ( 2 , 53 ). within individual subunits, the lbd is formed by two disparate sequences, s1 and s2, that fold up into the d1 and d2 lobes in the 3-dimensional structure (83, 84). in general, d1 is made up primarily of s1 and d2 of s2, but there is considerable swapping between lobes. the tmd is formed by three transmembrane segments, m1, m3, and m4, and an m2-pore loop. the lbd-tmd linkers are referred to as either s1-m1, m3-s2, and s2-m4 (linear sequence) or d2-m1, m3-d2, and d1-m4 (3-dimensional structure). this distinction is important functionally since d1 and d2 undergo different movements that are translated differently to the tmd ( 85 , 86 ). nevertheless, we will use the s1-m1, m3-s2, and s2-m4 designation for ease of comparison to structures. we began by dividing the zebrafish paralogues into their respective domains and subdomains based on our breakpoints (see methods). interestingly, the amino acid sequences surrounding these breakpoints were well conserved in zebrafish – the majority were identical (not shown). this allowed for ease of division of the zebrafish proteins into their respective domains and subdomains. as an initial assessment of the structural similarity between human and zebrafish nmdars, we made detailed sequence alignments using the ncbi blastp or emboss psa tools (table 1 , fig. 1 ). in terms of length of domains, the zebrafish protein sequence for glun1, glun2, and glun3 subunits was identical to the human sequence for the tmd (m1, m1-m2, m2, m2-m3, m3, &amp; m4) and lbd-tmd linkers (s1-m1, m3-s2, &amp; s2-m4) (fig. 1 c). the lbd (s1 &amp; s2) was also quite similar in length, with no changes in the length of s2 across the paralogs and only slight increases or decreases in amino acid number in the s1 of zglun1a, zglun1b, zglun2ab, zglun2ca, zglun2ca, zglun2da, and zglun2db (fig. 1 c). the ntd and ctd were less conserved in terms of size, wherein almost every zebrafish paralog has a different number of amino acids compared to the human. this is especially notable for the glun2b ntd which either lacks 130 amino acids (zglun2ba) or has an additional 74 amino acids (zglun2bb) as well as for the ctd where, for example, zglun2da and zglun2db have a ctd that is more than twice as long as that in the human protein (fig. 1 c). table 2 variant designation glun designation clinvar gnomad 1 2a 2b 2c 2d 3a 3b pathogenic “pathogenic” absent 49 (5%) 58 (4%) 81 (5%) 0 12 (&lt; 1%) 0 0 putatively pathogenic “uncertain” absent 123 (13%) 214 (15%) 182 (12%) 4 (&lt; 1%) 111 (8%) 5 (&lt; 1%) 8 (&lt; 1%) complex “pathogenic” present 3 (&lt; 1%) 5 (&lt; 1%) 6 (&lt; 1%) 0 1 (&lt; 1%) 0 0 putatively benign “uncertain” present 80 (9%) 320 (22%) 199 (13%) 31 (3%) 223 (17%) 30 (3%) 68 (7%) benign “benign” or absent absent/ present or present 401 (43%) 813 (55%) 672 (45%) 945 (77%) 863 (27%) 59 (5%) 945 (91%) number of positions with missense variants that have been identified in each subunit per designation. percentages report the proportion of the total protein amino acids at which such variants have been identified. we used these sequence alignments to assess the degree of amino acid identity and similarity between domains and subdomains (fig. 1 d,e). the obligatory glun1 subunit showed the highest degree of conservation. the tmd and lbd-tmd linkers are 100% conserved. the lbd is also well conserved with s1 and s2 showing ≥ 92% identity and ≥ 93% similarity. the ntd is somewhat less conserved (≥ 80% identity, ≥ 88% similarity), as is the ctd (≥ 53% identity; ≥58% similarity). this trend of conservation across nmdar subunits – the lbd and tmd are most highly conserved and the ntd and ctd less so – is maintained in the glun2 and glun3 subunits (fig. 1 d,e). generally, the glun2 subunits maintain a higher sequence identity than the glun3 subunits. overall, this sequence analysis suggests that the lbd and tmd are highly conserved whereas the ntd and ctd show less conservation. categorizing the pathogenicity of missense variants nmdars with missense, frameshift, and nonsense variants have been identified in pediatric patients with neurodevelopmental disorders ( 21 , 22 , 87 ). we therefore wanted to assess the extent to which variants are conserved in zebrafish to aid in guiding future studies. for this analysis, we focused solely on missense variants. while nonsense and frameshift mutation can be modelled in various ways, such as by mutating a portion of the protein to induce a loss-of-function, the conservation of the specific protein sequence is somewhat less relevant. in contrast, the single amino acid change associated with missense variants is critical to dissecting how such variants alter receptor function and lead to disease phenotypes. using publicly available datasets of nmdar variants from clinvar and gnomad, we identified missense variants in all nmdar subunits (see methods). we first categorized these variants with regards to their pathogenicity (table 2 ). clinvar reports variants that have been identified in patients with symptomatic disease ( 88 – 90 ), while gnomad reports variants identified in the healthy population ( 91 ). thus, in general, we define variants present in clinvar but absent in gnomad as pathogenic while those with the opposite reporting as benign. with this approach, pathogenic and putatively pathogenic variants were identified in glun1, glun2a, glun2b, and glun2d while minimal to no such variants were identified in glun2c, glun3a, and glun3b (table 2 ). notably, these genes with known pathogenic variants are expressed early in development, contributing to developmental processes while their dysfunction often leads to disease phenotypes ( 21 , 87 ). on the other hand, glun2c, glun3a, and glun3b have no pathogenic and a small number of putatively pathogenic variants (table 2 ). we therefore focus solely on glun1, glun2a, glun2b, and glun2d for further analysis. pathogenic and putatively pathogenic variant positions are well conserved in zebrafish using the criteria in table 2 , amino acid positions with pathogenic missense variants in glun1, glun2a, glun2b, and glun2d are completely conserved in zebrafish paralogues (fig. 2 a-d). putatively pathogenic variants are also conserved in these nmdar subunits, though less so than pathogenic variants. the highest degree of conservation of putatively pathogenic variants is in the glun1 paralogues (≥ 88% identity; 93% similarity). putatively pathogenic variants in the glun2 subunits are also reasonably conserved: zglun2aa (70% identity; 92% similarity), zglun2ab (62% identity; 84% similarity), zglun2ba (65% identity; 71% similarity), zglun2bb (72% identity; 80% similarity), and zglun2da and zglundb (≥ 46% identity; ≥59% similarity) (fig. 2 e,f). in general, the highest level of conservation is in the tmd and tmd-lbd linkers, followed by the lbd, and less so in the ntd and ctd (fig. 2 g,h). benign and putatively benign variant positions show reduced conservation we carried out a comparable analysis as above for benign and putatively benign variants (table 2 ). we hypothesized that positions with more benign variants would be less well conserved than pathogenic ones. indeed, the general trend is consistent with this idea (fig. 3 ). putatively benign variants showed a reduced conservation compared to putatively pathogenic variants (putatively pathogenic vs putatively benign): zglun1a (88% vs. 78% identity; 93% vs. 88% similarity), zglun1b (89% vs. 79% identity; 93% vs. 84% similarity), zglun2aa (70% vs. 60% identity; 92% vs. 86% similarity), zglun2ab (62% vs. 48% identity; 84% vs. 68% similarity), zglun2ba (65% vs. 45% identity; 71% vs. 55% similarity), zglun2bb (72% vs. 55% identity; 80% vs. 70% similarity), zglun2da (46% vs. 47% identity; 59% vs. 61% similarity), and zglun2db (47% vs. 44% identity; 61% vs. 56% similarity) (fig. 2 e,f vs fig. 3 a,b). for these putatively benign variants, the greatest extent of conservation tends to occur in the tmd and lbd-tmd linkers, followed by the lbd, and then the ntd and ctd (fig. 3 c,d). for benign variants, however, the decrease in conservation plateaus, with identity and similarity values comparable to the putatively benign variants, if not slightly higher (fig. 3 e-f). the subdomain-specific conservation trend for these variants follows that seen on the level of the whole receptor (fig. 1 ), wherein the tmd, lbd-tmd linkers, and lbd are generally more conserved and the ntd and ctd less so (fig. 3 g,h). although diverging from our prediction as we do not note a substantial decrease in the conservation of benign variants with respect to the putatively benign variants, this result reflects the overall conservation of the nmdar subunits themselves (benign vs total protein): zglun1 (78% vs. ≥84% identity; 84% vs. ≥89% similarity), zglun2aa (62% vs. 68% identity; 86% vs. 89% similarity), zglun2ab (49% vs. 58% identity; 74% vs. 79% similarity), zglun2ba (54% vs. 65% identity; 64% vs. 73% similarity), and zglun2bb (62% vs. 67% identity; 74% vs. 81% similarity), zglun2da (50% vs. 43% identity; 64% vs. 54% similarity), and zglun2db (49% vs. 46% identity; 62% vs. 57% similarity) (fig. 3 e-f, table 1 ). furthermore, this result reflects the relatively large number of benign variants in comparison to other variant groups (table 2 ). critical tmd and lbd-tmd linker components are completely conserved elements forming the ion channel – the tmd (m1, m1-m2, m2, m2-m3, m3, &amp; m4) – and the lbd-tmd linkers (s1-m1, m3-s2, &amp; s2-m4) were consistently highly conserved (fig. 1 ). nevertheless, we assayed conservation of critical functional motifs. the s1-m1 linker is a critical element for channel gating and is completely conserved (fig. s1 ) ( 56 ). the m2 subdomain contains what is known as the ‘n’ and ‘n + 1’ sites, which participate in mg 2+ block and ca 2+ permeation through the channel ( 2 , 55 ) and are conserved in zebrafish (fig. s1 , dark orange). the tmd m3 segment contains the sytanlaaf motif, which plays a key role in ion channel gating and is known to be highly conserved across species. this motif is conserved in all zebrafish glun1 and glun2 paralogs (fig. s1 , orange). lastly, the ‘drpeer’ motif in m3-s2 of glun1, an important component in the high ca 2+ permeability of nmdars ( 52 ) is conserved (fig. s1 , light orange). hence, core gating and permeation elements of the tmd and associated regions are highly conserved suggesting that basic mechanisms of ion channel gating and block/permeation are preserved. lbd elements required for agonist binding are completely conserved the glun1 lbd binds glycine or d-serine while the glun2 lbd binds glutamate or aspartate (fig. s2a,b). overall, the lbds are highly conserved in zebrafish relative to humans (fig. 1 ), as are specific amino acid side chains in s1 and s2 that are involved in binding endogenous agonists ( 47 ) (fig. s2c). these include sites that participate in van-der waals interactions with the α-carbon of the ligand (darkest red), interactions with the α-amino group (dark red), electrostatic interactions with the α-carboxyl group (red), and interactions with the amino acid side chain (light red) (fig. s2c). conservation of structure-function elements in the ntd in terms of overall sequence comparisons, the ntd shows variable degrees of conservation across nmdar subunits. the glun1 zebrafish paralogs maintain the highest degree of conservation relative to human (80% identity; ≥88% similarity) (fig. 1 ). the glun2 subunits show decreased ntd conservation (49–75% identity; 59–88% similarity) (fig. 1 ). given this reduced conservation, we assayed the conservation of key structure-function motifs in zebrafish paralogues. among its many roles, the ntd participates in interactions with ions and molecules that modulate receptor activity ( 58 , 59 ) (fig. 4 a,b). this is most notable for glun2a and to a lesser extent glun2b as they bind zinc, which inhibits receptor function by reducing channel open probability ( 58 ). in glun2a, four amino acids coordinate zinc binding: glun2a-his44, glun2a-his128, glun2a-glu266, and glun2a-asp282 ( 58 ). of these key sites, three are conserved (fig. 4 c) but in zglun2aa and zglun2ab, a key residue involved in zn 2+ binding, glun2a-his44, is not conserved (fig. 4 c). for glun2b, only two amino acid residues participate in zinc binding: glun2b-his127 and glun2b-glu284 ( 58 ). while glun2b-glu284 is conserved, the glun2b-his127 position in zglun2ba is missing due to alterations in ntd sequence length leading to gaps in the alignments, though it is similar now as a positively charged arginine instead of a histidine in zglun2bb (fig. 4 c). the ntd of glun2b also binds polyamines, which enhance receptor activity ( 59 ). nine amino acids are involved in the binding of polyamines. these sites in zglun2ba and zglun2bb are completely conserved except for one amino acid in zglun2bb that is similar but not identical, now a negatively charged aspartate instead of glutamate (fig. 4 d). most of this functional component of glun2b is, thus, conserved in zebrafish. additionally, the nmdar ntd has glycosylation sites, and their glycosylation regulates the trafficking of nmdars to the cell membrane ( 60 ). given the consensus sequence for arginine (n-) glycosylation (n- x -s/t), glun1 has seven, glun2a has three, and glun2b has three putative glycosylation sites in the ntd ( 60 ). their conservation in the zebrafish paralogues is somewhat variable: zglun1a &amp; zglun1b (5 of 7 sites), zglun2aa (3 of 3 sites), zglun2ab (2 of 3 sites), zglun2ba (2 of 3 sites), and zglun2bb (3 of 3 sites) (fig. 4 e). to the best of our knowledge, specific glycosylation sites in glun2d have yet to be identified. of note, the glun1 glycosylation sites glun1-n203 and glun1-n368 are necessary for receptor surface expression ( 60 ), and these sites are conserved in the zebrafish paralogues. while the critical functional features of the lbd were completely conserved in zebrafish relative to human nmdars, the conservation of key regions in the ntd occasionally diverged, most notably that for zn 2+ binding. to further explore the structural composition of the glun1, glun2a, and glun2b ntds, we used swiss-model to generate homology models for the human and zebrafish nmdar subunits, which we subsequently aligned in pymole to assess conservation at the level of the three-dimensional structure (see methods) (fig. 4 f). in general, this analysis revealed a high degree of similarity between the human ntd and zebrafish paralogs. the zebrafish glun1 ntd models display similar overall topology to human: glun1 vs zglun1a, reference structure 5tp0 (rmsd = 0.071); glun1 vs zglun1a, reference structure 5tpz (rmsd = 0.085); glun1 vs zglun1b, reference structure 5tp0 (rmsd = 0.094); and glun1 vs zglun1b, reference structure 5tpz (rmsd = 0.097) (table s3). the glun2a and glun2b subunits also reflect a high degree of structural similarity: glun2a vs zglun2aa, reference structure 5tp0 (rmsd = 0.085); glun2a vs zglun2ab, reference structure 5tp0 (rmsd = 0.095); glun2a vs zglun2ab, reference structure 5tpw (rmsd = 0.111); glun2b vs zglun2ba, reference structure 5tpz (rmsd = 0.028); and glun2b vs glun2bb, reference structure 5tpz (rmsd = 0.044) (table s3). the only exception here is the structural comparison of glun2a and zglun2aa using reference structure 5tpw, in which the ntd is interacting with zinc (rmsd = 2.149), suggesting that the structural components of zglun2aa may not effectively complex with zinc. interestingly, zglun2ba is &gt; 100 amino acids shorter and zglun2bb is &gt; 70 amino acids longer than human glun2b. though the zglun2ba model is missing an α-helix relative to the human protein, the zglun2bb modelled structure aligns well with its human counterpart, suggesting that the zebrafish sequences maintain a large degree of structural integrity. conservation of post-translational modification (ptm) sites in the ctd the intracellular ctd is a key regulatory domain of nmdar cell biology and function and shows distinct subunit-specific elements ( 11 ). for our analyses, we focused on the glun1-1 ctd splice variant, which is the longest variant containing the c0-c1-c2 cassettes ( 11 ). across the nmdar subunits, the zebrafish ctds are the most poorly conserved (fig. 1 ). with a sequence identity generally around 50–60% for the glun1, glun2a, and glun2b zebrafish paralogs, with some exceptions: zglun1a (53% identity; 58% similarity), zglun1b (54% identity; 59% similarity), zglun2aa (57% identity; 86% identity), zglun2ab (38% identity; 65% similarity), zglun2ba (57% identity, 66% similarity), and zglun2bb (60% identity, 71% similarity) while that of glun2d is much lower (≥ 18% identity, ≥ 28% similarity) (fig. 1 ). given these deviations in protein sequences, we first examined the conservation of ctd regions critical for post-translational modifications (ptms), excluding glun2d due its poor conservation in zebrafish. phosphorylation of nmdar subunit ctds confers key functional roles to the receptor ( 2 , 11 ). initially, we examined experimentally validated sites at which phosphorylation occurs by known kinases and calculated the percent conservation, reported in identity and similarity. similarity, here, is defined as a site at the homologous position that can be phosphorylated (e.g., s, t, or y), if not identical. the glun1 ctd contains phosphorylation sites for src kinase (src), protein kinase c (pkc), and protein kinase a (pka), (fig. 5 a, top ) ( 11 ). note specific sites have also been identified that are also targeted by the serine/threonine protein phosphatase 2b (pp2b). these ptm sites are perfectly conserved in zglun1a, while three of five are conserved in zglun1b (60% identity &amp; similarity) (fig. 5 b). the glun2a ctd is phosphorylated by src, pka, dual specificity tyrosine-phosphorylation-regulated kinase 1a (dyrk1), cyclin-dependent kinase 5 (cdk5), pkc, and calcium/calmodulin-dependent protein kinase iiα (camkiiα) (fig. 5 a, middle ) ( 11 ): zglun2aa (79% identity &amp; similarity) and zglun2ab (64% identity, 71% similarity) (fig. 5 b). the ctd of glun2b is phosphorylated by src, camkiiα, cdk5, pka, pkc, death-associated protein kinase 1 (dapk1), proto-oncogene tyrosine-protein kinase fyn (fyn), and casein kinase ii (ck2) (fig. 5 a, bottom ) ( 11 ). glun2b also contains sites for the protein tyrosine phosphatase non-receptor 11 (ptpn11, or shp2) and protein phosphatase 1 (pp1). the conservation of these ptms is slightly higher than that in glun2a: zglun2ba (73% identity &amp; similarity) and zglun2bb (82% identity, 100% similarity) (fig. 5 b). in addition to these kinase-regulated sites, we also examined the conservation of other known phosphorylation sites in these subunits identified by phosphoproteomics (psp) (see methods). the conservation of these sites in the zebrafish paralogs was generally lower than that of the kinase-regulated sites: glun1 (67% identity, 100% similarity), zglun2aa (51% identity, 66% similarity), zglun2ab (34% identity, 57% similarity), zglun2ba (60% identity &amp; similarity), and zglun2bb (67% identity, 70% similarity) (fig. 5 c). we next examined the conservation of ctd palmitoylation and ubiquitylation sites. glun1 has a single ubiquitylation site (lys860) (fig. 6 a, top ) ( 11 ), and it is conserved in the zebrafish paralogs (fig. 6 c). glun2a contains seven palmitoylation sites (fig. 6 a, middle ) ( 11 ), which are likewise completely conserved in zebrafish (fig. 6 b). the glun2a ctd also has four ubiquitylation sites (fig. 6 a, middle) ( 11 ), of which three are conserved in zglun2aa (75%) and one in zglun2ab (25%) (fig. 6 c). with the same approach for glun2b, we identified eight palmitoylation and six ubiquitylation sites (fig. 6 a, bottom ) ( 11 ). here, all palmitoylation sites are conserved in zebrafish while five of six ubiquitylation sites (83%) are conserved in zglun2ba and all six ubiquitylation sites (100%) are conserved in zglun2bb (fig. 5 b,c). in summary, the global conservation of ptm sites is as follows: zglun1a and zglun1b (89% identity, 100% similarity), zglun2aa (62% identity, 72% similarity), zglun2ab (45% identity, 62% similarity), zglun2ba (66% identity, 67% similarity), and zglun2bb (75% identity, 78% similarity). conservation of protein-binding motifs (pbm) in the ctd in addition to numerous ptm sites, a variety of protein-protein interactions are critical to the role of the ctd in receptor cell biology (surface expression, distribution) and function ( 11 , 63 ). these protein-binding elements include either defined short linear motifs (slims) or broader regions known to serve as protein docking sites. the ctd of glun1 contains a slim known as the kkk/rrr er retention motif, which is conserved in zebrafish ( 66 ) (fig. 7 a,b). it, likewise, has binding regions for calmodulin (cam)/calcium/calmodulin-dependent protein kinase ii (camkii) and α-actinin 2 ( 64 , 65 , 67 , 78 ), which is conserved and yotiao, also known as a-kinase anchoring protein 9 (akap9) ( 67 ), which is generally well conserved: zglun1a (90% identity, 98% similarity) and zglun1b (90% identity, 98% similarity) (fig. 8 a,b). the glun2 subunit ctds contain a variety of pbms. for glun2a, the conservation of slims is: the yxxø motif, where y is tyrosine, x is any amino acid, and ø is a bulky hydrophobic residue like leucine, isoleucine, phenylalanine, methionine, or valine ( 73 ) – both zglun2aa and zglun2ab show 100% identity &amp; similarity; the guanine-nucleotide exchange factor brag2 ( 70 ) – zglun2aa (80% identity &amp; similarity) and zglun2ab (40% identity, 60% similarity); postsynaptic density protein 95 (psd-95) ( 72 ) – zglun2aa (50% identity; 63% similarity) and zglun2ab (38% identity, 63% similarity); and the pdz domain that interacts with the proteins psd-95, discs large (dlg), and zonula occludens-1 (zo-1) ( 11 ) – zglun2aa and zglun2ab (100% identity &amp; similarity) (fig. 7 a,c). interacting regions in glun2a include: ring finger protein 10 (rnf10) ( 70 ) – zglun2aa (51% identity, 72% similarity) and zglun2ab (32% identity, 66% similarity); rabphilin 3a (rph3a) ( 68 ) – zglun2aa (71% identity, 80% similarity) and zglun2ab (49% identity, 71% similarity); flotillin-1 (flot-1) ( 71 ) – zglun2aa (61% identity, 81% similarity) and zglun2ab (31% identity, 52% similarity); and c-terminal binding protein 1 (ctbp1) ( 69 ) – zglun2aa (51% identity, 85% similarity) and zglun2ab (36% identity, 63% similarity) (fig. 8 a,c, s3a,b). for glun2b, the conservation of slims is: the guanine-nucleotide exchange factor brag1 ( 76 ) – zglun2ba (60% identity; 80% similarity) and zglun2bb (60% identity, 100% similarity); psd-95 ( 72 ) – zglun2ba (56% identity &amp; similarity) and zglun2bb (33% identity, 56% similarity); camkii and death-associated protein kinase 1 (dapk1) ( 11 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (92% identity, 100% similarity); synapse-associated protein 102 (sap102) ( 75 ) – zglun2ba (0% identity &amp; similarity) and zglun2bb (50% identity &amp; similarity); the adaptor complex ap2 ( 74 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (100% identity &amp; similarity); and pdz ( 11 ) – zglun2ba (100% identity &amp; similarity) and zglun2bb (100% identity &amp; similarity) (fig. 7 a,d). the ctd interaction regions for glun2b are conserved as follows: receptor for activated c kinase 1 (rack1) ( 77 ) – zglun2ba (0% identity, 7% similarity) and zglun2bb (13% identity, 20% similarity); flot-1 ( 71 ) – zglun2ba (23% identity; 47% similarity) and zglun2bb (35% identity, 68% similarity); spectrin ( 79 ) – zglun2ba (40% identity; 59% similarity) and zglun2bb (35% identity, 62% similarity); α-actinin 2 ( 78 ) – zglun2ba (47% identity; 67% similarity) and zglun2bb (51% identity, 76% similarity); and ras protein-specific guanine nucleotide-releasing factor 1 (rasgrf1) ( 80 ) – zglun2ba (59% identity; 74% similarity) and zglun2bb (60% identity, 78% similarity) (fig. 8 a,d, s3a,c). when compared globally across subunits, slims always show greater conservation than regions. this conservation is always greatest in the zebrafish glun1 paralogs, followed by the glun2b and then glun2a paralogs. for the analyzed slims, conservation grouped across each subunit is as follows: zglun1a and zglun1b (100% identity &amp; similarity), zglun2aa (76% identity, 81% similarity), zglun2ab (62% identity, 76% similarity), zglun2ba (78% identity, 81% similarity), and zglun2bb (72% identity, 86% similarity). the regions show the following degrees of conservation: zglun1a and zglun1b (94% identity, 98% similarity), zglun2aa (55% identity, 81% similarity), zglun2ab (36% identity, 62% similarity), zglun2ba (59% identity, 74% similarity), and zglun2bb (60% identity, 78% similarity). discussion here, we compared the conservation of the zebrafish nmdar subunit paralogues, including key structure-function motifs, to their human counterparts. we find that the core gating machinery – the lbd, the lbd-tmd linkers and the tmd – are highly conserved both in terms of overall identity and specific functional motifs. the more peripheral domains – the ntd and ctd – show less conservation overall, though functional motifs are reasonably conserved with the notable exception of zn 2+ binding in the glun2a ntd. notably, disease-associated variants are generally conserved, highlighting that zebrafish represent a useful model to study nmdars. nmdar conservation in zebrafish across most of our examined parameters, the obligatory glun1 subunit shows the highest degree of conservation in zebrafish (fig. 1 ). generally, conservation is then greatest in the glun2(a-d) followed by the glun3(a-b) subunits. within each subunit, sequence conservation is consistently highest in the tmd and lbd-tmd linkers, followed by the lbd. conservation is generally poorer in the ntd and even worse in the ctd (fig. 1 ). a key question with regards to this work is defining what constitutes as a ‘well conserved’ sequence. previous studies regarding the conservation of protein structure and function suggest that sequence identities between 40–70% confer functional conservation ( 92 – 94 ). they regard 50% sequence identity as the threshold below which function drastically diverges ( 93 , 94 ). our sequence conservation analyses almost always led to sequence identities &gt; 50%, except for zglun2da. nevertheless, we carried out more detailed examinations of more poorly conserved domains, e.g., the ntd and ctd, to assess the conservation of key structure-function motifs. nmdar missense variant conservation in zebrafish to date, a multitude of variants, including missense, frameshift, and nonsense mutations, have been identified in the genes encoding the nmdar subunits ( 21 , 22 , 87 ). we categorized identified missense variants by pathogenicity and analyzed the extent of conservation of the positions at which they occur. we focused only on the glun1, glun2a, glun2b, and glun2d subunits for this analysis, as this is where most disease-associated variants have been identified. generally, our results demonstrate a higher degree of variant position conservation with increasing pathogenicity (fig. 2 , 3 ). the sites of pathogenic missense variants are almost perfectly conserved (fig. 2 ). given the high conservation of the amino acid positions at which they occur, these protein regions are likely most essential for receptor function. across our categories of variants, with the exclusion of complex variants, the proportion of the total protein at which missense variants have been identified is lowest in the pathogenic group (table 2 ). it is possible that additional variants that might be categorized as pathogenic are so devastating to development as to be embryonic lethal. this reflects the notion that more integral regions of the receptor are less tolerant to variants ( 95 ). zebrafish, nevertheless, emerge as a useful model for the study of pathogenic nmdar variants. putatively pathogenic variants are conserved to a lesser extent than their pathogenic counterparts (fig. 2 ). according to our designations, these variants have the potential to cause disease, having been identified in patients, but are currently classified as having uncertain significance. thus, it aligns that these variants are comparatively less well conserved – they are likely occurring in less critical regions of the receptor and are likewise less likely to cause disease phenotypes. our subsequent investigation of putatively benign and benign variants revealed a further decrease in variant position conservation. this finding further supported our prediction that variants more highly associated with disease occur at positions with higher degrees of conservation and, thus, are more critical to protein structure and function. the conservation of positions with putatively benign and benign variants is quite comparable. for the benign variants, in particular, conservation percentages also closely reflect total protein conservation values for each paralog (fig. 3 , table 1 ). this follows from the finding that positions with identified benign variants, for most subunits, make up a substantial portion of the protein (table 2 ). nevertheless, this investigation both demonstrates the advantage of the zebrafish model in studying nmdar-associated disease and reflects the notion that the positions most essential for receptor function are those that are best conserved and more likely to induce disease when absent. overall, the trend for variant conservation is the same as that for the whole protein. conservation is always greatest in the tmd and lbd-tmd linkers, followed by the lbd, next the ntd, and last the ctd (fig. 2 , 3 ). these findings suggest that zebrafish serve as an effective model for variants, especially pathogenic and putatively pathogenic variants in both the tmd and lbd-tmd linkers. some caution, however, must be exercised when using this model system for variants in the lbd, ntd, and ctd. tmd, tmd-lbd linker, and lbd conservation our examination of key functional regions and motifs in the tmd, lbd-tmd linkers, and lbd consistently revealed complete conservation in their respective zebrafish paralogs (fig. s1 ,s2). this suggests that the basic mechanism of ion channel gating is conserved in zebrafish relative to human. as these domains and subdomains contribute to the formation of the core of the receptor – the ion channel pore allowing for ion flux – it is expected that they would be most highly conserved across species, since the major functional role of nmdars is current flux during synaptic transmission. ntd conservation the ntd is the first region in which nmdar sequences in zebrafish begin to diverge from those in human. modelling ntd variants in zebrafish must, thus, be approached with caution. for example, polyamine-binding sites in glun2b are almost completely conserved in zebrafish (fig. 4 d), so they have the potential to serve as an effective model for this functional role of nmdars. on the other hand, zinc-binding sites in glun2a and glun2b are more poorly conserved in zebrafish (fig. 4 c), so they would likely serve as a poor model for zinc-related studies in nmdars. this also aligns with our homology model finding that the zglun2aa model aligns well with glun2a when not bound to zinc, but the rmsd of the alignment is substantially poorer when glun2a interacting with zinc is used as the reference structure (table s3). ctd conservation the c-terminal domains (ctds) exhibit the lowest level of overall conservation across nmdar domains (fig. 1 ). this is consistent with the ctd’s classification as an intrinsically disordered region (idr), which lacks fixed secondary or tertiary structure and is more tolerant to evolutionary divergence ( 96 , 97 ). nonetheless, functional conservation within idrs is often maintained through the retention of post-translational modification (ptm) hotspots and short linear motifs (slims) that regulate protein interactions and intracellular signaling ( 98 ). we assessed ctd conservation in glun1, glun2a, and glun2b by analyzing experimentally validated phosphorylation, palmitoylation, and ubiquitylation sites, as well as defined protein-binding motifs. phosphorylation sites with known kinase interactions exhibited relatively high conservation, consistent with their well-characterized regulatory roles (figs. 5 a,b). in contrast, phosphorylation sites without identified kinases were less consistently retained (fig. 5 c). palmitoylation sites were fully conserved across all subunits, while ubiquitylation sites were generally retained, though conservation was notably lower in zglun2ab (fig. 5 f) and may reflect paralogue-specific divergence in degradation or trafficking pathways. protein-binding motif conservation varied by subunit. glun1 retained nearly all canonical interaction motifs, while glun2a and glun2b displayed region-specific conservation. notably well-conserved elements in the glun2 subunits included the pdz-binding motifs as well as docking regions for camkii, dapk1, and ap2 (fig. 6 ). slims overall showed higher conservation, likely due to their short, sequence-specific structure, which may place them under stronger evolutionary constraint than broader, more flexible interaction regions. despite broad sequence divergence, nmdar ctds retain a core set of conserved motifs involved in receptor trafficking, synaptic localization, and plasticity. kinase-regulated phosphorylation sites, palmitoylation sites, and pdz-binding motifs were consistently preserved. conservation of glun1 sites was always greatest in the zebrafish paralogs. among the glun2 subunits, glun2b displayed the highest overall conservation of functional elements, aligning with its essential role in neurodevelopment and synaptic signaling. these findings are especially relevant to the study of disease-associated truncating variants in the nmdar subunits. nonsense variants that introduce premature stop codons in the ctd eliminate key regulatory motifs critical for trafficking, anchoring, or signaling. when these affected regions correspond to conserved functional elements, zebrafish may serve as an appropriate in vitro and in vivo model for studying ctd-mediated disease mechanisms. conversely, for mutations in poorly conserved regions, the translational utility of this model may become limited. conclusions zebrafish represent a valuable system to study nmdar-associated neurodevelopmental disorders, but one must be judicious in choosing which variants to study in this model. our unique approach of sequence alignments per individual nmdar domains and subdomains highlights the high degree of conversation of some regions of the protein relative to others. additionally, our analysis of identified variants indicates that zebrafish are an effective model for pathogenic variants as these occur at conserved locations along the protein. generally, the utility of zebrafish is most apparent for the study of the tmd, lbd-tmd linkers, and lbd as these regions are most well conserved. on the other hand, studying the ntd and ctd in this species requires extra consideration stemming from deviations in sequence conservation. our work will enable future nmdar-related studies that can be effectively and efficiently conducted in zebrafish. abbreviations nmdar n -methyl-d-aspartate receptor ntd n-terminal domain lbd ligand-binding domain tmd transmembrane domain ctd c-terminal domain declarations ethics approval and consent to participate: not applicable. consent for publication: not applicable. availability of data and materials: the datasets used and/or analyzed during the current study are available either at osf (https://osf.io/h2zf9/) or from the corresponding author on reasonable request. competing interests: no competing interests declared. funding: this work was supported by nih grant r01ns088479 to lpw. authors’ contributions: e.r.n., c.a., j.d.z., h.i.s., and l.p.w. contributed to the conceptualization and methodology of this project. e.r.n., c.a., and j.d.z. conducted the investigation and analysis. w.r. assisted with data curation. e.r.n, c.a., and l.p.w. wrote the manuscript, which was revised with insights from j.d.z. and h.i.s. acknowledgements: the authors thank the clinvar and gnomad databases and the swiss-model server for their important contributions to this work. references paoletti p, bellone c, zhou q. nmda receptor subunit diversity: impact on receptor properties, synaptic plasticity and disease. nat rev neurosci. 2013;14(6):383-400. hansen kb, wollmuth lp, bowie d, furukawa h, menniti fs, sobolevsky ai, et al. structure, function, and pharmacology of glutamate receptor ion channels. pharmacol rev. 2021;73(4):298-487. chakraborty a, murphy s, coleman n. the role of nmda receptors in neural stem cell proliferation and differentiation. stem cells dev. 2017;26(11):798-807. nacher j, mcewen bs. the role of n-methyl-d-asparate receptors in neurogenesis. hippocampus. 2006;16(3):267-70. komuro h, rakic p. modulation of neuronal migration by nmda receptors. science. 1993;260(5104):95-7. pearce ia, cambray-deakin ma, burgoyne rd. glutamate acting on nmda receptors stimulates neurite outgrowth from cerebellar granule cells. febs lett. 1987;223(1):143-7. booker sa, wyllie dja. nmda receptor function in inhibitory neurons. neuropharmacology. 2021;196:108609. gambrill ac, barria a. nmda receptor subunit composition controls synaptogenesis and synapse stabilization. proc natl acad sci u s a. 2011;108(14):5855-60. napoli aj, laderwager s, zoodsma jd, biju b, mucollari o, schubel sk, et al. developmental loss of nmda receptors results in supernumerary forebrain neurons through delayed maturation of transit-amplifying neuroblasts. sci rep. 2024;14(1):3395. rajani v, sengar as, salter mw. tripartite signalling by nmda receptors. mol brain. 2020;13(1):23. ishchenko y, carrizales mg, koleske aj. regulation of the nmda receptor by its cytoplasmic domains: (how) is the tail wagging the dog? neuropharmacology. 2021;195:108634. mony l, paoletti p. mechanisms of nmda receptor regulation. curr opin neurobiol. 2023;83:102815. nielsen ac, anderson cl, ens c, boyce akj, thompson rj. non-ionotropic nmdar signalling activates panx1 to induce p2x4r-dependent long-term depression in the hippocampus. j physiol. 2024. barnes sa, thomazeau a, finnie psb, heinrich mj, heynen aj, komiyama nh, et al. non-ionotropic signaling through the nmda receptor glun2b carboxy-terminal domain drives dendritic spine plasticity and reverses fragile x phenotypes. cell rep. 2025;44(3):115311. platzer k, lemke jr. grin1-related neurodevelopmental disorder. genereviews((r)). 2021. strehlow v, myers ka, morgan at, scheffer ie, lemke jr. grin2a-related disorders. genereviews((r)). 2024. platzer k, lemke jr. grin2b-related neurodevelopmental disorder. genereviews((r)). 2021. platzer k, krey i, lemke jr. grin2d-related developmental and epileptic encephalopathy. genereviews((r)). 2022. zehavi y, mandel h, zehavi a, rashid ma, straussberg r, jabur b, et al. de novo grin1 mutations: an emerging cause of severe early infantile encephalopathy. eur j med genet. 2017;60(6):317-20. sabo sl, lahr jm, offer m, weekes a, sceniak mp. grin2b-related neurodevelopmental disorder: current understanding of pathophysiological mechanisms. front synaptic neurosci. 2022;14:1090865. gjerulfsen ce, krey i, klockner c, rubboli g, lemke jr, moller rs. spectrum of nmda receptor variants in neurodevelopmental disorders and epilepsy. methods mol biol. 2024;2799:1-11. amin jb, moody gr, wollmuth lp. from bedside-to-bench: what disease-associated variants are teaching us about the nmda receptor. j physiol. 2021;599(2):397-416. hu c, chen w, myers sj, yuan h, traynelis sf. human grin2b variants in neurodevelopmental disorders. j pharmacol sci. 2016;132(2):115-21. cox ja, kucenas s, voigt mm. molecular characterization and embryonic expression of the family of n-methyl-d-aspartate receptor subunit genes in the zebrafish. dev dyn. 2005;234(3):756-66. veldman mb, lin s. zebrafish as a developmental model organism for pediatric research. pediatr res. 2008;64(5):470-6. choi ty, choi ti, lee yr, choe sk, kim ch. zebrafish as an animal model for biomedical research. exp mol med. 2021;53(3):310-7. zoodsma jd, chan k, bhandiwad aa, golann dr, liu g, syed sa, et al. a model to study nmda receptors in early nervous system development. j neurosci. 2020;40(18):3631-45. zoodsma jd, keegan ej, moody gr, bhandiwad aa, napoli aj, burgess ha, et al. disruption of grin2b, an asd-associated gene, produces social deficits in zebrafish. mol autism. 2022;13(1):38. baraban sc, taylor mr, castro pa, baier h. pentylenetetrazole induced changes in zebrafish behavior, neural activity and c-fos expression. neuroscience. 2005;131(3):759-68. shams s, rihel j, ortiz jg, gerlai r. the zebrafish as a promising tool for modeling human brain disorders: a review based upon an ibns symposium. neurosci biobehav rev. 2018;85:176-90. de abreu ms, genario r, giacomini a, demin ka, lakstygal am, amstislavskaya tg, et al. zebrafish as a model of neurodevelopmental disorders. neuroscience. 2020;445:3-11. weinschutz mendes h, neelakantan u, liu y, fitzpatrick se, chen t, wu w, et al. high-throughput functional analysis of autism genes in zebrafish identifies convergence in dopaminergic and neuroimmune pathways. cell rep. 2023;42(3):112243. howe k, clark md, torroja cf, torrance j, berthelot c, muffato m, et al. the zebrafish reference genome sequence and its relationship to the human genome. nature. 2013;496(7446):498-503. rihel j, schier af. behavioral screening for neuroactive drugs in zebrafish. dev neurobiol. 2012;72(3):373-85. baraban sc, dinday mt, hortopan ga. drug screening in scn1a zebrafish mutant identifies clemizole as a potential dravet syndrome treatment. nat commun. 2013;4:2410. zhang y, kecskes a, copmans d, langlois m, crawford ad, ceulemans b, et al. pharmacological characterization of an antisense knockdown zebrafish model of dravet syndrome: inhibition of epileptic seizures by the serotonin agonist fenfluramine. plos one. 2015;10(5):e0125898. dingemans ajm, jansen s, van reeuwijk j, de leeuw n, pfundt r, schuurs-hoeijmakers j, et al. prevalence of comorbidities in individuals with neurodevelopmental disorders from the aggregated phenomics data of 51,227 pediatric individuals. nat med. 2024;30(7):1994-2003. ohtsuki t, sakurai k, dou h, toru m, yamakawa-kobayashi k, arinami t. mutation analysis of the nmdar2b (grin2b) gene in schizophrenia. mol psychiatry. 2001;6(2):211-6. arnold pd, rosenberg dr, mundo e, tharmalingam s, kennedy jl, richter ma. association of a glutamate (nmda) subunit receptor gene (grin2b) with obsessive-compulsive disorder: a preliminary study. psychopharmacology (berl). 2004;174(4):530-8. saunders ja, tatard-leitman vm, suh j, billingslea en, roberts tp, siegel sj. knockout of nmda receptors in parvalbumin interneurons recreates autism-like phenotypes. autism res. 2013;6(2):69-77. gandal mj, sisti j, klook k, ortinski pi, leitman v, liang y, et al. gabab-mediated rescue of altered excitatory-inhibitory balance, gamma synchrony and behavioral deficits following constitutive nmdar-hypofunction. transl psychiatry. 2012;2(7):e142. barnby g, abbott a, sykes n, morris a, weeks de, mott r, et al. candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at grin2a and abat. am j hum genet. 2005;76(6):950-66. hung ly, margolis kg. autism spectrum disorders and the gastrointestinal tract: insights into mechanisms and clinical relevance. nat rev gastroenterol hepatol. 2024;21(3):142-63. owens ap, mathias cj, iodice v. autonomic dysfunction in autism spectrum disorder. front integr neurosci. 2021;15:787037. orefice ll. peripheral somatosensory neuron dysfunction: emerging roles in autism spectrum disorders. neuroscience. 2020;445:120-9. makris g, agorastos a, chrousos gp, pervanidou p. stress system activation in children and adolescents with autism spectrum disorder. front neurosci. 2021;15:756628. ramos-vicente d, grant sg, bayes a. metazoan evolution and diversity of glutamate receptors and their auxiliary subunits. neuropharmacology. 2021;195:108640. amores a, force a, yan yl, joly l, amemiya c, fritz a, et al. zebrafish hox clusters and vertebrate genome evolution. science. 1998;282(5394):1711-4. chou th, tajima n, romero-hernandez a, furukawa h. structural basis of functional transitions in mammalian nmda receptors. cell. 2020;182(2):357-71 e13. zhang y, ye f, zhang t, lv s, zhou l, du d, et al. structural basis of ketamine action on human nmda receptors. nature. 2021;596(7871):301-5. wollmuth lp, sakmann b. different mechanisms of ca2+ transport in nmda and ca2+-permeable ampa glutamate receptor channels. j gen physiol. 1998;112(5):623-36. watanabe j, beck c, kuner t, premkumar ls, wollmuth lp. drpeer: a motif in the extracellular vestibule conferring high ca2+ flux rates in nmda receptor channels. j neurosci. 2002;22(23):10209-16. wollmuth lp, sobolevsky ai. structure and gating of the glutamate receptor ion channel. trends neurosci. 2004;27(6):321-8. ogden kk, khatri a, traynelis sf, heldt sa. potentiation of glun2c/d nmda receptor subtypes in the amygdala facilitates the retention of fear and extinction learning in mice. neuropsychopharmacology. 2014;39(3):625-37. wollmuth lp. ion permeation in ionotropic glutamate receptors: still dynamic after all these years. curr opin physiol. 2018;2:36-41. mcdaniel mj, ogden kk, kell sa, burger pb, liotta dc, traynelis sf. nmda receptor channel gating control by the pre-m1 helix. j gen physiol. 2020;152(4). wollmuth lp, kuner t, sakmann b. adjacent asparagines in the nr2-subunit of the nmda receptor channel control the voltage-dependent block by extracellular mg2+. j physiol. 1998;506 ( pt 1)(pt 1):13-32. romero-hernandez a, simorowski n, karakas e, furukawa h. molecular basis for subtype specificity and high-affinity zinc inhibition in the glun1-glun2a nmda receptor amino-terminal domain. neuron. 2016;92(6):1324-36. mony l, zhu s, carvalho s, paoletti p. molecular basis of positive allosteric modulation of glun2b nmda receptors by polyamines. embo j. 2011;30(15):3134-46. lichnerova k, kaniakova m, park sp, skrenkova k, wang yx, petralia rs, et al. two n-glycosylation sites in the glun1 subunit are essential for releasing n-methyl-d-aspartate (nmda) receptors from the endoplasmic reticulum. j biol chem. 2015;290(30):18379-90. waterhouse a, bertoni m, bienert s, studer g, tauriello g, gumienny r, et al. swiss-model: homology modelling of protein structures and complexes. nucleic acids res. 2018;46(w1):w296-w303. bienert s, waterhouse a, de beer ta, tauriello g, studer g, bordoli l, et al. the swiss-model repository-new features and functionality. nucleic acids res. 2017;45(d1):d313-d9. warnet xl, bakke krog h, sevillano-quispe og, poulsen h, kjaergaard m. the c-terminal domains of the nmda receptor: how intrinsically disordered tails affect signalling, plasticity and disease. eur j neurosci. 2021;54(8):6713-39. kim k, saneyoshi t, hosokawa t, okamoto k, hayashi y. interplay of enzymatic and structural functions of camkii in long-term potentiation. j neurochem. 2016;139(6):959-72. bej a, ames jb. chemical shift assignments of calmodulin bound to a cytosolic domain of glun2a (residues 1004-1024) from the nmda receptor. biomol nmr assign. 2023;17(1):89-93. horak m, wenthold rj. different roles of c-terminal cassettes in the trafficking of full-length nr1 subunits to the cell surface. j biol chem. 2009;284(15):9683-91. lin jw, wyszynski m, madhavan r, sealock r, kim ju, sheng m. yotiao, a novel protein of neuromuscular junction and brain that interacts with specific splice variants of nmda receptor subunit nr1. j neurosci. 1998;18(6):2017-27. stanic j, carta m, eberini i, pelucchi s, marcello e, genazzani aa, et al. rabphilin 3a retains nmda receptors at synaptic sites through interaction with glun2a/psd-95 complex. nat commun. 2015;6:10181. cousins sl, stephenson fa. identification of c-terminal binding protein 1 as a novel nmda receptor interactor. neurochem res. 2019;44(6):1437-45. franchini l, carrano n, di luca m, gardoni f. synaptic glun2a-containing nmda receptors: from physiology to pathological synaptic plasticity. int j mol sci. 2020;21(4). swanwick cc, shapiro me, yi z, chang k, wenthold rj. nmda receptors interact with flotillin-1 and -2, lipid raft-associated proteins. febs lett. 2009;583(8):1226-30. cousins sl, stephenson fa. identification of n-methyl-d-aspartic acid (nmda) receptor subtype-specific binding sites that mediate direct interactions with scaffold protein psd-95. j biol chem. 2012;287(16):13465-76. vissel b, krupp jj, heinemann sf, westbrook gl. a use-dependent tyrosine dephosphorylation of nmda receptors is independent of ion flux. nat neurosci. 2001;4(6):587-96. lavezzari g, mccallum j, lee r, roche kw. differential binding of the ap-2 adaptor complex and psd-95 to the c-terminus of the nmda receptor subunit nr2b regulates surface expression. neuropharmacology. 2003;45(6):729-37. chen bs, gray ja, sanz-clemente a, wei z, thomas ev, nicoll ra, et al. sap102 mediates synaptic clearance of nmda receptors. cell rep. 2012;2(5):1120-8. elagabani mn, brisevac d, kintscher m, pohle j, kohr g, schmitz d, et al. subunit-selective n-methyl-d-aspartate (nmda) receptor signaling through brefeldin a-resistant arf guanine nucleotide exchange factors brag1 and brag2 during synapse maturation. j biol chem. 2016;291(17):9105-18. yaka r, thornton c, vagts aj, phamluong k, bonci a, ron d. nmda receptor function is regulated by the inhibitory scaffolding protein, rack1. proc natl acad sci u s a. 2002;99(8):5710-5. wyszynski m, lin j, rao a, nigh e, beggs ah, craig am, et al. competitive binding of alpha-actinin and calmodulin to the nmda receptor. nature. 1997;385(6615):439-42. wechsler a, teichberg vi. brain spectrin binding to the nmda receptor is regulated by phosphorylation, calcium and calmodulin. embo j. 1998;17(14):3931-9. krapivinsky g, krapivinsky l, manasian y, ivanov a, tyzio r, pellegrino c, et al. the nmda receptor is coupled to the erk pathway by a direct interaction between nr2b and rasgrf1. neuron. 2003;40(4):775-84. mukherjee p, roy s, ghosh d, nandi sk. role of animal models in biomedical research: a review. lab anim res. 2022;38(1):18. chou th, epstein m, fritzemeier rg, akins ns, paladugu s, ullman ez, et al. molecular mechanism of ligand gating and opening of nmda receptor. nature. 2024;632(8023):209-17. armstrong n, sun y, chen gq, gouaux e. structure of a glutamate-receptor ligand-binding core in complex with kainate. nature. 1998;395(6705):913-7. armstrong n, gouaux e. mechanisms for activation and antagonism of an ampa-sensitive glutamate receptor: crystal structures of the glur2 ligand binding core. neuron. 2000;28(1):165-81. kazi r, gan q, talukder i, markowitz m, salussolia cl, wollmuth lp. asynchronous movements prior to pore opening in nmda receptors. j neurosci. 2013;33(29):12052-66. amin jb, gochman a, he m, certain n, wollmuth lp. nmda receptors require multiple pre-opening gating steps for efficient synaptic activity. neuron. 2021;109(3):488-501 e4. hardingham ge, do kq. linking early-life nmdar hypofunction and oxidative stress in schizophrenia pathogenesis. nat rev neurosci. 2016;17(2):125-34. landrum mj, lee jm, riley gr, jang w, rubinstein ws, church dm, et al. clinvar: public archive of relationships among sequence variation and human phenotype. nucleic acids res. 2014;42(database issue):d980-5. landrum mj, lee jm, benson m, brown g, chao c, chitipiralla s, et al. clinvar: public archive of interpretations of clinically relevant variants. nucleic acids res. 2016;44(d1):d862-8. landrum mj, lee jm, benson m, brown gr, chao c, chitipiralla s, et al. clinvar: improving access to variant interpretations and supporting evidence. nucleic acids res. 2018;46(d1):d1062-d7. gudmundsson s, singer-berk m, watts na, phu w, goodrich jk, solomonson m, et al. variant interpretation using population databases: lessons from gnomad. hum mutat. 2022;43(8):1012-30. tian w, skolnick j. how well is enzyme function conserved as a function of pairwise sequence identity? j mol biol. 2003;333(4):863-82. addou s, rentzsch r, lee d, orengo ca. domain-based and family-specific sequence identity thresholds increase the levels of reliable protein function transfer. j mol biol. 2009;387(2):416-30. sangar v, blankenberg dj, altman n, lesk am. quantitative sequence-function relationships in proteins based on gene ontology. bmc bioinformatics. 2007;8:294. perszyk re, kristensen as, lyuboslavsky p, traynelis sf. three-dimensional missense tolerance ratio analysis. genome res. 2021;31(8):1447-61. uversky vn. intrinsically disordered proteins from a to z. int j biochem cell biol. 2011;43(8):1090-103. singleton md, eisen mb. evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. plos comput biol. 2024;20(4):e1012028. davey ne, cowan jl, shields dc, gibson tj, coldwell mj, edwards rj. slimprints: conservation-based discovery of functional motif fingerprints in intrinsically disordered protein regions. nucleic acids res. 2012;40(21):10628-41. additional declarations no competing interests reported. supplementary files zfishnmdarssupplement20250716.docx cite share download pdf status: under review version 1 posted editorial decision: revision requested 24 sep, 2025 reviews received at journal 19 sep, 2025 reviewers agreed at journal 19 sep, 2025 reviews received at journal 09 sep, 2025 reviews received at journal 18 aug, 2025 reviewers agreed at journal 04 aug, 2025 reviewers agreed at journal 28 jul, 2025 reviewers invited by journal 28 jul, 2025 editor assigned by journal 28 jul, 2025 editor invited by journal 23 jul, 2025 submission checks completed at journal 22 jul, 2025 first submitted to journal 22 jul, 2025 you are reading this latest preprint version research square lets you share your work early, gain feedback from the community, and start making changes to your manuscript prior to peer review in a journal. as a division of research square company, we’re committed to making research communication faster, fairer, and more useful. we do this by developing innovative software and high quality services for the global research community. our growing team is made up of researchers and industry professionals working together to solve the most critical problems facing scientific publishing. also discoverable on platform about our team in review editorial policies advisory board help center resources author services research quality evaluation accessibility api access rss feed manage cookie preferences © research square 2025 | issn 2693-5015 (online) privacy policy terms of service do not sell my personal information {"props":{"pageprops":{"initialdata":{"identity":"rs-7151578","acceptedtermsandconditions":true,"allowdirectsubmit":false,"archivedversions":[],"articletype":"research article","associatedpublications":[],"authors":[{"id":492782335,"identity":"6084adf8-9103-4ca8-8de5-679d2a4ea2bb","order_by":0,"name":"erica r. nebet","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"erica","middlename":"r.","lastname":"nebet","suffix":""},{"id":492782336,"identity":"68eb1705-4eac-413e-bc82-7b9943220d12","order_by":1,"name":"christieann aprea","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"christieann","middlename":"","lastname":"aprea","suffix":""},{"id":492782337,"identity":"776ee37e-7ff9-4a8a-bc43-f965303e90a8","order_by":2,"name":"josiah d. zoodsma","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"josiah","middlename":"d.","lastname":"zoodsma","suffix":""},{"id":492782338,"identity":"ef7f492d-83d8-41a3-aa81-6be2af423f07","order_by":3,"name":"william raab","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"william","middlename":"","lastname":"raab","suffix":""},{"id":492782339,"identity":"6949754f-6430-4ffc-8673-da5548d44bd6","order_by":4,"name":"howard i. sirotkin","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"howard","middlename":"i.","lastname":"sirotkin","suffix":""},{"id":492782340,"identity":"a4f8ecb1-612e-43db-8923-d325a2be05c2","order_by":5,"name":"lonnie p. wollmuth","email":"data:image/png;base64,ivborw0kggoaaaansuheugaaazaaaaayaqmaaabi0h/eaaaablbmvex///8aaabvwtn+aaaacxbiwxmaaa7eaaaoxagvkw4baaabhuleqvriiwnggawjybacphcraclymrua5ae5sdajim0dxwiada6fgaqs4yaxwiigfgyklyknblww9x578kccibgfmbf1m++po+nz5zcfe8y7454cp88bxgdv2zc18jxln0g4w5a4s5mx7tzpwrpcdcfy0o15zxqbs/y2mbvoxajfisdmirgnixfdybcww7kb2hjmphnbehi3ngdgazjwapmh0jiu34bqwv4bp5yghjyehmmwlwcb2jixaee5yyarceyihusxm3psdhtuojawjjn3tikxzm/bzsk55zc08lp3men+qlex5mdvpnbjjc1hefnmw8ck3u5iaizy8sepb8owqxkcjfc5tjdoamcokydxb/fqr8eogawjypgdabgnaum7/svdaaaaaelftksuqmcc","orcid":"","institution":"stony brook university","correspondingauthor":true,"prefix":"","firstname":"lonnie","middlename":"p.","lastname":"wollmuth","suffix":""}],"badges":[],"createdat":"2025-07-17 18:08:21","currentversioncode":1,"declarations":"","doi":"10.21203/rs.3.rs-7151578/v1","doiurl":"https://doi.org/10.21203/rs.3.rs-7151578/v1","draftversion":[],"editorialevents":[],"editorialnote":"","failedworkflow":false,"files":[{"id":87954191,"identity":"93406acd-9b5c-4bbe-b8e4-476bfc8e3591","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":0,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":48663694,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarsmain20250722.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/72a01d08b07202a8dd3432a7.docx"},{"id":87954161,"identity":"09dcee36-3c3d-4bae-a5b6-a8031a04e2f8","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"json","order_by":1,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":8698,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b.json","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/088093afe258318de15a5f12.json"},{"id":87954169,"identity":"c1d3acb0-9e25-4b8a-9228-e0e154b314b2","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":2,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":15353369,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarssupplement20250716.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a3a65bcc549c7954c295d2d1.docx"},{"id":87954163,"identity":"6fde5d88-4da9-48b5-ac47-0b0e0f6354d8","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"xml","order_by":3,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":242943,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b1enriched.xml","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/aad420877d9070a2aedac4cc.xml"},{"id":87954167,"identity":"b5c71071-9fd7-476b-9303-17dcb5b1c845","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"jpeg","order_by":4,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":7224546,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage1.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/239b3c6e0e27c5d5f3a3c485.jpeg"},{"id":87954176,"identity":"66a11160-0f4e-44e0-a6b3-26a6c526f9c1","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":5,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5495318,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage2.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/7c759429dc2c4ccf2d95e1ed.jpeg"},{"id":87954168,"identity":"e3a84d67-d0ce-4822-8529-9e1cc04acd3d","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":6,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5497046,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage3.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/5ab1e35184bc289007b868d7.jpeg"},{"id":87954408,"identity":"4452df13-a563-4564-9a2c-9a250a285ebb","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"jpeg","order_by":7,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4536836,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage4.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/34cad84600342cebbb343c8d.jpeg"},{"id":87954171,"identity":"94757fe0-c77c-46b6-8a13-76b5920bc170","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":8,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4702238,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage5.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/704be5ce4c0cc10b6f98b684.jpeg"},{"id":87954173,"identity":"bbb91acc-360b-43aa-99a5-89fbc5989777","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":9,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4378230,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage6.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/85acc94c54964dbcd4ee6caa.jpeg"},{"id":87954192,"identity":"2447fad7-ffab-4e33-9b2f-a71d2af30f9e","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":10,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5735470,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage7.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3a0654f7ccd9aa40cc5d5f69.jpeg"},{"id":87954413,"identity":"29bec889-0025-4009-9267-6438ea9c5740","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"jpeg","order_by":11,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5327046,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage8.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/c3009b0732aa42082efe8df3.jpeg"},{"id":87954188,"identity":"20ba216a-d094-494f-9894-1937509c8d81","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":12,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":222456,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage1.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/08aba5e904980f68ab32639e.png"},{"id":87954187,"identity":"d99856fc-69f6-4dda-9eeb-4cafd6cc99a0","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":13,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":175317,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage2.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a0ea7afa8616702c9ac66b10.png"},{"id":87954174,"identity":"de47bd92-e161-4fbe-a9b2-437c4b5f0573","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":14,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":140021,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage3.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/bb8abfaa1c199dae266df4dd.png"},{"id":87954412,"identity":"87f7cc6b-e10f-4300-9539-2c2a02855ddb","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"png","order_by":15,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":138717,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage4.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a6b8e16a616611723ae39dab.png"},{"id":87954190,"identity":"ad2ebfe9-5bbc-4128-8d93-eaa9d7470a57","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":16,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":96612,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage5.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/99939544fae92f971073fd51.png"},{"id":87954189,"identity":"84923e46-044f-4cd2-9f5a-b3a9fa41caab","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":17,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":72313,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage6.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/92848d9ff37daaad2b48e9ff.png"},{"id":87954755,"identity":"eaa7ae44-7e94-4117-97cb-d03c1d07f90a","added_by":"auto","created_at":"2025-07-30 18:41:44","extension":"png","order_by":18,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":83342,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage7.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/845b774d666648bfa0a844ab.png"},{"id":87954410,"identity":"13827b86-cf06-4cf3-9952-4b424318037f","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"png","order_by":19,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":117211,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage8.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/af95a59a499464e286bef54b.png"},{"id":87954411,"identity":"7741fc55-f9dc-44b6-9b97-8875d99f5f4b","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"xml","order_by":20,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":240176,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b1structuring.xml","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/dfb8648302675ebf344a2fa6.xml"},{"id":87954756,"identity":"beb85617-8838-4c99-8712-1a7d62a23eff","added_by":"auto","created_at":"2025-07-30 18:41:44","extension":"html","order_by":21,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":257708,"visible":true,"origin":"","legend":"","description":"","filename":"earlyproof.html","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/01f35be0d8432473e2775041.html"},{"id":87954160,"identity":"3920a076-09de-49c0-b764-c081048b3b14","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":1,"title":"figure 1","display":"","copyasset":false,"role":"figure","size":795350,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of human nmdar subunits in zebrafish\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a)\u003c/strong\u003e nmdars are composed of two obligatory glun1 and typically two glun2a-d subunits. the tetrameric complex consists of four domains: extracellular n-terminal (ntd) and ligand-binding (lbd) domains; a transmembrane domain (tmd); and an intracellular c-terminal domain (ctd). glun1 subunits are colored dark blue (ntd), teal (lbd), red (lbd-tmd linkers), light orange (tmd), and gray (ctd); and glun2b subunits gray. 9are (82)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b)\u003c/strong\u003e topology of an individual glun1 subunit. lbd lobes (d1 and d2) are shown; d1 is mainly comprisedof s1 and d2 of s2. the ion channel is formed by 3 transmembrane segments, m1, m3, and m4, and an intracellular m2 pore loop. lbd-tmd linkers are s1-m1 (or d2-m1), m3-s2 (or d2-m3), and s2-m4 (or d1-m4).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c)\u003c/strong\u003e nmdar subunits are separated into 13 structural subdivisions: ntd, lbd (s1 \u0026amp; s2), lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4), tmd (m1, intracellular m1-m2 linker, m2 loop, m2-m3 linker, m3, \u0026amp; m4), and ctd. amino acid length of each zebrafish subdivision is indicated. those sharing amino acid length with their human orthologues are indicated in white, while increasingly darker pink indicates those with increasingly greater amino acid differences (greater (+) or fewer (-)).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(d-e)\u003c/strong\u003e nmdar subunits are grouped as in \u003cstrong\u003e(c)\u003c/strong\u003e. homology between human and zebrafish nmdar subunit paralogues is reported in amino acid identity \u003cstrong\u003e(d)\u003c/strong\u003e or similarity \u003cstrong\u003e(e)\u003c/strong\u003e. heat map with darker green indicating greater percent of homology.\u003c/p\u003e","description":"","filename":"floatimage1.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/5b4e27c91809ed8be6a2b3b5.png"},{"id":87954407,"identity":"47d4fb26-35bf-49ca-aae6-ed196317e2be","added_by":"auto","created_at":"2025-07-30 18:33:43","extension":"png","order_by":2,"title":"figure 2","display":"","copyasset":false,"role":"figure","size":712960,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of positions with pathogenic and putatively pathogenic missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a-b) \u003c/strong\u003epercent conservation of pathogenic variant positions in zebrafish genes, as defined in table 2. conservation is reported in amino acid identity \u003cstrong\u003e(a) \u003c/strong\u003eand similarity \u003cstrong\u003e(b)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c-d)\u003c/strong\u003e conservation of pathogenic variant positions in each nmdar domain, grouped as in figure 1. homology is reported in identity \u003cstrong\u003e(c)\u003c/strong\u003eand similarity \u003cstrong\u003e(d)\u003c/strong\u003e. dashes indicate domains without reported pathogenic variants. along with the percent of conservation, the total number of identified variants in the human protein is indicated in parentheses in the ‘a’ paralog column in \u003cstrong\u003e(c)\u003c/strong\u003e.\u003c/p\u003e","description":"","filename":"floatimage2.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/d7d962b025e722b029324c59.png"},{"id":87954162,"identity":"02953530-369f-4594-b587-6d86079d6a68","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":3,"title":"figure 3","display":"","copyasset":false,"role":"figure","size":667858,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of positions with putatively benign and benign missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a-b) \u003c/strong\u003epercent conservation of putatively benign variant positions in zebrafish genes, as defined in table 2. conservation is reported in identity \u003cstrong\u003e(a) \u003c/strong\u003eand similarity \u003cstrong\u003e(b)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c-d) \u003c/strong\u003econservation of putatively benign variant positions in each nmdar domain, grouped as in figure 2. homology is reported in identity \u003cstrong\u003e(c)\u003c/strong\u003e and similarity \u003cstrong\u003e(d)\u003c/strong\u003e. dashes indicate domains without reported pathogenic variants. along with the percent of conservation, the total number of identified variants in the human protein is indicated in parentheses in the ‘a’ paralog column in \u003cstrong\u003e(c)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(e-h) \u003c/strong\u003eas in a-d but for benign variant positions.\u003c/p\u003e","description":"","filename":"floatimage3.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/069549024abcf2bb76a43e0e.png"},{"id":87954166,"identity":"f30749e8-12a9-42b8-89f5-26186b255ecb","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":4,"title":"figure 4","display":"","copyasset":false,"role":"figure","size":473605,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key functional elements and structure of the nmdar ntd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003etopology of glun2a ntd, showing r1 and r2 subdomains, bound to zinc (red). 5tpw (58)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b) \u003c/strong\u003eclose-up image of structure in \u003cstrong\u003e(a)\u003c/strong\u003ehighlighting amino acids that participate in zinc binding. his44 is demarcated as it is not conserved in the zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c) \u003c/strong\u003econservation of amino acids that participate in ntd zinc binding in glun2a and glun2b, reported in identity (dark red) and similarity (light red). sequences are shown to the right.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(d) \u003c/strong\u003econservation of glun2b ntd regions that participate in interactions with polyamines, reported in identity (dark gray) and similarity (light gray). sequences are shown to the right.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(e) \u003c/strong\u003econservation of ntd glycosylation sites, reported in identity (dark gray).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(f) \u003c/strong\u003ealigned ntd homology models of the human (blue) and zebrafish (gray) protein for glun1 vs zglun1a (reference structure, 5tp0) (\u003cem\u003eleft\u003c/em\u003e), glun2a vs zglun2aa (reference structure, 5tp0) (\u003cem\u003emiddle\u003c/em\u003e), and glun2b vs zglun2bb (reference structure, 5tpz) (\u003cem\u003eright\u003c/em\u003e). secondary structures colored both blue and gray indicate areas of structural overlap.\u003c/p\u003e","description":"","filename":"floatimage4.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3693c16c36a6ea0f931e6343.png"},{"id":87954406,"identity":"16a86e66-9b13-469f-91ca-fe2142d4ce9c","added_by":"auto","created_at":"2025-07-30 18:33:43","extension":"png","order_by":5,"title":"figure 5","display":"","copyasset":false,"role":"figure","size":427432,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key nmdar ctd phosphorylation sites\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd kinase-regulated phosphorylation sites for the glun1, glun2a and glun2b ctds. the specific phosphorylation sites and the corresponding human and zebrafish sequences are indicated. positions sharing identity are indicated in dark gray. similarity is shown in light gray; here, a residue is only considered similar if it is ‘phosphorylatable’ (e.g., contains a s, t, or y) (see methods).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-c) \u003c/strong\u003epercent conservation of kinase-regulated phosphorylation sites \u003cstrong\u003e(b)\u003c/strong\u003e and other phosphorylation sites identified by phosphoproteomics \u003cstrong\u003e(c)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray); similarity is defined in \u003cstrong\u003e(a)\u003c/strong\u003e.\u003c/p\u003e","description":"","filename":"floatimage5.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/25001bb8418306120cd3e005.png"},{"id":87954184,"identity":"a64d1f54-7d76-472b-a396-d9efcf5f86c8","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":6,"title":"figure 6","display":"","copyasset":false,"role":"figure","size":333594,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key nmdar ctd post-translational modification (ptm) sites\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd palmitoylation and ubiquitylation sites for the glun1, glun2a and glun2b ctds. the specific phosphorylation sites and the corresponding human and zebrafish sequences are indicated. positions sharing identity are indicated in dark gray.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-c) \u003c/strong\u003epercent conservation of palmitoylation \u003cstrong\u003e(b)\u003c/strong\u003e and ubiquitylation \u003cstrong\u003e(c)\u003c/strong\u003e sites, reported in identity (dark gray).\u003c/p\u003e","description":"","filename":"floatimage6.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/fac7b19510f29b178f7e3661.png"},{"id":87954182,"identity":"d09e842a-ccf2-41a7-bd3e-0935363761f9","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":7,"title":"figure 7","display":"","copyasset":false,"role":"figure","size":403546,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of nmdar short linear motif (slim) protein-binding motifs (pbms) in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd short linear motifs (slims) for the glun1, glun2a and glun2b ctds.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-d) \u003c/strong\u003epercent conservation of ctd slims in glun1 \u003cstrong\u003e(b)\u003c/strong\u003e, glun2a \u003cstrong\u003e(c)\u003c/strong\u003e, and glun2b \u003cstrong\u003e(d)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray).\u003c/p\u003e","description":"","filename":"floatimage7.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/b6119c0825c4063a5e6bb6ec.png"},{"id":87954181,"identity":"cb3e5a1b-cd98-4155-84bb-c4d2fbb40f7e","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":8,"title":"figure 8","display":"","copyasset":false,"role":"figure","size":518323,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of nmdar protein-binding motifs (pbms) regions in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd regions for the glun1, glun2a and glun2b ctds.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-d) \u003c/strong\u003epercent conservation of ctd regions in glun1 \u003cstrong\u003e(b)\u003c/strong\u003e, glun2a \u003cstrong\u003e(c)\u003c/strong\u003e, and glun2b \u003cstrong\u003e(d)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray).\u003c/p\u003e","description":"","filename":"floatimage8.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/ccea59247fac75e14ad8ad14.png"},{"id":87954866,"identity":"ae52deed-9d7c-4d30-94e3-0b4ac04fe145","added_by":"auto","created_at":"2025-07-30 18:49:46","extension":"pdf","order_by":0,"title":"","display":"","copyasset":false,"role":"manuscript-pdf","size":5866105,"visible":true,"origin":"","legend":"","description":"","filename":"manuscript.pdf","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/24ce2fd9-cdac-44be-947d-44a6c3564886.pdf"},{"id":87954175,"identity":"cf7d81ad-b94d-4e4f-95f1-e45dee844a82","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":0,"title":"","display":"","copyasset":false,"role":"supplement","size":15353369,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarssupplement20250716.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3ada87e108301cd5d027f600.docx"}],"financialinterests":"no competing interests reported.","formattedtitle":"conservation of human nmda receptor subunits and disease variants in zebrafish","fulltext":[{"header":"background","content":"\u003cp\u003erapid cell-to-cell signaling in the nervous system occurs at specialized structures called synapses. at these sites, the electrical activity of the upstream or presynaptic neuron is converted into the release of a chemical neurotransmitter. this is then converted back into an electrical signal in the postsynaptic neuron by specialized ion channels that are gated by a neurotransmitter, initiating downstream biological signaling cascades. glutamate, the primary excitatory neurotransmitter in the brain, activates a variety of receptors that are essential for synaptic transmission and plasticity. glutamate-gated ion channels play fundamental roles in signaling in both the developing and mature nervous systems (\u003cspan additionalcitationids=\"cr2 cr3 cr4 cr5 cr6 cr7 cr8\" citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e–\u003cspan citationid=\"cr9\" class=\"citationref\"\u003e9\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ethe nmda receptor (nmdar) is a prominent glutamate-gated ion channel subtype (\u003cspan citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e, \u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e). due to its distinct biophysical properties and unique ionotropic and non-ionotropic signaling mechanisms, it contributes to a wide array of nervous and non-nervous system functions (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan additionalcitationids=\"cr11 cr12 cr13\" citationid=\"cr10\" class=\"citationref\"\u003e10\u003c/span\u003e–\u003cspan citationid=\"cr14\" class=\"citationref\"\u003e14\u003c/span\u003e). numerous missense, frameshift, and nonsense mutations (or variants) have been identified in the genes that encode nmdar subunits and are associated with neurodevelopmental disorders, including autism spectrum disorders (asd), intellectual disability, developmental delay, seizures, and schizophrenia (\u003cspan additionalcitationids=\"cr16 cr17 cr18 cr19 cr20 cr21 cr22\" citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e–\u003cspan citationid=\"cr23\" class=\"citationref\"\u003e23\u003c/span\u003e). however, how these variants lead to clinical phenotypes is poorly understood.\u003c/p\u003e\u003cp\u003ezebrafish are a powerful model organism to study neurodevelopmental phenotypes and disorders, including those linked to nmdars (\u003cspan citationid=\"cr19\" class=\"citationref\"\u003e19\u003c/span\u003e, \u003cspan additionalcitationids=\"cr25 cr26 cr27 cr28 cr29 cr30 cr31\" citationid=\"cr24\" class=\"citationref\"\u003e24\u003c/span\u003e–\u003cspan citationid=\"cr32\" class=\"citationref\"\u003e32\u003c/span\u003e). zebrafish develop rapidly and share fundamental developmental and signaling pathways with humans and other vertebrates (\u003cspan citationid=\"cr33\" class=\"citationref\"\u003e33\u003c/span\u003e); they are transparent, allowing for dynamic and non-invasive observations of developmental and cellular processes; and they display robust behaviors that are easily assayed at early larval stages. moreover, larval zebrafish readily take up small molecules from their environment, enabling non-invasive and high-throughput readout of nervous system function in the context of behavioral drug screens (\u003cspan additionalcitationids=\"cr35\" citationid=\"cr34\" class=\"citationref\"\u003e34\u003c/span\u003e–\u003cspan citationid=\"cr36\" class=\"citationref\"\u003e36\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enmdars are heterotetramers composed of two obligatory glun1 (encoded by \u003cem\u003egrin1\u003c/em\u003e) subunits. they are expressed as either diheterotetramers with two glun1 and two glun2 (\u003cem\u003egrin2a-d\u003c/em\u003e) subunits or triheterotetramers of some combination of glun2 and/or glun3 (\u003cem\u003egrin3a-b\u003c/em\u003e) subunits (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e) (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003ea). these nmdar subunits are differentially expressed over the course of neurodevelopment, with glun2a, glun2b, and glun2d being highly expressed during early development, while glun2c, glun3a, and glun3b expression increases into adulthood (\u003cspan citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e). moreover, nmdars display distinct functional and pharmacological properties based on their glun2 or glun3 composition (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e). each subunit consists of four principal domains: an n-terminal domain (ntd), a ligand-binding domain (lbd), a transmembrane domain (tmd) that forms the ion channel, and a c-terminal domain (ctd) (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003eb).\u003c/p\u003e\u003cp\u003e\u003c/p\u003e\u003cp\u003edisease-associated variants (davs) in \u003cem\u003egrin1\u003c/em\u003e, \u003cem\u003egrin2a\u003c/em\u003e, \u003cem\u003egrin2b\u003c/em\u003e, and \u003cem\u003egrin2d\u003c/em\u003e have been identified in pediatric patients with neurodevelopmental disorders (\u003cspan additionalcitationids=\"cr16 cr17\" citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e–\u003cspan citationid=\"cr18\" class=\"citationref\"\u003e18\u003c/span\u003e, \u003cspan citationid=\"cr21\" class=\"citationref\"\u003e21\u003c/span\u003e). these disorders often present with multiple comorbidities, reflecting overlapping neurodevelopmental phenotypes (\u003cspan citationid=\"cr37\" class=\"citationref\"\u003e37\u003c/span\u003e). generally, patients with davs in \u003cem\u003egrin1\u003c/em\u003e present with developmental delay and intellectual disability, though a large proportion of patients also experience seizures (\u003cspan citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e). in contrast, \u003cem\u003egrin2a\u003c/em\u003e and \u003cem\u003egrin2d\u003c/em\u003e are predominantly associated with seizure disorders (\u003cspan citationid=\"cr16\" class=\"citationref\"\u003e16\u003c/span\u003e, \u003cspan citationid=\"cr18\" class=\"citationref\"\u003e18\u003c/span\u003e). \u003cem\u003egrin2b\u003c/em\u003e is a high-risk gene for autism spectrum disorder (asd) (\u003cspan citationid=\"cr17\" class=\"citationref\"\u003e17\u003c/span\u003e, \u003cspan citationid=\"cr38\" class=\"citationref\"\u003e38\u003c/span\u003e, \u003cspan citationid=\"cr39\" class=\"citationref\"\u003e39\u003c/span\u003e), as are \u003cem\u003egrin1\u003c/em\u003e and \u003cem\u003egrin2a\u003c/em\u003e (\u003cspan additionalcitationids=\"cr41\" citationid=\"cr40\" class=\"citationref\"\u003e40\u003c/span\u003e–\u003cspan citationid=\"cr42\" class=\"citationref\"\u003e42\u003c/span\u003e). still, patients with such neurodevelopmental disorders generally experience a variety of both central and peripheral nervous system symptoms (\u003cspan additionalcitationids=\"cr44 cr45\" citationid=\"cr43\" class=\"citationref\"\u003e43\u003c/span\u003e–\u003cspan citationid=\"cr46\" class=\"citationref\"\u003e46\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enmda receptors are highly conserved across species (\u003cspan citationid=\"cr47\" class=\"citationref\"\u003e47\u003c/span\u003e). due to an ancient genome duplication event, zebrafish retain two paralogues for most nmdar genes (e.g., \u003cem\u003egrin1a\u003c/em\u003e \u0026amp; \u003cem\u003egrin1b\u003c/em\u003e) (\u003cspan citationid=\"cr24\" class=\"citationref\"\u003e24\u003c/span\u003e, \u003cspan citationid=\"cr48\" class=\"citationref\"\u003e48\u003c/span\u003e). patch clamp electrophysiology experiments show that zebrafish paralogues display similar functional properties to each other and to their mammalian orthologues (\u003cspan citationid=\"cr27\" class=\"citationref\"\u003e27\u003c/span\u003e, \u003cspan citationid=\"cr28\" class=\"citationref\"\u003e28\u003c/span\u003e). nevertheless, it remains unclear how well the human nmdar genes, including positions with identified missense variants, are conserved in zebrafish. to address this, we investigated nmdar conservation in zebrafish by taking advantage of amino acid sequence alignments (\u003cspan citationid=\"cr28\" class=\"citationref\"\u003e28\u003c/span\u003e) and assessed the conservation of all human nmdar subunits and zebrafish paralogues. we also compared the conservation of positions with missense variants identified in human patients for nmdar subunits expressed during early neurodevelopment. finally, we investigated the conservation of critical functional regions within each structural domain. through this analysis, we define the extent to which critical functional components and regions, with relevance to disease, are conserved across zebrafish nmdars, providing a template to use zebrafish as a model to study nmdars in neurodevelopment and disease pathology.\u003c/p\u003e"},{"header":"methods","content":"\u003cp\u003e\u003cb\u003enmdar sequences and sequence alignments\u003c/b\u003e\u003c/p\u003e\u003cp\u003eamino acid sequences for human genes are (uniprot with protein ids): glun1 (q05586-5; ntd splice isoform, nr1-3), glun2a (q12879-1), glun2b (q13224), glun2c (q14957), glun2d (o15399), glun3a (q8tcu5), and glun3b (o60391).\u003c/p\u003e\u003cp\u003efor mouse subunits: mglun1 (p35438), mglun2a (p35436), mglun2b (q01097), mglun2c (q01098), mglun2d (q03391), mglun2a (a2air5), and mglun3b (q91zu9).\u003c/p\u003e\u003cp\u003efor zebrafish paralogs: zglun1a (f1r366), zglun1b (q6zm67), zglun2aa (a0a8m9pxd6), zglun2ab (f1qde5), zglun2ba (a0a8m9q0i1), zglun2bb (a0a8m1nh89), zglun2ca (e7fh62), zglun2cb (a0a8m3b7k5), zglun2da (i3ni77), zglun2db (e7f3z4), zglun3a (a0a8m3awh5), zglun3ba (a0a8m9p9f6), and zglun3bb (a0a8m6z151).\u003c/p\u003e\u003cp\u003ebreakpoints for nmdar subunit domains and subdomains are based on structures 6whr (rat glun1-glun2b) and 7eu7 (human glun1-glun2a) (\u003cspan citationid=\"cr49\" class=\"citationref\"\u003e49\u003c/span\u003e, \u003cspan citationid=\"cr50\" class=\"citationref\"\u003e50\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ehuman and zebrafish protein sequences were aligned and analyzed for amino acid identity and similarity using align sequences protein blast (blastp), which is publicly available through the national center for biotechnology information (ncbi) or emboss needle pairwise sequence alignment (psa). for each nmdar subunit, the zebrafish paralogs were individually aligned to the human protein. amino acid similarity is defined based on the blosum 62 substitution matrix. for the ntd, lbd, and tmd, these alignments were generally straightforward as the sequences do not diverge greatly in length. in contrast, the ctd is quite large, with the zebrafish sequences generally larger than human. for these alignments, we entered the entire sequence into the alignment tool and used the given result, though this led to a fragmentation of the ctd such that certain portions of the sequence were aligned while others were not. we used the resulting domain-specific conservation values to calculate the conservation of the total protein.\u003c/p\u003e\u003cp\u003e\u003cb\u003evariants analysis\u003c/b\u003e\u003c/p\u003e\u003cp\u003ehuman nmdar variants were identified using clinvar and gnomad (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003eclinvar\u003c/em\u003e. results for each gene were downloaded on september 28th, 2023, using clinvar's bulk download option (\u003cspan class=\"externalref\"\u003e\u003cspan class=\"refsource\"\u003ehttps://www.ncbi.nlm.nih.gov/clinvar/\u003c/span\u003e\u003cspan address=\"https://www.ncbi.nlm.nih.gov/clinvar/\" targettype=\"url\" class=\"reftarget\"\u003e\u003c/span\u003e\u003c/span\u003e). genomic variants that did not result in protein coding changes were removed from the analysis. in the case of \u003cem\u003egrin3a\u003c/em\u003e, protein coding changes in the ppp3r2 gene, a single exon gene whose coding sequence resides within the genomic region of \u003cem\u003egrin3a\u003c/em\u003e and is downloaded alongside \u003cem\u003egrin3a\u003c/em\u003e due to this, was also removed. in cases where multiple protein coding annotations exist for the same variant, only the annotation fitting the canonical transcript was retained.\u003c/p\u003e\u003cp\u003e\u003cem\u003egnomad\u003c/em\u003e: genes were searched for by name on gnomad v4.0.0 (\u003cspan class=\"externalref\"\u003e\u003cspan class=\"refsource\"\u003ehttps://gnomad.broadinstitute.org/\u003c/span\u003e\u003cspan address=\"https://gnomad.broadinstitute.org/\" targettype=\"url\" class=\"reftarget\"\u003e\u003c/span\u003e\u003c/span\u003e) and missense, insertion, deletion, and loss-of-function variant information was exported on september 28th, 2023 as comma separated value (csv) files for each gene. hgvs annotation (protein change) was converted to letter-number-letter annotation.\u003c/p\u003e\u003cp\u003egene variants from clinvar and gnomad were merged using r studio 2023.06.1 build 524 based on the letter-number-letter annotation. the resulting merged file was checked for redundant information or variant duplicates, which were removed.\u003c/p\u003e\u003cp\u003emissense variants were extracted and grouped by pathogenicity as pathogenic, putatively pathogenic, complex, putatively benign, and benign variants based on the criteria in table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e. assessment of conservation of positions with identified human variants was performed using the generated sequence alignments.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar domain-specific analysis\u003c/b\u003e\u003c/p\u003e\u003cp\u003eto further refine the relationship between human and zebrafish nmdar proteins, we analyzed the tmd, lbd-tmd linkers, lbd, ntd, and ctd of glun1, glun2a, glun2b, and glun2d for conservation of key structural and functional features.\u003c/p\u003e\u003cp\u003e\u003cem\u003etransmembrane domain (tmd) and ligand-binding domain-transmembrane domain (lbd-tmd) linkers.\u003c/em\u003e key regions involved in channel gating, mg\u003csup\u003e2+\u003c/sup\u003e block, and ca\u003csup\u003e2+\u003c/sup\u003e flux were identified as described in the literature and analyzed for conservation (\u003cspan additionalcitationids=\"cr52 cr53 cr54 cr55 cr56\" citationid=\"cr51\" class=\"citationref\"\u003e51\u003c/span\u003e–\u003cspan citationid=\"cr57\" class=\"citationref\"\u003e57\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003eligand-binding domain (lbd)\u003c/em\u003e. key regions and specific amino acids involved in van-der-waals interactions with the α-carbon of the ligand, interactions with the α-amino group, electrostatic interactions with the α-carboxyl group, and interactions with the amino acid side chain were identified as described in ramos-vicente et. al. 2021 and analyzed for conservation (\u003cspan citationid=\"cr47\" class=\"citationref\"\u003e47\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003en-terminal domain (ntd)\u003c/em\u003e. key amino acid positions required for the binding of zinc and polyamines were identified via a literature search and their conservation assessed (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr58\" class=\"citationref\"\u003e58\u003c/span\u003e, \u003cspan citationid=\"cr59\" class=\"citationref\"\u003e59\u003c/span\u003e). we also examined the conservation of putative n-linked glycosylation sites (\u003cspan citationid=\"cr60\" class=\"citationref\"\u003e60\u003c/span\u003e). to generate homology models of the zebrafish ntds, we used the template function in swiss-model (\u003cspan citationid=\"cr61\" class=\"citationref\"\u003e61\u003c/span\u003e, \u003cspan citationid=\"cr62\" class=\"citationref\"\u003e62\u003c/span\u003e). as reference structures, we used rodent ntd structures 5tq0 (glun1-glun2a in the presence of edta, no bound zn\u003csup\u003e2+\u003c/sup\u003e for glun1 and glun2a), 5tpz (glun1-glun2b apo state for glun1 and glun2b), and 5tpw (glun1-glun2a with zn\u003csup\u003e2+\u003c/sup\u003e in complex with glun2a) (\u003cspan citationid=\"cr58\" class=\"citationref\"\u003e58\u003c/span\u003e). sequences for human (hglun1, hglun2a, and hglun2b) or zebrafish (zglun1a, zglun1b, glun2aa, glun2ab, glun2ba, glun2bb) ntds were entered individually into swiss-model to generate homology models. the subsequent models were then loaded into pymole (human and zebrafish paralog) and aligned. the resultant root mean square deviation (rmsd) is reported.\u003c/p\u003e\u003cp\u003e\u003cem\u003ecarboxy-terminal domain (ctd)\u003c/em\u003e. ctd-specific conservation in zebrafish was assessed for the glun1, glun2a, and glun2b subunits. glun2d was excluded from this analysis due to the poor conservation of the ctd in these paralogs relative to human. to assess functional relevance, we focused on post-translational modifications (ptms), specifically phosphorylation sites (serine, threonine, tyrosine), ubiquitylation sites (lysine), and palmitoylation sites (cysteine), using a combination of experimentally validated data from phosphositeplus (psp) and published literature (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e). phosphorylation sites with identified kinases or well-characterized functional roles were classified as ‘kinase-regulated phosphorylation sites.’ the remaining phosphorylation sites identified in psp were defined as ‘phosphoproteomic-identified sites.’ note, sites in psp (based on rodents) that did not have a phosphorylation site in humans were excluded from our analysis. in addition, known linear protein-binding motifs were annotated and analyzed for conservation (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e). to assess conservation of protein-binding motifs in the ctds, we categorized sites as either short linear motifs (slims) or broader interaction ‘regions’. slims were defined as experimentally validated, compact sequence elements (typically \u0026lt; 15 residues) that mediate specific and direct interactions with known protein partners. broader regions were defined as extended sequences (\u0026gt; 15 residues) shown to directly bind proteins in biochemical or structural studies but lacking a discrete consensus sequence. all motifs were curated from published literature where discrete sequences were experimentally validated per motif (\u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e, \u003cspan additionalcitationids=\"cr64 cr65 cr66 cr67 cr68 cr69 cr70 cr71 cr72 cr73 cr74 cr75 cr76 cr77 cr78 cr79\" citationid=\"cr63\" class=\"citationref\"\u003e63\u003c/span\u003e–\u003cspan citationid=\"cr80\" class=\"citationref\"\u003e80\u003c/span\u003e), and sequence conservation was examined.\u003c/p\u003e"},{"header":"results","content":"\u003cp\u003eour goal is to compare the structural similarity of zebrafish nmdar (zglun) paralogues to human nmdars (glun), especially regarding human variants. since mice are a common animal model (\u003cspan class=\"citationref\"\u003e81\u003c/span\u003e), as a reference-point for nmdar gene conservation, we initially made global comparisons of both mouse and zebrafish nmdar subunits to human.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eglobal comparison of mouse and zebrafish orthologues to human nmdars\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003enmdars are heterotetramers composed of two obligate glycine-binding glun1 subunits and two glutamate-binding glun2 subunits or a combination of glun2 and glycine-binding glun3 subunits (\u003cspan class=\"citationref\"\u003e1\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e2\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ea,b). glycine and glutamate act as co-agonists for glun2-containing receptors, with binding of both required for receptor activation, while glun3-containing receptors are activated by glycine binding. as an initial assessment of the structural similarity between nmdar orthologues, we made global sequence alignments using the ncbi blastp or emboss psa tools. not surprisingly, given evolutionary proximity, the mouse subunits are highly conserved relative to human: glun1 (97% identity \u0026amp; similarity), glun2a (95% identity; 98% similarity), glun2b (98% identity; 99% similarity), glun2c (88% identity; 90% similarity), glun2d (97% identity \u0026amp; similarity), glun3a (92% identity; 95% similarity), and glun3b (77% identity; 83% similarity) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003eon the other hand, the zebrafish paralogues show reduced conservation. the obligatory glun1 subunits are the most highly conserved, both in terms of identity (\u0026ge;\u0026thinsp;84%) and similarity (\u0026ge;\u0026thinsp;89%) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the other subunits show various degrees of identity and similarity: glun2a (\u0026ge;\u0026thinsp;58% identity; \u0026ge;79% similarity), glun2b (\u0026ge;\u0026thinsp;65% identity; \u0026ge;73% similarity), glun2c (\u0026ge;\u0026thinsp;50% identity; \u0026ge;64% similarity), glun2d (\u0026ge;\u0026thinsp;43% identity; \u0026ge;54% similarity), glun3a (63% identity; 78% similarity), and glun3b (\u0026ge;\u0026thinsp;54% identity, \u0026ge;\u0026thinsp;62% similarity) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). these subunits, thus, retain\u0026thinsp;\u0026ge;\u0026thinsp;50% identity with respect to the human protein, except for zglun2da and zglun2db, which only share 43% and 46% sequence identity, respectively (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). with respect to the zebrafish paralog pairs, there are instances in which one paralog is better conserved than the other. the glun1 paralogs are comparably conserved (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). however, the zglun2aa paralog is more highly conserved than zglun2ab while the zglun2bb shows increased conservation relative to zglun2ba (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the glun2c, glun2d, and glun3b paralogs are conserved to similar extents (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003enevertheless, despite lower conservation relative to mouse, zebrafish offer many technical advantages (see background). in addition, what is critical is the conservation of specific structural and functional domains and subdomains.\u003c/p\u003e\n\u003cdiv align=\"left\" class=\"colspec\"\u003e\u003cbr\u003e\u003c/div\u003e\n\u003ctable id=\"tab1\" border=\"1\"\u003e\n \u003ccaption language=\"en\"\u003e\n \u003cdiv class=\"captionnumber\"\u003etable 1\u003c/div\u003e\n \u003cdiv class=\"captioncontent\"\u003e\n \u003cp\u003eoverall protein conservation in mouse and zebrafish relative to human\u003c/p\u003e\n \u003c/div\u003e\n \u003c/caption\u003e\n \u003cthead\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003ehuman\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003emouse\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003ezebrafish\u003c/p\u003e\n \u003cp\u003eparalogue a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003ezebrafish\u003c/p\u003e\n \u003cp\u003eparalogue b\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003c/thead\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en1\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e938\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e938\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e84%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e89%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e966\u003c/p\u003e\n \u003cp\u003e(-28)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e86%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e90%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e937\u003c/p\u003e\n \u003cp\u003e(-1)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2a\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1464\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e95%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e98%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1464\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e68%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e89%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1460\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;6)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e58%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e79%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1445\u003c/p\u003e\n \u003cp\u003e(-9)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2b\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1484\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e98%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e99%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1482\u003c/p\u003e\n \u003cp\u003e(-2)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e65%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e73%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1417\u003c/p\u003e\n \u003cp\u003e(-31)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e67%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e81%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1770 (+\u0026thinsp;322)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2c\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1233\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e88%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e90%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1239\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;6)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e50%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e64%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1400\u003c/p\u003e\n \u003cp\u003e(-176)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e59%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e79%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1328\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;95)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2d\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1336\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1323\u003c/p\u003e\n \u003cp\u003e(-13)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e43%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e54%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1916 (+\u0026thinsp;580)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e46%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e57%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1676 (+\u0026thinsp;340)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en3a\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1115\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e92%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e95%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1135\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;20)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e63%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e78%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1111\u003c/p\u003e\n \u003cp\u003e(-4)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en3b\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1043\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e77%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e83%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1003\u003c/p\u003e\n \u003cp\u003e(-40)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e54%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e62%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1118\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;75)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e62%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e68%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1114\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;71)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003c/p\u003e\n\u003cp\u003enumbers indicate the percent of mouse or zebrafish paralog conservation with respect to the human protein \u0026ndash; \u0026lsquo;id.\u0026rsquo; is identity and \u0026lsquo;sim.\u0026rsquo; is similarity. the zebrafish a and b paralogs are indicated for each protein. glun3a has only a single paralog in zebrafish. for the mouse and zebrafish protein length, numbers in parentheses indicate the difference in amino acid number relative to human.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of specific structural elements in zebrafish paralogues\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003efor more rigorous comparisons, we examined the conservation within individual subunit domains and subdomains (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the structure of nmdars, like all iglurs, consists of 4 domains that are intrinsic to each individual subunit (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ea,b): the extracellular ntd and lbd; the tmd forming the ion channel as well as the linkers that connect the lbd to the tmd, the lbd-tmd linkers; and the intracellular ctd (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e53\u003c/span\u003e). within individual subunits, the lbd is formed by two disparate sequences, s1 and s2, that fold up into the d1 and d2 lobes in the 3-dimensional structure (83, 84). in general, d1 is made up primarily of s1 and d2 of s2, but there is considerable swapping between lobes. the tmd is formed by three transmembrane segments, m1, m3, and m4, and an m2-pore loop. the lbd-tmd linkers are referred to as either s1-m1, m3-s2, and s2-m4 (linear sequence) or d2-m1, m3-d2, and d1-m4 (3-dimensional structure). this distinction is important functionally since d1 and d2 undergo different movements that are translated differently to the tmd (\u003cspan class=\"citationref\"\u003e85\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e86\u003c/span\u003e). nevertheless, we will use the s1-m1, m3-s2, and s2-m4 designation for ease of comparison to structures.\u003c/p\u003e\n\u003cp\u003ewe began by dividing the zebrafish paralogues into their respective domains and subdomains based on our breakpoints (see methods). interestingly, the amino acid sequences surrounding these breakpoints were well conserved in zebrafish \u0026ndash; the majority were identical (not shown). this allowed for ease of division of the zebrafish proteins into their respective domains and subdomains.\u003c/p\u003e\n\u003cp\u003eas an initial assessment of the structural similarity between human and zebrafish nmdars, we made detailed sequence alignments using the ncbi blastp or emboss psa tools (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e, fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). in terms of length of domains, the zebrafish protein sequence for glun1, glun2, and glun3 subunits was identical to the human sequence for the tmd (m1, m1-m2, m2, m2-m3, m3, \u0026amp; m4) and lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec). the lbd (s1 \u0026amp; s2) was also quite similar in length, with no changes in the length of s2 across the paralogs and only slight increases or decreases in amino acid number in the s1 of zglun1a, zglun1b, zglun2ab, zglun2ca, zglun2ca, zglun2da, and zglun2db (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec). the ntd and ctd were less conserved in terms of size, wherein almost every zebrafish paralog has a different number of amino acids compared to the human. this is especially notable for the glun2b ntd which either lacks 130 amino acids (zglun2ba) or has an additional 74 amino acids (zglun2bb) as well as for the ctd where, for example, zglun2da and zglun2db have a ctd that is more than twice as long as that in the human protein (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec).\u003c/p\u003e\n\u003ctable id=\"tab2\" border=\"1\"\u003e\n \u003ccaption language=\"en\"\u003e\n \u003cdiv class=\"captionnumber\"\u003etable 2\u003c/div\u003e\n \u003cdiv class=\"captioncontent\"\u003e\n \u003cp\u003evariant designation\u003c/p\u003e\n \u003c/div\u003e\n \u003c/caption\u003e\n \u003cthead\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eglun\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003edesignation\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eclinvar\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003egnomad\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e1\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2b\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2c\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2d\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e3a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e3b\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003c/thead\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epathogenic\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;pathogenic\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e49\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e58\u003c/p\u003e\n \u003cp\u003e(4%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e81\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e12\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eputatively\u003c/p\u003e\n \u003cp\u003epathogenic\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;uncertain\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e123\u003c/p\u003e\n \u003cp\u003e(13%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e214\u003c/p\u003e\n \u003cp\u003e(15%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e182\u003c/p\u003e\n \u003cp\u003e(12%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e4\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e111\u003c/p\u003e\n \u003cp\u003e(8%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e5\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e8\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003ecomplex\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;pathogenic\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e3\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e5\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e6\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eputatively benign\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;uncertain\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e80\u003c/p\u003e\n \u003cp\u003e(9%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e320\u003c/p\u003e\n \u003cp\u003e(22%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e199\u003c/p\u003e\n \u003cp\u003e(13%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e31\u003c/p\u003e\n \u003cp\u003e(3%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e223\u003c/p\u003e\n \u003cp\u003e(17%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e30\u003c/p\u003e\n \u003cp\u003e(3%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e68\u003c/p\u003e\n \u003cp\u003e(7%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003ebenign\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;benign\u0026rdquo;\u003c/p\u003e\n \u003cp\u003eor\u003c/p\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent/\u003c/p\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003cp\u003eor\u003c/p\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e401\u003c/p\u003e\n \u003cp\u003e(43%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e813\u003c/p\u003e\n \u003cp\u003e(55%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e672\u003c/p\u003e\n \u003cp\u003e(45%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e945\u003c/p\u003e\n \u003cp\u003e(77%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e863\u003c/p\u003e\n \u003cp\u003e(27%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e59\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e945\u003c/p\u003e\n \u003cp\u003e(91%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003c/p\u003e\n\u003cp\u003enumber of positions with missense variants that have been identified in each subunit per designation. percentages report the proportion of the total protein amino acids at which such variants have been identified.\u003c/p\u003e\n\u003cp\u003ewe used these sequence alignments to assess the degree of amino acid identity and similarity between domains and subdomains (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ed,e). the obligatory glun1 subunit showed the highest degree of conservation. the tmd and lbd-tmd linkers are 100% conserved. the lbd is also well conserved with s1 and s2 showing\u0026thinsp;\u0026ge;\u0026thinsp;92% identity and \u0026ge;\u0026thinsp;93% similarity. the ntd is somewhat less conserved (\u0026ge;\u0026thinsp;80% identity, \u0026ge;\u0026thinsp;88% similarity), as is the ctd (\u0026ge;\u0026thinsp;53% identity; \u0026ge;58% similarity).\u003c/p\u003e\n\u003cp\u003ethis trend of conservation across nmdar subunits \u0026ndash; the lbd and tmd are most highly conserved and the ntd and ctd less so \u0026ndash; is maintained in the glun2 and glun3 subunits (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ed,e). generally, the glun2 subunits maintain a higher sequence identity than the glun3 subunits. overall, this sequence analysis suggests that the lbd and tmd are highly conserved whereas the ntd and ctd show less conservation.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecategorizing the pathogenicity of missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003enmdars with missense, frameshift, and nonsense variants have been identified in pediatric patients with neurodevelopmental disorders (\u003cspan class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e22\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e87\u003c/span\u003e). we therefore wanted to assess the extent to which variants are conserved in zebrafish to aid in guiding future studies. for this analysis, we focused solely on missense variants. while nonsense and frameshift mutation can be modelled in various ways, such as by mutating a portion of the protein to induce a loss-of-function, the conservation of the specific protein sequence is somewhat less relevant. in contrast, the single amino acid change associated with missense variants is critical to dissecting how such variants alter receptor function and lead to disease phenotypes.\u003c/p\u003e\n\u003cp\u003eusing publicly available datasets of nmdar variants from clinvar and gnomad, we identified missense variants in all nmdar subunits (see methods). we first categorized these variants with regards to their pathogenicity (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). clinvar reports variants that have been identified in patients with symptomatic disease (\u003cspan class=\"citationref\"\u003e88\u003c/span\u003e\u0026ndash;\u003cspan class=\"citationref\"\u003e90\u003c/span\u003e), while gnomad reports variants identified in the healthy population (\u003cspan class=\"citationref\"\u003e91\u003c/span\u003e). thus, in general, we define variants present in clinvar but absent in gnomad as pathogenic while those with the opposite reporting as benign. with this approach, pathogenic and putatively pathogenic variants were identified in glun1, glun2a, glun2b, and glun2d while minimal to no such variants were identified in glun2c, glun3a, and glun3b (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). notably, these genes with known pathogenic variants are expressed early in development, contributing to developmental processes while their dysfunction often leads to disease phenotypes (\u003cspan class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e87\u003c/span\u003e). on the other hand, glun2c, glun3a, and glun3b have no pathogenic and a small number of putatively pathogenic variants (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). we therefore focus solely on glun1, glun2a, glun2b, and glun2d for further analysis.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003epathogenic and putatively pathogenic variant positions are well conserved in zebrafish\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003eusing the criteria in table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e, amino acid positions with pathogenic missense variants in glun1, glun2a, glun2b, and glun2d are completely conserved in zebrafish paralogues (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ea-d).\u003c/p\u003e\n\u003cp\u003eputatively pathogenic variants are also conserved in these nmdar subunits, though less so than pathogenic variants. the highest degree of conservation of putatively pathogenic variants is in the glun1 paralogues (\u0026ge;\u0026thinsp;88% identity; 93% similarity). putatively pathogenic variants in the glun2 subunits are also reasonably conserved: zglun2aa (70% identity; 92% similarity), zglun2ab (62% identity; 84% similarity), zglun2ba (65% identity; 71% similarity), zglun2bb (72% identity; 80% similarity), and zglun2da and zglundb (\u0026ge;\u0026thinsp;46% identity; \u0026ge;59% similarity) (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ee,f). in general, the highest level of conservation is in the tmd and tmd-lbd linkers, followed by the lbd, and less so in the ntd and ctd (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003eg,h).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ebenign and putatively benign variant positions show reduced conservation\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ewe carried out a comparable analysis as above for benign and putatively benign variants (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). we hypothesized that positions with more benign variants would be less well conserved than pathogenic ones. indeed, the general trend is consistent with this idea (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003e). putatively benign variants showed a reduced conservation compared to putatively pathogenic variants (putatively pathogenic vs putatively benign): zglun1a (88% vs. 78% identity; 93% vs. 88% similarity), zglun1b (89% vs. 79% identity; 93% vs. 84% similarity), zglun2aa (70% vs. 60% identity; 92% vs. 86% similarity), zglun2ab (62% vs. 48% identity; 84% vs. 68% similarity), zglun2ba (65% vs. 45% identity; 71% vs. 55% similarity), zglun2bb (72% vs. 55% identity; 80% vs. 70% similarity), zglun2da (46% vs. 47% identity; 59% vs. 61% similarity), and zglun2db (47% vs. 44% identity; 61% vs. 56% similarity) (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ee,f vs fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ea,b). for these putatively benign variants, the greatest extent of conservation tends to occur in the tmd and lbd-tmd linkers, followed by the lbd, and then the ntd and ctd (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ec,d).\u003c/p\u003e\n\u003cp\u003efor benign variants, however, the decrease in conservation plateaus, with identity and similarity values comparable to the putatively benign variants, if not slightly higher (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ee-f). the subdomain-specific conservation trend for these variants follows that seen on the level of the whole receptor (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e), wherein the tmd, lbd-tmd linkers, and lbd are generally more conserved and the ntd and ctd less so (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003eg,h). although diverging from our prediction as we do not note a substantial decrease in the conservation of benign variants with respect to the putatively benign variants, this result reflects the overall conservation of the nmdar subunits themselves (benign vs total protein): zglun1 (78% vs. \u0026ge;84% identity; 84% vs. \u0026ge;89% similarity), zglun2aa (62% vs. 68% identity; 86% vs. 89% similarity), zglun2ab (49% vs. 58% identity; 74% vs. 79% similarity), zglun2ba (54% vs. 65% identity; 64% vs. 73% similarity), and zglun2bb (62% vs. 67% identity; 74% vs. 81% similarity), zglun2da (50% vs. 43% identity; 64% vs. 54% similarity), and zglun2db (49% vs. 46% identity; 62% vs. 57% similarity) (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ee-f, table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). furthermore, this result reflects the relatively large number of benign variants in comparison to other variant groups (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecritical tmd and lbd-tmd linker components are completely conserved\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003eelements forming the ion channel \u0026ndash; the tmd (m1, m1-m2, m2, m2-m3, m3, \u0026amp; m4) \u0026ndash; and the lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4) were consistently highly conserved (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). nevertheless, we assayed conservation of critical functional motifs. the s1-m1 linker is a critical element for channel gating and is completely conserved (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e) (\u003cspan class=\"citationref\"\u003e56\u003c/span\u003e). the m2 subdomain contains what is known as the \u0026lsquo;n\u0026rsquo; and \u0026lsquo;n\u0026thinsp;+\u0026thinsp;1\u0026rsquo; sites, which participate in mg\u003csup\u003e2+\u003c/sup\u003e block and ca\u003csup\u003e2+\u003c/sup\u003e permeation through the channel (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e55\u003c/span\u003e) and are conserved in zebrafish (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, dark orange). the tmd m3 segment contains the sytanlaaf motif, which plays a key role in ion channel gating and is known to be highly conserved across species. this motif is conserved in all zebrafish glun1 and glun2 paralogs (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, orange). lastly, the \u0026lsquo;drpeer\u0026rsquo; motif in m3-s2 of glun1, an important component in the high ca\u003csup\u003e2+\u003c/sup\u003e permeability of nmdars (\u003cspan class=\"citationref\"\u003e52\u003c/span\u003e) is conserved (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, light orange). hence, core gating and permeation elements of the tmd and associated regions are highly conserved suggesting that basic mechanisms of ion channel gating and block/permeation are preserved.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003elbd elements required for agonist binding are completely conserved\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ethe glun1 lbd binds glycine or d-serine while the glun2 lbd binds glutamate or aspartate (fig. s2a,b). overall, the lbds are highly conserved in zebrafish relative to humans (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e), as are specific amino acid side chains in s1 and s2 that are involved in binding endogenous agonists (\u003cspan class=\"citationref\"\u003e47\u003c/span\u003e) (fig. s2c). these include sites that participate in van-der waals interactions with the \u0026alpha;-carbon of the ligand (darkest red), interactions with the \u0026alpha;-amino group (dark red), electrostatic interactions with the \u0026alpha;-carboxyl group (red), and interactions with the amino acid side chain (light red) (fig. s2c).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of structure-function elements in the ntd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ein terms of overall sequence comparisons, the ntd shows variable degrees of conservation across nmdar subunits. the glun1 zebrafish paralogs maintain the highest degree of conservation relative to human (80% identity; \u0026ge;88% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the glun2 subunits show decreased ntd conservation (49\u0026ndash;75% identity; 59\u0026ndash;88% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). given this reduced conservation, we assayed the conservation of key structure-function motifs in zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003eamong its many roles, the ntd participates in interactions with ions and molecules that modulate receptor activity (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e59\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ea,b). this is most notable for glun2a and to a lesser extent glun2b as they bind zinc, which inhibits receptor function by reducing channel open probability (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). in glun2a, four amino acids coordinate zinc binding: glun2a-his44, glun2a-his128, glun2a-glu266, and glun2a-asp282 (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). of these key sites, three are conserved (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec) but in zglun2aa and zglun2ab, a key residue involved in zn\u003csup\u003e2+\u003c/sup\u003e binding, glun2a-his44, is not conserved (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec). for glun2b, only two amino acid residues participate in zinc binding: glun2b-his127 and glun2b-glu284 (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). while glun2b-glu284 is conserved, the glun2b-his127 position in zglun2ba is missing due to alterations in ntd sequence length leading to gaps in the alignments, though it is similar now as a positively charged arginine instead of a histidine in zglun2bb (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec).\u003c/p\u003e\n\u003cp\u003ethe ntd of glun2b also binds polyamines, which enhance receptor activity (\u003cspan class=\"citationref\"\u003e59\u003c/span\u003e). nine amino acids are involved in the binding of polyamines. these sites in zglun2ba and zglun2bb are completely conserved except for one amino acid in zglun2bb that is similar but not identical, now a negatively charged aspartate instead of glutamate (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ed). most of this functional component of glun2b is, thus, conserved in zebrafish.\u003c/p\u003e\n\u003cp\u003eadditionally, the nmdar ntd has glycosylation sites, and their glycosylation regulates the trafficking of nmdars to the cell membrane (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e). given the consensus sequence for arginine (n-) glycosylation (n-\u003cem\u003ex\u003c/em\u003e-s/t), glun1 has seven, glun2a has three, and glun2b has three putative glycosylation sites in the ntd (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e). their conservation in the zebrafish paralogues is somewhat variable: zglun1a \u0026amp; zglun1b (5 of 7 sites), zglun2aa (3 of 3 sites), zglun2ab (2 of 3 sites), zglun2ba (2 of 3 sites), and zglun2bb (3 of 3 sites) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ee). to the best of our knowledge, specific glycosylation sites in glun2d have yet to be identified. of note, the glun1 glycosylation sites glun1-n203 and glun1-n368 are necessary for receptor surface expression (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e), and these sites are conserved in the zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003ewhile the critical functional features of the lbd were completely conserved in zebrafish relative to human nmdars, the conservation of key regions in the ntd occasionally diverged, most notably that for zn\u003csup\u003e2+\u003c/sup\u003e binding. to further explore the structural composition of the glun1, glun2a, and glun2b ntds, we used swiss-model to generate homology models for the human and zebrafish nmdar subunits, which we subsequently aligned in pymole to assess conservation at the level of the three-dimensional structure (see methods) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ef). in general, this analysis revealed a high degree of similarity between the human ntd and zebrafish paralogs. the zebrafish glun1 ntd models display similar overall topology to human: glun1 vs zglun1a, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.071); glun1 vs zglun1a, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.085); glun1 vs zglun1b, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.094); and glun1 vs zglun1b, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.097) (table s3).\u003c/p\u003e\n\u003cp\u003ethe glun2a and glun2b subunits also reflect a high degree of structural similarity: glun2a vs zglun2aa, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.085); glun2a vs zglun2ab, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.095); glun2a vs zglun2ab, reference structure 5tpw (rmsd\u0026thinsp;=\u0026thinsp;0.111); glun2b vs zglun2ba, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.028); and glun2b vs glun2bb, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.044) (table s3). the only exception here is the structural comparison of glun2a and zglun2aa using reference structure 5tpw, in which the ntd is interacting with zinc (rmsd\u0026thinsp;=\u0026thinsp;2.149), suggesting that the structural components of zglun2aa may not effectively complex with zinc. interestingly, zglun2ba is \u0026gt;\u0026thinsp;100 amino acids shorter and zglun2bb is \u0026gt;\u0026thinsp;70 amino acids longer than human glun2b. though the zglun2ba model is missing an \u0026alpha;-helix relative to the human protein, the zglun2bb modelled structure aligns well with its human counterpart, suggesting that the zebrafish sequences maintain a large degree of structural integrity.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of post-translational modification (ptm) sites in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ethe intracellular ctd is a key regulatory domain of nmdar cell biology and function and shows distinct subunit-specific elements (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). for our analyses, we focused on the glun1-1 ctd splice variant, which is the longest variant containing the c0-c1-c2 cassettes (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003eacross the nmdar subunits, the zebrafish ctds are the most poorly conserved (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). with a sequence identity generally around 50\u0026ndash;60% for the glun1, glun2a, and glun2b zebrafish paralogs, with some exceptions: zglun1a (53% identity; 58% similarity), zglun1b (54% identity; 59% similarity), zglun2aa (57% identity; 86% identity), zglun2ab (38% identity; 65% similarity), zglun2ba (57% identity, 66% similarity), and zglun2bb (60% identity, 71% similarity) while that of glun2d is much lower (\u0026ge;\u0026thinsp;18% identity, \u0026ge;\u0026thinsp;28% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003egiven these deviations in protein sequences, we first examined the conservation of ctd regions critical for post-translational modifications (ptms), excluding glun2d due its poor conservation in zebrafish. phosphorylation of nmdar subunit ctds confers key functional roles to the receptor (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). initially, we examined experimentally validated sites at which phosphorylation occurs by known kinases and calculated the percent conservation, reported in identity and similarity. similarity, here, is defined as a site at the homologous position that can be phosphorylated (e.g., s, t, or y), if not identical. the glun1 ctd contains phosphorylation sites for src kinase (src), protein kinase c (pkc), and protein kinase a (pka), (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003etop\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). note specific sites have also been identified that are also targeted by the serine/threonine protein phosphatase 2b (pp2b). these ptm sites are perfectly conserved in zglun1a, while three of five are conserved in zglun1b (60% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb). the glun2a ctd is phosphorylated by src, pka, dual specificity tyrosine-phosphorylation-regulated kinase 1a (dyrk1), cyclin-dependent kinase 5 (cdk5), pkc, and calcium/calmodulin-dependent protein kinase ii\u0026alpha; (camkii\u0026alpha;) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003emiddle\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e): zglun2aa (79% identity \u0026amp; similarity) and zglun2ab (64% identity, 71% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb). the ctd of glun2b is phosphorylated by src, camkii\u0026alpha;, cdk5, pka, pkc, death-associated protein kinase 1 (dapk1), proto-oncogene tyrosine-protein kinase fyn (fyn), and casein kinase ii (ck2) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003ebottom\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). glun2b also contains sites for the protein tyrosine phosphatase non-receptor 11 (ptpn11, or shp2) and protein phosphatase 1 (pp1). the conservation of these ptms is slightly higher than that in glun2a: zglun2ba (73% identity \u0026amp; similarity) and zglun2bb (82% identity, 100% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb).\u003c/p\u003e\n\u003cp\u003ein addition to these kinase-regulated sites, we also examined the conservation of other known phosphorylation sites in these subunits identified by phosphoproteomics (psp) (see methods). the conservation of these sites in the zebrafish paralogs was generally lower than that of the kinase-regulated sites: glun1 (67% identity, 100% similarity), zglun2aa (51% identity, 66% similarity), zglun2ab (34% identity, 57% similarity), zglun2ba (60% identity \u0026amp; similarity), and zglun2bb (67% identity, 70% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ec).\u003c/p\u003e\n\u003cp\u003ewe next examined the conservation of ctd palmitoylation and ubiquitylation sites. glun1 has a single ubiquitylation site (lys860) (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003etop\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), and it is conserved in the zebrafish paralogs (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ec). glun2a contains seven palmitoylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003emiddle\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), which are likewise completely conserved in zebrafish (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003eb). the glun2a ctd also has four ubiquitylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003emiddle)\u003c/em\u003e (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), of which three are conserved in zglun2aa (75%) and one in zglun2ab (25%) (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ec). with the same approach for glun2b, we identified eight palmitoylation and six ubiquitylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003ebottom\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). here, all palmitoylation sites are conserved in zebrafish while five of six ubiquitylation sites (83%) are conserved in zglun2ba and all six ubiquitylation sites (100%) are conserved in zglun2bb (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb,c).\u003c/p\u003e\n\u003cp\u003ein summary, the global conservation of ptm sites is as follows: zglun1a and zglun1b (89% identity, 100% similarity), zglun2aa (62% identity, 72% similarity), zglun2ab (45% identity, 62% similarity), zglun2ba (66% identity, 67% similarity), and zglun2bb (75% identity, 78% similarity).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of protein-binding motifs (pbm) in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ein addition to numerous ptm sites, a variety of protein-protein interactions are critical to the role of the ctd in receptor cell biology (surface expression, distribution) and function (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e63\u003c/span\u003e). these protein-binding elements include either defined short linear motifs (slims) or broader regions known to serve as protein docking sites.\u003c/p\u003e\n\u003cp\u003ethe ctd of glun1 contains a slim known as the kkk/rrr er retention motif, which is conserved in zebrafish (\u003cspan class=\"citationref\"\u003e66\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,b). it, likewise, has binding regions for calmodulin (cam)/calcium/calmodulin-dependent protein kinase ii (camkii) and \u0026alpha;-actinin 2 (\u003cspan class=\"citationref\"\u003e64\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e65\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e67\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e78\u003c/span\u003e), which is conserved and yotiao, also known as a-kinase anchoring protein 9 (akap9) (\u003cspan class=\"citationref\"\u003e67\u003c/span\u003e), which is generally well conserved: zglun1a (90% identity, 98% similarity) and zglun1b (90% identity, 98% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,b).\u003c/p\u003e\n\u003cp\u003ethe glun2 subunit ctds contain a variety of pbms. for glun2a, the conservation of slims is: the yxx\u0026oslash; motif, where y is tyrosine, x is any amino acid, and \u0026oslash; is a bulky hydrophobic residue like leucine, isoleucine, phenylalanine, methionine, or valine (\u003cspan class=\"citationref\"\u003e73\u003c/span\u003e) \u0026ndash; both zglun2aa and zglun2ab show 100% identity \u0026amp; similarity; the guanine-nucleotide exchange factor brag2 (\u003cspan class=\"citationref\"\u003e70\u003c/span\u003e) \u0026ndash; zglun2aa (80% identity \u0026amp; similarity) and zglun2ab (40% identity, 60% similarity); postsynaptic density protein 95 (psd-95) (\u003cspan class=\"citationref\"\u003e72\u003c/span\u003e) \u0026ndash; zglun2aa (50% identity; 63% similarity) and zglun2ab (38% identity, 63% similarity); and the pdz domain that interacts with the proteins psd-95, discs large (dlg), and zonula occludens-1 (zo-1) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2aa and zglun2ab (100% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,c). interacting regions in glun2a include: ring finger protein 10 (rnf10) (\u003cspan class=\"citationref\"\u003e70\u003c/span\u003e) \u0026ndash; zglun2aa (51% identity, 72% similarity) and zglun2ab (32% identity, 66% similarity); rabphilin 3a (rph3a) (\u003cspan class=\"citationref\"\u003e68\u003c/span\u003e) \u0026ndash; zglun2aa (71% identity, 80% similarity) and zglun2ab (49% identity, 71% similarity); flotillin-1 (flot-1) (\u003cspan class=\"citationref\"\u003e71\u003c/span\u003e) \u0026ndash; zglun2aa (61% identity, 81% similarity) and zglun2ab (31% identity, 52% similarity); and c-terminal binding protein 1 (ctbp1) (\u003cspan class=\"citationref\"\u003e69\u003c/span\u003e) \u0026ndash; zglun2aa (51% identity, 85% similarity) and zglun2ab (36% identity, 63% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,c, s3a,b).\u003c/p\u003e\n\u003cp\u003efor glun2b, the conservation of slims is: the guanine-nucleotide exchange factor brag1 (\u003cspan class=\"citationref\"\u003e76\u003c/span\u003e) \u0026ndash; zglun2ba (60% identity; 80% similarity) and zglun2bb (60% identity, 100% similarity); psd-95 (\u003cspan class=\"citationref\"\u003e72\u003c/span\u003e) \u0026ndash; zglun2ba (56% identity \u0026amp; similarity) and zglun2bb (33% identity, 56% similarity); camkii and death-associated protein kinase 1 (dapk1) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (92% identity, 100% similarity); synapse-associated protein 102 (sap102) (\u003cspan class=\"citationref\"\u003e75\u003c/span\u003e) \u0026ndash; zglun2ba (0% identity \u0026amp; similarity) and zglun2bb (50% identity \u0026amp; similarity); the adaptor complex ap2 (\u003cspan class=\"citationref\"\u003e74\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (100% identity \u0026amp; similarity); and pdz (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (100% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,d). the ctd interaction regions for glun2b are conserved as follows: receptor for activated c kinase 1 (rack1) (\u003cspan class=\"citationref\"\u003e77\u003c/span\u003e) \u0026ndash; zglun2ba (0% identity, 7% similarity) and zglun2bb (13% identity, 20% similarity); flot-1 (\u003cspan class=\"citationref\"\u003e71\u003c/span\u003e) \u0026ndash; zglun2ba (23% identity; 47% similarity) and zglun2bb (35% identity, 68% similarity); spectrin (\u003cspan class=\"citationref\"\u003e79\u003c/span\u003e) \u0026ndash; zglun2ba (40% identity; 59% similarity) and zglun2bb (35% identity, 62% similarity); \u0026alpha;-actinin 2 (\u003cspan class=\"citationref\"\u003e78\u003c/span\u003e) \u0026ndash; zglun2ba (47% identity; 67% similarity) and zglun2bb (51% identity, 76% similarity); and ras protein-specific guanine nucleotide-releasing factor 1 (rasgrf1) (\u003cspan class=\"citationref\"\u003e80\u003c/span\u003e) \u0026ndash; zglun2ba (59% identity; 74% similarity) and zglun2bb (60% identity, 78% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,d, s3a,c).\u003c/p\u003e\n\u003cp\u003ewhen compared globally across subunits, slims always show greater conservation than regions. this conservation is always greatest in the zebrafish glun1 paralogs, followed by the glun2b and then glun2a paralogs. for the analyzed slims, conservation grouped across each subunit is as follows: zglun1a and zglun1b (100% identity \u0026amp; similarity), zglun2aa (76% identity, 81% similarity), zglun2ab (62% identity, 76% similarity), zglun2ba (78% identity, 81% similarity), and zglun2bb (72% identity, 86% similarity). the regions show the following degrees of conservation: zglun1a and zglun1b (94% identity, 98% similarity), zglun2aa (55% identity, 81% similarity), zglun2ab (36% identity, 62% similarity), zglun2ba (59% identity, 74% similarity), and zglun2bb (60% identity, 78% similarity).\u003c/p\u003e"},{"header":"discussion","content":"\u003cp\u003ehere, we compared the conservation of the zebrafish nmdar subunit paralogues, including key structure-function motifs, to their human counterparts. we find that the core gating machinery \u0026ndash; the lbd, the lbd-tmd linkers and the tmd \u0026ndash; are highly conserved both in terms of overall identity and specific functional motifs. the more peripheral domains \u0026ndash; the ntd and ctd \u0026ndash; show less conservation overall, though functional motifs are reasonably conserved with the notable exception of zn\u003csup\u003e2+\u003c/sup\u003e binding in the glun2a ntd. notably, disease-associated variants are generally conserved, highlighting that zebrafish represent a useful model to study nmdars.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar conservation in zebrafish\u003c/b\u003e\u003c/p\u003e\u003cp\u003eacross most of our examined parameters, the obligatory glun1 subunit shows the highest degree of conservation in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e). generally, conservation is then greatest in the glun2(a-d) followed by the glun3(a-b) subunits. within each subunit, sequence conservation is consistently highest in the tmd and lbd-tmd linkers, followed by the lbd. conservation is generally poorer in the ntd and even worse in the ctd (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ea key question with regards to this work is defining what constitutes as a \u0026lsquo;well conserved\u0026rsquo; sequence. previous studies regarding the conservation of protein structure and function suggest that sequence identities between 40\u0026ndash;70% confer functional conservation (\u003cspan additionalcitationids=\"cr93\" citationid=\"cr92\" class=\"citationref\"\u003e92\u003c/span\u003e\u0026ndash;\u003cspan citationid=\"cr94\" class=\"citationref\"\u003e94\u003c/span\u003e). they regard 50% sequence identity as the threshold below which function drastically diverges (\u003cspan citationid=\"cr93\" class=\"citationref\"\u003e93\u003c/span\u003e, \u003cspan citationid=\"cr94\" class=\"citationref\"\u003e94\u003c/span\u003e). our sequence conservation analyses almost always led to sequence identities\u0026thinsp;\u0026gt;\u0026thinsp;50%, except for zglun2da. nevertheless, we carried out more detailed examinations of more poorly conserved domains, e.g., the ntd and ctd, to assess the conservation of key structure-function motifs.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar missense variant conservation in zebrafish\u003c/b\u003e\u003c/p\u003e\u003cp\u003eto date, a multitude of variants, including missense, frameshift, and nonsense mutations, have been identified in the genes encoding the nmdar subunits (\u003cspan citationid=\"cr21\" class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan citationid=\"cr22\" class=\"citationref\"\u003e22\u003c/span\u003e, \u003cspan citationid=\"cr87\" class=\"citationref\"\u003e87\u003c/span\u003e). we categorized identified missense variants by pathogenicity and analyzed the extent of conservation of the positions at which they occur. we focused only on the glun1, glun2a, glun2b, and glun2d subunits for this analysis, as this is where most disease-associated variants have been identified. generally, our results demonstrate a higher degree of variant position conservation with increasing pathogenicity (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e,\u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e). the sites of pathogenic missense variants are almost perfectly conserved (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e). given the high conservation of the amino acid positions at which they occur, these protein regions are likely most essential for receptor function. across our categories of variants, with the exclusion of complex variants, the proportion of the total protein at which missense variants have been identified is lowest in the pathogenic group (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e). it is possible that additional variants that might be categorized as pathogenic are so devastating to development as to be embryonic lethal. this reflects the notion that more integral regions of the receptor are less tolerant to variants (\u003cspan citationid=\"cr95\" class=\"citationref\"\u003e95\u003c/span\u003e). zebrafish, nevertheless, emerge as a useful model for the study of pathogenic nmdar variants.\u003c/p\u003e\u003cp\u003eputatively pathogenic variants are conserved to a lesser extent than their pathogenic counterparts (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e). according to our designations, these variants have the potential to cause disease, having been identified in patients, but are currently classified as having uncertain significance. thus, it aligns that these variants are comparatively less well conserved \u0026ndash; they are likely occurring in less critical regions of the receptor and are likewise less likely to cause disease phenotypes.\u003c/p\u003e\u003cp\u003eour subsequent investigation of putatively benign and benign variants revealed a further decrease in variant position conservation. this finding further supported our prediction that variants more highly associated with disease occur at positions with higher degrees of conservation and, thus, are more critical to protein structure and function. the conservation of positions with putatively benign and benign variants is quite comparable. for the benign variants, in particular, conservation percentages also closely reflect total protein conservation values for each paralog (fig.\u0026nbsp;\u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e, table\u0026nbsp;\u003cspan refid=\"tab1\" class=\"internalref\"\u003e1\u003c/span\u003e). this follows from the finding that positions with identified benign variants, for most subunits, make up a substantial portion of the protein (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enevertheless, this investigation both demonstrates the advantage of the zebrafish model in studying nmdar-associated disease and reflects the notion that the positions most essential for receptor function are those that are best conserved and more likely to induce disease when absent.\u003c/p\u003e\u003cp\u003eoverall, the trend for variant conservation is the same as that for the whole protein. conservation is always greatest in the tmd and lbd-tmd linkers, followed by the lbd, next the ntd, and last the ctd (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e, \u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e). these findings suggest that zebrafish serve as an effective model for variants, especially pathogenic and putatively pathogenic variants in both the tmd and lbd-tmd linkers. some caution, however, must be exercised when using this model system for variants in the lbd, ntd, and ctd.\u003c/p\u003e\u003cp\u003e\u003cb\u003etmd, tmd-lbd linker, and lbd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003eour examination of key functional regions and motifs in the tmd, lbd-tmd linkers, and lbd consistently revealed complete conservation in their respective zebrafish paralogs (fig. \u003cspan refid=\"moesm1\" class=\"internalref\"\u003es1\u003c/span\u003e,s2). this suggests that the basic mechanism of ion channel gating is conserved in zebrafish relative to human. as these domains and subdomains contribute to the formation of the core of the receptor \u0026ndash; the ion channel pore allowing for ion flux \u0026ndash; it is expected that they would be most highly conserved across species, since the major functional role of nmdars is current flux during synaptic transmission.\u003c/p\u003e\u003cp\u003e\u003cb\u003entd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003ethe ntd is the first region in which nmdar sequences in zebrafish begin to diverge from those in human. modelling ntd variants in zebrafish must, thus, be approached with caution. for example, polyamine-binding sites in glun2b are almost completely conserved in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig8\" class=\"internalref\"\u003e4\u003c/span\u003ed), so they have the potential to serve as an effective model for this functional role of nmdars. on the other hand, zinc-binding sites in glun2a and glun2b are more poorly conserved in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig8\" class=\"internalref\"\u003e4\u003c/span\u003ec), so they would likely serve as a poor model for zinc-related studies in nmdars. this also aligns with our homology model finding that the zglun2aa model aligns well with glun2a when not bound to zinc, but the rmsd of the alignment is substantially poorer when glun2a interacting with zinc is used as the reference structure (table s3).\u003c/p\u003e\u003cp\u003e\u003cb\u003ectd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003ethe c-terminal domains (ctds) exhibit the lowest level of overall conservation across nmdar domains (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e). this is consistent with the ctd\u0026rsquo;s classification as an intrinsically disordered region (idr), which lacks fixed secondary or tertiary structure and is more tolerant to evolutionary divergence (\u003cspan citationid=\"cr96\" class=\"citationref\"\u003e96\u003c/span\u003e, \u003cspan citationid=\"cr97\" class=\"citationref\"\u003e97\u003c/span\u003e). nonetheless, functional conservation within idrs is often maintained through the retention of post-translational modification (ptm) hotspots and short linear motifs (slims) that regulate protein interactions and intracellular signaling (\u003cspan citationid=\"cr98\" class=\"citationref\"\u003e98\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ewe assessed ctd conservation in glun1, glun2a, and glun2b by analyzing experimentally validated phosphorylation, palmitoylation, and ubiquitylation sites, as well as defined protein-binding motifs. phosphorylation sites with known kinase interactions exhibited relatively high conservation, consistent with their well-characterized regulatory roles (figs.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ea,b). in contrast, phosphorylation sites without identified kinases were less consistently retained (fig.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ec). palmitoylation sites were fully conserved across all subunits, while ubiquitylation sites were generally retained, though conservation was notably lower in zglun2ab (fig.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ef) and may reflect paralogue-specific divergence in degradation or trafficking pathways.\u003c/p\u003e\u003cp\u003eprotein-binding motif conservation varied by subunit. glun1 retained nearly all canonical interaction motifs, while glun2a and glun2b displayed region-specific conservation. notably well-conserved elements in the glun2 subunits included the pdz-binding motifs as well as docking regions for camkii, dapk1, and ap2 (fig.\u0026nbsp;\u003cspan refid=\"fig12\" class=\"internalref\"\u003e6\u003c/span\u003e). slims overall showed higher conservation, likely due to their short, sequence-specific structure, which may place them under stronger evolutionary constraint than broader, more flexible interaction regions.\u003c/p\u003e\u003cp\u003edespite broad sequence divergence, nmdar ctds retain a core set of conserved motifs involved in receptor trafficking, synaptic localization, and plasticity. kinase-regulated phosphorylation sites, palmitoylation sites, and pdz-binding motifs were consistently preserved. conservation of glun1 sites was always greatest in the zebrafish paralogs. among the glun2 subunits, glun2b displayed the highest overall conservation of functional elements, aligning with its essential role in neurodevelopment and synaptic signaling.\u003c/p\u003e\u003cp\u003ethese findings are especially relevant to the study of disease-associated truncating variants in the nmdar subunits. nonsense variants that introduce premature stop codons in the ctd eliminate key regulatory motifs critical for trafficking, anchoring, or signaling. when these affected regions correspond to conserved functional elements, zebrafish may serve as an appropriate \u003cem\u003ein vitro\u003c/em\u003e and \u003cem\u003ein vivo\u003c/em\u003e model for studying ctd-mediated disease mechanisms. conversely, for mutations in poorly conserved regions, the translational utility of this model may become limited.\u003c/p\u003e"},{"header":"conclusions","content":"\u003cp\u003ezebrafish represent a valuable system to study nmdar-associated neurodevelopmental disorders, but one must be judicious in choosing which variants to study in this model. our unique approach of sequence alignments per individual nmdar domains and subdomains highlights the high degree of conversation of some regions of the protein relative to others. additionally, our analysis of identified variants indicates that zebrafish are an effective model for pathogenic variants as these occur at conserved locations along the protein. generally, the utility of zebrafish is most apparent for the study of the tmd, lbd-tmd linkers, and lbd as these regions are most well conserved. on the other hand, studying the ntd and ctd in this species requires extra consideration stemming from deviations in sequence conservation. our work will enable future nmdar-related studies that can be effectively and efficiently conducted in zebrafish.\u003c/p\u003e"},{"header":"abbreviations","content":"\u003ctable border=\"0\" cellspacing=\"0\" cellpadding=\"0\"\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003enmdar\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003e\u003cem\u003en\u003c/em\u003e-methyl-d-aspartate receptor\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003entd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003en-terminal domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003elbd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003eligand-binding domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003etmd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003etransmembrane domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003ectd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003ec-terminal domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e"},{"header":"declarations","content":"\u003cp\u003e\u003cstrong\u003eethics approval and consent to participate:\u0026nbsp;\u003c/strong\u003enot applicable.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econsent for publication:\u0026nbsp;\u003c/strong\u003enot applicable.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eavailability of data and materials:\u003c/strong\u003e the datasets used and/or analyzed during the current study are available either at osf (https://osf.io/h2zf9/) or from the corresponding author on reasonable request.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecompeting interests:\u0026nbsp;\u003c/strong\u003eno competing interests declared.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003efunding:\u003c/strong\u003e this work was supported by nih grant r01ns088479 to lpw.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eauthors\u0026rsquo; contributions:\u0026nbsp;\u003c/strong\u003ee.r.n., c.a., j.d.z., h.i.s., and l.p.w. contributed to the conceptualization and methodology of this project. e.r.n., c.a., and j.d.z. conducted the investigation and analysis. w.r. assisted with data curation. e.r.n, c.a., and l.p.w. wrote the manuscript, which was revised with insights from j.d.z. and h.i.s.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eacknowledgements:\u0026nbsp;\u003c/strong\u003ethe authors thank the clinvar and gnomad databases and the swiss-model server for their important contributions to this work.\u003c/p\u003e"},{"header":"references","content":"\u003col\u003e\n\u003cli\u003epaoletti p, bellone c, zhou q. nmda receptor subunit diversity: impact on receptor properties, synaptic plasticity and disease. nat rev neurosci. 2013;14(6):383-400.\u003c/li\u003e\n\u003cli\u003ehansen kb, wollmuth lp, bowie d, furukawa h, menniti fs, sobolevsky ai, et al. structure, function, and pharmacology of glutamate receptor ion channels. pharmacol rev. 2021;73(4):298-487.\u003c/li\u003e\n\u003cli\u003echakraborty a, murphy s, coleman n. the role of nmda receptors in neural stem cell proliferation and differentiation. stem cells dev. 2017;26(11):798-807.\u003c/li\u003e\n\u003cli\u003enacher j, mcewen bs. the role of n-methyl-d-asparate receptors in neurogenesis. hippocampus. 2006;16(3):267-70.\u003c/li\u003e\n\u003cli\u003ekomuro h, rakic p. modulation of neuronal migration by nmda receptors. science. 1993;260(5104):95-7.\u003c/li\u003e\n\u003cli\u003epearce ia, cambray-deakin ma, burgoyne rd. glutamate acting on nmda receptors stimulates neurite outgrowth from cerebellar granule cells. febs lett. 1987;223(1):143-7.\u003c/li\u003e\n\u003cli\u003ebooker sa, wyllie dja. nmda receptor function in inhibitory neurons. neuropharmacology. 2021;196:108609.\u003c/li\u003e\n\u003cli\u003egambrill ac, barria a. nmda receptor subunit composition controls synaptogenesis and synapse stabilization. proc natl acad sci u s a. 2011;108(14):5855-60.\u003c/li\u003e\n\u003cli\u003enapoli aj, laderwager s, zoodsma jd, biju b, mucollari o, schubel sk, et al. developmental loss of nmda receptors results in supernumerary forebrain neurons through delayed maturation of transit-amplifying neuroblasts. sci rep. 2024;14(1):3395.\u003c/li\u003e\n\u003cli\u003erajani v, sengar as, salter mw. tripartite signalling by nmda receptors. mol brain. 2020;13(1):23.\u003c/li\u003e\n\u003cli\u003eishchenko y, carrizales mg, koleske aj. regulation of the nmda receptor by its cytoplasmic domains: (how) is the tail wagging the dog? neuropharmacology. 2021;195:108634.\u003c/li\u003e\n\u003cli\u003emony l, paoletti p. mechanisms of nmda receptor regulation. curr opin neurobiol. 2023;83:102815.\u003c/li\u003e\n\u003cli\u003enielsen ac, anderson cl, ens c, boyce akj, thompson rj. non-ionotropic nmdar signalling activates panx1 to induce p2x4r-dependent long-term depression in the hippocampus. j physiol. 2024.\u003c/li\u003e\n\u003cli\u003ebarnes sa, thomazeau a, finnie psb, heinrich mj, heynen aj, komiyama nh, et al. non-ionotropic signaling through the nmda receptor glun2b carboxy-terminal domain drives dendritic spine plasticity and reverses fragile x phenotypes. cell rep. 2025;44(3):115311.\u003c/li\u003e\n\u003cli\u003eplatzer k, lemke jr. grin1-related neurodevelopmental disorder. genereviews((r)). 2021.\u003c/li\u003e\n\u003cli\u003estrehlow v, myers ka, morgan at, scheffer ie, lemke jr. grin2a-related disorders. genereviews((r)). 2024.\u003c/li\u003e\n\u003cli\u003eplatzer k, lemke jr. grin2b-related neurodevelopmental disorder. genereviews((r)). 2021.\u003c/li\u003e\n\u003cli\u003eplatzer k, krey i, lemke jr. grin2d-related developmental and epileptic encephalopathy. genereviews((r)). 2022.\u003c/li\u003e\n\u003cli\u003ezehavi y, mandel h, zehavi a, rashid ma, straussberg r, jabur b, et al. de novo grin1 mutations: an emerging cause of severe early infantile encephalopathy. eur j med genet. 2017;60(6):317-20.\u003c/li\u003e\n\u003cli\u003esabo sl, lahr jm, offer m, weekes a, sceniak mp. grin2b-related neurodevelopmental disorder: current understanding of pathophysiological mechanisms. front synaptic neurosci. 2022;14:1090865.\u003c/li\u003e\n\u003cli\u003egjerulfsen ce, krey i, klockner c, rubboli g, lemke jr, moller rs. spectrum of nmda receptor variants in neurodevelopmental disorders and epilepsy. methods mol biol. 2024;2799:1-11.\u003c/li\u003e\n\u003cli\u003eamin jb, moody gr, wollmuth lp. from bedside-to-bench: what disease-associated variants are teaching us about the nmda receptor. j physiol. 2021;599(2):397-416.\u003c/li\u003e\n\u003cli\u003ehu c, chen w, myers sj, yuan h, traynelis sf. human grin2b variants in neurodevelopmental disorders. j pharmacol sci. 2016;132(2):115-21.\u003c/li\u003e\n\u003cli\u003ecox ja, kucenas s, voigt mm. molecular characterization and embryonic expression of the family of n-methyl-d-aspartate receptor subunit genes in the zebrafish. dev dyn. 2005;234(3):756-66.\u003c/li\u003e\n\u003cli\u003eveldman mb, lin s. zebrafish as a developmental model organism for pediatric research. pediatr res. 2008;64(5):470-6.\u003c/li\u003e\n\u003cli\u003echoi ty, choi ti, lee yr, choe sk, kim ch. zebrafish as an animal model for biomedical research. exp mol med. 2021;53(3):310-7.\u003c/li\u003e\n\u003cli\u003ezoodsma jd, chan k, bhandiwad aa, golann dr, liu g, syed sa, et al. a model to study nmda receptors in early nervous system development. j neurosci. 2020;40(18):3631-45.\u003c/li\u003e\n\u003cli\u003ezoodsma jd, keegan ej, moody gr, bhandiwad aa, napoli aj, burgess ha, et al. disruption of grin2b, an asd-associated gene, produces social deficits in zebrafish. mol autism. 2022;13(1):38.\u003c/li\u003e\n\u003cli\u003ebaraban sc, taylor mr, castro pa, baier h. pentylenetetrazole induced changes in zebrafish behavior, neural activity and c-fos expression. neuroscience. 2005;131(3):759-68.\u003c/li\u003e\n\u003cli\u003eshams s, rihel j, ortiz jg, gerlai r. the zebrafish as a promising tool for modeling human brain disorders: a review based upon an ibns symposium. neurosci biobehav rev. 2018;85:176-90.\u003c/li\u003e\n\u003cli\u003ede abreu ms, genario r, giacomini a, demin ka, lakstygal am, amstislavskaya tg, et al. zebrafish as a model of neurodevelopmental disorders. neuroscience. 2020;445:3-11.\u003c/li\u003e\n\u003cli\u003eweinschutz mendes h, neelakantan u, liu y, fitzpatrick se, chen t, wu w, et al. high-throughput functional analysis of autism genes in zebrafish identifies convergence in dopaminergic and neuroimmune pathways. cell rep. 2023;42(3):112243.\u003c/li\u003e\n\u003cli\u003ehowe k, clark md, torroja cf, torrance j, berthelot c, muffato m, et al. the zebrafish reference genome sequence and its relationship to the human genome. nature. 2013;496(7446):498-503.\u003c/li\u003e\n\u003cli\u003erihel j, schier af. behavioral screening for neuroactive drugs in zebrafish. dev neurobiol. 2012;72(3):373-85.\u003c/li\u003e\n\u003cli\u003ebaraban sc, dinday mt, hortopan ga. drug screening in scn1a zebrafish mutant identifies clemizole as a potential dravet syndrome treatment. nat commun. 2013;4:2410.\u003c/li\u003e\n\u003cli\u003ezhang y, kecskes a, copmans d, langlois m, crawford ad, ceulemans b, et al. pharmacological characterization of an antisense knockdown zebrafish model of dravet syndrome: inhibition of epileptic seizures by the serotonin agonist fenfluramine. plos one. 2015;10(5):e0125898.\u003c/li\u003e\n\u003cli\u003edingemans ajm, jansen s, van reeuwijk j, de leeuw n, pfundt r, schuurs-hoeijmakers j, et al. prevalence of comorbidities in individuals with neurodevelopmental disorders from the aggregated phenomics data of 51,227 pediatric individuals. nat med. 2024;30(7):1994-2003.\u003c/li\u003e\n\u003cli\u003eohtsuki t, sakurai k, dou h, toru m, yamakawa-kobayashi k, arinami t. mutation analysis of the nmdar2b (grin2b) gene in schizophrenia. mol psychiatry. 2001;6(2):211-6.\u003c/li\u003e\n\u003cli\u003earnold pd, rosenberg dr, mundo e, tharmalingam s, kennedy jl, richter ma. association of a glutamate (nmda) subunit receptor gene (grin2b) with obsessive-compulsive disorder: a preliminary study. psychopharmacology (berl). 2004;174(4):530-8.\u003c/li\u003e\n\u003cli\u003esaunders ja, tatard-leitman vm, suh j, billingslea en, roberts tp, siegel sj. knockout of nmda receptors in parvalbumin interneurons recreates autism-like phenotypes. autism res. 2013;6(2):69-77.\u003c/li\u003e\n\u003cli\u003egandal mj, sisti j, klook k, ortinski pi, leitman v, liang y, et al. gabab-mediated rescue of altered excitatory-inhibitory balance, gamma synchrony and behavioral deficits following constitutive nmdar-hypofunction. transl psychiatry. 2012;2(7):e142.\u003c/li\u003e\n\u003cli\u003ebarnby g, abbott a, sykes n, morris a, weeks de, mott r, et al. candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at grin2a and abat. am j hum genet. 2005;76(6):950-66.\u003c/li\u003e\n\u003cli\u003ehung ly, margolis kg. autism spectrum disorders and the gastrointestinal tract: insights into mechanisms and clinical relevance. nat rev gastroenterol hepatol. 2024;21(3):142-63.\u003c/li\u003e\n\u003cli\u003eowens ap, mathias cj, iodice v. autonomic dysfunction in autism spectrum disorder. front integr neurosci. 2021;15:787037.\u003c/li\u003e\n\u003cli\u003eorefice ll. peripheral somatosensory neuron dysfunction: emerging roles in autism spectrum disorders. neuroscience. 2020;445:120-9.\u003c/li\u003e\n\u003cli\u003emakris g, agorastos a, chrousos gp, pervanidou p. stress system activation in children and adolescents with autism spectrum disorder. front neurosci. 2021;15:756628.\u003c/li\u003e\n\u003cli\u003eramos-vicente d, grant sg, bayes a. metazoan evolution and diversity of glutamate receptors and their auxiliary subunits. neuropharmacology. 2021;195:108640.\u003c/li\u003e\n\u003cli\u003eamores a, force a, yan yl, joly l, amemiya c, fritz a, et al. zebrafish hox clusters and vertebrate genome evolution. science. 1998;282(5394):1711-4.\u003c/li\u003e\n\u003cli\u003echou th, tajima n, romero-hernandez a, furukawa h. structural basis of functional transitions in mammalian nmda receptors. cell. 2020;182(2):357-71 e13.\u003c/li\u003e\n\u003cli\u003ezhang y, ye f, zhang t, lv s, zhou l, du d, et al. structural basis of ketamine action on human nmda receptors. nature. 2021;596(7871):301-5.\u003c/li\u003e\n\u003cli\u003ewollmuth lp, sakmann b. different mechanisms of ca2+ transport in nmda and ca2+-permeable ampa glutamate receptor channels. j gen physiol. 1998;112(5):623-36.\u003c/li\u003e\n\u003cli\u003ewatanabe j, beck c, kuner t, premkumar ls, wollmuth lp. drpeer: a motif in the extracellular vestibule conferring high ca2+ flux rates in nmda receptor channels. j neurosci. 2002;22(23):10209-16.\u003c/li\u003e\n\u003cli\u003ewollmuth lp, sobolevsky ai. structure and gating of the glutamate receptor ion channel. trends neurosci. 2004;27(6):321-8.\u003c/li\u003e\n\u003cli\u003eogden kk, khatri a, traynelis sf, heldt sa. potentiation of glun2c/d nmda receptor subtypes in the amygdala facilitates the retention of fear and extinction learning in mice. neuropsychopharmacology. 2014;39(3):625-37.\u003c/li\u003e\n\u003cli\u003ewollmuth lp. ion permeation in ionotropic glutamate receptors: still dynamic after all these years. curr opin physiol. 2018;2:36-41.\u003c/li\u003e\n\u003cli\u003emcdaniel mj, ogden kk, kell sa, burger pb, liotta dc, traynelis sf. nmda receptor channel gating control by the pre-m1 helix. j gen physiol. 2020;152(4).\u003c/li\u003e\n\u003cli\u003ewollmuth lp, kuner t, sakmann b. adjacent asparagines in the nr2-subunit of the nmda receptor channel control the voltage-dependent block by extracellular mg2+. j physiol. 1998;506 ( pt 1)(pt 1):13-32.\u003c/li\u003e\n\u003cli\u003eromero-hernandez a, simorowski n, karakas e, furukawa h. molecular basis for subtype specificity and high-affinity zinc inhibition in the glun1-glun2a nmda receptor amino-terminal domain. neuron. 2016;92(6):1324-36.\u003c/li\u003e\n\u003cli\u003emony l, zhu s, carvalho s, paoletti p. molecular basis of positive allosteric modulation of glun2b nmda receptors by polyamines. embo j. 2011;30(15):3134-46.\u003c/li\u003e\n\u003cli\u003elichnerova k, kaniakova m, park sp, skrenkova k, wang yx, petralia rs, et al. two n-glycosylation sites in the glun1 subunit are essential for releasing n-methyl-d-aspartate (nmda) receptors from the endoplasmic reticulum. j biol chem. 2015;290(30):18379-90.\u003c/li\u003e\n\u003cli\u003ewaterhouse a, bertoni m, bienert s, studer g, tauriello g, gumienny r, et al. swiss-model: homology modelling of protein structures and complexes. nucleic acids res. 2018;46(w1):w296-w303.\u003c/li\u003e\n\u003cli\u003ebienert s, waterhouse a, de beer ta, tauriello g, studer g, bordoli l, et al. the swiss-model repository-new features and functionality. nucleic acids res. 2017;45(d1):d313-d9.\u003c/li\u003e\n\u003cli\u003ewarnet xl, bakke krog h, sevillano-quispe og, poulsen h, kjaergaard m. the c-terminal domains of the nmda receptor: how intrinsically disordered tails affect signalling, plasticity and disease. eur j neurosci. 2021;54(8):6713-39.\u003c/li\u003e\n\u003cli\u003ekim k, saneyoshi t, hosokawa t, okamoto k, hayashi y. interplay of enzymatic and structural functions of camkii in long-term potentiation. j neurochem. 2016;139(6):959-72.\u003c/li\u003e\n\u003cli\u003ebej a, ames jb. chemical shift assignments of calmodulin bound to a cytosolic domain of glun2a (residues 1004-1024) from the nmda receptor. biomol nmr assign. 2023;17(1):89-93.\u003c/li\u003e\n\u003cli\u003ehorak m, wenthold rj. different roles of c-terminal cassettes in the trafficking of full-length nr1 subunits to the cell surface. j biol chem. 2009;284(15):9683-91.\u003c/li\u003e\n\u003cli\u003elin jw, wyszynski m, madhavan r, sealock r, kim ju, sheng m. yotiao, a novel protein of neuromuscular junction and brain that interacts with specific splice variants of nmda receptor subunit nr1. j neurosci. 1998;18(6):2017-27.\u003c/li\u003e\n\u003cli\u003estanic j, carta m, eberini i, pelucchi s, marcello e, genazzani aa, et al. rabphilin 3a retains nmda receptors at synaptic sites through interaction with glun2a/psd-95 complex. nat commun. 2015;6:10181.\u003c/li\u003e\n\u003cli\u003ecousins sl, stephenson fa. identification of c-terminal binding protein 1 as a novel nmda receptor interactor. neurochem res. 2019;44(6):1437-45.\u003c/li\u003e\n\u003cli\u003efranchini l, carrano n, di luca m, gardoni f. synaptic glun2a-containing nmda receptors: from physiology to pathological synaptic plasticity. int j mol sci. 2020;21(4).\u003c/li\u003e\n\u003cli\u003eswanwick cc, shapiro me, yi z, chang k, wenthold rj. nmda receptors interact with flotillin-1 and -2, lipid raft-associated proteins. febs lett. 2009;583(8):1226-30.\u003c/li\u003e\n\u003cli\u003ecousins sl, stephenson fa. identification of n-methyl-d-aspartic acid (nmda) receptor subtype-specific binding sites that mediate direct interactions with scaffold protein psd-95. j biol chem. 2012;287(16):13465-76.\u003c/li\u003e\n\u003cli\u003evissel b, krupp jj, heinemann sf, westbrook gl. a use-dependent tyrosine dephosphorylation of nmda receptors is independent of ion flux. nat neurosci. 2001;4(6):587-96.\u003c/li\u003e\n\u003cli\u003elavezzari g, mccallum j, lee r, roche kw. differential binding of the ap-2 adaptor complex and psd-95 to the c-terminus of the nmda receptor subunit nr2b regulates surface expression. neuropharmacology. 2003;45(6):729-37.\u003c/li\u003e\n\u003cli\u003echen bs, gray ja, sanz-clemente a, wei z, thomas ev, nicoll ra, et al. sap102 mediates synaptic clearance of nmda receptors. cell rep. 2012;2(5):1120-8.\u003c/li\u003e\n\u003cli\u003eelagabani mn, brisevac d, kintscher m, pohle j, kohr g, schmitz d, et al. subunit-selective n-methyl-d-aspartate (nmda) receptor signaling through brefeldin a-resistant arf guanine nucleotide exchange factors brag1 and brag2 during synapse maturation. j biol chem. 2016;291(17):9105-18.\u003c/li\u003e\n\u003cli\u003eyaka r, thornton c, vagts aj, phamluong k, bonci a, ron d. nmda receptor function is regulated by the inhibitory scaffolding protein, rack1. proc natl acad sci u s a. 2002;99(8):5710-5.\u003c/li\u003e\n\u003cli\u003ewyszynski m, lin j, rao a, nigh e, beggs ah, craig am, et al. competitive binding of alpha-actinin and calmodulin to the nmda receptor. nature. 1997;385(6615):439-42.\u003c/li\u003e\n\u003cli\u003ewechsler a, teichberg vi. brain spectrin binding to the nmda receptor is regulated by phosphorylation, calcium and calmodulin. embo j. 1998;17(14):3931-9.\u003c/li\u003e\n\u003cli\u003ekrapivinsky g, krapivinsky l, manasian y, ivanov a, tyzio r, pellegrino c, et al. the nmda receptor is coupled to the erk pathway by a direct interaction between nr2b and rasgrf1. neuron. 2003;40(4):775-84.\u003c/li\u003e\n\u003cli\u003emukherjee p, roy s, ghosh d, nandi sk. role of animal models in biomedical research: a review. lab anim res. 2022;38(1):18.\u003c/li\u003e\n\u003cli\u003echou th, epstein m, fritzemeier rg, akins ns, paladugu s, ullman ez, et al. molecular mechanism of ligand gating and opening of nmda receptor. nature. 2024;632(8023):209-17.\u003c/li\u003e\n\u003cli\u003earmstrong n, sun y, chen gq, gouaux e. structure of a glutamate-receptor ligand-binding core in complex with kainate. nature. 1998;395(6705):913-7.\u003c/li\u003e\n\u003cli\u003earmstrong n, gouaux e. mechanisms for activation and antagonism of an ampa-sensitive glutamate receptor: crystal structures of the glur2 ligand binding core. neuron. 2000;28(1):165-81.\u003c/li\u003e\n\u003cli\u003ekazi r, gan q, talukder i, markowitz m, salussolia cl, wollmuth lp. asynchronous movements prior to pore opening in nmda receptors. j neurosci. 2013;33(29):12052-66.\u003c/li\u003e\n\u003cli\u003eamin jb, gochman a, he m, certain n, wollmuth lp. nmda receptors require multiple pre-opening gating steps for efficient synaptic activity. neuron. 2021;109(3):488-501 e4.\u003c/li\u003e\n\u003cli\u003ehardingham ge, do kq. linking early-life nmdar hypofunction and oxidative stress in schizophrenia pathogenesis. nat rev neurosci. 2016;17(2):125-34.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, riley gr, jang w, rubinstein ws, church dm, et al. clinvar: public archive of relationships among sequence variation and human phenotype. nucleic acids res. 2014;42(database issue):d980-5.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, benson m, brown g, chao c, chitipiralla s, et al. clinvar: public archive of interpretations of clinically relevant variants. nucleic acids res. 2016;44(d1):d862-8.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, benson m, brown gr, chao c, chitipiralla s, et al. clinvar: improving access to variant interpretations and supporting evidence. nucleic acids res. 2018;46(d1):d1062-d7.\u003c/li\u003e\n\u003cli\u003egudmundsson s, singer-berk m, watts na, phu w, goodrich jk, solomonson m, et al. variant interpretation using population databases: lessons from gnomad. hum mutat. 2022;43(8):1012-30.\u003c/li\u003e\n\u003cli\u003etian w, skolnick j. how well is enzyme function conserved as a function of pairwise sequence identity? j mol biol. 2003;333(4):863-82.\u003c/li\u003e\n\u003cli\u003eaddou s, rentzsch r, lee d, orengo ca. domain-based and family-specific sequence identity thresholds increase the levels of reliable protein function transfer. j mol biol. 2009;387(2):416-30.\u003c/li\u003e\n\u003cli\u003esangar v, blankenberg dj, altman n, lesk am. quantitative sequence-function relationships in proteins based on gene ontology. bmc bioinformatics. 2007;8:294.\u003c/li\u003e\n\u003cli\u003eperszyk re, kristensen as, lyuboslavsky p, traynelis sf. three-dimensional missense tolerance ratio analysis. genome res. 2021;31(8):1447-61.\u003c/li\u003e\n\u003cli\u003euversky vn. intrinsically disordered proteins from a to z. int j biochem cell biol. 2011;43(8):1090-103.\u003c/li\u003e\n\u003cli\u003esingleton md, eisen mb. evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. plos comput biol. 2024;20(4):e1012028.\u003c/li\u003e\n\u003cli\u003edavey ne, cowan jl, shields dc, gibson tj, coldwell mj, edwards rj. slimprints: conservation-based discovery of functional motif fingerprints in intrinsically disordered protein regions. nucleic acids res. 2012;40(21):10628-41.\u003c/li\u003e\n\u003c/ol\u003e"}],"fulltextsource":"","fulltext":"","funders":[],"hasadminpriorityonworkflow":false,"hasmanuscriptdocx":true,"hasoptedintopreprint":true,"haspassedjournalqc":"","hasanypriority":false,"hidejournal":false,"highlight":"","institution":"","isacceptedbyjournal":false,"isauthorsuppliedpdf":false,"isdeskrejected":"","ishiddenfromsearch":false,"isinqc":false,"isinworkflow":false,"ispdf":false,"ispdfuptodate":true,"iswithdrawnorretracted":false,"journal":{"display":true,"email":"info@researchsquare.com","identity":"bmc-genomics","isnaturejournal":false,"hasqc":true,"allowdirectsubmit":false,"externalidentity":"gics","sidebox":"learn more about [bmc genomics](http://bmcgenomics.biomedcentral.com/)","snpcode":"","submissionurl":"https://www.editorialmanager.com/gics","title":"bmc genomics","twitterhandle":"#bmcgenomics","acdcenabled":true,"dfaenabled":false,"editorialsystem":"em","reportingportfolio":"bmc series","inreviewenabled":true,"inreviewrevisionsenabled":true},"keywords":"grin genes, clinvar, gnomad, sequence alignments, disease-associated variants, n-terminal domain, ligand-binding domain, transmembrane domain, c-terminal domain, ion channel","lastpublisheddoi":"10.21203/rs.3.rs-7151578/v1","lastpublisheddoiurl":"https://doi.org/10.21203/rs.3.rs-7151578/v1","license":{"name":"cc by 4.0","url":"https://creativecommons.org/licenses/by/4.0/"},"manuscriptabstract":"\u003ch2\u003ebackground\u003c/h2\u003e\u003cp\u003enmda receptors (nmdars) are widely expressed, glutamate-gated ion channels that play key roles in brain development and function. variants have been identified in the \u003cem\u003egrin\u003c/em\u003e genes encoding nmdar subunits that are linked to neurodevelopmental disorders, among other manifestations. zebrafish are a powerful model to study brain development and function given their rapid development and ease of genetic manipulation. as a result of an ancient genome duplication, zebrafish possess two paralogues for most human nmdar subunits. to evaluate the degree of conservation between human nmdar subunits and their respective zebrafish paralogues, we carried out detailed \u003cem\u003ein silico\u003c/em\u003e analyses, with an emphasis on key functional elements. to further assess the suitability of zebrafish for modeling nmdar-associated neurodevelopmental disorders, we analyzed the conservation of positions with identified missense variants.\u003c/p\u003e\u003ch2\u003eresults\u003c/h2\u003e\u003cp\u003ewe find that the human nmdar subunits are generally well conserved across zebrafish paralogs. moreover, variants classified as pathogenic and putatively pathogenic are highly conserved, reflecting the importance of key protein regions to neurotypical receptor function. positions with putatively benign and benign variants are less conserved. across nmdar domains, the transmembrane domain is most highly conserved, followed by the ligand-binding domain, which maintains conservation of amino acids that participate in the binding of ligands. the n-terminal domain is less well conserved but aligned homology models show high degrees of structural similarity. the c-terminal domain is the most poorly conserved region across zebrafish paralogs, but certain key regions that undergo phosphorylation, palmitoylation, and ubiquitylation as well as protein-binding motifs are better conserved.\u003c/p\u003e\u003ch2\u003econclusions\u003c/h2\u003e\u003cp\u003eour findings highlight a strong conservation of human nmdar subunits in zebrafish, with some exceptions. the ligand-binding domain, the transmembrane domain forming the ion channel and the short polypeptide linkers that connect them are highly conserved. the n- and c-terminal domains are less conserved but functional motifs in general, except for the zn\u003csup\u003e2+\u003c/sup\u003e binding site in glun2a paralogues, are more highly conserved relative to the entire domain. overall, our findings support the utility of zebrafish as a model for studying neurodevelopment and disease mechanisms and provide a template for rigorously considering the relationship between human and zebrafish paralogues.\u003c/p\u003e","manuscripttitle":"conservation of human nmda receptor subunits and disease variants in zebrafish","msid":"","msnumber":"","nondraftversions":[{"code":1,"date":"2025-07-30 18:25:39","doi":"10.21203/rs.3.rs-7151578/v1","editorialevents":[{"type":"communitycomments","content":0},{"type":"decision","content":"revision requested","date":"2025-09-24t19:23:39+00:00","index":"","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-09-19t21:34:21+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"269262391104669611722251259561774961128","date":"2025-09-19t14:02:55+00:00","index":"hide","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-09-09t17:52:57+00:00","index":"hide","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-08-18t16:11:58+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"201191728820772676273599414131783545114","date":"2025-08-04t10:39:46+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"230034227985411933590242857371107944843","date":"2025-07-28t13:05:21+00:00","index":"hide","fulltext":""},{"type":"reviewersinvited","content":"","date":"2025-07-28t07:38:37+00:00","index":"","fulltext":""},{"type":"editorassigned","content":"","date":"2025-07-28t07:10:47+00:00","index":"","fulltext":""},{"type":"editorinvited","content":"","date":"2025-07-23t18:56:26+00:00","index":"","fulltext":""},{"type":"checkscomplete","content":"","date":"2025-07-22t16:36:43+00:00","index":"","fulltext":""},{"type":"submitted","content":"bmc genomics","date":"2025-07-22t16:32:19+00:00","index":"","fulltext":""}],"status":"published","journal":{"display":true,"email":"info@researchsquare.com","identity":"bmc-genomics","isnaturejournal":false,"hasqc":true,"allowdirectsubmit":false,"externalidentity":"gics","sidebox":"learn more about [bmc genomics](http://bmcgenomics.biomedcentral.com/)","snpcode":"","submissionurl":"https://www.editorialmanager.com/gics","title":"bmc genomics","twitterhandle":"#bmcgenomics","acdcenabled":true,"dfaenabled":false,"editorialsystem":"em","reportingportfolio":"bmc series","inreviewenabled":true,"inreviewrevisionsenabled":true}}],"origin":"","owneridentity":"2b85bbad-8f38-4c35-89b3-5949e6b6d3c0","owner":[],"posteddate":"july 30th, 2025","published":true,"recenteditorialevents":[],"rejectedjournal":[],"revision":"","amendment":"","status":"under-review","subjectareas":[],"tags":[],"updatedat":"2025-10-14t16:08:28+00:00","versionofrecord":[],"versioncreatedat":"2025-07-30 18:25:39","video":"","vordoi":"","vordoiurl":"","workflowstages":[]},"version":"v1","identity":"rs-7151578","journalconfig":"researchsquare"},"__n_ssp":true},"page":"/article/[identity]/[[...version]]","query":{"identity":"rs-7151578","version":["v1"]},"buildid":"idgtm6tzxtmdq-mqt1zbo","isfallback":false,"dynamicids":[87228],"gssp":true,"scriptloader":[]}

## Discussion
here, we compared the conservation of the zebrafish nmdar subunit paralogues, including key structure-function motifs, to their human counterparts. we find that the core gating machinery – the lbd, the lbd-tmd linkers and the tmd – are highly conserved both in terms of overall identity and specific functional motifs. the more peripheral domains – the ntd and ctd – show less conservation overall, though functional motifs are reasonably conserved with the notable exception of zn 2+ binding in the glun2a ntd. notably, disease-associated variants are generally conserved, highlighting that zebrafish represent a useful model to study nmdars. nmdar conservation in zebrafish across most of our examined parameters, the obligatory glun1 subunit shows the highest degree of conservation in zebrafish (fig. 1 ). generally, conservation is then greatest in the glun2(a-d) followed by the glun3(a-b) subunits. within each subunit, sequence conservation is consistently highest in the tmd and lbd-tmd linkers, followed by the lbd. conservation is generally poorer in the ntd and even worse in the ctd (fig. 1 ). a key question with regards to this work is defining what constitutes as a ‘well conserved’ sequence. previous studies regarding the conservation of protein structure and function suggest that sequence identities between 40–70% confer functional conservation ( 92 – 94 ). they regard 50% sequence identity as the threshold below which function drastically diverges ( 93 , 94 ). our sequence conservation analyses almost always led to sequence identities &gt; 50%, except for zglun2da. nevertheless, we carried out more detailed examinations of more poorly conserved domains, e.g., the ntd and ctd, to assess the conservation of key structure-function motifs. nmdar missense variant conservation in zebrafish to date, a multitude of variants, including missense, frameshift, and nonsense mutations, have been identified in the genes encoding the nmdar subunits ( 21 , 22 , 87 ). we categorized identified missense variants by pathogenicity and analyzed the extent of conservation of the positions at which they occur. we focused only on the glun1, glun2a, glun2b, and glun2d subunits for this analysis, as this is where most disease-associated variants have been identified. generally, our results demonstrate a higher degree of variant position conservation with increasing pathogenicity (fig. 2 , 3 ). the sites of pathogenic missense variants are almost perfectly conserved (fig. 2 ). given the high conservation of the amino acid positions at which they occur, these protein regions are likely most essential for receptor function. across our categories of variants, with the exclusion of complex variants, the proportion of the total protein at which missense variants have been identified is lowest in the pathogenic group (table 2 ). it is possible that additional variants that might be categorized as pathogenic are so devastating to development as to be embryonic lethal. this reflects the notion that more integral regions of the receptor are less tolerant to variants ( 95 ). zebrafish, nevertheless, emerge as a useful model for the study of pathogenic nmdar variants. putatively pathogenic variants are conserved to a lesser extent than their pathogenic counterparts (fig. 2 ). according to our designations, these variants have the potential to cause disease, having been identified in patients, but are currently classified as having uncertain significance. thus, it aligns that these variants are comparatively less well conserved – they are likely occurring in less critical regions of the receptor and are likewise less likely to cause disease phenotypes. our subsequent investigation of putatively benign and benign variants revealed a further decrease in variant position conservation. this finding further supported our prediction that variants more highly associated with disease occur at positions with higher degrees of conservation and, thus, are more critical to protein structure and function. the conservation of positions with putatively benign and benign variants is quite comparable. for the benign variants, in particular, conservation percentages also closely reflect total protein conservation values for each paralog (fig. 3 , table 1 ). this follows from the finding that positions with identified benign variants, for most subunits, make up a substantial portion of the protein (table 2 ). nevertheless, this investigation both demonstrates the advantage of the zebrafish model in studying nmdar-associated disease and reflects the notion that the positions most essential for receptor function are those that are best conserved and more likely to induce disease when absent. overall, the trend for variant conservation is the same as that for the whole protein. conservation is always greatest in the tmd and lbd-tmd linkers, followed by the lbd, next the ntd, and last the ctd (fig. 2 , 3 ). these findings suggest that zebrafish serve as an effective model for variants, especially pathogenic and putatively pathogenic variants in both the tmd and lbd-tmd linkers. some caution, however, must be exercised when using this model system for variants in the lbd, ntd, and ctd. tmd, tmd-lbd linker, and lbd conservation our examination of key functional regions and motifs in the tmd, lbd-tmd linkers, and lbd consistently revealed complete conservation in their respective zebrafish paralogs (fig. s1 ,s2). this suggests that the basic mechanism of ion channel gating is conserved in zebrafish relative to human. as these domains and subdomains contribute to the formation of the core of the receptor – the ion channel pore allowing for ion flux – it is expected that they would be most highly conserved across species, since the major functional role of nmdars is current flux during synaptic transmission. ntd conservation the ntd is the first region in which nmdar sequences in zebrafish begin to diverge from those in human. modelling ntd variants in zebrafish must, thus, be approached with caution. for example, polyamine-binding sites in glun2b are almost completely conserved in zebrafish (fig. 4 d), so they have the potential to serve as an effective model for this functional role of nmdars. on the other hand, zinc-binding sites in glun2a and glun2b are more poorly conserved in zebrafish (fig. 4 c), so they would likely serve as a poor model for zinc-related studies in nmdars. this also aligns with our homology model finding that the zglun2aa model aligns well with glun2a when not bound to zinc, but the rmsd of the alignment is substantially poorer when glun2a interacting with zinc is used as the reference structure (table s3). ctd conservation the c-terminal domains (ctds) exhibit the lowest level of overall conservation across nmdar domains (fig. 1 ). this is consistent with the ctd’s classification as an intrinsically disordered region (idr), which lacks fixed secondary or tertiary structure and is more tolerant to evolutionary divergence ( 96 , 97 ). nonetheless, functional conservation within idrs is often maintained through the retention of post-translational modification (ptm) hotspots and short linear motifs (slims) that regulate protein interactions and intracellular signaling ( 98 ). we assessed ctd conservation in glun1, glun2a, and glun2b by analyzing experimentally validated phosphorylation, palmitoylation, and ubiquitylation sites, as well as defined protein-binding motifs. phosphorylation sites with known kinase interactions exhibited relatively high conservation, consistent with their well-characterized regulatory roles (figs. 5 a,b). in contrast, phosphorylation sites without identified kinases were less consistently retained (fig. 5 c). palmitoylation sites were fully conserved across all subunits, while ubiquitylation sites were generally retained, though conservation was notably lower in zglun2ab (fig. 5 f) and may reflect paralogue-specific divergence in degradation or trafficking pathways. protein-binding motif conservation varied by subunit. glun1 retained nearly all canonical interaction motifs, while glun2a and glun2b displayed region-specific conservation. notably well-conserved elements in the glun2 subunits included the pdz-binding motifs as well as docking regions for camkii, dapk1, and ap2 (fig. 6 ). slims overall showed higher conservation, likely due to their short, sequence-specific structure, which may place them under stronger evolutionary constraint than broader, more flexible interaction regions. despite broad sequence divergence, nmdar ctds retain a core set of conserved motifs involved in receptor trafficking, synaptic localization, and plasticity. kinase-regulated phosphorylation sites, palmitoylation sites, and pdz-binding motifs were consistently preserved. conservation of glun1 sites was always greatest in the zebrafish paralogs. among the glun2 subunits, glun2b displayed the highest overall conservation of functional elements, aligning with its essential role in neurodevelopment and synaptic signaling. these findings are especially relevant to the study of disease-associated truncating variants in the nmdar subunits. nonsense variants that introduce premature stop codons in the ctd eliminate key regulatory motifs critical for trafficking, anchoring, or signaling. when these affected regions correspond to conserved functional elements, zebrafish may serve as an appropriate in vitro and in vivo model for studying ctd-mediated disease mechanisms. conversely, for mutations in poorly conserved regions, the translational utility of this model may become limited. conclusions zebrafish represent a valuable system to study nmdar-associated neurodevelopmental disorders, but one must be judicious in choosing which variants to study in this model. our unique approach of sequence alignments per individual nmdar domains and subdomains highlights the high degree of conversation of some regions of the protein relative to others. additionally, our analysis of identified variants indicates that zebrafish are an effective model for pathogenic variants as these occur at conserved locations along the protein. generally, the utility of zebrafish is most apparent for the study of the tmd, lbd-tmd linkers, and lbd as these regions are most well conserved. on the other hand, studying the ntd and ctd in this species requires extra consideration stemming from deviations in sequence conservation. our work will enable future nmdar-related studies that can be effectively and efficiently conducted in zebrafish. abbreviations nmdar n -methyl-d-aspartate receptor ntd n-terminal domain lbd ligand-binding domain tmd transmembrane domain ctd c-terminal domain declarations ethics approval and consent to participate: not applicable. consent for publication: not applicable. availability of data and materials: the datasets used and/or analyzed during the current study are available either at osf (https://osf.io/h2zf9/) or from the corresponding author on reasonable request. competing interests: no competing interests declared. funding: this work was supported by nih grant r01ns088479 to lpw. authors’ contributions: e.r.n., c.a., j.d.z., h.i.s., and l.p.w. contributed to the conceptualization and methodology of this project. e.r.n., c.a., and j.d.z. conducted the investigation and analysis. w.r. assisted with data curation. e.r.n, c.a., and l.p.w. wrote the manuscript, which was revised with insights from j.d.z. and h.i.s. acknowledgements: the authors thank the clinvar and gnomad databases and the swiss-model server for their important contributions to this work. references paoletti p, bellone c, zhou q. nmda receptor subunit diversity: impact on receptor properties, synaptic plasticity and disease. nat rev neurosci. 2013;14(6):383-400. hansen kb, wollmuth lp, bowie d, furukawa h, menniti fs, sobolevsky ai, et al. structure, function, and pharmacology of glutamate receptor ion channels. pharmacol rev. 2021;73(4):298-487. chakraborty a, murphy s, coleman n. the role of nmda receptors in neural stem cell proliferation and differentiation. stem cells dev. 2017;26(11):798-807. nacher j, mcewen bs. the role of n-methyl-d-asparate receptors in neurogenesis. hippocampus. 2006;16(3):267-70. komuro h, rakic p. modulation of neuronal migration by nmda receptors. science. 1993;260(5104):95-7. pearce ia, cambray-deakin ma, burgoyne rd. glutamate acting on nmda receptors stimulates neurite outgrowth from cerebellar granule cells. febs lett. 1987;223(1):143-7. booker sa, wyllie dja. nmda receptor function in inhibitory neurons. neuropharmacology. 2021;196:108609. gambrill ac, barria a. nmda receptor subunit composition controls synaptogenesis and synapse stabilization. proc natl acad sci u s a. 2011;108(14):5855-60. napoli aj, laderwager s, zoodsma jd, biju b, mucollari o, schubel sk, et al. developmental loss of nmda receptors results in supernumerary forebrain neurons through delayed maturation of transit-amplifying neuroblasts. sci rep. 2024;14(1):3395. rajani v, sengar as, salter mw. tripartite signalling by nmda receptors. mol brain. 2020;13(1):23. ishchenko y, carrizales mg, koleske aj. regulation of the nmda receptor by its cytoplasmic domains: (how) is the tail wagging the dog? neuropharmacology. 2021;195:108634. mony l, paoletti p. mechanisms of nmda receptor regulation. curr opin neurobiol. 2023;83:102815. nielsen ac, anderson cl, ens c, boyce akj, thompson rj. non-ionotropic nmdar signalling activates panx1 to induce p2x4r-dependent long-term depression in the hippocampus. j physiol. 2024. barnes sa, thomazeau a, finnie psb, heinrich mj, heynen aj, komiyama nh, et al. non-ionotropic signaling through the nmda receptor glun2b carboxy-terminal domain drives dendritic spine plasticity and reverses fragile x phenotypes. cell rep. 2025;44(3):115311. platzer k, lemke jr. grin1-related neurodevelopmental disorder. genereviews((r)). 2021. strehlow v, myers ka, morgan at, scheffer ie, lemke jr. grin2a-related disorders. genereviews((r)). 2024. platzer k, lemke jr. grin2b-related neurodevelopmental disorder. genereviews((r)). 2021. platzer k, krey i, lemke jr. grin2d-related developmental and epileptic encephalopathy. genereviews((r)). 2022. zehavi y, mandel h, zehavi a, rashid ma, straussberg r, jabur b, et al. de novo grin1 mutations: an emerging cause of severe early infantile encephalopathy. eur j med genet. 2017;60(6):317-20. sabo sl, lahr jm, offer m, weekes a, sceniak mp. grin2b-related neurodevelopmental disorder: current understanding of pathophysiological mechanisms. front synaptic neurosci. 2022;14:1090865. gjerulfsen ce, krey i, klockner c, rubboli g, lemke jr, moller rs. spectrum of nmda receptor variants in neurodevelopmental disorders and epilepsy. methods mol biol. 2024;2799:1-11. amin jb, moody gr, wollmuth lp. from bedside-to-bench: what disease-associated variants are teaching us about the nmda receptor. j physiol. 2021;599(2):397-416. hu c, chen w, myers sj, yuan h, traynelis sf. human grin2b variants in neurodevelopmental disorders. j pharmacol sci. 2016;132(2):115-21. cox ja, kucenas s, voigt mm. molecular characterization and embryonic expression of the family of n-methyl-d-aspartate receptor subunit genes in the zebrafish. dev dyn. 2005;234(3):756-66. veldman mb, lin s. zebrafish as a developmental model organism for pediatric research. pediatr res. 2008;64(5):470-6. choi ty, choi ti, lee yr, choe sk, kim ch. zebrafish as an animal model for biomedical research. exp mol med. 2021;53(3):310-7. zoodsma jd, chan k, bhandiwad aa, golann dr, liu g, syed sa, et al. a model to study nmda receptors in early nervous system development. j neurosci. 2020;40(18):3631-45. zoodsma jd, keegan ej, moody gr, bhandiwad aa, napoli aj, burgess ha, et al. disruption of grin2b, an asd-associated gene, produces social deficits in zebrafish. mol autism. 2022;13(1):38. baraban sc, taylor mr, castro pa, baier h. pentylenetetrazole induced changes in zebrafish behavior, neural activity and c-fos expression. neuroscience. 2005;131(3):759-68. shams s, rihel j, ortiz jg, gerlai r. the zebrafish as a promising tool for modeling human brain disorders: a review based upon an ibns symposium. neurosci biobehav rev. 2018;85:176-90. de abreu ms, genario r, giacomini a, demin ka, lakstygal am, amstislavskaya tg, et al. zebrafish as a model of neurodevelopmental disorders. neuroscience. 2020;445:3-11. weinschutz mendes h, neelakantan u, liu y, fitzpatrick se, chen t, wu w, et al. high-throughput functional analysis of autism genes in zebrafish identifies convergence in dopaminergic and neuroimmune pathways. cell rep. 2023;42(3):112243. howe k, clark md, torroja cf, torrance j, berthelot c, muffato m, et al. the zebrafish reference genome sequence and its relationship to the human genome. nature. 2013;496(7446):498-503. rihel j, schier af. behavioral screening for neuroactive drugs in zebrafish. dev neurobiol. 2012;72(3):373-85. baraban sc, dinday mt, hortopan ga. drug screening in scn1a zebrafish mutant identifies clemizole as a potential dravet syndrome treatment. nat commun. 2013;4:2410. zhang y, kecskes a, copmans d, langlois m, crawford ad, ceulemans b, et al. pharmacological characterization of an antisense knockdown zebrafish model of dravet syndrome: inhibition of epileptic seizures by the serotonin agonist fenfluramine. plos one. 2015;10(5):e0125898. dingemans ajm, jansen s, van reeuwijk j, de leeuw n, pfundt r, schuurs-hoeijmakers j, et al. prevalence of comorbidities in individuals with neurodevelopmental disorders from the aggregated phenomics data of 51,227 pediatric individuals. nat med. 2024;30(7):1994-2003. ohtsuki t, sakurai k, dou h, toru m, yamakawa-kobayashi k, arinami t. mutation analysis of the nmdar2b (grin2b) gene in schizophrenia. mol psychiatry. 2001;6(2):211-6. arnold pd, rosenberg dr, mundo e, tharmalingam s, kennedy jl, richter ma. association of a glutamate (nmda) subunit receptor gene (grin2b) with obsessive-compulsive disorder: a preliminary study. psychopharmacology (berl). 2004;174(4):530-8. saunders ja, tatard-leitman vm, suh j, billingslea en, roberts tp, siegel sj. knockout of nmda receptors in parvalbumin interneurons recreates autism-like phenotypes. autism res. 2013;6(2):69-77. gandal mj, sisti j, klook k, ortinski pi, leitman v, liang y, et al. gabab-mediated rescue of altered excitatory-inhibitory balance, gamma synchrony and behavioral deficits following constitutive nmdar-hypofunction. transl psychiatry. 2012;2(7):e142. barnby g, abbott a, sykes n, morris a, weeks de, mott r, et al. candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at grin2a and abat. am j hum genet. 2005;76(6):950-66. hung ly, margolis kg. autism spectrum disorders and the gastrointestinal tract: insights into mechanisms and clinical relevance. nat rev gastroenterol hepatol. 2024;21(3):142-63. owens ap, mathias cj, iodice v. autonomic dysfunction in autism spectrum disorder. front integr neurosci. 2021;15:787037. orefice ll. peripheral somatosensory neuron dysfunction: emerging roles in autism spectrum disorders. neuroscience. 2020;445:120-9. makris g, agorastos a, chrousos gp, pervanidou p. stress system activation in children and adolescents with autism spectrum disorder. front neurosci. 2021;15:756628. ramos-vicente d, grant sg, bayes a. metazoan evolution and diversity of glutamate receptors and their auxiliary subunits. neuropharmacology. 2021;195:108640. amores a, force a, yan yl, joly l, amemiya c, fritz a, et al. zebrafish hox clusters and vertebrate genome evolution. science. 1998;282(5394):1711-4. chou th, tajima n, romero-hernandez a, furukawa h. structural basis of functional transitions in mammalian nmda receptors. cell. 2020;182(2):357-71 e13. zhang y, ye f, zhang t, lv s, zhou l, du d, et al. structural basis of ketamine action on human nmda receptors. nature. 2021;596(7871):301-5. wollmuth lp, sakmann b. different mechanisms of ca2+ transport in nmda and ca2+-permeable ampa glutamate receptor channels. j gen physiol. 1998;112(5):623-36. watanabe j, beck c, kuner t, premkumar ls, wollmuth lp. drpeer: a motif in the extracellular vestibule conferring high ca2+ flux rates in nmda receptor channels. j neurosci. 2002;22(23):10209-16. wollmuth lp, sobolevsky ai. structure and gating of the glutamate receptor ion channel. trends neurosci. 2004;27(6):321-8. ogden kk, khatri a, traynelis sf, heldt sa. potentiation of glun2c/d nmda receptor subtypes in the amygdala facilitates the retention of fear and extinction learning in mice. neuropsychopharmacology. 2014;39(3):625-37. wollmuth lp. ion permeation in ionotropic glutamate receptors: still dynamic after all these years. curr opin physiol. 2018;2:36-41. mcdaniel mj, ogden kk, kell sa, burger pb, liotta dc, traynelis sf. nmda receptor channel gating control by the pre-m1 helix. j gen physiol. 2020;152(4). wollmuth lp, kuner t, sakmann b. adjacent asparagines in the nr2-subunit of the nmda receptor channel control the voltage-dependent block by extracellular mg2+. j physiol. 1998;506 ( pt 1)(pt 1):13-32. romero-hernandez a, simorowski n, karakas e, furukawa h. molecular basis for subtype specificity and high-affinity zinc inhibition in the glun1-glun2a nmda receptor amino-terminal domain. neuron. 2016;92(6):1324-36. mony l, zhu s, carvalho s, paoletti p. molecular basis of positive allosteric modulation of glun2b nmda receptors by polyamines. embo j. 2011;30(15):3134-46. lichnerova k, kaniakova m, park sp, skrenkova k, wang yx, petralia rs, et al. two n-glycosylation sites in the glun1 subunit are essential for releasing n-methyl-d-aspartate (nmda) receptors from the endoplasmic reticulum. j biol chem. 2015;290(30):18379-90. waterhouse a, bertoni m, bienert s, studer g, tauriello g, gumienny r, et al. swiss-model: homology modelling of protein structures and complexes. nucleic acids res. 2018;46(w1):w296-w303. bienert s, waterhouse a, de beer ta, tauriello g, studer g, bordoli l, et al. the swiss-model repository-new features and functionality. nucleic acids res. 2017;45(d1):d313-d9. warnet xl, bakke krog h, sevillano-quispe og, poulsen h, kjaergaard m. the c-terminal domains of the nmda receptor: how intrinsically disordered tails affect signalling, plasticity and disease. eur j neurosci. 2021;54(8):6713-39. kim k, saneyoshi t, hosokawa t, okamoto k, hayashi y. interplay of enzymatic and structural functions of camkii in long-term potentiation. j neurochem. 2016;139(6):959-72. bej a, ames jb. chemical shift assignments of calmodulin bound to a cytosolic domain of glun2a (residues 1004-1024) from the nmda receptor. biomol nmr assign. 2023;17(1):89-93. horak m, wenthold rj. different roles of c-terminal cassettes in the trafficking of full-length nr1 subunits to the cell surface. j biol chem. 2009;284(15):9683-91. lin jw, wyszynski m, madhavan r, sealock r, kim ju, sheng m. yotiao, a novel protein of neuromuscular junction and brain that interacts with specific splice variants of nmda receptor subunit nr1. j neurosci. 1998;18(6):2017-27. stanic j, carta m, eberini i, pelucchi s, marcello e, genazzani aa, et al. rabphilin 3a retains nmda receptors at synaptic sites through interaction with glun2a/psd-95 complex. nat commun. 2015;6:10181. cousins sl, stephenson fa. identification of c-terminal binding protein 1 as a novel nmda receptor interactor. neurochem res. 2019;44(6):1437-45. franchini l, carrano n, di luca m, gardoni f. synaptic glun2a-containing nmda receptors: from physiology to pathological synaptic plasticity. int j mol sci. 2020;21(4). swanwick cc, shapiro me, yi z, chang k, wenthold rj. nmda receptors interact with flotillin-1 and -2, lipid raft-associated proteins. febs lett. 2009;583(8):1226-30. cousins sl, stephenson fa. identification of n-methyl-d-aspartic acid (nmda) receptor subtype-specific binding sites that mediate direct interactions with scaffold protein psd-95. j biol chem. 2012;287(16):13465-76. vissel b, krupp jj, heinemann sf, westbrook gl. a use-dependent tyrosine dephosphorylation of nmda receptors is independent of ion flux. nat neurosci. 2001;4(6):587-96. lavezzari g, mccallum j, lee r, roche kw. differential binding of the ap-2 adaptor complex and psd-95 to the c-terminus of the nmda receptor subunit nr2b regulates surface expression. neuropharmacology. 2003;45(6):729-37. chen bs, gray ja, sanz-clemente a, wei z, thomas ev, nicoll ra, et al. sap102 mediates synaptic clearance of nmda receptors. cell rep. 2012;2(5):1120-8. elagabani mn, brisevac d, kintscher m, pohle j, kohr g, schmitz d, et al. subunit-selective n-methyl-d-aspartate (nmda) receptor signaling through brefeldin a-resistant arf guanine nucleotide exchange factors brag1 and brag2 during synapse maturation. j biol chem. 2016;291(17):9105-18. yaka r, thornton c, vagts aj, phamluong k, bonci a, ron d. nmda receptor function is regulated by the inhibitory scaffolding protein, rack1. proc natl acad sci u s a. 2002;99(8):5710-5. wyszynski m, lin j, rao a, nigh e, beggs ah, craig am, et al. competitive binding of alpha-actinin and calmodulin to the nmda receptor. nature. 1997;385(6615):439-42. wechsler a, teichberg vi. brain spectrin binding to the nmda receptor is regulated by phosphorylation, calcium and calmodulin. embo j. 1998;17(14):3931-9. krapivinsky g, krapivinsky l, manasian y, ivanov a, tyzio r, pellegrino c, et al. the nmda receptor is coupled to the erk pathway by a direct interaction between nr2b and rasgrf1. neuron. 2003;40(4):775-84. mukherjee p, roy s, ghosh d, nandi sk. role of animal models in biomedical research: a review. lab anim res. 2022;38(1):18. chou th, epstein m, fritzemeier rg, akins ns, paladugu s, ullman ez, et al. molecular mechanism of ligand gating and opening of nmda receptor. nature. 2024;632(8023):209-17. armstrong n, sun y, chen gq, gouaux e. structure of a glutamate-receptor ligand-binding core in complex with kainate. nature. 1998;395(6705):913-7. armstrong n, gouaux e. mechanisms for activation and antagonism of an ampa-sensitive glutamate receptor: crystal structures of the glur2 ligand binding core. neuron. 2000;28(1):165-81. kazi r, gan q, talukder i, markowitz m, salussolia cl, wollmuth lp. asynchronous movements prior to pore opening in nmda receptors. j neurosci. 2013;33(29):12052-66. amin jb, gochman a, he m, certain n, wollmuth lp. nmda receptors require multiple pre-opening gating steps for efficient synaptic activity. neuron. 2021;109(3):488-501 e4. hardingham ge, do kq. linking early-life nmdar hypofunction and oxidative stress in schizophrenia pathogenesis. nat rev neurosci. 2016;17(2):125-34. landrum mj, lee jm, riley gr, jang w, rubinstein ws, church dm, et al. clinvar: public archive of relationships among sequence variation and human phenotype. nucleic acids res. 2014;42(database issue):d980-5. landrum mj, lee jm, benson m, brown g, chao c, chitipiralla s, et al. clinvar: public archive of interpretations of clinically relevant variants. nucleic acids res. 2016;44(d1):d862-8. landrum mj, lee jm, benson m, brown gr, chao c, chitipiralla s, et al. clinvar: improving access to variant interpretations and supporting evidence. nucleic acids res. 2018;46(d1):d1062-d7. gudmundsson s, singer-berk m, watts na, phu w, goodrich jk, solomonson m, et al. variant interpretation using population databases: lessons from gnomad. hum mutat. 2022;43(8):1012-30. tian w, skolnick j. how well is enzyme function conserved as a function of pairwise sequence identity? j mol biol. 2003;333(4):863-82. addou s, rentzsch r, lee d, orengo ca. domain-based and family-specific sequence identity thresholds increase the levels of reliable protein function transfer. j mol biol. 2009;387(2):416-30. sangar v, blankenberg dj, altman n, lesk am. quantitative sequence-function relationships in proteins based on gene ontology. bmc bioinformatics. 2007;8:294. perszyk re, kristensen as, lyuboslavsky p, traynelis sf. three-dimensional missense tolerance ratio analysis. genome res. 2021;31(8):1447-61. uversky vn. intrinsically disordered proteins from a to z. int j biochem cell biol. 2011;43(8):1090-103. singleton md, eisen mb. evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. plos comput biol. 2024;20(4):e1012028. davey ne, cowan jl, shields dc, gibson tj, coldwell mj, edwards rj. slimprints: conservation-based discovery of functional motif fingerprints in intrinsically disordered protein regions. nucleic acids res. 2012;40(21):10628-41. additional declarations no competing interests reported. supplementary files zfishnmdarssupplement20250716.docx cite share download pdf status: under review version 1 posted editorial decision: revision requested 24 sep, 2025 reviews received at journal 19 sep, 2025 reviewers agreed at journal 19 sep, 2025 reviews received at journal 09 sep, 2025 reviews received at journal 18 aug, 2025 reviewers agreed at journal 04 aug, 2025 reviewers agreed at journal 28 jul, 2025 reviewers invited by journal 28 jul, 2025 editor assigned by journal 28 jul, 2025 editor invited by journal 23 jul, 2025 submission checks completed at journal 22 jul, 2025 first submitted to journal 22 jul, 2025 you are reading this latest preprint version research square lets you share your work early, gain feedback from the community, and start making changes to your manuscript prior to peer review in a journal. as a division of research square company, we’re committed to making research communication faster, fairer, and more useful. we do this by developing innovative software and high quality services for the global research community. our growing team is made up of researchers and industry professionals working together to solve the most critical problems facing scientific publishing. also discoverable on platform about our team in review editorial policies advisory board help center resources author services research quality evaluation accessibility api access rss feed manage cookie preferences © research square 2025 | issn 2693-5015 (online) privacy policy terms of service do not sell my personal information {"props":{"pageprops":{"initialdata":{"identity":"rs-7151578","acceptedtermsandconditions":true,"allowdirectsubmit":false,"archivedversions":[],"articletype":"research article","associatedpublications":[],"authors":[{"id":492782335,"identity":"6084adf8-9103-4ca8-8de5-679d2a4ea2bb","order_by":0,"name":"erica r. nebet","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"erica","middlename":"r.","lastname":"nebet","suffix":""},{"id":492782336,"identity":"68eb1705-4eac-413e-bc82-7b9943220d12","order_by":1,"name":"christieann aprea","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"christieann","middlename":"","lastname":"aprea","suffix":""},{"id":492782337,"identity":"776ee37e-7ff9-4a8a-bc43-f965303e90a8","order_by":2,"name":"josiah d. zoodsma","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"josiah","middlename":"d.","lastname":"zoodsma","suffix":""},{"id":492782338,"identity":"ef7f492d-83d8-41a3-aa81-6be2af423f07","order_by":3,"name":"william raab","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"william","middlename":"","lastname":"raab","suffix":""},{"id":492782339,"identity":"6949754f-6430-4ffc-8673-da5548d44bd6","order_by":4,"name":"howard i. sirotkin","email":"","orcid":"","institution":"stony brook university","correspondingauthor":false,"prefix":"","firstname":"howard","middlename":"i.","lastname":"sirotkin","suffix":""},{"id":492782340,"identity":"a4f8ecb1-612e-43db-8923-d325a2be05c2","order_by":5,"name":"lonnie p. wollmuth","email":"data:image/png;base64,ivborw0kggoaaaansuheugaaazaaaaayaqmaaabi0h/eaaaablbmvex///8aaabvwtn+aaaacxbiwxmaaa7eaaaoxagvkw4baaabhuleqvriiwnggawjybacphcraclymrua5ae5sdajim0dxwiada6fgaqs4yaxwiigfgyklyknblww9x578kccibgfmbf1m++po+nz5zcfe8y7454cp88bxgdv2zc18jxln0g4w5a4s5mx7tzpwrpcdcfy0o15zxqbs/y2mbvoxajfisdmirgnixfdybcww7kb2hjmphnbehi3ngdgazjwapmh0jiu34bqwv4bp5yghjyehmmwlwcb2jixaee5yyarceyihusxm3psdhtuojawjjn3tikxzm/bzsk55zc08lp3men+qlex5mdvpnbjjc1hefnmw8ck3u5iaizy8sepb8owqxkcjfc5tjdoamcokydxb/fqr8eogawjypgdabgnaum7/svdaaaaaelftksuqmcc","orcid":"","institution":"stony brook university","correspondingauthor":true,"prefix":"","firstname":"lonnie","middlename":"p.","lastname":"wollmuth","suffix":""}],"badges":[],"createdat":"2025-07-17 18:08:21","currentversioncode":1,"declarations":"","doi":"10.21203/rs.3.rs-7151578/v1","doiurl":"https://doi.org/10.21203/rs.3.rs-7151578/v1","draftversion":[],"editorialevents":[],"editorialnote":"","failedworkflow":false,"files":[{"id":87954191,"identity":"93406acd-9b5c-4bbe-b8e4-476bfc8e3591","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":0,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":48663694,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarsmain20250722.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/72a01d08b07202a8dd3432a7.docx"},{"id":87954161,"identity":"09dcee36-3c3d-4bae-a5b6-a8031a04e2f8","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"json","order_by":1,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":8698,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b.json","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/088093afe258318de15a5f12.json"},{"id":87954169,"identity":"c1d3acb0-9e25-4b8a-9228-e0e154b314b2","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":2,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":15353369,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarssupplement20250716.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a3a65bcc549c7954c295d2d1.docx"},{"id":87954163,"identity":"6fde5d88-4da9-48b5-ac47-0b0e0f6354d8","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"xml","order_by":3,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":242943,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b1enriched.xml","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/aad420877d9070a2aedac4cc.xml"},{"id":87954167,"identity":"b5c71071-9fd7-476b-9303-17dcb5b1c845","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"jpeg","order_by":4,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":7224546,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage1.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/239b3c6e0e27c5d5f3a3c485.jpeg"},{"id":87954176,"identity":"66a11160-0f4e-44e0-a6b3-26a6c526f9c1","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":5,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5495318,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage2.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/7c759429dc2c4ccf2d95e1ed.jpeg"},{"id":87954168,"identity":"e3a84d67-d0ce-4822-8529-9e1cc04acd3d","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":6,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5497046,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage3.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/5ab1e35184bc289007b868d7.jpeg"},{"id":87954408,"identity":"4452df13-a563-4564-9a2c-9a250a285ebb","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"jpeg","order_by":7,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4536836,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage4.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/34cad84600342cebbb343c8d.jpeg"},{"id":87954171,"identity":"94757fe0-c77c-46b6-8a13-76b5920bc170","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":8,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4702238,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage5.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/704be5ce4c0cc10b6f98b684.jpeg"},{"id":87954173,"identity":"bbb91acc-360b-43aa-99a5-89fbc5989777","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":9,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":4378230,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage6.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/85acc94c54964dbcd4ee6caa.jpeg"},{"id":87954192,"identity":"2447fad7-ffab-4e33-9b2f-a71d2af30f9e","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"jpeg","order_by":10,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5735470,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage7.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3a0654f7ccd9aa40cc5d5f69.jpeg"},{"id":87954413,"identity":"29bec889-0025-4009-9267-6438ea9c5740","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"jpeg","order_by":11,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":5327046,"visible":true,"origin":"","legend":"","description":"","filename":"floatimage8.jpeg","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/c3009b0732aa42082efe8df3.jpeg"},{"id":87954188,"identity":"20ba216a-d094-494f-9894-1937509c8d81","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":12,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":222456,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage1.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/08aba5e904980f68ab32639e.png"},{"id":87954187,"identity":"d99856fc-69f6-4dda-9eeb-4cafd6cc99a0","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":13,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":175317,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage2.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a0ea7afa8616702c9ac66b10.png"},{"id":87954174,"identity":"de47bd92-e161-4fbe-a9b2-437c4b5f0573","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":14,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":140021,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage3.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/bb8abfaa1c199dae266df4dd.png"},{"id":87954412,"identity":"87f7cc6b-e10f-4300-9539-2c2a02855ddb","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"png","order_by":15,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":138717,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage4.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/a6b8e16a616611723ae39dab.png"},{"id":87954190,"identity":"ad2ebfe9-5bbc-4128-8d93-eaa9d7470a57","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":16,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":96612,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage5.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/99939544fae92f971073fd51.png"},{"id":87954189,"identity":"84923e46-044f-4cd2-9f5a-b3a9fa41caab","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":17,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":72313,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage6.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/92848d9ff37daaad2b48e9ff.png"},{"id":87954755,"identity":"eaa7ae44-7e94-4117-97cb-d03c1d07f90a","added_by":"auto","created_at":"2025-07-30 18:41:44","extension":"png","order_by":18,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":83342,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage7.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/845b774d666648bfa0a844ab.png"},{"id":87954410,"identity":"13827b86-cf06-4cf3-9952-4b424318037f","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"png","order_by":19,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":117211,"visible":true,"origin":"","legend":"","description":"","filename":"onlinefloatimage8.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/af95a59a499464e286bef54b.png"},{"id":87954411,"identity":"7741fc55-f9dc-44b6-9b97-8875d99f5f4b","added_by":"auto","created_at":"2025-07-30 18:33:44","extension":"xml","order_by":20,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":240176,"visible":true,"origin":"","legend":"","description":"","filename":"b48938f334554205a41ce6e526b2cd8b1structuring.xml","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/dfb8648302675ebf344a2fa6.xml"},{"id":87954756,"identity":"beb85617-8838-4c99-8712-1a7d62a23eff","added_by":"auto","created_at":"2025-07-30 18:41:44","extension":"html","order_by":21,"title":"","display":"","copyasset":false,"role":"acdc-reference","size":257708,"visible":true,"origin":"","legend":"","description":"","filename":"earlyproof.html","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/01f35be0d8432473e2775041.html"},{"id":87954160,"identity":"3920a076-09de-49c0-b764-c081048b3b14","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":1,"title":"figure 1","display":"","copyasset":false,"role":"figure","size":795350,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of human nmdar subunits in zebrafish\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a)\u003c/strong\u003e nmdars are composed of two obligatory glun1 and typically two glun2a-d subunits. the tetrameric complex consists of four domains: extracellular n-terminal (ntd) and ligand-binding (lbd) domains; a transmembrane domain (tmd); and an intracellular c-terminal domain (ctd). glun1 subunits are colored dark blue (ntd), teal (lbd), red (lbd-tmd linkers), light orange (tmd), and gray (ctd); and glun2b subunits gray. 9are (82)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b)\u003c/strong\u003e topology of an individual glun1 subunit. lbd lobes (d1 and d2) are shown; d1 is mainly comprisedof s1 and d2 of s2. the ion channel is formed by 3 transmembrane segments, m1, m3, and m4, and an intracellular m2 pore loop. lbd-tmd linkers are s1-m1 (or d2-m1), m3-s2 (or d2-m3), and s2-m4 (or d1-m4).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c)\u003c/strong\u003e nmdar subunits are separated into 13 structural subdivisions: ntd, lbd (s1 \u0026amp; s2), lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4), tmd (m1, intracellular m1-m2 linker, m2 loop, m2-m3 linker, m3, \u0026amp; m4), and ctd. amino acid length of each zebrafish subdivision is indicated. those sharing amino acid length with their human orthologues are indicated in white, while increasingly darker pink indicates those with increasingly greater amino acid differences (greater (+) or fewer (-)).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(d-e)\u003c/strong\u003e nmdar subunits are grouped as in \u003cstrong\u003e(c)\u003c/strong\u003e. homology between human and zebrafish nmdar subunit paralogues is reported in amino acid identity \u003cstrong\u003e(d)\u003c/strong\u003e or similarity \u003cstrong\u003e(e)\u003c/strong\u003e. heat map with darker green indicating greater percent of homology.\u003c/p\u003e","description":"","filename":"floatimage1.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/5b4e27c91809ed8be6a2b3b5.png"},{"id":87954407,"identity":"47d4fb26-35bf-49ca-aae6-ed196317e2be","added_by":"auto","created_at":"2025-07-30 18:33:43","extension":"png","order_by":2,"title":"figure 2","display":"","copyasset":false,"role":"figure","size":712960,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of positions with pathogenic and putatively pathogenic missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a-b) \u003c/strong\u003epercent conservation of pathogenic variant positions in zebrafish genes, as defined in table 2. conservation is reported in amino acid identity \u003cstrong\u003e(a) \u003c/strong\u003eand similarity \u003cstrong\u003e(b)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c-d)\u003c/strong\u003e conservation of pathogenic variant positions in each nmdar domain, grouped as in figure 1. homology is reported in identity \u003cstrong\u003e(c)\u003c/strong\u003eand similarity \u003cstrong\u003e(d)\u003c/strong\u003e. dashes indicate domains without reported pathogenic variants. along with the percent of conservation, the total number of identified variants in the human protein is indicated in parentheses in the ‘a’ paralog column in \u003cstrong\u003e(c)\u003c/strong\u003e.\u003c/p\u003e","description":"","filename":"floatimage2.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/d7d962b025e722b029324c59.png"},{"id":87954162,"identity":"02953530-369f-4594-b587-6d86079d6a68","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":3,"title":"figure 3","display":"","copyasset":false,"role":"figure","size":667858,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of positions with putatively benign and benign missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a-b) \u003c/strong\u003epercent conservation of putatively benign variant positions in zebrafish genes, as defined in table 2. conservation is reported in identity \u003cstrong\u003e(a) \u003c/strong\u003eand similarity \u003cstrong\u003e(b)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c-d) \u003c/strong\u003econservation of putatively benign variant positions in each nmdar domain, grouped as in figure 2. homology is reported in identity \u003cstrong\u003e(c)\u003c/strong\u003e and similarity \u003cstrong\u003e(d)\u003c/strong\u003e. dashes indicate domains without reported pathogenic variants. along with the percent of conservation, the total number of identified variants in the human protein is indicated in parentheses in the ‘a’ paralog column in \u003cstrong\u003e(c)\u003c/strong\u003e.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(e-h) \u003c/strong\u003eas in a-d but for benign variant positions.\u003c/p\u003e","description":"","filename":"floatimage3.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/069549024abcf2bb76a43e0e.png"},{"id":87954166,"identity":"f30749e8-12a9-42b8-89f5-26186b255ecb","added_by":"auto","created_at":"2025-07-30 18:25:43","extension":"png","order_by":4,"title":"figure 4","display":"","copyasset":false,"role":"figure","size":473605,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key functional elements and structure of the nmdar ntd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003etopology of glun2a ntd, showing r1 and r2 subdomains, bound to zinc (red). 5tpw (58)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b) \u003c/strong\u003eclose-up image of structure in \u003cstrong\u003e(a)\u003c/strong\u003ehighlighting amino acids that participate in zinc binding. his44 is demarcated as it is not conserved in the zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(c) \u003c/strong\u003econservation of amino acids that participate in ntd zinc binding in glun2a and glun2b, reported in identity (dark red) and similarity (light red). sequences are shown to the right.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(d) \u003c/strong\u003econservation of glun2b ntd regions that participate in interactions with polyamines, reported in identity (dark gray) and similarity (light gray). sequences are shown to the right.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(e) \u003c/strong\u003econservation of ntd glycosylation sites, reported in identity (dark gray).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(f) \u003c/strong\u003ealigned ntd homology models of the human (blue) and zebrafish (gray) protein for glun1 vs zglun1a (reference structure, 5tp0) (\u003cem\u003eleft\u003c/em\u003e), glun2a vs zglun2aa (reference structure, 5tp0) (\u003cem\u003emiddle\u003c/em\u003e), and glun2b vs zglun2bb (reference structure, 5tpz) (\u003cem\u003eright\u003c/em\u003e). secondary structures colored both blue and gray indicate areas of structural overlap.\u003c/p\u003e","description":"","filename":"floatimage4.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3693c16c36a6ea0f931e6343.png"},{"id":87954406,"identity":"16a86e66-9b13-469f-91ca-fe2142d4ce9c","added_by":"auto","created_at":"2025-07-30 18:33:43","extension":"png","order_by":5,"title":"figure 5","display":"","copyasset":false,"role":"figure","size":427432,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key nmdar ctd phosphorylation sites\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd kinase-regulated phosphorylation sites for the glun1, glun2a and glun2b ctds. the specific phosphorylation sites and the corresponding human and zebrafish sequences are indicated. positions sharing identity are indicated in dark gray. similarity is shown in light gray; here, a residue is only considered similar if it is ‘phosphorylatable’ (e.g., contains a s, t, or y) (see methods).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-c) \u003c/strong\u003epercent conservation of kinase-regulated phosphorylation sites \u003cstrong\u003e(b)\u003c/strong\u003e and other phosphorylation sites identified by phosphoproteomics \u003cstrong\u003e(c)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray); similarity is defined in \u003cstrong\u003e(a)\u003c/strong\u003e.\u003c/p\u003e","description":"","filename":"floatimage5.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/25001bb8418306120cd3e005.png"},{"id":87954184,"identity":"a64d1f54-7d76-472b-a396-d9efcf5f86c8","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":6,"title":"figure 6","display":"","copyasset":false,"role":"figure","size":333594,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of key nmdar ctd post-translational modification (ptm) sites\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd palmitoylation and ubiquitylation sites for the glun1, glun2a and glun2b ctds. the specific phosphorylation sites and the corresponding human and zebrafish sequences are indicated. positions sharing identity are indicated in dark gray.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-c) \u003c/strong\u003epercent conservation of palmitoylation \u003cstrong\u003e(b)\u003c/strong\u003e and ubiquitylation \u003cstrong\u003e(c)\u003c/strong\u003e sites, reported in identity (dark gray).\u003c/p\u003e","description":"","filename":"floatimage6.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/fac7b19510f29b178f7e3661.png"},{"id":87954182,"identity":"d09e842a-ccf2-41a7-bd3e-0935363761f9","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":7,"title":"figure 7","display":"","copyasset":false,"role":"figure","size":403546,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of nmdar short linear motif (slim) protein-binding motifs (pbms) in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd short linear motifs (slims) for the glun1, glun2a and glun2b ctds.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-d) \u003c/strong\u003epercent conservation of ctd slims in glun1 \u003cstrong\u003e(b)\u003c/strong\u003e, glun2a \u003cstrong\u003e(c)\u003c/strong\u003e, and glun2b \u003cstrong\u003e(d)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray).\u003c/p\u003e","description":"","filename":"floatimage7.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/b6119c0825c4063a5e6bb6ec.png"},{"id":87954181,"identity":"cb3e5a1b-cd98-4155-84bb-c4d2fbb40f7e","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"png","order_by":8,"title":"figure 8","display":"","copyasset":false,"role":"figure","size":518323,"visible":true,"origin":"","legend":"\u003cp\u003e\u003cstrong\u003econservation of nmdar protein-binding motifs (pbms) regions in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(a) \u003c/strong\u003eschematics of ctd regions for the glun1, glun2a and glun2b ctds.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003e(b-d) \u003c/strong\u003epercent conservation of ctd regions in glun1 \u003cstrong\u003e(b)\u003c/strong\u003e, glun2a \u003cstrong\u003e(c)\u003c/strong\u003e, and glun2b \u003cstrong\u003e(d)\u003c/strong\u003e. conservation is reported in identity (dark gray) and similarity (light gray).\u003c/p\u003e","description":"","filename":"floatimage8.png","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/ccea59247fac75e14ad8ad14.png"},{"id":87954866,"identity":"ae52deed-9d7c-4d30-94e3-0b4ac04fe145","added_by":"auto","created_at":"2025-07-30 18:49:46","extension":"pdf","order_by":0,"title":"","display":"","copyasset":false,"role":"manuscript-pdf","size":5866105,"visible":true,"origin":"","legend":"","description":"","filename":"manuscript.pdf","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/24ce2fd9-cdac-44be-947d-44a6c3564886.pdf"},{"id":87954175,"identity":"cf7d81ad-b94d-4e4f-95f1-e45dee844a82","added_by":"auto","created_at":"2025-07-30 18:25:44","extension":"docx","order_by":0,"title":"","display":"","copyasset":false,"role":"supplement","size":15353369,"visible":true,"origin":"","legend":"","description":"","filename":"zfishnmdarssupplement20250716.docx","url":"https://assets-eu.researchsquare.com/files/rs-7151578/v1/3ada87e108301cd5d027f600.docx"}],"financialinterests":"no competing interests reported.","formattedtitle":"conservation of human nmda receptor subunits and disease variants in zebrafish","fulltext":[{"header":"background","content":"\u003cp\u003erapid cell-to-cell signaling in the nervous system occurs at specialized structures called synapses. at these sites, the electrical activity of the upstream or presynaptic neuron is converted into the release of a chemical neurotransmitter. this is then converted back into an electrical signal in the postsynaptic neuron by specialized ion channels that are gated by a neurotransmitter, initiating downstream biological signaling cascades. glutamate, the primary excitatory neurotransmitter in the brain, activates a variety of receptors that are essential for synaptic transmission and plasticity. glutamate-gated ion channels play fundamental roles in signaling in both the developing and mature nervous systems (\u003cspan additionalcitationids=\"cr2 cr3 cr4 cr5 cr6 cr7 cr8\" citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e–\u003cspan citationid=\"cr9\" class=\"citationref\"\u003e9\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ethe nmda receptor (nmdar) is a prominent glutamate-gated ion channel subtype (\u003cspan citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e, \u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e). due to its distinct biophysical properties and unique ionotropic and non-ionotropic signaling mechanisms, it contributes to a wide array of nervous and non-nervous system functions (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan additionalcitationids=\"cr11 cr12 cr13\" citationid=\"cr10\" class=\"citationref\"\u003e10\u003c/span\u003e–\u003cspan citationid=\"cr14\" class=\"citationref\"\u003e14\u003c/span\u003e). numerous missense, frameshift, and nonsense mutations (or variants) have been identified in the genes that encode nmdar subunits and are associated with neurodevelopmental disorders, including autism spectrum disorders (asd), intellectual disability, developmental delay, seizures, and schizophrenia (\u003cspan additionalcitationids=\"cr16 cr17 cr18 cr19 cr20 cr21 cr22\" citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e–\u003cspan citationid=\"cr23\" class=\"citationref\"\u003e23\u003c/span\u003e). however, how these variants lead to clinical phenotypes is poorly understood.\u003c/p\u003e\u003cp\u003ezebrafish are a powerful model organism to study neurodevelopmental phenotypes and disorders, including those linked to nmdars (\u003cspan citationid=\"cr19\" class=\"citationref\"\u003e19\u003c/span\u003e, \u003cspan additionalcitationids=\"cr25 cr26 cr27 cr28 cr29 cr30 cr31\" citationid=\"cr24\" class=\"citationref\"\u003e24\u003c/span\u003e–\u003cspan citationid=\"cr32\" class=\"citationref\"\u003e32\u003c/span\u003e). zebrafish develop rapidly and share fundamental developmental and signaling pathways with humans and other vertebrates (\u003cspan citationid=\"cr33\" class=\"citationref\"\u003e33\u003c/span\u003e); they are transparent, allowing for dynamic and non-invasive observations of developmental and cellular processes; and they display robust behaviors that are easily assayed at early larval stages. moreover, larval zebrafish readily take up small molecules from their environment, enabling non-invasive and high-throughput readout of nervous system function in the context of behavioral drug screens (\u003cspan additionalcitationids=\"cr35\" citationid=\"cr34\" class=\"citationref\"\u003e34\u003c/span\u003e–\u003cspan citationid=\"cr36\" class=\"citationref\"\u003e36\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enmdars are heterotetramers composed of two obligatory glun1 (encoded by \u003cem\u003egrin1\u003c/em\u003e) subunits. they are expressed as either diheterotetramers with two glun1 and two glun2 (\u003cem\u003egrin2a-d\u003c/em\u003e) subunits or triheterotetramers of some combination of glun2 and/or glun3 (\u003cem\u003egrin3a-b\u003c/em\u003e) subunits (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e) (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003ea). these nmdar subunits are differentially expressed over the course of neurodevelopment, with glun2a, glun2b, and glun2d being highly expressed during early development, while glun2c, glun3a, and glun3b expression increases into adulthood (\u003cspan citationid=\"cr1\" class=\"citationref\"\u003e1\u003c/span\u003e). moreover, nmdars display distinct functional and pharmacological properties based on their glun2 or glun3 composition (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e). each subunit consists of four principal domains: an n-terminal domain (ntd), a ligand-binding domain (lbd), a transmembrane domain (tmd) that forms the ion channel, and a c-terminal domain (ctd) (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003eb).\u003c/p\u003e\u003cp\u003e\u003c/p\u003e\u003cp\u003edisease-associated variants (davs) in \u003cem\u003egrin1\u003c/em\u003e, \u003cem\u003egrin2a\u003c/em\u003e, \u003cem\u003egrin2b\u003c/em\u003e, and \u003cem\u003egrin2d\u003c/em\u003e have been identified in pediatric patients with neurodevelopmental disorders (\u003cspan additionalcitationids=\"cr16 cr17\" citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e–\u003cspan citationid=\"cr18\" class=\"citationref\"\u003e18\u003c/span\u003e, \u003cspan citationid=\"cr21\" class=\"citationref\"\u003e21\u003c/span\u003e). these disorders often present with multiple comorbidities, reflecting overlapping neurodevelopmental phenotypes (\u003cspan citationid=\"cr37\" class=\"citationref\"\u003e37\u003c/span\u003e). generally, patients with davs in \u003cem\u003egrin1\u003c/em\u003e present with developmental delay and intellectual disability, though a large proportion of patients also experience seizures (\u003cspan citationid=\"cr15\" class=\"citationref\"\u003e15\u003c/span\u003e). in contrast, \u003cem\u003egrin2a\u003c/em\u003e and \u003cem\u003egrin2d\u003c/em\u003e are predominantly associated with seizure disorders (\u003cspan citationid=\"cr16\" class=\"citationref\"\u003e16\u003c/span\u003e, \u003cspan citationid=\"cr18\" class=\"citationref\"\u003e18\u003c/span\u003e). \u003cem\u003egrin2b\u003c/em\u003e is a high-risk gene for autism spectrum disorder (asd) (\u003cspan citationid=\"cr17\" class=\"citationref\"\u003e17\u003c/span\u003e, \u003cspan citationid=\"cr38\" class=\"citationref\"\u003e38\u003c/span\u003e, \u003cspan citationid=\"cr39\" class=\"citationref\"\u003e39\u003c/span\u003e), as are \u003cem\u003egrin1\u003c/em\u003e and \u003cem\u003egrin2a\u003c/em\u003e (\u003cspan additionalcitationids=\"cr41\" citationid=\"cr40\" class=\"citationref\"\u003e40\u003c/span\u003e–\u003cspan citationid=\"cr42\" class=\"citationref\"\u003e42\u003c/span\u003e). still, patients with such neurodevelopmental disorders generally experience a variety of both central and peripheral nervous system symptoms (\u003cspan additionalcitationids=\"cr44 cr45\" citationid=\"cr43\" class=\"citationref\"\u003e43\u003c/span\u003e–\u003cspan citationid=\"cr46\" class=\"citationref\"\u003e46\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enmda receptors are highly conserved across species (\u003cspan citationid=\"cr47\" class=\"citationref\"\u003e47\u003c/span\u003e). due to an ancient genome duplication event, zebrafish retain two paralogues for most nmdar genes (e.g., \u003cem\u003egrin1a\u003c/em\u003e \u0026amp; \u003cem\u003egrin1b\u003c/em\u003e) (\u003cspan citationid=\"cr24\" class=\"citationref\"\u003e24\u003c/span\u003e, \u003cspan citationid=\"cr48\" class=\"citationref\"\u003e48\u003c/span\u003e). patch clamp electrophysiology experiments show that zebrafish paralogues display similar functional properties to each other and to their mammalian orthologues (\u003cspan citationid=\"cr27\" class=\"citationref\"\u003e27\u003c/span\u003e, \u003cspan citationid=\"cr28\" class=\"citationref\"\u003e28\u003c/span\u003e). nevertheless, it remains unclear how well the human nmdar genes, including positions with identified missense variants, are conserved in zebrafish. to address this, we investigated nmdar conservation in zebrafish by taking advantage of amino acid sequence alignments (\u003cspan citationid=\"cr28\" class=\"citationref\"\u003e28\u003c/span\u003e) and assessed the conservation of all human nmdar subunits and zebrafish paralogues. we also compared the conservation of positions with missense variants identified in human patients for nmdar subunits expressed during early neurodevelopment. finally, we investigated the conservation of critical functional regions within each structural domain. through this analysis, we define the extent to which critical functional components and regions, with relevance to disease, are conserved across zebrafish nmdars, providing a template to use zebrafish as a model to study nmdars in neurodevelopment and disease pathology.\u003c/p\u003e"},{"header":"methods","content":"\u003cp\u003e\u003cb\u003enmdar sequences and sequence alignments\u003c/b\u003e\u003c/p\u003e\u003cp\u003eamino acid sequences for human genes are (uniprot with protein ids): glun1 (q05586-5; ntd splice isoform, nr1-3), glun2a (q12879-1), glun2b (q13224), glun2c (q14957), glun2d (o15399), glun3a (q8tcu5), and glun3b (o60391).\u003c/p\u003e\u003cp\u003efor mouse subunits: mglun1 (p35438), mglun2a (p35436), mglun2b (q01097), mglun2c (q01098), mglun2d (q03391), mglun2a (a2air5), and mglun3b (q91zu9).\u003c/p\u003e\u003cp\u003efor zebrafish paralogs: zglun1a (f1r366), zglun1b (q6zm67), zglun2aa (a0a8m9pxd6), zglun2ab (f1qde5), zglun2ba (a0a8m9q0i1), zglun2bb (a0a8m1nh89), zglun2ca (e7fh62), zglun2cb (a0a8m3b7k5), zglun2da (i3ni77), zglun2db (e7f3z4), zglun3a (a0a8m3awh5), zglun3ba (a0a8m9p9f6), and zglun3bb (a0a8m6z151).\u003c/p\u003e\u003cp\u003ebreakpoints for nmdar subunit domains and subdomains are based on structures 6whr (rat glun1-glun2b) and 7eu7 (human glun1-glun2a) (\u003cspan citationid=\"cr49\" class=\"citationref\"\u003e49\u003c/span\u003e, \u003cspan citationid=\"cr50\" class=\"citationref\"\u003e50\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ehuman and zebrafish protein sequences were aligned and analyzed for amino acid identity and similarity using align sequences protein blast (blastp), which is publicly available through the national center for biotechnology information (ncbi) or emboss needle pairwise sequence alignment (psa). for each nmdar subunit, the zebrafish paralogs were individually aligned to the human protein. amino acid similarity is defined based on the blosum 62 substitution matrix. for the ntd, lbd, and tmd, these alignments were generally straightforward as the sequences do not diverge greatly in length. in contrast, the ctd is quite large, with the zebrafish sequences generally larger than human. for these alignments, we entered the entire sequence into the alignment tool and used the given result, though this led to a fragmentation of the ctd such that certain portions of the sequence were aligned while others were not. we used the resulting domain-specific conservation values to calculate the conservation of the total protein.\u003c/p\u003e\u003cp\u003e\u003cb\u003evariants analysis\u003c/b\u003e\u003c/p\u003e\u003cp\u003ehuman nmdar variants were identified using clinvar and gnomad (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003eclinvar\u003c/em\u003e. results for each gene were downloaded on september 28th, 2023, using clinvar's bulk download option (\u003cspan class=\"externalref\"\u003e\u003cspan class=\"refsource\"\u003ehttps://www.ncbi.nlm.nih.gov/clinvar/\u003c/span\u003e\u003cspan address=\"https://www.ncbi.nlm.nih.gov/clinvar/\" targettype=\"url\" class=\"reftarget\"\u003e\u003c/span\u003e\u003c/span\u003e). genomic variants that did not result in protein coding changes were removed from the analysis. in the case of \u003cem\u003egrin3a\u003c/em\u003e, protein coding changes in the ppp3r2 gene, a single exon gene whose coding sequence resides within the genomic region of \u003cem\u003egrin3a\u003c/em\u003e and is downloaded alongside \u003cem\u003egrin3a\u003c/em\u003e due to this, was also removed. in cases where multiple protein coding annotations exist for the same variant, only the annotation fitting the canonical transcript was retained.\u003c/p\u003e\u003cp\u003e\u003cem\u003egnomad\u003c/em\u003e: genes were searched for by name on gnomad v4.0.0 (\u003cspan class=\"externalref\"\u003e\u003cspan class=\"refsource\"\u003ehttps://gnomad.broadinstitute.org/\u003c/span\u003e\u003cspan address=\"https://gnomad.broadinstitute.org/\" targettype=\"url\" class=\"reftarget\"\u003e\u003c/span\u003e\u003c/span\u003e) and missense, insertion, deletion, and loss-of-function variant information was exported on september 28th, 2023 as comma separated value (csv) files for each gene. hgvs annotation (protein change) was converted to letter-number-letter annotation.\u003c/p\u003e\u003cp\u003egene variants from clinvar and gnomad were merged using r studio 2023.06.1 build 524 based on the letter-number-letter annotation. the resulting merged file was checked for redundant information or variant duplicates, which were removed.\u003c/p\u003e\u003cp\u003emissense variants were extracted and grouped by pathogenicity as pathogenic, putatively pathogenic, complex, putatively benign, and benign variants based on the criteria in table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e. assessment of conservation of positions with identified human variants was performed using the generated sequence alignments.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar domain-specific analysis\u003c/b\u003e\u003c/p\u003e\u003cp\u003eto further refine the relationship between human and zebrafish nmdar proteins, we analyzed the tmd, lbd-tmd linkers, lbd, ntd, and ctd of glun1, glun2a, glun2b, and glun2d for conservation of key structural and functional features.\u003c/p\u003e\u003cp\u003e\u003cem\u003etransmembrane domain (tmd) and ligand-binding domain-transmembrane domain (lbd-tmd) linkers.\u003c/em\u003e key regions involved in channel gating, mg\u003csup\u003e2+\u003c/sup\u003e block, and ca\u003csup\u003e2+\u003c/sup\u003e flux were identified as described in the literature and analyzed for conservation (\u003cspan additionalcitationids=\"cr52 cr53 cr54 cr55 cr56\" citationid=\"cr51\" class=\"citationref\"\u003e51\u003c/span\u003e–\u003cspan citationid=\"cr57\" class=\"citationref\"\u003e57\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003eligand-binding domain (lbd)\u003c/em\u003e. key regions and specific amino acids involved in van-der-waals interactions with the α-carbon of the ligand, interactions with the α-amino group, electrostatic interactions with the α-carboxyl group, and interactions with the amino acid side chain were identified as described in ramos-vicente et. al. 2021 and analyzed for conservation (\u003cspan citationid=\"cr47\" class=\"citationref\"\u003e47\u003c/span\u003e).\u003c/p\u003e\u003cp\u003e\u003cem\u003en-terminal domain (ntd)\u003c/em\u003e. key amino acid positions required for the binding of zinc and polyamines were identified via a literature search and their conservation assessed (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr58\" class=\"citationref\"\u003e58\u003c/span\u003e, \u003cspan citationid=\"cr59\" class=\"citationref\"\u003e59\u003c/span\u003e). we also examined the conservation of putative n-linked glycosylation sites (\u003cspan citationid=\"cr60\" class=\"citationref\"\u003e60\u003c/span\u003e). to generate homology models of the zebrafish ntds, we used the template function in swiss-model (\u003cspan citationid=\"cr61\" class=\"citationref\"\u003e61\u003c/span\u003e, \u003cspan citationid=\"cr62\" class=\"citationref\"\u003e62\u003c/span\u003e). as reference structures, we used rodent ntd structures 5tq0 (glun1-glun2a in the presence of edta, no bound zn\u003csup\u003e2+\u003c/sup\u003e for glun1 and glun2a), 5tpz (glun1-glun2b apo state for glun1 and glun2b), and 5tpw (glun1-glun2a with zn\u003csup\u003e2+\u003c/sup\u003e in complex with glun2a) (\u003cspan citationid=\"cr58\" class=\"citationref\"\u003e58\u003c/span\u003e). sequences for human (hglun1, hglun2a, and hglun2b) or zebrafish (zglun1a, zglun1b, glun2aa, glun2ab, glun2ba, glun2bb) ntds were entered individually into swiss-model to generate homology models. the subsequent models were then loaded into pymole (human and zebrafish paralog) and aligned. the resultant root mean square deviation (rmsd) is reported.\u003c/p\u003e\u003cp\u003e\u003cem\u003ecarboxy-terminal domain (ctd)\u003c/em\u003e. ctd-specific conservation in zebrafish was assessed for the glun1, glun2a, and glun2b subunits. glun2d was excluded from this analysis due to the poor conservation of the ctd in these paralogs relative to human. to assess functional relevance, we focused on post-translational modifications (ptms), specifically phosphorylation sites (serine, threonine, tyrosine), ubiquitylation sites (lysine), and palmitoylation sites (cysteine), using a combination of experimentally validated data from phosphositeplus (psp) and published literature (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e). phosphorylation sites with identified kinases or well-characterized functional roles were classified as ‘kinase-regulated phosphorylation sites.’ the remaining phosphorylation sites identified in psp were defined as ‘phosphoproteomic-identified sites.’ note, sites in psp (based on rodents) that did not have a phosphorylation site in humans were excluded from our analysis. in addition, known linear protein-binding motifs were annotated and analyzed for conservation (\u003cspan citationid=\"cr2\" class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e). to assess conservation of protein-binding motifs in the ctds, we categorized sites as either short linear motifs (slims) or broader interaction ‘regions’. slims were defined as experimentally validated, compact sequence elements (typically \u0026lt; 15 residues) that mediate specific and direct interactions with known protein partners. broader regions were defined as extended sequences (\u0026gt; 15 residues) shown to directly bind proteins in biochemical or structural studies but lacking a discrete consensus sequence. all motifs were curated from published literature where discrete sequences were experimentally validated per motif (\u003cspan citationid=\"cr11\" class=\"citationref\"\u003e11\u003c/span\u003e, \u003cspan additionalcitationids=\"cr64 cr65 cr66 cr67 cr68 cr69 cr70 cr71 cr72 cr73 cr74 cr75 cr76 cr77 cr78 cr79\" citationid=\"cr63\" class=\"citationref\"\u003e63\u003c/span\u003e–\u003cspan citationid=\"cr80\" class=\"citationref\"\u003e80\u003c/span\u003e), and sequence conservation was examined.\u003c/p\u003e"},{"header":"results","content":"\u003cp\u003eour goal is to compare the structural similarity of zebrafish nmdar (zglun) paralogues to human nmdars (glun), especially regarding human variants. since mice are a common animal model (\u003cspan class=\"citationref\"\u003e81\u003c/span\u003e), as a reference-point for nmdar gene conservation, we initially made global comparisons of both mouse and zebrafish nmdar subunits to human.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eglobal comparison of mouse and zebrafish orthologues to human nmdars\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003enmdars are heterotetramers composed of two obligate glycine-binding glun1 subunits and two glutamate-binding glun2 subunits or a combination of glun2 and glycine-binding glun3 subunits (\u003cspan class=\"citationref\"\u003e1\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e2\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ea,b). glycine and glutamate act as co-agonists for glun2-containing receptors, with binding of both required for receptor activation, while glun3-containing receptors are activated by glycine binding. as an initial assessment of the structural similarity between nmdar orthologues, we made global sequence alignments using the ncbi blastp or emboss psa tools. not surprisingly, given evolutionary proximity, the mouse subunits are highly conserved relative to human: glun1 (97% identity \u0026amp; similarity), glun2a (95% identity; 98% similarity), glun2b (98% identity; 99% similarity), glun2c (88% identity; 90% similarity), glun2d (97% identity \u0026amp; similarity), glun3a (92% identity; 95% similarity), and glun3b (77% identity; 83% similarity) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003eon the other hand, the zebrafish paralogues show reduced conservation. the obligatory glun1 subunits are the most highly conserved, both in terms of identity (\u0026ge;\u0026thinsp;84%) and similarity (\u0026ge;\u0026thinsp;89%) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the other subunits show various degrees of identity and similarity: glun2a (\u0026ge;\u0026thinsp;58% identity; \u0026ge;79% similarity), glun2b (\u0026ge;\u0026thinsp;65% identity; \u0026ge;73% similarity), glun2c (\u0026ge;\u0026thinsp;50% identity; \u0026ge;64% similarity), glun2d (\u0026ge;\u0026thinsp;43% identity; \u0026ge;54% similarity), glun3a (63% identity; 78% similarity), and glun3b (\u0026ge;\u0026thinsp;54% identity, \u0026ge;\u0026thinsp;62% similarity) (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). these subunits, thus, retain\u0026thinsp;\u0026ge;\u0026thinsp;50% identity with respect to the human protein, except for zglun2da and zglun2db, which only share 43% and 46% sequence identity, respectively (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). with respect to the zebrafish paralog pairs, there are instances in which one paralog is better conserved than the other. the glun1 paralogs are comparably conserved (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). however, the zglun2aa paralog is more highly conserved than zglun2ab while the zglun2bb shows increased conservation relative to zglun2ba (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the glun2c, glun2d, and glun3b paralogs are conserved to similar extents (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003enevertheless, despite lower conservation relative to mouse, zebrafish offer many technical advantages (see background). in addition, what is critical is the conservation of specific structural and functional domains and subdomains.\u003c/p\u003e\n\u003cdiv align=\"left\" class=\"colspec\"\u003e\u003cbr\u003e\u003c/div\u003e\n\u003ctable id=\"tab1\" border=\"1\"\u003e\n \u003ccaption language=\"en\"\u003e\n \u003cdiv class=\"captionnumber\"\u003etable 1\u003c/div\u003e\n \u003cdiv class=\"captioncontent\"\u003e\n \u003cp\u003eoverall protein conservation in mouse and zebrafish relative to human\u003c/p\u003e\n \u003c/div\u003e\n \u003c/caption\u003e\n \u003cthead\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003ehuman\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003emouse\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003ezebrafish\u003c/p\u003e\n \u003cp\u003eparalogue a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\" colspan=\"3\"\u003e\n \u003cp\u003ezebrafish\u003c/p\u003e\n \u003cp\u003eparalogue b\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eid.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003esim.\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eprotein\u003c/p\u003e\n \u003cp\u003elength\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003c/thead\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en1\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e938\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e938\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e84%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e89%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e966\u003c/p\u003e\n \u003cp\u003e(-28)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e86%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e90%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e937\u003c/p\u003e\n \u003cp\u003e(-1)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2a\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1464\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e95%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e98%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1464\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e68%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e89%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1460\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;6)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e58%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e79%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1445\u003c/p\u003e\n \u003cp\u003e(-9)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2b\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1484\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e98%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e99%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1482\u003c/p\u003e\n \u003cp\u003e(-2)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e65%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e73%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1417\u003c/p\u003e\n \u003cp\u003e(-31)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e67%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e81%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1770 (+\u0026thinsp;322)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2c\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1233\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e88%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e90%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1239\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;6)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e50%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e64%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1400\u003c/p\u003e\n \u003cp\u003e(-176)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e59%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e79%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1328\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;95)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en2d\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1336\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e97%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1323\u003c/p\u003e\n \u003cp\u003e(-13)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e43%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e54%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1916 (+\u0026thinsp;580)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e46%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e57%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1676 (+\u0026thinsp;340)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en3a\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1115\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e92%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e95%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1135\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;20)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e63%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e78%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1111\u003c/p\u003e\n \u003cp\u003e(-4)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003en3b\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1043\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e77%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e83%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1003\u003c/p\u003e\n \u003cp\u003e(-40)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e54%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e62%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1118\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;75)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\u0026nbsp;\u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e62%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e68%\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e1114\u003c/p\u003e\n \u003cp\u003e(+\u0026thinsp;71)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003c/p\u003e\n\u003cp\u003enumbers indicate the percent of mouse or zebrafish paralog conservation with respect to the human protein \u0026ndash; \u0026lsquo;id.\u0026rsquo; is identity and \u0026lsquo;sim.\u0026rsquo; is similarity. the zebrafish a and b paralogs are indicated for each protein. glun3a has only a single paralog in zebrafish. for the mouse and zebrafish protein length, numbers in parentheses indicate the difference in amino acid number relative to human.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of specific structural elements in zebrafish paralogues\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003efor more rigorous comparisons, we examined the conservation within individual subunit domains and subdomains (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the structure of nmdars, like all iglurs, consists of 4 domains that are intrinsic to each individual subunit (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ea,b): the extracellular ntd and lbd; the tmd forming the ion channel as well as the linkers that connect the lbd to the tmd, the lbd-tmd linkers; and the intracellular ctd (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e53\u003c/span\u003e). within individual subunits, the lbd is formed by two disparate sequences, s1 and s2, that fold up into the d1 and d2 lobes in the 3-dimensional structure (83, 84). in general, d1 is made up primarily of s1 and d2 of s2, but there is considerable swapping between lobes. the tmd is formed by three transmembrane segments, m1, m3, and m4, and an m2-pore loop. the lbd-tmd linkers are referred to as either s1-m1, m3-s2, and s2-m4 (linear sequence) or d2-m1, m3-d2, and d1-m4 (3-dimensional structure). this distinction is important functionally since d1 and d2 undergo different movements that are translated differently to the tmd (\u003cspan class=\"citationref\"\u003e85\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e86\u003c/span\u003e). nevertheless, we will use the s1-m1, m3-s2, and s2-m4 designation for ease of comparison to structures.\u003c/p\u003e\n\u003cp\u003ewe began by dividing the zebrafish paralogues into their respective domains and subdomains based on our breakpoints (see methods). interestingly, the amino acid sequences surrounding these breakpoints were well conserved in zebrafish \u0026ndash; the majority were identical (not shown). this allowed for ease of division of the zebrafish proteins into their respective domains and subdomains.\u003c/p\u003e\n\u003cp\u003eas an initial assessment of the structural similarity between human and zebrafish nmdars, we made detailed sequence alignments using the ncbi blastp or emboss psa tools (table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e, fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). in terms of length of domains, the zebrafish protein sequence for glun1, glun2, and glun3 subunits was identical to the human sequence for the tmd (m1, m1-m2, m2, m2-m3, m3, \u0026amp; m4) and lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec). the lbd (s1 \u0026amp; s2) was also quite similar in length, with no changes in the length of s2 across the paralogs and only slight increases or decreases in amino acid number in the s1 of zglun1a, zglun1b, zglun2ab, zglun2ca, zglun2ca, zglun2da, and zglun2db (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec). the ntd and ctd were less conserved in terms of size, wherein almost every zebrafish paralog has a different number of amino acids compared to the human. this is especially notable for the glun2b ntd which either lacks 130 amino acids (zglun2ba) or has an additional 74 amino acids (zglun2bb) as well as for the ctd where, for example, zglun2da and zglun2db have a ctd that is more than twice as long as that in the human protein (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ec).\u003c/p\u003e\n\u003ctable id=\"tab2\" border=\"1\"\u003e\n \u003ccaption language=\"en\"\u003e\n \u003cdiv class=\"captionnumber\"\u003etable 2\u003c/div\u003e\n \u003cdiv class=\"captioncontent\"\u003e\n \u003cp\u003evariant designation\u003c/p\u003e\n \u003c/div\u003e\n \u003c/caption\u003e\n \u003cthead\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eglun\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003cth align=\"left\"\u003e\u0026nbsp;\u003c/th\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003edesignation\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003eclinvar\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003egnomad\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e1\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2b\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2c\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e2d\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e3a\u003c/p\u003e\n \u003c/th\u003e\n \u003cth align=\"left\"\u003e\n \u003cp\u003e3b\u003c/p\u003e\n \u003c/th\u003e\n \u003c/tr\u003e\n \u003c/thead\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epathogenic\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;pathogenic\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e49\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e58\u003c/p\u003e\n \u003cp\u003e(4%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e81\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e12\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eputatively\u003c/p\u003e\n \u003cp\u003epathogenic\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;uncertain\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e123\u003c/p\u003e\n \u003cp\u003e(13%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e214\u003c/p\u003e\n \u003cp\u003e(15%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e182\u003c/p\u003e\n \u003cp\u003e(12%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e4\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e111\u003c/p\u003e\n \u003cp\u003e(8%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e5\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e8\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003ecomplex\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;pathogenic\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e3\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e5\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e6\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e1\u003c/p\u003e\n \u003cp\u003e(\u0026lt;\u0026thinsp;1%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e0\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eputatively benign\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;uncertain\u0026rdquo;\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e80\u003c/p\u003e\n \u003cp\u003e(9%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e320\u003c/p\u003e\n \u003cp\u003e(22%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e199\u003c/p\u003e\n \u003cp\u003e(13%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e31\u003c/p\u003e\n \u003cp\u003e(3%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e223\u003c/p\u003e\n \u003cp\u003e(17%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e30\u003c/p\u003e\n \u003cp\u003e(3%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e68\u003c/p\u003e\n \u003cp\u003e(7%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003ebenign\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003e\u0026ldquo;benign\u0026rdquo;\u003c/p\u003e\n \u003cp\u003eor\u003c/p\u003e\n \u003cp\u003eabsent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"left\"\u003e\n \u003cp\u003eabsent/\u003c/p\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003cp\u003eor\u003c/p\u003e\n \u003cp\u003epresent\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e401\u003c/p\u003e\n \u003cp\u003e(43%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e813\u003c/p\u003e\n \u003cp\u003e(55%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e672\u003c/p\u003e\n \u003cp\u003e(45%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e945\u003c/p\u003e\n \u003cp\u003e(77%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e863\u003c/p\u003e\n \u003cp\u003e(27%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e59\u003c/p\u003e\n \u003cp\u003e(5%)\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd align=\"char\"\u003e\n \u003cp\u003e945\u003c/p\u003e\n \u003cp\u003e(91%)\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003c/p\u003e\n\u003cp\u003enumber of positions with missense variants that have been identified in each subunit per designation. percentages report the proportion of the total protein amino acids at which such variants have been identified.\u003c/p\u003e\n\u003cp\u003ewe used these sequence alignments to assess the degree of amino acid identity and similarity between domains and subdomains (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ed,e). the obligatory glun1 subunit showed the highest degree of conservation. the tmd and lbd-tmd linkers are 100% conserved. the lbd is also well conserved with s1 and s2 showing\u0026thinsp;\u0026ge;\u0026thinsp;92% identity and \u0026ge;\u0026thinsp;93% similarity. the ntd is somewhat less conserved (\u0026ge;\u0026thinsp;80% identity, \u0026ge;\u0026thinsp;88% similarity), as is the ctd (\u0026ge;\u0026thinsp;53% identity; \u0026ge;58% similarity).\u003c/p\u003e\n\u003cp\u003ethis trend of conservation across nmdar subunits \u0026ndash; the lbd and tmd are most highly conserved and the ntd and ctd less so \u0026ndash; is maintained in the glun2 and glun3 subunits (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003ed,e). generally, the glun2 subunits maintain a higher sequence identity than the glun3 subunits. overall, this sequence analysis suggests that the lbd and tmd are highly conserved whereas the ntd and ctd show less conservation.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecategorizing the pathogenicity of missense variants\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003enmdars with missense, frameshift, and nonsense variants have been identified in pediatric patients with neurodevelopmental disorders (\u003cspan class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e22\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e87\u003c/span\u003e). we therefore wanted to assess the extent to which variants are conserved in zebrafish to aid in guiding future studies. for this analysis, we focused solely on missense variants. while nonsense and frameshift mutation can be modelled in various ways, such as by mutating a portion of the protein to induce a loss-of-function, the conservation of the specific protein sequence is somewhat less relevant. in contrast, the single amino acid change associated with missense variants is critical to dissecting how such variants alter receptor function and lead to disease phenotypes.\u003c/p\u003e\n\u003cp\u003eusing publicly available datasets of nmdar variants from clinvar and gnomad, we identified missense variants in all nmdar subunits (see methods). we first categorized these variants with regards to their pathogenicity (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). clinvar reports variants that have been identified in patients with symptomatic disease (\u003cspan class=\"citationref\"\u003e88\u003c/span\u003e\u0026ndash;\u003cspan class=\"citationref\"\u003e90\u003c/span\u003e), while gnomad reports variants identified in the healthy population (\u003cspan class=\"citationref\"\u003e91\u003c/span\u003e). thus, in general, we define variants present in clinvar but absent in gnomad as pathogenic while those with the opposite reporting as benign. with this approach, pathogenic and putatively pathogenic variants were identified in glun1, glun2a, glun2b, and glun2d while minimal to no such variants were identified in glun2c, glun3a, and glun3b (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). notably, these genes with known pathogenic variants are expressed early in development, contributing to developmental processes while their dysfunction often leads to disease phenotypes (\u003cspan class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e87\u003c/span\u003e). on the other hand, glun2c, glun3a, and glun3b have no pathogenic and a small number of putatively pathogenic variants (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). we therefore focus solely on glun1, glun2a, glun2b, and glun2d for further analysis.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003epathogenic and putatively pathogenic variant positions are well conserved in zebrafish\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003eusing the criteria in table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e, amino acid positions with pathogenic missense variants in glun1, glun2a, glun2b, and glun2d are completely conserved in zebrafish paralogues (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ea-d).\u003c/p\u003e\n\u003cp\u003eputatively pathogenic variants are also conserved in these nmdar subunits, though less so than pathogenic variants. the highest degree of conservation of putatively pathogenic variants is in the glun1 paralogues (\u0026ge;\u0026thinsp;88% identity; 93% similarity). putatively pathogenic variants in the glun2 subunits are also reasonably conserved: zglun2aa (70% identity; 92% similarity), zglun2ab (62% identity; 84% similarity), zglun2ba (65% identity; 71% similarity), zglun2bb (72% identity; 80% similarity), and zglun2da and zglundb (\u0026ge;\u0026thinsp;46% identity; \u0026ge;59% similarity) (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ee,f). in general, the highest level of conservation is in the tmd and tmd-lbd linkers, followed by the lbd, and less so in the ntd and ctd (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003eg,h).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ebenign and putatively benign variant positions show reduced conservation\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ewe carried out a comparable analysis as above for benign and putatively benign variants (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e). we hypothesized that positions with more benign variants would be less well conserved than pathogenic ones. indeed, the general trend is consistent with this idea (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003e). putatively benign variants showed a reduced conservation compared to putatively pathogenic variants (putatively pathogenic vs putatively benign): zglun1a (88% vs. 78% identity; 93% vs. 88% similarity), zglun1b (89% vs. 79% identity; 93% vs. 84% similarity), zglun2aa (70% vs. 60% identity; 92% vs. 86% similarity), zglun2ab (62% vs. 48% identity; 84% vs. 68% similarity), zglun2ba (65% vs. 45% identity; 71% vs. 55% similarity), zglun2bb (72% vs. 55% identity; 80% vs. 70% similarity), zglun2da (46% vs. 47% identity; 59% vs. 61% similarity), and zglun2db (47% vs. 44% identity; 61% vs. 56% similarity) (fig. \u003cspan class=\"internalref\"\u003e2\u003c/span\u003ee,f vs fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ea,b). for these putatively benign variants, the greatest extent of conservation tends to occur in the tmd and lbd-tmd linkers, followed by the lbd, and then the ntd and ctd (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ec,d).\u003c/p\u003e\n\u003cp\u003efor benign variants, however, the decrease in conservation plateaus, with identity and similarity values comparable to the putatively benign variants, if not slightly higher (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ee-f). the subdomain-specific conservation trend for these variants follows that seen on the level of the whole receptor (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e), wherein the tmd, lbd-tmd linkers, and lbd are generally more conserved and the ntd and ctd less so (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003eg,h). although diverging from our prediction as we do not note a substantial decrease in the conservation of benign variants with respect to the putatively benign variants, this result reflects the overall conservation of the nmdar subunits themselves (benign vs total protein): zglun1 (78% vs. \u0026ge;84% identity; 84% vs. \u0026ge;89% similarity), zglun2aa (62% vs. 68% identity; 86% vs. 89% similarity), zglun2ab (49% vs. 58% identity; 74% vs. 79% similarity), zglun2ba (54% vs. 65% identity; 64% vs. 73% similarity), and zglun2bb (62% vs. 67% identity; 74% vs. 81% similarity), zglun2da (50% vs. 43% identity; 64% vs. 54% similarity), and zglun2db (49% vs. 46% identity; 62% vs. 57% similarity) (fig. \u003cspan class=\"internalref\"\u003e3\u003c/span\u003ee-f, table \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). furthermore, this result reflects the relatively large number of benign variants in comparison to other variant groups (table \u003cspan class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecritical tmd and lbd-tmd linker components are completely conserved\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003eelements forming the ion channel \u0026ndash; the tmd (m1, m1-m2, m2, m2-m3, m3, \u0026amp; m4) \u0026ndash; and the lbd-tmd linkers (s1-m1, m3-s2, \u0026amp; s2-m4) were consistently highly conserved (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). nevertheless, we assayed conservation of critical functional motifs. the s1-m1 linker is a critical element for channel gating and is completely conserved (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e) (\u003cspan class=\"citationref\"\u003e56\u003c/span\u003e). the m2 subdomain contains what is known as the \u0026lsquo;n\u0026rsquo; and \u0026lsquo;n\u0026thinsp;+\u0026thinsp;1\u0026rsquo; sites, which participate in mg\u003csup\u003e2+\u003c/sup\u003e block and ca\u003csup\u003e2+\u003c/sup\u003e permeation through the channel (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e55\u003c/span\u003e) and are conserved in zebrafish (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, dark orange). the tmd m3 segment contains the sytanlaaf motif, which plays a key role in ion channel gating and is known to be highly conserved across species. this motif is conserved in all zebrafish glun1 and glun2 paralogs (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, orange). lastly, the \u0026lsquo;drpeer\u0026rsquo; motif in m3-s2 of glun1, an important component in the high ca\u003csup\u003e2+\u003c/sup\u003e permeability of nmdars (\u003cspan class=\"citationref\"\u003e52\u003c/span\u003e) is conserved (fig. \u003cspan class=\"internalref\"\u003es1\u003c/span\u003e, light orange). hence, core gating and permeation elements of the tmd and associated regions are highly conserved suggesting that basic mechanisms of ion channel gating and block/permeation are preserved.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003elbd elements required for agonist binding are completely conserved\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ethe glun1 lbd binds glycine or d-serine while the glun2 lbd binds glutamate or aspartate (fig. s2a,b). overall, the lbds are highly conserved in zebrafish relative to humans (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e), as are specific amino acid side chains in s1 and s2 that are involved in binding endogenous agonists (\u003cspan class=\"citationref\"\u003e47\u003c/span\u003e) (fig. s2c). these include sites that participate in van-der waals interactions with the \u0026alpha;-carbon of the ligand (darkest red), interactions with the \u0026alpha;-amino group (dark red), electrostatic interactions with the \u0026alpha;-carboxyl group (red), and interactions with the amino acid side chain (light red) (fig. s2c).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of structure-function elements in the ntd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ein terms of overall sequence comparisons, the ntd shows variable degrees of conservation across nmdar subunits. the glun1 zebrafish paralogs maintain the highest degree of conservation relative to human (80% identity; \u0026ge;88% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). the glun2 subunits show decreased ntd conservation (49\u0026ndash;75% identity; 59\u0026ndash;88% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). given this reduced conservation, we assayed the conservation of key structure-function motifs in zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003eamong its many roles, the ntd participates in interactions with ions and molecules that modulate receptor activity (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e59\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ea,b). this is most notable for glun2a and to a lesser extent glun2b as they bind zinc, which inhibits receptor function by reducing channel open probability (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). in glun2a, four amino acids coordinate zinc binding: glun2a-his44, glun2a-his128, glun2a-glu266, and glun2a-asp282 (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). of these key sites, three are conserved (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec) but in zglun2aa and zglun2ab, a key residue involved in zn\u003csup\u003e2+\u003c/sup\u003e binding, glun2a-his44, is not conserved (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec). for glun2b, only two amino acid residues participate in zinc binding: glun2b-his127 and glun2b-glu284 (\u003cspan class=\"citationref\"\u003e58\u003c/span\u003e). while glun2b-glu284 is conserved, the glun2b-his127 position in zglun2ba is missing due to alterations in ntd sequence length leading to gaps in the alignments, though it is similar now as a positively charged arginine instead of a histidine in zglun2bb (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ec).\u003c/p\u003e\n\u003cp\u003ethe ntd of glun2b also binds polyamines, which enhance receptor activity (\u003cspan class=\"citationref\"\u003e59\u003c/span\u003e). nine amino acids are involved in the binding of polyamines. these sites in zglun2ba and zglun2bb are completely conserved except for one amino acid in zglun2bb that is similar but not identical, now a negatively charged aspartate instead of glutamate (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ed). most of this functional component of glun2b is, thus, conserved in zebrafish.\u003c/p\u003e\n\u003cp\u003eadditionally, the nmdar ntd has glycosylation sites, and their glycosylation regulates the trafficking of nmdars to the cell membrane (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e). given the consensus sequence for arginine (n-) glycosylation (n-\u003cem\u003ex\u003c/em\u003e-s/t), glun1 has seven, glun2a has three, and glun2b has three putative glycosylation sites in the ntd (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e). their conservation in the zebrafish paralogues is somewhat variable: zglun1a \u0026amp; zglun1b (5 of 7 sites), zglun2aa (3 of 3 sites), zglun2ab (2 of 3 sites), zglun2ba (2 of 3 sites), and zglun2bb (3 of 3 sites) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ee). to the best of our knowledge, specific glycosylation sites in glun2d have yet to be identified. of note, the glun1 glycosylation sites glun1-n203 and glun1-n368 are necessary for receptor surface expression (\u003cspan class=\"citationref\"\u003e60\u003c/span\u003e), and these sites are conserved in the zebrafish paralogues.\u003c/p\u003e\n\u003cp\u003ewhile the critical functional features of the lbd were completely conserved in zebrafish relative to human nmdars, the conservation of key regions in the ntd occasionally diverged, most notably that for zn\u003csup\u003e2+\u003c/sup\u003e binding. to further explore the structural composition of the glun1, glun2a, and glun2b ntds, we used swiss-model to generate homology models for the human and zebrafish nmdar subunits, which we subsequently aligned in pymole to assess conservation at the level of the three-dimensional structure (see methods) (fig. \u003cspan class=\"internalref\"\u003e4\u003c/span\u003ef). in general, this analysis revealed a high degree of similarity between the human ntd and zebrafish paralogs. the zebrafish glun1 ntd models display similar overall topology to human: glun1 vs zglun1a, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.071); glun1 vs zglun1a, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.085); glun1 vs zglun1b, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.094); and glun1 vs zglun1b, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.097) (table s3).\u003c/p\u003e\n\u003cp\u003ethe glun2a and glun2b subunits also reflect a high degree of structural similarity: glun2a vs zglun2aa, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.085); glun2a vs zglun2ab, reference structure 5tp0 (rmsd\u0026thinsp;=\u0026thinsp;0.095); glun2a vs zglun2ab, reference structure 5tpw (rmsd\u0026thinsp;=\u0026thinsp;0.111); glun2b vs zglun2ba, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.028); and glun2b vs glun2bb, reference structure 5tpz (rmsd\u0026thinsp;=\u0026thinsp;0.044) (table s3). the only exception here is the structural comparison of glun2a and zglun2aa using reference structure 5tpw, in which the ntd is interacting with zinc (rmsd\u0026thinsp;=\u0026thinsp;2.149), suggesting that the structural components of zglun2aa may not effectively complex with zinc. interestingly, zglun2ba is \u0026gt;\u0026thinsp;100 amino acids shorter and zglun2bb is \u0026gt;\u0026thinsp;70 amino acids longer than human glun2b. though the zglun2ba model is missing an \u0026alpha;-helix relative to the human protein, the zglun2bb modelled structure aligns well with its human counterpart, suggesting that the zebrafish sequences maintain a large degree of structural integrity.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of post-translational modification (ptm) sites in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ethe intracellular ctd is a key regulatory domain of nmdar cell biology and function and shows distinct subunit-specific elements (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). for our analyses, we focused on the glun1-1 ctd splice variant, which is the longest variant containing the c0-c1-c2 cassettes (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003eacross the nmdar subunits, the zebrafish ctds are the most poorly conserved (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e). with a sequence identity generally around 50\u0026ndash;60% for the glun1, glun2a, and glun2b zebrafish paralogs, with some exceptions: zglun1a (53% identity; 58% similarity), zglun1b (54% identity; 59% similarity), zglun2aa (57% identity; 86% identity), zglun2ab (38% identity; 65% similarity), zglun2ba (57% identity, 66% similarity), and zglun2bb (60% identity, 71% similarity) while that of glun2d is much lower (\u0026ge;\u0026thinsp;18% identity, \u0026ge;\u0026thinsp;28% similarity) (fig. \u003cspan class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\n\u003cp\u003egiven these deviations in protein sequences, we first examined the conservation of ctd regions critical for post-translational modifications (ptms), excluding glun2d due its poor conservation in zebrafish. phosphorylation of nmdar subunit ctds confers key functional roles to the receptor (\u003cspan class=\"citationref\"\u003e2\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). initially, we examined experimentally validated sites at which phosphorylation occurs by known kinases and calculated the percent conservation, reported in identity and similarity. similarity, here, is defined as a site at the homologous position that can be phosphorylated (e.g., s, t, or y), if not identical. the glun1 ctd contains phosphorylation sites for src kinase (src), protein kinase c (pkc), and protein kinase a (pka), (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003etop\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). note specific sites have also been identified that are also targeted by the serine/threonine protein phosphatase 2b (pp2b). these ptm sites are perfectly conserved in zglun1a, while three of five are conserved in zglun1b (60% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb). the glun2a ctd is phosphorylated by src, pka, dual specificity tyrosine-phosphorylation-regulated kinase 1a (dyrk1), cyclin-dependent kinase 5 (cdk5), pkc, and calcium/calmodulin-dependent protein kinase ii\u0026alpha; (camkii\u0026alpha;) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003emiddle\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e): zglun2aa (79% identity \u0026amp; similarity) and zglun2ab (64% identity, 71% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb). the ctd of glun2b is phosphorylated by src, camkii\u0026alpha;, cdk5, pka, pkc, death-associated protein kinase 1 (dapk1), proto-oncogene tyrosine-protein kinase fyn (fyn), and casein kinase ii (ck2) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ea, \u003cem\u003ebottom\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). glun2b also contains sites for the protein tyrosine phosphatase non-receptor 11 (ptpn11, or shp2) and protein phosphatase 1 (pp1). the conservation of these ptms is slightly higher than that in glun2a: zglun2ba (73% identity \u0026amp; similarity) and zglun2bb (82% identity, 100% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb).\u003c/p\u003e\n\u003cp\u003ein addition to these kinase-regulated sites, we also examined the conservation of other known phosphorylation sites in these subunits identified by phosphoproteomics (psp) (see methods). the conservation of these sites in the zebrafish paralogs was generally lower than that of the kinase-regulated sites: glun1 (67% identity, 100% similarity), zglun2aa (51% identity, 66% similarity), zglun2ab (34% identity, 57% similarity), zglun2ba (60% identity \u0026amp; similarity), and zglun2bb (67% identity, 70% similarity) (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003ec).\u003c/p\u003e\n\u003cp\u003ewe next examined the conservation of ctd palmitoylation and ubiquitylation sites. glun1 has a single ubiquitylation site (lys860) (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003etop\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), and it is conserved in the zebrafish paralogs (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ec). glun2a contains seven palmitoylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003emiddle\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), which are likewise completely conserved in zebrafish (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003eb). the glun2a ctd also has four ubiquitylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003emiddle)\u003c/em\u003e (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e), of which three are conserved in zglun2aa (75%) and one in zglun2ab (25%) (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ec). with the same approach for glun2b, we identified eight palmitoylation and six ubiquitylation sites (fig. \u003cspan class=\"internalref\"\u003e6\u003c/span\u003ea, \u003cem\u003ebottom\u003c/em\u003e) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e). here, all palmitoylation sites are conserved in zebrafish while five of six ubiquitylation sites (83%) are conserved in zglun2ba and all six ubiquitylation sites (100%) are conserved in zglun2bb (fig. \u003cspan class=\"internalref\"\u003e5\u003c/span\u003eb,c).\u003c/p\u003e\n\u003cp\u003ein summary, the global conservation of ptm sites is as follows: zglun1a and zglun1b (89% identity, 100% similarity), zglun2aa (62% identity, 72% similarity), zglun2ab (45% identity, 62% similarity), zglun2ba (66% identity, 67% similarity), and zglun2bb (75% identity, 78% similarity).\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econservation of protein-binding motifs (pbm) in the ctd\u003c/strong\u003e\u003c/p\u003e\n\u003cp\u003ein addition to numerous ptm sites, a variety of protein-protein interactions are critical to the role of the ctd in receptor cell biology (surface expression, distribution) and function (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e63\u003c/span\u003e). these protein-binding elements include either defined short linear motifs (slims) or broader regions known to serve as protein docking sites.\u003c/p\u003e\n\u003cp\u003ethe ctd of glun1 contains a slim known as the kkk/rrr er retention motif, which is conserved in zebrafish (\u003cspan class=\"citationref\"\u003e66\u003c/span\u003e) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,b). it, likewise, has binding regions for calmodulin (cam)/calcium/calmodulin-dependent protein kinase ii (camkii) and \u0026alpha;-actinin 2 (\u003cspan class=\"citationref\"\u003e64\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e65\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e67\u003c/span\u003e, \u003cspan class=\"citationref\"\u003e78\u003c/span\u003e), which is conserved and yotiao, also known as a-kinase anchoring protein 9 (akap9) (\u003cspan class=\"citationref\"\u003e67\u003c/span\u003e), which is generally well conserved: zglun1a (90% identity, 98% similarity) and zglun1b (90% identity, 98% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,b).\u003c/p\u003e\n\u003cp\u003ethe glun2 subunit ctds contain a variety of pbms. for glun2a, the conservation of slims is: the yxx\u0026oslash; motif, where y is tyrosine, x is any amino acid, and \u0026oslash; is a bulky hydrophobic residue like leucine, isoleucine, phenylalanine, methionine, or valine (\u003cspan class=\"citationref\"\u003e73\u003c/span\u003e) \u0026ndash; both zglun2aa and zglun2ab show 100% identity \u0026amp; similarity; the guanine-nucleotide exchange factor brag2 (\u003cspan class=\"citationref\"\u003e70\u003c/span\u003e) \u0026ndash; zglun2aa (80% identity \u0026amp; similarity) and zglun2ab (40% identity, 60% similarity); postsynaptic density protein 95 (psd-95) (\u003cspan class=\"citationref\"\u003e72\u003c/span\u003e) \u0026ndash; zglun2aa (50% identity; 63% similarity) and zglun2ab (38% identity, 63% similarity); and the pdz domain that interacts with the proteins psd-95, discs large (dlg), and zonula occludens-1 (zo-1) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2aa and zglun2ab (100% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,c). interacting regions in glun2a include: ring finger protein 10 (rnf10) (\u003cspan class=\"citationref\"\u003e70\u003c/span\u003e) \u0026ndash; zglun2aa (51% identity, 72% similarity) and zglun2ab (32% identity, 66% similarity); rabphilin 3a (rph3a) (\u003cspan class=\"citationref\"\u003e68\u003c/span\u003e) \u0026ndash; zglun2aa (71% identity, 80% similarity) and zglun2ab (49% identity, 71% similarity); flotillin-1 (flot-1) (\u003cspan class=\"citationref\"\u003e71\u003c/span\u003e) \u0026ndash; zglun2aa (61% identity, 81% similarity) and zglun2ab (31% identity, 52% similarity); and c-terminal binding protein 1 (ctbp1) (\u003cspan class=\"citationref\"\u003e69\u003c/span\u003e) \u0026ndash; zglun2aa (51% identity, 85% similarity) and zglun2ab (36% identity, 63% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,c, s3a,b).\u003c/p\u003e\n\u003cp\u003efor glun2b, the conservation of slims is: the guanine-nucleotide exchange factor brag1 (\u003cspan class=\"citationref\"\u003e76\u003c/span\u003e) \u0026ndash; zglun2ba (60% identity; 80% similarity) and zglun2bb (60% identity, 100% similarity); psd-95 (\u003cspan class=\"citationref\"\u003e72\u003c/span\u003e) \u0026ndash; zglun2ba (56% identity \u0026amp; similarity) and zglun2bb (33% identity, 56% similarity); camkii and death-associated protein kinase 1 (dapk1) (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (92% identity, 100% similarity); synapse-associated protein 102 (sap102) (\u003cspan class=\"citationref\"\u003e75\u003c/span\u003e) \u0026ndash; zglun2ba (0% identity \u0026amp; similarity) and zglun2bb (50% identity \u0026amp; similarity); the adaptor complex ap2 (\u003cspan class=\"citationref\"\u003e74\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (100% identity \u0026amp; similarity); and pdz (\u003cspan class=\"citationref\"\u003e11\u003c/span\u003e) \u0026ndash; zglun2ba (100% identity \u0026amp; similarity) and zglun2bb (100% identity \u0026amp; similarity) (fig. \u003cspan class=\"internalref\"\u003e7\u003c/span\u003ea,d). the ctd interaction regions for glun2b are conserved as follows: receptor for activated c kinase 1 (rack1) (\u003cspan class=\"citationref\"\u003e77\u003c/span\u003e) \u0026ndash; zglun2ba (0% identity, 7% similarity) and zglun2bb (13% identity, 20% similarity); flot-1 (\u003cspan class=\"citationref\"\u003e71\u003c/span\u003e) \u0026ndash; zglun2ba (23% identity; 47% similarity) and zglun2bb (35% identity, 68% similarity); spectrin (\u003cspan class=\"citationref\"\u003e79\u003c/span\u003e) \u0026ndash; zglun2ba (40% identity; 59% similarity) and zglun2bb (35% identity, 62% similarity); \u0026alpha;-actinin 2 (\u003cspan class=\"citationref\"\u003e78\u003c/span\u003e) \u0026ndash; zglun2ba (47% identity; 67% similarity) and zglun2bb (51% identity, 76% similarity); and ras protein-specific guanine nucleotide-releasing factor 1 (rasgrf1) (\u003cspan class=\"citationref\"\u003e80\u003c/span\u003e) \u0026ndash; zglun2ba (59% identity; 74% similarity) and zglun2bb (60% identity, 78% similarity) (fig. \u003cspan class=\"internalref\"\u003e8\u003c/span\u003ea,d, s3a,c).\u003c/p\u003e\n\u003cp\u003ewhen compared globally across subunits, slims always show greater conservation than regions. this conservation is always greatest in the zebrafish glun1 paralogs, followed by the glun2b and then glun2a paralogs. for the analyzed slims, conservation grouped across each subunit is as follows: zglun1a and zglun1b (100% identity \u0026amp; similarity), zglun2aa (76% identity, 81% similarity), zglun2ab (62% identity, 76% similarity), zglun2ba (78% identity, 81% similarity), and zglun2bb (72% identity, 86% similarity). the regions show the following degrees of conservation: zglun1a and zglun1b (94% identity, 98% similarity), zglun2aa (55% identity, 81% similarity), zglun2ab (36% identity, 62% similarity), zglun2ba (59% identity, 74% similarity), and zglun2bb (60% identity, 78% similarity).\u003c/p\u003e"},{"header":"discussion","content":"\u003cp\u003ehere, we compared the conservation of the zebrafish nmdar subunit paralogues, including key structure-function motifs, to their human counterparts. we find that the core gating machinery \u0026ndash; the lbd, the lbd-tmd linkers and the tmd \u0026ndash; are highly conserved both in terms of overall identity and specific functional motifs. the more peripheral domains \u0026ndash; the ntd and ctd \u0026ndash; show less conservation overall, though functional motifs are reasonably conserved with the notable exception of zn\u003csup\u003e2+\u003c/sup\u003e binding in the glun2a ntd. notably, disease-associated variants are generally conserved, highlighting that zebrafish represent a useful model to study nmdars.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar conservation in zebrafish\u003c/b\u003e\u003c/p\u003e\u003cp\u003eacross most of our examined parameters, the obligatory glun1 subunit shows the highest degree of conservation in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e). generally, conservation is then greatest in the glun2(a-d) followed by the glun3(a-b) subunits. within each subunit, sequence conservation is consistently highest in the tmd and lbd-tmd linkers, followed by the lbd. conservation is generally poorer in the ntd and even worse in the ctd (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ea key question with regards to this work is defining what constitutes as a \u0026lsquo;well conserved\u0026rsquo; sequence. previous studies regarding the conservation of protein structure and function suggest that sequence identities between 40\u0026ndash;70% confer functional conservation (\u003cspan additionalcitationids=\"cr93\" citationid=\"cr92\" class=\"citationref\"\u003e92\u003c/span\u003e\u0026ndash;\u003cspan citationid=\"cr94\" class=\"citationref\"\u003e94\u003c/span\u003e). they regard 50% sequence identity as the threshold below which function drastically diverges (\u003cspan citationid=\"cr93\" class=\"citationref\"\u003e93\u003c/span\u003e, \u003cspan citationid=\"cr94\" class=\"citationref\"\u003e94\u003c/span\u003e). our sequence conservation analyses almost always led to sequence identities\u0026thinsp;\u0026gt;\u0026thinsp;50%, except for zglun2da. nevertheless, we carried out more detailed examinations of more poorly conserved domains, e.g., the ntd and ctd, to assess the conservation of key structure-function motifs.\u003c/p\u003e\u003cp\u003e\u003cb\u003enmdar missense variant conservation in zebrafish\u003c/b\u003e\u003c/p\u003e\u003cp\u003eto date, a multitude of variants, including missense, frameshift, and nonsense mutations, have been identified in the genes encoding the nmdar subunits (\u003cspan citationid=\"cr21\" class=\"citationref\"\u003e21\u003c/span\u003e, \u003cspan citationid=\"cr22\" class=\"citationref\"\u003e22\u003c/span\u003e, \u003cspan citationid=\"cr87\" class=\"citationref\"\u003e87\u003c/span\u003e). we categorized identified missense variants by pathogenicity and analyzed the extent of conservation of the positions at which they occur. we focused only on the glun1, glun2a, glun2b, and glun2d subunits for this analysis, as this is where most disease-associated variants have been identified. generally, our results demonstrate a higher degree of variant position conservation with increasing pathogenicity (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e,\u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e). the sites of pathogenic missense variants are almost perfectly conserved (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e). given the high conservation of the amino acid positions at which they occur, these protein regions are likely most essential for receptor function. across our categories of variants, with the exclusion of complex variants, the proportion of the total protein at which missense variants have been identified is lowest in the pathogenic group (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e). it is possible that additional variants that might be categorized as pathogenic are so devastating to development as to be embryonic lethal. this reflects the notion that more integral regions of the receptor are less tolerant to variants (\u003cspan citationid=\"cr95\" class=\"citationref\"\u003e95\u003c/span\u003e). zebrafish, nevertheless, emerge as a useful model for the study of pathogenic nmdar variants.\u003c/p\u003e\u003cp\u003eputatively pathogenic variants are conserved to a lesser extent than their pathogenic counterparts (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e). according to our designations, these variants have the potential to cause disease, having been identified in patients, but are currently classified as having uncertain significance. thus, it aligns that these variants are comparatively less well conserved \u0026ndash; they are likely occurring in less critical regions of the receptor and are likewise less likely to cause disease phenotypes.\u003c/p\u003e\u003cp\u003eour subsequent investigation of putatively benign and benign variants revealed a further decrease in variant position conservation. this finding further supported our prediction that variants more highly associated with disease occur at positions with higher degrees of conservation and, thus, are more critical to protein structure and function. the conservation of positions with putatively benign and benign variants is quite comparable. for the benign variants, in particular, conservation percentages also closely reflect total protein conservation values for each paralog (fig.\u0026nbsp;\u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e, table\u0026nbsp;\u003cspan refid=\"tab1\" class=\"internalref\"\u003e1\u003c/span\u003e). this follows from the finding that positions with identified benign variants, for most subunits, make up a substantial portion of the protein (table\u0026nbsp;\u003cspan refid=\"tab2\" class=\"internalref\"\u003e2\u003c/span\u003e).\u003c/p\u003e\u003cp\u003enevertheless, this investigation both demonstrates the advantage of the zebrafish model in studying nmdar-associated disease and reflects the notion that the positions most essential for receptor function are those that are best conserved and more likely to induce disease when absent.\u003c/p\u003e\u003cp\u003eoverall, the trend for variant conservation is the same as that for the whole protein. conservation is always greatest in the tmd and lbd-tmd linkers, followed by the lbd, next the ntd, and last the ctd (fig.\u0026nbsp;\u003cspan refid=\"fig4\" class=\"internalref\"\u003e2\u003c/span\u003e, \u003cspan refid=\"fig6\" class=\"internalref\"\u003e3\u003c/span\u003e). these findings suggest that zebrafish serve as an effective model for variants, especially pathogenic and putatively pathogenic variants in both the tmd and lbd-tmd linkers. some caution, however, must be exercised when using this model system for variants in the lbd, ntd, and ctd.\u003c/p\u003e\u003cp\u003e\u003cb\u003etmd, tmd-lbd linker, and lbd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003eour examination of key functional regions and motifs in the tmd, lbd-tmd linkers, and lbd consistently revealed complete conservation in their respective zebrafish paralogs (fig. \u003cspan refid=\"moesm1\" class=\"internalref\"\u003es1\u003c/span\u003e,s2). this suggests that the basic mechanism of ion channel gating is conserved in zebrafish relative to human. as these domains and subdomains contribute to the formation of the core of the receptor \u0026ndash; the ion channel pore allowing for ion flux \u0026ndash; it is expected that they would be most highly conserved across species, since the major functional role of nmdars is current flux during synaptic transmission.\u003c/p\u003e\u003cp\u003e\u003cb\u003entd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003ethe ntd is the first region in which nmdar sequences in zebrafish begin to diverge from those in human. modelling ntd variants in zebrafish must, thus, be approached with caution. for example, polyamine-binding sites in glun2b are almost completely conserved in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig8\" class=\"internalref\"\u003e4\u003c/span\u003ed), so they have the potential to serve as an effective model for this functional role of nmdars. on the other hand, zinc-binding sites in glun2a and glun2b are more poorly conserved in zebrafish (fig.\u0026nbsp;\u003cspan refid=\"fig8\" class=\"internalref\"\u003e4\u003c/span\u003ec), so they would likely serve as a poor model for zinc-related studies in nmdars. this also aligns with our homology model finding that the zglun2aa model aligns well with glun2a when not bound to zinc, but the rmsd of the alignment is substantially poorer when glun2a interacting with zinc is used as the reference structure (table s3).\u003c/p\u003e\u003cp\u003e\u003cb\u003ectd conservation\u003c/b\u003e\u003c/p\u003e\u003cp\u003ethe c-terminal domains (ctds) exhibit the lowest level of overall conservation across nmdar domains (fig.\u0026nbsp;\u003cspan refid=\"fig2\" class=\"internalref\"\u003e1\u003c/span\u003e). this is consistent with the ctd\u0026rsquo;s classification as an intrinsically disordered region (idr), which lacks fixed secondary or tertiary structure and is more tolerant to evolutionary divergence (\u003cspan citationid=\"cr96\" class=\"citationref\"\u003e96\u003c/span\u003e, \u003cspan citationid=\"cr97\" class=\"citationref\"\u003e97\u003c/span\u003e). nonetheless, functional conservation within idrs is often maintained through the retention of post-translational modification (ptm) hotspots and short linear motifs (slims) that regulate protein interactions and intracellular signaling (\u003cspan citationid=\"cr98\" class=\"citationref\"\u003e98\u003c/span\u003e).\u003c/p\u003e\u003cp\u003ewe assessed ctd conservation in glun1, glun2a, and glun2b by analyzing experimentally validated phosphorylation, palmitoylation, and ubiquitylation sites, as well as defined protein-binding motifs. phosphorylation sites with known kinase interactions exhibited relatively high conservation, consistent with their well-characterized regulatory roles (figs.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ea,b). in contrast, phosphorylation sites without identified kinases were less consistently retained (fig.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ec). palmitoylation sites were fully conserved across all subunits, while ubiquitylation sites were generally retained, though conservation was notably lower in zglun2ab (fig.\u0026nbsp;\u003cspan refid=\"fig10\" class=\"internalref\"\u003e5\u003c/span\u003ef) and may reflect paralogue-specific divergence in degradation or trafficking pathways.\u003c/p\u003e\u003cp\u003eprotein-binding motif conservation varied by subunit. glun1 retained nearly all canonical interaction motifs, while glun2a and glun2b displayed region-specific conservation. notably well-conserved elements in the glun2 subunits included the pdz-binding motifs as well as docking regions for camkii, dapk1, and ap2 (fig.\u0026nbsp;\u003cspan refid=\"fig12\" class=\"internalref\"\u003e6\u003c/span\u003e). slims overall showed higher conservation, likely due to their short, sequence-specific structure, which may place them under stronger evolutionary constraint than broader, more flexible interaction regions.\u003c/p\u003e\u003cp\u003edespite broad sequence divergence, nmdar ctds retain a core set of conserved motifs involved in receptor trafficking, synaptic localization, and plasticity. kinase-regulated phosphorylation sites, palmitoylation sites, and pdz-binding motifs were consistently preserved. conservation of glun1 sites was always greatest in the zebrafish paralogs. among the glun2 subunits, glun2b displayed the highest overall conservation of functional elements, aligning with its essential role in neurodevelopment and synaptic signaling.\u003c/p\u003e\u003cp\u003ethese findings are especially relevant to the study of disease-associated truncating variants in the nmdar subunits. nonsense variants that introduce premature stop codons in the ctd eliminate key regulatory motifs critical for trafficking, anchoring, or signaling. when these affected regions correspond to conserved functional elements, zebrafish may serve as an appropriate \u003cem\u003ein vitro\u003c/em\u003e and \u003cem\u003ein vivo\u003c/em\u003e model for studying ctd-mediated disease mechanisms. conversely, for mutations in poorly conserved regions, the translational utility of this model may become limited.\u003c/p\u003e"},{"header":"conclusions","content":"\u003cp\u003ezebrafish represent a valuable system to study nmdar-associated neurodevelopmental disorders, but one must be judicious in choosing which variants to study in this model. our unique approach of sequence alignments per individual nmdar domains and subdomains highlights the high degree of conversation of some regions of the protein relative to others. additionally, our analysis of identified variants indicates that zebrafish are an effective model for pathogenic variants as these occur at conserved locations along the protein. generally, the utility of zebrafish is most apparent for the study of the tmd, lbd-tmd linkers, and lbd as these regions are most well conserved. on the other hand, studying the ntd and ctd in this species requires extra consideration stemming from deviations in sequence conservation. our work will enable future nmdar-related studies that can be effectively and efficiently conducted in zebrafish.\u003c/p\u003e"},{"header":"abbreviations","content":"\u003ctable border=\"0\" cellspacing=\"0\" cellpadding=\"0\"\u003e\n \u003ctbody\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003enmdar\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003e\u003cem\u003en\u003c/em\u003e-methyl-d-aspartate receptor\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003entd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003en-terminal domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003elbd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003eligand-binding domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003etmd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003etransmembrane domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003ctr\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003ectd\u003c/p\u003e\n \u003c/td\u003e\n \u003ctd valign=\"top\" style=\"width: 312px;\"\u003e\n \u003cp\u003ec-terminal domain\u003c/p\u003e\n \u003c/td\u003e\n \u003c/tr\u003e\n \u003c/tbody\u003e\n\u003c/table\u003e"},{"header":"declarations","content":"\u003cp\u003e\u003cstrong\u003eethics approval and consent to participate:\u0026nbsp;\u003c/strong\u003enot applicable.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003econsent for publication:\u0026nbsp;\u003c/strong\u003enot applicable.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eavailability of data and materials:\u003c/strong\u003e the datasets used and/or analyzed during the current study are available either at osf (https://osf.io/h2zf9/) or from the corresponding author on reasonable request.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003ecompeting interests:\u0026nbsp;\u003c/strong\u003eno competing interests declared.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003efunding:\u003c/strong\u003e this work was supported by nih grant r01ns088479 to lpw.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eauthors\u0026rsquo; contributions:\u0026nbsp;\u003c/strong\u003ee.r.n., c.a., j.d.z., h.i.s., and l.p.w. contributed to the conceptualization and methodology of this project. e.r.n., c.a., and j.d.z. conducted the investigation and analysis. w.r. assisted with data curation. e.r.n, c.a., and l.p.w. wrote the manuscript, which was revised with insights from j.d.z. and h.i.s.\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eacknowledgements:\u0026nbsp;\u003c/strong\u003ethe authors thank the clinvar and gnomad databases and the swiss-model server for their important contributions to this work.\u003c/p\u003e"},{"header":"references","content":"\u003col\u003e\n\u003cli\u003epaoletti p, bellone c, zhou q. nmda receptor subunit diversity: impact on receptor properties, synaptic plasticity and disease. nat rev neurosci. 2013;14(6):383-400.\u003c/li\u003e\n\u003cli\u003ehansen kb, wollmuth lp, bowie d, furukawa h, menniti fs, sobolevsky ai, et al. structure, function, and pharmacology of glutamate receptor ion channels. pharmacol rev. 2021;73(4):298-487.\u003c/li\u003e\n\u003cli\u003echakraborty a, murphy s, coleman n. the role of nmda receptors in neural stem cell proliferation and differentiation. stem cells dev. 2017;26(11):798-807.\u003c/li\u003e\n\u003cli\u003enacher j, mcewen bs. the role of n-methyl-d-asparate receptors in neurogenesis. hippocampus. 2006;16(3):267-70.\u003c/li\u003e\n\u003cli\u003ekomuro h, rakic p. modulation of neuronal migration by nmda receptors. science. 1993;260(5104):95-7.\u003c/li\u003e\n\u003cli\u003epearce ia, cambray-deakin ma, burgoyne rd. glutamate acting on nmda receptors stimulates neurite outgrowth from cerebellar granule cells. febs lett. 1987;223(1):143-7.\u003c/li\u003e\n\u003cli\u003ebooker sa, wyllie dja. nmda receptor function in inhibitory neurons. neuropharmacology. 2021;196:108609.\u003c/li\u003e\n\u003cli\u003egambrill ac, barria a. nmda receptor subunit composition controls synaptogenesis and synapse stabilization. proc natl acad sci u s a. 2011;108(14):5855-60.\u003c/li\u003e\n\u003cli\u003enapoli aj, laderwager s, zoodsma jd, biju b, mucollari o, schubel sk, et al. developmental loss of nmda receptors results in supernumerary forebrain neurons through delayed maturation of transit-amplifying neuroblasts. sci rep. 2024;14(1):3395.\u003c/li\u003e\n\u003cli\u003erajani v, sengar as, salter mw. tripartite signalling by nmda receptors. mol brain. 2020;13(1):23.\u003c/li\u003e\n\u003cli\u003eishchenko y, carrizales mg, koleske aj. regulation of the nmda receptor by its cytoplasmic domains: (how) is the tail wagging the dog? neuropharmacology. 2021;195:108634.\u003c/li\u003e\n\u003cli\u003emony l, paoletti p. mechanisms of nmda receptor regulation. curr opin neurobiol. 2023;83:102815.\u003c/li\u003e\n\u003cli\u003enielsen ac, anderson cl, ens c, boyce akj, thompson rj. non-ionotropic nmdar signalling activates panx1 to induce p2x4r-dependent long-term depression in the hippocampus. j physiol. 2024.\u003c/li\u003e\n\u003cli\u003ebarnes sa, thomazeau a, finnie psb, heinrich mj, heynen aj, komiyama nh, et al. non-ionotropic signaling through the nmda receptor glun2b carboxy-terminal domain drives dendritic spine plasticity and reverses fragile x phenotypes. cell rep. 2025;44(3):115311.\u003c/li\u003e\n\u003cli\u003eplatzer k, lemke jr. grin1-related neurodevelopmental disorder. genereviews((r)). 2021.\u003c/li\u003e\n\u003cli\u003estrehlow v, myers ka, morgan at, scheffer ie, lemke jr. grin2a-related disorders. genereviews((r)). 2024.\u003c/li\u003e\n\u003cli\u003eplatzer k, lemke jr. grin2b-related neurodevelopmental disorder. genereviews((r)). 2021.\u003c/li\u003e\n\u003cli\u003eplatzer k, krey i, lemke jr. grin2d-related developmental and epileptic encephalopathy. genereviews((r)). 2022.\u003c/li\u003e\n\u003cli\u003ezehavi y, mandel h, zehavi a, rashid ma, straussberg r, jabur b, et al. de novo grin1 mutations: an emerging cause of severe early infantile encephalopathy. eur j med genet. 2017;60(6):317-20.\u003c/li\u003e\n\u003cli\u003esabo sl, lahr jm, offer m, weekes a, sceniak mp. grin2b-related neurodevelopmental disorder: current understanding of pathophysiological mechanisms. front synaptic neurosci. 2022;14:1090865.\u003c/li\u003e\n\u003cli\u003egjerulfsen ce, krey i, klockner c, rubboli g, lemke jr, moller rs. spectrum of nmda receptor variants in neurodevelopmental disorders and epilepsy. methods mol biol. 2024;2799:1-11.\u003c/li\u003e\n\u003cli\u003eamin jb, moody gr, wollmuth lp. from bedside-to-bench: what disease-associated variants are teaching us about the nmda receptor. j physiol. 2021;599(2):397-416.\u003c/li\u003e\n\u003cli\u003ehu c, chen w, myers sj, yuan h, traynelis sf. human grin2b variants in neurodevelopmental disorders. j pharmacol sci. 2016;132(2):115-21.\u003c/li\u003e\n\u003cli\u003ecox ja, kucenas s, voigt mm. molecular characterization and embryonic expression of the family of n-methyl-d-aspartate receptor subunit genes in the zebrafish. dev dyn. 2005;234(3):756-66.\u003c/li\u003e\n\u003cli\u003eveldman mb, lin s. zebrafish as a developmental model organism for pediatric research. pediatr res. 2008;64(5):470-6.\u003c/li\u003e\n\u003cli\u003echoi ty, choi ti, lee yr, choe sk, kim ch. zebrafish as an animal model for biomedical research. exp mol med. 2021;53(3):310-7.\u003c/li\u003e\n\u003cli\u003ezoodsma jd, chan k, bhandiwad aa, golann dr, liu g, syed sa, et al. a model to study nmda receptors in early nervous system development. j neurosci. 2020;40(18):3631-45.\u003c/li\u003e\n\u003cli\u003ezoodsma jd, keegan ej, moody gr, bhandiwad aa, napoli aj, burgess ha, et al. disruption of grin2b, an asd-associated gene, produces social deficits in zebrafish. mol autism. 2022;13(1):38.\u003c/li\u003e\n\u003cli\u003ebaraban sc, taylor mr, castro pa, baier h. pentylenetetrazole induced changes in zebrafish behavior, neural activity and c-fos expression. neuroscience. 2005;131(3):759-68.\u003c/li\u003e\n\u003cli\u003eshams s, rihel j, ortiz jg, gerlai r. the zebrafish as a promising tool for modeling human brain disorders: a review based upon an ibns symposium. neurosci biobehav rev. 2018;85:176-90.\u003c/li\u003e\n\u003cli\u003ede abreu ms, genario r, giacomini a, demin ka, lakstygal am, amstislavskaya tg, et al. zebrafish as a model of neurodevelopmental disorders. neuroscience. 2020;445:3-11.\u003c/li\u003e\n\u003cli\u003eweinschutz mendes h, neelakantan u, liu y, fitzpatrick se, chen t, wu w, et al. high-throughput functional analysis of autism genes in zebrafish identifies convergence in dopaminergic and neuroimmune pathways. cell rep. 2023;42(3):112243.\u003c/li\u003e\n\u003cli\u003ehowe k, clark md, torroja cf, torrance j, berthelot c, muffato m, et al. the zebrafish reference genome sequence and its relationship to the human genome. nature. 2013;496(7446):498-503.\u003c/li\u003e\n\u003cli\u003erihel j, schier af. behavioral screening for neuroactive drugs in zebrafish. dev neurobiol. 2012;72(3):373-85.\u003c/li\u003e\n\u003cli\u003ebaraban sc, dinday mt, hortopan ga. drug screening in scn1a zebrafish mutant identifies clemizole as a potential dravet syndrome treatment. nat commun. 2013;4:2410.\u003c/li\u003e\n\u003cli\u003ezhang y, kecskes a, copmans d, langlois m, crawford ad, ceulemans b, et al. pharmacological characterization of an antisense knockdown zebrafish model of dravet syndrome: inhibition of epileptic seizures by the serotonin agonist fenfluramine. plos one. 2015;10(5):e0125898.\u003c/li\u003e\n\u003cli\u003edingemans ajm, jansen s, van reeuwijk j, de leeuw n, pfundt r, schuurs-hoeijmakers j, et al. prevalence of comorbidities in individuals with neurodevelopmental disorders from the aggregated phenomics data of 51,227 pediatric individuals. nat med. 2024;30(7):1994-2003.\u003c/li\u003e\n\u003cli\u003eohtsuki t, sakurai k, dou h, toru m, yamakawa-kobayashi k, arinami t. mutation analysis of the nmdar2b (grin2b) gene in schizophrenia. mol psychiatry. 2001;6(2):211-6.\u003c/li\u003e\n\u003cli\u003earnold pd, rosenberg dr, mundo e, tharmalingam s, kennedy jl, richter ma. association of a glutamate (nmda) subunit receptor gene (grin2b) with obsessive-compulsive disorder: a preliminary study. psychopharmacology (berl). 2004;174(4):530-8.\u003c/li\u003e\n\u003cli\u003esaunders ja, tatard-leitman vm, suh j, billingslea en, roberts tp, siegel sj. knockout of nmda receptors in parvalbumin interneurons recreates autism-like phenotypes. autism res. 2013;6(2):69-77.\u003c/li\u003e\n\u003cli\u003egandal mj, sisti j, klook k, ortinski pi, leitman v, liang y, et al. gabab-mediated rescue of altered excitatory-inhibitory balance, gamma synchrony and behavioral deficits following constitutive nmdar-hypofunction. transl psychiatry. 2012;2(7):e142.\u003c/li\u003e\n\u003cli\u003ebarnby g, abbott a, sykes n, morris a, weeks de, mott r, et al. candidate-gene screening and association analysis at the autism-susceptibility locus on chromosome 16p: evidence of association at grin2a and abat. am j hum genet. 2005;76(6):950-66.\u003c/li\u003e\n\u003cli\u003ehung ly, margolis kg. autism spectrum disorders and the gastrointestinal tract: insights into mechanisms and clinical relevance. nat rev gastroenterol hepatol. 2024;21(3):142-63.\u003c/li\u003e\n\u003cli\u003eowens ap, mathias cj, iodice v. autonomic dysfunction in autism spectrum disorder. front integr neurosci. 2021;15:787037.\u003c/li\u003e\n\u003cli\u003eorefice ll. peripheral somatosensory neuron dysfunction: emerging roles in autism spectrum disorders. neuroscience. 2020;445:120-9.\u003c/li\u003e\n\u003cli\u003emakris g, agorastos a, chrousos gp, pervanidou p. stress system activation in children and adolescents with autism spectrum disorder. front neurosci. 2021;15:756628.\u003c/li\u003e\n\u003cli\u003eramos-vicente d, grant sg, bayes a. metazoan evolution and diversity of glutamate receptors and their auxiliary subunits. neuropharmacology. 2021;195:108640.\u003c/li\u003e\n\u003cli\u003eamores a, force a, yan yl, joly l, amemiya c, fritz a, et al. zebrafish hox clusters and vertebrate genome evolution. science. 1998;282(5394):1711-4.\u003c/li\u003e\n\u003cli\u003echou th, tajima n, romero-hernandez a, furukawa h. structural basis of functional transitions in mammalian nmda receptors. cell. 2020;182(2):357-71 e13.\u003c/li\u003e\n\u003cli\u003ezhang y, ye f, zhang t, lv s, zhou l, du d, et al. structural basis of ketamine action on human nmda receptors. nature. 2021;596(7871):301-5.\u003c/li\u003e\n\u003cli\u003ewollmuth lp, sakmann b. different mechanisms of ca2+ transport in nmda and ca2+-permeable ampa glutamate receptor channels. j gen physiol. 1998;112(5):623-36.\u003c/li\u003e\n\u003cli\u003ewatanabe j, beck c, kuner t, premkumar ls, wollmuth lp. drpeer: a motif in the extracellular vestibule conferring high ca2+ flux rates in nmda receptor channels. j neurosci. 2002;22(23):10209-16.\u003c/li\u003e\n\u003cli\u003ewollmuth lp, sobolevsky ai. structure and gating of the glutamate receptor ion channel. trends neurosci. 2004;27(6):321-8.\u003c/li\u003e\n\u003cli\u003eogden kk, khatri a, traynelis sf, heldt sa. potentiation of glun2c/d nmda receptor subtypes in the amygdala facilitates the retention of fear and extinction learning in mice. neuropsychopharmacology. 2014;39(3):625-37.\u003c/li\u003e\n\u003cli\u003ewollmuth lp. ion permeation in ionotropic glutamate receptors: still dynamic after all these years. curr opin physiol. 2018;2:36-41.\u003c/li\u003e\n\u003cli\u003emcdaniel mj, ogden kk, kell sa, burger pb, liotta dc, traynelis sf. nmda receptor channel gating control by the pre-m1 helix. j gen physiol. 2020;152(4).\u003c/li\u003e\n\u003cli\u003ewollmuth lp, kuner t, sakmann b. adjacent asparagines in the nr2-subunit of the nmda receptor channel control the voltage-dependent block by extracellular mg2+. j physiol. 1998;506 ( pt 1)(pt 1):13-32.\u003c/li\u003e\n\u003cli\u003eromero-hernandez a, simorowski n, karakas e, furukawa h. molecular basis for subtype specificity and high-affinity zinc inhibition in the glun1-glun2a nmda receptor amino-terminal domain. neuron. 2016;92(6):1324-36.\u003c/li\u003e\n\u003cli\u003emony l, zhu s, carvalho s, paoletti p. molecular basis of positive allosteric modulation of glun2b nmda receptors by polyamines. embo j. 2011;30(15):3134-46.\u003c/li\u003e\n\u003cli\u003elichnerova k, kaniakova m, park sp, skrenkova k, wang yx, petralia rs, et al. two n-glycosylation sites in the glun1 subunit are essential for releasing n-methyl-d-aspartate (nmda) receptors from the endoplasmic reticulum. j biol chem. 2015;290(30):18379-90.\u003c/li\u003e\n\u003cli\u003ewaterhouse a, bertoni m, bienert s, studer g, tauriello g, gumienny r, et al. swiss-model: homology modelling of protein structures and complexes. nucleic acids res. 2018;46(w1):w296-w303.\u003c/li\u003e\n\u003cli\u003ebienert s, waterhouse a, de beer ta, tauriello g, studer g, bordoli l, et al. the swiss-model repository-new features and functionality. nucleic acids res. 2017;45(d1):d313-d9.\u003c/li\u003e\n\u003cli\u003ewarnet xl, bakke krog h, sevillano-quispe og, poulsen h, kjaergaard m. the c-terminal domains of the nmda receptor: how intrinsically disordered tails affect signalling, plasticity and disease. eur j neurosci. 2021;54(8):6713-39.\u003c/li\u003e\n\u003cli\u003ekim k, saneyoshi t, hosokawa t, okamoto k, hayashi y. interplay of enzymatic and structural functions of camkii in long-term potentiation. j neurochem. 2016;139(6):959-72.\u003c/li\u003e\n\u003cli\u003ebej a, ames jb. chemical shift assignments of calmodulin bound to a cytosolic domain of glun2a (residues 1004-1024) from the nmda receptor. biomol nmr assign. 2023;17(1):89-93.\u003c/li\u003e\n\u003cli\u003ehorak m, wenthold rj. different roles of c-terminal cassettes in the trafficking of full-length nr1 subunits to the cell surface. j biol chem. 2009;284(15):9683-91.\u003c/li\u003e\n\u003cli\u003elin jw, wyszynski m, madhavan r, sealock r, kim ju, sheng m. yotiao, a novel protein of neuromuscular junction and brain that interacts with specific splice variants of nmda receptor subunit nr1. j neurosci. 1998;18(6):2017-27.\u003c/li\u003e\n\u003cli\u003estanic j, carta m, eberini i, pelucchi s, marcello e, genazzani aa, et al. rabphilin 3a retains nmda receptors at synaptic sites through interaction with glun2a/psd-95 complex. nat commun. 2015;6:10181.\u003c/li\u003e\n\u003cli\u003ecousins sl, stephenson fa. identification of c-terminal binding protein 1 as a novel nmda receptor interactor. neurochem res. 2019;44(6):1437-45.\u003c/li\u003e\n\u003cli\u003efranchini l, carrano n, di luca m, gardoni f. synaptic glun2a-containing nmda receptors: from physiology to pathological synaptic plasticity. int j mol sci. 2020;21(4).\u003c/li\u003e\n\u003cli\u003eswanwick cc, shapiro me, yi z, chang k, wenthold rj. nmda receptors interact with flotillin-1 and -2, lipid raft-associated proteins. febs lett. 2009;583(8):1226-30.\u003c/li\u003e\n\u003cli\u003ecousins sl, stephenson fa. identification of n-methyl-d-aspartic acid (nmda) receptor subtype-specific binding sites that mediate direct interactions with scaffold protein psd-95. j biol chem. 2012;287(16):13465-76.\u003c/li\u003e\n\u003cli\u003evissel b, krupp jj, heinemann sf, westbrook gl. a use-dependent tyrosine dephosphorylation of nmda receptors is independent of ion flux. nat neurosci. 2001;4(6):587-96.\u003c/li\u003e\n\u003cli\u003elavezzari g, mccallum j, lee r, roche kw. differential binding of the ap-2 adaptor complex and psd-95 to the c-terminus of the nmda receptor subunit nr2b regulates surface expression. neuropharmacology. 2003;45(6):729-37.\u003c/li\u003e\n\u003cli\u003echen bs, gray ja, sanz-clemente a, wei z, thomas ev, nicoll ra, et al. sap102 mediates synaptic clearance of nmda receptors. cell rep. 2012;2(5):1120-8.\u003c/li\u003e\n\u003cli\u003eelagabani mn, brisevac d, kintscher m, pohle j, kohr g, schmitz d, et al. subunit-selective n-methyl-d-aspartate (nmda) receptor signaling through brefeldin a-resistant arf guanine nucleotide exchange factors brag1 and brag2 during synapse maturation. j biol chem. 2016;291(17):9105-18.\u003c/li\u003e\n\u003cli\u003eyaka r, thornton c, vagts aj, phamluong k, bonci a, ron d. nmda receptor function is regulated by the inhibitory scaffolding protein, rack1. proc natl acad sci u s a. 2002;99(8):5710-5.\u003c/li\u003e\n\u003cli\u003ewyszynski m, lin j, rao a, nigh e, beggs ah, craig am, et al. competitive binding of alpha-actinin and calmodulin to the nmda receptor. nature. 1997;385(6615):439-42.\u003c/li\u003e\n\u003cli\u003ewechsler a, teichberg vi. brain spectrin binding to the nmda receptor is regulated by phosphorylation, calcium and calmodulin. embo j. 1998;17(14):3931-9.\u003c/li\u003e\n\u003cli\u003ekrapivinsky g, krapivinsky l, manasian y, ivanov a, tyzio r, pellegrino c, et al. the nmda receptor is coupled to the erk pathway by a direct interaction between nr2b and rasgrf1. neuron. 2003;40(4):775-84.\u003c/li\u003e\n\u003cli\u003emukherjee p, roy s, ghosh d, nandi sk. role of animal models in biomedical research: a review. lab anim res. 2022;38(1):18.\u003c/li\u003e\n\u003cli\u003echou th, epstein m, fritzemeier rg, akins ns, paladugu s, ullman ez, et al. molecular mechanism of ligand gating and opening of nmda receptor. nature. 2024;632(8023):209-17.\u003c/li\u003e\n\u003cli\u003earmstrong n, sun y, chen gq, gouaux e. structure of a glutamate-receptor ligand-binding core in complex with kainate. nature. 1998;395(6705):913-7.\u003c/li\u003e\n\u003cli\u003earmstrong n, gouaux e. mechanisms for activation and antagonism of an ampa-sensitive glutamate receptor: crystal structures of the glur2 ligand binding core. neuron. 2000;28(1):165-81.\u003c/li\u003e\n\u003cli\u003ekazi r, gan q, talukder i, markowitz m, salussolia cl, wollmuth lp. asynchronous movements prior to pore opening in nmda receptors. j neurosci. 2013;33(29):12052-66.\u003c/li\u003e\n\u003cli\u003eamin jb, gochman a, he m, certain n, wollmuth lp. nmda receptors require multiple pre-opening gating steps for efficient synaptic activity. neuron. 2021;109(3):488-501 e4.\u003c/li\u003e\n\u003cli\u003ehardingham ge, do kq. linking early-life nmdar hypofunction and oxidative stress in schizophrenia pathogenesis. nat rev neurosci. 2016;17(2):125-34.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, riley gr, jang w, rubinstein ws, church dm, et al. clinvar: public archive of relationships among sequence variation and human phenotype. nucleic acids res. 2014;42(database issue):d980-5.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, benson m, brown g, chao c, chitipiralla s, et al. clinvar: public archive of interpretations of clinically relevant variants. nucleic acids res. 2016;44(d1):d862-8.\u003c/li\u003e\n\u003cli\u003elandrum mj, lee jm, benson m, brown gr, chao c, chitipiralla s, et al. clinvar: improving access to variant interpretations and supporting evidence. nucleic acids res. 2018;46(d1):d1062-d7.\u003c/li\u003e\n\u003cli\u003egudmundsson s, singer-berk m, watts na, phu w, goodrich jk, solomonson m, et al. variant interpretation using population databases: lessons from gnomad. hum mutat. 2022;43(8):1012-30.\u003c/li\u003e\n\u003cli\u003etian w, skolnick j. how well is enzyme function conserved as a function of pairwise sequence identity? j mol biol. 2003;333(4):863-82.\u003c/li\u003e\n\u003cli\u003eaddou s, rentzsch r, lee d, orengo ca. domain-based and family-specific sequence identity thresholds increase the levels of reliable protein function transfer. j mol biol. 2009;387(2):416-30.\u003c/li\u003e\n\u003cli\u003esangar v, blankenberg dj, altman n, lesk am. quantitative sequence-function relationships in proteins based on gene ontology. bmc bioinformatics. 2007;8:294.\u003c/li\u003e\n\u003cli\u003eperszyk re, kristensen as, lyuboslavsky p, traynelis sf. three-dimensional missense tolerance ratio analysis. genome res. 2021;31(8):1447-61.\u003c/li\u003e\n\u003cli\u003euversky vn. intrinsically disordered proteins from a to z. int j biochem cell biol. 2011;43(8):1090-103.\u003c/li\u003e\n\u003cli\u003esingleton md, eisen mb. evolutionary analyses of intrinsically disordered regions reveal widespread signals of conservation. plos comput biol. 2024;20(4):e1012028.\u003c/li\u003e\n\u003cli\u003edavey ne, cowan jl, shields dc, gibson tj, coldwell mj, edwards rj. slimprints: conservation-based discovery of functional motif fingerprints in intrinsically disordered protein regions. nucleic acids res. 2012;40(21):10628-41.\u003c/li\u003e\n\u003c/ol\u003e"}],"fulltextsource":"","fulltext":"","funders":[],"hasadminpriorityonworkflow":false,"hasmanuscriptdocx":true,"hasoptedintopreprint":true,"haspassedjournalqc":"","hasanypriority":false,"hidejournal":false,"highlight":"","institution":"","isacceptedbyjournal":false,"isauthorsuppliedpdf":false,"isdeskrejected":"","ishiddenfromsearch":false,"isinqc":false,"isinworkflow":false,"ispdf":false,"ispdfuptodate":true,"iswithdrawnorretracted":false,"journal":{"display":true,"email":"info@researchsquare.com","identity":"bmc-genomics","isnaturejournal":false,"hasqc":true,"allowdirectsubmit":false,"externalidentity":"gics","sidebox":"learn more about [bmc genomics](http://bmcgenomics.biomedcentral.com/)","snpcode":"","submissionurl":"https://www.editorialmanager.com/gics","title":"bmc genomics","twitterhandle":"#bmcgenomics","acdcenabled":true,"dfaenabled":false,"editorialsystem":"em","reportingportfolio":"bmc series","inreviewenabled":true,"inreviewrevisionsenabled":true},"keywords":"grin genes, clinvar, gnomad, sequence alignments, disease-associated variants, n-terminal domain, ligand-binding domain, transmembrane domain, c-terminal domain, ion channel","lastpublisheddoi":"10.21203/rs.3.rs-7151578/v1","lastpublisheddoiurl":"https://doi.org/10.21203/rs.3.rs-7151578/v1","license":{"name":"cc by 4.0","url":"https://creativecommons.org/licenses/by/4.0/"},"manuscriptabstract":"\u003ch2\u003ebackground\u003c/h2\u003e\u003cp\u003enmda receptors (nmdars) are widely expressed, glutamate-gated ion channels that play key roles in brain development and function. variants have been identified in the \u003cem\u003egrin\u003c/em\u003e genes encoding nmdar subunits that are linked to neurodevelopmental disorders, among other manifestations. zebrafish are a powerful model to study brain development and function given their rapid development and ease of genetic manipulation. as a result of an ancient genome duplication, zebrafish possess two paralogues for most human nmdar subunits. to evaluate the degree of conservation between human nmdar subunits and their respective zebrafish paralogues, we carried out detailed \u003cem\u003ein silico\u003c/em\u003e analyses, with an emphasis on key functional elements. to further assess the suitability of zebrafish for modeling nmdar-associated neurodevelopmental disorders, we analyzed the conservation of positions with identified missense variants.\u003c/p\u003e\u003ch2\u003eresults\u003c/h2\u003e\u003cp\u003ewe find that the human nmdar subunits are generally well conserved across zebrafish paralogs. moreover, variants classified as pathogenic and putatively pathogenic are highly conserved, reflecting the importance of key protein regions to neurotypical receptor function. positions with putatively benign and benign variants are less conserved. across nmdar domains, the transmembrane domain is most highly conserved, followed by the ligand-binding domain, which maintains conservation of amino acids that participate in the binding of ligands. the n-terminal domain is less well conserved but aligned homology models show high degrees of structural similarity. the c-terminal domain is the most poorly conserved region across zebrafish paralogs, but certain key regions that undergo phosphorylation, palmitoylation, and ubiquitylation as well as protein-binding motifs are better conserved.\u003c/p\u003e\u003ch2\u003econclusions\u003c/h2\u003e\u003cp\u003eour findings highlight a strong conservation of human nmdar subunits in zebrafish, with some exceptions. the ligand-binding domain, the transmembrane domain forming the ion channel and the short polypeptide linkers that connect them are highly conserved. the n- and c-terminal domains are less conserved but functional motifs in general, except for the zn\u003csup\u003e2+\u003c/sup\u003e binding site in glun2a paralogues, are more highly conserved relative to the entire domain. overall, our findings support the utility of zebrafish as a model for studying neurodevelopment and disease mechanisms and provide a template for rigorously considering the relationship between human and zebrafish paralogues.\u003c/p\u003e","manuscripttitle":"conservation of human nmda receptor subunits and disease variants in zebrafish","msid":"","msnumber":"","nondraftversions":[{"code":1,"date":"2025-07-30 18:25:39","doi":"10.21203/rs.3.rs-7151578/v1","editorialevents":[{"type":"communitycomments","content":0},{"type":"decision","content":"revision requested","date":"2025-09-24t19:23:39+00:00","index":"","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-09-19t21:34:21+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"269262391104669611722251259561774961128","date":"2025-09-19t14:02:55+00:00","index":"hide","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-09-09t17:52:57+00:00","index":"hide","fulltext":""},{"type":"editorinvitedreview","content":"","date":"2025-08-18t16:11:58+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"201191728820772676273599414131783545114","date":"2025-08-04t10:39:46+00:00","index":"hide","fulltext":""},{"type":"revieweragreed","content":"230034227985411933590242857371107944843","date":"2025-07-28t13:05:21+00:00","index":"hide","fulltext":""},{"type":"reviewersinvited","content":"","date":"2025-07-28t07:38:37+00:00","index":"","fulltext":""},{"type":"editorassigned","content":"","date":"2025-07-28t07:10:47+00:00","index":"","fulltext":""},{"type":"editorinvited","content":"","date":"2025-07-23t18:56:26+00:00","index":"","fulltext":""},{"type":"checkscomplete","content":"","date":"2025-07-22t16:36:43+00:00","index":"","fulltext":""},{"type":"submitted","content":"bmc genomics","date":"2025-07-22t16:32:19+00:00","index":"","fulltext":""}],"status":"published","journal":{"display":true,"email":"info@researchsquare.com","identity":"bmc-genomics","isnaturejournal":false,"hasqc":true,"allowdirectsubmit":false,"externalidentity":"gics","sidebox":"learn more about [bmc genomics](http://bmcgenomics.biomedcentral.com/)","snpcode":"","submissionurl":"https://www.editorialmanager.com/gics","title":"bmc genomics","twitterhandle":"#bmcgenomics","acdcenabled":true,"dfaenabled":false,"editorialsystem":"em","reportingportfolio":"bmc series","inreviewenabled":true,"inreviewrevisionsenabled":true}}],"origin":"","owneridentity":"2b85bbad-8f38-4c35-89b3-5949e6b6d3c0","owner":[],"posteddate":"july 30th, 2025","published":true,"recenteditorialevents":[],"rejectedjournal":[],"revision":"","amendment":"","status":"under-review","subjectareas":[],"tags":[],"updatedat":"2025-10-14t16:08:28+00:00","versionofrecord":[],"versioncreatedat":"2025-07-30 18:25:39","video":"","vordoi":"","vordoiurl":"","workflowstages":[]},"version":"v1","identity":"rs-7151578","journalconfig":"researchsquare"},"__n_ssp":true},"page":"/article/[identity]/[[...version]]","query":{"identity":"rs-7151578","version":["v1"]},"buildid":"idgtm6tzxtmdq-mqt1zbo","isfallback":false,"dynamicids":[87228],"gssp":true,"scriptloader":[]}

## Scraping Notes

- Successfully scraped from DOI.org
