# Genome-wide analyses identify 30 loci associated with obsessive-compulsive disorder.

**DOI:** 10.1038/s41588-025-02189-z
**Journal:** Nature genetics
**Date:** 2025-05-14
**Authors:** ['Nora IStrom', 'Zachary FGerring', 'MarcoGalimberti', 'DongmeiYu', 'Matthew WHalvorsen', 'AbdelAbdellaoui', 'CristinaRodriguez-Fontenla', 'Julia MSealock', 'TimBigdeli', 'Jonathan RColeman', 'BehrangMahjani', 'Jackson GThorp', 'KatharinaBey', 'Christie LBurton', 'Jurjen JLuykx', 'GwynethZai', 'SilviaAlemany', 'ChristineAndre', 'Kathleen DAskland', 'JuliaBäckman', 'NerisaBanaj', 'CristinaBarlassina', 'Judith BeckerNissen', 'O JosephBienvenu', 'DonaldBlack', 'Michael HBloch', 'SigridBørte', 'RosaBosch', 'MichaelBreen', 'Brian PBrennan', 'HelenaBrentani', 'Joseph DBuxbaum', 'JonasBybjerg-Grauholm', 'Enda MByrne', 'JuditCabana-Dominguez', 'BeatrizCamarena', 'AdrianCamarena', 'CarolinaCappi', 'AngelCarracedo', 'MiguelCasas', 'Maria CristinaCavallini', 'ValentinaCiullo', 'Edwin HCook', 'JesseCrosby', 'Bernadette ACullen', 'Elles JDe Schipper', 'RichardDelorme', 'SrdjanDjurovic', 'Jason AElias', 'XavierEstivill', 'Martha JFalkenstein', 'Bengt TFundin', 'LaurynGarner', 'ChristinaGironda', 'Fernando SGoes', 'Marco AGrados', 'JakobGrove', 'WeiGuo', 'JanHaavik', 'KristenHagen', 'KellyHarrington', 'AlexandraHavdahl', 'Kira DHöffler', 'Ana GHounie', 'DonaldHucks', 'ChristinaHultman', 'MagdalenaJanecka', 'EricJenike', 'Elinor KKarlsson', 'KaraKelley', 'JuliaKlawohn', 'Janice EKrasnow', 'KristiKrebs', 'ChristophLange', 'NuriaLanzagorta', 'DanielLevey', 'KerstinLindblad-Toh', 'FabioMacciardi', 'BrionMaher', 'BrittanyMathes', 'EvonneMcArthur', 'NathanielMcGregor', 'Nicole CMcLaughlin', 'SandraMeier', 'Euripedes CMiguel', 'MaureenMulhern', 'Paul SNestadt', 'Erika LNurmi', "Kevin SO'Connell", 'LisaOsiecki', 'Olga ThereseOusdal', 'TeemuPalviainen', 'Nancy LPedersen', 'FabrizioPiras', 'FedericaPiras', 'SriramyaPotluri', 'RaquelRabionet', 'AlfredoRamirez', 'ScottRauch', 'AbrahamReichenberg', 'Mark ARiddle', 'StephanRipke', 'Maria CRosário', 'Aline SSampaio', 'Miriam ASchiele', 'Anne HeidiSkogholt', 'Laura GSloofman', 'JanSmit', 'María SolerArtigas', 'Laurent FThomas', 'EricTifft', 'HomeroVallada', 'Nathanialvan Kirk', 'JeremyVeenstra-VanderWeele', 'Nienke NVulink', 'Christopher PWalker', 'YingWang', 'Jens RWendland', 'Bendik SWinsvold', 'YinYao', 'HangZhou', 'NoneNone', 'NoneNone', 'ArpanaAgrawal', 'PinoAlonso', 'GötzBerberich', 'Kathleen KBucholz', 'Cynthia MBulik', 'DanielleCath', 'DamiaanDenys', 'ValsammaEapen', 'HowardEdenberg', 'PeterFalkai', 'Thomas VFernandez', 'Abby JFyer', 'J MGaziano', 'Dan AGeller', 'Hans JGrabe', 'Benjamin DGreenberg', 'Gregory LHanna', 'Ian BHickie', 'David MHougaard', 'NorbertKathmann', 'JamesKennedy', 'DongbingLai', 'MikaelLandén', 'Stéphanie LeHellard', 'MarionLeboyer', 'ChristineLochner', 'James TMcCracken', 'Sarah EMedland', 'Preben BMortensen', 'Benjamin MNeale', 'HumbertoNicolini', 'MereteNordentoft', 'MichelePato', 'CarlosPato', 'David LPauls', 'JohnPiacentini', 'ChristopherPittenger', 'DaniellePosthuma', 'Josep AntoniRamos-Quiroga', 'Steven ARasmussen', 'Margaret ARichter', 'David RRosenberg', 'StephanRuhrmann', 'Jack FSamuels', 'SvenSandin', 'PaulSandor', 'GianfrancoSpalletta', 'Dan JStein', 'S EvelynStewart', 'Eric AStorch', 'Barbara EStranger', 'MaurizioTuriel', 'ThomasWerge', 'Ole AAndreassen', 'Anders DBørglum', 'SusanneWalitza', 'KristianHveem', 'Bjarne KHansen', 'ChristianRück', 'Nicholas GMartin', 'LiliMilani', 'OleMors', 'TedReichborn-Kjennerud', 'MartaRibasés', 'GerdKvale', 'DavidMataix-Cols', 'KatharinaDomschke', 'EdnaGrünblatt', 'MichaelWagner', 'John-AnkerZwart', 'GeromeBreen', 'GeraldNestadt', 'JaakkoKaprio', 'Paul DArnold', 'Dorothy EGrice', 'James AKnowles', 'HelgaAsk', 'Karin JVerweij', 'Lea KDavis', 'Dirk JSmit', 'James JCrowley', 'Jeremiah MScharf', 'Murray BStein', 'JoelGelernter', 'Carol AMathews', 'Eske MDerks', 'ManuelMattheisen']
**Category:** tourette_syndrome
**Source:** pubmed
**Scraped by:** agent_a
**Scraped at:** 2025-10-21T10:51:28.025443
**Source URL:** https://doi.org/10.1038/s41588-025-02189-z

## Abstract

Obsessive-compulsive disorder (OCD) affects ~1% of children and adults and is partly caused by genetic factors. We conducted a genome-wide association study (GWAS) meta-analysis combining 53,660 OCD cases and 2,044,417 controls and identified 30 independent genome-wide significant loci. Gene-based approaches identified 249 potential effector genes for OCD, with 25 of these classified as the most likely causal candidates, including WDR6, DALRD3 and CTNND1 and multiple genes in the major histocompatibility complex (MHC) region. We estimated that ~11,500 genetic variants explained 90% of OCD genetic heritability. OCD genetic risk was associated with excitatory neurons in the hippocampus and the cortex, along with D

## Full Text Content

Download PDFArticleOpen accessPublished:13 May 2025Genome-wide analyses identify 30 loci associated with obsessive–compulsive disorderNora I. StromORCID:orcid.org/0000-0002-5261-88521,2,3,4,5na1,Zachary F. GerringORCID:orcid.org/0000-0002-2445-12666,7na1,Marco GalimbertiORCID:orcid.org/0000-0001-6052-156X8,9na1,Dongmei YuORCID:orcid.org/0000-0001-7901-436510,11na1,Matthew W. Halvorsen12,Abdel AbdellaouiORCID:orcid.org/0000-0003-1088-678413,Cristina Rodriguez-FontenlaORCID:orcid.org/0000-0002-9829-639314,15,Julia M. Sealock16,Tim Bigdeli17,18,Jonathan R. ColemanORCID:orcid.org/0000-0002-6759-094419,20,Behrang MahjaniORCID:orcid.org/0000-0001-6087-953721,22,Jackson G. ThorpORCID:orcid.org/0000-0002-9461-641723,24,Katharina BeyORCID:orcid.org/0000-0001-6392-499725,Christie L. BurtonORCID:orcid.org/0000-0002-8955-652826,Jurjen J. LuykxORCID:orcid.org/0000-0002-6439-277427,28,Gwyneth Zai29,30,Silvia AlemanyORCID:orcid.org/0000-0002-7925-676731,32,33,Christine Andre34,Kathleen D. Askland35,Julia Bäckman3,Nerisa BanajORCID:orcid.org/0000-0002-6317-854936,Cristina Barlassina37,Judith Becker Nissen38,39,O. Joseph Bienvenu40,Donald Black41,Michael H. Bloch42,Sigrid BørteORCID:orcid.org/0000-0001-9540-325643,44,45,Rosa BoschORCID:orcid.org/0000-0002-7596-183X46,47,Michael Breen21,48,49,Brian P. Brennan34,50,Helena BrentaniORCID:orcid.org/0000-0001-5192-468251,Joseph D. BuxbaumORCID:orcid.org/0000-0001-8898-831321,Jonas Bybjerg-GrauholmORCID:orcid.org/0000-0003-1705-400852,Enda M. ByrneORCID:orcid.org/0000-0002-9491-779753,Judit Cabana-DominguezORCID:orcid.org/0000-0002-4732-728431,32,33,Beatriz Camarena54,Adrian Camarena55,Carolina Cappi56,57,Angel Carracedo58,59,60,Miguel Casas61,62,Maria Cristina Cavallini63,Valentina Ciullo36,Edwin H. CookORCID:orcid.org/0000-0002-5848-511464,Jesse Crosby34,50,Bernadette A. Cullen65,66,Elles J. De Schipper3,Richard Delorme67,Srdjan DjurovicORCID:orcid.org/0000-0002-8140-806145,68,Jason A. Elias69,70,Xavier Estivill71,Martha J. Falkenstein34,50,Bengt T. Fundin72,Lauryn Garner34,Christina Gironda34,Fernando S. GoesORCID:orcid.org/0000-0001-6262-826473,Marco A. Grados74,Jakob GroveORCID:orcid.org/0000-0003-2284-57444,75,76,77,Wei Guo78,Jan HaavikORCID:orcid.org/0000-0001-7865-280879,80,Kristen Hagen81,82,83,Kelly Harrington84,85,Alexandra HavdahlORCID:orcid.org/0000-0002-9268-042386,87,Kira D. HöfflerORCID:orcid.org/0000-0002-0837-146968,88,89,Ana G. Hounie57,Donald Hucks90,Christina Hultman22,Magdalena Janecka21,91,Eric Jenike34,Elinor K. Karlsson92,93,Kara Kelley34,Julia Klawohn1,94,Janice E. Krasnow95,Kristi KrebsORCID:orcid.org/0000-0003-0494-275196,Christoph Lange97,98,Nuria LanzagortaORCID:orcid.org/0000-0001-6769-681399,Daniel LeveyORCID:orcid.org/0000-0001-8431-9569100,101,Kerstin Lindblad-TohORCID:orcid.org/0000-0001-8338-025393,102,103,Fabio Macciardi104,Brion MaherORCID:orcid.org/0000-0001-9809-0064105,Brittany Mathes34,Evonne McArthur106,Nathaniel McGregorORCID:orcid.org/0000-0001-7195-4993107,Nicole C. McLaughlin108,109,Sandra Meier110,Euripedes C. MiguelORCID:orcid.org/0000-0002-9393-3103111,Maureen Mulhern25,Paul S. NestadtORCID:orcid.org/0000-0002-2479-703X112,Erika L. Nurmi113,Kevin S. O’ConnellORCID:orcid.org/0000-0002-6865-8795114,115,Lisa Osiecki116,117,Olga Therese Ousdal79,118,Teemu PalviainenORCID:orcid.org/0000-0002-7847-8384119,Nancy L. Pedersen22,Fabrizio Piras36,Federica PirasORCID:orcid.org/0000-0002-9546-7038120,Sriramya Potluri34,Raquel RabionetORCID:orcid.org/0000-0001-5006-8140121,122,123,Alfredo RamirezORCID:orcid.org/0000-0003-4991-763X124,125,126,127,128,Scott Rauch129,Abraham Reichenberg130,Mark A. Riddle131,Stephan RipkeORCID:orcid.org/0000-0003-3622-835X132,133,134,Maria C. Rosário135,Aline S. Sampaio136,Miriam A. Schiele137,Anne Heidi SkogholtORCID:orcid.org/0000-0001-7466-0405138,Laura G. SloofmanORCID:orcid.org/0000-0001-7628-437821,Jan Smit139,María Soler ArtigasORCID:orcid.org/0000-0002-3213-110731,32,33,140,Laurent F. ThomasORCID:orcid.org/0000-0003-0548-2486141,142,143,144,Eric Tifft34,Homero Vallada51,145,Nathanial van KirkORCID:orcid.org/0000-0002-1975-4238146,147,Jeremy Veenstra-VanderWeele148,149,Nienke N. Vulink150,Christopher P. WalkerORCID:orcid.org/0000-0002-6069-0184151,Ying Wang152,Jens R. WendlandORCID:orcid.org/0000-0001-5106-5013153,Bendik S. WinsvoldORCID:orcid.org/0000-0003-4171-891943,154,155,Yin YaoORCID:orcid.org/0000-0003-3053-8429156,Hang ZhouORCID:orcid.org/0000-0002-7694-6391157,158,159,Estonian Biobank,23andMe Inc.,Arpana AgrawalORCID:orcid.org/0000-0002-0313-793X160,Pino Alonso161,162,163,164,Götz Berberich165,Kathleen K. Bucholz166,Cynthia M. BulikORCID:orcid.org/0000-0001-7772-326422,167,168,Danielle Cath169,170,Damiaan Denys171,Valsamma EapenORCID:orcid.org/0000-0001-6296-8306172,173,Howard EdenbergORCID:orcid.org/0000-0003-0344-9690174,Peter FalkaiORCID:orcid.org/0000-0002-0284-2290175,176,Thomas V. FernandezORCID:orcid.org/0000-0003-0830-022X177,Abby J. FyerORCID:orcid.org/0000-0002-0944-2505178,179,J. M. Gaziano180,181,Dan A. GellerORCID:orcid.org/0000-0001-8692-415650,182,Hans J. Grabe183,Benjamin D. Greenberg184,185,186,Gregory L. HannaORCID:orcid.org/0000-0002-0742-6990187,Ian B. HickieORCID:orcid.org/0000-0001-8832-9895188,David M. HougaardORCID:orcid.org/0000-0001-5928-351752,76,Norbert Kathmann1,James KennedyORCID:orcid.org/0000-0002-8733-380630,Dongbing LaiORCID:orcid.org/0000-0001-7803-580X189,Mikael LandénORCID:orcid.org/0000-0002-4496-645122,190,Stéphanie Le HellardORCID:orcid.org/0000-0002-8085-051X68,80,Marion LeboyerORCID:orcid.org/0000-0001-5473-3697191,Christine Lochner192,James T. McCracken113,Sarah E. MedlandORCID:orcid.org/0000-0003-1382-380X193,Preben B. MortensenORCID:orcid.org/0000-0002-5230-986576,194,195,Benjamin M. NealeORCID:orcid.org/0000-0003-1513-6077133,196,Humberto Nicolini197,198,Merete Nordentoft199,200,Michele Pato201,Carlos Pato201,David L. Pauls50,John Piacentini202,Christopher Pittenger203,Danielle PosthumaORCID:orcid.org/0000-0001-7582-2365204,205,Josep Antoni Ramos-QuirogaORCID:orcid.org/0000-0003-1622-0350206,207,208,209,Steven A. Rasmussen108,Margaret A. Richter30,210,David R. Rosenberg211,Stephan RuhrmannORCID:orcid.org/0000-0002-6022-2364212,Jack F. Samuels213,Sven SandinORCID:orcid.org/0000-0001-6994-488421,22,Paul SandorORCID:orcid.org/0000-0003-2397-075830,Gianfranco Spalletta36,214,Dan J. SteinORCID:orcid.org/0000-0001-7218-7810215,S. Evelyn Stewart216,217,218,Eric A. Storch219,Barbara E. StrangerORCID:orcid.org/0000-0001-9021-7331220,221,Maurizio Turiel222,Thomas WergeORCID:orcid.org/0000-0003-1829-0766223,224,Ole A. Andreassen225,226,Anders D. BørglumORCID:orcid.org/0000-0001-8627-72194,227,228,Susanne Walitza229,230,231,Kristian Hveem155,232,233,Bjarne K. Hansen82,234,Christian Rück3,Nicholas G. MartinORCID:orcid.org/0000-0003-4069-8020235,Lili MilaniORCID:orcid.org/0000-0002-5323-310296,Ole Mors236,Ted Reichborn-Kjennerud237,238,Marta RibasésORCID:orcid.org/0000-0003-1039-111631,32,33,140,Gerd Kvale83,239,David Mataix-ColsORCID:orcid.org/0000-0002-4545-09243,Katharina DomschkeORCID:orcid.org/0000-0002-2550-9132137,240,Edna GrünblattORCID:orcid.org/0000-0001-8505-7265229,230,231,Michael Wagner25,241,242,John-Anker ZwartORCID:orcid.org/0000-0001-5721-0154155,243,Gerome Breen244,Gerald NestadtORCID:orcid.org/0000-0001-9461-5249112,Jaakko KaprioORCID:orcid.org/0000-0002-3716-2455245,Paul D. Arnold246,247,Dorothy E. GriceORCID:orcid.org/0000-0002-8833-254921,James A. Knowles248,Helga AskORCID:orcid.org/0000-0003-0149-5319249,250,Karin J. VerweijORCID:orcid.org/0000-0001-9868-7821150,Lea K. DavisORCID:orcid.org/0000-0001-5143-2282251,Dirk J. SmitORCID:orcid.org/0000-0001-8301-8860252,James J. CrowleyORCID:orcid.org/0000-0001-9051-15573,12,167,Jeremiah M. Scharf116,133na2,Murray B. SteinORCID:orcid.org/0000-0001-9564-2871253,254na2,Joel GelernterORCID:orcid.org/0000-0002-4067-1859255,256na2,Carol A. Mathews257na2,Eske M. DerksORCID:orcid.org/0000-0002-6292-6883258na2&…Manuel MattheisenORCID:orcid.org/0000-0002-8442-493X2,4,259na2Show authorsNature Geneticsvolume57,pages1389–1401 (2025)Cite this article58kAccesses8Citations458AltmetricMetricsdetailsSubjectsGenetics researchGenome-wide association studiesAbstractObsessive–compulsive disorder (OCD) affects ~1% of children and adults and is partly caused by genetic factors. We conducted a genome-wide association study (GWAS) meta-analysis combining 53,660 OCD cases and 2,044,417 controls and identified 30 independent genome-wide significant loci. Gene-based approaches identified 249 potential effector genes for OCD, with 25 of these classified as the most likely causal candidates, includingWDR6,DALRD3andCTNND1and multiple genes in the major histocompatibility complex (MHC) region. We estimated that ~11,500 genetic variants explained 90% of OCD genetic heritability. OCD genetic risk was associated with excitatory neurons in the hippocampus and the cortex, along with D1and D2type dopamine receptor-containing medium spiny neurons. OCD genetic risk was shared with 65 of 112 additional phenotypes, including all the psychiatric disorders we examined. In particular, OCD shared genetic risk with anxiety, depression, anorexia nervosa and Tourette syndrome and was negatively associated with inflammatory bowel diseases, educational attainment and body mass index.Similar content being viewed by othersShared genetic loci and causal relations between schizophrenia and obsessive-compulsive disorderArticleOpen access07 April 2023Genome-wide association study of pediatric obsessive-compulsive traits: shared genetic risk between traits and disorderArticleOpen access02 February 2021The genetic epidemiology of obsessive-compulsive disorder: a systematic review and meta-analysisArticleOpen access28 June 2023MainOCD is a chronic psychiatric disorder that affects 1–3% of the population1and is characterized by obsessions and compulsions that vary in type and severity and over time. OCD is responsible for profound personal and societal costs2, including increased risk of suicide3and overall mortality4. OCD is moderately heritable; twin-based heritability estimates range between 27% and 47% in adults and between 45% and 65% in children5,6,7,8, with SNP-based heritability estimates between 28% and 37%9,10,11.Two earlier OCD GWAS meta-analyses, both containing a subset of the data included in this analysis12,13, showed SNP-based heritabilities of 8.5% (assuming a 3% population prevalence) and 16% (assuming a 2% population prevalence). The first GWAS (ncases= 14,140,ncontrols= 562,117)12found one genome-wide significant locus associated with OCD, while the second (ncases= 37,015,ncontrols= 948,616)13identified 15 independent genome-wide significant loci. As with other complex traits, increased sample sizes are needed for a more comprehensive understanding of the underlying genetic etiology of OCD and its genetic relationships with related disorders.The current study combines data from the two unpublished OCD GWASs described above and includes additional cohorts (~9,000 cases). This results in one of the largest and most well-powered GWAS of OCD so far, with a ~20-fold increase of OCD cases compared to the previously published OCD GWASs10. Based on the results from the meta-analysis, we conducted secondary analyses, including positional and functional fine-mapping of SNPs and genes, structural equation modeling to examine possible genetic differences in sample ascertainment across cohorts, protein and transcriptome-wide association analyses, single-cell enrichment and genetic correlations with other traits (Supplementary Fig.1). Our results provide more detailed insight into the genetic underpinnings and biology of OCD.ResultsGWAS meta-analysis identifies 30 genome-wide significant lociWe conducted a GWAS meta-analysis of 28 OCD case–control cohorts of European ancestry, comprising 53,660 cases and 2,044,417 controls (effective sample size, ~210,000 individuals). Ascertainment of cases varied across cohorts: OCD diagnosis was determined (1) by a healthcare professional in a clinical setting (18 cohorts,n= 9,089 cases), (2) from health records or biobanks (seven cohorts,n= 9,138 cases), (3) in a clinical setting or from health records with the additional characteristic that all OCD cases were primarily collected for another psychiatric disorder (three cohorts,n= 5,266 cases) or (4) by self-reported diagnosis in a consumer-based setting (23andMe, Inc.,n= 30,167 cases). Cohort details, including phenotypic assessment, quality control and individual cohort GWAS analyses, are described in Supplementary Note2and Supplementary Table1. We identified 30 independent (defined in Supplementary Note3) loci among the 1,672 SNPs that exceeded the genome-wide threshold for significance (\(P < 5{\times 10}^{-8}\); Manhattan plot in Fig.1, regional association plots and forest plots in Supplementary Figs.2–31and a list of all independent genome-wide significant SNPs in Table1with additional details in Supplementary Tables2and3). The independence of the 30 lead SNPs was subsequently validated using conditional and joint analysis (GCTA-COJO)14(Supplementary Table4). Analysis of the X chromosome, conducted in a subset of the data for which this information was available (23andMe), yielded no significant associations (Supplementary Note4and Supplementary Fig.37e). Of the 15 genome-wide significant loci previously reported in preprints12,13, 13 were genome-wide significant in the current GWAS, with the remaining two showing suggestive significance (\(P=5.23{\times 10}^{-8}\,{\rm{and}}\,{P}=2.2{\times 10}^{-7}\); Supplementary Table5). Using MiXeR15, we estimated that approximately 11,500 (standard error of the effect estimate (s.e.) = 607) causal variants account for 90% of the OCD SNP-based heritability.Fig. 1: Manhattan plot of OCD GWAS meta-analysis.Theyaxis represents −log10(Pvalues) (two sided, not adjusted for multiple testing) for the association of variants with OCD using an inverse-variance-weighted fixed-effects model (ncases= 53,660 andncontrols= 2,044,417). Thexaxis shows chromosomes 1–22. The horizontal red line represents the threshold for genome-wide significance (\(P=5\times {10}^{-8})\). Index variants of genome-wide significant loci are highlighted as green diamonds.Full size imageTable 1 Genome-wide-significant loci associated with OCDFull size tableNo statistically significant heterogeneity was observed across individual cohorts for the 30 genome-wide significant loci, as assessed with Cochran’sQ-test (Supplementary Fig.32), theI2statistic and the genomic structural equation modeling (GenomicSEM)\({Q}_{{\rm{SNP}}}\)statistic16(Supplementary Table2). Genome-wide analyses of samples grouped by clinical, comorbid, biobank and 23andMe information (Supplementary Table3and Supplementary Figs.33–37) showed evidence that sample ascertainment impacted results at a genome-wide scale, although not beyond what is observed with closely related psychiatric disorders17,18. We observed moderate to high genetic correlations across the subgroups (between 0.63, s.e. = 0.11 for biobanks and comorbid information and 0.92, s.e. = 0.07 for 23andMe and comorbid information; Supplementary Table7) and a satisfactory fit for a one-factor GenomicSEM model (Supplementary Table8and Supplementary Fig.39). A common factor GWAS based on the one-factor GenomicSEM model resulted in 20 significant loci, all of which were also significant in the primary GWAS (Supplementary Table8and Supplementary Fig.40; analysis details in Supplementary Note5). SNP heritability (assuming a 1% population prevalence) was 6.7% (s.e. = 0.3%), with slightly higher estimates for the clinical (\({h}_{{\rm{SNP}}}^{2}\)= 16.4%, s.e. = 1.5%) and comorbid (\({h}_{{\rm{SNP}}}^{2}\)= 13.3%, s.e. = 1.7%) subgroups (Supplementary Table1).Gene-based findingsWe prioritized putative risk genes for OCD using six positional and functional QTL gene-based mapping approaches. Positional mapping was performed with mBAT-combo19. Functional expression quantitative trait locus (eQTL) mapping was performed with transcriptome-wide association study (TWAS)20, using PsychENCODE gene expression weights21, and summary-based Mendelian randomization (SMR)22using the whole-blood eQTLGen23and MetaBrain24datasets. Functional protein QTL mapping was done using a protein-wide association study (PWAS) of human brain protein expression panels25. Finally, we used the psychiatric omnilocus prioritization score (PsyOPS)26, which combines positional mapping with biological annotations, to further prioritize risk genes within genome-wide significant loci. We identified 207 significant genes (Bonferroni correction,P< 2.67 × 10−6) with mBAT-combo and 24 genes using TWAS (P< 4.76 × 10−6), 14 of which were conditionally independent. The SMR–eQTLGen analysis identified 39 significant risk genes (P< 4.28 × 10−6), and the SMR–MetaBrain analysis identified 14 risk genes (P< 9.23 × 10−6). The PWAS identified three significant genes (P< 3.39 × 10−5), while PsyOPS prioritized 29 genes. In total, 251 genes were significantly associated with OCD through at least one gene-based approach, and 48 were implicated by at least two methods (Methods, Supplementary Note7and Supplementary Tables9–14).From the 48 genes implicated by at least two approaches, we prioritized likely causal genes for OCD using colocalization (TWAS-COLOC)27,28and SMR–heterogeneity in dependent instruments (SMR-HEIDI)22tests. Colocalization was used to identify significant TWAS associations for which the underlying GWAS and eQTL summary statistics are likely to share a single causal variant. Similarly, HEIDI was used to select SMR associations for which the same causal variant affects gene expression and trait variation. Of the 48 genes implicated by at least two gene-based tests, 25 were also significant in either the TWAS-COLOC or the SMR-HEIDI tests, suggesting causality (Fig.2a). Only 2 of these 25 genes were prioritized by both TWAS-COLOC and SMR-HEIDI:WDR6(WD repeat domain 6) andDALRD3(DALR anticodon binding domain-containing 3). Another gene of interest,CTNND1(catenin δ1), was implicated by three of our five approaches (multivariate set-based association test (mBAT-combo), TWAS, PWAS) and showed evidence for colocalization. Only three genes were implicated in the PWAS; of these,CTNND1was the only gene also implicated in the TWAS. In the PWAS, downregulation of CTNND1 protein expression in the human dorsolateral prefrontal cortex (dlPFC) was significantly associated with OCD risk (\(Z=-4.49,P=7.11\times {10}^{-6}\); Supplementary Table13), consistent with the downregulation ofCTNND1gene expression in the prefrontal cortex seen in the TWAS (\(Z=-6.86,P=6.90\times {10}^{-12};\)Supplementary Table10). For a discussion of the overlap between the gene findings with rare coding variants in OCD, see Supplementary Table6and Supplementary Note7.Fig. 2: Gene-based, tissue and cell type enrichment analyses.a, List of 25 genes that were implicated in at least two of the five different gene-based tests (significance indicated by gray dots) and passed the TWAS colocalization and/or SMR-HEIDI filters (significance indicated by orange dots). Conditionally independent (cond. ind.) genes within each locus are indicated by blue dots.b, Enrichment of OCD GWAS signal in human brain-related tissues from GTEx (version 8). No significant enrichment was observed in the peripheral tissues (not included in the figure). The horizontal bar size represents the significance of the enrichment measured using the MAGMA gene set enrichment test or partitioned LDSC.c, Top 20 groups of brain cell types (n= 35 total tested) enriched with OCD GWAS signal using MAGMA. Dots represent −log10(Pvalues) from MAGMA gene set enrichment tests of individual neuronal cell types from Zeisel et al.30. Vertical crosses represent the mean −log10(Pvalue) observed for each brain cell type group. Blue crosses represent a significant enrichment of OCD GWAS signals (FDR across 35 groups, FDR < 0.05), while pink crosses indicate nonsignificant enrichment. Gray points represent the association (−log10(Pvalue)) for each single cell cluster (‘level 5’ analysis defined by Zeisel et al.30) in a given cell type (for example, excitatory neurons, cerebral cortex). CCK, cholecystokinin-expressing; R-LM, stratum radiatum-stratum lacunosum-moleculare.Full size imageTissue and cell type enrichment analysisAfter mapping significantly associated SNPs from the GWAS meta-analysis to likely causal genes, we explored which tissues or cell types showed enriched gene expression of OCD-associated genetic signals using a previously described approach29on published human gene expression datasets from bulk tissue RNA-seq data from the Genotype–Tissue Expression (GTEx) project and single-cell RNA-sequencing data from the adult mouse central and peripheral nervous systems30. We found enrichment of OCD GWAS signals in six of 13 human brain tissue types in GTEx but no enrichment in human peripheral tissues (Fig.2band Supplementary Table15). In the adult mouse central and peripheral nervous systems, we found enrichment of OCD GWAS signals in 41 of 166 tested specific single cell types using the MAGMA gene set enrichment test (Supplementary Table16). When summarizing results of individual single cell types into groups of cell types defined by the same region or tissue and cell type, nine of 35 were enriched for OCD GWAS signals (top 20 shown in Fig.2c). Strong enrichment of OCD GWAS signal was especially observed in excitatory neurons of the hippocampus and the cerebral cortex as well as in D1and D2medium spiny neurons (MSNs).Genetic relationship of OCD with other phenotypesUsing phenome-wide association analysis, we examined whether the 30 independent OCD-associated loci identified by our GWAS meta-analysis have previously been associated with other phenotypes (see Supplementary Tables17a–dfor lookups in four, partially overlapping GWAS databanks and Table1for highlighted associations). We found that 22 of the 30 loci were associated with other phenotypes, including schizophrenia (seven loci), depression and major depressive disorder (two loci), bipolar disorder (one locus), neuroticism (seven loci), educational attainment (seven loci) and body fat mass or body mass index (eight loci).We further used bivariate linkage disequilibrium score regression (LDSC)31to investigate the extent of genetic correlations between OCD and 112 previously published GWASs encompassing psychiatric, substance use and neurological phenotypes, among others (Fig.3). We found that 65 phenotypes were significantly correlated with OCD after correcting for multiple testing using the Benjamini–Hochberg32procedure to control the false discovery rate (FDR) at a threshold of 0.05. OCD was significantly positively correlated with all tested psychiatric phenotypes; the highest correlations were with anxiety (\({r}_{\rm{G}}=0.70\)), depression (\({r}_{\rm{G}}=0.60\)), anorexia nervosa (\({r}_{\rm{G}}=0.52\)), Tourette syndrome (\({r}_{\rm{G}}=0.47\)) and post-traumatic stress disorder (PTSD;\({r}_{\rm{G}}=0.48\)). Significant positive genetic correlations were also obtained for neuroticism (\({r}_{\rm{G}}=0.53\)), in particular for the worry subcluster (\({r}_{\rm{G}}=0.64\)), and all individual items in the worry subcluster, with slightly lower estimates for the depressive subcluster (\({r}_{\rm{G}}=0.35\)). Suicide attempt (\({r}_{\rm{G}}=0.40\)), history of childhood maltreatment (\({r}_{\rm{G}}=0.37\)) and tiredness (\({r}_{\rm{G}}=0.36\)) were also notable for strong positive associations with OCD. Of the assessed neurological disorders, OCD was only significantly correlated with migraine (\({r}_{\rm{G}}=0.15\)). Some autoimmune disorders, such as Crohn’s disease (\({r}_{\rm{G}}=-0.13\)), ulcerative colitis (\({r}_{\rm{G}}=-0.14\)) and inflammatory bowel disease (\({r}_{\rm{G}}=-0.14\)), showed negative correlations with OCD (see Fig.3and Supplementary Table18for all genetic correlation estimates, 95% confidence intervals andPvalues, Supplementary Note6for a more in-depth discussion of all significant genetic correlations and Supplementary Table19and Supplementary Figs.41and42for subgroup-specific genetic correlation estimates).Fig. 3: Genetic correlations (rG) between OCD and 112 phenotypes.This includes psychiatric, substance use, cognition–socioeconomic status (SES), personality, psychological, neurological, autoimmune, cardiovascular (cardiovasc.), anthropomorphic–diet, fertility and other phenotypes. References and sample sizes of the corresponding summary statistics of the GWAS studies can be found in Supplementary Table18. The OCD summary statistics are of the main meta-analysis (ncases= 53,660 andncontrols= 2,044,417). Error bars represent the 95% confidence intervals for the genetic correlation estimates (rG). Red circles indicate significant associations witha Pvalue adjusted for multiple testing with the Benjamini–Hochberg procedure to control the FDR (<0.05). Black circles indicate associations that are not significant. a., after; ADHD, attention-deficit hyperactivity disorder; ALS, amyotrophic lateral sclerosis; BMI, body mass index; embarras., embarrassment; freq, frequency; fr., from; HDL, high-density lipoprotein; IQ, intelligence quotient; LDL, low-density lipoprotein; neurot., neuroticism; nr., number; PTSD, post-traumatic stress disorder; sat., satisfaction; VN, verbal-numerical.Full size imageDiscussionThe OCD GWAS reported here, comprising over 53,000 cases, identified 30 independent genome-wide significant loci. Common SNPs explained 6.7% of the variation in OCD risk in our meta-analysis (LDSC with an assumed population prevalence of 1%), a significant reduction from the 28% reported previously10. However, differences in the assumed population prevalence (where a lower assumed prevalence for LDSC heritability calculation results in a lower heritability estimate) and an increase in sample heterogeneity likely contributed to this discrepancy. The reduction in SNP heritability is in line with previous observations for closely related psychiatric disorders such as attention deficit hyperactivity disorder (ADHD)33,34or depression17,35,36,37, where expanding the phenotype definition increased genetic heterogeneity, potentially accounting for the observed decrease in SNP heritability. This aligns with the fact that heritability estimates for more homogeneous OCD subgroups were higher: 16.4% for the clinically ascertained subgroup and 13.3% for the comorbid subgroup (Supplementary Note10). The current estimates are comparable to those of other psychiatric and substance use disorders, with SNP heritability estimates ranging between 9% and 28%38.The most significant SNP (rs78587207(\(P=5.28\times 1{0}^{-12}\))) identified in the GWAS is located on chr11q12.1 and has been previously associated with several traits, including neuropsychiatric phenotypes39such as depressive symptoms40and neuroticism40. Gene-based analyses identified four putative causal genes within this locus. The closest gene tors78587207isCTNND1, which encodes the cell adhesion molecule p120 catenin. This gene was associated with OCD using three gene-based tests (mBAT-combo, TWAS and PWAS), and we found strong evidence for colocalization of the TWAS signal forCTNND1in the dlPFC. The dlPFC has been consistently implicated in the neural circuitry of OCD as well as in compulsivity more broadly as part of the cortico–striatal–thalamo–cortical circuitry41,42. The protein product ofCTNND1is a regulator of cell–cell adhesion43and has a crucial role in gene transcription, Rho GTPase activity and cytoskeletal organization44,45,46. Other credible causal genes in the locus includeCLP1(cleavage factor polyribonucleotide kinase subunit 1),TMX2(thioredoxin-related transmembrane protein 2) andZDHHC5(zinc finger DHHC type palmitoyltransferase 5). Rare genetic mutations inCLP1are associated with pontocerebellar hypoplasia type 10, a very rare autosomal recessive neurodegenerative disease characterized by brain atrophy and delayed myelination resulting in intellectual disability47.TMX2is associated with increased risk of neurodevelopmental disorders with microcephaly, cortical malformations, spasticity and congenital nervous system abnormalities48.ZDHHC5is broadly expressed in the brain, including the frontal cortex.ZDHHC5has not been implicated in brain development but has been linked to lung acinar adenocarcinoma and lung papillary adenocarcinoma in prior studies49.Our finding that approximately 11,500 (s.e. = 607) causal variants account for 90% of the SNP-based heritability of OCD suggests that OCD is more polygenic than other complex traits such as height (ncausal= 4,000), schizophrenia (ncausal= 9,600) and ADHD (ncausal= 5,600) but less polygenic than major depression (ncausal= 14,500) and educational attainment (ncausal= 13,200)50.We identified a total of 25 credible causal genes based on robust evidence using multiple positional and functionally informed gene-based approaches. Notably,DLGAP1, which has been previously implicated in OCD pathogenesis10,51, was not identified in either the GWAS or in the gene-based analyses. Of the 25 genes that were implicated, 15 were within 6.5 kb of a SNP that surpassed genome-wide significance in the meta-analysis. In addition to the four genes discussed above, several others are of particular interest, includingWDR6andDALRD3, which had the strongest evidence from the gene-based analyses. These genes lie in a gene-rich region on chr3p21.31, which, in addition to harboring multiple genome-wide significant SNPs, has been previously associated with a broad range of psychiatric disorders and related traits, including schizophrenia39, well-being52and the worry subcluster of neuroticism53.WDR6is broadly expressed in the brain, particularly the hypothalamus. Its protein product is involved in cell growth arrest54, and recent studies have implicated it in anorexia nervosa55and Parkinson’s disease56.DALRD3is located on chromosome 3 in the same region asWDR6. DALRD3, when fully disrupted, is implicated in a form of epileptic encephalopathy with associated developmental delay57. Finally, a third gene in the 3p21 locus,CELSR3(cadherin EGF LAG seven-pass G type receptor 3), encodes a protocadherin that is highly expressed in the developing basal ganglia58. Multiple loss-of-function mutations inCELSR3have been associated with Tourette syndrome59,60, which co-occurs with OCD in 10–20% of patients.Four other genes identified through these analyses are located in the MHC locus, a region on chromosome 6 that has a major role in the adaptive immune system and has been repeatedly linked to major psychiatric disorders61. The newly identified MHC association for OCD is noteworthy given evidence linking OCD with autoimmune disorders62,63,64. Genetic pleiotropy may underlie this connection, with variants predisposing individuals to both autoimmune conditions and OCD65. Furthermore, some OCD subtypes, such as pediatric acute-onset neuropsychiatric disorders associated withStreptococcusand pediatric acute-onset neuropsychiatric syndrome, may have autoimmune origins66,67. Nevertheless, we were surprised to discover several negative genetic correlations between OCD and autoimmune disorders such as Crohn’s disease, ulcerative colitis and inflammatory bowel disease in our analyses, suggesting that there is heterogeneity (and perhaps pleiotropy) in the genetic relationships between autoimmune disorders and OCD.Tissue and cell type enrichment analysis revealed significant enrichment of OCD SNP heritability in several tissues and cell types, with the strongest enrichment in excitatory neurons of the hippocampus and the cerebral cortex and in dopamine D1receptor (D1R)-positive and dopamine D2receptor (D2R)-positive MSNs in the striatum. These findings are in line with traditional neural circuitry models of OCD, which focus on frontal cortical–striatal pathways68,69. These findings are consistent with and build on previous work linking various neuronal cell types to psychiatric and cognitive phenotypes70.Interestingly, the frontal and anterior cingulate cortices, which were enriched in our tissue-based analyses, as well as the hippocampus and the striatum, which were implicated in our cell type-based analyses, are among the regions that are consistently implicated in neuroimaging studies of OCD41,71,72,73. Enrichment in MSNs in the striatum is consistent with their role in the observed aberrant circuitry in OCD, where the D1MSNs project to the globus pallidus interna and the substantia nigra in the direct pathway and the D2type MSNs project to the globus pallidus externa in the indirect pathway74. However, MSNs are also enriched in major depressive disorder75, schizophrenia76and intelligence77, suggesting that the observed enrichment is not specific for OCD.Our analyses of the shared genetic risk between OCD and other psychiatric disorders provides further insights into the etiology of OCD. In line with previous observations38,78, OCD was significantly genetically correlated with multiple psychiatric disorders and traits. The strongest genetic correlations were observed for anxiety disorders, depression and anorexia nervosa, all of which are highly comorbid with OCD79. This aligns with previous findings from cross-disorder analyses suggesting a shared genetic susceptibility among most psychiatric disorders38,80,81. A notable exception is our finding that risk variants for OCD are protective for alcohol dependence82, which is at odds with epidemiological evidence strongly linking OCD and alcohol-related disorders83but in line with a recent paper79reporting a lower-than-expected lifetime comorbidity of substance use disorders in OCD. The observed pattern of correlations with other phenotypes can be thought of as falling into two categories: compulsivity–impulsivity and rumination–worry–neuroticism. In both categories, the patterns of genetic correlations appear to follow a gradient across disorders and traits. For example, in the compulsivity–impulsivity category, strong positive correlations are seen with anorexia nervosa and Tourette syndrome, which are disorders with strong compulsive features, with less positive associations seen with ADHD and negative correlations with alcohol dependence and risk-taking behaviors, which are all phenotypes characterized by impulsivity. A similar gradient is observed for the rumination–worry–neuroticism-related phenotypes, with strong positive correlations with anxiety and other ruminative phenotypes such as worry, transitioning to less strong correlations with individual depression-related items.This study marks the transition from the flat (sample-building) phase of SNP discovery described for GWAS84(Supplementary Fig.20), where few to no genome-wide significant loci are identified10,12,51,85, to the linear phase of SNP discovery, where even relatively small increases in sample size identify additional genome-wide significant loci18. The strengths of the current study therefore include the marked increase in the number of OCD cases and the rigorous analytic methods, including two multivariate approaches (multi-trait analysis of GWAS (MTAG) and GenomicSEM) to control for potential overlapping study participants and to examine potential heterogeneity between the multiple ascertainment approaches. Potential weaknesses include the inability to document comorbid psychiatric disorders in the majority of cases that were not ascertained from clinical collections or electronic registries, the lack of inclusion of non-European ancestries and the limited availability of sex chromosome data. Owing to the nature of our study, imputation references used in the different cohorts were heterogeneous and did not allow for confident analysis of rare variant associations. Future larger-scale sequencing studies that are currently underway will be needed to identify associations in this allele frequency spectrum. We also note that the genetic correlation analyses are impacted by residual heterogeneity in genetic signals owing to the employment of heterogeneous ascertainment strategies.In summary, this work substantially advances the field of OCD genetics by identifying new OCD genetic risk loci and multiple credible candidate causal genes, including those expressed in brain regions and cell types previously implicated in OCD86. We have also shown that OCD is highly polygenic in nature, with many variants implicated not only in OCD but also in commonly comorbid disorders or traits, in particular, anxiety, neuroticism, anorexia nervosa and depression. The observation that common variants explain only a modest amount of the phenotypic variation in OCD suggests that other types of genetic variation may also contribute to the etiology of OCD. Notably, whole-exome-sequencing studies have suggested that a substantial proportion of OCD cases (22%) may be influenced by rare de novo coding variants87, especially in genes that are intolerant to loss of function88. Similarly, rare potentially damaging copy number variations represent part of the risk architecture for OCD9. These findings emphasize the need for a comprehensive exploration of the contribution of both common and rare genetic factors as well as their interplay to OCD risk. Finally, with the implication of the MHC complex, we provide additional evidence for potential shared genetic influences underlying both OCD and increased liability to autoimmune processes, although the directionality of those relationships remains to be definitively elucidated. In addition to continuing to increase sample sizes, future studies will require ancestrally diverse samples to further facilitate the discovery of additional OCD risk variants. Similarly, sex-specific analyses and additional clinical phenotyping will allow for the further elucidation of genetic and clinical relationships between OCD and co-occurring disorders. Finally, with the emergence of drug databases describing the relations between drugs and molecular phenotypes89, our results may be useful for drug repurposing (that is, identifying existing drugs targeting OCD risk genes), leading to new opportunities to find more effective treatments.MethodsEthicsAll relevant ethics approvals have been obtained by the respective cohort’s institutions, and a list of all respective approvals can be found in Supplementary Note2.Study participantsWe analyzed genomic data from 28 OCD case–control cohorts including 53,660 OCD cases and 2,044,417 controls of European ancestry. Supplementary Table1provides an overview of the individual cohorts. A subset of the cases and controls have been included in previous studies10,51,85and preprints12,13, as described in Supplementary Note2. Among all included individuals, 323 cases were part of a parent–proband trio; in these cases, parents were used as pseudocontrols. A total of 20,427 cases met DSM-5 (ref.93) or ICD-10 (https://icd.who.int/) criteria for OCD as assessed by a healthcare professional or derived from (electronic) health records, while the remaining 32,233 cases were based on self-reported OCD diagnosis (23andMe, AGDS and parts of UKBB). Cohort-specific sample and analytic details can be found in Supplementary Note2. Data collections were approved by the relevant institutional review boards at all participating sites, and all participants provided written informed consent.Individual GWAS analyses and harmonizing of resultsFirst, the data of each participating cohort were analyzed individually (see Supplementary Note2for details). Genetic data were imputed using either the Haplotype Reference Consortium (HRC)94or 1000 Genomes Project Phase 3 reference panels95. The resulting GWAS summary statistics were then harmonized before a conjoint meta-analysis of all autosomes was conducted. Each summary statistic dataset was transformed to the ‘daner’ file format following RICOPILI96specifications. All variants had to meet the following criteria for inclusion: minor allele frequency (MAF) > 1% in cases and controls, INFO score > 0.8 and <1.2. If the effect measure,Pvalue or s.e. was missing or was out of bounds (infinite), the SNP was removed. Once cleaned summary statistics were produced, all datasets were aligned to the HRC reference panel. If variants were reported on different strands, they were flipped to the orientation in the HRC reference. Furthermore, strand-ambiguous A/T and C/G SNPs were removed if their MAF was >0.4. In the case that A/T and C/G SNPs showed a MAF < 0.4, allele frequencies were compared to frequencies in the HRC reference. If an allele frequency match was found, that is, minor alleles were the same in the summary statistics and the HRC reference, the same strand orientation was assumed. If an allele mismatch was found, that is, the allele had a frequency > 0.5 in the HRC reference, it was assumed that alleles were reported on different strands, and alleles were flipped subsequently. Marker names were uniformly switched to those present in the HRC reference. If a variant did not overlap with the variants in the HRC reference, it was removed.GWAS meta-analysisInverse-variance-weighted meta-analysis was conducted on 28 European cohorts using METAL97. Weighting was based on standard error primarily to account for the large case–control imbalances in cohorts that used linear mixed model approaches in their primary GWAS. Heterogeneity was assessed with Cochran’sQstatistic and theI2statistic98,99(see Supplementary Note5for details). The genomic control factor lambda (λ) was calculated for each individual GWAS and for the overall meta-analysis to identify residual population stratification or systematic technical artifacts. GWAS summary statistics were subjected to LDSC analyses on high-quality common SNPs (INFO score > 0.9) to examine the LDSC intercept to distinguish polygenicity from other types of inflation and to estimate the genetic heritability from the meta-analysis and genetic correlations between cohorts. The genomic inflation factorλwas estimated at 1.330 with aλ1000of 1.033, while the LDSC intercept was 1.0155 (s.e. = 0.0085), indicating that the inflation was mostly due to polygenic signal and unlikely to be substantially confounded by population structure. The genome-wide significance threshold for the GWAS was set ata Pvalue of\(5.0\times 1{0}^{-8}\). The 23andMe data included information on the X chromosome; as this information was not present for all other cohorts, analysis of the X chromosome was only conducted in this subcohort (see Supplementary Note4for details).We further conducted GWAS meta-analyses on the following four subgroups, defined by differences in their sample ascertainment: (1) clinical OCD cases diagnosed by a healthcare professional in a clinical setting (ncases= 9,089,ncontrols= 21,077; including IOCDF, IOCDF_trio, EPOC, NORDiC-nor, NORDiC-swe, EGOS, OCGAS, OCGAS-ab, OCGAS-gh, OCGAS-nes, Psych_Broad, WWF, MVP, Michigan/Toronto IGS, YalePenn, Chop, CoGa), (2) comorbid individuals who were primarily ascertained for another comorbid psychiatric disorder (ncases= 5,266,ncontrols= 43,760; AGDS, iPSYCH), (3) biobank data from large-scale biobanks or registries with ICD or DSM codes (ncases= 9,138,ncontrols= 1,049,776; BioVU, EstBB, FinnGen, HUNT, MoBa, UKBB) or (4) 23andMe data (ncases= 30,167,ncontrols= 929,804). While these groups are not exclusive (for example, diagnoses in health records were originally given in a clinical setting or comorbid cases were also assessed in a clinical setting or derived from health records), we defined these groups by the cohort’s primary characteristic. We also conducted one meta-analysis including all clinical, comorbid and biobank subgroups, while excluding the 23andMe data, resulting in 23,493 cases and 1,114,613 controls. As 23andMe is the only consumer-based dataset, we intended to compare this dataset to all others.Number of trait-specific causal variants (MiXeR analysis)We applied MiXeR version 1.3 (ref.15) to quantify the polygenicity of OCD (that is, estimate the total number of trait-influencing genetic variants). MiXeR fits a Gaussian mixture model assuming that common genetic effects on a trait are a mixture of causal variants and noncausal variants. Polygenicity is reported as the number of causal variants that explain 90% of SNP heritability of OCD (to avoid extrapolating model parameters into the area of infinitesimally small effects).SNP-based fine-mapping (GCTA-COJO)We performed a conditional and joint analysis (GCTA-COJO)14to identify independent signals within significant OCD loci. This approach performs a conditional and joint analysis on the basis of conditionalPvalues before calculating the joint effects of all selected SNPs. We used the stepwise model selection procedure to select independently associated SNPs. The linkage disequilibrium reference sample was created from 73,005 individuals from the QIMR Berghofer Medical Research Institute genetic epidemiology cohort. The distance assumed for complete linkage disequilibrium was 10 Mb, and we used the defaultP-value threshold of\(5\times {10}^{-8}\)to define a genome-wide significant hit.Multi-trait analysis of ascertainment subgroupsWe used MTAG100to conduct multivariable GWAS analyses, reporting GWAS results for each of the ascertainment-specific subgroups. Through this approach, we aimed to address potential concerns about heterogeneity in genetic liability for individual subgroups following different ascertainment strategies. MTAG is a multi-trait analysis that is usually used to combine different but related traits into one meta-analysis by leveraging the shared heritability among the different traits and thereby gaining power. In this case, our aim was to generate ascertainment-specific estimates, while boosting power by leveraging the high shared heritability between the subgroups. The MTAG analysis resulted in four different GWAS summary statistics, one for each subgroup (clinical, comorbid, biobanks, 23andMe). We performed maxFDR analyses to approximate the upper bound on the FDR of MTAG results.GenomicSEMSimilarly, we used GenomicSEM16to model the joint genetic architecture of the four subgroups. First, we ran a common factor model without individual SNP effects, following the tutorial ‘Models without individual SNP effects’ on the GenomicSEM GitHub website (Code availability). Second, we ran a multivariate GWAS of the common factor (see Supplementary Note5for details). We specified the model using unit variance identification, for which the latent factor variance is fixed to 1 and the loadings of the traits are estimated freely. This ensures that we capture how much of each subgroup contributes to the latent factor. GenomicSEM also generates\({Q}_{{\rm{SNP}}}\)values, which indicate possible heterogeneous effects across the subgroups. The\({Q}_{{\rm{SNP}}}\)statistic is mathematically similar to theQstatistic from standard meta-analysis and is a\({ X}^{2}\)-distributed test statistic, with larger values indexing a violation of the null hypothesis that the SNP acts entirely through the common factor.SNP heritability estimationThe proportion of the phenotypic variance that could be explained by the aggregated effect of all included SNPs (SNP-based heritability,\({h}_{{\rm{SNP}}}^{2}\)) was estimated using LDSC31. The analysis was performed using precomputed linkage disequilibrium scores from samples restricted to European ancestry in the 1000 Genomes Project95, filtered for SNPs included in the HapMap 3 reference panel101. SNP heritability was estimated based on the slope of the LDSC, with heritability on the liability scale calculated assuming a 1% population prevalence of OCD1. To omit a downward bias in our estimates of liability-scale heritability, following Grotzinger et al.102, we accounted for varying levels of ascertainment across cohorts in our meta-analysis by summing the effective sample sizes across the contributing cohorts and using that as the input sample size for LDSC. For conversion to the liability scale (1%), the sample prevalence was then specified as 0.5. The SNP heritability was calculated for the whole OCD sample as well as for ascertainment-specific subgroups.Genetic correlationsWe used cross-trait LDSC31, a method that computes genetic correlations between GWASs without bias from ancestry differences or sample overlap to calculate genetic correlations between the primary OCD meta-analysis and other phenotypes of interest. The selection of traits was based on phenotypic relevance and/or prior report of a genetic relationship with OCD. The genetic correlation between traits was based on the estimated slope from the regression of the product ofZscores from two GWASs on the linkage disequilibrium score and represents the genetic covariation between two traits based on all polygenic effects captured by the included SNPs. The genome-wide linkage disequilibrium information used by these methods was based on European populations from the HapMap 3 reference panel101, and GWAS summary statistics were filtered to only include SNPs that were part of the 1,290,028 HapMap 3 SNPs.To ensure the internal consistency of the datasets included in our meta-analysis, we calculated genetic correlations between all cohorts we considered to have a sample size large enough for LDSC (effective sample size of ≥1,000) and between the four ascertainment-specific subgroups.We further calculated genetic correlations between OCD and 112 other disorders and traits. The source studies of the GWAS summary statistics can be found in Supplementary Table18. As a follow-up, we also calculated genetic correlations between the 112 phenotypes and each ascertainment-specific subcohort and compared the genetic correlation patterns between the four groups. For all cross-phenotype genetic correlation analyses, we adjustedPvalues for multiple testing using the Benjamini–Hochberg procedure to control for the FDR (<0.05).Gene-based analysesTo match the significant SNPs to the genes for which they likely influence function, we conducted a series of positional and functional gene-mapping analyses. The positional mapping employed MBAT-combo19, while the functional mapping tested whether genetic variants associated with OCD were also associated with differential expression of nearby genes (within a 1-Mb window) using (1) TWAS20using PsychENCODE data and included colocalization with COLOC27,28, and (2) SMR22using whole-blood eQTL information and brain tissues from MetaBrain, alongside the HEIDI test, which tests for heterogeneity in GWAS signal and eQTL association. Furthermore, a PWAS was conducted. As a final step, genes within each locus were prioritized using PsyOPS26, which integrates both positional and functional information. The details of each method are described below.Positional gene mapping (MBAT-combo)A gene-based analysis was conducted using mBAT-combo19within GCTA version 1.94.1 (ref.14). The European subsample (n= 503 individuals) from phase 3 of the 1000 Genomes Project95was used as the linkage disequilibrium reference panel with the fastBAT default linkage disequilibrium cutoff of 0.9 applied. After filtering SNPs with MAF > 0.01, there were 6,629,124 SNPs for analysis in our sample. A gene list consisting of 19,899 protein-coding genes was used to map the base pair position of genes using genome build hg19 (see Supplementary Note7for details).Functional gene mappingTranscriptome-wide association studyWe used TWAS FUSION20to perform a TWAS of OCD. We used brain gene expression weights from the PsychENCODE103and linkage disequilibrium information from the 1000 Genomes Project Phase 3 (ref.95). TWAS FUSION uses reference linkage disequilibrium and reference gene expression panels with GWAS summary statistics to estimate the association between gene expression and OCD risk. These data were processed with the test statistics from the OCD GWAS to estimate the expression–GWAS association statistic. We corrected for multiple testing using Bonferroni correction.We performed colocalization analyses using the COLOC R function27,28implemented in TWAS FUSION. Colocalization is a Bayesian method used to calculate the posterior probabilities (PP) that individual lead SNPs within a significant TWAS locus are (1) independent (for example, two causal SNPs in linkage disequilibrium, one affecting transcription and one affecting OCD; PP3) or (2) share the same associated variant (for example, a single causal SNP affects both transcription and OCD (PP4)). We also performed a conditional analysis to determine whether identified associations represented independent associations. This was performed using the FUSION software, which jointly estimates the effect of all significant features within each locus by using residual SNP associations with OCD after accounting for the predicted expression of other features.Summary-based Mendelian randomizationSMR22was performed using default settings and eQTL meta-analysis summary statistics from European populations for whole blood from eQTLGen23and all five nervous system tissues from MetaBrain (basal ganglia, cerebellum, cortex, hippocampus and spinal cord)104. The HEIDI test was performed alongside SMR to test for effect size heterogeneity between the GWAS and eQTL summary statistics. Both SMR and TWAS have a number of important assumptions and limitations, which we discuss in Supplementary Note9.Psychiatric omnilocus prioritization scoreWe used the gene prioritization method PsyOPS26to rank genes within genome-wide significant loci. This supervised approach integrates biological annotations on mutational intolerance, brain-specific expression and involvement in neurodevelopmental disorder for genes within significant loci. Genes with the top PsyOPS score within each locus were used for further gene prioritization (Gene prioritization). In the instance where two genes in the same locus had the same PsyOPS score, the gene nearest the index SNP was prioritized.Protein-wide association studyWe performed a PWAS using protein expression data from human brain samples. Human brain proteome reference weight data were obtained using the Religious Orders Study and Rush Memory and Aging Project (ROS/MAP) and the Banner Sun Health Research Institute (Banner) study. The ROS/MAP proteomes were generated from the dlPFC of 376 participants of European ancestry and included 1,476 proteins with significant SNP-based heritability (P< 0.01). The Banner PWAS weights were generated from 152 individuals of European ancestry and included 1,147 proteins with significant SNP-based heritability. The PWAS was performed using the TWAS FUSION software20with linkage disequilibrium reference information from the 1000 Genomes Project Phase 3 (ref.95). We corrected for multiple testing using Bonferroni correction.Gene prioritizationWe created a list of prioritized genes using both gene-based tests and colocalization–HEIDI filters. Results from each gene-based test were first restricted to protein-coding genes with unique gene identifiers based on the release from GENCODE (version 40) for hg19. The following criteria were then used to prioritize genes: (1) a significant (Bonferroni-corrected) association from at least two gene-based tests (mBAT-combo, TWAS FUSION, SMR or PsyOPS) and (2) evidence of colocalization (COLOC PP4> 0.8) and/or significant SMR association with HEIDIP> 0.05. Joint–conditional tests of association and significant PWAS associations were used as ancillary approaches to further annotate the prioritized gene list.Tissue and cell type enrichment analysisAn analysis of tissue and cell type enrichment of OCD GWAS association signals was conducted using MAGMA (version 1.08)105and partitioned LDSC106. We used the previously described approach29to determine gene expression specificity in bulk tissue RNA-seq data from 37 tissues in GTEx (version 8) and single-cell RNA-sequencing data from 19 regions in the mouse central and peripheral nervous systems30. The analysis was limited to protein-coding genes with 1:1 orthologs between mice and humans. Gene expression in each tissue or cell type was calculated relative to total expression across all tissues or cell types. Enrichment analysis was performed on genes with the top 10% specificity values in each tissue or cell type, as previously defined29.To evaluate the enrichment of tissue- and cell type-specific genes in OCD genetic association signals, we applied MAGMA and partitioned LDSC. We restricted the analysis to summary statistics for SNPs with a high INFO score (>0.6) and frequency in the entire cohort (MAF > 0.01). Using MAGMA (version 1.08), we tested whether genes with the top 10% specificity in a tissue or cell type showed enrichment in gene-level genetic associations for OCD, with the 1000 Genomes Phase 3 European sample genotypes serving as the linkage disequilibrium reference panel. We used standard gene boundaries (35 kb upstream of the transcription start site to 10 kb downstream of the transcription stop site). Partitioned LDSC was used to examine whether SNPs within 100-kb regions of the top 10% specifically expressed genes were enriched for SNP-based heritability for OCD. All results were corrected for multiple testing with an FDR threshold of 0.05.SNP and gene findings in the context of previous analysesPreviously reported associations for significant SNPs (PheWAS)Multiple resources were used to identify previously reported associations of our 30 significant SNPs with other phenotypes. We used the IEU Open GWAS project92, PheWAS analysis of GWAS ATLAS52and the NHGRI-EBI GWAS Catalog91and identified credible SNPs through CAUSALdb90. CAUSALdb estimates causal probabilities of all genetic variants in GWAS significant loci using three state-of-the-art fine-mapping tools including PAINTOR, CAVIARBF and FINEMAP107,108,109,110. We used default settings for our CAUSALdb queries.Lookup of previous OCD GWAS findingsWe performed a lookup of SNPs identified to be significantly associated with OCD-related phenotypes in previous GWASs. Note that this is not an independent replication, as previous studies partially overlap with the cohorts included in this GWAS.Overlap of previous rare coding variants in OCD and GWAS gene findingsWe performed a bidirectional lookup, assessing (1) whether gene findings from our GWAS showed evidence for rare variant involvement and (2) vice versa, whether findings from rare variant testing showed evidence of common variant association in our GWAS.First, we comprehensively assessed the overlap between 251 genes that we highlighted in our study as carrying common risk variation for OCD (Supplementary Table14) and current gene-based summary statistics from OCD exome-sequencing data. We used data from Halvorsen et al.88because it is the largest published exome-sequencing study of OCD presently. The supplementary materials from that paper include de novo variant calls from 771 case trios and 1,911 controls (supplementary table 14 in ref.88). We compared the burden of de novo variants, partitioned by variant annotation (synonymous, missense, loss of function) in trio cases versus trio controls within these 251 GWAS-prioritized genes. As described previously88, we only included de novo variants that were in loci well covered in both case and control data (In_Jointly_Covered_Loci==TRUE). We also excluded all calls from quartet samples in ref.88(Cohort!=“OCD_JHU_quartets”). For each of the four variant annotation classes, we compared the proportion of cases that had at least one qualifying de novo variant to the proportion of controls using a two-sided Fisher’s exact test.Second, as Halvorsen et al.88describe an overall excess of loss-of-function variants in OCD cases relative to controls specifically within loss-of-function intolerant genes (supplementary table 13 in ref.88), we analyzed the overlap between those genes and our GWAS-derived genes. We looked up 200 genes with a probability of loss-of-function intolerance > 0.995 (derived from ref.111) and effect size estimate > 1. We further tested for a difference in the proportion of these pLI > 0.995 genes with effect size estimate > 1 versus ≤1 within the set of genes highlighted in the OCD GWAS (n= 251) versus outside this set using a two-sided Fisher’s exact test.Reporting summaryFurther information on research design is available in theNature Portfolio Reporting Summarylinked to this article.Data availabilityThe meta-analyzed summary statistics (not including 23andMe data) are available from the Psychiatric Genomics Consortium Download page (https://www.med.unc.edu/pgc/download-results/). In line with 23andMe regulations, 10,000 SNPs from the full GWAS including 23andMe are also being made available athttps://www.med.unc.edu/pgc/download-results/. The full GWAS summary statistics for the 23andMe discovery dataset will be made available through 23andMe to qualified researchers under an agreement with 23andMe that protects the privacy of the 23andMe participants. Datasets will be made available at no cost for academic use. Please visithttps://research.23andme.com/collaborate/#dataset-access/for more information and to apply to access the data. MVP summary statistics are made available through dbGAP request under accessionhttps://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001672.v12.p1phs001672.v12.p1.Code availabilityCore analysis code for RICOPILI can be found athttps://sites.google.com/a/broadinstitute.org/ricopili/. This includes PLINK (https://www.cog-genomics.org/plink2/), EIGENSOFT (https://www.hsph.harvard.edu/alkes-price/software/), Eagle2 (https://alkesgroup.broadinstitute.org/Eagle/), Minimac3 (https://genome.sph.umich.edu/wiki/Minimac3), SHAPEIT3 (https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html), METAL (https://genome.sph.umich.edu/wiki/METAL_Documentation) and LDSR (https://github.com/bulik/ldsc). MAGMA can be found athttps://ctg.cncr.nl/software/magma. GenomicSEM, specifically the tutorial ‘Models without Individual SNP effects’ can be found here:https://github.com/GenomicSEM/GenomicSEM/wiki/3.-Models-without-Individual-SNP-effects. TWAS FUSION can be found athttp://gusevlab.org/projects/fusion/. PWAS: for access to the protein weights, seehttps://www.synapse.org/#!Synapse:syn24872746. GCTA (mBAT-combo and COJO) can be found athttps://yanglab.westlake.edu.cn/software/gcta/#Overview. LDSC and partitioned heritability can be found athttps://github.com/bulik/ldsc. Additional code for data processing (for example, harmonization of summary statistics) can be found athttps://doi.org/10.6084/m9.figshare.28451894(ref.112).ReferencesFawcett, E. J., Power, H. & Fawcett, J. M. Women are at greater risk of OCD than men: a meta-analytic review of OCD prevalence worldwide.J. Clin. Psychiatry81, 19r13085 (2020).ArticlePubMedGoogle ScholarWorld Health Organization.The Global Burden of Disease: 2004 Update(WHO Press, 2008).Fernández de la Cruz, L. et al. Suicide in obsessive–compulsive disorder: a population-based study of 36 788 Swedish patients.Mol. Psychiatry22, 1626–1632 (2017).ArticlePubMedGoogle ScholarMeier, S. M. et al. Mortality among persons with obsessive–compulsive disorder in Denmark.JAMA Psychiatry73, 268–274 (2016).ArticlePubMedPubMed CentralGoogle ScholarBlanco-Vieira, T. et al. The genetic epidemiology of obsessive–compulsive disorder: a systematic review and meta-analysis.Transl. Psychiatry13, 230 (2023).ArticlePubMedPubMed CentralGoogle ScholarBurton, C. L. et al. Heritability of obsessive–compulsive trait dimensions in youth from the general population.Transl. Psychiatry8, 191 (2018).ArticlePubMedPubMed CentralGoogle ScholarPauls, D. L. The genetics of obsessive compulsive disorder: a review of the evidence.Am. J. Med. Genet. C Semin. Med. Genet.148, 133–139 (2008).ArticleGoogle Scholarvan Grootheest, D. S., Cath, D. C., Beekman, A. T. & Boomsma, D. I. Twin studies on obsessive–compulsive disorder: a review.Twin Res. Hum. Genet.8, 450–458 (2005).ArticlePubMedGoogle ScholarMahjani, B. et al. The genetic architecture of obsessive–compulsive disorder: contribution of liability to OCD from alleles across the frequency spectrum.Am. J. Psychiatry179, 216–225 (2022).ArticlePubMedGoogle ScholarInternational Obsessive Compulsive Disorder Foundation Genetics Collaborative (IOCDF-GC) and OCD Collaborative Genetics Association Studies (OCGAS). Revealing the complex genetic architecture of obsessive–compulsive disorder using meta-analysis.Mol. Psychiatry23, 1181–1188 (2018).Davis, L. K. et al. Partitioning the heritability of Tourette syndrome and obsessive compulsive disorder reveals differences in genetic architecture.PLoS Genet.9, e1003864 (2013).ArticlePubMedPubMed CentralGoogle ScholarStrom, N. I. et al. Genome-wide association study identifies new locus associated with OCD. Preprint atmedRxivhttps://doi.org/10.1101/2021.10.13.21261078(2021).Strom, N. I. et al. Genome-wide association study identifies new loci associated with OCD. Preprint atmedRxivhttps://doi.org/10.1101/2024.03.06.24303776(2024).Yang, J., Lee, S. H., Goddard, M. E. & Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis.Am. J. Hum. Genet.88, 76–82 (2011).ArticleCASPubMedPubMed CentralGoogle ScholarHolland, D. et al. Beyond SNP heritability: polygenicity and discoverability of phenotypes estimated with a univariate Gaussian mixture model.PLoS Genet.16, e1008612 (2020).ArticleCASPubMedPubMed CentralGoogle ScholarGrotzinger, A. D. et al. Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits.Nat. Hum. Behav.3, 513–525 (2019).ArticlePubMedPubMed CentralGoogle ScholarWray, N. R. et al. Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression.Nat. Genet.50, 668–681 (2018).ArticleCASPubMedPubMed CentralGoogle ScholarStrom, N. I. et al. Genome-wide association study of major anxiety disorders in 122,341 European-ancestry cases identifies 58 loci and highlights GABAergic signaling. Preprint atmedRxivhttps://doi.org/10.1101/2024.07.03.24309466(2024).Li, A. et al. mBAT-combo: a more powerful test to detect gene–trait associations from GWAS data.Am. J. Hum. Genet.110, 30–43 (2023).ArticleCASPubMedPubMed CentralGoogle ScholarGusev, A. et al. Integrative approaches for large-scale transcriptome-wide association studies.Nat. Genet.48, 245–252 (2016).ArticleCASPubMedPubMed CentralGoogle ScholarGandal, M. J. et al. Shared molecular neuropathology across major psychiatric disorders parallels polygenic overlap.Science359, 693–697 (2018).ArticleCASPubMedPubMed CentralGoogle ScholarZhu, Z. et al. Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets.Nat. Genet.48, 481–487 (2016).ArticleCASPubMedGoogle ScholarVõsa, U. et al. Large-scalecis- andtrans-eQTL analyses identify thousands of genetic loci and polygenic scores that regulate blood gene expression.Nat. Genet.53, 1300–1310 (2021).ArticlePubMedPubMed CentralGoogle ScholarQi, T. et al. Identifying gene targets for brain-related traits using transcriptomic and methylomic data from blood.Nat. Commun.9, 2282 (2018).ArticlePubMedPubMed CentralGoogle ScholarWingo, T. S. et al. Brain proteome-wide association study implicates novel proteins in depression pathogenesis.Nat. Neurosci.24, 810–817 (2021).ArticleCASPubMedPubMed CentralGoogle ScholarWainberg, M., Merico, D., Keller, M. C., Fauman, E. B. & Tripathy, S. J. Predicting causal genes from psychiatric genome-wide association studies using high-level etiological knowledge.Mol. Psychiatry27, 3095–3106 (2022).ArticlePubMedGoogle ScholarGiambartolomei, C. et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics.PLoS Genet.10, e1004383 (2014).ArticlePubMedPubMed CentralGoogle ScholarWallace, C. Eliciting priors and relaxing the single causal variant assumption in colocalisation analyses.PLoS Genet.16, e1008720 (2020).ArticleCASPubMedPubMed CentralGoogle ScholarBryois, J. et al. Genetic identification of cell types underlying brain complex traits yields insights into the etiology of Parkinson’s disease.Nat. Genet.52, 482–493 (2020).ArticleCASPubMedPubMed CentralGoogle ScholarZeisel, A. et al. Molecular architecture of the mouse nervous system.Cell174, 999–1014 (2018).ArticleCASPubMedPubMed CentralGoogle ScholarBulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies.Nat. Genet.47, 291–295 (2015).ArticleCASPubMedPubMed CentralGoogle ScholarBenjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing.J. R. Stat. Soc. B57, 289–300 (1995).ArticleGoogle ScholarDemontis, D. et al. Discovery of the first genome-wide significant risk loci for attention deficit/hyperactivity disorder.Nat. Genet.51, 63–75 (2019).ArticleCASPubMedGoogle ScholarDemontis, D. et al. Genome-wide analyses of ADHD identify 27 risk loci, refine the genetic architecture and implicate several cognitive domains.Nat. Genet.55, 198–208 (2023).ArticleCASPubMedPubMed CentralGoogle ScholarHall, L. S. et al. Genome-wide meta-analyses of stratified depression in Generation Scotland and UK Biobank.Transl. Psychiatry8, 9 (2018).ArticlePubMedPubMed CentralGoogle ScholarAls, T. D. et al. Depression pathophysiology, risk prediction of recurrence and comorbid psychiatric disorders using genome-wide analyses.Nat. Med.29, 1832–1844 (2023).ArticleCASPubMedPubMed CentralGoogle ScholarCai, N. et al. Minimal phenotyping yields genome-wide association signals of low specificity for major depression.Nat. Genet.52, 437–447 (2020).ArticleCASPubMedPubMed CentralGoogle ScholarDerks, E. M., Thorp, J. G. & Gerring, Z. F. Ten challenges for clinical translation in psychiatric genetics.Nat. Genet.54, 1457–1465 (2022).ArticleCASPubMedGoogle ScholarPardiñas, A. F. et al. Common schizophrenia alleles are enriched in mutation-intolerant genes and in regions under strong background selection.Nat. Genet.50, 381–389 (2018).ArticlePubMedPubMed CentralGoogle ScholarBaselmans, B. M. L. et al. Multivariate genome-wide analyses of the well-being spectrum.Nat. Genet.51, 445–451 (2019).ArticleCASPubMedGoogle Scholarvan den Heuvel, O. A. et al. Brain circuitry of compulsivity.Eur. Neuropsychopharmacol.26, 810–827 (2016).ArticlePubMedGoogle ScholarShephard, E. et al. Toward a neurocircuit-based taxonomy to guide treatment of obsessive–compulsive disorder.Mol. Psychiatry26, 4583–4604 (2021).ArticlePubMedPubMed CentralGoogle ScholarDavis, M. A., Ireton, R. C. & Reynolds, A. B. A core function for p120-catenin in cadherin turnover.J. Cell Biol.163, 525–534 (2003).ArticleCASPubMedPubMed CentralGoogle ScholarYanagisawa, M. et al. A p120 catenin isoform switch affects Rho activity, induces tumor cell invasion, and predicts metastatic disease.J. Biol. Chem.283, 18344–18354 (2008).ArticleCASPubMedPubMed CentralGoogle ScholarDaniel, J. M. & Reynolds, A. B. The catenin p120ctninteracts with Kaiso, a novel BTB/POZ domain zinc finger transcription factor.Mol. Cell. Biol.19, 3614–3623 (1999).ArticleCASPubMedPubMed CentralGoogle ScholarIshiyama, N. et al. Dynamic and static interactions between p120 catenin and E-cadherin regulate the stability of cell–cell adhesion.Cell141, 117–128 (2010).ArticleCASPubMedGoogle ScholarSchaffer, A. E. et al.CLP1founder mutation links tRNA splicing and maturation to cerebellar development and neurodegeneration.Cell157, 651–663 (2014).ArticleCASPubMedPubMed CentralGoogle ScholarVandervore, L. V. et al. TMX2 is a crucial regulator of cellular redox state, and its dysfunction causes severe brain developmental abnormalities.Am. J. Hum. Genet.105, 1126–1147 (2019).ArticleCASPubMedPubMed CentralGoogle ScholarZhang, Y. et al. Potential role ofS-palmitoylation in cancer stem cells of lung adenocarcinoma.Front. Cell Dev. Biol.9, 734897 (2021).ArticlePubMedPubMed CentralGoogle ScholarHindley, G. et al. Charting the landscape of genetic overlap between mental disorders and related traits beyond genetic correlation.Am. J. Psychiatry179, 833–843 (2022).ArticlePubMedPubMed CentralGoogle ScholarMattheisen, M. et al. Genome-wide association study in obsessive–compulsive disorder: results from the OCGAS.Mol. Psychiatry20, 337–344 (2015).ArticleCASPubMedGoogle ScholarWatanabe, K. et al. A global overview of pleiotropy and genetic architecture in complex traits.Nat. Genet.51, 1339–1348 (2019).ArticleCASPubMedGoogle ScholarNagel, M., Watanabe, K., Stringer, S., Posthuma, D. & van der Sluis, S. Item-level analyses reveal genetic heterogeneity in neuroticism.Nat. Commun.9, 905 (2018).ArticlePubMedPubMed CentralGoogle ScholarXie, X., Wang, Z. & Chen, Y. Association of LKB1 with a WD-repeat protein WDR6 is implicated in cell growth arrest and p27Kip1induction.Mol. Cell. Biochem.301, 115–122 (2007).ArticleCASPubMedGoogle ScholarAdams, D. M., Reay, W. R. & Cairns, M. J. Multiomic prioritisation of risk genes for anorexia nervosa.Psychol. Med.53, 6754–6762 (2023).ArticlePubMedPubMed CentralGoogle ScholarKia, D. A. et al. Identification of candidate Parkinson disease genes by integrating genome-wide association study, expression, and epigenetic data sets.JAMA Neurol.78, 464–472 (2021).ArticlePubMedGoogle ScholarLentini, J. M., Alsaif, H. S., Faqeih, E., Alkuraya, F. S. & Fu, D.DALRD3encodes a protein mutated in epileptic encephalopathy that targets arginine tRNAs for 3-methylcytosine modification.Nat. Commun.11, 2510 (2020).ArticleCASPubMedPubMed CentralGoogle ScholarWu, J., Poppi, L. A. & Tischfield, M. A. Planar cell polarity and the pathogenesis of Tourette disorder: new hypotheses and perspectives.Dev. Biol.489, 14–20 (2022).ArticleCASPubMedGoogle ScholarWillsey, A. J. et al. De novo coding variants are strongly associated with Tourette disorder.Neuron94, 486–499 (2017).ArticleCASPubMedPubMed CentralGoogle ScholarZhao, X. et al. A whole-exome sequencing study of Tourette disorder in a Chinese population.DNA Cell Biol.39, 63–68 (2020).ArticlePubMedGoogle ScholarDebnath, M., Berk, M., Leboyer, M. & Tamouza, R. The MHC/HLA gene complex in major psychiatric disorders: emerging roles and implications.Curr. Behav. Neurosci. Rep.5, 179–188 (2018).ArticleGoogle ScholarMataix-Cols, D. et al. A total-population multigenerational family clustering study of autoimmune diseases in obsessive–compulsive disorder and Tourette’s/chronic tic disorders.Mol. Psychiatry23, 1652–1658 (2018).ArticleCASPubMedGoogle ScholarTylee, D. S. et al. Genetic correlations among psychiatric and immune-related phenotypes based on genome-wide association data.Am. J. Med. Genet. B Neuropsychiatr. Genet.177, 641–657 (2018).ArticlePubMedPubMed CentralGoogle ScholarWestwell-Roper, C. et al. Immune-related comorbidities in childhood-onset obsessive compulsive disorder: lifetime prevalence in the Obsessive Compulsive Disorder Collaborative Genetics Association Study.J. Child Adolesc. Psychopharmacol.29, 615–624 (2019).ArticlePubMedPubMed CentralGoogle ScholarZhang, T. et al. Prenatal and early childhood infections and subsequent risk of obsessive–compulsive disorder and tic disorders: a nationwide, sibling-controlled study.Biol. Psychiatry93, 1023–1030 (2023).ArticlePubMedGoogle ScholarSwedo, S. E. et al. Pediatric autoimmune neuropsychiatric disorders associated with streptococcal infections: clinical description of the first 50 cases.Am. J. Psychiatry155, 264–271 (1998).ArticleCASPubMedGoogle ScholarWilbur, C. et al. PANDAS/PANS in childhood: controversies and evidence.Paediatr. Child Health24, 85–91 (2019).ArticlePubMedGoogle ScholarAhmari, S. E. et al. Repeated cortico–striatal stimulation generates persistent OCD-like behavior.Science340, 1234–1239 (2013).ArticleCASPubMedPubMed CentralGoogle ScholarAhmari, S. E. & Rauch, S. L. The prefrontal cortex and OCD.Neuropsychopharmacology47, 211–224 (2022).ArticlePubMedGoogle ScholarOlislagers, M., Rademaker, K., Adan, R. A. H., Lin, B. D. & Luykx, J. J. Comprehensive analyses of RNA-seq and genome-wide data point to enrichment of neuronal cell type subsets in neuropsychiatric disorders.Mol. Psychiatry27, 947–955 (2022).ArticleCASPubMedGoogle ScholarBoedhoe, P. S. W. et al. Cortical abnormalities associated with pediatric and adult obsessive–compulsive disorder: findings from the ENIGMA Obsessive–Compulsive Disorder Working Group.Am. J. Psychiatry175, 453–462 (2018).ArticlePubMedGoogle ScholarBruin, W. B. et al. Structural neuroimaging biomarkers for obsessive–compulsive disorder in the ENIGMA-OCD consortium: medication matters.Transl. Psychiatry10, 342 (2020).ArticlePubMedPubMed CentralGoogle Scholarvan den Heuvel, O. A. et al. An overview of the first 5 years of the ENIGMA obsessive–compulsive disorder working group: the power of worldwide collaboration.Hum.BrainMapp.43, 23–36 (2022).Google ScholarHaber, S. N. Corticostriatal circuitry.DialoguesClin. Neurosci.18, 7–21 (2016).Google ScholarMajor Depressive Disorder Working Group of the Psychiatric Genomics Consortium. Trans-ancestry genome-wide study of depression identifies 697 associations implicating cell types and pharmacotherapies.Cell188, 640–652 (2025).ArticlePubMed CentralGoogle ScholarTrubetskoy, V. et al. Mapping genomic loci implicates genes and synaptic biology in schizophrenia.Nature604, 502–508 (2022).ArticleCASPubMedPubMed CentralGoogle ScholarSavage, J. E. et al. Genome-wide association meta-analysis in 269,867 individuals identifies new genetic and functional links to intelligence.Nat. Genet.50, 912–919 (2018).ArticleCASPubMedPubMed CentralGoogle ScholarRomero, C. et al. Exploring the genetic overlap between twelve psychiatric disorders.Nat. Genet.54, 1795–1802 (2022).ArticleCASPubMedGoogle ScholarSharma, E. et al. Comorbidities in obsessive–compulsive disorder across the lifespan: a systematic review and meta-analysis.Front. Psychiatry12, 703701 (2021).ArticlePubMedPubMed CentralGoogle ScholarLee, P. H. et al. Genomic relationships, novel loci, and pleiotropic mechanisms across eight psychiatric disorders.Cell179, 1469–1482 (2019).ArticleGoogle ScholarGrotzinger, A. D. Shared genetic architecture across psychiatric disorders.Psychol. Med.51, 2210–2216 (2021).ArticlePubMedPubMed CentralGoogle ScholarWalters, R. K. et al. Transancestral GWAS of alcohol dependence reveals common genetic underpinnings with psychiatric disorders.Nat. Neurosci.21, 1656–1669 (2018).ArticleCASPubMedPubMed CentralGoogle ScholarVirtanen, S. et al. Association of obsessive–compulsive disorder and obsessive–compulsive symptoms with substance misuse in 2 longitudinal cohorts in Sweden.JAMA Netw. Open5, e2214779 (2022).ArticlePubMedPubMed CentralGoogle ScholarSullivan, P. F. et al. Psychiatric genomics: an update and an agenda.Am. J. Psychiatry175, 15–27 (2018).ArticlePubMedGoogle ScholarStewart, S. E. et al. Genome-wide association study of obsessive–compulsive disorder.Mol. Psychiatry18, 788–798 (2013).ArticleCASPubMedGoogle ScholarPiantadosi, S. C. et al. Transcriptome alterations are enriched for synapse-associated genes in the striatum of subjects with obsessive–compulsive disorder.Transl. Psychiatry11, 171 (2021).ArticleCASPubMedPubMed CentralGoogle ScholarCappi, C. et al. De novo damaging DNA coding mutations are associated with obsessive–compulsive disorder and overlap with Tourette’s disorder and autism.Biol. Psychiatry87, 1035–1044 (2020).ArticleCASPubMedGoogle ScholarHalvorsen, M. et al. Exome sequencing in obsessive–compulsive disorder reveals a burden of rare damaging coding variants.Nat. Neurosci.24, 1071–1076 (2021).ArticleCASPubMedGoogle ScholarKnox, C. et al. DrugBank 6.0: the DrugBank Knowledgebase for 2024.Nucleic Acids Res.52, D1265–D1275 (2024).ArticleCASPubMedGoogle ScholarWang, J. et al. CAUSALdb: a database for disease/trait causal variants identified using summary statistics of genome-wide association studies.Nucleic Acids Res.48, D807–D816 (2020).CASPubMedGoogle ScholarBuniello, A. et al. The NHGRI-EBI GWAS Catalog of published genome-wide association studies, targeted arrays and summary statistics 2019.Nucleic Acids Res.47, D1005–D1012 (2019).ArticleCASPubMedGoogle ScholarElsworth, B. et al. The MRC IEU OpenGWAS data infrastructure. Preprint atbioRxivhttps://doi.org/10.1101/2020.08.10.244293(2020).DSM-5.Diagnostic and Statistical Manual of Mental Disorders: DSM-5237–242 (American Psychiatric Association, 2013).McCarthy, S. et al. A reference panel of 64,976 haplotypes for genotype imputation.Nat. Genet.48, 1279–1283 (2016).ArticleCASPubMedPubMed CentralGoogle ScholarThe 1000 Genomes Project Consortium. A global reference for human genetic variation.Nature526, 68–74 (2015).ArticleGoogle ScholarLam, M. et al. RICOPILI: Rapid Imputation for Consortias Pipeline.Bioinformatics36, 930–933 (2020).ArticleCASPubMedGoogle ScholarWiller, C. J., Li, Y. & Abecasis, G. R. METAL: fast and efficient meta-analysis of genomewide association scans.Bioinformatics26, 2190–2191 (2010).ArticleCASPubMedPubMed CentralGoogle ScholarHiggins, J. P. T. & Thompson, S. G. Quantifying heterogeneity in a meta-analysis.Stat. Med.21, 1539–1558 (2002).ArticlePubMedGoogle ScholarHiggins, J. P. T., Thompson, S. G., Deeks, J. J. & Altman, D. G. Measuring inconsistency in meta-analyses.BMJ327, 557–560 (2003).Google ScholarTurley, P. et al. Multi-trait analysis of genome-wide association summary statistics using MTAG.Nat. Genet.50, 229–237 (2018).ArticleCASPubMedPubMed CentralGoogle ScholarAltshuler, D. M. et al. Integrating common and rare genetic variation in diverse human populations.Nature467, 52–58 (2010).ArticleCASPubMedGoogle ScholarGrotzinger, A. D., de la Fuente, J., Privé, F., Nivard, M. G. & Tucker-Drob, E. M. Pervasive downward bias in estimates of liability-scale heritability in genome-wide association study meta-analysis: a simple solution.Biol. Psychiatry93, 29–36 (2023).ArticleCASPubMedGoogle ScholarWang, D. et al. Comprehensive functional genomic resource and integrative model for the human brain.Science362, eaat8464 (2018).ArticleCASPubMedPubMed CentralGoogle Scholarde Klein, N. et al. Brain expression quantitative trait locus and network analyses reveal downstream effects and putative drivers for brain-related diseases.Nat. Genet.55, 377–388 (2023).ArticlePubMedPubMed CentralGoogle Scholarde Leeuw, C. A., Mooij, J. M., Heskes, T. & Posthuma, D. MAGMA: generalized gene-set analysis of GWAS data.PLoS Comput. Biol.11, e1004219 (2015).ArticlePubMedPubMed CentralGoogle ScholarFinucane, H. K. et al. Partitioning heritability by functional annotation using genome-wide association summary statistics.Nat. Genet.47, 1228–1235 (2015).ArticleCASPubMedPubMed CentralGoogle ScholarBenner, C. et al. FINEMAP: efficient variable selection using summary data from genome-wide association studies.Bioinformatics32, 1493–1501 (2016).ArticleCASPubMedPubMed CentralGoogle ScholarChen, W. et al. Fine mapping causal variants with an approximate Bayesian method using marginal test statistics.Genetics200, 719–736 (2015).ArticlePubMedPubMed CentralGoogle ScholarKichaev, G. et al. Integrating functional data to prioritize causal variants in statistical fine-mapping studies.PLoS Genet.10, e1004722 (2014).ArticlePubMedPubMed CentralGoogle ScholarKichaev, G. & Pasaniuc, B. Leveraging functional-annotation data in trans-ethnic fine-mapping studies.Am. J. Hum. Genet.97, 260–271 (2015).ArticleCASPubMedPubMed CentralGoogle ScholarLek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans.Nature536, 285–291 (2016).ArticleCASPubMedPubMed CentralGoogle ScholarMattheisen, M. PGC OCD3 code.Figsharehttps://doi.org/10.6084/m9.figshare.28451894.v1(2025).Download referencesAcknowledgementsWe thank the research participants and employees of all cohorts included in this study for making this work possible. A list of members of the 23andMe Research Team, HUNT, CoGa and the MVP who contributed to this study can be found in Supplementary Note1. We thank the research participants and employees of 23andMe for making this work possible. The Trøndelag Health Study (HUNT) is a collaboration between the HUNT Research Centre (Faculty of Medicine and Health Sciences, NTNU), Trøndelag County Council, Central Norway Regional Health Authority and the Norwegian Institute of Public Health. Genotype quality control and imputation were conducted by the K.G. Jebsen Center for Genetic Epidemiology, Department of Public Health and Nursing, Faculty of Medicine and Health Sciences, NTNU. HUNT analyses were performed in digital laboratories at HUNT Cloud, HUNT Research Centre Department of Public Health and Nursing, Faculty of Medicine and Health Sciences, NTNU, Trondheim, Norway. NORDiC is funded by National Institute of Mental Health (NIMH) R01 MH110427 (PI J.J.C.), NIMH R01 MH105500 (PI J.J.C.) and the Swedish Research Council grant 2015–02271 (PI D.M.-C.). NORDiC was further supported by the Swedish Research Council (grants 2012-07111 and 2018-02487), Swedish Research Council for Health, Working Life and Welfare 2018-00221 and the Center for Innovative Medicine (CIMED). We are deeply grateful for the study participants contributing to the NORDiC research. We thank the collection team that worked to recruit them: A. Juréus, J. Pege, M. Rådström, R. Satgunanthan-Dawoud, M. Krestelica and B. Ohlander, as well as the data manager B. Iliadou. We also thank the National Quality Registry for Eating Disorders (RIKSÄT) for help with recruiting patients. We finally thank the BBMRI.se and KI Biobank at Karolinska Institutet for professional biobank services. Grant support for the MoBa team was also provided from RCN (273291, 262656, 248778, 223273) and the KG Jebsen Stiftelsen. MoBa is supported by the Norwegian Ministry of Health and Care Services and the Ministry of Education and Research. We are grateful to all the participating families in Norway who take part in this ongoing cohort study. The AGDS was primarily funded by National Health and Medical Research Council (NHMRC) of Australia grant 1086683. This work was further supported by NHMRC grants 1145645, 1078901 and 1087889. We thank all the people who helped in the conception, implementation, beta testing, media campaign and data cleaning of the AGDS data. We specifically acknowledge D. Nyholt for advice on using the PBS for research; K. Kendler, P. Sullivan, A. McIntosh and C. Lewis for input on the questionnaire; L. Nunn, M. Ferguson, L. Winkler and N. Garden for data and sample collection; N. Zmicerevska, A. Nichles and C. Brennan for participant recruitment support; J. Davies, L. Lowrey and V. Antonini for support with IT aspects; and V. Morgan and K. Kirkby for help with the media campaign. We thank VIVA! Communications for their effort in promoting the study. We also acknowledge D. Whiteman and C. Olsen from QSkin. The work done by the EstBB team has received funding from the European Union’s Horizon 2020 Research and Innovation Programme under grant agreement 847776 (CoMorMent). Data analysis for EstBB was carried out in part at the High-Performance Computing Center of the University of Tartu. We thank participants, families and staff of primary and secondary schools who kindly contribute to this research (M.R., Metal-Cat and INSchool). EGOS was supported by a grant from the Beatrice and Samuel A. Seaver Foundation to D.E.G. The genotyping of HUNT was financed by the National Institutes of Health (NIH), the University of Michigan, the Norwegian Research Council and Central Norway Regional Health Authority and the Faculty of Medicine and Health Sciences, NTNU. This research is based in part on data from the MVP, Office of Research and Development, Veterans Health Administration and was supported by awards CSP575b, I01CX001849-01 and 1P1HX002375 and the National Center for PTSD Research. The MVP was supported by funding from the Department of Veterans Affairs Office of Research and Development, USVA, grants CSP575B and I01CX001849, MVP-025 and the VA Cooperative Studies Program study no. 575B, the VA National Center for PTSD Research and the West Haven VA Mental Illness Research, Education and Clinical Center and by NIH grant R01 AA026364 (J. Gelernter). This publication does not represent the views of the Department of Veterans Affairs or the United States government. The EPOC study was funded by the Deutsche Forschungsgemeinschaft (DFG; KA815/6-1 and WA731/10-1). LifeGene was supported by the Swedish Research Council, Karolinska Institutet–Stockholm County Council research grants, AFA Insurance and the Torsten and Ragnar Söderbergs Foundation. GENOS was supported by the German Research Foundation (GR 1912/1-1). The OCD Collaborative Genetics Association Study (OCGAS) is a collaborative research study and was funded by the following NIMH grant numbers: MH071507, MH079489, MH079487, MH079488 and MH079494. This work (OCGAS and IOCDF) is supported by the Netherlands Organization for Scientific Research—Gravitation project ‘BRAINSCAPES: a Roadmap from Neurogenetics to Neurobiology’ (024.004.012) and the European Research Council advanced grant ‘From GWAS to Function’ (2018-ADG 834057). The OCGAS and IOCDF samples are supported through NIMH grant numbers MH071507 (G.N.), MH079489 (D.A.G.), MH079487 (J.T.M.), MH079488 (A.J.F.) and MH079494 (J.A.K.). The iPSYCH team was supported by grants from the Lundbeck Foundation (R102-A9118, R155-2014-1724 and R248-2017-2003), the NIH/NIMH (1R01MH124851-01 to A.D.B.) and the universities and university hospitals of Aarhus and Copenhagen. The Danish National Biobank resource was supported by the Novo Nordisk Foundation. High-performance computer capacity for handling and statistical analysis of iPSYCH data in the GenomeDK HPC facility was provided by the Center for Genomics and Personalized Medicine and the Centre for Integrative Sequencing, iSEQ, Aarhus University, Denmark (grant to A.D.B.). A.D.B. was also supported by the EU’s HORIZON-HLTH-2021-STAYHLTH-01 program, project number 101057385: Risk and Resilience in Developmental Diversity and Mental Health (R2D2-MH). Mental-Cat and INSchool were supported by the Agència de Gestió d’Ajuts Universitaris i de Recerca (2017SGR-1461, 2021SGR-00840), the Instituto de Salud Carlos III (PI20/00041, PI23/00404 and PI23/00026), the European Regional Development Fund and the ECNP Network (‘ADHD across the Lifespan’ and ‘la Marató de TV3’, 202228-30 and 202228-31). BioVU: CTSA (S.D., Vanderbilt Resources) was supported by the National Center for Research Resources, grant UL1 RR024975-01 and is now at the National Center for Advancing Translational Sciences (grant 2 UL1 TR000445-06). The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH. The dataset(s) used for the analyses (BioVU) described were obtained from Vanderbilt University Medical Center’s BioVU, which is supported by numerous sources: institutional funding, private agencies and federal grants. These include the NIH-funded Shared Instrumentation Grant S10RR025141 and CTSA grants UL1 TR002243, UL1 TR000445 and UL1 RR024975. Genomic data are also supported by investigator-led projects that include U01HG004798, R01NS032830, RC2GM092618, P50GM115305, U01HG006378, U19HL065962 and R01HD074711 and additional funding sources listed athttps://victr.vumc.org/biovu-funding/. S.A. acknowledges a Miguel Servet contract (CP22/00026) awarded by the Instituto de Salud Carlos III and cofunded by the European Union Fund: Fondo Social Europeo Plus, FSE+. HYPERGENES and InterOmics cohorts provided controls of Italian origin for the present study. J.C.-D. acknowledges her contract from the Network Center for Biomedical Research (CIBER). R.D. acknowledges the Clinical Investigation Centre, Robert Debré Hospital. INSERM at APHP granted the study. B.T.F. acknowledges the Anorexia Nervosa Genetics Initiative, an initiative of the Klarman Family Foundation. J.H. acknowledges the Trond Mohn Foundation, Bergen, Norway. C. Lochner acknowledges the South African Medical Research Council and the National Research Foundation for their support. T.V.F: research reported in this publication was supported by the NIMH of the NIH under award number R01MH114927. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH. N.G.M. has received funding from a project grant from the Australian NHMRC. The Research Council of Norway supported H.A., A.H. and T.R.-K. (274611). A. Havdahl was also supported by South East Norway Health Authority (2020022). Z.F.G. is supported by an Australian NHMRC EL1 Investigator Grant (2034743) and NIH/NIA grant AG068026. M.G. received support from the following grants (J. Gelernter): CSP575b, I01CX001849-01, 1P1HX002375, the National Center for PTSD Research, 5R01DA054869-01. A. Abdellaoui was supported by the Foundation Volksbond Rotterdam. T.B. is supported by NIMH grant 7R01MH103657 (GPC-OCD). J.R.C.: this study represents independent research partly funded by the National Institute for Health Research Maudsley Biomedical Research Centre at South London and Maudsley NHS Foundation Trust and King’s College London. The views expressed are those of the authors and not necessarily those of the NHS, the National Institute for Health Research or the Department of Health and Social Care. C.B. was supported by grant EU FP7-HEALTH-2007-A-201550 and grant MIUR-CNR PB05. E.M.B. was supported by NHMRC project grant 1145645 and the University of Queensland Health Research Accelerator Program. C.C. was supported by grant K99MH128540-01A1. V.C. was supported by the Italian Ministry of Health grant RC18-19-20-21/A. M.A.G. was supported by NIMH K23 MH066284. J.H. was supported by Stiftelsen KG Jebsen (SKGJ MED-02). K. Hagen was supported by the Trond Mohn Foundation. E.K.K. was supported by NIH R21 MH109938. P.S.N. was supported by R01MH071507. Fabrizio and Federica Piras are supported by the Italian Ministry of Health RC18-19-20-21/A grant. This work was in part supported by the German Research Foundation (DFG) grants RA1971/8-1 and RA1971/7-1 and by the Bundesministerium für Bildung und Forschung grant 01ED2007A to A. Ramirez. S. Ripke was supported by research grant 1R01MH124873-01. M.S.A. was supported by the Instituto de Salud Carlos III (P19/01224, PI22/00464 and CP22/00128) and the European Regional Development Fund. A. Agrawal was supported by grant U10AA008401. P.A. was supported by the Spanish Ministry of Science, Innovation and Universities (ISCIII PI22/00752) and Fundació La Marató 202201-30. C.M.B. was supported by R01 MH124871 (Sullivan and Bulik) PGC4. H.E. was supported by grant U10AA008401. D.A.G. was supported by the NIMH (OCGAS and OCGS). G.L.H. was supported by the NIMH (R01 MH58376, K20 MH01065, R01 MH101493, R01 MH085321). N.K. has received funding from DFG KA815/6-1. S.E.M. is supported by an Australian NHMRC Investigator Grant (APP1172917). B.M.N. is funded by grant R01MH124851. M.P. and C. Pato have received support from R01MH103657 and R01MH079494 from the NIMH and the Della Martin Foundation, Los Angeles, CA. J.P. has received support through the NIMH (R01MH50214: Collaborative OCD Genetics Study (G.N., PI; J.T.M., UCLA PI)). M.A. Richter was supported by funding from the Canadian Institutes for Health Research and the Ontario Mental Health Foundation. D.R.R. was supported by NIMH R01MH059299. J.F.S. was supported by NIMH grant number MH071507. G.S. is supported by the Italian Ministry of Health RC18-19-20-21/A grant. E.A.S. collected data as part of the NIH grant 1R01MH093381. O.A.A. (MoBa) has received grant support from RCN (324499, 273291, 262656, 248778, 223273), KG Jebsen Stiftelsen and NordForsk 164218. J. Kaprio has been supported by the Academy of Finland (grant 336823). P.D.A. is supported by the Alberta Innovates Translational Health Chair in Child and Youth Mental Health. D.E.G. is supported by grant MH124679-01. J.A.K. is supported by the grants R01MH103657 and R01MH079494 from the NIMH and the Della Martin Foundation, Los Angeles, CA. K.J.V. is supported by the Foundation Volksbond Rotterdam. L.K.D. was supported by grants from the NIH including R01NS102371, R01MH113362, R01MH118223, R01NS105746 and R56MH120736. J.S. was supported by an NIH Training Grant in Human Genetics (2T32GM080178). J.J.C. was supported by NIH grants R01 MH105500 and R01 MH110427. M.B.S. has been funded by the US Veterans Affairs Administration.FundingOpen access funding provided by Humboldt-Universität zu Berlin.Author informationAuthor notesThese authors contributed equally: Nora I. Strom, Zachary F. Gerring, Marco Galimberti, Dongmei Yu.These authors jointly supervised this work: Jeremiah M. Scharf, Murray B. Stein, Joel Gelernter, Carol A. Mathews, Eske M. Derks, Manuel Mattheisen.Authors and AffiliationsDepartment of Psychology, Humboldt-Universität zu Berlin, Berlin, GermanyNora I. Strom, Julia Klawohn & Norbert KathmannDepartment of Psychiatric Phenomics and Genomics (IPPG), Ludwig-Maximilians University Munich, Munich, GermanyNora I. Strom & Manuel MattheisenCentre for Psychiatry Research, Department of Clinical Neuroscience, Karolinska Institutet and Stockholm Health Services, Region Stockholm, Stockholm, SwedenNora I. Strom, Julia Bäckman, Elles J. De Schipper, Christian Rück, David Mataix-Cols & James J. CrowleyDepartment of Biomedicine, Aarhus University, Aarhus, DenmarkNora I. Strom, Jakob Grove, Anders D. Børglum & Manuel MattheisenUniversity Hospital of Psychiatry and Psychotherapy, University of Bern, Bern, SwitzerlandNora I. StromDepartment of Mental Health and Neuroscience, Translational Neurogenomics, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, AustraliaZachary F. GerringDepartment of Population Health and Immunity, Healthy Development and Ageing, Walter and Eliza Hall Institute of Medical Research, Melbourne, Victoria, AustraliaZachary F. GerringDepartment of Psychiatry, Human Genetics, Yale University, New Haven, CT, USAMarco GalimbertiVA Connecticut Healthcare System, West Haven, CT, USAMarco GalimbertiDepartment of Center for Genomic Medicine, Psychiatric and Neurodevelopmental Genetics Unit, Massachusetts General Hospital, Boston, MA, USADongmei YuStanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USADongmei YuDepartment of Genetics, University of North Carolina at Chapel Hill, Chapel Hill, NC, USAMatthew W. Halvorsen & James J. CrowleyDepartment of Psychiatry, Amsterdam University Medical Centers, University of Amsterdam, Amsterdam, the NetherlandsAbdel AbdellaouiCenter for Research in Molecular Medicine and Chronic Diseases (CiMUS), Genomics and Bioinformatics, University of Santiago de Compostela, Santiago de Compostela, SpainCristina Rodriguez-FontenlaGrupo de Medicina Xenómica, Genetics, Instituto de Investigación Sanitaria de Santiago de Compostela (FIDIS), Santiago de Compostela, SpainCristina Rodriguez-FontenlaVanderbilt Genetics Institute, Vanderbilt University, Nashville, TN, USAJulia M. SealockDepartment of Psychiatry and Behavioral Sciences, SUNY Downstate Health Sciences University, Brooklyn, NY, USATim BigdeliVA NY Harbor Healthcare System, Brooklyn, NY, USATim BigdeliSocial, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, Psychology and Neuroscience, King’s College London, London, UKJonathan R. ColemanNational Institute for Health and Care Research Maudsley Biomedical Research Centre, South London and Maudsley NHS Trust, London, UKJonathan R. ColemanDepartment of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USABehrang Mahjani, Michael Breen, Joseph D. Buxbaum, Magdalena Janecka, Laura G. Sloofman, Sven Sandin & Dorothy E. GriceDepartment of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, SwedenBehrang Mahjani, Christina Hultman, Nancy L. Pedersen, Cynthia M. Bulik, Mikael Landén & Sven SandinMental Health and Neuroscience Program, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, AustraliaJackson G. ThorpFaculty of Medicine, School of Biomedical Sciences, University of Queensland, Brisbane, Queensland, AustraliaJackson G. ThorpDepartment of Psychiatry and Psychotherapy, University Hospital Bonn, Bonn, GermanyKatharina Bey, Maureen Mulhern & Michael WagnerDepartment of Neurosciences and Mental Health, Hospital for Sick Children, Toronto, Ontario, CanadaChristie L. BurtonDepartment of Psychiatry, Brain University Medical Center Utrecht, Utrecht, the NetherlandsJurjen J. LuykxSecond Opinion Outpatient Clinic, GGNet, Warnsveld, the NetherlandsJurjen J. LuykxMolecular Brain Science Department, Campbell Family Mental Health Research Institute, Centre for Addiction and Mental Health, Toronto, Ontario, CanadaGwyneth ZaiDepartment of Psychiatry, University of Toronto, Toronto, Ontario, CanadaGwyneth Zai, James Kennedy, Margaret A. Richter & Paul SandorPsychiatric Genetics Unit, Group of Psychiatry, Mental Health and Addiction, Vall d’Hebron Research Institute, Universitat Autònoma de Barcelona, Barcelona, SpainSilvia Alemany, Judit Cabana-Dominguez, María Soler Artigas & Marta RibasésDepartment of Mental Health, Hospital Universitari Vall d’Hebron, Barcelona, SpainSilvia Alemany, Judit Cabana-Dominguez, María Soler Artigas & Marta RibasésBiomedical Network Research Centre on Mental Health (CIBERSAM), Madrid, SpainSilvia Alemany, Judit Cabana-Dominguez, María Soler Artigas & Marta RibasésObsessive–Compulsive Disorder Institute, McLean Hospital, Belmont, MA, USAChristine Andre, Brian P. Brennan, Jesse Crosby, Martha J. Falkenstein, Lauryn Garner, Christina Gironda, Eric Jenike, Kara Kelley, Brittany Mathes, Sriramya Potluri & Eric TifftDepartment of Psychiatry and Behavioural Neurosciences, McMaster University, Hamilton, Hamilton, Ontario, CanadaKathleen D. AsklandLaboratory of Neuropsychiatry, IRCCS Santa Lucia Foundation, Rome, ItalyNerisa Banaj, Valentina Ciullo, Fabrizio Piras & Gianfranco SpallettaDepartment of Health Sciences, University of Milano, Milano, ItalyCristina BarlassinaDepartment of Child and Adolescent Psychiatry, Aarhus University Hospital, Psychiatry, Aarhus, DenmarkJudith Becker NissenInstitute of Clinical Medicine, Health, Aarhus University, Aarhus, DenmarkJudith Becker NissenDepartment of Psychiatry and Behavioral Sciences, General Hospital Psychiatry, Johns Hopkins University School of Medicine, Baltimore, MD, USAO. Joseph BienvenuDepartments of Roy J. and Lucille A. Carver College of Medicine, University of Iowa, Iowa City, IA, USADonald BlackDepartment of Child Study Center and Psychiatry, Yale University, New Haven, CT, USAMichael H. BlochDepartment of Research and Innovation, Division of Clinical Neuroscience, Oslo University Hospital, Oslo, NorwaySigrid Børte & Bendik S. WinsvoldDepartment of Public Health and Nursing, Faculty of Medicine and Health Sciences, HUNT Center for Molecular and Clinical Epidemiology, Norwegian University of Science and Technology (NTNU), Trondheim, NorwaySigrid BørteDepartment of Medical Genetics, Oslo University Hospital, Oslo, NorwaySigrid Børte & Srdjan DjurovicDepartment of Child and Adolescent Mental Health, Hospital Sant Joan de Déu, Esplugues de Llobregat, SpainRosa BoschInstituto de Salut Carlos III, Centro de Investigación Biomédica en Red de Salut Mental (CIBERSAM), Madrid, SpainRosa BoschSeaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USAMichael BreenThe Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USAMichael BreenDepartment of Psychiatry, Harvard Medical School, Boston, MA, USABrian P. Brennan, Jesse Crosby, Martha J. Falkenstein, Dan A. Geller & David L. PaulsDepartment of Psychiatry, Universidade de São Paulo, São Paulo, BrazilHelena Brentani & Homero ValladaDepartment for Congenital Disorders, Statens Serum Institut, Copenhagen, DenmarkJonas Bybjerg-Grauholm & David M. HougaardChild Health Research Centre, University of Queensland, Brisbane, Queensland, AustraliaEnda M. ByrnePharmacogenetics Department, Investigaciones Clínicas, Instituto Nacional de Psiquiatría Ramon de la Fuente Muñiz, Mexico City, MéxicoBeatriz CamarenaDepartment of Surgery, Duke University, Durham, NC, USAAdrian CamarenaDepartment of Psychiatry, Icahn School of Medicine at Mount Sinai Hospital, New York, NY, USACarolina CappiDepartment of Psychiatry, University of São Paulo, São Paulo, BrazilCarolina Cappi & Ana G. HounieCiMUS, Genomics and Bioinformatics Group, University of Santiago de Compostela, Santiago de Compostela, SpainAngel CarracedoGalician Foundation of Genomic Medicine, Grupo de Medicina Xenómica, Instituto de Investigación Sanitaria de Santiago (IDIS), Santiago de Compostela, SpainAngel CarracedoMedicina Genómica, Centro de Investigación Biomédica en Red, Enfermedades Raras (CIBERER), Santiago de Compostela, SpainAngel CarracedoPrograma MIND Escoles, Hospital Sant Joan de Déu, Esplugues de Llobregat, SpainMiguel CasasDepartamento de Psiquiatría y Medicina Legal, Universitat Autònoma de Barcelona, Bellaterra, SpainMiguel CasasDepartment of Psychiatry, Ospedale San Raffaele, Milano, ItalyMaria Cristina CavalliniDepartment of Psychiatry, University of Illinois Chicago, Chicago, IL, USAEdwin H. CookDepartment of Psychiatry and Behavioral Sciences, Johns Hopkins Medical Institutions, Baltimore, MD, USABernadette A. CullenDepartment of Mental Health, Bloomberg School of Public Health, Baltimore, MD, USABernadette A. CullenChild and Adolesccent Psychiatry Department, APHP, Paris, FranceRichard DelormeDepartment of Clinical Science, University of Bergen, Bergen, NorwaySrdjan Djurovic, Kira D. Höffler & Stéphanie Le HellardPsychiatry, McLean Hospital OCDI, Harvard Medical School, Belmont, MA, USAJason A. EliasAdult Psychological Services, CBTeam LLC, Lexington, MA, USAJason A. EliasqGenomics (Quantitative Genomics Laboratories), Esplugues de Llobregat, SpainXavier EstivillDepartment of Medical Epidemiology and Biostatistics, Center for Eating Disorders Innovation, Karolinska Institutet, Stockholm, SwedenBengt T. FundinDepartment of Psychiatry, Johns Hopkins University, Baltimore, MD, USAFernando S. GoesDepartment of Psychiatry and Behavioral Sciences, Child and Adolescent Psychiatry, Johns Hopkins University, Baltimore, MD, USAMarco A. GradosThe Lundbeck Foundation Initiative for Integrative Psychiatric Research, iPSYCH, Aarhus, DenmarkJakob GroveCenter for Genomics and Personalized Medicine, Aarhus, DenmarkJakob Grove, David M. Hougaard & Preben B. MortensenBioinformatics Research Centre, Aarhus, DenmarkJakob GroveGenetic Epidemiology Research Branch, National Institute of Mental Health, Bethesda, MD, USAWei GuoDepartment of Biomedicine, University of Bergen, Bergen, NorwayJan Haavik & Olga Therese OusdalBergen Center for Brain Plasticity, Division of Psychiatry, Haukeland University Hospital, Bergen, NorwayJan Haavik & Stéphanie Le HellardDepartment of Psychiatry, Møre og Romsdal Hospital Trust, Molde, NorwayKristen HagenBergen Center for Brain Plasticity, Psychiatry, Haukeland University Hospital, Bergen, NorwayKristen Hagen & Bjarne K. HansenDepartment of Mental Health, Norwegian University for Science and Technology, Trondheim, NorwayKristen Hagen & Gerd KvaleMillion Veteran Program (MVP) Coordinating Center, VA Boston Healthcare System, Boston, MA, USAKelly HarringtonDepartment of Psychiatry, Boston University Chobanian and Avedisian School of Medicine, Boston, MA, USAKelly HarringtonPsychGen Centre for Genetic Epidemiology and Mental Health, Norwegian Institute of Public Health, Oslo, NorwayAlexandra HavdahlNic Waals Institute, Lovisenberg Diaconal Hospital, Oslo, NorwayAlexandra HavdahlBergen Center for Brain Plasticity, Haukeland University Hospital, Bergen, NorwayKira D. HöfflerDepartment of Medical Genetics, Dr. Einar Martens Research Group for Biological Psychiatry, Haukeland University Hospital, Bergen, NorwayKira D. HöfflerDepartment of Medicine, Genetic Medicine, Vanderbilt University Medical Center, Nashville, TN, USADonald HucksDepartment of Child and Adolescent Psychiatry, NYU Grossman School of Medicine, New York, NY, USAMagdalena JaneckaDepartment of Bioinformatics and Integrative Biology, University of Massachusetts Medical School, Worcester, MA, USAElinor K. KarlssonDepartment of Vertebrate Genomics, Broad Institute of MIT and Harvard, Cambridge, MA, USAElinor K. Karlsson & Kerstin Lindblad-TohDepartment of Medicine, MSB Medical School Berlin, Berlin, GermanyJulia KlawohnDepartment of Psychiatry and Behavioral Sciences, Johns Hopkins University, Baltimore, MD, USAJanice E. KrasnowEstonian Genome Centre, Institute of Genomics, University of Tartu, Tartu, EstoniaKristi Krebs, Andres Metspalu, Tõnu Esko, Reedik Mägi, Mari Nelis, Georgi Hudjashov & Lili MilaniDepartment of Biostatistics, T.H. Chan School of Public Health, Boston, MA, USAChristoph LangeDepartment of Medicine, Channing Division of Network Medicine, Brigham and Women’s Hospital, Boston, MA, USAChristoph LangeResearch Department, Grupo Medico Carracci, Mexico City, MexicoNuria LanzagortaDepartment of Psychiatry, Yale University, West Haven, CT, USADaniel LeveyOffice of Research and Development, United States Department of Veterans Affairs, West Haven, CT, USADaniel LeveyDepartment of Medical Biochemistry and Microbiology, Uppsala University, Uppsala, SwedenKerstin Lindblad-TohSciLifeLab, Uppsala University, Uppsala, SwedenKerstin Lindblad-TohDepartment of Psychiatry, University of California, Irvine, Irvine, CA, USAFabio MacciardiDepartment of Mental Health, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USABrion MaherVanderbilt Genetics Institute, Vanderbilt University Medical Center, Nashville, TN, USAEvonne McArthurCOBRE Center for Neuromodulation, Butler Hospital, Providence, RI, USANathaniel McGregorDepartment of Psychiatry and Human Behavior, Alpert Medical School, Brown University, Providence, RI, USANicole C. McLaughlin & Steven A. RasmussenButler Hospital, Providence, RI, USANicole C. McLaughlinDepartment of Psychiatry, Dalhousie University, Halifax, Nova Scotia, CanadaSandra MeierDepartment of Psychiatry, Faculdade de Medicina, Universidade de São Paulo, São Paulo, BrazilEuripedes C. MiguelDepartment of Psychiatry and Behavioral Science, Johns Hopkins University, Baltimore, MD, USAPaul S. Nestadt & Gerald NestadtDepartment of Psychiatry and Biobehavioral Sciences, Division of Child and Adolescent Psychiatry, University of California, Los Angeles, Los Angeles, CA, USAErika L. Nurmi & James T. McCrackenDepartment of Clinical Medicine, Division of Mental Health and Addiction, Oslo University Hospital, Oslo, NorwayKevin S. O’ConnellNORMENT, University of Oslo, Oslo, NorwayKevin S. O’ConnellDepartment of Psychiatry, Massachusetts General Hospital, Boston, MA, USALisa Osiecki & Jeremiah M. ScharfPsychiatric and Neurodevelopmental Genetics Unit, Harvard Medical School, Boston, MA, USALisa OsieckiDepartment of Biomedicine, Haukeland University Hospital, Bergen, NorwayOlga Therese OusdalInstitute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, FinlandTeemu PalviainenDepartment of Clinical Neuroscience and Neurorehabilitation, Neuropsychiatry Laboratory, IRCCS Santa Lucia Foundation, Rome, ItalyFederica PirasDepartment of Genetics, Microbiology and Statistics, IBUB, Universitat de Barcelona, Barcelona, SpainRaquel RabionetCIBERER, Centro de Investigación Biomédica en Red, Madrid, SpainRaquel RabionetDepartment of Human Molecular Genetics, Institut de Recerca Sant Joan de Déu, Barcelona, SpainRaquel RabionetDepartment of Psychiatry and Psychotherapy, Division of Neurogenetics and Molecular Psychiatry, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, GermanyAlfredo RamirezDepartment of Neurodegenerative Diseases and Geriatric Psychiatry, Medical Faculty, University Hospital Bonn, Bonn, GermanyAlfredo RamirezDZNE Bonn, German Center for Neurodegenerative Diseases (DZNE), Bonn, GermanyAlfredo RamirezDepartment of Psychiatry and Glenn Biggs Institute for Alzheimer’s and Neurodegenerative Diseases, UT Health San Antonio, San Antonio, TX, USAAlfredo RamirezCologne Excellence Cluster for Stress Responses in Ageing-associated Diseases (CECAD), University of Cologne, Cologne, GermanyAlfredo RamirezDepartment of Psychiatry, McLean Hospital, Harvard Medical School, Belmont, MA, USAScott RauchDepartment of Mental Disorders, Norwegian Institute of Public Health, New York, NY, USAAbraham ReichenbergDepartment of Psychiatry and Behavioral Sciences, Child and Adolescent, Johns Hopkins University School of Medicine, Baltimore, MD, USAMark A. RiddleDepartment of Psychiatry and Psychotherapy, Charité—Universitätsmedizin, Berlin, GermanyStephan RipkeStanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USAStephan Ripke, Benjamin M. Neale & Jeremiah M. ScharfSite Berlin–Potsdam, German Center for Mental Health (DZPG), Berlin, GermanyStephan RipkeDepartment of Psychiatry, Child and Adolescent Psychiatry Unit (UPIA), Federal University of São Paulo (UNIFESP), São Paulo, BrazilMaria C. RosárioDepartment of Neurosciences and Mental Health, Medical School, Federal University of Bahia, Salvador, BrazilAline S. SampaioDepartment of Psychiatry and Psychotherapy, Faculty of Medicine, University of Freiburg, Medical Center—University of Freiburg, Freiburg, GermanyMiriam A. Schiele & Katharina DomschkeDepartment of Public Health and Nursing, HUNT Center for Molecular and Clinical Epidemiology, Trondheim, NorwayAnne Heidi SkogholtDepartment of Psychiatry, Faculty of Medicine, Locaion VUmc, Amsterdam UMC, University of Amsterdam, Amsterdam, the NetherlandsJan SmitDepartment of Genetics, Microbiology, and Statistics, Faculty of Biology, Universitat de Barcelona, Barcelona, SpainMaría Soler Artigas & Marta RibasésDepartment of Clinical and Molecular Medicine, NTNU, Trondheim, NorwayLaurent F. ThomasDepartment of Public Health and Nursing, K.G. Jebsen Center for Genetic Epidemiology, NTNU, Trondheim, NorwayLaurent F. ThomasBioCore, Bioinformatics Core Facility, NTNU, Trondheim, NorwayLaurent F. ThomasClinic of Laboratory Medicine, St. Olavs Hospital, Trondheim University Hospital, Trondheim, NorwayLaurent F. ThomasDepartment of Molecular Medicine and Surgery, CMM, Karolinska Institutet, Stockholm, SwedenHomero ValladaOCD Institute, Division of Depression and Anxiety, McLean Hospital, Belmont, MA, USANathanial van KirkDepartment of Psychiatry, Harvard Medical School, Belmont, MA, USANathanial van KirkDepartment of Psychiatry, Division of Child and Adolescent Psychiatry, Columbia University, New York, NY, USAJeremy Veenstra-VanderWeeleDepartment of Child and Adolescent Psychiatry, New York State Psychiatric Institute, New York, NY, USAJeremy Veenstra-VanderWeeleDepartment of Psychiatry, Amsterdam UMC, University of Amsterdam, Amsterdam, the NetherlandsNienke N. Vulink & Karin J. VerweijHuman Genetics Institute of New Jersey, Rutgers University, Piscataway, NJ, USAChristopher P. WalkerDepartment of Neurology, the Johns Hopkins University School of Medicine, Baltimore, MD, USAYing WangLaboratory of Clinical Science, NIMH Intramural Research Program, Bethesda, MD, USAJens R. WendlandDepartment of Neurology, Oslo University Hospital, Oslo, NorwayBendik S. WinsvoldHUNT Center for Molecular and Clinical Epidemiology, Department of Public Health and Nursing, Faculty of Medicine and Health Sciences, NTNU, Trondheim, NorwayBendik S. Winsvold, Kristian Hveem & John-Anker ZwartDepartment of Computional Biology, Institute of Life Science, Fudan University, Fudan, ChinaYin YaoDepartment of Psychiatry, Yale School of Medicine, New Haven, CT, USAHang ZhouDepartment of Psychiatry, Veterans Affairs Connecticut Healthcare System, West Haven, CT, USAHang ZhouSection of Biomedical Informatics and Data Science, Yale School of Medicine, New Haven, CT, USAHang ZhouDepartment of Psychiatry, Washington University School of Medicine, Saint Louis, MO, USAArpana AgrawalDepartment of Psychiatry, OCD Clinical and Research Unit, Bellvitge Hospital, Barcelona, SpainPino AlonsoDepartment of Clinical Sciences, University of Barcelona, Barcelona, SpainPino AlonsoDepartment of Psychiatry and Mental Health, Bellvitge Biomedical Research Institute IDIBELLL, Barcelona, SpainPino AlonsoCIBERSAM, Mental Health Network Biomedical Research Center, Madrid, SpainPino AlonsoPsychosomatic Department, Windach Hospital of Neurobehavioural Research and Therapy, Windach, GermanyGötz BerberichDepartment of Psychiatry, Washington University School of Medicine, St Louis, MO, USAKathleen K. BucholzDepartment of Psychiatry, University of North Carolina at Chapel Hill, Chapel Hill, NC, USACynthia M. Bulik & James J. CrowleyDepartment of Nutrition, University of North Carolina at Chapel Hill, Chapel Hill, NC, USACynthia M. BulikDepartments of Rijksuniversiteit Groningen and Psychiatry, University Medical Center Groningen, Groningen, the NetherlandsDanielle CathDepartment of Specialized Training, Drenthe Mental Health Care Institute, Groningen, the NetherlandsDanielle CathDepartment of Psychiatry, Institute of the Royal Netherlands Academy of Arts and Sciences (NIN-KNAW), Amsterdam, the NetherlandsDamiaan DenysDiscipline of Psychiatry and Mental Health, School of Clinical Medicine, UNSW, Sydney, New South Wales, AustraliaValsamma EapenAcademic Unit of Child Psychiatry South-West Sydney, South-West Sydney Clinical School, SWSLHD and Ingham Institute, Sydney, New South Wales, AustraliaValsamma EapenDepartment of Biochemistry and Molecular Biology, Indiana University School of Medicine, Indianapolis, IN, USAHoward EdenbergDepartment of Psychiatry and Psychotherapy, University Hospital LMU, Munich, GermanyPeter FalkaiDepartment of Psychiatry, Max Planck Institute, Munich, GermanyPeter FalkaiChild Study Center and Psychiatry, Yale School of Medicine, New Haven, CT, USAThomas V. FernandezDepartment of Psychiatry, New York State Psychiatric Institute, New York, NY, USAAbby J. FyerDepartment of Psychiatry, Columbia University Medical Center, New York, NY, USAAbby J. FyerDepartment of Medicine, VA Boston Healthcare System, Boston, MA, USAJ. M. GazianoDepartment of Medicine, Mass General Brigham, Boston, MA, USAJ. M. GazianoDepartment of Psychiatry, Child Psychiatry, Massachusetts General Hospital, Boston, MA, USADan A. GellerDepartment of Psychiatry and Psychotherapy, University Medicine Greifswald, Greifswald, GermanyHans J. GrabeCOBRE Center on Neuromodulation, Butler Hospital, Providence, RI, USABenjamin D. GreenbergCenter for Neurorestoration and Neurotechnology, VA Providence Healthcare System, Providence, RI, USABenjamin D. GreenbergDepartment of Psychiatry and Human Behavior, Alpert Medical School, Brown University, Providence, RI, USABenjamin D. GreenbergDepartment of Psychiatry, Child and Adolescent Psychiatry, University of Michigan, Ann Arbor, MI, USAGregory L. HannaBrain and Mind Centre, the University of Sydney, Sydney, New South Wales, AustraliaIan B. HickieDepartment of Medical and Molecular Genetics, Indiana University School of Medicine, Indianapolis, IN, USADongbing LaiInstitute of Neuroscience and Physiology, Department of Psychiatry and Neurochemistry, University of Gothenburg, Gothenburg, SwedenMikael LandénDepartment of Addictology and Psychiatry, Université Paris-Est Créteil, AP-HP, Inserm, Paris, FranceMarion LeboyerDepartment of Psychiatry, SA MRC Unit on Risk and Resilience in Mental Disorders, Stellenbosch University, Stellenbosch, South AfricaChristine LochnerDepartment of Mental Health, Psychiatric Genetics, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, AustraliaSarah E. MedlandNational Centre for Register-based Research, Aarhus University, Aarhus, DenmarkPreben B. MortensenCentre for Integrated Register-based Research, Aarhus University, Aarhus, DenmarkPreben B. MortensenAnalytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USABenjamin M. NealeDepartment of Psychiatry, Psychiatry, Carracci Medical Group, Mexico City, MéxicoHumberto NicoliniPsiquiatría, Instituto Nacional de Medicina Genómica, Mexico City, MéxicoHumberto NicoliniMental Health Center Copenhagen, Copenhagen Research Center for Mental Health, Mental Health Services in the Capital Region of Denmark, Copenhagen, DenmarkMerete NordentoftFaculty of Health and Medical Sciences, Department of Clinical Medicine, University of Copenhagen, Copenhagen, DenmarkMerete NordentoftDepartment of Psychiatry, Rutgers University, Piscataway, NJ, USAMichele Pato & Carlos PatoDepartment of Psychiatry and Biobehavioral Sciences, Child and Adolescent Psychiatry, UCLA Semel Institute for Neuroscience and Human Behavior, Los Angeles, CA, USAJohn PiacentiniDepartment of Psychiatry, Yale University, New Haven, CT, USAChristopher PittengerDepartment of Complex Trait Genetics, Vrije Universiteit Amsterdam, Center for Neurogenomics and Cognitive Research, Amsterdam Neuroscience, Amsterdam, the NetherlandsDanielle PosthumaDepartment of Child and Adolescent Psychiatric, Section Complex Trait Genetics, VU Medical Center Amsterdam, Amsterdam, the NetherlandsDanielle PosthumaDepartment of Psychiatry, Hospital Universitari Vall d’Hebron, Barcelona, SpainJosep Antoni Ramos-QuirogaGroup of Psychiatry, Mental Health and Addictions, Psychiatric Genetics Unit, Vall d’Hebron Research Institute, Barcelona, SpainJosep Antoni Ramos-QuirogaCIBERSAM, Barcelona, SpainJosep Antoni Ramos-QuirogaDepartment of Psychiatry and Forensic Medicine, Universitat Autònoma de Barcelona, Barcelona, SpainJosep Antoni Ramos-QuirogaDepartment of Psychiatry, Sunnybrook Health Sciences Centre, Toronto, Ontario, CanadaMargaret A. RichterDepartment of Psychiatry and Behavioral Neurosciences, Child and Adolescent Psychiatry, Wayne State University School of Medicine, Detroit, MI, USADavid R. RosenbergDepartment of Psychiatry and Psychotherapy, University of Cologne, Cologne, GermanyStephan RuhrmannDepartment of Psychiatry and Behavioral Sciences, the Johns Hopkins University School of Medicine, Baltimore, MD, USAJack F. SamuelsDepartment of Psychiatry and Behavioral Sciences, Division of Neuropsychiatry, Baylor College of Medicine, Houston, TX, USAGianfranco SpallettaDepartment of Psychiatry and Neuroscience Institute, SAMRC Unit on Risk and Resilience in Mental Disorders, University of Cape Town, Cape Town, South AfricaDan J. SteinDepartment of Psychiatry, University of British Columbia, Vancouver, British Columbia, CanadaS. Evelyn StewartBritish Columbia Children’s Hospital Research Institute, Vancouver, British Columbia, CanadaS. Evelyn StewartBritish Columbia Mental Health and Substance Use Services Research Institute, Vancouver, British Columbia, CanadaS. Evelyn StewartDepartment of Psychiatry and Behavioral Sciences, Baylor College of Medicine, Houston, TX, USAEric A. StorchDepartment of Pharmacology, Northwestern University Feinberg School of Medicine, Chicago, IL, USABarbara E. StrangerCenter for Genetic Medicine, Northwestern University Feinberg School of Medicine, Chicago, IL, USABarbara E. StrangerDepartment of Cardiology, University of Milan, Milan, ItalyMaurizio TurielInstitute of Biological Psychiatry, Mental Health Center Sct. Hans, Copenhagen University Hospital, Mental Health Services (RHP), Copenhagen, DenmarkThomas WergeInstitute of Clinical Medicine, University of Copenhagen, Copenhagen, DenmarkThomas WergeInstitute of Clinical Medicine, NORMENT Centre, University of Oslo, Oslo, NorwayOle A. AndreassenDivision of Mental Health and Addiction, Center for Precision Psychiatry, Oslo University Hospital, Oslo, NorwayOle A. AndreassenThe Lundbeck Foundation Initiative for Integrative Psychiatric Research, iPSYCH, Aarhus University, Aarhus, DenmarkAnders D. BørglumCenter for Genomics and Personalized Medicine, Aarhus University, Aarhus, DenmarkAnders D. BørglumDepartment of Child and Adolescent Psychiatry and Psychotherapy, Psychiatric University Hospital Zurich (PUK), University of Zurich, Zürich, SwitzerlandSusanne Walitza & Edna GrünblattNeuroscience Center Zurich, University of Zurich and the ETH Zurich, Zürich, SwitzerlandSusanne Walitza & Edna GrünblattZurich Center for Integrative Human Physiology, University of Zurich, Zürich, SwitzerlandSusanne Walitza & Edna GrünblattHUNT Research Center, Department of Public Health and Nursing, Faculty of Medicine and Health Sciences, NTNU, Trondheim, NorwayKristian HveemDepartment of Research, Innovation and Education, St. Olavs Hospital, Trondheim University Hospital, Trondheim, NorwayKristian HveemCentre for Crisis Psychology, Psychology, University of Bergen, Bergen, NorwayBjarne K. HansenDepartment of Genetic Epidemiology, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, AustraliaNicholas G. MartinPsychosis Research Unit, Psychiatry, Aarhus University Hospital, Aarhus, DenmarkOle MorsDepartment of Mental Disorders, Norwegian Institute of Public Health, Oslo, NorwayTed Reichborn-KjennerudInstitute of Clinical Medicine, University of Oslo, Oslo, NorwayTed Reichborn-KjennerudDepartment of Clinical Psychology, Faculty of Psychology, University of Bergen, Bergen, NorwayGerd KvalePartner Site Berlin, DZPG, Berlin, GermanyKatharina DomschkeDepartment of Neurodegenerative Diseases and Geriatric Psychiatry, University Hospital Bonn, Bonn, GermanyMichael WagnerDZNE, Bonn, GermanyMichael WagnerDepartment of Research and Innovation, Clinical Neuroscience, Oslo University Hospital and University of Oslo, Oslo, NorwayJohn-Anker ZwartSocial, Genetic and Developmental Psychiatric Centre, Institute of Psychiatry, Psychology and Neuroscience, King’s College London, London, UKGerome BreenFIMM, University of Helsinki, Helsinki, FinlandJaakko KaprioDepartment of Psychiatry, the Mathison Centre for Mental Health Research and Education, Cumming School of Medicine, University of Calgary, Calgary, Alberta, CanadaPaul D. ArnoldProgram in Genetics and Genome Biology, Hospital for Sick Children, Toronto, Ontario, CanadaPaul D. ArnoldDepartment of Genetics, Human Genetics Institute of New Jersey, Rutgers University, Piscataway, NJ, USAJames A. KnowlesPsychGen Center for Genetic Epidemiology, Norwegian Institute of Public Health, Oslo, NorwayHelga AskPROMENTA Research Center, Department of Psychology, University of Oslo, Oslo, NorwayHelga AskDepartment of Medicine, Division of Genetic Medicine, Vanderbilt University Medical Center, Nashville, TN, USALea K. DavisDepartment of Psychiatry, Amsterdam UMC location AMC, Amsterdam, the NetherlandsDirk J. SmitPsychiatry Service, VA San Diego Healthcare System, San Diego, CA, USAMurray B. SteinDepartment of Psychiatry and School of Public Health, University of California San Diego, La Jolla, CA, USAMurray B. SteinDepartment of Psychiatry, Human Genetics (Psychiatry), Yale University School of Medicine, West Haven, CT, USAJoel GelernterDepartment of Psychiatry, Veterans Affairs Connecticut Healthcare Center, West Haven, CT, USAJoel GelernterPsychiatry and Genetics Institute, Evelyn F. and William L. Mc Knight Brain Institute, Center for OCD, Anxiety and Related Disorders, University of Florida, Gainesville, FL, USACarol A. MathewsDepartment of Mental Health and Neuroscience, QIMR Berghofer, Brisbane, Queensland, AustraliaEske M. DerksDepartment of Community Health and Epidemiology and Faculty of Computer Science, Dalhousie University, Halifax, Nova Scotia, CanadaManuel Mattheisen23andMe, Inc., Sunnyvale, CA, USAChris GermanAuthorsNora I. StromView author publicationsSearch author on:PubMedGoogle ScholarZachary F. GerringView author publicationsSearch author on:PubMedGoogle ScholarMarco GalimbertiView author publicationsSearch author on:PubMedGoogle ScholarDongmei YuView author publicationsSearch author on:PubMedGoogle ScholarMatthew W. HalvorsenView author publicationsSearch author on:PubMedGoogle ScholarAbdel AbdellaouiView author publicationsSearch author on:PubMedGoogle ScholarCristina Rodriguez-FontenlaView author publicationsSearch author on:PubMedGoogle ScholarJulia M. SealockView author publicationsSearch author on:PubMedGoogle ScholarTim BigdeliView author publicationsSearch author on:PubMedGoogle ScholarJonathan R. ColemanView author publicationsSearch author on:PubMedGoogle ScholarBehrang MahjaniView author publicationsSearch author on:PubMedGoogle ScholarJackson G. ThorpView author publicationsSearch author on:PubMedGoogle ScholarKatharina BeyView author publicationsSearch author on:PubMedGoogle ScholarChristie L. BurtonView author publicationsSearch author on:PubMedGoogle ScholarJurjen J. LuykxView author publicationsSearch author on:PubMedGoogle ScholarGwyneth ZaiView author publicationsSearch author on:PubMedGoogle ScholarSilvia AlemanyView author publicationsSearch author on:PubMedGoogle ScholarChristine AndreView author publicationsSearch author on:PubMedGoogle ScholarKathleen D. AsklandView author publicationsSearch author on:PubMedGoogle ScholarJulia BäckmanView author publicationsSearch author on:PubMedGoogle ScholarNerisa BanajView author publicationsSearch author on:PubMedGoogle ScholarCristina BarlassinaView author publicationsSearch author on:PubMedGoogle ScholarJudith Becker NissenView author publicationsSearch author on:PubMedGoogle ScholarO. Joseph BienvenuView author publicationsSearch author on:PubMedGoogle ScholarDonald BlackView author publicationsSearch author on:PubMedGoogle ScholarMichael H. BlochView author publicationsSearch author on:PubMedGoogle ScholarSigrid BørteView author publicationsSearch author on:PubMedGoogle ScholarRosa BoschView author publicationsSearch author on:PubMedGoogle ScholarMichael BreenView author publicationsSearch author on:PubMedGoogle ScholarBrian P. BrennanView author publicationsSearch author on:PubMedGoogle ScholarHelena BrentaniView author publicationsSearch author on:PubMedGoogle ScholarJoseph D. BuxbaumView author publicationsSearch author on:PubMedGoogle ScholarJonas Bybjerg-GrauholmView author publicationsSearch author on:PubMedGoogle ScholarEnda M. ByrneView author publicationsSearch author on:PubMedGoogle ScholarJudit Cabana-DominguezView author publicationsSearch author on:PubMedGoogle ScholarBeatriz CamarenaView author publicationsSearch author on:PubMedGoogle ScholarAdrian CamarenaView author publicationsSearch author on:PubMedGoogle ScholarCarolina CappiView author publicationsSearch author on:PubMedGoogle ScholarAngel CarracedoView author publicationsSearch author on:PubMedGoogle ScholarMiguel CasasView author publicationsSearch author on:PubMedGoogle ScholarMaria Cristina CavalliniView author publicationsSearch author on:PubMedGoogle ScholarValentina CiulloView author publicationsSearch author on:PubMedGoogle ScholarEdwin H. CookView author publicationsSearch author on:PubMedGoogle ScholarJesse CrosbyView author publicationsSearch author on:PubMedGoogle ScholarBernadette A. CullenView author publicationsSearch author on:PubMedGoogle ScholarElles J. De SchipperView author publicationsSearch author on:PubMedGoogle ScholarRichard DelormeView author publicationsSearch author on:PubMedGoogle ScholarSrdjan DjurovicView author publicationsSearch author on:PubMedGoogle ScholarJason A. EliasView author publicationsSearch author on:PubMedGoogle ScholarXavier EstivillView author publicationsSearch author on:PubMedGoogle ScholarMartha J. FalkensteinView author publicationsSearch author on:PubMedGoogle ScholarBengt T. FundinView author publicationsSearch author on:PubMedGoogle ScholarLauryn GarnerView author publicationsSearch author on:PubMedGoogle ScholarChristina GirondaView author publicationsSearch author on:PubMedGoogle ScholarFernando S. GoesView author publicationsSearch author on:PubMedGoogle ScholarMarco A. GradosView author publicationsSearch author on:PubMedGoogle ScholarJakob GroveView author publicationsSearch author on:PubMedGoogle ScholarWei GuoView author publicationsSearch author on:PubMedGoogle ScholarJan HaavikView author publicationsSearch author on:PubMedGoogle ScholarKristen HagenView author publicationsSearch author on:PubMedGoogle ScholarKelly HarringtonView author publicationsSearch author on:PubMedGoogle ScholarAlexandra HavdahlView author publicationsSearch author on:PubMedGoogle ScholarKira D. HöfflerView author publicationsSearch author on:PubMedGoogle ScholarAna G. HounieView author publicationsSearch author on:PubMedGoogle ScholarDonald HucksView author publicationsSearch author on:PubMedGoogle ScholarChristina HultmanView author publicationsSearch author on:PubMedGoogle ScholarMagdalena JaneckaView author publicationsSearch author on:PubMedGoogle ScholarEric JenikeView author publicationsSearch author on:PubMedGoogle ScholarElinor K. KarlssonView author publicationsSearch author on:PubMedGoogle ScholarKara KelleyView author publicationsSearch author on:PubMedGoogle ScholarJulia KlawohnView author publicationsSearch author on:PubMedGoogle ScholarJanice E. KrasnowView author publicationsSearch author on:PubMedGoogle ScholarKristi KrebsView author publicationsSearch author on:PubMedGoogle ScholarChristoph LangeView author publicationsSearch author on:PubMedGoogle ScholarNuria LanzagortaView author publicationsSearch author on:PubMedGoogle ScholarDaniel LeveyView author publicationsSearch author on:PubMedGoogle ScholarKerstin Lindblad-TohView author publicationsSearch author on:PubMedGoogle ScholarFabio MacciardiView author publicationsSearch author on:PubMedGoogle ScholarBrion MaherView author publicationsSearch author on:PubMedGoogle ScholarBrittany MathesView author publicationsSearch author on:PubMedGoogle ScholarEvonne McArthurView author publicationsSearch author on:PubMedGoogle ScholarNathaniel McGregorView author publicationsSearch author on:PubMedGoogle ScholarNicole C. McLaughlinView author publicationsSearch author on:PubMedGoogle ScholarSandra MeierView author publicationsSearch author on:PubMedGoogle ScholarEuripedes C. MiguelView author publicationsSearch author on:PubMedGoogle ScholarMaureen MulhernView author publicationsSearch author on:PubMedGoogle ScholarPaul S. NestadtView author publicationsSearch author on:PubMedGoogle ScholarErika L. NurmiView author publicationsSearch author on:PubMedGoogle ScholarKevin S. O’ConnellView author publicationsSearch author on:PubMedGoogle ScholarLisa OsieckiView author publicationsSearch author on:PubMedGoogle ScholarOlga Therese OusdalView author publicationsSearch author on:PubMedGoogle ScholarTeemu PalviainenView author publicationsSearch author on:PubMedGoogle ScholarNancy L. PedersenView author publicationsSearch author on:PubMedGoogle ScholarFabrizio PirasView author publicationsSearch author on:PubMedGoogle ScholarFederica PirasView author publicationsSearch author on:PubMedGoogle ScholarSriramya PotluriView author publicationsSearch author on:PubMedGoogle ScholarRaquel RabionetView author publicationsSearch author on:PubMedGoogle ScholarAlfredo RamirezView author publicationsSearch author on:PubMedGoogle ScholarScott RauchView author publicationsSearch author on:PubMedGoogle ScholarAbraham ReichenbergView author publicationsSearch author on:PubMedGoogle ScholarMark A. RiddleView author publicationsSearch author on:PubMedGoogle ScholarStephan RipkeView author publicationsSearch author on:PubMedGoogle ScholarMaria C. RosárioView author publicationsSearch author on:PubMedGoogle ScholarAline S. SampaioView author publicationsSearch author on:PubMedGoogle ScholarMiriam A. SchieleView author publicationsSearch author on:PubMedGoogle ScholarAnne Heidi SkogholtView author publicationsSearch author on:PubMedGoogle ScholarLaura G. SloofmanView author publicationsSearch author on:PubMedGoogle ScholarJan SmitView author publicationsSearch author on:PubMedGoogle ScholarMaría Soler ArtigasView author publicationsSearch author on:PubMedGoogle ScholarLaurent F. ThomasView author publicationsSearch author on:PubMedGoogle ScholarEric TifftView author publicationsSearch author on:PubMedGoogle ScholarHomero ValladaView author publicationsSearch author on:PubMedGoogle ScholarNathanial van KirkView author publicationsSearch author on:PubMedGoogle ScholarJeremy Veenstra-VanderWeeleView author publicationsSearch author on:PubMedGoogle ScholarNienke N. VulinkView author publicationsSearch author on:PubMedGoogle ScholarChristopher P. WalkerView author publicationsSearch author on:PubMedGoogle ScholarYing WangView author publicationsSearch author on:PubMedGoogle ScholarJens R. WendlandView author publicationsSearch author on:PubMedGoogle ScholarBendik S. WinsvoldView author publicationsSearch author on:PubMedGoogle ScholarYin YaoView author publicationsSearch author on:PubMedGoogle ScholarHang ZhouView author publicationsSearch author on:PubMedGoogle ScholarArpana AgrawalView author publicationsSearch author on:PubMedGoogle ScholarPino AlonsoView author publicationsSearch author on:PubMedGoogle ScholarGötz BerberichView author publicationsSearch author on:PubMedGoogle ScholarKathleen K. BucholzView author publicationsSearch author on:PubMedGoogle ScholarCynthia M. BulikView author publicationsSearch author on:PubMedGoogle ScholarDanielle CathView author publicationsSearch author on:PubMedGoogle ScholarDamiaan DenysView author publicationsSearch author on:PubMedGoogle ScholarValsamma EapenView author publicationsSearch author on:PubMedGoogle ScholarHoward EdenbergView author publicationsSearch author on:PubMedGoogle ScholarPeter FalkaiView author publicationsSearch author on:PubMedGoogle ScholarThomas V. FernandezView author publicationsSearch author on:PubMedGoogle ScholarAbby J. FyerView author publicationsSearch author on:PubMedGoogle ScholarJ. M. GazianoView author publicationsSearch author on:PubMedGoogle ScholarDan A. GellerView author publicationsSearch author on:PubMedGoogle ScholarHans J. GrabeView author publicationsSearch author on:PubMedGoogle ScholarBenjamin D. GreenbergView author publicationsSearch author on:PubMedGoogle ScholarGregory L. HannaView author publicationsSearch author on:PubMedGoogle ScholarIan B. HickieView author publicationsSearch author on:PubMedGoogle ScholarDavid M. HougaardView author publicationsSearch author on:PubMedGoogle ScholarNorbert KathmannView author publicationsSearch author on:PubMedGoogle ScholarJames KennedyView author publicationsSearch author on:PubMedGoogle ScholarDongbing LaiView author publicationsSearch author on:PubMedGoogle ScholarMikael LandénView author publicationsSearch author on:PubMedGoogle ScholarStéphanie Le HellardView author publicationsSearch author on:PubMedGoogle ScholarMarion LeboyerView author publicationsSearch author on:PubMedGoogle ScholarChristine LochnerView author publicationsSearch author on:PubMedGoogle ScholarJames T. McCrackenView author publicationsSearch author on:PubMedGoogle ScholarSarah E. MedlandView author publicationsSearch author on:PubMedGoogle ScholarPreben B. MortensenView author publicationsSearch author on:PubMedGoogle ScholarBenjamin M. NealeView author publicationsSearch author on:PubMedGoogle ScholarHumberto NicoliniView author publicationsSearch author on:PubMedGoogle ScholarMerete NordentoftView author publicationsSearch author on:PubMedGoogle ScholarMichele PatoView author publicationsSearch author on:PubMedGoogle ScholarCarlos PatoView author publicationsSearch author on:PubMedGoogle ScholarDavid L. PaulsView author publicationsSearch author on:PubMedGoogle ScholarJohn PiacentiniView author publicationsSearch author on:PubMedGoogle ScholarChristopher PittengerView author publicationsSearch author on:PubMedGoogle ScholarDanielle PosthumaView author publicationsSearch author on:PubMedGoogle ScholarJosep Antoni Ramos-QuirogaView author publicationsSearch author on:PubMedGoogle ScholarSteven A. RasmussenView author publicationsSearch author on:PubMedGoogle ScholarMargaret A. RichterView author publicationsSearch author on:PubMedGoogle ScholarDavid R. RosenbergView author publicationsSearch author on:PubMedGoogle ScholarStephan RuhrmannView author publicationsSearch author on:PubMedGoogle ScholarJack F. SamuelsView author publicationsSearch author on:PubMedGoogle ScholarSven SandinView author publicationsSearch author on:PubMedGoogle ScholarPaul SandorView author publicationsSearch author on:PubMedGoogle ScholarGianfranco SpallettaView author publicationsSearch author on:PubMedGoogle ScholarDan J. SteinView author publicationsSearch author on:PubMedGoogle ScholarS. Evelyn StewartView author publicationsSearch author on:PubMedGoogle ScholarEric A. StorchView author publicationsSearch author on:PubMedGoogle ScholarBarbara E. StrangerView author publicationsSearch author on:PubMedGoogle ScholarMaurizio TurielView author publicationsSearch author on:PubMedGoogle ScholarThomas WergeView author publicationsSearch author on:PubMedGoogle ScholarOle A. AndreassenView author publicationsSearch author on:PubMedGoogle ScholarAnders D. BørglumView author publicationsSearch author on:PubMedGoogle ScholarSusanne WalitzaView author publicationsSearch author on:PubMedGoogle ScholarKristian HveemView author publicationsSearch author on:PubMedGoogle ScholarBjarne K. HansenView author publicationsSearch author on:PubMedGoogle ScholarChristian RückView author publicationsSearch author on:PubMedGoogle ScholarNicholas G. MartinView author publicationsSearch author on:PubMedGoogle ScholarLili MilaniView author publicationsSearch author on:PubMedGoogle ScholarOle MorsView author publicationsSearch author on:PubMedGoogle ScholarTed Reichborn-KjennerudView author publicationsSearch author on:PubMedGoogle ScholarMarta RibasésView author publicationsSearch author on:PubMedGoogle ScholarGerd KvaleView author publicationsSearch author on:PubMedGoogle ScholarDavid Mataix-ColsView author publicationsSearch author on:PubMedGoogle ScholarKatharina DomschkeView author publicationsSearch author on:PubMedGoogle ScholarEdna GrünblattView author publicationsSearch author on:PubMedGoogle ScholarMichael WagnerView author publicationsSearch author on:PubMedGoogle ScholarJohn-Anker ZwartView author publicationsSearch author on:PubMedGoogle ScholarGerome BreenView author publicationsSearch author on:PubMedGoogle ScholarGerald NestadtView author publicationsSearch author on:PubMedGoogle ScholarJaakko KaprioView author publicationsSearch author on:PubMedGoogle ScholarPaul D. ArnoldView author publicationsSearch author on:PubMedGoogle ScholarDorothy E. GriceView author publicationsSearch author on:PubMedGoogle ScholarJames A. KnowlesView author publicationsSearch author on:PubMedGoogle ScholarHelga AskView author publicationsSearch author on:PubMedGoogle ScholarKarin J. VerweijView author publicationsSearch author on:PubMedGoogle ScholarLea K. DavisView author publicationsSearch author on:PubMedGoogle ScholarDirk J. SmitView author publicationsSearch author on:PubMedGoogle ScholarJames J. CrowleyView author publicationsSearch author on:PubMedGoogle ScholarJeremiah M. ScharfView author publicationsSearch author on:PubMedGoogle ScholarMurray B. SteinView author publicationsSearch author on:PubMedGoogle ScholarJoel GelernterView author publicationsSearch author on:PubMedGoogle ScholarCarol A. MathewsView author publicationsSearch author on:PubMedGoogle ScholarEske M. DerksView author publicationsSearch author on:PubMedGoogle ScholarManuel MattheisenView author publicationsSearch author on:PubMedGoogle ScholarConsortiaEstonian BiobankAndres Metspalu, Tõnu Esko, Reedik Mägi, Mari Nelis& Georgi Hudjashov23andMe Inc.Chris GermanContributionsJ.M. Scharf, M.B.S., J. Gelernter, C.A.M., E.M.D. and M. Mattheisen designed the study. N.I.S., Z.F.G., M.G., D.Y. and M.W.H. conducted data analysis. N.I.S., Z.F.G., M.G., D.Y., M.W.H., A. Abdellaoui, C.R.-F., J.M. Sealock, T.B., J.R.C., B. Mahjani, J.G.T., K.B., C.L.B., J.J.L., G.Z., S.A., C.A., K.D.A., J.B., N.B., C.B., J.B.N., O.J.B., D.B., M.H.B., S.B., R.B., M.B., B.P.B., H.B., J.D.B., J.B.-G., E.M.B., J.C.-D., B.C., A. Camarena, C.C., A. Carracedo, M.C., M.C.C., V.C., E.H.C., J.C., B.A.C., E.J.D.S., R.D., S.D., J.A.E., X.E., M.J.F., B.T.F., L.G., C. Gironda, F.S.G., M.A.G., J. Grove, W.G., J.H., K. Hagen, K. Harrington, A.H., K.D.H., A.G.H., D.H., C.H., M.J., E.J., E.K.K., K. Kelley, J. Klawohn, J.E.K., K. Krebs, C. Lange, N.L., D. Levey, K.L.-T., F.M., B. Maher, B. Mathes, E.M., N.M., N.C.M., S.M., E.C.M., M. Mulhem, P.S.N., E.L.N., K.S.O’C., L.O., O.T.O., T.P., N.L.P., Fabrizio Piras, Federica Piras, S.P., R.R., A. Ramirez, S. Rauch, A. Reichenberg, M.A. Riddle, S. Ripke, M.C.R., A.S.S., M.A.S., A.H.S., L.G.S., J.S., M.S.A., L.F.T., E.T., H.V., N.v.K., J.V.-V., N.N.V., C.P.W., Y.W., J.R.W., B.S.W., Y.Y., H.Z., A. Agrawal, P.A., G. Berberich, K.K.B., C.M.B., D.C., D.D., V.E., H.E., P.F., T.V.F., A.J.F., J.M.G., D.A.G., H.J.G., B.D.G., G.L.H., I.B.H., D.M.H., N.K., J. Kennedy, D. Lai, M. Landén, S.L.H., M. Leboyer, C. Lochner, J.T.M., S.E.M., P.B.M., B.M.N., H.N., M. Nordentoft, M.P., C. Pato, D.L.P., J.P., C. Pittenger, D.P., J.A.R.-Q., S.A.R., M.A. Richter, D.R.R., S. Ruhrmann, J.F.S., S.S., P.S., G.S., D.J. Stein, S.E.S., E.A.S., B.E.S., M.T., T.W., O.A.A., A.D.B., S.W., K. Hveem, B.K.H., C.R., N.G.M., L.M., O.M., T.R.-K., M.R., G.K., D.M.-C., K.D., E.G., M.W., J.-A.Z., G. Breen, G.N., J. Kaprio, P.D.A., D.E.G., J.A.K., H.A., K.J.V., L.K.D., D.J. Smit, J.J.C., J.M. Scharf, M.B.S., J. Gelernter, C.A.M., E.M.D. and M. Mattheisen provided samples and/or processed individual cohort data. N.I.S., Z.F.G., M.G., D.Y., M.W.H., J.M. Scharf, M.B.S., J. Gelernter, C.A.M., E.M.D. and M. Mattheisen wrote the paper and formed the core revision group. J.M. Scharf, M.B.S., J. Gelernter, C.A.M., E.M.D. and M. Mattheisen supervised and directed the study. All authors discussed the results and approved the final version of the manuscript.Corresponding authorsCorrespondence toNora I. StromorManuel Mattheisen.Ethics declarationsCompeting interestsC. German is employed by and holds stock or stock options in 23andMe. E.L.N. is on the scientific advisory board for Myriad Genetics and the medical advisory board for the Tourette Association of America and received clinical trial funding from Emalex and Octapharma Pharmaceuticals. J.V.-V. has served on advisory boards or consulted with Roche, Novartis and SynapDx; received research funding from Roche, Novartis, SynapDx, Seaside Therapeutics, Forest, Janssen, Acadia, Yamo and MapLight; and received stipends for editorial work from Wiley and Springer. J.R.W. is a current employee and shareholder of Takeda Pharmaceuticals and a past employee and shareholder of F. Hoffmann-La Roche, Pfizer and Nestle Health Science. C.M.B. reports Pearson (author, royalty recipient). P.F. reports no conflict of interest regarding this study and reports having received financial support and served on the advisory board for Richter, Recordati, Boehringer Ingelheim, Otsuka, Janssen and Lundbeck. H.J.G. has received travel grants and speaker’s honoraria from Fresenius Medical Care, Neuraxpharm, Servier and Janssen-Cilag as well as research funding from Fresenius Medical Care. I.B.H. is the co-director of health and policy at the Brain and Mind Centre of the University of Sydney, Australia. The Brain and Mind Centre operates early-intervention youth services at Camperdown under contract to headspace. I.B.H. has previously led community-based and pharmaceutical industry-supported (Wyeth, Eli Lily, Servier, Pfizer, AstraZeneca, Janssen-Cilag) projects focused on the identification and better management of anxiety and depression. He is the chief scientific advisor to and a 3.2% equity shareholder in InnoWell, which aims to transform mental health services through the use of innovative technologies. B.M.N. is a member of the scientific advisory board at Deep Genomics and Neumora. C. Pittenger consults and/or receives research support from Biohaven Pharmaceuticals, Freedom Biosciences, Ceruvia Lifesciences, Transcend Therapeutics, UCB BioPharma and F-Prime Capital Partners. He owns equity in Alco Therapeutics. These relationships are not related to the current work. D.J. Stein has received consultancy honoraria from Discovery Vitality, Johnson & Johnson, Kanna, L’Oreal, Lundbeck, Orion, Sanofi, Servier, Takeda and Vistagen. E.A.S. reports receiving research funding to his institution from the Ream Foundation, the International OCD Foundation and the NIH. He was formerly a consultant for Brainsway and Biohaven Pharmaceuticals in the past 12 months. He owns stock less than $5,000 in nView–Proem for distribution related to the YBOCS scales. He receives book royalties from Elsevier, Wiley, Oxford, the American Psychological Association, Guildford, Springer, Routledge and Jessica Kingsley. O.A.A. reports being a consultant to Cortechs.ai and Precision Health and speaker honoraria from Otsuka, Lundbeck, Sunovion and Janssen. A.D.B. has received a speaker fee from Lundbeck. D.M.-C. receives royalties for contributing articles to UpToDate and Wolters Kluwer Health and personal fees for editorial work from Elsevier, all unrelated to the current work. M.B.S. has in the past 3 years received consulting income from Acadia Pharmaceuticals, Big Health, Biogen, Bionomics, Boehringer Ingelheim, Clexio, Eisai, EmpowerPharm, Engrail Therapeutics, Janssen, Jazz Pharmaceuticals, NeuroTrauma Sciences, Otsuka, PureTech Health, Sage Therapeutics, Sumitomo Pharma and Roche–Genentech. M.B.S. has stock options in Oxeia Biopharmaceuticals and EpiVario. He has been paid for his editorial work onDepression and Anxiety(editor in chief),Biological Psychiatry(deputy editor) and UpToDate (co-editor in chief for psychiatry). J. Gelernter is paid for editorial work by the journalComplex Psychiatry. P.A. has received funding from Biohaven, Boston Scientific and Medtronic. All other authors report no conflicts of interest.Peer reviewPeer review informationNature Geneticsthanks Sarah Gagliano Taliun and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.Additional informationPublisher’s noteSpringer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.Supplementary informationSupplementary InformationSupplementary Notes 1–10 and Figs. 1–43Reporting SummarySupplementary TablesSupplementary Tables 1–20.Rights and permissionsOpen AccessThis article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visithttp://creativecommons.org/licenses/by/4.0/.Reprints and permissionsAbout this articleCite this articleStrom, N.I., Gerring, Z.F., Galimberti, M.et al.Genome-wide analyses identify 30 loci associated with obsessive–compulsive disorder.Nat Genet57, 1389–1401 (2025). https://doi.org/10.1038/s41588-025-02189-zDownload citationReceived:13 March 2024Accepted:07 April 2025Published:13 May 2025Issue date:June 2025DOI:https://doi.org/10.1038/s41588-025-02189-zShare this articleAnyone you share the following link with will be able to read this content:Get shareable linkSorry, a shareable link is not currently available for this article.Copy shareable link to clipboardProvided by the Springer Nature SharedIt content-sharing initiative Shared genetic loci and causal relations between schizophrenia and obsessive-compulsive disorderArticleOpen access07 April 2023 Genome-wide association study of pediatric obsessive-compulsive traits: shared genetic risk between traits and disorderArticleOpen access02 February 2021 The genetic epidemiology of obsessive-compulsive disorder: a systematic review and meta-analysisArticleOpen access28 June 2023

---
*This content was automatically scraped by Webscraping Agent A*
